<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Overview of Chromatin Accessibility</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Overview of Chromatin Accessibility</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreec50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0targetblankc50b98fa">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0" target="_blank">c50b98f</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreec50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0targetblankc50b98fa"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0" target="_blank">c50b98f</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  rstudio_server_config_midway3.Rmd

Unstaged changes:
    Modified:   analysis/Trans.Rmd
    Deleted:    analysis/rstudio_server_config_midway3.Rmd
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/chromatin.Rmd</code>) and HTML
(<code>docs/chromatin.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0/docs/chromatin.html" target="_blank">c50b98f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/6db7e76c607c54d59f79e2c372613ea124842205/docs/chromatin.html" target="_blank">6db7e76</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b4a686861792d873ddecf9297235da455aad759d/analysis/chromatin.Rmd" target="_blank">b4a6868</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Chromatin review
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/34704ea6cbda8a773a5baca24a2add12393de8eb/docs/chromatin.html" target="_blank">34704ea</a>
</td>
<td>
liliw-w
</td>
<td>
2022-11-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/86b930d0cb530fbb52457ddf1204e34c6a96600d/analysis/chromatin.Rmd" target="_blank">86b930d</a>
</td>
<td>
liliw-w
</td>
<td>
2022-11-21
</td>
<td>
Initial commit on chromatin work
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="basics" class="section level1">
<h1>Basics</h1>
<div id="chromatin-structure" class="section level2">
<h2><a href="https://en.wikipedia.org/wiki/Chromatin">Chromatin
structure</a></h2>
<ul>
<li><p><strong>Chromatin</strong> is a complex of DNA and protein found
in eukaryotic cells.</p>
<ul>
<li><p>The primary function is to package long DNA molecules into more
compact, denser structures.</p></li>
<li><p>Histones prevent DNA from becoming tangled and protect it from
DNA damage. In addition, histones play important roles in gene
regulation and DNA replication.</p></li>
</ul></li>
<li><p>Three levels of chromatin organization &amp; The overall
structure of the chromatin network further depends on the stage of the
cell cycle.</p>
<center>
<div class="float">
<img src="asset/2023/Chromatin_Structures.png" style="width:120.0%"
alt="Chromatin Structures" />
<div class="figcaption">Chromatin Structures</div>
</div>
</center></li>
<li><p>The basic unit is a <strong>nucleosome</strong>: histone wound by
DNA.</p>
<center>
<div class="float">
<img src="asset/2023/Nucleosome_structure.png" style="width:80.0%"
alt="Nucleosome Structures" />
<div class="figcaption">Nucleosome Structures</div>
</div>
</center>
<ul>
<li><p>The primary protein components of chromatin (DNA packaging
protein) are <strong><a
href="https://en.wikipedia.org/wiki/Histone#Histone_modification">histones</a></strong>.</p>
<ul>
<li>There are five major families of histones which are designated H1/H5
(linker histones), H2A, H2B, H3, and H4 (core histones).</li>
</ul></li>
<li><p>An octamer of two sets of four histone cores (Histone H2A,
Histone H2B, Histone H3, and Histone H4) bind to DNA and function as
“anchors” around which the strands are wound.</p></li>
<li><p>The tight wrapping of DNA around histones is to a large degree a
result of electrostatic attraction between the positively charged
histones and negatively charged phosphate backbone of DNA.</p></li>
</ul></li>
</ul>
</div>
<div
id="chromatin-accessibilitycompaction---alter-the-local-chromatin-structure"
class="section level2">
<h2>Chromatin accessibility/compaction - Alter the local chromatin
structure</h2>
<div id="how-is-it-related-with-gene-regulation" class="section level3">
<h3>How is it related with gene regulation?</h3>
<ul>
<li><p>By altering the level of chromatin compaction, e.g. structurally
loose to allow access to RNA and DNA polymerases that transcribe and
replicate the DNA.</p></li>
<li><p>Open chromatin regions (OCRs) are nucleosome-depleted regions
that can be bound by protein factors.</p></li>
</ul>
</div>
<div id="change-chromatin-accessibility-by-histone-modification"
class="section level3">
<h3>Change chromatin accessibility by histone modification</h3>
<ul>
<li><p>The chromatin state, whether it’s open or closed, is controlled
by epigenetic modifications (to the histone, or to the DNA
itself).</p></li>
<li><p>Histone proteins are the basic packers and arrangers of chromatin
and can be modified by various post-translational modifications to
<strong>alter chromatin packing</strong> and their interaction with DNA
and nuclear proteins (histone modification).</p></li>
<li><p>The H3 and H4 histones have long tails protruding from the
nucleosome, which can be covalently modified at several places.
<strong>Modifications of the tail include</strong>,</p>
<ul>
<li><p>methylation</p></li>
<li><p>acetylation</p></li>
<li><p>phosphorylation</p></li>
<li><p>ubiquitination</p></li>
<li><p>SUMOylation</p></li>
<li><p>citrullination</p></li>
<li><p>ADP-ribosylation</p></li>
</ul>
<p>(The core of the histones H2A and H2B can also be modified.)</p></li>
<li><p>Combinations of modifications, known as <strong>histone
marks</strong>. The common nomenclature of histone modifications is,</p>
<ul>
<li><p>The name of the histone (e.g., H3)</p></li>
<li><p>The single-letter amino acid abbreviation (e.g., K for
Lysine)</p></li>
<li><p>the amino acid position in the protein</p></li>
<li><p>The type of modification (Me: methyl, P: phosphate, Ac: acetyl,
Ub: ubiquitin)</p></li>
<li><p>The number of modifications (only Me is known to occur in more
than one copy per residue. 1, 2 or 3 is mono-, di- or
tri-methylation)</p></li>
</ul>
<p>So H3K4me1 denotes the monomethylation of the 4th residue (a lysine)
from the start (i.e., the N-terminal) of the H3 protein.</p></li>
<li><p>Example of histonen modifications</p>
<ul>
<li><p>Methylation of arginine or lysine residues</p>
<p>Methylation can affect how other protein such as transcription
factors interact with the nucleosomes.</p></li>
<li><p>Acetylation of lysine</p>
<p>Lysine acetylation eliminates a positive charge on lysine thereby
weakening the electrostatic attraction between histone and DNA resulting
in partial unwinding of the DNA making it more accessible for gene
expression.</p></li>
</ul></li>
</ul>
</div>
</div>
<div
id="methods-to-profile-chromatin-accessibility-in-open-chromatin-regions"
class="section level2">
<h2>Methods to profile chromatin accessibility in open chromatin
regions</h2>
<ul>
<li><p>chromatin accessibility assays that directly (DNase-seq,
FAIRE-seq and ATAC-seq) isolate accessible locations of a
genome</p></li>
<li><p>in contrast to histone mark ChIP-seq, where such effects must be
inferred by presence or absence of overlapping histone tail
modifications.</p></li>
</ul>
<div id="atac-seq" class="section level3">
<h3>ATAC-seq</h3>
<p>(Assay for Transposable Accessible Chromatin sequencing) uses the Tn5
transposase to integrate (synthetic) transposons into accessible regions
of the genome consequentially highlighting the localisation of
nucleosomes and transcription factors across the genome</p>
</div>
<div id="dnase-seq" class="section level3">
<h3>DNase-seq</h3>
<p>(DNase I hypersensitive sites Sequencing) uses the sensitivity of
accessible regions in the genome to the DNase I enzyme to map open or
accessible regions in the genome.</p>
</div>
<div id="chip-seq-to-assay-histone-modifications"
class="section level3">
<h3><a
href="https://www.youtube.com/watch?v=nkWGmaYRues&amp;t=144s">ChIP-seq</a>
to assay histone modifications</h3>
<ul>
<li><p>Identify the binding sites for any DNA-associated proteins of
interest.</p>
<ul>
<li>Applied to the set of ChIP-able proteins and modifications, such as
transcription factors, polymerases and transcriptional machinery,
structural proteins, protein modifications, and DNA modifications.</li>
</ul></li>
<li><p>It is primarily used to determine how transcription factors and
other chromatin-associated proteins influence phenotype-affecting
mechanisms.</p></li>
</ul>
<p><strong>Types</strong></p>
<ul>
<li><p>TF ChIP-seq: sonication to fragment nucleosome, antibody against
TF</p></li>
<li><p>Histone mark ChIP-seq: sonication or MNase to fragment
nucleosome, antibody against histone modification</p></li>
</ul>
<center>
<div class="float">
<img src="asset/2023/chip_seq_histone_mark.png" style="width:50.0%"
alt="Chromatin Structures" />
<div class="figcaption">Chromatin Structures</div>
</div>
</center>
</div>
</div>
</div>
<div id="chromatin-accesibility-analysis" class="section level1">
<h1>Chromatin accesibility analysis</h1>
<div id="bulk-chromatin-accessibility-data" class="section level2">
<h2>Bulk chromatin accessibility data</h2>
<center>
<div class="float">
<img src="asset/2023/common_task_bulk.png" style="width:80.0%"
alt="Overview of common tasks in the analysis of bulk chromatin accessibility data" />
<div class="figcaption">Overview of common tasks in the analysis of bulk
chromatin accessibility data</div>
</div>
</center>
</div>
<div id="scatac-seq-data" class="section level2">
<h2>scATAC-seq data</h2>
<center>
<div class="float">
<img src="asset/2023/common_task_sc.png" style="width:80.0%"
alt="Overview of common tasks in the analysis of scATAC-seq data" />
<div class="figcaption">Overview of common tasks in the analysis of
scATAC-seq data</div>
</div>
</center>
</div>
</div>
<div id="peak-calling-detect-regions-of-enrichment"
class="section level1">
<h1>Peak calling: detect regions of enrichment</h1>
<center>
<div class="float">
<img src="asset/2023/peak_calling.png" style="width:80.0%"
alt="Chromatin Structures" />
<div class="figcaption">Chromatin Structures</div>
</div>
</center>
<ul>
<li><p>The most widely used tool for peak calling is MACS2, which is
also the default in the ENCODE ATAC-seq pipeline.</p></li>
<li><p>Other general and method-specific peak callers exist, for
example, ZINBA (general), HMMRATAC and Genrich (ATAC-seq), and F-seq and
Hotspot (DNase-seq and ATAC-seq).</p></li>
</ul>
<div id="basics-1" class="section level2">
<h2>Basics</h2>
<ul>
<li><p>3 prime end &amp; 5 prime end</p>
<p>A nucleic acid strand is inherently directional, and the “5 prime
end” has a free hydroxyl (or phosphate) on a 5’ carbon and the “3 prime
end” has a free hydroxyl (or phosphate) on a 3’ carbon.</p></li>
<li><p>tag</p>
<p>a tag sequence read</p></li>
<li><p>tag seq v.s. standard RNA-seq</p>
<ul>
<li><p>Standard RNA-seq: generate sequencing libraries for full-length
mRNAs</p></li>
<li><p>TagSeq: only generate a single fragment from the 3’/5’/paired end
of each transcript (a tag read) and quantifies gene expression by tag
abundance</p></li>
</ul></li>
<li><p>paired-end tag (ditags) &amp; single-end tag</p>
<ul>
<li><p>Paired-end tags (PET) are the short sequences at the 5’ and 3’
ends of a DNA fragment, which are unique enough that they
(theoretically) exist together only once in a genome, therefore making
the sequence of the DNA in between them available upon search (if
full-genome sequence data is available) or upon further sequencing
(since tag sites are unique enough to serve as primer annealing
sites).</p></li>
<li><p>a PET “represents” a larger fragment of genomic</p></li>
<li><p>Since the tag sequences are short, individual PETs are well
suited for next-generation sequencing that has short read lengths and
higher throughput.</p></li>
<li><p>increased specificity when aligning back to the genome compared
to single tags, which involves only one end of the DNA
fragment.</p></li>
</ul></li>
</ul>
</div>
<div id="how-macs2-call-peaks-specifically" class="section level2">
<h2>How MACS2 call peaks specifically?</h2>
<p>[Ref: Shirley Liu, et al. Genome Biology, 2008.]</p>
<div id="steps" class="section level3">
<h3><a
href="https://hbctraining.github.io/Intro-to-ChIPseq/lessons/05_peak_calling_macs.html">Steps</a></h3>
<ul>
<li><p>Determine the site length d</p>
<blockquote>
<p>The distance between the modes of the two peaks in the alignment is
defined as ‘d’ and represents the estimated fragment length.</p>
</blockquote></li>
<li><p>Define sliding windows</p>
<blockquote>
<p>After MACS shifts every tag by d/2, it then slides across the genome
using a window size of 2d to find candidate peaks.</p>
</blockquote></li>
<li><p>Identify windows with significant tag enrichment</p>
<blockquote>
<p>A region is considered to have a significant tag enrichment if the
p-value &lt; 10e-5 (this can be changed from the default). This is a
Poisson distribution p-value based on <span
class="math inline">\(\lambda\)</span>.</p>
</blockquote></li>
<li><p>Merge overlapping peaks with enriched tags</p>
<blockquote>
<p>Overlapping enriched peaks are merged, and each tag position is
extended ‘d’ bases from its center.</p>
</blockquote></li>
<li><p>Correct for multiple testing</p>
<blockquote>
<p>in MACS2, p-values are now corrected for multiple comparison using
the Benjamini-Hochberg correction.</p>
</blockquote></li>
<li><p>MACS2 was originally designed for ChIP–seq data, specific
parameters (for example, –nomodel) need to be used for peak calling in
ATAC-seq or DNase-seq data.</p></li>
<li><p>MACS2 is a model-based algorithm originally designed for ChIP–seq
data analysis. It implements a dynamic Poisson distribution to capture
local background biases in the genome and to effectively detect
peaks</p></li>
</ul>
<pre><code>significantly enriched tags in the ChIP sample relative to the control, 

although a small ChIP peak region often contains too few control tags to robustly estimate the background biases.</code></pre>
<pre><code>ChIP-seq analysis algorithms have specialized in identifying one of two types of enrichment: broad domains (i.e. histone modifications that cover entire gene bodies) or narrow/sharp peaks (i.e. a transcription factor bound to an enhancer).</code></pre>
<ul>
<li><p>control samples: to estimate the background biases</p></li>
<li><p>paired-end</p></li>
<li><p>shift-extend (for single-end tag)</p></li>
</ul>
</div>
<div
id="construction-of-a-common-set-of-features-across-different-samples"
class="section level3">
<h3>Construction of a common set of features across different
samples</h3>
<p>To be able to compare samples with each other in downstream
steps.</p>
<ul>
<li><p>Construct a consensus peak file</p>
<ul>
<li>comprises the set of peaks that are shared between samples, and in
which the start and end location of overlapping peaks are adjusted
(through the so-called merging of peaks) to thus yield one consensus
peak</li>
</ul></li>
<li><p>A predefined set of regions</p></li>
<li><p>A binned genome</p></li>
<li><p>Peak calling is a computational method used to identify areas in
the genome that have been enriched with aligned reads.</p></li>
</ul>
</div>
</div>
</div>
<div id="thoughts" class="section level1">
<h1>Thoughts</h1>
<div id="peak-calling-using-macs2" class="section level2">
<h2>Peak calling using MACS2</h2>
<ol style="list-style-type: decimal">
<li><p>How often that a called peak has multiple sub-peaks?</p></li>
<li><p>Even when sub-peaks exist, how to determine the “sub-regions” to
apply multivariate association?</p></li>
<li><p>What’s the power improvement?</p></li>
</ol>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       compiler_4.2.3    pillar_1.9.0      bslib_0.5.0      
 [5] later_1.3.1       git2r_0.32.0      jquerylib_0.1.4   tools_4.2.3      
 [9] getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7    evaluate_0.21    
[13] lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3   rlang_1.1.1      
[17] cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7        xfun_0.39        
[21] fastmap_1.1.1     httr_1.4.6        stringr_1.5.0     knitr_1.43       
[25] fs_1.6.2          vctrs_0.6.3       sass_0.4.6        rprojroot_2.0.3  
[29] glue_1.6.2        R6_2.5.1          processx_3.8.2    fansi_1.0.4      
[33] rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3    whisker_0.4.1    
[37] ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5   httpuv_1.6.11    
[41] utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
