<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log.html">Log</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Log</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2020-11-05
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges"> <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges" class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of the R Markdown file created these results, you’ll want to first commit it to the Git repo. If you’re still working on the analysis, you can ignore this warning. When you’re finished, you can run <code>wflow_publish</code> to commit the R Markdown file and build the HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree5bc34ec29c863fe52b0cbea6219fd45fae83a381targetblank5bc34eca"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liliw-w/GradLog/tree/5bc34ec29c863fe52b0cbea6219fd45fae83a381" target="_blank">5bc34ec</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree5bc34ec29c863fe52b0cbea6219fd45fae83a381targetblank5bc34eca" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liliw-w/GradLog/tree/5bc34ec29c863fe52b0cbea6219fd45fae83a381" target="_blank">5bc34ec</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    code/.DS_Store
    Ignored:    data/.DS_Store
    Ignored:    output/.DS_Store

Unstaged changes:
    Modified:   analysis/Log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Log.Rmd</code>) and HTML (<code>docs/Log.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bc34ec29c863fe52b0cbea6219fd45fae83a381/analysis/Log.Rmd" target="_blank">5bc34ec</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5bc34ec29c863fe52b0cbea6219fd45fae83a381/docs/Log.html" target="_blank">5bc34ec</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8fe41846bceb86adb07e5260904835194eba55ea/analysis/Log.Rmd" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8fe41846bceb86adb07e5260904835194eba55ea/docs/Log.html" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="nov-02" class="section level2">
<h2>Nov 02</h2>
<div id="why-use-minmodulesize30-in-wgcna-for-tcga" class="section level3">
<h3>Why use <code>minModuleSize=30</code> in WGCNA for TCGA?</h3>
<p>There are three reasons why I set the minimum number of genes in modules to be 30.</p>
<p><strong>1. DGN used this setting.</strong></p>
<p>At first, we looked at the variance explained by the top three PC of each modules under different settings, i.e. <code>minModuleSize=100, 50, 30</code>. We found that under setting of 30, the top three PC’s explained more variance, so we tended to use 30.</p>
<p><img src="1101.fig1.minModuleSize.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101.fig1.minModuleSize">
Past versions of 1101.fig1.minModuleSize.png
</button>
</p>
<div id="fig-1101.fig1.minModuleSize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8fe41846bceb86adb07e5260904835194eba55ea/docs/1101.fig1.minModuleSize.png" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The above plots used all 13634 genes without removing any poorly mapped genes. Next, I removed the poorly mapped genes, with 5258 genes left. I used 30 to cluster these 5258 genes and resulted in 18 modules. We then looked at the explained variance and found nothing seemed wrong. So we decided to use 30 for WGCNA and these 18 modules for downstream analysis.</p>
<p><img src="1101.fig2.minModuleSize.png" width="80%" height="80%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101.fig2.minModuleSize">
Past versions of 1101.fig2.minModuleSize.png
</button>
</p>
<div id="fig-1101.fig2.minModuleSize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8fe41846bceb86adb07e5260904835194eba55ea/docs/1101.fig2.minModuleSize.png" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>2. 30 is the default module size in WGCNA tutorial, and also used by the elife paper.</strong></p>
<p><a href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/Tutorials/FemaleLiver-02-networkConstr-auto.pdf">WGCNA tutorial</a> used 30 as default. <del>The <a href="https://elifesciences.org/articles/58705#s4">elife method</a> also used the default parameters.</del> (Correction: elife method used the default parameters in the R package, where <code>minClusterSize = 20</code>.)</p>
<p><strong>3. I tried other parameters, e.g. 100. 50, which both increase the unclassified genes.</strong></p>
<table>
<thead>
<tr class="header">
<th align="left">minModuleSize</th>
<th align="left">Num.unclassified.genes</th>
<th align="left">Num.modules</th>
<th align="left">Max.num.genes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">100</td>
<td align="left">5247</td>
<td align="left">4</td>
<td align="left">1972</td>
</tr>
<tr class="even">
<td align="left">50</td>
<td align="left">3251</td>
<td align="left">33</td>
<td align="left">397</td>
</tr>
<tr class="odd">
<td align="left">30</td>
<td align="left">2908</td>
<td align="left">57</td>
<td align="left">356</td>
</tr>
<tr class="even">
<td align="left">20</td>
<td align="left">2813</td>
<td align="left">81</td>
<td align="left">326</td>
</tr>
</tbody>
</table>
</div>
<div id="look-into-enrichment-of-tcga-modules." class="section level3">
<h3>Look into enrichment of TCGA modules.</h3>
<p>Here is the <a href="https://biit.cs.ut.ee/gplink/l/5R4sgZawQC">complete enrichment result</a>.</p>
<div id="kegg-pathway-enrichment" class="section level4">
<h4>KEGG pathway enrichment</h4>
<table>
<thead>
<tr class="header">
<th align="left">source</th>
<th align="left">term_name</th>
<th align="left">sig_module</th>
<th align="left">p_adjusted</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">KEGG</td>
<td align="left">ErbB signaling pathway</td>
<td align="left">21</td>
<td align="left">0.008964903</td>
</tr>
<tr class="even">
<td align="left">KEGG</td>
<td align="left">Neuroactive ligand-receptor interaction</td>
<td align="left">3</td>
<td align="left">0.015524997</td>
</tr>
<tr class="odd">
<td align="left">KEGG</td>
<td align="left">Mucin type O-glycan biosynthesis</td>
<td align="left">20</td>
<td align="left">0.016931426</td>
</tr>
<tr class="even">
<td align="left">KEGG</td>
<td align="left">Human papillomavirus infection</td>
<td align="left">2</td>
<td align="left">0.019400355</td>
</tr>
<tr class="odd">
<td align="left">KEGG</td>
<td align="left">Prolactin signaling pathway</td>
<td align="left">2</td>
<td align="left">0.041320205</td>
</tr>
<tr class="even">
<td align="left">KEGG</td>
<td align="left">Cocaine addiction</td>
<td align="left">2;26</td>
<td align="left">0.04269822;0.049447144</td>
</tr>
<tr class="odd">
<td align="left">KEGG</td>
<td align="left">PI3K-Akt signaling pathway</td>
<td align="left">21</td>
<td align="left">0.048894331</td>
</tr>
</tbody>
</table>
<ul>
<li><p><strong>ErbB signaling pathway</strong>: "ErbB family members and some of their ligands are often over-expressed, amplified, or mutated in many forms of cancer, making them important therapeutic targets. For example, researchers have found EGFR to be amplified and/or mutated in gliomas and NSCLC while ErbB2 amplifications are seen in breast, ovarian, bladder, NSCLC, as well as several other tumor types. <a href="https://www.cellsignal.com/contents/science-cst-pathways-kinase-signaling/erbb-her-signaling/pathways-erbb#:~:text=The%20ErbB%20receptors%20signal%20through,making%20them%20important%20therapeutic%20targets.">[source]</a></p></li>
<li><p><strong>Neuroactive ligand-receptor interaction</strong>:</p>
<ul>
<li>Influence on breast cancer MCF-7 cells: “Telocinobufagin might regulate neuroactive ligand-receptor interaction pathway to exert its influences in breast cancer MCF-7 cells.” <a href="https://www-sciencedirect-com.proxy.uchicago.edu/science/article/pii/S0344033818301018">[source]</a></li>
<li>Involved in the regulation of renal cell carcinoma. <a href="https://www.karger.com/Article/Pdf/368504">[source]</a></li>
</ul></li>
<li><p><strong>Mucin type O-glycan biosynthesis</strong>: “Changes in mucin-type O-linked glycosylation are seen in over 90% of breast cancers” <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6103458/">[source]</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1479595/">[source]</a></p></li>
<li><p><strong>Human papillomavirus infection</strong>: “We demonstrated that HPV is associated with breast cancer development, although the role of HPV in breast cancers is still questionable and further research is required to investigate, in more detail, the role of HPV infection in breast cancer.” <a href="https://bmccancer.biomedcentral.com/articles/10.1186/s12885-019-5286-0#Sec17">[source]</a></p></li>
<li><p><strong>Prolactin signaling pathway</strong>: “elevated PRL levels are correlated with increased breast cancer risk and metastasis” “In vitro studies have indicated a role for PRL in breast cancer proliferation and survival.” <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5653461/">[source]</a></p></li>
<li><p><strong>Cocaine addiction</strong>: ???</p></li>
<li><p><strong>PI3K-Akt signaling pathway</strong>: “PI3K/Akt signaling pathway is key in the development of BC” (breast cancer) <a href="https://link.springer.com/article/10.1007%2Fs10555-010-9261-0">[source]</a></p></li>
</ul>
</div>
</div>
<div id="debug-code-for-truncpco." class="section level3">
<h3>Debug code for TruncPCO.</h3>
<p>I was doubting that TruncPCO (using only <span class="math inline">\(\lambda&gt;1\)</span>) code may have bugs because of the following weird qqplot and histogram of pvalues.</p>
<p><img src="1101.fig3.TruncPCO.png" width="70%" height="70%" style="display: block; margin: auto;" /></p>
<p>Rows are qqplot and histogram of pvalues. These pvalues are observed pvalues rather than null pvalues. Left column is for standard PCO (using all <span class="math inline">\(\lambda\)</span>’s); right is for TruncPCO. I thought TruncPCO is weird because a histogram of pvalues shouldn’t be as skewed as the lower right one. This skewed plot means pvalues by TruncPCO are overall smaller than PCO. We thought this may be due to LD among snps, i.e. the tests are not independent. However, even with the LD complication, the histogram should be approximately flat as in the lower left plot, where the tests are not independent either.</p>
<p>So I looked into the code for TruncPCO and see if there are bugs I didn’t notice. <em>(This paragraph explains what the bug is. See the histogram by the debugged code in next paragraph.)</em> PCO uses the minimum pvalue of six tests as the test statistic, i.e. <span class="math display">\[T_{PCO}=min(p_{PCMinP}, p_{PCFisher}, p_{PCLC}, p_{WI}, p_{Wald}, p_{VC})\]</span></p>
<p>Then use a inverse normal distribution to calculate pvalue of <span class="math inline">\(T_{PCO}\)</span>. So the key step is to compute the six pvalues. The bug happens in the step of computing <span class="math inline">\(p_{WI}\)</span>. The test statistic for this pvalus is, <span class="math display">\[T_{WI}=\sum_{k=1}^K {z_k^2} = \sum_{k=1}^K {PC_k^2}=\sum_{k=1}^K {\lambda_k \chi_{1k}^2}\]</span> ,where <span class="math inline">\(K\)</span> is number of all PC’s of expression matrix, <span class="math inline">\(PC_k=u_k^Tz \sim N(0, \lambda_k)\)</span>, <span class="math inline">\(u_k\)</span> is k-th eigenvector with eigenvalue <span class="math inline">\(\lambda_k\)</span>; <span class="math inline">\(z\)</span> is zscore vector for one snp to <span class="math inline">\(K\)</span> genes, <span class="math inline">\(\chi_{1k}\)</span> is chi-square distribution with DF 1. If we only use <span class="math inline">\(\lambda&gt;1\)</span> for TruncPCO, i.e. <span class="math inline">\(T_{WI}^{Trunc}=\sum_{k=1}^{k_0} {PC_k^2}=\sum_{k=1}^{k_0} {\lambda_k \chi_{1k}^2}\)</span>, what I only need to do is to subset <span class="math inline">\(\lambda\)</span>’s and <span class="math inline">\(PC_k\)</span>’s. That is what I did for other five tests. However, in the original PCO package, <span class="math inline">\(T_{WI}\)</span> is calculated using <span class="math inline">\(\sum_{k=1}^K {z_k^2}\)</span> and I should have changed <span class="math inline">\(z_k\)</span>’s to <span class="math inline">\(PC_k\)</span>’s. But I missed this. So, <span class="math inline">\(T_{WI}\)</span> actually used all <span class="math inline">\(K\)</span> <span class="math inline">\(z_k\)</span>’s, and it’s larger than the real <span class="math inline">\(T_{WI}^{Trunc}\)</span>, therefore the pvalue is smaller than the real <span class="math inline">\(p_{WI}^{Trunc}\)</span>. That is why in the above plot, TruncPCO has many small pvalues and its histogram skews to the left.</p>
<p>So I used the debuged code and re-computed pvalues for the same (module, chr). The result is in the third column.</p>
<p><img src="1101.fig4.TruncPCO.png" style="display: block; margin: auto;" /></p>
<p>The histogram became approximately flat. For this (module13, chr19), there seems to be some signals. Next, I give another example (module, chr), where there seems no signals.</p>
<p><img src="1101.fig5.TruncPCO.png" style="display: block; margin: auto;" /></p>
<p>The histogram by the debugged TruncPCO is still almost flat in this cese, which makes sense.</p>
<p>Now I have the updated pvalues. Next, I rerun TruncPCO for the null zscore, and use the null p’s to adjust for these updated pvalues. Then see how many snps are significant (<span class="math inline">\(p_{adj}&lt;\frac{0.05}{18\times22}\)</span>). I do this because I want to see how are the signals by this debugged TruncPCO compared with those by the previous TruncPCO. For now, I checked for the above two (module, chr) and the signals are almost the same.</p>
<p>From now on, I will use this correct version of TruncPCO to compute p’s for other datasets and update p’s for datasets I’ve analyzed.</p>
</div>
<div id="use-tensorqtl-do-more-permutation" class="section level3">
<h3>Use tensorQTL do more permutation?</h3>
<p>So far, the way we use to adjust p is to permute samples once, calculate null z, then null p, then using this null p distribution to adjust the observe p’s. This one-time permutation procedure should work. We do the permutation once because multiple permutation is computation-time heavy. So can we use tensorQTL to make permutation faster?</p>
<p>As far as I understand, tensorQTL is like FastQTL, which does fast permutation to provide the adjusted minimum pvalues for each gene. This is gene-level adjustment, not the snp-level adjustment as we want. So we can’t have more faster permutations by tensorQTL.</p>
</div>
<div id="do-another-round-of-permutation." class="section level3">
<h3>Do another round of permutation.</h3>
<p>To see how stable the signals are, I do another round of permutation for the above two (module, chr). Here I use TruncPCO for pvalues and empirical null for p adjustment. The following table gives the results.</p>
<table>
<thead>
<tr class="header">
<th align="left">example</th>
<th align="right">oldTruncPCO.perm.1</th>
<th align="right">TruncPCO.perm.1</th>
<th align="right">TruncPCO.perm.2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">(module13,chr19)</td>
<td align="right">157</td>
<td align="right">159</td>
<td align="right">155</td>
</tr>
<tr class="even">
<td align="left">(module17,chr15)</td>
<td align="right">1</td>
<td align="right">0</td>
<td align="right">1</td>
</tr>
</tbody>
</table>
</div>
<div id="notation-table" class="section level3">
<h3>Notation table</h3>
<p>To make it clear, the following table is for notations of different methods.</p>
<table>
<thead>
<tr class="header">
<th align="left">data</th>
<th align="left">PCO</th>
<th align="left">TruncPCO.old</th>
<th align="left">TruncPCO</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">observe</td>
<td align="left">p.PCO</td>
<td align="left">NA</td>
<td align="left">p.TruncPCO</td>
</tr>
<tr class="even">
<td align="left">null</td>
<td align="left">p.null.PCO</td>
<td align="left">NA</td>
<td align="left">p.null.TruncPCO</td>
</tr>
</tbody>
</table>
<ul>
<li>“data” is for the data used. “observe” means pvalues are calculated from the observed data. “null” means pvalues are computed from permutation data. This is needed because we need to adjust pvalues in the downstream FDR analysis using the empirical null distribution of pvalues.
<ul>
<li>We use empirical null because snps are in LD, so the tests are not independent and therefore the null distribution of p is not necessarily uniform.</li>
<li>Why not using BH? BH assumes independent tests and uniform null.</li>
<li>Why not using Bonferroni? Conservative for snp-level FDR correction, particularly for the trans snp-level FDR.</li>
<li><del>Approaches to deal with the multiple testing in QTL studies, cis and trans. (???)</del></li>
</ul></li>
<li>“PCO” is the standard PCO using all <span class="math inline">\(\lambda\)</span>’s. “TruncPCO.old” is PCO using only <span class="math inline">\(\lambda&gt;1\)</span>. This is the method used for all results obtained before 11/02/2020. I added “.old” because the code of this TruncPCO has bugs. “TruncPCO” uses the debugded code.</li>
</ul>
</div>
<div id="signals-we-have-so-far." class="section level3">
<h3>Signals we have so far.</h3>
<p><img src="1101.fig6.signals.png" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.6

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] workflowr_1.6.2 Rcpp_1.0.4      rprojroot_1.3-2 digest_0.6.25  
 [5] later_1.0.0     R6_2.4.1        backports_1.1.5 git2r_0.27.1   
 [9] magrittr_1.5    evaluate_0.14   highr_0.8       stringi_1.4.6  
[13] rlang_0.4.7     fs_1.3.2        promises_1.1.0  whisker_0.4    
[17] rmarkdown_2.1   tools_3.6.3     stringr_1.4.0   glue_1.3.2     
[21] httpuv_1.5.2    xfun_0.12       yaml_2.2.1      compiler_3.6.3 
[25] htmltools_0.4.0 knitr_1.28     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
