<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2021</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2021</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree37d45d6d8b462485ccb203e9190ef29db5a8d493targetblank37d45d6a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/37d45d6d8b462485ccb203e9190ef29db5a8d493" target="_blank">37d45d6</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree37d45d6d8b462485ccb203e9190ef29db5a8d493targetblank37d45d6a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/37d45d6d8b462485ccb203e9190ef29db5a8d493" target="_blank">37d45d6</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  rstudio_server_config_midway3.Rmd

Unstaged changes:
    Modified:   analysis/Trans.Rmd
    Deleted:    analysis/rstudio_server_config_midway3.Rmd
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2021.Rmd</code>) and HTML
(<code>docs/Log2021.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ee589703409e38f18cd760bf00a45681c72df3b6/docs/Log2021.html" target="_blank">ee58970</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0/docs/Log2021.html" target="_blank">c50b98f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/dacdae72dc64e4e22ad915006c0cb24a5c938cc1/docs/Log2021.html" target="_blank">dacdae7</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c82dd26f86f5ee6780f4110703ac59beb57fa72c/docs/Log2021.html" target="_blank">c82dd26</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9e1307d0b2645a00c8945ad2ec73b6239053f9ac/analysis/Log2021.Rmd" target="_blank">9e1307d</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
minor edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/da2f8892d3e0e30c6fbaa3e10cb3ec61e9f48cfd/docs/Log2021.html" target="_blank">da2f889</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f855bc7173536cff9aa681d2e66e46844fe23a38/analysis/Log2021.Rmd" target="_blank">f855bc7</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
minor edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e26f3bc3dc98faa86b954ddd5259b6a8c0d90f13/docs/Log2021.html" target="_blank">e26f3bc</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5765087f435765a803ed3257f94d5e5acebb49bb/analysis/Log2021.Rmd" target="_blank">5765087</a>
</td>
<td>
llw
</td>
<td>
2022-01-16
</td>
<td>
Add coloc analysis for trait MS
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/97b4ca1336ea92e396991f1569177c14360eb07c/docs/Log2021.html" target="_blank">97b4ca1</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f6c2add8784cd0d07374cb04ffcc6d391bd0971f/docs/Log2021.html" target="_blank">f6c2add</a>
</td>
<td>
llw
</td>
<td>
2021-12-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/74b31a96ab38989afd3765d7f878f0a2615f0b8a/analysis/Log2021.Rmd" target="_blank">74b31a9</a>
</td>
<td>
llw
</td>
<td>
2021-12-22
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/087078dcbf0b488112f547e5c50bd5c7655fbf14/docs/Log2021.html" target="_blank">087078d</a>
</td>
<td>
llw
</td>
<td>
2021-12-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/562e3b262564c0a15f0dffdbaf04cf1cff8c977e/analysis/Log2021.Rmd" target="_blank">562e3b2</a>
</td>
<td>
llw
</td>
<td>
2021-12-12
</td>
<td>
wrong figure name
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/b1ee5d276e501b0414452fc55818ce059a752d68/docs/Log2021.html" target="_blank">b1ee5d2</a>
</td>
<td>
llw
</td>
<td>
2021-12-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cdc581e5e25014ca7b0152b11d15547fd4524102/analysis/Log2021.Rmd" target="_blank">cdc581e</a>
</td>
<td>
llw
</td>
<td>
2021-12-12
</td>
<td>
update progress
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5565fdd73ab08feb41746c72cb0a7ef119b32adf/docs/Log2021.html" target="_blank">5565fdd</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/94a7bb9e36654ec06af3e122a9fcb14e5b22cf8f/analysis/Log2021.Rmd" target="_blank">94a7bb9</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-24
</td>
<td>
add notes
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/91631c920d01f81af3611c92c61cdc365f1a5a91/docs/Log2021.html" target="_blank">91631c9</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/c50e156165e9d5b6476cde458a0c6965b55a66a5/analysis/Log2021.Rmd" target="_blank">c50e156</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
<td>
update log
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/82ed164bf101d1ad9a5a3fdd6d1ceed86c3e0561/docs/Log2021.html" target="_blank">82ed164</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b7769260a045ab7d7952f05cb2d4938b953752c2/analysis/Log2021.Rmd" target="_blank">b776926</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-07
</td>
<td>
Add Sep 07
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/671dc182ed62e67fd811d2ee5de9ff1c53cfb96b/docs/Log2021.html" target="_blank">671dc18</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/60d26b9d6b2c6bd9a4411a0ce838d1089fd96c93/analysis/Log2021.Rmd" target="_blank">60d26b9</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Minor edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/88f3dab8faaa1e33d17122c00183db653c9a74ff/docs/Log2021.html" target="_blank">88f3dab</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87062821ea63571f96cdbd22798a65e5b510d723/analysis/Log2021.Rmd" target="_blank">8706282</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Minor edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/90ef72c8a7ce631d955ba5b01620d6de88206671/docs/Log2021.html" target="_blank">90ef72c</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/786b185f8b0b3379fcb5c81afb08770cbd3718b1/analysis/Log2021.Rmd" target="_blank">786b185</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
<td>
Keep cross mappable genes in the filtering step
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/52d05f092f868e22e3034d1f4b22e31c6d247eff/docs/Log2021.html" target="_blank">52d05f0</a>
</td>
<td>
liliw-w
</td>
<td>
2021-08-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1c75f6fe28c06004bfbe6259798f0c8ddd88d085/analysis/Log2021.Rmd" target="_blank">1c75f6f</a>
</td>
<td>
liliw-w
</td>
<td>
2021-08-17
</td>
<td>
wflow_publish("analysis/Log2021.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/6dac5dc88e79c45e59b9659f0b78e85520f9029d/docs/Log2021.html" target="_blank">6dac5dc</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6edd0eb19f5ac9820f74904a0f7a86167064bc71/analysis/Log2021.Rmd" target="_blank">6edd0eb</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Add July 06
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0894fed478d7262d46e94162f84e6f75b5a064ab/docs/Log2021.html" target="_blank">0894fed</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Update figures location
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/analysis/Log2021.Rmd" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Move figures for Log2021 to asset folder
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/331721bf31d6a3b1b51b174538dd1b2c487b004a/docs/Log2021.html" target="_blank">331721b</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8bef12c1d327e3302b0df742da198670733254b0/analysis/Log2021.Rmd" target="_blank">8bef12c</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
<td>
Add July 08
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f6c37ba2a5136e03f230aa60da69547fe91f9a8d/docs/Log2021.html" target="_blank">f6c37ba</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b79abbfc3a37486b82a1da080a41c5c0f384809/analysis/Log2021.Rmd" target="_blank">1b79abb</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-29
</td>
<td>
Add ‘estimate eQTLGen sigma by indepdent null eQTLGen zscores’
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/15631a852eeb691fda1e40929a43dc15e7fc2320/docs/Log2021.html" target="_blank">15631a8</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/bdc8aeceaab1683e68dde3db74c4469a1ffad757/analysis/Log2021.Rmd" target="_blank">bdc8aec</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-25
</td>
<td>
minor edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ea89d4f1dd9e555fc6993966fcbf6e68376c4d4e/docs/Log2021.html" target="_blank">ea89d4f</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0f607209229988d6a203f2951470251684069e51/analysis/Log2021.Rmd" target="_blank">0f60720</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-25
</td>
<td>
Replication of eQTLGen results in trans-PCO of eQTLGen summary stats
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/25a87c0f0409f1dbb98f615a7393b30c183665b8/docs/Log2021.html" target="_blank">25a87c0</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/4fc3dffcaa851ffd73851b931c99222ae428d814/analysis/Log2021.Rmd" target="_blank">4fc3dff</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-14
</td>
<td>
Add eQTLGen’s pvalues QQplot
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/7ca8ec5664d806720b7e6c4cb076e49dc8f3ad4e/docs/Log2021.html" target="_blank">7ca8ec5</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/170be3805403d7ec3e28149f4faf4100d573f34d/analysis/Log2021.Rmd" target="_blank">170be38</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-11
</td>
<td>
revise ‘comparison with univariate’
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8ab02f2e0ee6e4899f24cb6261229dd57c78021a/docs/Log2021.html" target="_blank">8ab02f2</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/52166c09bceb3f48a7d061b72ad84777ef4b144e/analysis/Log2021.Rmd" target="_blank">52166c0</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-10
</td>
<td>
update Snakemake file with python lines
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/04c44fc61b812b6245d10b6c2d75ce94e0e57646/docs/Log2021.html" target="_blank">04c44fc</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1f02e91fb8206c139bf84a6a9146d941cd3aeb0c/analysis/Log2021.Rmd" target="_blank">1f02e91</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-26
</td>
<td>
edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/096ef88fdffa71c6c1d50ffabb455e646dc58b28/docs/Log2021.html" target="_blank">096ef88</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f509189c9a4d79b7c9a937a0131bc4ebc6d3f33f/analysis/Log2021.Rmd" target="_blank">f509189</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-24
</td>
<td>
edit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/94425cabab6b19a717275404183bbb0608fe781c/docs/Log2021.html" target="_blank">94425ca</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d7396a450ff5774a47f10ad89114cd20a395e542/analysis/Log2021.Rmd" target="_blank">d7396a4</a>
</td>
<td>
liliw-w
</td>
<td>
2021-05-24
</td>
<td>
Add ‘Are trans-eQTLs cis-eQTLs?’
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c54b15bd91dad1459ffe156846cbaa18289dcf23/docs/Log2021.html" target="_blank">c54b15b</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-03-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/93bcb5b70a7b99286667c458824294a15b2ff46d/analysis/Log2021.Rmd" target="_blank">93bcb5b</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-03-08
</td>
<td>
wflow_publish("analysis/Log2021.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c8841009156dfc134b37deb2c1a9955fe7c4973d/docs/Log2021.html" target="_blank">c884100</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-03-01
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/da50130dba82cdeebed3ac2ef40385a43f07dedf/analysis/Log2021.Rmd" target="_blank">da50130</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-03-01
</td>
<td>
wflow_publish("analysis/Log2021.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/40088b2b65bf282c4b52744096f4154154ede4be/docs/Log2021.html" target="_blank">40088b2</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-23
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cbbe31912c59713652e5dd47a717dc0ff7263547/analysis/Log2021.Rmd" target="_blank">cbbe319</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-23
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/power.lambda0.1.png",
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/85072fc9f9f6e1b79da237ed672b71a3427efe4f/docs/Log2021.html" target="_blank">85072fc</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8960692331102d9c1f1dcb7766e23d5bbc783c7b/analysis/Log2021.Rmd" target="_blank">8960692</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-16
</td>
<td>
Update simulation.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5f143be496f7db7241fbab3dc61ba9dda45e195f/docs/Log2021.html" target="_blank">5f143be</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/912aceee499f2c36a3957a2681af9b5d270cc343/analysis/Log2021.Rmd" target="_blank">912acee</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/N.png", "docs/caus.png",
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e15b1db4e9b2c3759724165a42d7bd512586dab6/docs/Log2021.html" target="_blank">e15b1db</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d/analysis/Log2021.Rmd" target="_blank">74c629e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/power.png",
"docs/Muscle_Skeleta.p.funcExplorer.png"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/86b2e826a9533b862b6715d0f7de193374e4a3b3/docs/Log2021.html" target="_blank">86b2e82</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/analysis/Log2021.Rmd" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/modules.png",
"docs/Muscle_Skeleta.p.WGCNA.png",
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bb3c0167c3a5fe978c67b3ee26743b21e08d7514/docs/Log2021.html" target="_blank">bb3c016</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/83d3f1df74b031551273c819b1badc3cad2ddcce/docs/Log2021.html" target="_blank">83d3f1d</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/301726295134b063cc529b09d126e305e44f4b04/analysis/Log2021.Rmd" target="_blank">3017262</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "data/signals.all.txt",
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a753e96616e6107b3903d6520d76e907334055b3/docs/Log2021.html" target="_blank">a753e96</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b7111b226ef2465c1082c1bf829c3c72e56de4d3/analysis/Log2021.Rmd" target="_blank">b7111b2</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/signalsv.s.FDR.png"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1b48368459d0a166d3bfc3723f18a344babcb81c/docs/Log2021.html" target="_blank">1b48368</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9526e7a4db2194d4466bfb704904b23f4e671808/analysis/Log2021.Rmd" target="_blank">9526e7a</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "docs/signalsv.s.FDR.png"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/9b6da5ccad8ca7b6d03a1d25d165d6e69310e006/docs/Log2021.html" target="_blank">9b6da5c</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/16f7005f5912525fa5678a416cabad511d9c0e42/analysis/Log2021.Rmd" target="_blank">16f7005</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c("analysis/Log2021.Rmd", "data/signals.all.txt"))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/82152fd9cf25b5181433540129f1caabec8921f3/docs/Log2021.html" target="_blank">82152fd</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a3644196362b1562516fb85a55f6f3f2103f171e/docs/Log2021.html" target="_blank">a364419</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b8815e3a5cad91b286dd209681a60875e4ea6d2c/analysis/Log2021.Rmd" target="_blank">b8815e3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
docs/nCross.png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0611c355e9eae2bcad4336efdf32a86a777ee6e4/docs/Log2021.html" target="_blank">0611c35</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/11c79eb5f073cdff8f48f909a8c3d1e3974d2b03/analysis/Log2021.Rmd" target="_blank">11c79eb</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Add simulation.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/51465aef79db94aa2c17b98069e03c1e23630d1f/docs/Log2021.html" target="_blank">51465ae</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cc5fc2819c50a235676ebf78877d78ee4499b218/analysis/Log2021.Rmd" target="_blank">cc5fc28</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
wflow_publish("analysis/Log2021.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1f9d60f1b33018a546171bd15d14f3a20120eb57/docs/Log2021.html" target="_blank">1f9d60f</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b4ce27bdf62d804dca1bd3ff81f689620dddf4dd/analysis/Log2021.Rmd" target="_blank">b4ce27b</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Update GTEx.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/51b1a4cc02e4dd63c9a84c02ea8c597e171bc5ca/docs/Log2021.html" target="_blank">51b1a4c</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/23d78aacbfd5ed4ed07e702b4071961aec7928bb/analysis/Log2021.Rmd" target="_blank">23d78aa</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
<td>
wflow_git_commit(c("analysis/_site.yml", "analysis/index.Rmd",
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="dec-20" class="section level2">
<h2>Dec 20</h2>
<div id="what_is_gene_SENP7" class="section level3">
<h3>1. What is gene <em>SENP7</em>?</h3>
<p>In <a href="#coloc-region-between-module-and-immune-trait">earlier
section</a>, we observed that, there are 4 modules that have coloc
regions with the three immune traits crohn’s disease, Inflammaroty bowel
disease, Allergic diseases.</p>
<p>Genes of these modules are mostly zinc finger genes, and enriched in
relevant terms.</p>
<p>The lead SNPs of the coloc regions are close to several genes,
including <em>SENP7</em>.</p>
<p>So we wanted to see if <em>SENP7</em> is close to any zinc finger
genes.</p>
<ul>
<li><p>Located on Chr 3. Protein coding gene.</p></li>
<li><p>Closest to gene <a
href="https://www.ncbi.nlm.nih.gov/gene/?term=ZNF90P1"><em>ZNF90P1</em></a>,
a pseudogene. Its location on <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr3%3A101043118%2D101232085&amp;hgsid=1242828115_b2ZR5XmJBYJAa0EEIja9ez9peZeK">Genome
browser</a>.</p></li>
<li><p>Does <em>SENP7</em> have functions relevant to autoimmune
disease? See <a
href="https://www.nature.com/articles/s41467-018-05321-2">paper</a>.</p>
<p>Inflammasome activity &lt;– <em>NLRP3</em> activation &lt;–
SUMO-mediated regulation (SUMOylation) &lt;– SUMO-specific proteases
<em>SENP7</em> promotes NLRP3 activation</p>
<p>Also <a
href="https://www.cell.com/cell-reports/pdf/S2211-1247(19)31499-8.pdf">paper</a>.</p></li>
</ul>
</div>
</div>
<div id="dec-13" class="section level2">
<h2>Dec 13</h2>
<div id="add-trait-ms-to-coloc-analysis" class="section level3">
<h3>1. Add trait MS to coloc analysis</h3>
<ul>
<li><p>Request summary statistic from <a
href="https://imsgc.net/">link</a>.</p></li>
<li><p>Meta analysis from 15 GWASs.</p></li>
</ul>
<p>I added coloc results of MS to <a
href="#Proportion-of-colocalized-regions-for-immune-traits">earlier
section</a>.</p>
</div>
<div id="coloc-region-between-module-and-immune-trait"
class="section level3">
<h3>2. Look at the colocalized regions between trans and immune
traits</h3>
<div id="coloc-regions-overview" class="section level4">
<h4>coloc regions overview</h4>
<ul>
<li>I specifically looked at the following three traits, as they have a
relatively high proportion of coloc regions from <a
href="#Proportion-of-colocalized-regions-for-immune-traits">earlier
section</a>.</li>
</ul>
<table>
<colgroup>
<col width="38%" />
<col width="24%" />
<col width="23%" />
<col width="12%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">disease</th>
<th align="right">#light green regions</th>
<th align="right">#dark green regions</th>
<th align="right">coloc prop</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">crohn’s disease (cd)</td>
<td align="right">51</td>
<td align="right">20</td>
<td align="right">0.3921569</td>
</tr>
<tr class="even">
<td align="left">Inflammaroty bowel disease (ibd)</td>
<td align="right">48</td>
<td align="right">18</td>
<td align="right">0.3750000</td>
</tr>
<tr class="odd">
<td align="left">Allergy</td>
<td align="right">43</td>
<td align="right">11</td>
<td align="right">0.2558140</td>
</tr>
<tr class="even">
<td align="left">MS</td>
<td align="right">20</td>
<td align="right">1</td>
<td align="right">0.0500000</td>
</tr>
</tbody>
</table>
<p><u>Note: I added module44 on the above table.</u></p>
<p>*light green regions: regions with lead module-QTL
pvalue&lt;1e-8.</p>
<p>*dark green regions: coloc regions with lead module-QTL
pvalue&lt;1e-8.</p>
<ul>
<li>What are those dark green regions in specific?</li>
</ul>
<p>Fig: Colocalized regions summary of crohn’s disease, Inflammaroty
bowel disease, Allergic disases. <strong>Specifically for dark green
regions</strong>, i.e. coloc regions with lead module-QTL
pvalue&lt;1e-8.</p>
<p><img src="asset/1213_cd.coloc.Pval.region.summary.png"
style="width:32.0%" /> <img
src="asset/1213_ibd.coloc.Pval.region.summary.png"
style="width:32.0%" /> <img
src="asset/1213_Allergy.coloc.Pval.region.summary.png"
style="width:32.0%" /></p>
<p><u><strong>For MS, there is only one dark green region,
module44:7:50308527. This region has 69 SNPs, and corresponds to module
44 and chr 7.</strong></u></p>
</div>
<div id="observations" class="section level4">
<h4>Observations</h4>
<p><img src="asset/1213_cd_ibd_Allergy_coloc_info.png"
style="width:30.0%" /> <img
src="asset/1213_cd_ibd_Allergy_coloc_region_overlap.png"
style="width:25.0%" /></p>
<ul>
<li><p>These three traits share very similar coloc regions.</p>
<p>See the above venn diagram. The coloc regions of three traits are
largely overlapped.</p></li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">module</th>
<th align="right">#Genes</th>
<th align="left">Enrichment</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module25</td>
<td align="right">123</td>
<td align="left">some genes are ZNF* genes</td>
</tr>
<tr class="even">
<td align="left">module51</td>
<td align="right">61</td>
<td align="left">almost all genes are ZNF* genes</td>
</tr>
<tr class="odd">
<td align="left">module153</td>
<td align="right">15</td>
<td align="left">no genes are ZNF* genes</td>
</tr>
<tr class="even">
<td align="left">module156</td>
<td align="right">13</td>
<td align="left">12 genes are ZNF* genes</td>
</tr>
<tr class="odd">
<td align="left">module44</td>
<td align="right">70</td>
<td align="left">no genes are ZNF* genes</td>
</tr>
</tbody>
</table>
<p><u>Note: I added module44 on the above table.</u></p>
<ul>
<li><p>Module enrichment, relevant to immune?</p>
<ul>
<li><p>See <a
href="https://biit.cs.ut.ee/gplink/l/VQIAZG_ZQT">enrichment results
here</a>.</p></li>
<li><p>See <a
href="asset/1213_cd_ibd_Allergy_coloc_module_enrich.png">g:profiler
enrichment overview</a>.</p></li>
<li><p>See <a
href="asset/1213_cd_ibd_Allergy_coloc_module_gprofiler_query.txt">g:profiler
enrichment query file</a>.</p></li>
<li><p>See <a
href="asset/1213_cd_ibd_Allergy_coloc_module_david_query.xlsx">David
enrichment query file and gene descriptions</a>.</p></li>
</ul>
<p><strong>Question: These four modules contain many zinc finger
genes.</strong></p>
<ul>
<li><u><strong>For MS, module 44’s contained genes and enrichment result
see <a
href="https://biit.cs.ut.ee/gplink/l/t6CyzBjcS9">here</a>.</strong></u></li>
</ul></li>
<li><p>SNP variants</p>
<ul>
<li>cis-genes?</li>
</ul>
<p>For the 20 lead SNPs, they are, Intron Variant of gene
<em>SENP7</em>, <em>IMPG2</em>, <em>CEP97</em>, Non Coding Transcript
Variant/2KB Upstream Variant of gene <em>SENP7</em>, or 500B Downstream
Variant of gene <em>FAM172BP</em>.</p>
<ul>
<li><p><u><strong>For MS, cis- genes?</strong></u></p>
<p>The lead SNP of region module44:7:50308527 is close to gene <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr7%3A50295000%2D50472798&amp;hgsid=1260587557_latSAw7QoV0gBDVTJRA2kKabpth8"><em>IKZF1</em></a>.</p></li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="nov-29-dec-06" class="section level2">
<h2>Nov 29, Dec 06</h2>
<div id="out-130-regions-of-colocalization" class="section level3">
<h3>0. out 130 regions, % of colocalization</h3>
<p>I updated it to <a
href="#coloc-between-cis--and-trans--signals">earlier section</a> -&gt;
<code>Observations</code> -&gt;
<code>Distribution of coloc regions</code>.</p>
</div>
<div id="other-blood-related-traits-immune-diseases"
class="section level3">
<h3>1. Other blood related traits: immune diseases</h3>
<p>Traits refer to <a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02334-x#MOESM1">Paper</a>.</p>
<p>72 GWAS of immune-related and non-immune traits.</p>
<blockquote>
<p>We compiled a set of 72 well-powered GWAS, including 14 for
autoimmune diseases (11 unique disease types), 36 blood traits, and 22
other traits (Additional File 1: Table S4),</p>
</blockquote>
<blockquote>
<p>72 publicly available GWAS summary statistics for 11 autoimmune
diseases (14 studies).</p>
</blockquote>
<div id="autoimmune-diseases-overview" class="section level4">
<h4>14 autoimmune diseases overview</h4>
<p>See <a href="../docs/asset/1129_72_traits_GWAS.xlsx">file</a>.</p>
<ul>
<li><p>There are 14 studies (datasets) for 11 immune diseases.</p></li>
<li><p>For two studies (MS, ApD), the summary stat is only available
upon request. (Update: we have requested sum stat for MS and added its
coloc results here.)</p></li>
<li><p>For one study (sle), the previous dataset is no longer available.
So I downloaded the updated version (meta- v.s. EUR-).</p></li>
<li><p><strong>Many studies meta-analyze multiple datasets</strong></p>
<ul>
<li>Use meta sum stat for coloc?</li>
<li>Use meta sample sizes of cases and controls?</li>
</ul></li>
</ul>
</div>
<div id="Proportion-of-colocalized-regions-for-immune-traits"
class="section level4">
<h4>coloc proportion</h4>
<p>The following figure is similar to <a
href="#Proportion-of-colocalized-regions-for-29-blood-related-UKBB-traits">earlier
section</a>, which gives regions and proportions of coloc regions.</p>
<div class="figure" style="text-align: center">
<img src="asset/1129_all.traits.coloc.region.summary.png" alt="The proportion of colocalized regions for 11 auto-immune diseases." width="70%" />
<p class="caption">
The proportion of colocalized regions for 11 auto-immune diseases.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1129_all.traits.coloc.region.summary">
Past versions of 1129_all.traits.coloc.region.summary.png
</button>
</p>
<div id="fig-1129_all.traits.coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/69b76c8e4b08c292abc095812021b9d55ac2f74c/docs/asset/1129_all.traits.coloc.region.summary.png" target="_blank">69b76c8</a>
</td>
<td>
llw
</td>
<td>
2021-12-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For a specific trait, the coloc regions’ summary and visualization
are in folder
<code>/scratch/midway2/liliw1/coloc/pmid*_*/plot</code>.</p>
</div>
<div id="observations-1" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>These 11 immune diseases generally have less blue regions (GWAS
regions with minimum QTL <span
class="math inline">\(p&gt;10^{-5}\)</span>).</p></li>
<li><p>Also less green regions (regions with very small module-QTL
pvalues).</p></li>
<li><p>Also lower proportion of coloc regions, out of blue and green
regions. 8 diseases have 0 coloc regions.</p></li>
<li><p>Due to meta-analysis? Cell type complexity?</p></li>
</ul>
</div>
</div>
<div id="mr-under-assumption-of-textvarbeta-on-slack"
class="section level3">
<h3>2. MR under assumption of <span
class="math inline">\(\text{var}{\beta}\)</span> (on slack)</h3>
<div id="procedure" class="section level4">
<h4>Procedure</h4>
<p>For MR R package and code tutorial, see <a
href="#Mendelian-Randomization-MR-among-SNPs-module-trait">earlier
section</a>.</p>
<p>Basically, we will need <span class="math inline">\(\beta\)</span>,
<span class="math inline">\(\text{se}({\beta})\)</span> for both
module-QTLs and GWAS-QTLs. However, we don’t have estiamted <span
class="math inline">\(\beta\)</span> for module-QTLs. What we have is
just <span class="math inline">\(p\)</span>.</p>
<p>Therefore, on slack, assumptions to make TwoSampleMR work,</p>
<ul>
<li><p>(Strong assumption) set <span
class="math inline">\(\beta=z\)</span>, <span
class="math inline">\(\text{se}({\beta})=1\)</span>.</p></li>
<li><p>(Strong assumption) Since we only have <span
class="math inline">\(p\)</span>, we will only have absolute zscores. So
one way to deal with this is to assume <span
class="math inline">\(\beta_{SNP-module}\)</span> and <span
class="math inline">\(\beta_{SNP-GWAS}\)</span> have the same sign (+),
so that we only consider the consistency of effect magnitude.</p></li>
<li><p>We look at each trans-eQTL region separately, rather than combine
all trans-eQTL regions for a module, as what we do here is “like
mediation analysis, SNP affect trait through this gene module”, “rather
than to infer if the module is causal for the trait, the causality
between module and trait”.</p></li>
</ul>
</div>
<div id="results" class="section level4">
<h4>Results</h4>
<p>Here I take trait 30090 (platelet crit) for an example.</p>
<p>MR result in figure <code>pheno30090.region.mr.png</code>.</p>
</div>
<div id="observations-2" class="section level4">
<h4>Observations</h4>
<ul>
<li><p><span class="math inline">\(\beta_{SNP-module}\)</span> is much
larger than <span
class="math inline">\(\beta_{SNP-GWAS}\)</span>.</p></li>
<li><p>Insufficient (1) or only a few (mostly 2, a few 3) independent
SNPs (IV) used for MR, as we consider one region each time and the
region is quite short (100k).</p></li>
</ul>
</div>
</div>
</div>
<div id="nov-23" class="section level2">
<h2>Nov 23</h2>
<div id="are-the-dark-green-signals-eqtlgen-signals"
class="section level3">
<h3>1. Are the dark green signals eQTLGen signals?</h3>
<div id="procedure-1" class="section level4">
<h4>Procedure</h4>
<ul>
<li><p>Check those dark green signals (colocalized regions): region lead
SNP’s module-QTL <span class="math inline">\(p&lt;10^{-8}\)</span>,
<span class="math inline">\(\text{PP4} &gt; 0.75\)</span>, and <span
class="math inline">\(\text{#SNPs in region} \ge 5\)</span>.</p></li>
<li><p>eQTLGen SNPs, cis- &amp; trans- signals.</p></li>
</ul>
</div>
<div id="observations-3" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>trans-</p>
<p>#all regions for all traits: 551 .<br />
#unique regions: 105 .<br />
#unique lead SNPs: 74 .<br />
Out of 74 unique lead SNPs, 9 are included in eQTLGen trans- SNPs.<br />
Out of 9 SNPs, 9 are eQTLGen trans- signals.</p></li>
<li><p>cis-</p>
<p>#all regions for all traits: 551 .<br />
#unique regions: 105 .<br />
#unique lead SNPs: 74 .<br />
Out of 74 unique lead SNPs, 65 are included in eQTLGen cis- SNPs.<br />
Out of 65 SNPs, 61 are eQTLGen cis- signals.<br />
</p></li>
<li><p>trans- &amp; cis-</p>
<p>Out of 74 unique lead SNPs, 65 are included in eQTLGen SNPs.<br />
Out of 65 SNPs, 63 are eQTLGen cis- or trans- signals.</p></li>
</ul>
</div>
</div>
<div id="look-at-module-15" class="section level3">
<h3>2. Look at module 15</h3>
<p>We wanted to look at module15, because in <a
href="#Enrichment-of-colocalized-regions">earlier section</a>, we found
that module15 correspond to multiple regions extensively colocalized
with multiple ukbb traits, but we didn’t find any enrichment of the
genes in module15.</p>
<ul>
<li><p>coloc region &amp; traits</p>
<p>See <a href="../data/module15.coloc.traits.txt">file</a> for detailed
traits and regions.</p></li>
</ul>
<table style="width:100%;">
<caption>The colocalization between the module and traits are from the
following regions (marked by lead SNPs):</caption>
<colgroup>
<col width="9%" />
<col width="3%" />
<col width="7%" />
<col width="40%" />
<col width="35%" />
<col width="3%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Region</th>
<th align="right">traits</th>
<th align="left">cis_genes</th>
<th align="left">link_gene</th>
<th align="left">link_snp</th>
<th align="left">note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module15:17:33867678</td>
<td align="right">5</td>
<td align="left">SLFN12L;SLFN14</td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/gene/100506736;https://www.ncbi.nlm.nih.gov/gene/342618"
class="uri">https://www.ncbi.nlm.nih.gov/gene/100506736;https://www.ncbi.nlm.nih.gov/gene/342618</a></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">module15:19:4502201</td>
<td align="right">7</td>
<td align="left">PLIN4</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/729359"
class="uri">https://www.ncbi.nlm.nih.gov/gene/729359</a></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">module15:2:60725451</td>
<td align="right">1</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">module15:2:8762414</td>
<td align="right">14</td>
<td align="left">ID2</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/3398"
class="uri">https://www.ncbi.nlm.nih.gov/gene/3398</a></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">module15:6:139840693</td>
<td align="right">7</td>
<td align="left">CITED2</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/10370"
class="uri">https://www.ncbi.nlm.nih.gov/gene/10370</a></td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/snp/rs592423?horizontal_tab=true#publications"
class="uri">https://www.ncbi.nlm.nih.gov/snp/rs592423?horizontal_tab=true#publications</a></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">module15:7:50426097</td>
<td align="right">10</td>
<td align="left">IKZF1</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/10320"
class="uri">https://www.ncbi.nlm.nih.gov/gene/10320</a></td>
<td align="left"></td>
<td align="left">intron</td>
</tr>
</tbody>
</table>
<table>
<caption>The coloc traits are from the follwing trait groups, mostly
<code>Red blood cells</code>. (bc we have more Red blood cells
traits?)</caption>
<thead>
<tr class="header">
<th align="left">GWAS Group</th>
<th align="right">#traits</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Platelets</td>
<td align="right">1</td>
</tr>
<tr class="even">
<td align="left">Red blood cells</td>
<td align="right">14</td>
</tr>
<tr class="odd">
<td align="left">White blood cells</td>
<td align="right">2</td>
</tr>
</tbody>
</table>
<ul>
<li><p>module15</p>
<p>191 genes. <a
href="https://biit.cs.ut.ee/gplink/l/4H3dKiAaRm">Enrichment in
g:profiler</a></p></li>
</ul>
<p>??? genes: number, enrichment in David</p>
</div>
<div id="look-at-module-4" class="section level3">
<h3>3. Look at module 4</h3>
<p>We wanted to look at module4, because in <a
href="#Enrichment-of-colocalized-regions">earlier section</a>, we found
that module4 correspond to multiple regions extensively colocalized with
multiple ukbb traits.</p>
<ul>
<li><p>coloc region &amp; traits</p>
<p>See <a href="../data/module4.coloc.traits.txt">file</a> for detailed
traits and regions.</p></li>
</ul>
<table>
<caption>The colocalization between the module and traits are from the
following regions (marked by lead SNPs):</caption>
<colgroup>
<col width="9%" />
<col width="3%" />
<col width="6%" />
<col width="37%" />
<col width="35%" />
<col width="8%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Region</th>
<th align="right">traits</th>
<th align="left">cis_genes</th>
<th align="left">link_gene</th>
<th align="left">link_snp</th>
<th align="left">note</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module4:1:248039451</td>
<td align="right">15</td>
<td align="left">OR2W3;TRIM58</td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/gene/343171;https://www.ncbi.nlm.nih.gov/gene/25893"
class="uri">https://www.ncbi.nlm.nih.gov/gene/343171;https://www.ncbi.nlm.nih.gov/gene/25893</a></td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/snp/rs3811444?horizontal_tab=true#publications"
class="uri">https://www.ncbi.nlm.nih.gov/snp/rs3811444?horizontal_tab=true#publications</a></td>
<td align="left">introns</td>
</tr>
<tr class="even">
<td align="left">module4:3:56848999</td>
<td align="right">3</td>
<td align="left">ARHGEF3</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/50650"
class="uri">https://www.ncbi.nlm.nih.gov/gene/50650</a></td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/snp/rs6445826?horizontal_tab=true"
class="uri">https://www.ncbi.nlm.nih.gov/snp/rs6445826?horizontal_tab=true</a></td>
<td align="left">intron of ARHGEF3</td>
</tr>
<tr class="odd">
<td align="left">module4:3:56901292</td>
<td align="right">4</td>
<td align="left">ARHGEF3</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/50650"
class="uri">https://www.ncbi.nlm.nih.gov/gene/50650</a></td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/snp/rs6445826?horizontal_tab=true"
class="uri">https://www.ncbi.nlm.nih.gov/snp/rs6445826?horizontal_tab=true</a></td>
<td align="left">intron of ARHGEF3</td>
</tr>
<tr class="even">
<td align="left">module4:9:99118445</td>
<td align="right">4</td>
<td align="left">SLC35D2</td>
<td align="left"><a href="https://www.ncbi.nlm.nih.gov/gene/11046"
class="uri">https://www.ncbi.nlm.nih.gov/gene/11046</a></td>
<td align="left"><a
href="https://www.ncbi.nlm.nih.gov/snp/rs11794772?horizontal_tab=true"
class="uri">https://www.ncbi.nlm.nih.gov/snp/rs11794772?horizontal_tab=true</a></td>
<td align="left">intron</td>
</tr>
</tbody>
</table>
<table>
<caption>The coloc traits are from the follwing trait groups, mostly
<code>Red blood cells</code></caption>
<thead>
<tr class="header">
<th align="left">GWAS Group</th>
<th align="right">#traits</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Platelets</td>
<td align="right">4</td>
</tr>
<tr class="even">
<td align="left">Red blood cells</td>
<td align="right">12</td>
</tr>
<tr class="odd">
<td align="left">White blood cells</td>
<td align="right">5</td>
</tr>
</tbody>
</table>
</div>
<div id="coloc-between-cis--and-trans--signals" class="section level3">
<h3>4. coloc between cis- and trans- signals</h3>
<div id="procedure-2" class="section level4">
<h4>Procedure</h4>
<ul>
<li><p>130 regions, based on module-QTLs. With region lead SNP’s
module-QTL <span class="math inline">\(p&lt;10^{-8}\)</span> and with at
least one cis- gene.</p></li>
<li><p>I will test colocalization between module and a gene, if the
genes’ 500k flanking region have (&gt;5) overlapped SNPs with the module
region.</p></li>
<li><p>Update the region by the overlapped SNPs of both module-QTLs and
gene eQTLs.</p></li>
<li><p>Exclude the region if the minimum eQTL <span
class="math inline">\(p&gt;10^{-5}\)</span>.</p></li>
</ul>
</div>
<div id="basics" class="section level4">
<h4>Basics</h4>
<p>130 regions. With region lead SNP’s module-QTL <span
class="math inline">\(p&lt;10^{-8}\)</span> and with at least one cis-
gene.</p>
<div class="figure" style="text-align: center">
<img src="asset/1123_region.cis.gene.png" alt="Fig A: region v.s. #cis genes. Fig B: region v.s. dis" width="70%" />
<p class="caption">
Fig A: region v.s. #cis genes. Fig B: region v.s. dis
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1123_region.cis.gene">
Past versions of 1123_region.cis.gene.png
</button>
</p>
<div id="fig-1123_region.cis.gene" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/eb8733a621537abd39a691427a450badecbb1c0f/docs/asset/1123_region.cis.gene.png" target="_blank">eb8733a</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations-4" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>As a result, there are 429 (region, module, gene) sets
remained.</p></li>
<li><p>Out of which, 89 regions are colocalized (PP.H4.abf &gt;
0.75).</p></li>
<li><p>Distribution of coloc regions</p></li>
</ul>
<p><img src="asset/1123_cis_coloc.region.summary.png" width="50%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1123_cis_coloc.region.summary">
Past versions of 1123_cis_coloc.region.summary.png
</button>
</p>
<div id="fig-1123_cis_coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/eb8733a621537abd39a691427a450badecbb1c0f/docs/asset/1123_cis_coloc.region.summary.png" target="_blank">eb8733a</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/1123_fig_bar_prop_cis.png" alt="Region v.s. #cis genes. Light blue is for the cis genes near the region. Dark blue is for the colocalized cis genes. There are 120 regions in total, out of which 54 have at least one colocalized cis- gene." width="50%" />
<p class="caption">
Region v.s. #cis genes. Light blue is for the cis genes near the region.
Dark blue is for the colocalized cis genes. There are 120 regions in
total, out of which 54 have at least one colocalized cis- gene.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1123_fig_bar_prop_cis">
Past versions of 1123_fig_bar_prop_cis.png
</button>
</p>
<div id="fig-1123_fig_bar_prop_cis" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/eb8733a621537abd39a691427a450badecbb1c0f/docs/asset/1123_fig_bar_prop_cis.png" target="_blank">eb8733a</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Note</strong>: Here there are 120 regions instead of 130
regions, because I excluded the cis- gene regions with <span
class="math inline">\(min(p)&gt;10^{-5}\)</span> before I do the
coloc.</p>
<ul>
<li>Colocalized region visualization</li>
</ul>
<p><img
src="asset/1123_1.reg.module25:3:101011537.gene.SENP7.reg.coloc.png"
style="width:45.0%" /> <img
src="asset/1123_2.reg.module66:12:54685880.gene.NFE2.reg.coloc.png"
style="width:45.0%" /></p>
</div>
</div>
</div>
<div id="nov-15" class="section level2">
<h2>Nov 15</h2>
<div
id="Proportion-of-colocalized-regions-for-29-blood-related-UKBB-traits"
class="section level3">
<h3>1. Proportion of colocalized regions for 29 blood related UKBB
traits</h3>
<div id="procedure-3" class="section level4">
<h4>Procedure</h4>
<ul>
<li><p>Aim: to see how many regions have shared signals for both trait
and module. (PP4 &gt; 0.75)</p></li>
<li><p>I consider two types of regions:</p>
<ol style="list-style-type: decimal">
<li>One type as stated at last week. Based on module-QTLs and GWAS-QTLs
with <span class="math inline">\(p&lt;10^{-5}\)</span>.</li>
<li>Among the above regions, only use regions whose lead SNPs are
module-QTLs signals. (Note, here I used the threshold <span
class="math inline">\(p&lt;10^{-8}\)</span>, as I don’t have
module-signals based on empirical null FDR correction yet.)</li>
</ol></li>
<li><p>Colocalized regions</p>
<p>Defined as regions with <span class="math inline">\(\text{PP4} &gt;
0.75\)</span> and <span class="math inline">\(\text{#SNPs in region} \ge
5\)</span>.</p></li>
</ul>
</div>
<div id="results-1" class="section level4">
<h4>Results</h4>
<div class="figure" style="text-align: center">
<img src="asset/1115_all.traits.coloc.region.summary.png" alt="The proportion of colocalized regions for 29 blood-related UKBB traits." width="80%" />
<p class="caption">
The proportion of colocalized regions for 29 blood-related UKBB traits.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1115_all.traits.coloc.region.summary">
Past versions of 1115_all.traits.coloc.region.summary.png
</button>
</p>
<div id="fig-1115_all.traits.coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1115_all.traits.coloc.region.summary.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>This figure gives (1) the number of regions at the upper panel,
(2) proportion of colocalized regions at the lower panel.</p></li>
<li><p>Blue is for the frist type of regions. Green is for the second
type of reigons, based on module-QTL signals.</p></li>
<li><p>Dark color is for the colocalized regions, while light color is
for the base regions.</p></li>
</ul>
</div>
</div>
<div id="enrichment-of-colocalized-regions" class="section level3">
<h3>2. Enrichment of colocalized regions</h3>
<div id="procedure-4" class="section level4">
<h4>Procedure</h4>
<ul>
<li><p>colocalized regions: region lead SNP’s module-QTL <span
class="math inline">\(p&lt;10^{-8}\)</span>, <span
class="math inline">\(\text{PP4} &gt; 0.75\)</span>, and <span
class="math inline">\(\text{#SNPs in region} \ge 5\)</span>.</p></li>
<li><p>Look at the enrichment of these colocalized regions, and the
relationship with the corresponding trait.</p></li>
</ul>
</div>
<div id="results-2" class="section level4">
<h4>Results</h4>
<div class="figure" style="text-align: center">
<img src="asset/1115_traits.module.coloc.region.png" alt="traits v.s. module v.s. coloc region" width="50%" />
<p class="caption">
traits v.s. module v.s. coloc region
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1115_traits.module.coloc.region">
Past versions of 1115_traits.module.coloc.region.png
</button>
</p>
<div id="fig-1115_traits.module.coloc.region" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1115_traits.module.coloc.region.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>The above figure shows, what are the modules that have
colocalized regions with a trait.</p></li>
<li><p>Similar traits should have similar colocalized module
patterns.</p></li>
</ul>
<table>
<colgroup>
<col width="9%" />
<col width="17%" />
<col width="34%" />
<col width="38%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">Phenocode</th>
<th align="left">Trait Abbreviation</th>
<th align="left">Trait</th>
<th align="left">g:Profiler</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">30000</td>
<td align="left">WBC</td>
<td align="left">White blood cell count</td>
<td align="left"><a href="https://biit.cs.ut.ee/gplink/l/jWgRmfj9SM"
class="uri">https://biit.cs.ut.ee/gplink/l/jWgRmfj9SM</a></td>
</tr>
<tr class="even">
<td align="right">30140</td>
<td align="left">NEUT</td>
<td align="left">Neutrophill count</td>
<td align="left"><a href="https://biit.cs.ut.ee/gplink/l/Hs8Nsb99QV"
class="uri">https://biit.cs.ut.ee/gplink/l/Hs8Nsb99QV</a></td>
</tr>
<tr class="odd">
<td align="right">30200</td>
<td align="left">NEUT_P</td>
<td align="left">Neutrophill percentage</td>
<td align="left"><a href="https://biit.cs.ut.ee/gplink/l/NKfAXoO9Sp"
class="uri">https://biit.cs.ut.ee/gplink/l/NKfAXoO9Sp</a></td>
</tr>
<tr class="even">
<td align="right">30300</td>
<td align="left">HLSR</td>
<td align="left">High light scatter reticulocyte count</td>
<td align="left"><a href="https://biit.cs.ut.ee/gplink/l/VJWuuEEzTF"
class="uri">https://biit.cs.ut.ee/gplink/l/VJWuuEEzTF</a></td>
</tr>
</tbody>
</table>
<ul>
<li><p>This table gives the enrichment of the module genes for each
trait.</p></li>
<li><p>“module4” enriched in platelet activation. Many traits among
these 29 blood-related traits are colocalized with “module4”.</p></li>
<li><p>Although “module15” is extensively colocalized with many traits,
it’s not enriched in any pathway.</p></li>
</ul>
</div>
</div>
<div id="Mendelian-Randomization-MR-among-SNPs-module-trait"
class="section level3">
<h3>3. Mendelian Randomization (MR) among SNPs, module, trait</h3>
<ul>
<li><p>Use R package <a
href="https://mrcieu.github.io/TwoSampleMR/#installation">TwoSampleMR</a>.</p></li>
<li><p><strong>Problem</strong>: no <span class="math inline">\(\beta,
\text{se}(\beta)\)</span> for module as exposure. Exposure file
requirements see <a
href="https://mrcieu.github.io/TwoSampleMR/articles/exposure.html">here</a>.</p></li>
<li><p>Use <a
href="https://www.nature.com/articles/ng.3538#Sec11">SMR</a>? No. As SMR
is also <span class="math inline">\(\beta, \text{se}(\beta)\)</span>
based. In addition to that, it only uses one SNP as instrument
variable.</p>
<ul>
<li>SMR only uses one SNP as instrument variable. “top associated
cis-eQTL”.</li>
<li>Distinguishing pleiotropy from linkage; test for pleiotropic
association.</li>
</ul></li>
<li><p>How people do MR? <a
href="https://www.medrxiv.org/content/10.1101/2021.06.01.21258159v1.full-text">Paper-Yanyu</a></p>
<ol style="list-style-type: decimal">
<li>the instrument variants were selected using LD clumping function
(ld_clump) in the R package ieugwasr</li>
</ol></li>
</ul>
<blockquote>
<p>We used the EUR super-population in 1000 Genomes data (1000 Genomes
Project Consortium, 2015) as the LD reference panel and the data was
downloaded from <a href="http://fileserve.mrcieu.ac.uk/ld/1kg.v3.tgz"
class="uri">http://fileserve.mrcieu.ac.uk/ld/1kg.v3.tgz</a>. The LD
clumping parameters were clump_kb = 10000 and clump_r2 = 0.001. The
p-value parameter (clump_p) in the LD clumping was 10−5 for IDP GWAS and
5 × 10−8 for phenotype GWAS, which gave approximately independent and
significant variant instruments.</p>
</blockquote>
<ol start="2" style="list-style-type: decimal">
<li>The MR analysis was performed using the R package TwoSampleMR</li>
</ol>
<blockquote>
<p>reported the MR results using three MR methods: i) inverse variance
weighted MR (Burgess et al., 2013); ii) median-based estimator: weighted
median (Bowden et al., 2016); iii) MR Egger analysis (Bowden et al.,
2015), which corresponds to mr_ivw, mr_weighted_median, and
mr_egger_regression in TwoSampleMR.</p>
</blockquote>
<ol start="3" style="list-style-type: decimal">
<li>further reported a meta-analyzed p-value summarizing the results of
the three MR tests being performed</li>
</ol>
<blockquote>
<p>The meta-analysis is based on an extension of ACAT method (Liu et
al., 2019) that takes into account the direction of the effects.</p>
</blockquote>
</div>
</div>
<div id="nov-08" class="section level2">
<h2>Nov 08</h2>
<div id="some-statistics" class="section level3">
<h3>Some statistics</h3>
<ul>
<li><p>Out of 6,203,169 SNPs in DGN, 6,133,791 (~98.88%) are also
included in UKBB GWAS SNPs, 69,378 snps are missed.</p></li>
<li><p><del>Among 6,133,791 overlapped SNPs, 5,753,809 (~93.81%) are <a
href="https://pan.ukbb.broadinstitute.org/docs/per-phenotype-files"><code>high_quality</code></a>
variants.</del></p></li>
</ul>
</div>
<div id="coloc-procedure" class="section level3">
<h3>coloc procedure</h3>
<ul>
<li><p>To do coloc, I compare the pvalues of SNPs within a specific
region from each module in DGN and a GWAS trait.</p>
<ul>
<li><p>For DGN, I use the pvalues from the procedure where I didn’t
filter genes based on mappability. There are 166 modules.</p></li>
<li><p>For GWAS, I use the summary statistics of EUR
population.</p></li>
</ul></li>
<li><p>Define Regions</p>
<p>I define coloc regions based on DGN pvalues. For a module, I first
sort the pvalues and pick the SNP with the smallest p. Then collect the
SNPs in flanking regions within <strong>100kb</strong>. These SNPs
consist of the first region. Then use the second smallest p and define a
second region out of the remaining SNPs. I only consider the regions
within which at least one SNP has ** <span
class="math inline">\(p&lt;1e-5\)</span> **.</p></li>
<li><p>SNP filtering</p>
<p>Based on the above preliminary regions, I updated the regions based
on the following criterias:</p>
<ul>
<li>Remove SNPs not included in GWAS.</li>
<li>Remove SNPs without rsid.</li>
<li>Remove SNPs with low_confidence, no high_quality.</li>
<li>Remove duplicated SNPs.</li>
<li>Remove the regions within which all SNPs have GWAS ** <span
class="math inline">\(p&gt;1e-5\)</span> **.</li>
<li>Remove SNPs with GWAS <span
class="math inline">\(se(\beta)=0\)</span>, for <code>coloc</code>
purpose.</li>
</ul></li>
<li><p>Do coloc</p>
<ul>
<li>Use the simple coloc, <code>coloc.abf</code>.</li>
<li>A region with <strong>PP4 &gt; 0.75</strong> is defined as being
colocalized.</li>
</ul></li>
</ul>
</div>
<div id="results-3" class="section level3">
<h3>Results</h3>
<ol style="list-style-type: decimal">
<li>Trait v.s. #regions v.s. #coloc regions</li>
</ol>
<table>
<thead>
<tr class="header">
<th align="right">UKBB trait id</th>
<th align="left">Trait</th>
<th align="right">#regions</th>
<th align="right">#coloc regions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">30080</td>
<td align="left">Platelet count</td>
<td align="right">508</td>
<td align="right">54</td>
</tr>
<tr class="even">
<td align="right">30090</td>
<td align="left">Platelet crit</td>
<td align="right">506</td>
<td align="right">85</td>
</tr>
<tr class="odd">
<td align="right">30100</td>
<td align="left">Mean platelet volume</td>
<td align="right">560</td>
<td align="right">35</td>
</tr>
</tbody>
</table>
<ol start="2" style="list-style-type: decimal">
<li>#SNPs in colocalized regions</li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/1108_coloc.region.summary.png" alt="An example of colocalized region visualization. Trait Platelet count." width="70%" />
<p class="caption">
An example of colocalized region visualization. Trait Platelet count.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1108_coloc.region.summary">
Past versions of 1108_coloc.region.summary.png
</button>
</p>
<div id="fig-1108_coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1108_coloc.region.summary.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/1108_pheno30090.coloc.region.summary.png" alt="An example of colocalized region visualization. Trait Platelet crit" width="70%" />
<p class="caption">
An example of colocalized region visualization. Trait Platelet crit
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1108_pheno30090.coloc.region.summary">
Past versions of 1108_pheno30090.coloc.region.summary.png
</button>
</p>
<div id="fig-1108_pheno30090.coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1108_pheno30090.coloc.region.summary.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/1108_pheno30100.coloc.region.summary.png" alt="An example of colocalized region visualization. Trait Mean platelet volume" width="70%" />
<p class="caption">
An example of colocalized region visualization. Trait Mean platelet
volume
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1108_pheno30100.coloc.region.summary">
Past versions of 1108_pheno30100.coloc.region.summary.png
</button>
</p>
<div id="fig-1108_pheno30100.coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1108_pheno30100.coloc.region.summary.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="3" style="list-style-type: decimal">
<li>Colocalized region visualization by <a
href="https://github.com/boxiangliu/locuscomparer"><code>locuscomparer</code></a></li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/1108_pheno30080.module102:7:50427982.reg.coloc.png" alt="An example of colocalized region visualization." width="50%" />
<p class="caption">
An example of colocalized region visualization.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1108_pheno30080.module102:7:50427982.reg.coloc">
Past versions of 1108_pheno30080.module102:7:50427982.reg.coloc.png
</button>
</p>
<div id="fig-1108_pheno30080.module102:7:50427982.reg.coloc"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1108_pheno30080.module102:7:50427982.reg.coloc.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="4" style="list-style-type: decimal">
<li><a
href="https://www.biorxiv.org/content/10.1101/2021.04.07.438882v1.full">29
UKBB blood traits</a></li>
</ol>
<blockquote>
<p>For our study, we examined 29 blood trait GWASs in the UK Biobank,
including traits from platelets, red blood cells, white blood cells and
reticulocytes (Table S1A).</p>
</blockquote>
</div>
</div>
<div id="nov-01" class="section level2">
<h2>Nov 01</h2>
<div id="colocalization" class="section level3">
<h3>Colocalization</h3>
<div id="enloc" class="section level4">
<h4>enloc</h4>
<p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1006646">Paper</a>:</p>
<div class="figure" style="text-align: center">
<img src="asset/1101_enloc_theory.png" alt="enloc equation" width="80%" />
<p class="caption">
enloc equation
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101_enloc_theory">
Past versions of 1101_enloc_theory.png
</button>
</p>
<div id="fig-1101_enloc_theory" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1101_enloc_theory.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a
href="https://github.com/xqwen/fastenloc/tree/master/tutorial">Code
tutorial</a>:</p>
<div class="figure" style="text-align: center">
<img src="asset/1101_enloc_code_pipeline.png" alt="enloc code workflow" width="80%" />
<p class="caption">
enloc code workflow
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101_enloc_code_pipeline">
Past versions of 1101_enloc_code_pipeline.png
</button>
</p>
<div id="fig-1101_enloc_code_pipeline" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1101_enloc_code_pipeline.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Need to provide “expression file” for the first fine-mapping. But
we don’t have this file in the case of a module being the “expression
trait”.</p></li>
<li><p>DAP can be applied to each LD block independently. <a
href="https://academic.oup.com/bioinformatics/article/32/2/283/1743626">independent
LD block</a> provided a working recipe to segment the full genome based
on the population-specific LD structures.</p></li>
<li><p>Example of using enloc for colocalization: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7737656/">GTEx
2020</a>. Used for “cis-QTL-GWAS colocalization: ENLOC.”</p></li>
</ul>
</div>
<div id="coloc" class="section level4">
<h4>coloc</h4>
<p><a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004383">Paper-coloc
single</a>, <a
href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009440">Paper-coloc
multiple</a>:</p>
<div class="figure" style="text-align: center">
<img src="asset/1101_coloc_theory.png" alt="coloc equation" width="30%" />
<p class="caption">
coloc equation
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101_coloc_theory">
Past versions of 1101_coloc_theory.png
</button>
</p>
<div id="fig-1101_coloc_theory" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1101_coloc_theory.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="https://chr1swallace.github.io/coloc/">Code
tutorial</a>:</p>
<p>First, segment the whole genome. Then fine-map each traits by <a
href="https://stephenslab.github.io/susieR/articles/index.html"><code>susieR</code></a>,
better than <code>coloc.abf</code> as it allows multiple causal
variants. Finally, coloc.</p>
<div class="figure" style="text-align: center">
<img src="asset/1101_coloc_code_workflow.png" alt="coloc code workflow" width="80%" />
<p class="caption">
coloc code workflow
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1101_coloc_code_workflow">
Past versions of 1101_coloc_code_workflow.png
</button>
</p>
<div id="fig-1101_coloc_code_workflow" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1101_coloc_code_workflow.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Determine LD block? Independent LD block + small GWAS pvalue loci
+ small module pvalue loci</p></li>
<li><p>Example of using coloc for colocalization:</p>
<ul>
<li><a
href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02334-x#Sec11">Mu
&amp; Li</a>): <code>coloc</code> + default prior.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7737656/">GTEx
2020</a>: Used for “Colocalization of cis- and trans-QTLs.”,
<code>coloc</code> + default prior. In addition, used for
“Colocalization of neutrophil ieQTLs and GWAS traits”:
<code>coloc</code> + <code>enloc</code> prior.</li>
</ul></li>
</ul>
</div>
<div id="ref" class="section level4">
<h4>Ref</h4>
</div>
</div>
</div>
<div id="oct-25" class="section level2">
<h2>Oct 25</h2>
<div id="pan-ukbb-website" class="section level3">
<h3>Pan UKBB website</h3>
<p>See website <a
href="https://pan.ukbb.broadinstitute.org/">here</a>.</p>
<p>Files website <a
href="https://pan.ukbb.broadinstitute.org/docs/technical-overview">here</a>.</p>
<ul>
<li>SNPs: 28,987,534 variants</li>
<li>Trait:</li>
<li>Population? meta? for (<span class="math inline">\(\beta, \text{se},
p, \text{low_confidence}\)</span>)</li>
</ul>
<div id="trait" class="section level4">
<h4>Trait</h4>
<ul>
<li><code>category</code>: A categorization of the phenotype</li>
</ul>
<p>blood related traits:</p>
<ol style="list-style-type: decimal">
<li>“Biological samples &gt; Assay results &gt; Blood assays &gt; Blood
count”</li>
</ol>
<p>e.g. Platelet count, Monocyte percentage, el al</p>
<ol start="2" style="list-style-type: decimal">
<li>“Biological samples &gt; Assay results &gt; Blood assays &gt; Blood
biochemistry”</li>
</ol>
<p>e.g. HDL cholesterol, Rheumatoid factor, et al</p>
<ol start="3" style="list-style-type: decimal">
<li><p>“UK Biobank Assessment Centre &gt; Physical measures &gt; Blood
pressure”</p></li>
<li><p>“Chapter III Diseases of the blood and blood-forming organs and
certain disorders involving the immune mechanism …”</p></li>
</ol>
<p>e.g. anaemias, D57 Sickle-cell disorders, el al</p>
<ul>
<li><p><code>pheno_sex</code>: Use which one? “both_sexes”, “females”,
“males”</p></li>
<li><p><code>modifier</code>: <code>irnt</code> or
<code>raw</code>?</p></li>
</ul>
</div>
<div id="variants-filtering" class="section level4">
<h4>Variants filtering</h4>
<blockquote>
<p>summary statistics with low-confidence statistics removed and only
considering high-quality variants.</p>
</blockquote>
<ul>
<li><code>af</code>: allele frequency</li>
<li><code>low_confidence</code>: criteria about allele count; includes
some (non-exhaustive) basic quality control filters.</li>
<li><code>high_quality</code>: A boolean corresponding to a high-quality
variant based on these filters … on “Files website”.</li>
</ul>
</div>
</div>
</div>
<div id="october-07-october-18" class="section level2">
<h2>October 07, October 18</h2>
<div id="msigdb-gene-modules-incorporated-with-dgn-genes"
class="section level3">
<h3>MSigDB gene modules, incorporated with DGN genes</h3>
<p>MSigDB hallmark gene sets have 50 clusters, consisting of 4383 genes.
3276 genes are included in 12132 DGN genes (without removing
cross-mappable genes).</p>
<p>Run Trans-PCO on the MSigDB hallmark gene sets.</p>
<div id="pvalues-distribution---qqplot" class="section level4">
<h4>pvalues’ distribution - qqplot</h4>
<p>Marked modules with significant signals by eye. See figure <a
href="asset/1007_MSigDB_p_distribution.png">here</a>.</p>
</div>
<div id="signals-under-empirical-fdr" class="section level4">
<h4>Signals under empirical FDR</h4>
<ol style="list-style-type: decimal">
<li>signals summary, under FDR 0.05, 0.1</li>
</ol>
<table>
<thead>
<tr class="header">
<th align="right">FDR level</th>
<th align="right">#signals</th>
<th align="right">#unique signals</th>
<th align="right">#independent signals</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">0.05</td>
<td align="right">808</td>
<td align="right">321</td>
<td align="right">23</td>
</tr>
<tr class="even">
<td align="right">0.10</td>
<td align="right">965</td>
<td align="right">411</td>
<td align="right">42</td>
</tr>
</tbody>
</table>
<p>cross-mapped genes in 50 gene sets? already removed cross-mapped
reads?</p>
<ol start="2" style="list-style-type: decimal">
<li>module (with annotation, size) v.s. chromosome signals</li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/1018-num_signals_module_vs_chr_.png" alt="#signals of MSigDB modules for each module v.s. chr."  />
<p class="caption">
#signals of MSigDB modules for each module v.s. chr.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1018-num_signals_module_vs_chr_">
Past versions of 1018-num_signals_module_vs_chr_.png
</button>
</p>
<div id="fig-1018-num_signals_module_vs_chr_" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1018-num_signals_module_vs_chr_.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="string-ppi-interaction-network-clustered-by-hierarchical-clustering."
class="section level3">
<h3>STRING PPI interaction network, clustered by hierarchical
clustering.</h3>
<p>Out of 12132 DGN genes, 11911 genes in the data included in STRING
PPI network. As a result of hierarchical clustering, 11911 genes consist
of 198 modules, with module size ranging from 210 ~ 26.</p>
<p>Run Trans-PCO on these modules.</p>
<div id="pvalues-distribution---qqplot-1" class="section level4">
<h4>pvalues’ distribution - qqplot</h4>
<p>A selected set of modules.</p>
<p>Marked modules with significant signals by eye. See figure <a
href="">here</a>.</p>
</div>
</div>
<div id="dgn_no_filter_on_mappability" class="section level3">
<h3>DGN_no_filter_on_mappability</h3>
<p>Without removing genes with mappability issues.</p>
<div id="pvalues-distribution---qqplot-2" class="section level4">
<h4>pvalues’ distribution - qqplot</h4>
</div>
</div>
</div>
<div id="october-04" class="section level2">
<h2>October 04</h2>
<div id="molecular-signatures-database" class="section level3">
<h3>Molecular Signatures Database</h3>
<p>The <a
href="http://www.gsea-msigdb.org/gsea/msigdb/index.jsp">MSigDB</a> gene
sets are divided into <a
href="http://www.gsea-msigdb.org/gsea/msigdb/collections.jsp">9 major
collections</a>: the hallmark gene sets, and C1 - C8 gene sets.</p>
</div>
<div id="hallmark-gene-sets" class="section level3">
<h3>Hallmark gene sets</h3>
<blockquote>
<p>What? Hallmark gene sets summarize and represent specific
well-defined biological states or processes and display coherent
expression.</p>
</blockquote>
<blockquote>
<p>How? These gene sets were generated by a computational methodology
based on identifying gene set overlaps and retaining genes that display
coordinate expression.</p>
</blockquote>
<div class="figure" style="text-align: center">
<img src="asset/1004_hallmark_pipeline.jpeg" alt="Pipeline to obtain hallmark gene sets from other MSigDB gene sets." width="50%" />
<p class="caption">
Pipeline to obtain hallmark gene sets from other MSigDB gene sets.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1004_hallmark_pipeline">
Past versions of 1004_hallmark_pipeline.jpeg
</button>
</p>
<div id="fig-1004_hallmark_pipeline" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1004_hallmark_pipeline.jpeg" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>See <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969/">paper
here</a>.</p>
<blockquote>
<p>pros? The hallmarks reduce noise and redundancy of the original
overlapping gene sets, from which a hallmark is derived (‘founder’
sets).</p>
</blockquote>
<p>Initial release. 50 hallmark gene sets. Comprise 4,022 original
overlapping gene sets from 8,380 v4.0 MSigDB gene sets of collections C1
through C6.</p>
<div class="figure" style="text-align: center">
<img src="asset/1004_geneSet_size_h.all.v7.4.symbols.gmt.png" alt="50 hallmark gene sets size" width="50%" />
<p class="caption">
50 hallmark gene sets size
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1004_geneSet_size_h.all.v7.4.symbols.gmt">
Past versions of 1004_geneSet_size_h.all.v7.4.symbols.gmt.png
</button>
</p>
<div id="fig-1004_geneSet_size_h.all.v7.4.symbols.gmt" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1004_geneSet_size_h.all.v7.4.symbols.gmt.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Summary of the hallmark gene sets: name, process category,
description, number of founder sets and number of genes it contains. <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969/table/T1/?report=objectonly">Table.</a></p>
<div class="figure" style="text-align: center">
<img src="asset/1004_geneSet_overlap_h.all.v7.4.symbols.gmt.png" alt="Overlapped genes among 50 hallmark gene sets" width="80%" />
<p class="caption">
Overlapped genes among 50 hallmark gene sets
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1004_geneSet_overlap_h.all.v7.4.symbols.gmt">
Past versions of 1004_geneSet_overlap_h.all.v7.4.symbols.gmt.png
</button>
</p>
<div id="fig-1004_geneSet_overlap_h.all.v7.4.symbols.gmt"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/1004_geneSet_overlap_h.all.v7.4.symbols.gmt.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="september-30" class="section level2">
<h2>September 30</h2>
<div id="string-ppi-networks-hierarchical-clustering"
class="section level3">
<h3>STRING PPI networks + Hierarchical clustering</h3>
<p>Question: Do the clusters make sense? Are the large clusters too
loose or the small clusters too tight?</p>
<ul>
<li>Large clusters too loose such that can be separated to a few smaller
cluster?</li>
</ul>
<p>I looked at the top two largest clusters: Module 1 with 214 genes,
and Module 2 with 188 genes.</p>
<p><img src="asset/0930_stringPPI_large_module.png" width="70%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0930_stringPPI_large_module">
Past versions of 0930_stringPPI_large_module.png
</button>
</p>
<div id="fig-0930_stringPPI_large_module" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/0930_stringPPI_large_module.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Small clusters too tight such that can be merged into less bigger
cluster?</li>
</ul>
<p>I looked at the last three smallest clusters: Module 194, 193, 192,
with 23, 26, 26 genes, correspondingly.</p>
<p><img src="asset/0930_stringPPI_small_module.png" width="80%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0930_stringPPI_small_module">
Past versions of 0930_stringPPI_small_module.png
</button>
</p>
<div id="fig-0930_stringPPI_small_module" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b27c4e1fcf611b039932dc2b8caca1241ea81803/docs/asset/0930_stringPPI_small_module.png" target="_blank">b27c4e1</a>
</td>
<td>
liliw-w
</td>
<td>
2021-11-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="septermber-23" class="section level2">
<h2>Septermber 23</h2>
<div id="string-ppi-networks" class="section level3">
<h3>STRING PPI networks</h3>
<div id="data" class="section level4">
<h4>Data</h4>
<ul>
<li><p>Protein network data downloaded from <a
href="https://www.string-db.org/cgi/download?sessionId=b0t6ChKtUCa8">here</a>.</p></li>
<li><p>Out of 12132 DGN genes, 11911 genes in the data included in
STRING network, with 3076398 edges.</p></li>
</ul>
</div>
<div id="clustering" class="section level4">
<h4>Clustering</h4>
<p>A <a
href="https://academic.oup.com/bioinformatics/article/30/10/1343/266738">paper</a>
reviews <a href="https://academic.oup.com/view-large/86243930">a list of
PPI networks clustering methods</a>.</p>
<blockquote>
<p>Clustering approaches to PPI networks can be broadly categorized as
<strong>topology-free</strong> and <strong>graph-based</strong> ones.
Topology-free approaches use traditional clustering techniques using
notions of distance between proteins that do not take into account the
topology of the network. Graph-based clustering approaches consider
instead the topology of the network, and usually rely on specialized
clustering techniques.</p>
</blockquote>
<ol style="list-style-type: decimal">
<li><p>MCODE &amp; MCL</p>
<p>Too many genes for MCODE, I waited 1h and there was no result coming
out.</p></li>
<li><p>Hierarchical clustering from clustermaker &amp; R</p>
<p>Hierarchical clustering in the Cytoscape plug-in, clustermaker, can’t
handle so many genes. So I tried to do hierarchical clustering in R.</p>
<p>Specifically, I used the “PPI scores” between two genes as the
similary measurement for computing distances. Those gene pairs without
edges (no PPI interaction) have 0 scores and 1 distance. Genes with
itself have 1 scores and 0 distance (diagonals of distance matrix). Then
I used the <code>hclust</code> in R to construct the hierarchical tree,
and the dynamic tree cut <code>cutreeDynamic</code> for branch cutting,
as used in WGCNA.</p>
<p>As a result, 11911 genes consist of 198 modules, with module size
ranging from 210 ~ 26.</p></li>
</ol>
</div>
</div>
<div id="humanbase-predicted-tissue-specific-interactions"
class="section level3">
<h3>HumanBase predicted tissue-specific interactions</h3>
<div id="data-1" class="section level4">
<h4>Data</h4>
<ul>
<li>Blood tissue interactions downloaded from <a
href="https://hb.flatironinstitute.org/download">here</a>.</li>
</ul>
</div>
<div id="clustering-1" class="section level4">
<h4>Clustering</h4>
<ol style="list-style-type: decimal">
<li><p>functional module detection</p></li>
<li><p>MCODE (graph-based clustering, since only have edge
info)</p></li>
</ol>
</div>
</div>
</div>
<div id="september-02-september-07-september-10" class="section level2">
<h2>September 02, September 07, September 10</h2>
<div id="pre-filter-cross-map-genes-before-grouping-genes-into-modules"
class="section level3">
<h3>Pre-filter cross-map genes before grouping genes into modules</h3>
<p>Out of 10461 genes, (after filtering based on meta info, protain and
lincRNA, low mappability), do additional gene filtering if genes have
too many cross-map genes (e.g. use 1K as the threshold). 4314 genes are
removed.</p>
<p>As a result, 6147 genes are remained and are grouped in 95
modules.</p>
</div>
<div id="computation-time" class="section level3">
<h3>Computation time</h3>
<div id="problem" class="section level4">
<h4>Problem</h4>
<p>Last time, we ran into the problem that it is computational
unfeasible if we want to run PCO without removing cross mappable genes
in the previous step. See the illustration in the following figure.</p>
<p>There are two main steps that need long computational time. (1) For
each SNP, remove genes from the module that are cross mappable with cis-
genes of the SNP. (2) In this case, we would have module containing
different genes for SNPs with different cis- genes, thus different <span
class="math inline">\(\Sigma\)</span> and <span
class="math inline">\(\Sigma^O\)</span>. Therefore, tremendously
increase the computational time.</p>
</div>
<div id="try-solution-1-filtering-snps-instead" class="section level4">
<h4>Try solution 1: filtering SNPs instead</h4>
<p>One of the possible solutions we mentioned is, instead of filtering
genes for each SNP, we can try filtering SNPs and keeping the module
unchanged. That being said, if a SNP has cis- genes cross mappable with
any genes in the module, then this SNP is removed from the following PCO
analysis.</p>
<p>In this way, there are two advantages. First, it’s faster to filter
SNPs than genes. Second, we will have same module for all remained SNPs,
thus same <span class="math inline">\(\Sigma\)</span> and <span
class="math inline">\(\Sigma^O\)</span>.</p>
<p>So, next I check how many SNPs will be left after this filtering
step.</p>
<div class="figure" style="text-align: center">
<img src="asset/0907_SNP_remain.png" alt="Figure. SNP remained. The bar plot gives the number of SNPs, while the line plot gives the percentage of the remained SNPs on each chr." width="70%" />
<p class="caption">
Figure. SNP remained. The bar plot gives the number of SNPs, while the
line plot gives the percentage of the remained SNPs on each chr.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_SNP_remain">
Past versions of 0907_SNP_remain.png
</button>
</p>
<div id="fig-0907_SNP_remain" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b7769260a045ab7d7952f05cb2d4938b953752c2/docs/asset/0907_SNP_remain.png" target="_blank">b776926</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We can see, if we filter SNPs instead, over half of the genome will
be removed from the p-value analysis. We could miss a lot of
signals.</p>
</div>
</div>
<div id="keep-cross-mappable-genes-in-the-filtering-step"
class="section level3">
<h3>Keep cross mappable genes in the filtering step</h3>
<ol style="list-style-type: decimal">
<li>Genes and modules containing cross mappable genes</li>
</ol>
<table style="width:65%;">
<colgroup>
<col width="31%" />
<col width="33%" />
</colgroup>
<tbody>
<tr class="odd">
<td>All genes:</td>
<td>13634</td>
</tr>
<tr class="even">
<td>With meta:</td>
<td>12585</td>
</tr>
<tr class="odd">
<td>Final use:</td>
<td>10461</td>
</tr>
<tr class="even">
<td>All removed:</td>
<td>3173</td>
</tr>
<tr class="odd">
<td>including:</td>
<td><ul>
<li>meta: 1049</li>
<li>auto: 1</li>
<li>prote&amp;linc: 453</li>
<li>low_map: 1798</li>
<li>cross_map: 0</li>
</ul></td>
</tr>
<tr class="even">
<td>Number of modules:</td>
<td>158</td>
</tr>
</tbody>
</table>
<p><img src="asset/0902_plot_module_size.png" style="width:30.0%" /><img
src="asset/0902_plot_module_size_cross_within_module.png"
style="width:30.0%" /></p>
<p>Figure. Module v.s. #Genes. Bar height for module size. Red/blue bar
for #cross mappable genes for each module. Left for genes cross mappable
with any genes. Right for genes cross mappable with genes within the
same module.</p>
<ol start="2" style="list-style-type: decimal">
<li>Run PCO with cross mappable genes</li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/0902_PCO_keep_crossmap.jpeg" alt="Figure. Illustration of PCO with cross mappable genes." width="70%" />
<p class="caption">
Figure. Illustration of PCO with cross mappable genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0902_PCO_keep_crossmap">
Past versions of 0902_PCO_keep_crossmap.jpeg
</button>
</p>
<div id="fig-0902_PCO_keep_crossmap" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/786b185f8b0b3379fcb5c81afb08770cbd3718b1/docs/asset/0902_PCO_keep_crossmap.jpeg" target="_blank">786b185</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Problem: Computational infeasible</p>
<ul>
<li>In phase 1, for each SNP, filter out genes within the module that
are cross mappable with any of its cis- genes. Not fast, as need to
search pairs of (cis- gene, module gene) in the cross-mappability file
with 23,727,385 cross-mappable gene pairs.</li>
</ul>
<p>The code I use,</p>
<pre class="r"><code>if_cross &lt;- sapply(cis_Genes[1:10], function(x){
  colSums(matrix(outer(x, module_Geneid, FUN = &quot;paste&quot;, sep=&quot;;&quot;) %in% cross_map$V4 |
                   t(outer(module_Geneid, x, FUN = &quot;paste&quot;, sep=&quot;;&quot;)) %in% cross_map$V4, ncol = length(module_Geneid)) ) == 0
} )</code></pre>
<p>Time elapsed,</p>
<div class="figure" style="text-align: center">
<img src="asset/0902_time_cross_map.png" alt="Figure. Time used for a SNP with ~40 cis- genes and a module with 15 genes." width="40%" />
<p class="caption">
Figure. Time used for a SNP with ~40 cis- genes and a module with 15
genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0902_time_cross_map">
Past versions of 0902_time_cross_map.png
</button>
</p>
<div id="fig-0902_time_cross_map" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/786b185f8b0b3379fcb5c81afb08770cbd3718b1/docs/asset/0902_time_cross_map.png" target="_blank">786b185</a>
</td>
<td>
liliw-w
</td>
<td>
2021-09-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>In phase 2, each SNP could correspond to modules with different gene
combinations, thus diff. <span class="math inline">\(\Sigma\)</span> for
with-in module correlation and <span
class="math inline">\(\Sigma^0\)</span>, which is the estimated
correlation of six tests involved in PCO using 2000 simulations. Thus,
computing <span class="math inline">\(\Sigma^0\)</span> for each SNP
should take a long time.</li>
</ul>
</div>
</div>
<div id="july-08" class="section level2">
<h2>July 08</h2>
<div id="how-expression-pcs-affect-the-signals" class="section level3">
<h3>How expression PCs affect the signals?</h3>
<p>The following table gives the number of unique signals under various
FDR levels for DGN regressing out expression PC as covariates
(“DGN_wo_expPC”) and keeping expression PCs (“DGN_with_expPC”).</p>
<table>
<colgroup>
<col width="11%" />
<col width="14%" />
<col width="20%" />
<col width="17%" />
<col width="22%" />
<col width="9%" />
<col width="4%" />
</colgroup>
<thead>
<tr class="header">
<th align="right">FDR level</th>
<th align="right">DGN_wo_expPC</th>
<th align="left">DGN_wo_expPC_maxp</th>
<th align="right">DGN_with_expPC</th>
<th align="left">DGN_with_expPC_maxp</th>
<th align="right">overlap</th>
<th align="right">#*</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">0.05</td>
<td align="right">313</td>
<td align="left">1.9e-08</td>
<td align="right">169</td>
<td align="left">4e-09</td>
<td align="right">125</td>
<td align="right">218</td>
</tr>
<tr class="even">
<td align="right">0.10</td>
<td align="right">512</td>
<td align="left">6.5e-08</td>
<td align="right">225</td>
<td align="left">2e-08</td>
<td align="right">167</td>
<td align="right">303</td>
</tr>
</tbody>
</table>
</div>
<div id="correlation-between-meta-ed-gene-zscores-from-eqtlgen"
class="section level3">
<h3>Correlation between meta-ed gene zscores from eQTLGen</h3>
<p>Check document <a
href="asset/0708-Correlation%20between%20meta-ed%20gene%20zscores%20from%20eQTLGen.pdf">here</a>.</p>
</div>
</div>
<div id="july-06" class="section level2">
<h2>July 06</h2>
<div
id="how-sigma-hatsigma_dgn_exp-and-hatsigma_nullz-affect-pco-pvalues-for-dgn"
class="section level3">
<h3>How Sigma <span
class="math inline">\(\hat{\Sigma}_{DGN_exp}\)</span> and <span
class="math inline">\(\hat{\Sigma}_{nullz}\)</span> affect PCO pvalues
for DGN?</h3>
<p>I inspected three modules. Check <a
href="asset/0706_Sigma_effect_on_PCO.pdf">here</a> for the
comparison.</p>
<p>For individual and clearer figures, check <a
href="asset/0706_qqplot.effect.of.Sigma.on.PCO.pvalue.module1.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module1.origp.png">here</a>,
<a
href="asset/0706_qqplot.effect.of.Sigma.on.PCO.pvalue.module20.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module1.log10.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module20.origp.png">here</a>,
<a
href="asset/0706_qqplot.effect.of.Sigma.on.PCO.pvalue.module73.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module20.log10.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module73.origp.png">here</a>,
<a
href="asset/0706_effect.of.Sigma.on.PCO.pvalue.module73.log10.png">here</a>.</p>
</div>
</div>
<div id="june-29" class="section level2">
<h2>June 29</h2>
<div
id="try-using-zscores-to-build-sigma-plot-of-number-of-snps-with-low-zscores"
class="section level3">
<h3>“Try using zscores to build sigma, plot of number of SNPs with low
zscores”</h3>
<ol style="list-style-type: decimal">
<li>plot of number of SNPs with low zscores</li>
</ol>
<p>In this section, I am going to check how the estimated Sigma <span
class="math inline">\(\hat{\Sigma}_{K}\)</span> (correlation matrix of
zscores/residule expression across genes) can affect PCO’s pvalues. Use
eQTLGen zscores. Consider two <span
class="math inline">\(\hat{\Sigma}\)</span>’s. (1) <span
class="math inline">\(\hat{\Sigma}_{DGN}\)</span>, computed by <span
class="math inline">\(cor(\tilde{Y}_{n \times K}^{DGN})\)</span>, where
<span class="math inline">\(\tilde{Y}\)</span> is the residual
expression data from DGN, n is sample size, K is gene module size. (2)
<span class="math inline">\(\hat{\Sigma}_{nullz}\)</span>, computed by
<span class="math inline">\(cor(\tilde{Z}_{m \times
K}^{eQTLGen})\)</span>, where z is genome-wide independent null zscores
from eQTLGen, m is the number of null SNPs (m is in fact much less than
genome-wide, but should be sufficient?).</p>
<p>Since for each null SNP, we should have its zscores on all genes (or
at least all genes in the module we consider), but eQTLGen doesn’t
provide zscores for all genome-wide SNPs (only disease-related variants
for trans-eQTLs and close-to-gene variants for cis-eQTLs). Therefore, I
will first extract the common SNPs for each module. Main steps
include:</p>
<ul>
<li><p>Combine the zscores of all the significant/non-significant
gene-SNP pairs in both eQTLGen cis-eQTL and trans-eQTL results.</p></li>
<li><p>For each module, extract the common SNPs that have zscores
available for all genes in the module.</p></li>
<li><p>For each module, further extract the common SNPs that are “NULL
SNPs”. A snp for a module is defined as a “null snp” if its z-scores for
all genes in this module are below some z-score cutoff (i.e. all
z-scores are small z’s”).</p></li>
<li><p>Among these common null SNPs, check how many of them are
independent. (Use DGN genotype ‘plink –indep-pairwise 50 5 0.2’ ??? in
this case, some SNPs not included in DGN are also considered as
“independent. ref panel including all eQTLGen snps?)</p></li>
</ul>
<p><img src="asset/0629_num_nullSNP.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0629_num_nullSNP">
Past versions of 0629_num_nullSNP.png
</button>
</p>
<div id="fig-0629_num_nullSNP" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b79abbfc3a37486b82a1da080a41c5c0f384809/docs/asset/0629_num_nullSNP.png" target="_blank">1b79abb</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Here I explain the above figure. x-axis is for each module (75),
y-axis is for number of SNPs under various scenarios that can be used
for computing <span class="math inline">\(\hat{\Sigma}_{nullz}\)</span>.
Color is for various z-scores cutoff for defining “NULL SNPs”. The
cutoff is pvalue of z. Smaller cutoff, more insignificant z. Point shape
is for unique SNPs and independent SNPs. The barplot in the lower panel
gives the number of genes in each module.</p>
<div id="observations-5" class="section level4">
<h4>Observations</h4>
<ol style="list-style-type: decimal">
<li><p>For larger gene module, the common null SNPs (unique and
independent) are fewer.</p></li>
<li><p>The average number of the unique null SNPs is ~8000 (under pvalue
of z &gt; 0.001). The independent null SNPs is ~4000.</p></li>
<li><p>Since we were particularly interested at eQTLGen’s module1, (as
all eQTLGen trans-variants are significant for module1,) next I will
focus on module1. I use the 2282 independent null SNPs (under pvalue of
z &gt; 0.001) for module1 to calculate <span
class="math inline">\(\hat{\Sigma}_{nullz}(module1)\)</span>.</p></li>
</ol>
</div>
</div>
<div
id="build-sigma-hatsigma_dgn-and-hatsigma_nullz-and-check-pco-pvalues."
class="section level3">
<h3>Build Sigma <span class="math inline">\(\hat{\Sigma}_{DGN}\)</span>
and <span class="math inline">\(\hat{\Sigma}_{nullz}\)</span> and check
PCO pvalues.</h3>
<p>eQTLGen’s module1. Run PCO based on two Sigmas, <span
class="math inline">\(\hat{\Sigma}_{DGN}\)</span> and <span
class="math inline">\(\hat{\Sigma}_{nullz}\)</span> built on 2282
independent null SNPs. Calculate pvalues of 9918 SNPs across
chromosomes.</p>
<p><img src="asset/0629_effect.of.Sigma.on.PCO.pvalue.module1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0629_effect.of.Sigma.on.PCO.pvalue.module1">
Past versions of 0629_effect.of.Sigma.on.PCO.pvalue.module1.png
</button>
</p>
<div id="fig-0629_effect.of.Sigma.on.PCO.pvalue.module1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b79abbfc3a37486b82a1da080a41c5c0f384809/docs/asset/0629_effect.of.Sigma.on.PCO.pvalue.module1.png" target="_blank">1b79abb</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In the above figure, x-axis is for snp across chromosomes, y-axis is
for log(p) for the module1, color is for different <span
class="math inline">\(\hat{\Sigma}\)</span>, grey dashed line is the
Bonferroni pvalue threshold (~1e-8).</p>
<p><img src="asset/0629_qqplot.effect.of.Sigma.on.PCO.pvalue.module1.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0629_qqplot.effect.of.Sigma.on.PCO.pvalue.module1">
Past versions of 0629_qqplot.effect.of.Sigma.on.PCO.pvalue.module1.png
</button>
</p>
<div id="fig-0629_qqplot.effect.of.Sigma.on.PCO.pvalue.module1"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b79abbfc3a37486b82a1da080a41c5c0f384809/docs/asset/0629_qqplot.effect.of.Sigma.on.PCO.pvalue.module1.png" target="_blank">1b79abb</a>
</td>
<td>
liliw-w
</td>
<td>
2021-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The above figure gives QQ-plot for all pvalues across 22 chr’s of
module1.</p>
<div id="observations-6" class="section level4">
<h4>Observations</h4>
<ol style="list-style-type: decimal">
<li>Using <span class="math inline">\(\hat{\Sigma}_{nullz}\)</span>
gives larger PCO pvalues, than using <span
class="math inline">\(\hat{\Sigma}_{DGN}\)</span>. Number of signals
decreases. Under Bonferroni pvalue threshold (~1e-8), when using <span
class="math inline">\(\hat{\Sigma}_{nullz}\)</span>, there are 790 out
of 9918 SNPs are significant, compared to 9794 significant SNPs using
<span class="math inline">\(\hat{\Sigma}_{DGN}\)</span>.</li>
</ol>
</div>
</div>
</div>
<div id="june-25" class="section level2">
<h2>June 25</h2>
<div
id="replication-of-eqtlgen-results-in-trans-pco-of-eqtlgen-summary-stats"
class="section level3">
<h3>Replication of eQTLGen results in trans-PCO of eQTLGen summary
stats?</h3>
<p>I first give some data statistics here.</p>
<table>
<colgroup>
<col width="20%" />
<col width="12%" />
<col width="30%" />
<col width="24%" />
<col width="13%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="left">Method</th>
<th align="right">#(gene, SNP)/(module, SNP)</th>
<th align="right">#unique genes/modules</th>
<th align="right">unique SNPs</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">eQTGen</td>
<td align="left">FDR&lt;0.05</td>
<td align="right">59786</td>
<td align="right">6298</td>
<td align="right">3853</td>
</tr>
<tr class="even">
<td align="left">eQTGen</td>
<td align="left">filter</td>
<td align="right">20394</td>
<td align="right">1857</td>
<td align="right">2469</td>
</tr>
<tr class="odd">
<td align="left">eQTLGen_trans-PCO</td>
<td align="left">qvalue</td>
<td align="right">406729</td>
<td align="right">75</td>
<td align="right">9918</td>
</tr>
<tr class="even">
<td align="left">eQTLGen_trans-PCO</td>
<td align="left">Bonferroni</td>
<td align="right">55634</td>
<td align="right">74</td>
<td align="right">9915</td>
</tr>
</tbody>
</table>
<ul>
<li><p>The eQTGen’s original result <a
href="https://www.eqtlgen.org/trans-eqtls.html">here</a> gives 59786
(gene, trans-eQTL) pairs under FDR <span
class="math inline">\(&lt;0.05\)</span>, corresponding to 6298 unique
genes and 3853 unique SNPs.</p></li>
<li><p>After some filtering, to make sure the eQTGen trans-eQTLs and
genes are included in out trans-PCO result using eQTLGen summary stats.
<strong>There are 20394 (gene, trans-eQTL) pairs left, corresponding to
1857 unique genes and 2469 unique SNPs.</strong></p></li>
<li><p>Orignal eQTLGen_trans-PCO result (under qvalue FDR correction):
406729 (module, trans-eQTL) pairs, corresponding to all 75 unique
modules and 9918 unique SNPs.</p></li>
<li><p>Under Bonferroni correction (p&lt;6.721785e-08): <strong>55634,
74, 9915</strong>.</p></li>
</ul>
<p>Next, I will check how eQTGen signals are replicated in
eQTLGen_trans-PCO results (see bold text above). The basic idea is, for
an eQTGen trans-eQTL, find its trans-target genes, then look at how
these genes are distributed in each gene modules we used for
eQTLGen_trans-PCO, and whether the corresponding (module, trans-eQTL)
are identified by eQTLGen_trans-PCO. Specifically, I will check two
things.</p>
<ol style="list-style-type: decimal">
<li><p>For a trans-eQTL whose trans-target genes are included in one of
our PCO module, is this (module, trans-eQTL) identified by
eQTLGen_trans-PCO? (Power)</p>
<p>Since in previous analysis, the intuition is PCO tends to pick
modules where there are multiple trans-target genes for a SNP.
Therefore, in the below figure, I picked out all the eQTGen trans-eQTLs
that have &gt;2 target genes and plotted them on x-axis. The target
genes are distributed on the Y-axis for each PCO module. Number/color is
for #target genes. These (module, SNP) pairs should be significant in
eQTLGen_trans-PCO. I circled out (red) those (module, SNP) pairs that
are missed by eQTLGen_trans-PCO. As a result, 11 (module, SNP) (11
unique SNPs) are missed.</p></li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/0625_power.png" alt="Power"  />
<p class="caption">
Power
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0625_power">
Past versions of 0625_power.png
</button>
</p>
<div id="fig-0625_power" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0625_power.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check <a href="asset/0625_power.png">here</a> for clearer
version.</p>
<ol start="2" style="list-style-type: decimal">
<li><p>For a trans-eQTL whose trans-target genes are not included in any
PCO modules (or only 1 or 2), is this (module, trans-eQTL) identified by
eQTLGen_trans-PCO? (TIE)</p>
<p>Similar plot, except Number/color is SNP with 0, 1, or 2 target genes
in each module. These (module, SNP) pairs are less likely to be
identified by eQTLGen_trans-PCO. Yet, I circled out (green) those
(module, SNP) pairs that are still identified by
eQTLGen_trans-PCO.</p></li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/0625_TIE.png" alt="TIE"  />
<p class="caption">
TIE
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0625_TIE">
Past versions of 0625_TIE.png
</button>
</p>
<div id="fig-0625_TIE" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0625_TIE.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Check <a href="asset/0625_TIE.png">here</a> for clearer version.</p>
<div id="observation" class="section level4">
<h4>Observation</h4>
<ol style="list-style-type: decimal">
<li><p>Almost all eQTGen trans-eQTLs with &gt;2 target genes and their
corresponding module are identified by eQTLGen_trans-PCO. This may
suggest a good power of eQTLGen_trans-PCO.</p></li>
<li><p>However, the false positiva rate may be inflated, as a large
amount of SNPs that has no target genes in the modules are also
identified. (<strong>Maybe although none of the genes in the module are
target genes for the SNPs, the genes may be cross-mappable with the
SNP’s other traget genes that are filter out?</strong>)</p></li>
</ol>
</div>
</div>
<div id="how-does-zero-values-affect-qvalue" class="section level3">
<h3>How does zero values affect qvalue?</h3>
<p>Short answer:</p>
</div>
</div>
<div id="june-14" class="section level2">
<h2>June 14</h2>
<div id="why-there-are-so-many-eqtlgen-signals-using-pco_lambda0.1"
class="section level3">
<h3>Why there are so many eQTLGen signals using PCO_<span
class="math inline">\({\lambda&lt;0.1}\)</span>?</h3>
<p>Check the QQ-plot of the observed pvalues from eQTLGen using
PCO_<span class="math inline">\({\lambda&lt;0.1}\)</span> (9918 unique
SNPs, all eQTLGen variants) and PCO_<span
class="math inline">\({\lambda&lt;1}\)</span> (909 unique SNPs).</p>
<p><img src="asset/0614_qqplot.p_obs.eQTLGen.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0614_qqplot.p_obs.eQTLGen">
Past versions of 0614_qqplot.p_obs.eQTLGen.png
</button>
</p>
<div id="fig-0614_qqplot.p_obs.eQTLGen" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0614_qqplot.p_obs.eQTLGen.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="june-11" class="section level2">
<h2>June 11</h2>
<div id="compare-with-truncated-dgn_transeqtl_battle2014"
class="section level3">
<h3>Compare with truncated ‘DGN_transeQTL_battle2014’</h3>
<p>In this section, I removed trans-eGenes from Battle’s result that are
not included in our analysis for being cross-mappable or poorly mapped.
I do this to makes sure univariate and multivariate are based on the
same set of genes. I go through the same analysis as below.</p>
<ul>
<li>Summary of the new version Battle trans-eQTL signals</li>
</ul>
<p>After filter our some trans-eGenes, there are <strong>157</strong>
(v.s. 736) significant (gene, SNP) pairs left, corresponding to 84
unique trans-eQTLs and 47 unique trans-eGenes.</p>
<p>The following is a a table of the number of trans-target genes of
these 84 unique trans-eQTLs.</p>
<pre><code>
 1  2  3  4  5 15 16 29 
75  1  1  3  1  1  1  1 </code></pre>
<p>We have identified 313 unique trans-eQTLs, among which 24 are also
detected by Battle’s univariate method. The above table gives which
Battle’s signals are replicated by trans-PCO.</p>
<p>Among 84 Battle’s trans-eQTLs, 24 (~28.57%) are also detected by
trans-PCO. For 9 signals with more than one trans-target genes included
in our analysis, all of them are detected by trans-PCO. Trans-PCO also
detected other 15 signals with only one trans-target gene used in our
analysis. For the other 60 univariate signals that are missed by
trans-PCO, they have only one trans-target gene included in
trans-PCO.</p>
<pre><code>            sig N_transeGene if_transPCO num_included
 1:  3:56849749           29        TRUE           29
 2:  3:56858686           16        TRUE           16
 3:  3:56865776           15        TRUE           15
 4:  3:56880444            5        TRUE            5
 5:  3:56871432            4        TRUE            4
 6:  3:56874033            4        TRUE            4
 7:  7:50366637            4        TRUE            4
 8:  3:56851387            3        TRUE            3
 9:  7:50428445            2        TRUE            2
10: 7:106337250            1       FALSE            1
11: 12:54723028            1       FALSE            1
12: 12:54729872            1       FALSE            1
13: 4:103432974            1       FALSE            1
14: 4:103443569            1       FALSE            1
15: 4:103448582            1       FALSE            1
16: 4:103511114            1       FALSE            1
17: 1:118147675            1       FALSE            1
18: 1:118158079            1       FALSE            1
19: 19:37441365            1       FALSE            1
20: 19:37469098            1       FALSE            1
21: 19:37503899            1       FALSE            1
22: 19:37504596            1       FALSE            1
23: 19:37507729            1       FALSE            1
24: 19:37516220            1       FALSE            1
25: 19:37520927            1       FALSE            1
26: 19:37547219            1       FALSE            1
27: 19:37563211            1       FALSE            1
28: 19:37573074            1       FALSE            1
29: 19:37582471            1       FALSE            1
30: 19:37584650            1       FALSE            1
31: 19:37597395            1       FALSE            1
32: 19:37620248            1       FALSE            1
33: 19:37627113            1       FALSE            1
34: 19:37632877            1       FALSE            1
35: 19:37634291            1       FALSE            1
36: 19:37642165            1       FALSE            1
37: 19:37649577            1       FALSE            1
38: 19:37650929            1       FALSE            1
39: 19:37684966            1       FALSE            1
40: 17:33875262            1       FALSE            1
41: 1:248039451            1        TRUE            1
42: 1:248039713            1       FALSE            1
43: 1:248041182            1       FALSE            1
44: 1:248058265            1       FALSE            1
45: 19:44889660            1       FALSE            1
46: 19:44930738            1        TRUE            1
47: 19:44931995            1       FALSE            1
48: 19:44932972            1        TRUE            1
49: 19:44933947            1        TRUE            1
50: 19:44938980            1        TRUE            1
51: 19:44947050            1       FALSE            1
52: 19:44947184            1        TRUE            1
53: 19:44959866            1       FALSE            1
54: 19:44973969            1       FALSE            1
55: 19:44978298            1       FALSE            1
56: 19:44979443            1       FALSE            1
57: 19:44995990            1       FALSE            1
58: 19:45001346            1       FALSE            1
59: 19:45001619            1       FALSE            1
60: 14:35523266            1       FALSE            1
61: 14:35571651            1        TRUE            1
62: 14:35624831            1       FALSE            1
63: 21:44473062            1       FALSE            1
64: 21:44473425            1       FALSE            1
65: 21:44473446            1       FALSE            1
66:  12:8142798            1        TRUE            1
67:  12:8143483            1        TRUE            1
68:  12:8161503            1        TRUE            1
69:  12:8179891            1       FALSE            1
70:  12:8185853            1       FALSE            1
71:  12:8190206            1       FALSE            1
72: 6:159498130            1       FALSE            1
73: 15:80260554            1        TRUE            1
74: 15:80263217            1        TRUE            1
75: 15:80263345            1        TRUE            1
76: 15:80263406            1        TRUE            1
77:  7:50435777            1       FALSE            1
78: 6:169749100            1       FALSE            1
79: 14:20811332            1        TRUE            1
80: 17:33902635            1       FALSE            1
81: 17:34019488            1       FALSE            1
82: 17:34043021            1       FALSE            1
83: 17:34051176            1       FALSE            1
84:  9:99288951            1       FALSE            1
            sig N_transeGene if_transPCO num_included</code></pre>
</div>
<div id="compare-with-dgn_transeqtl_battle2014" class="section level3">
<h3>Compare with ‘DGN_transeQTL_battle2014’</h3>
<ul>
<li>Summary of Battle trans-eQTL signals</li>
</ul>
<p>They gave 736 significant (gene, SNP) pairs, corresponding to 350
unique trans-eQTLs and 138 unique trans-eGenes.</p>
<p>The following is a a table of the number of trans-target genes of
these 350 unique trans-eQTLs. We can see most of the trans-eQTLs have
only 1 or 2 trans-target genes (<span class="math inline">\(283+34=317,
\approx 91\%\)</span>).</p>
<pre><code>
  1   2   3   4   5   6   7   8  11  13  14  22  23  24  29  33  59 
283  34   3   1  12   1   2   2   2   1   2   1   2   1   1   1   1 </code></pre>
<p>Let’s take a look at the trans-eQTLs with more than 2 target genes.
We can see these trans-eQTLs are mainly located around the regions
3:56849749, 3:101043565, 7:50366637, 17:33875262.</p>
<pre><code>            sig N_transeGene if_transPCO num_included
 1:  3:56849749           59        TRUE           29
 2:  3:56858686           33        TRUE           16
 3:  3:56865776           29        TRUE           15
 4: 3:101242751           24       FALSE            0
 5: 3:101043565           23       FALSE            0
 6: 3:101106527           23       FALSE            0
 7: 3:101232736           22       FALSE            0
 8: 3:100963761           14       FALSE            0
 9: 3:100970240           14       FALSE            0
10: 3:100994497           13       FALSE            0
11:  3:56851387           11        TRUE            3
12:  3:56880444           11        TRUE            5
13: 3:100915962            8       FALSE            0
14:  7:50366637            8        TRUE            4
15:  3:56874033            7        TRUE            4
16: 3:101068620            7       FALSE            0
17:  3:56871432            6        TRUE            4
18: 3:101042594            5       FALSE            0
19: 3:101080694            5       FALSE            0
20: 3:101085413            5       FALSE            0
21: 3:101135210            5       FALSE            0
22: 3:101160339            5       FALSE            0
23: 3:101184134            5       FALSE            0
24: 3:101191663            5       FALSE            0
25: 3:101231921            5       FALSE            0
26: 3:101237307            5       FALSE            0
27: 3:101252514            5       FALSE            0
28: 3:101253091            5       FALSE            0
29: 3:101253605            5       FALSE            0
30:  7:50428445            4        TRUE            2
31:  7:50435777            3       FALSE            1
32: 17:33875262            3       FALSE            1
33: 17:33902635            3       FALSE            1
            sig N_transeGene if_transPCO num_included</code></pre>
<ul>
<li>As for our results from trans-PCO</li>
</ul>
<p>We have identified 313 unique trans-eQTLs, among which 24 are also
detected by Battle’s univariate method. The above table gives which
Battle’s signals are replicated by trans-PCO.</p>
<p>Table of the number of target genes for replicated trans-eQTLs.</p>
<pre><code>
 1  2  4  6  7  8 11 29 33 59 
12  3  1  1  1  1  2  1  1  1 </code></pre>
<ol style="list-style-type: decimal">
<li><p>Signals detected by univariate but missed by trans-PCO.</p>
<p>Since trans-PCO tends to identify SNPs that have effects on multiple
genes, I will focus on the univariate trans-eQTLs that correspond to
multiple genes (<span class="math inline">\(&gt;2\)</span>). For this
type of signals, there are 33 trans-eQTLs in total, among which 9 are
detected by trans-PCO. For the rest of 24 missed trans-eQTLs, there are
basically three types: (1) 21 near the region “3:101043565”, (2) 1 at
“7:50435777”, (3) 2 at “17:33875262” and “17:33875262”.</p>
<ol style="list-style-type: decimal">
<li><p>Even though the 20 missed trans-eQTLs correspond to multiple
genes, ranging from 5 ~ 24, they are still missed by trans-PCO. So I
checked their trans-target genes, and it turns out that all of the
target genes are filtered out at the beginning of our analysis for being
cross-mappable or poorly mapped.</p></li>
<li><p>As for the rest of three trans-eQTLs that are missed by
trans-PCO, only one of their 3 trans-target genes are included in our
analysis.</p></li>
</ol></li>
</ol>
<div id="take-aways" class="section level4">
<h4>Take-aways</h4>
<ol style="list-style-type: decimal">
<li><p>Trans-PCO tends to be more likely to detect trans-eQTLs with
effects on multiple target genes.</p></li>
<li><p>Some signals could be missed at the step of filtering out
genes.</p></li>
</ol>
</div>
</div>
</div>
<div id="may-26" class="section level2">
<h2>May 26</h2>
<div id="are-trans-eqtls-cis-eqtls" class="section level3">
<h3>Are trans-eQTLs cis-eQTLs?</h3>
<p>Updated the way to define cis-eQTLs. Previously, cis-eQTL signals are
gene-based (from Pheonix using fastQTL). In this section, I compare
trans signals with SNP-based cis-eQTLs from two datasets, i.e. GTEx_v7
(whole blood) and DGN.</p>
<ul>
<li><p>GTEx_v7. <a
href="https://www.gtexportal.org/home/datasets">Here</a> provides all
significant variant-gene associations.</p></li>
<li><p>DGN. Two steps to obtain SNP-based significant associations.</p>
<ol style="list-style-type: decimal">
<li><p>Gene-based associations. (eGenes using fastQTL, result from
Pheonix)</p></li>
<li><p>Find all cis-eQTLs for each gene.</p>
<p>See “4.2 cis-eQTL mapping” described <a
href="https://science.sciencemag.org/content/sci/suppl/2020/09/09/369.6509.1318.DC1/aaz1776_Aguet_SM.pdf">here</a>.
(p.s. I checked GTEx source code and turns out that <a
href="https://github.com/francois-a/fastqtl/blob/master/R/calculateSignificanceFastQTL.R">the
actual code GTEx used</a> is a bit different. See below for key
lines.)</p></li>
</ol></li>
</ul>
<pre class="r"><code>pt = mean(c(max(eGene$bpval), min(eGene$bpval)))
eGene$pval_nominal_threshold = signif(qbeta(pt,
                                            eGene$shape1, eGene$shape2, ncp=0, lower.tail=TRUE, log.p=FALSE), 6)</code></pre>
<table style="width:100%;">
<colgroup>
<col width="7%" />
<col width="4%" />
<col width="18%" />
<col width="10%" />
<col width="15%" />
<col width="8%" />
<col width="15%" />
<col width="9%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="right">#signal</th>
<th align="right">GTEx_v7_is_ciseQTL(SNP-based)</th>
<th align="right">GTEx_v7_cis-eGene</th>
<th align="right">DGN_is_ciseQTL(SNP-based)</th>
<th align="right">DGN_cis-eGene</th>
<th align="right">DGN_is_cissQTL(SNP-based)</th>
<th align="right">DGN_cis-sIntro</th>
<th align="right">DGN_cis-e/sQTL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="right">313</td>
<td align="right">51</td>
<td align="right">16</td>
<td align="right">156</td>
<td align="right">39</td>
<td align="right">101</td>
<td align="right">71</td>
<td align="right">182</td>
</tr>
<tr class="even">
<td align="left">eQTLGen_DGN</td>
<td align="right">784</td>
<td align="right">221</td>
<td align="right">32</td>
<td align="right">502</td>
<td align="right">83</td>
<td align="right">381</td>
<td align="right">182</td>
<td align="right">551</td>
</tr>
</tbody>
</table>
<p><strong>A diagram version:</strong></p>
<div class="figure" style="text-align: center">
<img src="asset/0526-1.png" alt="Are trans-eQTLs cis-e/sQTLs?"  />
<p class="caption">
Are trans-eQTLs cis-e/sQTLs?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0526-1">
Past versions of 0526-1.png
</button>
</p>
<div id="fig-0526-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0526-1.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations:</strong></p>
<ul>
<li>For DGN, among 313 trans signals, 156 (~49.8%) are also cis-eQTLs,
corresponding to 39 eGenes, including:</li>
</ul>
<p><em>AC016753.7, BECN1, C17orf50, C2orf68, CFL2, CTD-2562J15.6, ERN1,
FAM177A1, FIGNL1, FKRP, GGCX, HABP4, HSD17B1P1, ICAM2, KIAA0391, MAT2A,
MLX, NEK6, NFE2, NFKBIA, OR2W3, PARP2, PEX12, PPP2R3C, PRKD2, PSMC3IP,
RETSAT, RNF181, RP11-1094M14.8, RP11-69M1.4, RP11-85K15.2, SLC1A5,
SLC2A3, SLFN12L, SYMPK, TMEM150A, TTC5, VAMP8, ZNF782</em></p>
<ul>
<li><p>For DGN, 101 (~32.3%) out of 313 trans signals are cis-sQTLs,
corresponding to 71 introns. 26 trans-eQTLs are only sQTLs but not
eQTLs. These signals are located on chromosome 15 and 19.</p></li>
<li><p>Among 313 trans signals, 182 (~58.1%) are either cis-eQTLs or
cis-sQTLs.</p></li>
</ul>
</div>
</div>
<div id="may-19" class="section level2">
<h2>May 19</h2>
<div
id="how-many-trans-eqtls-detected-in-dgngtex-whole-blood-are-replicated-in-eqtlgen"
class="section level3">
<h3>How many trans-eQTLs detected in DGN/GTEx whole blood are replicated
in eQTLGen?</h3>
<p>Replication of DGN signals in eQTLGen. Based on updated results,
i.e. smaller and more gene modules &amp; PCO<span
class="math inline">\(_{\lambda&gt;0.1}\)</span>.</p>
</div>
<div id="are-trans-eqtls-cis-eqtls-1" class="section level3">
<h3>Are trans-eQTLs cis-eQTLs?</h3>
<ol style="list-style-type: decimal">
<li>Dataset: DGN</li>
</ol>
<table>
<colgroup>
<col width="13%" />
<col width="8%" />
<col width="25%" />
<col width="24%" />
<col width="28%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="right">#signal</th>
<th align="right">is_ciseQTL(gene-based)</th>
<th align="right">is_cissQTL(SNP-based)</th>
<th align="right">is_cise(s)QTL(gene-based)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="right">313</td>
<td align="right">7</td>
<td align="right">9</td>
<td align="right">15</td>
</tr>
<tr class="even">
<td align="left">eQTLGen_DGN</td>
<td align="right">784</td>
<td align="right">17</td>
<td align="right">30</td>
<td align="right">45</td>
</tr>
</tbody>
</table>
<p><strong>Observations:</strong></p>
<p>1.1 DGN</p>
<ul>
<li><p>Among 313 trans signals, 7 are also (gene-based) cis-eQTLs,
corresponding to 5~7 independent SNPs. The cis-eGenes include
<strong>SLC2A3, NFE2, KIAA0391, NFKBIA, ERN1, AC016753.7,
MAT2A</strong>.</p></li>
<li><p>Among 313 unique trans signals, <strong>229 (~73%)</strong> have
a neareset gene that are eGenes (with cis-eQTLs). Among 28 independent
trans signals, <strong>19 (~68%)</strong> have a neareset gene that are
eGenes (with cis-eQTLs).</p></li>
<li><p>Among 313 trans signals, 9 are also cis-sQTLs, corresponding to 6
independent SNPs.</p></li>
</ul>
<p>1.2 eQTLGen_DGN</p>
<ul>
<li><p>Among 784 trans signals, 17 are also (gene-based) cis-eQTLs,
corresponding to 9 independent SNPs. The cis-eGenes include
<strong>AC016753.7, CPEB4, KIAA0391, MAT2A, MRFAP1, NFKBIA, PAQR6, PWP1,
RNF181, RP11-539L10.2, RP11-69M1.4, S100P, SENP7, SLC25A44, SLC2A3,
SMG5, TMEM79, ZNF782</strong>.</p></li>
<li><p>Among 784 unique trans signals, <strong>590 (~75%)</strong> have
a neareset gene that are eGenes (with cis-eQTLs). Among 50 independent
trans signals, <strong>29 (~58%)</strong> have a neareset gene that are
eGenes (with cis-eQTLs).</p></li>
<li><p>Among 784 trans signals, 30 are also cis-sQTLs, corresponding to
10 independent SNPs.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><p>Dataset: TCGA</p></li>
<li><p>eQTLGen?</p></li>
</ol>
</div>
</div>
<div id="march-8" class="section level2">
<h2>March 8</h2>
<div id="muscle" class="section level3">
<h3>Muscle</h3>
<ul>
<li><p>Less modules, PCO<span class="math inline">\(_{\{\lambda:
\lambda&gt;0.1\}}\)</span>. 1 independent signal, large p.</p></li>
<li><p>More modules, PCO<span class="math inline">\(_{\{\lambda:
\lambda&gt;0.1\}}\)</span>. 5 independent signal, small p.</p></li>
<li><p>More modules, PCO<span class="math inline">\(_{\{\lambda:
\lambda&gt;1\}}\)</span>. 5 independent signal, relatively large
p.</p></li>
</ul>
</div>
</div>
<div id="march-1" class="section level2">
<h2>March 1</h2>
<div id="real-data-analysis-by-pco-using-more-lambdas"
class="section level3">
<h3>Real data analysis by PCO using more <span
class="math inline">\(\lambda\)</span>’s</h3>
<p>Dataset: Muscle_Skeletal.</p>
<p>Fewer (independent) signals. <span
class="math inline">\(PCO_{\{\lambda: \lambda&gt;0.1\}}\)</span> (1
signal) v.s. <span class="math inline">\(PCO_{\{\lambda:
\lambda&gt;1\}}\)</span> (3 signals).</p>
</div>
<div id="proteomics-data" class="section level3">
<h3>proteomics data</h3>
<p>See the google doc <a
href="https://docs.google.com/spreadsheets/d/1PizRyY3VnvTHxu3PIM2_2sLbrj0Q_mxzRC9TrIm7vi4/edit?usp=sharing">here</a>.</p>
</div>
<div id="other-stuff" class="section level3">
<h3>Other stuff</h3>
<ul>
<li>RCC account storage</li>
</ul>
</div>
</div>
<div id="feb-23" class="section level2">
<h2>Feb 23</h2>
</div>
<div id="updated-simulation" class="section level2">
<h2>Updated simulation</h2>
<ol style="list-style-type: decimal">
<li>Adjusted variance of effects</li>
</ol>
<ul>
<li><p>tests</p>
<ul>
<li><p>Oracle: <span class="math inline">\(\beta^T \Sigma^{-1}Z \approx
\beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k:
\lambda_k&gt;0.1\}}\lambda_k u_k u_k^T) Z\)</span></p></li>
<li><p>PCO: Use <span class="math inline">\(\{\lambda_k:
\lambda_k&gt;0.1\}\)</span>.</p></li>
</ul></li>
<li><p>scenerios (<span class="math inline">\(K=100,K=105\)</span>)</p>
<ul>
<li><p>N: var.b = 0.003; caus = 30%; N.seq = c(200, 400, 600, 800,
1000)</p></li>
<li><p>caus: var.b = 0.003; N = 500; caus.seq = c(1, 5, 10, 30, 50, 70,
100)</p></li>
<li><p>caus_fix: var.b.fix = 0.1; N = 500; caus.seq = c(1, 5, 10, 30,
50, 70, 100)</p></li>
</ul></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><p>Run for other <span class="math inline">\(\Sigma\)</span>’s.
<span class="math inline">\(K=100,K=105\)</span></p></li>
<li><p>FDR level</p></li>
</ol>
<p>The minimum null pvalue of PCO is <span class="math inline">\(\ge
10^{-9}\)</span>, similar as the significance threshold we use here
<span class="math inline">\(10^{-9}\)</span>.</p>
<ol start="4" style="list-style-type: decimal">
<li>Results</li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/0223_power.lambda0.1.K100.png" alt="power under Bonferroni corretion cutoff"  />
<p class="caption">
power under Bonferroni corretion cutoff
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0223_power.lambda0.1.K100">
Past versions of 0223_power.lambda0.1.K100.png
</button>
</p>
<div id="fig-0223_power.lambda0.1.K100" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0223_power.lambda0.1.K100.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0223_power.lambda0.1.K105.png" alt="Another Sigma (K=105)"  />
<p class="caption">
Another Sigma (K=105)
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0223_power.lambda0.1.K105">
Past versions of 0223_power.lambda0.1.K105.png
</button>
</p>
<div id="fig-0223_power.lambda0.1.K105" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0223_power.lambda0.1.K105.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="feb-16" class="section level2">
<h2>Feb 16</h2>
<div id="simulation" class="section level3">
<h3>simulation</h3>
<ul>
<li><p>tests</p>
<ul>
<li><p>Oracle: <span class="math inline">\(\beta^T \Sigma^{-1}Z \approx
\beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k:
\lambda_k&gt;1\}}\lambda_k u_k u_k^T) Z\)</span></p></li>
<li><p>PC1;</p></li>
<li><p>minp = <span class="math inline">\(min\{p_1, \dots, p_K \} \times
K\)</span></p></li>
<li><p>PCO: Use <span class="math inline">\(\{\lambda_k:
\lambda_k&gt;1\}\)</span>.</p></li>
</ul></li>
<li><p>Multiple testing</p>
<p>The significance threshold is set based on the Bofferroni correction
as the <span class="math inline">\(\frac{0.05}{50\times10^6} =
10^{-9}\)</span> for univariate test and PC-based tests.</p></li>
<li><p>scenerios (<span class="math inline">\(K=100\)</span>)</p>
<ul>
<li><p>N: var.b = 0.01; caus = 100 (100%); N.seq = c(200, 400, 600, 800,
1000)</p></li>
<li><p>caus: var.b = 0.02; N = 500; caus.seq = c(1, 10, 30, 50, 70,
100)</p></li>
<li><p>caus_fix: var.b.fix = 0.2; N = 500; caus.seq = c(1, 10, 30, 50,
70, 100)</p></li>
</ul></li>
<li><p>results</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/0216_power.lambda1.png" alt="power under Bonferroni corretion cutoff"  />
<p class="caption">
power under Bonferroni corretion cutoff
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0216_power.lambda1">
Past versions of 0216_power.lambda1.png
</button>
</p>
<div id="fig-0216_power.lambda1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0216_power.lambda1.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Try other tests</p>
<ul>
<li><p>Oracle: <span class="math inline">\(\beta^T \Sigma^{-1}Z \approx
\beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k:
\lambda_k&gt;0.1\}}\lambda_k u_k u_k^T) Z\)</span></p></li>
<li><p>PCO: Use <span class="math inline">\(\{\lambda_k:
\lambda_k&gt;0.1\}\)</span>.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/0216_power.lambda0.1.png" alt="power under Bonferroni corretion cutoff"  />
<p class="caption">
power under Bonferroni corretion cutoff
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0216_power.lambda0.1">
Past versions of 0216_power.lambda0.1.png
</button>
</p>
<div id="fig-0216_power.lambda0.1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0216_power.lambda0.1.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>For talk</li>
</ul>
<p>(This figure should be updated as for the case of ‘u1’, the threshold
is falsely set as 0.05, which should be <span
class="math inline">\(10^{-9}\)</span>.)</p>
<div class="figure" style="text-align: center">
<img src="asset/0216_talk.png" alt="PC1 v.s. PCO"  />
<p class="caption">
PC1 v.s. PCO
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0216_talk">
Past versions of 0216_talk.png
</button>
</p>
<div id="fig-0216_talk" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0216_talk.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="jan-26" class="section level2">
<h2>Jan 26</h2>
<div id="simulation-1" class="section level3">
<h3>simulation</h3>
<ul>
<li><p>Methods: Oracle, PC1, PCO, MinP, minp.</p></li>
<li><p>Models: <span class="math inline">\(z \sim N(\sqrt{N} \beta,
\Sigma)\)</span>. <span class="math inline">\(\beta \sim N(0,
\sigma_{b}^2 I_{K \times K})\)</span>.</p></li>
</ul>
<ol style="list-style-type: decimal">
<li><p>various sample size <span class="math inline">\(N\)</span>. 200,
400, 600, 800, 1000.</p></li>
<li><p>various <span class="math inline">\(\sigma_{b}^2\)</span>. 0.005,
0.01, 0.05, 0.1, 0.2.</p></li>
<li><p>various causality percentage. ???</p></li>
</ol>
<ul>
<li><p>simulation.</p>
<p>Generate <span class="math inline">\(10^4\)</span> zscores under each
model. <span class="math inline">\(10^4\)</span> simulations.</p></li>
</ul>
<p>The following plot gives the boxplot, mean plot with standard
deviation, and mean plot with standard error of the mean. Which one to
use?</p>
<div class="figure" style="text-align: center">
<img src="asset/0126_power_05.png" alt="power under cutoff 0.05"  />
<p class="caption">
power under cutoff 0.05
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_power_05">
Past versions of 0126_power_05.png
</button>
</p>
<div id="fig-0126_power_05" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_power_05.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0126_plot.error_bar.png" alt="power comparison"  />
<p class="caption">
power comparison
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_plot.error_bar">
Past versions of 0126_plot.error_bar.png
</button>
</p>
<div id="fig-0126_plot.error_bar" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_plot.error_bar.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0126_test.png" alt="bar plot"  />
<p class="caption">
bar plot
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_test">
Past versions of 0126_test.png
</button>
</p>
<div id="fig-0126_test" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_test.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="larger-fdr-level-signals-in-large-and-small-module"
class="section level3">
<h3>Larger FDR level, #signals in large and small module?</h3>
<p>Here I will look at how the number of signals changes in the original
large and current small modules when I increase the FDR level.</p>
<p>The plot below is based on Muscle tissue from GTEx_v8. The x-axis is
for various FDR levels, including 5%, 10%, 15%, and 20%. The y-axis is
for the number of signals under these FDR levels. I plot three types of
signals, i.e. module and SNP pairs (left subplot), unique SNPs (middle
subplot), independent SNPs (right subplot). The two colors represent our
previous WGCNA modules (red) and current smaller WGCNA modules
(blue).</p>
<p>We can observe (1) When increasing FDR levels, number of signals only
increase a little bit, say 3-5 more signals. This means that the
corrected pvalues are bipolar, i.e. either very small or very large. (2)
Signals based on smaller modules are similar to that of previous
standard modules.</p>
<div class="figure" style="text-align: center">
<img src="asset/0126_signalsv.s.FDR.Muscle.png" alt="Muscle: #signals v.s.FDR"  />
<p class="caption">
Muscle: #signals v.s.FDR
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_signalsv.s.FDR.Muscle">
Past versions of 0126_signalsv.s.FDR.Muscle.png
</button>
</p>
<div id="fig-0126_signalsv.s.FDR.Muscle" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_signalsv.s.FDR.Muscle.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0126_signalsv.s.FDR.DGN.png" alt="DGN: #signals v.s.FDR"  />
<p class="caption">
DGN: #signals v.s.FDR
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_signalsv.s.FDR.DGN">
Past versions of 0126_signalsv.s.FDR.DGN.png
</button>
</p>
<div id="fig-0126_signalsv.s.FDR.DGN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_signalsv.s.FDR.DGN.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="smaller-gene-modules-more-signals" class="section level3">
<h3>Smaller gene modules, more signals?</h3>
<p>I changed the parameter <code>deepSplit</code> in function
<code>cutreeDynamic</code> from package WGCNA to get modules with
smaller size. I run it on the Muscle tissue from GTEx (see
<code>GTEx_v8.Muscle_Skeletal.WGCNA</code> in the GTEx results table).
Compared to <code>GTEx_v8.Muscle_Skeletal</code>, though more modules
(32 v.s. 18), they have same independent signals (3), located on chr5,
chr10, and chr22.</p>
<p>The corresponding modules of these signals from small and large WGCNA
share a large proportion of genes.</p>
<table style="width:100%;">
<colgroup>
<col width="15%" />
<col width="30%" />
<col width="28%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">signal.Chr</th>
<th align="left">module.small (#genes)</th>
<th align="left">module.big (#genes)</th>
<th align="right">#overlapped genes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr5</td>
<td align="left">module21 (89)</td>
<td align="left">module15 (114)</td>
<td align="right">88</td>
</tr>
<tr class="even">
<td align="left">chr10</td>
<td align="left">module21 (89)</td>
<td align="left">module15 (114)</td>
<td align="right">88</td>
</tr>
<tr class="odd">
<td align="left">chr22</td>
<td align="left">module2 (297)</td>
<td align="left">module4 (350)</td>
<td align="right">257</td>
</tr>
</tbody>
</table>
<p>The above table is for clusters obtained by method
‘WGCNA.min20_deep2’ (big) and ‘WGCNA.min20_deep4’ (small). Next, I give
more info about the clusters by more methods, i.e. #genes in modules
(A), #unclassfied genes (B), and #modules (C) by the six methods. I also
give the pvalue distribution corresponding to these clusters below.</p>
<p>Generally, funcExplorer gives more modules, smaller module, and much
more unclassified genes. Though the smaller modules, there aren’t many
extreme pvalues. Say, the minimum p of all is about <span
class="math inline">\(10^{-9}\)</span>, as compared to <span
class="math inline">\(10^{-14}\)</span> by other methods.</p>
<p>From the pvalue plot, it looks like ‘WGCNA.min10_deep4’ gives smaller
clusters and some extreme pvalues. So next step I will run the whole
pipleline for this setting.</p>
<div class="figure" style="text-align: center">
<img src="asset/0126_modules.png" alt="Modules v.s. methods/parameters"  />
<p class="caption">
Modules v.s. methods/parameters
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_modules">
Past versions of 0126_modules.png
</button>
</p>
<div id="fig-0126_modules" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_modules.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0126_Muscle_Skeleta.p.funcExplorer.png" alt="p distribution of Muscle"  />
<p class="caption">
p distribution of Muscle
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0126_Muscle_Skeleta.p.funcExplorer">
Past versions of 0126_Muscle_Skeleta.p.funcExplorer.png
</button>
</p>
<div id="fig-0126_Muscle_Skeleta.p.funcExplorer" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0126_Muscle_Skeleta.p.funcExplorer.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="jan-05-jan-19" class="section level2">
<h2>Jan 05, Jan 19</h2>
<div id="simulation-2" class="section level3">
<h3>Simulation</h3>
<p>The simulations here aim to compare the power of PC-based tests
including PC1 (using only the primary PC) and PCO (using combined PC’s),
and non-PC based test MinP. The simulations consist of two parts: (1)
verify that type 1 error is well controlled; (2) compare power of tests.
At this time, I simply assume there is no LD among SNPs, i.e. tests are
independent.</p>
<ol style="list-style-type: decimal">
<li>Verify that type 1 error is well controlled.</li>
</ol>
<p>Since the tests are assumed to be independent, the null pvalues
should be uniformly distributed. To verify that, I estimate the null
pvalues by empirical p’s obtained from the following steps.</p>
<p>Step1. Generate <span class="math inline">\(z_0 \sim N(0, \Sigma) |
H_0\)</span>.</p>
<p>Step2. Run tests including PC1, PCO, MinP for <span
class="math inline">\(10^6\)</span> times and obtain <span
class="math inline">\(10^6\)</span> p’s for each test.</p>
<p>Step3. Compute the empirical T1E at significance <span
class="math inline">\(\alpha\)</span> by <span
class="math inline">\(\frac{I\{p&lt;\alpha\}}{10^6}\)</span>. Draw
qqplot of <span class="math inline">\(-log_{10}p\)</span>.</p>
<p>The correlation matrix <span class="math inline">\(\Sigma\)</span> is
the correlation of genes in module15 of GTEx dataset Muscle_Skeletal
(Sigma.Muscle_Skeletal.module15.chr7.rds).</p>
<table>
<thead>
<tr class="header">
<th align="right">alpha</th>
<th align="right">PC1</th>
<th align="right">PCO</th>
<th align="left">MinP</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">5e-02</td>
<td align="right">0.050356</td>
<td align="right">0.053584</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="right">1e-04</td>
<td align="right">0.000093</td>
<td align="right">0.000114</td>
<td align="left"></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="asset/0119_qqplot.simulation.null.png" alt="qqplot.simulation.null"  />
<p class="caption">
qqplot.simulation.null
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_qqplot.simulation.null">
Past versions of 0119_qqplot.simulation.null.png
</button>
</p>
<div id="fig-0119_qqplot.simulation.null" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_qqplot.simulation.null.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>Compare power of tests.</li>
</ol>
<p>To compare the power of non-/PC-based tests, I will run the oracle
test in addition to PC1, PCO, and MinP to benchmark the power. It’s
mentioned in the PCO paper that given a fixed correlation relationship
among the phenotypes, the power of a test depends on the relationship
bewteen the true effects (<span class="math inline">\(\beta\)</span>)
and the phenotype correlation (<span
class="math inline">\(\Sigma\)</span>). Therefore, to see which tests
perform well in what cases, I will consider different models, including
(1) <span class="math inline">\(\beta=10 u_1\)</span>; (2) <span
class="math inline">\(\beta=4 u_k\)</span>; (3) <span
class="math inline">\(\beta=1.5 u_K\)</span>; (4) <span
class="math inline">\(\beta=rnorm(K)\)</span>; (5) <span
class="math inline">\(\beta=rnorm(0.7K)\)</span>; (6) <span
class="math inline">\(\beta=rnorm(0.3K)\)</span>.</p>
<p>Step1. Generate <span class="math inline">\(z_0 \sim N(\beta, \Sigma)
| H_1\)</span>.</p>
<p>Step2. Run tests including Oracle, PC1, PCO, MinP for <span
class="math inline">\(10^6\)</span> times and obtain <span
class="math inline">\(10^6\)</span> p’s for each test.</p>
<p>Step3. Compute the power at significance <span
class="math inline">\(\alpha\)</span> by <span
class="math inline">\(\frac{I\{p&lt;\alpha\}}{10^6}\)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="left">Model</th>
<th align="right">Oracle</th>
<th align="right">PC1</th>
<th align="right">PCO</th>
<th align="left">MinP</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">u1</td>
<td align="right">0.946341</td>
<td align="right">0.902540</td>
<td align="right">0.746301</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">uPCO</td>
<td align="right">0.989835</td>
<td align="right">0.050178</td>
<td align="right">0.713434</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">uK</td>
<td align="right">0.953727</td>
<td align="right">0.049719</td>
<td align="right">0.054140</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">100%</td>
<td align="right">1.000000</td>
<td align="right">0.058867</td>
<td align="right">0.945918</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">70%</td>
<td align="right">1.000000</td>
<td align="right">0.058910</td>
<td align="right">0.580368</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">30%</td>
<td align="right">0.999995</td>
<td align="right">0.050201</td>
<td align="right">0.149904</td>
<td align="left"></td>
</tr>
</tbody>
</table>
</div>
<div id="gtex" class="section level3">
<h3>GTEx</h3>
<div id="results-4" class="section level4">
<h4>Results</h4>
<p>The table below summarizes all results I have so far, followed by
figures of the distributions of pvalues in various datasets.</p>
<table style="width:100%;">
<colgroup>
<col width="21%" />
<col width="4%" />
<col width="3%" />
<col width="6%" />
<col width="5%" />
<col width="12%" />
<col width="3%" />
<col width="4%" />
<col width="12%" />
<col width="3%" />
<col width="6%" />
<col width="6%" />
<col width="9%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="left">Nsample</th>
<th align="left">All</th>
<th align="left">Annotated</th>
<th align="left">Filtered</th>
<th align="left">Filtered_info</th>
<th align="left">Final</th>
<th align="left">Nmodule</th>
<th align="left">FDR</th>
<th align="left">Nperm</th>
<th align="left">QTL_Module</th>
<th align="left">unique_QTL</th>
<th align="left">independent_QTL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">11979</td>
<td align="left">8120</td>
<td align="left">0;165;1663;8073</td>
<td align="left">3859</td>
<td align="left">21</td>
<td align="left">combined chr+module</td>
<td align="left">20</td>
<td align="left">659</td>
<td align="left">623</td>
<td align="left">40</td>
</tr>
<tr class="even">
<td align="left">DGN_new</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">12585</td>
<td align="left">9939</td>
<td align="left">1;453;1798;8705</td>
<td align="left">3695</td>
<td align="left">19</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">331</td>
<td align="left">275</td>
<td align="left">26</td>
</tr>
<tr class="odd">
<td align="left">TCGA</td>
<td align="left">788</td>
<td align="left">17656</td>
<td align="left">15994</td>
<td align="left">12694</td>
<td align="left">336;3504;10697;598</td>
<td align="left">4962</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Whole_Blood</td>
<td align="left">670</td>
<td align="left">20315</td>
<td align="left">20315</td>
<td align="left">15245</td>
<td align="left">619;4544;3677;13096</td>
<td align="left">5070</td>
<td align="left">22</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">4</td>
<td align="left">4</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Muscle_Skeletal</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">15601</td>
<td align="left">675;4447;3631;13490</td>
<td align="left">5430</td>
<td align="left">18</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">38</td>
<td align="left">38</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Skin_Sun_Exposed_Lower_leg</td>
<td align="left">605</td>
<td align="left">25196</td>
<td align="left">25196</td>
<td align="left">18801</td>
<td align="left">807;6458;4557;15650</td>
<td align="left">6395</td>
<td align="left">30</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">1</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Artery_Tibial</td>
<td align="left">584</td>
<td align="left">23304</td>
<td align="left">23304</td>
<td align="left">17390</td>
<td align="left">795;5588;4202;14730</td>
<td align="left">5914</td>
<td align="left">20</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">0</td>
<td align="left">0</td>
<td align="left">0</td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Muscle_Skeletal.cross</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">7141</td>
<td align="left">675;4447;3631;0</td>
<td align="left">13890</td>
<td align="left">39</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">43</td>
<td align="left">43</td>
<td align="left">4</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Muscle_Skeletal.WGCNA</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">15601</td>
<td align="left">675;4447;3631;13490</td>
<td align="left">5430</td>
<td align="left">32</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">44</td>
<td align="left">44</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">DGN_new.WGCNA</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">12585</td>
<td align="left">9939</td>
<td align="left">1;453;1798;8705</td>
<td align="left">3695</td>
<td align="left">40</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">TCGA_new</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="asset/0119_DGN.p.png" alt="DGN" width="70%" height="70%" />
<p class="caption">
DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_DGN.p">
Past versions of 0119_DGN.p.png
</button>
</p>
<div id="fig-0119_DGN.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_DGN.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_DGN_new.p.png" alt="DGN_new" width="70%" height="70%" />
<p class="caption">
DGN_new
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_DGN_new.p">
Past versions of 0119_DGN_new.p.png
</button>
</p>
<div id="fig-0119_DGN_new.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_DGN_new.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_Whole_Blood.p.png" alt="Whole_Blood" width="70%" height="70%" />
<p class="caption">
Whole_Blood
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_Whole_Blood.p">
Past versions of 0119_Whole_Blood.p.png
</button>
</p>
<div id="fig-0119_Whole_Blood.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_Whole_Blood.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_Muscle_Skeletal.p.png" alt="Muscle_Skeletal" width="70%" height="70%" />
<p class="caption">
Muscle_Skeletal
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_Muscle_Skeletal.p">
Past versions of 0119_Muscle_Skeletal.p.png
</button>
</p>
<div id="fig-0119_Muscle_Skeletal.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_Muscle_Skeletal.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_Skin_Sun_Exposed_Lower_leg.p.png" alt="Skin_Sun_Exposed_Lower_leg" width="70%" height="70%" />
<p class="caption">
Skin_Sun_Exposed_Lower_leg
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_Skin_Sun_Exposed_Lower_leg.p">
Past versions of 0119_Skin_Sun_Exposed_Lower_leg.p.png
</button>
</p>
<div id="fig-0119_Skin_Sun_Exposed_Lower_leg.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_Skin_Sun_Exposed_Lower_leg.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_Artery_Tibial.p.png" alt="Artery_Tibial" width="70%" height="70%" />
<p class="caption">
Artery_Tibial
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_Artery_Tibial.p">
Past versions of 0119_Artery_Tibial.p.png
</button>
</p>
<div id="fig-0119_Artery_Tibial.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_Artery_Tibial.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/0119_Muscle_Skeletal.cross.p.png" alt="Muscle_Skeletal.cross" width="70%" height="70%" />
<p class="caption">
Muscle_Skeletal.cross
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_Muscle_Skeletal.cross.p">
Past versions of 0119_Muscle_Skeletal.cross.p.png
</button>
</p>
<div id="fig-0119_Muscle_Skeletal.cross.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_Muscle_Skeletal.cross.p.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="remarks" class="section level4">
<h4>Remarks</h4>
<ol style="list-style-type: decimal">
<li><p>The dataset “DGN_new” represents DGN through the standard
filtering (see “Gene filter” on Dec 10).</p></li>
<li><p>GTEx datasets generally have few significant signals and
relatively large pvalues (compared to DGN).</p></li>
<li><p>The dataset “GTEx_v8.Muscle_Skeletal.cross” uses Muscle_Skeletal
samples from GTEx_v8 (similar as GTEx_v8.Muscle_Skeletal), but without
removing cross-mapped genes before constructing gene modules. Therefore,
there are 13890 genes (v.s. 5430 in GTEx_v8.Muscle_Skeletal) in total
which result in 39 modules (v.s. 18 in GTEx_v8.Muscle_Skeletal). We do
this step because we observed that there are relatively few signals
using our original pipeline and we wonder if the reason to this
observation is us filtering too many genes in the first step and leaving
too few signals. To check on this, we put the “filtering” to the last
step, i.e. including potentially cross-mapped genes into the analysis
and generate significant variant-module pairs. We then exclude those
where target eGene in the module is cross-mappable with any gene within
1Mb of the variant. Hopefully we could have more signals.</p>
<p>However, though the increased genes and modules, the number of
identified signals (43) is similar as that using the original pipeline
(38). Next, I will look into these signals.</p></li>
</ol>
<table>
<caption>GTEx_v8.Muscle_Skeletal</caption>
<thead>
<tr class="header">
<th align="left">variant-module</th>
<th align="left">p</th>
<th align="left">q</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module15:10:48930105</td>
<td align="left">8.63e-10</td>
<td align="left">8.33e-03</td>
</tr>
<tr class="even">
<td align="left">module15:5:132440814</td>
<td align="left">1.56e-09</td>
<td align="left">2.16e-02</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132448891</td>
<td align="left">3.36e-09</td>
<td align="left">4.74e-02</td>
</tr>
<tr class="even">
<td align="left">module15:5:132450078</td>
<td align="left">5.81e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132450726</td>
<td align="left">5.96e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132450916</td>
<td align="left">8.89e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132451361</td>
<td align="left">5.97e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132451586</td>
<td align="left">1.11e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132453865</td>
<td align="left">1.05e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132454053</td>
<td align="left">9.94e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132454171</td>
<td align="left">1.03e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132454631</td>
<td align="left">1.44e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132454724</td>
<td align="left">1.08e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132455672</td>
<td align="left">1.07e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132455979</td>
<td align="left">1.04e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132456154</td>
<td align="left">9.96e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132456710</td>
<td align="left">6.04e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132458606</td>
<td align="left">5.33e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132459905</td>
<td align="left">4.84e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132459971</td>
<td align="left">5.04e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132460190</td>
<td align="left">5.05e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132460375</td>
<td align="left">5.56e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132460917</td>
<td align="left">5.03e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132461111</td>
<td align="left">5.17e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132463834</td>
<td align="left">5.47e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132464413</td>
<td align="left">3.90e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132464907</td>
<td align="left">1.01e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132466034</td>
<td align="left">4.56e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132468333</td>
<td align="left">1.32e-10</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132468353</td>
<td align="left">5.74e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132468564</td>
<td align="left">8.53e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132469724</td>
<td align="left">5.63e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132469899</td>
<td align="left">6.96e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132470043</td>
<td align="left">1.03e-10</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132470796</td>
<td align="left">5.42e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132471932</td>
<td align="left">8.01e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132474927</td>
<td align="left">2.71e-10</td>
<td align="left">2.94e-03</td>
</tr>
<tr class="even">
<td align="left">module4:22:23508295</td>
<td align="left">3.76e-10</td>
<td align="left">5.71e-03</td>
</tr>
</tbody>
</table>
<table>
<caption>GTEx_v8.Muscle_Skeletal.cross</caption>
<thead>
<tr class="header">
<th align="left">variant-module</th>
<th align="left">p</th>
<th align="left">q</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module8:1:123955560</td>
<td align="left">9.97e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:1:123955561</td>
<td align="left">1.06e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:16:36353041</td>
<td align="left">3.49e-11</td>
<td align="left">6.67e-03</td>
</tr>
<tr class="even">
<td align="left">module8:16:36353056</td>
<td align="left">8.29e-11</td>
<td align="left">1.25e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132453865</td>
<td align="left">1.56e-09</td>
<td align="left">3.12e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132454053</td>
<td align="left">1.59e-09</td>
<td align="left">2.86e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132454171</td>
<td align="left">1.52e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132454631</td>
<td align="left">1.43e-09</td>
<td align="left">3.57e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132454724</td>
<td align="left">1.53e-09</td>
<td align="left">3.23e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132455672</td>
<td align="left">1.50e-09</td>
<td align="left">3.45e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132455979</td>
<td align="left">1.56e-09</td>
<td align="left">3.03e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132456154</td>
<td align="left">1.78e-09</td>
<td align="left">3.42e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132456710</td>
<td align="left">1.36e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132458606</td>
<td align="left">3.37e-10</td>
<td align="left">1.76e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132459905</td>
<td align="left">6.50e-10</td>
<td align="left">1.74e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132459971</td>
<td align="left">3.92e-10</td>
<td align="left">1.43e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132460190</td>
<td align="left">3.50e-10</td>
<td align="left">1.58e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132460375</td>
<td align="left">3.37e-10</td>
<td align="left">1.67e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132460917</td>
<td align="left">3.98e-10</td>
<td align="left">1.36e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132461111</td>
<td align="left">3.57e-10</td>
<td align="left">1.50e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132463834</td>
<td align="left">8.21e-10</td>
<td align="left">2.40e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132464413</td>
<td align="left">9.93e-10</td>
<td align="left">2.69e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132466034</td>
<td align="left">6.75e-10</td>
<td align="left">1.67e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132468333</td>
<td align="left">1.86e-09</td>
<td align="left">3.50e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132468353</td>
<td align="left">1.69e-09</td>
<td align="left">3.06e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132468564</td>
<td align="left">2.43e-09</td>
<td align="left">4.19e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132469724</td>
<td align="left">1.93e-09</td>
<td align="left">3.66e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132469899</td>
<td align="left">1.73e-09</td>
<td align="left">3.24e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132470043</td>
<td align="left">1.56e-09</td>
<td align="left">2.94e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132470796</td>
<td align="left">1.85e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132471932</td>
<td align="left">2.37e-09</td>
<td align="left">4.29e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:46659000</td>
<td align="left">9.86e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:46906585</td>
<td align="left">2.34e-11</td>
<td align="left">7.14e-03</td>
</tr>
<tr class="even">
<td align="left">module8:5:47027928</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47068540</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47166316</td>
<td align="left">3.33e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47210790</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47210990</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47258679</td>
<td align="left">4.44e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47258917</td>
<td align="left">3.33e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:49809727</td>
<td align="left">2.70e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:49930686</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:49936394</td>
<td align="left">1.10e-11</td>
<td align="left">0.00e+00</td>
</tr>
</tbody>
</table>
<pre><code>For GTEx_v8.Muscle_Skeletal, there are 38 variant-module pairs, corresponding to 2 module (module 15, module4) and 3 independent loci on (chr5, chr10, chr22). GTEx_v8.Muscle_Skeletal.cross has 43 variant-module pairs, corresponding to 1 module (module 8) and 4 independent loci on (chr1, chr5, chr16).

The signal on chr5 is significant for both module 15 (114 genes) and module 8 (394 genes) in two datasets, which have 82 shared genes. Take SNP [rs2706381][rs2706381 GTEx ref] (chr5:132474927) for example. It is [&quot;in cis with IRF1 ($P \le 2\times10^{-10}$; Fig. 6c), a transcription factor that facilitates regulation of the interferon-induced immune response&quot;][rs2706381 GTEx ref]. It is also [&quot;associated in trans with PSME1 ($P \le 1.1\times10^{-11}$) and PARP10 ($P \le 7.8\times10^{-10}$)&quot;][rs2706381 GTEx ref]. These two genes are included in module 15. The reference also gives additional results to &quot;suggest that cis-regulatory loci affecting IRF1 are regulators of interferon-responsive inflammatory processes involving genes including PSME1 and PARP10, with implications for complex traits specific to muscle tissue&quot;.

I also looked at the enrichment of the genes in module 15. These genes are mainly enriched in immunity-related terms and tuberculosis. To reproduce, use the gene list [here](asset/0119_gene.GTEx_v8.Muscle_Skeletal.module15.txt).

SNP 10:48930105. module 15.

module4:22:23508295

+ Check if the signals&#39; nearby genes are cross-mapped with genes in their corresponding module. The signal on chr5 corresponding to module8 has 20 genes +/- 500Kb away from its TSS. Many of them are crosspable with genes in the module 8. The number of of genes in the module crossmapped with each of the 20 genes are: `1, 144, 0, 62, 54, 0, 63, 4, 0, 0, 0, 59, 145, 0, 0, 128, 0, 23, 53`, where the nearest gene is crossmappable with 1 gene in the module.</code></pre>
<ul>
<li><p>The other signals on chr1, chr16, and chr5:47210790 don’t have
any nearby genes around its TSS.</p></li>
<li><p>See the following plot for number of cross-mapped genes in each
module.</p></li>
</ul>
<p><img src="asset/0119_nCross.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_nCross">
Past versions of 0119_nCross.png
</button>
</p>
<div id="fig-0119_nCross" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_nCross.png" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="new-tcga-by-the-standard-filtering" class="section level3">
<h3>New TCGA by the standard filtering</h3>
</div>
<div id="eqtlgen" class="section level3">
<h3>eQTLGen</h3>
<ol style="list-style-type: decimal">
<li><p>eQTLGen description</p>
<p><em>This full dataset includes 19942 genes that showed expression in
blood tested and 10317 SNPs that are trait-associated SNPs based on GWAS
Catalog.</em></p>
<p>After gene filter steps described in <code>Dec 10</code>, there are
4963 genes left. Applying the same filtering to 13634 DGN genes, there
are 3695 genes left, among which 3642 are also included in eQTLGen. So,
I will use these genes to do the downstream analysis, e.g. constructing
co-expressed gene modules. These 3642 genes result in 19 gene
modules.</p></li>
<li><p>Replication of DGN signals in eQTLGen</p>
<p>The table below gives the signals found in eQTLGen and DGN.</p>
<p>The first two rows give results based on eQTLGen zscores, with row 1
using <code>qvalue</code> for FDR correction (threshold <span
class="math inline">\(0.05\)</span>) and row 2 using <span
class="math inline">\(\frac{0.05}{\#DGN signals}\)</span> as
significance threshold. The third row is based on the same gene modules
and SNPs but tensorQTL zscores using DGN expression data. The FDR
correction uses the empirical distribution of pvalues from the combined
chr’s and modules (10 permutations).</p></li>
</ol>
<table>
<colgroup>
<col width="13%" />
<col width="32%" />
<col width="9%" />
<col width="15%" />
<col width="11%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="left">FDR</th>
<th align="right">minp</th>
<th align="right">(QTL, module)</th>
<th align="right">unique QTL</th>
<th align="right">independent QTL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">eQTLGen</td>
<td align="left">qvalue</td>
<td align="right">5.82e-04</td>
<td align="right">2195</td>
<td align="right">909</td>
<td align="right">348</td>
</tr>
<tr class="even">
<td align="left">eQTLGen</td>
<td align="left">0.05/#DGN signals</td>
<td align="right">1.18e-04</td>
<td align="right">1707</td>
<td align="right">762</td>
<td align="right">286</td>
</tr>
<tr class="odd">
<td align="left">eQTLGen_DGN</td>
<td align="left">combined chr+module #10-perms</td>
<td align="right">1.00e-07</td>
<td align="right">420</td>
<td align="right">374</td>
<td align="right">28</td>
</tr>
</tbody>
</table>
<p>Among 374 eQTLGen_DGN signals, 16 are replicated in 762 eQTLGen
signals. These 16 replicated signals consists of 6 independent SNPs,
including (based on GRCh37),</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs12485738">rs12485738</a>
: 3:56865776, intron variant of <em>ARHGEF3</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs643381">rs643381</a> :
6:139839423. (<a
href="https://www.ncbi.nlm.nih.gov/snp/rs590856">rs590856</a>,
6:139844429).</li>
<li><a
href="https://www.ncbi.nlm.nih.gov/snp/rs149007767">rs149007767</a>
7:50370254: , intron variant of <em>IKZF1</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs12718597">rs12718597</a>
: 7:50428428, intron variant of <em>IKZF1</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs35979828">rs35979828</a>
: 12:54685880, 500B downstream variant of <em>NFE2</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs7210990">rs7210990</a> :
17:16170764, intron variant of <em>PIGL</em>.</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Replication of DGN signals in eQTLGen</li>
</ol>
<div class="figure" style="text-align: center">
<img src="asset/0119_eQTLGen_DGN_venn.jpeg" alt="DGN" width="70%" height="70%" />
<p class="caption">
DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_eQTLGen_DGN_venn">
Past versions of 0119_eQTLGen_DGN_venn.jpeg
</button>
</p>
<div id="fig-0119_eQTLGen_DGN_venn" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9275134104895899d8e293e4982d51af90ae06ce/docs/asset/0119_eQTLGen_DGN_venn.jpeg" target="_blank">9275134</a>
</td>
<td>
liliw-w
</td>
<td>
2021-07-08
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] data.table_1.14.8 workflowr_1.7.0  

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       highr_0.10        compiler_4.2.3    pillar_1.9.0     
 [5] bslib_0.5.0       later_1.3.1       git2r_0.32.0      jquerylib_0.1.4  
 [9] tools_4.2.3       getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7   
[13] evaluate_0.21     lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3  
[17] rlang_1.1.1       cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7       
[21] xfun_0.39         fastmap_1.1.1     httr_1.4.6        stringr_1.5.0    
[25] knitr_1.43        fs_1.6.2          vctrs_0.6.3       sass_0.4.6       
[29] rprojroot_2.0.3   glue_1.6.2        R6_2.5.1          processx_3.8.2   
[33] fansi_1.0.4       rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3   
[37] whisker_0.4.1     ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5  
[41] httpuv_1.6.11     utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
