<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2023</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2023</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef1090b1fecef54522cbb07c77c3b22db0f7dfc3etargetblankf1090b1a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e" target="_blank">f1090b1</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef1090b1fecef54522cbb07c77c3b22db0f7dfc3etargetblankf1090b1a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e" target="_blank">f1090b1</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Unstaged changes:
    Modified:   analysis/Trans.Rmd
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2023_a_fresh_start.Rmd</code>)
and HTML (<code>docs/Log2023_a_fresh_start.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e/docs/Log2023_a_fresh_start.html" target="_blank">f1090b1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ee589703409e38f18cd760bf00a45681c72df3b6/docs/Log2023_a_fresh_start.html" target="_blank">ee58970</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0/docs/Log2023_a_fresh_start.html" target="_blank">c50b98f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/801032f881d9eb655999250ba37a61bae2e532a7/docs/Log2023_a_fresh_start.html" target="_blank">801032f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/24a6487ef9cf0acce6147b83c869fc123d73b0f9/analysis/Log2023_a_fresh_start.Rmd" target="_blank">24a6487</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
<td>
add archR peak calling details
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/052a48ccd36b09a5f9c76797a3a6d8bd3c8be15d/docs/Log2023_a_fresh_start.html" target="_blank">052a48c</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6289d6e2f7d4ed98ba31e74b350534925c543b8f/analysis/Log2023_a_fresh_start.Rmd" target="_blank">6289d6e</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
add weeks
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/analysis/Log2023_a_fresh_start.Rmd" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
<td>
updates to last week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/Log2023_a_fresh_start.html" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
<td>
updates to last week
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/analysis/Log2023_a_fresh_start.Rmd" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/17f3513a8f932600c6a08a4709de6fd87d490192/docs/Log2023_a_fresh_start.html" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bf01171fdb6e927955fd3500468d194b00c23ba0/docs/Log2023_a_fresh_start.html" target="_blank">bf01171</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/c34957bae42e29e3d3cb607c63032e73c63e55a6/analysis/Log2023_a_fresh_start.Rmd" target="_blank">c34957b</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-29
</td>
<td>
update a week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8ed8c82abde828dc1f3ea8b3ecfbdf95949dcf90/docs/Log2023_a_fresh_start.html" target="_blank">8ed8c82</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e446595eff32ea807f01aa455a23b119a412579e/analysis/Log2023_a_fresh_start.Rmd" target="_blank">e446595</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
<td>
Add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/210c2519c9b729e732e7f50cac31aace22b8b22a/docs/Log2023_a_fresh_start.html" target="_blank">210c251</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/dd31c888a6f4198b3bd9a962393373c2f45c6cb5/analysis/Log2023_a_fresh_start.Rmd" target="_blank">dd31c88</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-26
</td>
<td>
Add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8046bfaa70abeb1338e938838f26cc1754d444dc/docs/Log2023_a_fresh_start.html" target="_blank">8046bfa</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/46f2fde5deb67e2ca14c55523a3f43ca33ca43d0/analysis/Log2023_a_fresh_start.Rmd" target="_blank">46f2fde</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
<td>
Add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/64772c7f7de8e269e737f01eeec2be1dc2bace42/docs/Log2023_a_fresh_start.html" target="_blank">64772c7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5863abac67338aecc6ee1d931a3c4347019174a0/analysis/Log2023_a_fresh_start.Rmd" target="_blank">5863aba</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
<td>
add quick summary
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/61f98020cf8a33913295550933f197d16fa42691/docs/Log2023_a_fresh_start.html" target="_blank">61f9802</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c19b31946a56d03a5ca6c1d620d05f9b14b848b/analysis/Log2023_a_fresh_start.Rmd" target="_blank">6c19b31</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
<td>
Add new weeks
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a8ba9a35e0a05265e2ad801f12bada97b91ba752/analysis/Log2023_a_fresh_start.Rmd" target="_blank">a8ba9a3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-04
</td>
<td>
add results from using factor 1 and peak-based
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a8ba9a35e0a05265e2ad801f12bada97b91ba752/docs/Log2023_a_fresh_start.html" target="_blank">a8ba9a3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-04
</td>
<td>
add results from using factor 1 and peak-based
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0ee57c6050ffb139cb1e309383524bc47271b389/docs/Log2023_a_fresh_start.html" target="_blank">0ee57c6</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/39fcfad90d7b7bbdbd5ddf5af0cc27c6d5978df0/analysis/Log2023_a_fresh_start.Rmd" target="_blank">39fcfad</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d788d08437895f9b1518f27657847a581477dc66/docs/Log2023_a_fresh_start.html" target="_blank">d788d08</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3094ce997a05a84325f063673ce9eea4cdebacf7/analysis/Log2023_a_fresh_start.Rmd" target="_blank">3094ce9</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-24
</td>
<td>
add more details
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f5a165be0df7f78fcb8cbf9c5eb49cb71bb9797c/docs/Log2023_a_fresh_start.html" target="_blank">f5a165b</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/88593d492a7edee3dcdf85e9550875db6e27cec2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">88593d4</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/9ecc7a8af04512c57de40fb708845460f572f601/docs/Log2023_a_fresh_start.html" target="_blank">9ecc7a8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/7588eb75a18aee67634d632523da454d2f4cdb30/analysis/Log2023_a_fresh_start.Rmd" target="_blank">7588eb7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/analysis/Log2023_a_fresh_start.Rmd" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/Log2023_a_fresh_start.html" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/analysis/Log2023_a_fresh_start.Rmd" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/Log2023_a_fresh_start.html" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e225f5d45714ad8bf33e21a4da04a2c975cbf16e/docs/Log2023_a_fresh_start.html" target="_blank">e225f5d</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/analysis/Log2023_a_fresh_start.Rmd" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/15872511eb2e7eeb833bd01c6cc6b85473cc8c90/docs/Log2023_a_fresh_start.html" target="_blank">1587251</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/281c4602ae2a7ffa01409b9233c4b9fcee9272b2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">281c460</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
add reason why to test smaller window size
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/279fae6f6cb02c60adbf1d62cece753e127f1fc2/docs/Log2023_a_fresh_start.html" target="_blank">279fae6</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/analysis/Log2023_a_fresh_start.Rmd" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4104dc026ba662ac6d89da7b243ac22b89c55cf0/docs/Log2023_a_fresh_start.html" target="_blank">4104dc0</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/245fe228f05daceb01b1293b77c3cb5733f42c52/analysis/Log2023_a_fresh_start.Rmd" target="_blank">245fe22</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e7b284cf13b85c027f1760424017abe9745292c5/docs/Log2023_a_fresh_start.html" target="_blank">e7b284c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6e5a414fc6364413fa987028f07d382ec32dfd7f/analysis/Log2023_a_fresh_start.Rmd" target="_blank">6e5a414</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
correct a figure path
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a6d19e33b90fd7a2db46c7e4c55f5416b62f2828/docs/Log2023_a_fresh_start.html" target="_blank">a6d19e3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/bc0a2f7db3f7f82573f5f6297cebac528967c5c2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">bc0a2f7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/92bc59e2447452c7e7f39e4c8ed4eeb3d58a319b/docs/Log2023_a_fresh_start.html" target="_blank">92bc59e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/bf7aad20b6be25071e0a04bb971b7550ed02d8ff/analysis/Log2023_a_fresh_start.Rmd" target="_blank">bf7aad2</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/cdbf8d48acae909e8dae436520fbc510ffa7fd95/docs/Log2023_a_fresh_start.html" target="_blank">cdbf8d4</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f28308c60221d21a3fadd6691b79480fd5729a17/analysis/Log2023_a_fresh_start.Rmd" target="_blank">f28308c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Add a new wekk
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="dec-06" class="section level1">
<h1>Dec 06</h1>
<div id="run-cacti-p-on-all-cells-whole-blood" class="section level2">
<h2>2. Run CACTI-P on all cells (whole blood)</h2>
<div id="run-qtltools" class="section level3">
<h3>Run QTLTools</h3>
<p>To-do’s -</p>
<p><del>- Ask and confirm questions with Phoenix on slack</del></p>
<p><del>- Check if current QTLTools results fit our needs</del></p>
<p><del>- Re-run QTLTools if needed</del></p>
<ul>
<li>Check rasqual scripts &amp; results for pseodo-bulk (scqtl is not
what I need)</li>
</ul>
<p><strong>Take-aways -</strong></p>
<ul>
<li>Pseudo-bulk data is in rasqual folder. (scqtl folder is for single
cell)</li>
</ul>
</div>
<div id="basic-info" class="section level3">
<h3>Basic info</h3>
<ul>
<li><p>Whole blood (using all cells)</p></li>
<li><p>#individuals</p>
<ul>
<li><p>All individuals: 56</p></li>
<li><p>#cells for each individual</p></li>
<li><p>read coverage across individuals</p></li>
</ul></li>
<li><p>#peaks</p>
<ul>
<li><p>All peaks: 280,249</p></li>
<li><p>Peak width: 500bp</p></li>
<li><p>Peak distance:</p></li>
<li><p>#peaks across chr’s</p></li>
<li><p>Peak coverage</p></li>
<li><p>Signal peaks (fdr&lt;0.05, cis 500kb) by qtltools - 2561</p></li>
<li><p>Signal peaks (fdr&lt;0.05, cis 500kb) by rasqual -
11,848</p></li>
</ul></li>
<li><p>Window size to choose</p>
<p>Need to consider (1) window v.s. sample size, (2) ATAC-seq peak
charateristics.</p>
<ul>
<li><p>50kb?</p></li>
<li><p>#windows (non-overlapping) - 36,358</p></li>
<li><p>#peaks within windows (v.s. sample size) - 1~66. median 5 (v.s.
56)</p></li>
</ul></li>
<li><p>Cis region to use for association test</p>
<ul>
<li><p>Same as window size, 50kb, because …</p>
<ul>
<li><p>Many caQTLs are within or very close to peaks.</p></li>
<li><p>To make sure the leftmost and rightmost peaks in the window have
at least common cis snps within in the window.</p></li>
<li><p>To reduce computational burden. We have much more windows than
ChIP-seq data, and smaller window, so not to use same cis region
(500kb).</p></li>
</ul></li>
<li><p>Add 500kb as cis window to analysis.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/12_06_peak_num.pdf" title="Figure: Number of peaks across chr's." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of peaks across chr’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_06_peak_num">
Past versions of 12_06_peak_num.pdf
</button>
</p>
<div id="fig-12_06_peak_num" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/12_06_peak_num.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/12_06_peaks_on_chr_window50kb.pdf" title="Figure: Number of peak groups across chr's." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of peak groups across chr’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_06_peaks_on_chr_window50kb">
Past versions of 12_06_peaks_on_chr_window50kb.pdf
</button>
</p>
<div id="fig-12_06_peaks_on_chr_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/12_06_peaks_on_chr_window50kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/12_06_peaks_in_group_window50kb.pdf" title="Figure: Number of peaks in peak windows." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of peaks in peak windows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_06_peaks_in_group_window50kb">
Past versions of 12_06_peaks_in_group_window50kb.pdf
</button>
</p>
<div id="fig-12_06_peaks_in_group_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/12_06_peaks_in_group_window50kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/12_06_peaks_w_cqtl_in_group_window50kb.pdf" title="Figure: Number of cpeaks in peak windows." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of cpeaks in peak windows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_06_peaks_w_cqtl_in_group_window50kb">
Past versions of 12_06_peaks_w_cqtl_in_group_window50kb.pdf
</button>
</p>
<div id="fig-12_06_peaks_w_cqtl_in_group_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/12_06_peaks_w_cqtl_in_group_window50kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/12_06_peaks_in_group_w_cqtl_window50kb.pdf" title="Figure: Number of cpeaks in peak windows." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of cpeaks in peak windows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_06_peaks_in_group_w_cqtl_window50kb">
Past versions of 12_06_peaks_in_group_w_cqtl_window50kb.pdf
</button>
</p>
<div id="fig-12_06_peaks_in_group_w_cqtl_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/12_06_peaks_in_group_w_cqtl_window50kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="association-results" class="section level3">
<h3>Association results</h3>
<ol style="list-style-type: decimal">
<li>p-value distribution</li>
</ol>
<ul>
<li><p>Top p-values (bonbforroni corrected)</p></li>
<li><p>Top p-values (ACAT corrected)</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Number of significant peak groups</li>
</ol>
</div>
<div id="comparison-results" class="section level3">
<h3>Comparison results</h3>
<ul>
<li><p>Compare to uni-pop (qtltools)</p>
<ul>
<li><p>Signal numbers</p></li>
<li><p>Effect size?</p></li>
<li><p>Overlaps? (new vis) <a
href="atac/figs/whole_blood/plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf">fig</a></p></li>
</ul></li>
<li><p>Compare to uni-allelic (rasqual)</p>
<ul>
<li><p>Signal numbers</p></li>
<li><p>Effect size?</p></li>
<li><p>Overlaps? (new vis) <a
href="atac/figs/whole_blood/plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf">fig</a></p></li>
</ul></li>
</ul>
<p><del>- Compare to sc-PME ???</del></p>
<p>~~ - Signal numbers~~</p>
<p>~~ - Effect size?~~</p>
<p>~~ - Overlaps? (new vis)~~</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis500kb.pdf" title="Figure: Number of cpeaks in peak windows." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of cpeaks in peak windows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis500kb">
Past versions of
01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis500kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Choose new window size and cis region - </strong></p>
<p><a
href="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window50kb_cis50kb.pdf">window50kb_cis50kb</a></p>
<p><a
href="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis500kb.pdf">window25kb_cis500kb</a></p>
<p><a
href="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf">window25kb_cis25kb</a></p>
<p><a
href="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window5kb_cis5kb.pdf">window5kb_cis5kb</a></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Comparison of cwindows and cpeaks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison of cwindows and cpeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_plt_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_dis_leadsnp_cpeak.pdf" title="Figure: Distance from lead cQTL to cpeak/cwindow (and if within any peak for cwindow). Cis region used for (1) multi (500kb, 25kb), (2) qtltools (500kb), (3) rasqual (10kb)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Distance from lead cQTL to cpeak/cwindow (and if within any peak
for cwindow). Cis region used for (1) multi (500kb, 25kb), (2) qtltools
(500kb), (3) rasqual (10kb).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_dis_leadsnp_cpeak">
Past versions of 01_05_dis_leadsnp_cpeak.pdf
</button>
</p>
<div id="fig-01_05_dis_leadsnp_cpeak" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_dis_leadsnp_cpeak.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Observations -</p>
<ul>
<li><p>Most lead cQTLs are close to cpeaks.</p></li>
<li><p>Using 25kb as cis region is reasonable? 500kb too far</p></li>
</ul>
<p><strong>Change the way to visualize numbers - </strong></p>
<p>as the number of peaks and windows differ a lot. Focus on windows,
i.e. only use the right column, to show the number of cwindows not
overlap with any cpeaks, and non-significant windows overlap with any
cpeaks.</p>
<p>Better way - two figures with different angles -</p>
<p>(A1) Significant peaks, filled by overlap with cwindows. (replicated
cpeaks &amp; missed cpeaks)</p>
<p>(A2) Non significant peaks, filled by overlap with at least one
cwindow. (new cpeaks)</p>
<p>(B1) Significant windows, filled by overlap with cpeaks. (new
windows)</p>
<p>(B2) Non significant windows, filled by overlap with at least one
cpeak. (missed cpeaks)</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Comparison of cwindows and cpeaks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison of cwindows and cpeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87402c7054fa6d1127a4c4cb79a57153472fdf89/docs/asset/2023/01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">87402c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_plt_new_cpeak_cwindow_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<p>CACTI v.s. QTLTools -</p>
<ul>
<li><p>CACTI replicated most cpeaks by qtltools (upper left).</p></li>
<li><p>CACTI found many more new signals, much more new cpeaks (upper
right) and ~1k cwindows (lower left).</p></li>
<li><p>CACTI missed a few cpeaks (upper left) and cwindows (lower
right). Most missed cwindows have only 1 cpeak by QTLTools.</p></li>
</ul>
<p>CACTI v.s. RASQUAL -</p>
<ul>
<li><p>CACTI did not replicate most cpeaks by rasqual (upper
left).</p></li>
<li><p>CACTI found many more new new cpeaks (upper right), but only a
few cwindows (lower left).</p></li>
<li><p>CACTI missed many cpeaks (upper left) and cwindows (lower right).
Most missed cwindows have only 1 cpeak by QTLTools.</p></li>
</ul>
<p>Looks like CACTI and RASQUAL find different signals. CACTI finds
cwindows containing multiple cpeaks, whereas cpeaks missed by cwindows
are those containing only one cpeaks.</p>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Is it necessary to use large cis region? Decrease cis regions
from 500kb to 50kb found more signals. Check - Variants within 500kb
v.s. p-value.</p></li>
<li><p>Decrease window size found a bit more signals but not
much.</p></li>
<li><p>Use smaller window size &amp; smaller cis region?</p>
<ul>
<li><p>Using smaller cis region (25kb window, 25kb cis region) results
in more signals. More than qtltools, but still not good as
rasqual.</p></li>
<li><p>5kb window &amp; 5kb cis reigon is too small, resulting in missed
overlap windows with qtltools peaks, and with qtltools cpeaks.</p></li>
</ul></li>
<li><p>Why so few signals? (compared to the other two methods,
especially to rasqual)</p>
<ul>
<li><p>What are those missed signals?</p>
<ul>
<li><p>Those missed signals by multi (but identified by rasqual or
qtltools) tend to have low signal to noise ratio, i.e. in a window only
a few (1 or 2) peaks are cpeaks. rasqual tends to find these “isolated”
cpeaks, whereas cacti finds co-regulated cwindows.</p></li>
<li><p>For writing, we can emphasize that cacti finds co-regulated
cwindows, and examples of those new cwindows.</p></li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="show-cacti-and-rasqual-find-different-signals"
class="section level3">
<h3>Show CACTI and RASQUAL find different signals</h3>
<ol style="list-style-type: decimal">
<li>signal to noise ratio of missed and replicated windows</li>
</ol>
<blockquote>
<p>Looks like CACTI and RASQUAL find different signals. CACTI finds
cwindows containing multiple cpeaks, whereas cpeaks missed by cwindows
are those containing only one cpeaks.</p>
</blockquote>
<ul>
<li><p>missed and replicated windows have difference signal to noise
ratio, therefore, CACTI finds cQTLs of co-regulated peaks.</p>
<ul>
<li><p>num of cpeaks for missed and replicated windows</p></li>
<li><p>size of missed and replicated windows</p></li>
<li><p>signal to noise ratio of missed and replicated windows</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Missed and replicated windows have difference signal to noise ratio." width="40%" type="application/pdf" />
<p class="caption">
Figure: Missed and replicated windows have difference signal to noise
ratio.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>Compare to CRDs or TADs</li>
</ol>
<p><strong>Rational -</strong></p>
<p>[Ref]</p>
<ul>
<li><p>Physical interactions between REs are commonly mapped with
chromosome conformation capture (3C) techniques, such as Hi-C. In
practice, Hi-C measures 3D contacts between distinct DNA regions and
therefore potential functional links between REs.</p></li>
<li><p>By contrast, interindividual correlation captures the coordinated
activity of REs resulting from (direct or indirect) functional
links.</p></li>
<li><p>We therefore looked at how structure among peaks within cwindows
compare to functional links and 3D contacts.</p></li>
</ul>
<p><strong>Basis info about CRDs -</strong></p>
<ul>
<li><p>How?</p></li>
<li><p>“… regrouped 40.9% (n = 111,005) and 16.6% (n = 45,062) of the
chromatin peaks within 12,583 and 10,442 CRDs in LCLs and fibroblasts,
respectively.”</p></li>
</ul>
<p><strong>Do replicated cwindows more likely overlap with any CRDs than
missed cwindows by rasqual/qtltools?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_rep_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Do replicated cwindows more likely overlap with any CRDs than missed cwindows by rasqual/qtltools?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do replicated cwindows more likely overlap with any CRDs than
missed cwindows by rasqual/qtltools?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_rep_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_rep_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_rep_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_rep_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations -</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
<p><strong>Do replicated cwindows more likely fall within same CRDs than
missed cwindows by rasqual/qtltools?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_rep_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Do replicated cwindows more likely fall within same CRDs than missed cwindows by rasqual/qtltools?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do replicated cwindows more likely fall within same CRDs than
missed cwindows by rasqual/qtltools?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_rep_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_rep_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_rep_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_rep_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations -</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Compare to ChromHMM functional annotations</li>
</ol>
<p><strong>Rational -</strong></p>
<p>RASQUAL cpeaks less likely fall within functional annotations (false
positives).</p>
<p><strong>Basis info about ChromHMM functional annotations
-</strong></p>
<ul>
<li><p>ChromHMM: automating chromatin-state discovery and
characterization.</p></li>
<li><p>ChromHMM functional annotations of chromatin states in hg19,
across 127 Reference Epigenomes (111 Roadmap cell types and 16 ENCODE
cell lines), using 12 histone marks and 25 states.</p></li>
</ul>
<p><strong>Do replicated cwindows more likely fall within functional
annotations than missed cwindows by rasqual/qtltools?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_func_annot_new.pdf" title="Figure: Do replicated cwindows more likely fall within functional annotations than missed cwindows by rasqual/qtltools?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do replicated cwindows more likely fall within functional
annotations than missed cwindows by rasqual/qtltools?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_func_annot_new">
Past versions of 01_05_fig_func_annot_new.pdf
</button>
</p>
<div id="fig-01_05_fig_func_annot_new" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_func_annot_new.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="characterization-of-signals-new-signals"
class="section level3">
<h3>Characterization of signals &amp; new signals</h3>
<ol style="list-style-type: decimal">
<li>compare to CRDs or TADs</li>
</ol>
<p>Rational -</p>
<p>CACTI finds cwindows of co-regulated peaks.</p>
<p><strong>(all signals) Do cwindows more likely overlap with any CRDs
than non-cwindows?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_cwindow_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_cwindow_crd_num_overlap_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: if cwindows and CRDs frequently overlap" width="40%" type="application/pdf" />
<p class="caption">
Figure: if cwindows and CRDs frequently overlap
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_cwindow_crd_num_overlap_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_cwindow_crd_num_overlap_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_cwindow_crd_num_overlap_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_cwindow_crd_num_overlap_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_cwindow_crd_dis_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: if cwindows closer to CRDs than non-cwindows" width="40%" type="application/pdf" />
<p class="caption">
Figure: if cwindows closer to CRDs than non-cwindows
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_cwindow_crd_dis_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_cwindow_crd_dis_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_cwindow_crd_dis_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_cwindow_crd_dis_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations -</p>
<ul>
<li><p>cwindows are enriched for CRDs. They have correlations. TADS
functional links.</p></li>
<li><p>cwindows and CRDs frequently overlap.</p></li>
<li><p>But distance from cwindow and non-cwindows to CRDs no significant
difference.</p></li>
</ul></li>
</ul>
<p><strong>(all signals) Are cwindows more likely with same CRDs than
non-cwindows?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Are cwindows more likely with same CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Are cwindows more likely with same CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_05_fig_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_05_fig_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_cwindow_crd_same_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations -</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>compare to chromHMM functional annotations</li>
</ol>
<p><strong>Do cwindows more likely fall within functional annotations
than non-cwindows?</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_05_fig_func_annot_all.pdf" title="Figure: Do cwindows more likely fall within functional annotations than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely fall within functional annotations than
non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_05_fig_func_annot_all">
Past versions of 01_05_fig_func_annot_all.pdf
</button>
</p>
<div id="fig-01_05_fig_func_annot_all" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/01_05_fig_func_annot_all.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="3" style="list-style-type: decimal">
<li><p>eQTLs (uni-pop, uni-allelic, sc-PME)</p></li>
<li><p>GWAS (uni-pop, uni-allelic, sc-PME)</p></li>
</ol>
</div>
</div>
<div id="run-cacti-p-on-cell-types" class="section level2">
<h2>2. Run CACTI-P on cell types</h2>
<div id="steps-of-caution" class="section level3">
<h3>Steps of caution</h3>
<ul>
<li><p>Check rasqual scripts &amp; results for pseudo-bulk data</p></li>
<li><p>Run QTLTools on cell types</p></li>
<li><p>Data QC on several scales (see below)</p></li>
<li><p>Data normalization on several scales (see below)</p></li>
<li><p>Window size to choose</p></li>
</ul>
</div>
<div id="basic-info-1" class="section level3">
<h3>Basic info</h3>
<p>(Make a table for main cell types)</p>
</div>
</div>
<div id="phoenixs-results-of-pbmc-scatac-from-56-individuals"
class="section level2">
<h2>1. Phoenix’s results of PBMC scATAC from 56 individuals</h2>
<p>Base directory: <code>/project/yangili1/zpmu/xijing/ra/</code>.</p>
<p>For now, use cells/results from all threee datasets. (But downsample
to ~20 samples?)</p>
<div id="cell-level-data" class="section level3">
<h3>Cell level data</h3>
<div id="cell-by-peak-matrix" class="section level4">
<h4>Cell by peak matrix</h4>
<p>Single-cell level count matrix as <code>SummarizedExperiment</code>
objects.</p>
<p><code>data/scqtl/peakMtrx_chr{1..22}_3study.rds</code></p>
<ul>
<li><p>Cell-level metadata - individual information</p></li>
<li><p>Cell-level metadata - cell type information</p></li>
</ul>
</div>
<div id="covariates" class="section level4">
<h4>Covariates</h4>
<ul>
<li><p>Single-cell level phenotypic PC (LSI embedding)</p>
<p><code>data/scqtl/IterativeLSI50_3study.rds</code></p></li>
<li><p>Genotype PC calculated by smartPCA together with 1000G
samples</p>
<p><code>data/glimpse/1kg_merged3_chr1-22.merged.1kgfmt.pruned.eigenvec</code></p></li>
</ul>
</div>
</div>
<div id="pseudo-bulk-level-data" class="section level3">
<h3>Pseudo-bulk level data</h3>
<div id="individual-data" class="section level4">
<h4>individual data</h4>
<p>For <code>all cell1</code> (whole blood),
<code>data/rasqual/harmonised_smp.txt</code>.</p>
<p>For <code>L1</code> and <code>L2</code> cell types, some individuals
with low cell number were removed, therefore the list of individuals are
in these files,
<code>data/rasqual/smp_to_use_L{1..2}_{cell}.txt</code>.</p>
</div>
<div id="genotype" class="section level4">
<h4>Genotype</h4>
<p><code>data/rasqual/as_vcf_3study/chr{1..22}.filtered.dosage.txt.gz</code></p>
<p><code>data/rasqual/as_vcf_3study/chr{1..22}.filtered.vcf.gz</code>
(vcf format)</p>
</div>
<div id="pseudo-bulk-count-data" class="section level4">
<h4>Pseudo-bulk count data</h4>
<p>Whole-blood:
<code>data/rasqual/sum_3study_allCell_hornet.txt.gz</code></p>
<p>L1 cell types:
<code>data/rasqual/sum_3study_L1_hornet.txt.gz</code></p>
<p>L2 cell types:
<code>data/rasqual/sum_3study_L2_hornet.txt.gz</code></p>
</div>
<div id="covariates-for-count-data" class="section level4">
<h4>Covariates for count data</h4>
<p>Whole-blood:
<code>data/rasqual/input_table_3study/PC-{nPC}_3study_allCell_hornet.txt</code></p>
<p>L1 cell types:
<code>data/rasqual/input_table_3study/PC-{nPC}_3study_L1_hornet_{cell}.txt</code></p>
<p>L2 cell types:
<code>data/rasqual/input_table_3study/PC-{nPC}_3study_L2_hornet_{cell}.txt</code></p>
</div>
<div id="summary-stats" class="section level4">
<h4>Summary stats</h4>
<p>Summary stats for all SNPs from Metasoft in whole blood and seven
cell-types:
<code>results/metasoft/poi_scale_*_noGC_formatted_v1.txt.gz</code></p>
<p>Lead caQTL from Metasoft in whole blood and seven cell-types:
<code>results/metasoft/poi_scale_*_noGC_formatted_v1_lead.txt.gz</code></p>
</div>
</div>
</div>
<div id="data-qc" class="section level2">
<h2>2. Data QC</h2>
<ol style="list-style-type: decimal">
<li><p>Cells</p></li>
<li><p>Individuals</p></li>
<li><p>Peaks</p></li>
<li><p>Variants</p></li>
</ol>
<p><strong>All cells –&gt; Cell types –&gt; (1) Cell numbers, (2) Read
coverage –&gt; (1) Individuals (cell numbers), (2) Peaks (read coverage)
</strong></p>
</div>
<div id="data-normalization" class="section level2">
<h2>3. Data normalization</h2>
<ol style="list-style-type: decimal">
<li><p>Cells</p></li>
<li><p>Individuals</p></li>
<li><p>Peaks</p></li>
<li><p>Variants</p></li>
</ol>
</div>
<div id="cacti-p-in-whole-blood" class="section level2">
<h2>4. CACTI-P in whole blood</h2>
<ol style="list-style-type: decimal">
<li>Univariate z-scores of snps v.s. peaks (fastQTL)</li>
</ol>
<ul>
<li><p>Inputs of Metasoft (PME)? (ind by peak mat, covariates, geno)</p>
<ul>
<li><p>Pipeline
<code>/project2/yangili1/zpmu/xijing/ra/scripts/scqtl/scqtl.smk</code></p></li>
<li><p>Check what QC and normalization were done?</p>
<ul>
<li>Script?
<code>/project2/yangili1/zpmu/xijing/ra/scripts/scatac/16_merged3_analysis_clean.R</code></li>
</ul></li>
</ul></li>
<li><p>Ben’s fastQTL pipeline (double check outputs are same as Ben’s
previous results)</p></li>
</ul>
<p>ArchR project folder, containing all other types of data and many
files I used in my own analysis.</p>
<p>data/scatac/archR_3study_v1New_clean</p>
</div>
</div>
<div id="nov-15" class="section level1">
<h1>Nov 15</h1>
<div id="paper-revision-1" class="section level2">
<h2>1. Paper revision 1</h2>
<p>Table of signal numbers in googlesheets. A fews things to note -</p>
<ul>
<li><p>I have the numbers for LD independent signals in eQTLGen, but I
still don’t think it makes sense to report these numbers -</p>
<ul>
<li>As we already explained in main text (line 505)</li>
</ul>
<blockquote>
<p>“As in eQTLGen, we did not perform LD clumping on trans-eSNPs,
because they were GWAS SNPs associated with different traits and
diseases.”</p>
</blockquote>
<ul>
<li>To do LD clumping, we need to use DGN genotypes. However, not all
eQTLGen variants are in DGN, so many top trans variants are missed, and
#independent loci is underestimated and inaccurate.</li>
</ul></li>
<li><p>To answer question b, I think our main text description is clear,
but maybe adding the numbers as a supp table?</p></li>
<li><p>To answer the first part of question c,</p>
<p>The number of LD-independent signals and trans-eQTL module pairs
(179) are defined in different ways and for different purposes.</p>
<ul>
<li><p>The LD-independent signals are defined by using LD clumping while
accounting for LD (R2&lt;0.2). This is for reporting the number of
independent signals.</p></li>
<li><p>The 179 trans-eQTL loci are regions for performing colocalizaiton
analysis. They are defined as fixed-width (200kb) flanking genomic
regions centered at a lead signal. Please see Lines 812-822 for more
detailed descriptions.</p></li>
<li><p>These two types of regions are similar, as we expect those 200kb
flanking genomic regions have low LD. In fact, there are 145 (LD
independent trans-loci, module) pairs, which is similar to 179 pairs we
used for coloc.</p></li>
</ul></li>
<li><p>To answer the second part of question c, it’s the last row in the
table.</p></li>
</ul>
</div>
<div id="paper-revision-2" class="section level2">
<h2>2. Paper revision 2</h2>
<ul>
<li><p>Traits v.s. transQTL of co-expression gene module in DGN</p>
<p>I looked at those six immune traits, in terms of finding the coloc
region between a trait and a co-expression gene module in DGN. The coloc
region is defined as a 200kb region centered at a transQTL (before
merging with other neighboring coloc regions). I then take the transQTL
(lead SNP), and find its nearest/near genes. See files
attached.</p></li>
<li><p>Traits v.s. transQTL of co-expression gene module in eQTLGen</p>
<p>I looked at if trait signal snps are also transQTLs of co-expression
gene modules in eQTLGen. Trait signal snps are defined as those with
p&lt;5e-8. I then check if they are also transQTLs, and find its
nearest/near genes. See files attached. Each row of a file is a trait
signal snp, and its corresponding trans target module. One trait signal
snp can appear in multiple rows if they have multiple trans target
module in eQTLGen.</p></li>
</ul>
</div>
<div id="paper-revision-3" class="section level2">
<h2>3. Paper revision 3</h2>
<div id="v1" class="section level3">
<h3>V1</h3>
<p><strong>To look at the enrichment of target genes in the transeQTL
nearest genes</strong>, I did steps -</p>
<ul>
<li><p>I looked at 5 immune diseases.</p></li>
<li><p>I found significant trait-associated snps in eQTLGen for each
trait.</p></li>
<li><p>I looked at the nearest genes and check if they are trans
signals.</p></li>
<li><p>I also checked if they are target genes of the trait.</p></li>
</ul>
<p><strong>Figure -</strong></p>
<ul>
<li><p>x: if a snp is trans signal</p></li>
<li><p>y: number of snps</p></li>
<li><p>each panel: a trait</p></li>
<li><p>color: if the snp’s nearest gene is target gene</p></li>
<li><p>lower panel: I used only target genes of the corresponding
trait</p></li>
<li><p>upper [anel: I used all target genes of any trait</p></li>
</ul>
<p><strong>Observations -</strong></p>
<ul>
<li><p>There is no enrichment of target genes in nearest genes of trans
snps</p></li>
<li><p>If I use only target gene of the corresponding trait (lower
panel), there is basically no overlap between nearest genes of
trait-associated snps (no matter it’s trans or not) and target genes,
i.e. few red color.</p></li>
<li><p>If I use all target genes of any trait, there are some overlap
(more red), but the enrichment is not significant.</p></li>
</ul>
<p><strong>Some reasons/changes may improve enrichment -</strong></p>
<ul>
<li><p>Use other lists of target genes</p></li>
<li><p>We were looking at eQTLGen trans snps that are disease associated
snp. These snps are already trait-associated. If we directly look at the
enrichemnt, I think we are more looking at the “trait specific”
enrichment, as basically all those snps are expected to be enriched for
trans snps. it’s like the logic in Dutta et al. Should we use general
snps in DGN? or just use the absolute number of overlap genes?</p></li>
<li><p>Should we use trans snp across all traits instead of consider
each trait separately?</p></li>
<li><p>I think you mentioned to look at if snps are trans snps of only
those immune related module, but now I am using all modules to define
trans snps. If I use the former, the overlap will be less (red
color)</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/target_enrich.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<p class="caption">
Figure: To look at the enrichment of target genes in the transeQTL
nearest genes.
</p>
</div>
</div>
<div id="v2" class="section level3">
<h3>V2</h3>
<p><strong>Updated results for using transeqtl of only immune related
co-expression gene modules.</strong> Steps -</p>
<ul>
<li><p>I first go through all coexpression gene modules, and pick those
are immune/blood related. There are 26 such modules.</p></li>
<li><p>I then define a trait-associated snp to be transeqtl if it’s
associated at least one of those 26 modules.</p></li>
<li><p>Similarly as previous, I overlap trans eqtls’ nearest genes with
target genes</p></li>
<li><p>Updated results are in the third row.</p></li>
</ul>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Less transeqtls under this definition</p></li>
<li><p>Seems no significant enrichment in target genes</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/target_enrich.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<p class="caption">
Figure: To look at the enrichment of target genes in the transeQTL
nearest genes.
</p>
</div>
</div>
<div id="v3" class="section level3">
<h3>V3</h3>
<p><strong>Changes -</strong></p>
<ul>
<li><p>Change visualization to the opposite, i.e. to see if drug targets
are more likely to be trans signals of immune related modules.</p></li>
<li><p>Use genes instead of SNPs.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/target_enrich_target_vs_trans_snp.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<embed src="asset/2023/target_enrich_target_vs_trans_gene.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<p class="caption">
Figure: To look at the enrichment of target genes in the transeQTL
nearest genes.
</p>
</div>
<p><strong>Observations -</strong></p>
<ol style="list-style-type: decimal">
<li>Obs 1 for disease allergy</li>
</ol>
<p>It might be interesting to check out gene plot, Panel A &amp;C, of
allergy. Out of 3 trait-associated genes that are target, all of them
are near to a trans snp (A), and two of them are near to a trans snp
corresponding to a immune related gene module</p>
<p>The genes are -</p>
<ul>
<li><p>ERBB3 (chr12), corresponding to M87 and M97, all of which are
immune/blood related.</p></li>
<li><p>IL2 (chr4), corresponding to no modules that are immune/blood
related.</p></li>
<li><p>UGT3A1 (chr5), corresponding to M62, M76, M87, all of which are
immune/blood related.</p></li>
</ul>
<p>Their corresponding trans gene modules are -</p>
<pre><code>gene_module                                                                      GO:BP_term_name
1:          62 cytokine-mediated signaling pathway;tumor necrosis factor-mediated signaling pathway
2:          76                                system development;multicellular organism development
3:          87             positive regulation of T cell activation;regulation of T cell activation
4:          97                                                  cytoplasmic translation;translation
                                                   GO:MF_term_name
1:                                      cytokine receptor activity
2:                           molecular function regulator activity
3:                                         T cell receptor binding
4: structural constituent of ribosome;structural molecule activity
                                                                                         KEGG_term_name
1: Cytokine-cytokine receptor interaction;Viral protein interaction with cytokine and cytokine receptor
2:                                             Prolactin signaling pathway;NF-kappa B signaling pathway
3:                                           Th17 cell differentiation;Th1 and Th2 cell differentiation
4:                                                              Ribosome;Coronavirus disease - COVID-19
                                                                                                                    REAC_term_name
1:                                                          TNFR1-induced NFkappaB signaling pathway;Regulation of TNFR1 signaling
2: Nucleotide-binding domain, leucine rich repeat containing receptor (NLR) signaling pathways;Cytokine Signaling in Immune system
3:                                     Generation of second messenger molecules;Binding of TCF/LEF:CTNNB1 to target gene promoters
4:                                                                                 Viral mRNA Translation;Peptide chain elongation</code></pre>
<ol start="2" style="list-style-type: decimal">
<li>Obs 2 across all traits</li>
</ol>
<p>The above is for allergy trait. Across all traits -</p>
<ul>
<li>Genes that are target and near a trans eqtl -</li>
</ul>
<pre><code>        nearest_gene
 1:       DNMT3A
 2:        ERBB3
 3:       GPBAR1
 4:          IL2
 5:          IL3
 6:         NCF4
 7:         PPIF
 8:        PSMB8
 9:       PTGER4
10:        SH2B3
11:          TNF
12:       UGT3A1</code></pre>
<ul>
<li>Genes that are target and near a trans eqtl of an immune-related
module -</li>
</ul>
<pre><code>   nearest_gene
1:       DNMT3A
2:        ERBB3
3:         NCF4
4:         PPIF
5:        SH2B3
6:          TNF
7:       UGT3A1</code></pre>
</div>
<div id="hall-mark-gene-sets" class="section level3">
<h3>Hall mark gene sets</h3>
<p>Using hallmark gene sets as modules, 11 have signals, 5 are immune
related gene sets.</p>
<ul>
<li>I define a gene set as immune related if it has direct relation with
immune response or blood, e.g. genes are immune related genes, or
pathway description includes blood cells.</li>
</ul>
<p><strong>Figures -</strong></p>
<ul>
<li><p>1st fig: using snps</p></li>
<li><p>2nd fig: using genes</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/target_enrich_target_vs_trans_snp_hallmark.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<embed src="asset/2023/target_enrich_target_vs_trans_gene_hallmark.pdf" title="Figure: To look at the enrichment of target genes in the transeQTL nearest genes. " width="40%" type="application/pdf" />
<p class="caption">
Figure: To look at the enrichment of target genes in the transeQTL
nearest genes.
</p>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Less snps/genes are drug targets as well as being trans signal of
an immune related module.</p></li>
<li><p>Genes that are target and near a trans eqtl -</p></li>
</ul>
<pre><code>   nearest_gene
1:        ERBB3
2:          IL3
3:         NCF4
4:        SH2B3</code></pre>
<ul>
<li>Genes that are target and near a trans eqtl of an immune-related
module -</li>
</ul>
<pre><code>   nearest_gene
1:          IL3
2:         NCF4
3:        SH2B3</code></pre>
<ul>
<li><p>My definition of a gene set being immune related might be
inaccurate.</p>
<ul>
<li><p>In coexpression module, gene ERBB3 belongs to the second
category, its near trans eqtl of an immune related module. So I checked
the gene’s associated hallmark gene sets. There are two -
HALLMARK_CHOLESTEROL_HOMEOSTASIS and HALLMARK_PANCREAS_BETA_CELLS. I
digged deeper and found that HALLMARK_PANCREAS_BETA_CELLS can be immune
related. so does HALLMARK_CHOLESTEROL_HOMEOSTASIS. so, ERBB3 should also
be in the second categoty.</p></li>
<li><p>Def file
<code>/project2/xuanyao/llw/MODULES/MSigDB/msig_50_table_immune_related.txt</code></p></li>
<li><p>Modify the file …</p>
<ul>
<li><p>Out of 50 hallmark gene sets, 9 are immune related gene
sets.</p></li>
<li><p>Using updated immune gene set def, enrichment OR and p don’t
change much.</p></li>
</ul></li>
</ul></li>
</ul>
</div>
<div id="enrichment-or-p" class="section level3">
<h3>Enrichment OR &amp; p</h3>
<p>Results see slack.</p>
<p>Use genes and for the immune related modules only.</p>
<ul>
<li><p>Co-expression modules</p></li>
<li><p>Hallmark</p></li>
<li><p>Co-expression &amp; Hallmark</p></li>
</ul>
</div>
</div>
<div id="redo-ld-clumping" class="section level2">
<h2>4. Redo LD clumping</h2>
<p>Issue - there are 145 loci-module pairs and 145 loci too. Does that
mean there is no loci that regulate multiple modules?</p>
<p>Previously I had a bug in code of ld clumping, where I assign signal
files of all modules to plink clumping in one run. I thought plink would
do clumping for each file separately, but it actually take all signals
across all files. That’s why the number was the same when I consider
only the snps.</p>
<p>So I loop over each module instead and run plink clumping for each
module separately. The current numbers should be correct.</p>
<p>(LD independent trans-loci, module) is more than LD independent
trans-loci, which means some loci regulate multiple modules.</p>
</div>
<div id="a-few-numbers-of-allergy-example-for-drug-target"
class="section level2">
<h2>5. A few numbers of allergy example for drug target</h2>
<p><strong>Q 1) How many launched drug targets are there for allergy in
the broad database</strong></p>
<pre><code>There are 2184 targets in total.
1338 are launched.
1153 are with a disease,
55 are for allergy trait.</code></pre>
<p><strong>Q 2) How many are near a GWAS loci included in eQTLGen (eg,
within 1Mb)</strong></p>
<pre><code>47 are near a eQTGen snp (within 1Mb).</code></pre>
<p><strong>Q 3) How many are trans-eQTLs of immune related
pathways?</strong></p>
<ul>
<li><p>#1. trans-eQTLs of coexp-module</p>
<ul>
<li><p>23 targets are near (&lt;1Mb) a trans-qtl associated with at
least one coexp-module.</p></li>
<li><p>12 targets are near (&lt;1Mb) a trans-qtl associated with at
least one immune related coexp-module.</p></li>
</ul></li>
<li><p>#2. trans-eQTLs of hallmark-module</p>
<ul>
<li><p>10 targets are near (&lt;1Mb) a trans-qtl associated with at
least one hallmark-module.</p></li>
<li><p>9 targets are near (&lt;1Mb) a trans-qtl associated with at least
one immune related hallmark-module.</p></li>
</ul></li>
</ul>
<p><strong>Q 4) If the number is not zero for 2 and 3, can you also
provide the names of the genes and the gene modules</strong></p>
<ul>
<li><p>Coexp-module</p>
<p><code>/project2/xuanyao/llw/translation/target_eqtlgen_trans_module_coexp.txt</code></p></li>
<li><p>Hallmark-module</p>
<p><code>/project2/xuanyao/llw/translation/target_eqtlgen_trans_module_hallmark.txt</code></p></li>
<li><p>Columns are -</p>
<ul>
<li><p>target</p></li>
<li><p>n_trans_module: number of trans target modules regulated by any
trans-eqtl (1Mb) near the target</p></li>
<li><p>trans_module: trans target modules regulated by any trans-eqtl
(1Mb) near the target</p></li>
<li><p>n_trans_module_immune: number of trans target modules (immune
related) regulated by any trans-eqtl (1Mb) near the target</p></li>
<li><p>trans_module_immune: trans target modules (immune related)
regulated by any trans-eqtl (1Mb) near the target</p></li>
</ul></li>
</ul>
<p><strong>note -</strong></p>
<ul>
<li><p>this is for allergy trait only</p></li>
<li><p>By a snp near a gene, the def is if a snp is within 500kb of a
gene’s upstream or downstream (1Mb in total)</p></li>
</ul>
<div id="updated-results-using-allergy-gwas-significant-snps"
class="section level3">
<h3>Updated results using allergy gwas significant snps</h3>
<p><strong>Q 2) How many are near a GWAS loci included in eQTLGen (eg,
within 1Mb)</strong></p>
<pre><code>5 are near a eQTGen snp (within 1Mb).</code></pre>
<p><strong>Q 3) How many are trans-eQTLs of immune related
pathways?</strong></p>
<ul>
<li><p>#1. trans-eQTLs of coexp-module</p>
<ul>
<li><p>2 targets are near (&lt;1Mb) a trans-qtl associated with at least
one coexp-module.</p></li>
<li><p>2 targets are near (&lt;1Mb) a trans-qtl associated with at least
one immune related coexp-module.</p></li>
<li><p>Target genes are: UGT3A1 and IL3</p></li>
</ul></li>
<li><p>#2. trans-eQTLs of hallmark-module</p>
<ul>
<li><p>1 targets are near (&lt;1Mb) a trans-qtl associated with at least
one hallmark-module.</p></li>
<li><p>1 targets are near (&lt;1Mb) a trans-qtl associated with at least
one immune related hallmark-module.</p></li>
<li><p>Target genes are: IL3</p></li>
</ul></li>
</ul>
<p><strong>Q 4) If the number is not zero for 2 and 3, can you also
provide the names of the genes and the gene modules</strong></p>
<ul>
<li><p>Coexp-module</p>
<p><code>/project2/xuanyao/llw/translation/target_gwas_eqtlgen_trans_module_coexp.txt</code></p></li>
<li><p>Hallmark-module</p>
<p><code>/project2/xuanyao/llw/translation/target_gwas_eqtlgen_trans_module_hallmark.txt</code></p></li>
</ul>
<p><strong>A few things to note -</strong></p>
<ul>
<li><p>This is actually similar to the analysis I did before, as in
previous message, where I used the “nearest gene”, here I am using cis
genes &lt;1Mb</p></li>
<li><p>Previous version using nearest gene, there were three target
genes: ERBB3, IL2, UGT3A1 (compared to two genes: UGT3A1 and IL3, using
cis genes)</p>
<ul>
<li><p>UGT3A1 is also target gene using cis genes</p></li>
<li><p>IL2 is not, as it’s not corresponding to immune related modules,
whereas here we use only immune modules</p></li>
<li><p>ERBB3 is not target gene in current results, as it’s not a target
of allergy trait, so it’s filtered out (see table below for this target
info)</p></li>
</ul></li>
<li><p>Overall, using “cis genes” increases one gene “IL3” as a target
near a allergy associated snp in eQTLGen that regulates a tran target
coexpression-gene-module and hallmark-gene-set</p>
<ul>
<li><p>Regulated coexp-module (Module108)</p></li>
<li><p>Regulated hallmark gene set</p></li>
</ul></li>
</ul>
<pre><code>gene_module
108
                                                                                                       GO:BP_term_name
antigen processing and presentation of exogenous antigen;peptide antigen assembly with MHC class II protein complex
                                                    GO:MF_term_name
MHC class II protein complex binding;MHC protein complex binding
                               KEGG_term_name
Antigen processing and presentation;Asthma
                                                                       REAC_term_name
MHC class II antigen presentation;Translocation of ZAP-70 to Immunological synapse</code></pre>
<pre><code>HALLMARK_IL6_JAK_STAT3_SIGNALING
HALLMARK_NOTCH_SIGNALING</code></pre>
</div>
</div>
<div id="new-way-to-do-enrichment" class="section level2">
<h2>6. New way to do enrichment</h2>
<p>See results on slack thread -</p>
<p><a
href="https://liulabuchicago.slack.com/archives/DSHGE9T63/p1701392735355239"
class="uri">https://liulabuchicago.slack.com/archives/DSHGE9T63/p1701392735355239</a></p>
<div id="gene-based-enrichment" class="section level3">
<h3>Gene-based enrichment</h3>
<ol style="list-style-type: decimal">
<li><p>Basing on eQTLGen snps.</p></li>
<li><p>Basing on trans snps in eQTLGen.</p></li>
</ol>
<ul>
<li><p>Table 1</p></li>
<li><p>Table 2</p></li>
<li><p>Table 3</p></li>
<li><p>How I did?</p></li>
<li><p>Note</p></li>
</ul>
</div>
<div id="snp-based-enrichment" class="section level3">
<h3>SNP-based enrichment</h3>
<ol style="list-style-type: decimal">
<li><p>Basing on eQTLGen snps.</p></li>
<li><p>Basing on trans snps in eQTLGen.</p></li>
</ol>
</div>
<div id="immune-modules" class="section level3">
<h3>Immune modules</h3>
<p>How many co-expression gene modules and hall mark gene sets are
immune related? and how many gene modules and hallmark gene sets in
total?</p>
<ul>
<li><p>co-expression: 166 in total, 26 immune related. 129 used for
eQTLGen, 17 immune related.</p></li>
<li><p>hallmark: 50 in total, 9 immune related. 11 used for eQTLGen, 2
immune related.</p></li>
</ul>
</div>
</div>
<div id="peak-calling-in-archr" class="section level2">
<h2>3. Peak calling in ArchR</h2>
<p>Peak calling in scatac data was done using peak calling method in
ArchR (a software for integrative scatac data analysis).</p>
<p>Per pseudo-bulk replicate under each cell cluster. Summary of main
steps -</p>
<ol style="list-style-type: decimal">
<li><p>MACS2, peak calling, summit. The majority of ATAC-seq peaks are
less than 500bp.</p></li>
<li><p>Expand a 500bp fix-width peak centered at the summit.</p></li>
<li><p>Merged non-overlapping fixed-width peak set across replicates by
iterative overlap peak merging.</p></li>
</ol>
<p>Detailed explaination of these steps can be found in the <a
href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-021-00790-6/MediaObjects/41588_2021_790_MOESM1_ESM.pdf">supp
note of ArchR paper (“ArchR Methods – Peak Calling”)</a>. Also see
Supplementary Fig. 9b for other details on peak merging.</p>
</div>
<div id="apply-two-applications-of-cacti-to-single-cell-data"
class="section level2">
<h2>4. Apply two applications of CACTI to single cell data</h2>
<ul>
<li><p>Different goals</p>
<ul>
<li><p>Improve power by aggregating 500bp peaks based on ArchR results
(Mainly to compare QTL mapping methods, e.g. QTLTool, allele specific
mapping.)</p></li>
<li><p>Improve power by skip peak calling of MACS2 for narrow elements
(Genome-wide 500bp segments. Mainly to compare peak calling. Is it
necessary given that MACS2 calls accurate peaks for narrow
elements?)</p></li>
</ul></li>
</ul>
<p><strong>What is the goal of applying CACTI to single cell
data?</strong> To show CACTI can find more cQTLs even given small sample
size? To show CACTI is more powerful cQTL calling tool in single cell
data? Compare to single cell methods instead of bulk methods? Which way
we should apply CACTI?</p>
<ul>
<li><p>Implementation</p>
<ul>
<li><p>To tease out Xijing data, we need to redo single cell processing
from the beginning.</p>
<ul>
<li><p>We can’t simply remove cells of Xijing data from the step before
pseudo-bulk.</p></li>
<li><p>As removal of cells affect many pre-processing steps, e.g. batch
effect removal, cell clustering, ArchR project setup, etc.</p></li>
<li><p>It also affects manuscript writing, e.g. how we cluster cells
using three datasets and remove cells from clusters.</p></li>
</ul></li>
<li><p>Ask Phoenix to do this processing step. Two weeks.</p></li>
<li><p>Meanwhile, I work on “teasing cells from given cell clusters” as
trail.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="oct-04" class="section level1">
<h1>Oct 04</h1>
<div id="multicaq-s-coloc-with-gwas-loci" class="section level2">
<h2>1. MultiCaQ-S coloc with GWAS loci</h2>
<div id="multicaq-s-on-mark-h3k36me3" class="section level3">
<h3>MultiCaQ-S on mark H3K36me3</h3>
<ul>
<li><p>Window 50kb, Segment size 2kb, Segment filtering
<code>peak_overlap</code>, Start from GWAS loci.</p></li>
<li><p><a href="#multicaqp-H3K27ac-gwas-coloc">Coloc procedure with GWAS
as in previous section of MultiCaQ-P on H3K27ac</a></p></li>
<li><p>Numbers</p>
<ul>
<li><p>Summary of gwas loci and coloc loci.</p>
<p>Each trait has ~200 loci, ~3.5 (1.8%) are colocalized with
cQTLs.</p></li>
<li><p>How many coloc gwas loci are extra new cQTLs (than
univariate)?</p>
<p>In total, 52 coloc gwas loci are extra new cqtl. On average, traits
have 2 coloc loci coloc with new cQTLs.</p></li>
<li><p>How many coloc gwas loci are not eQTLs?</p>
<p>In total, 46 coloc gwas loci are not eqtl. On average, traits have
1.7 coloc loci not overlap with any eQTL.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/10_04_coloc_num.pdf" title="Figure: Summary of gwas loci and coloc loci. 36 blood traits. (A) x-axis: trait. Grey bar: GWAS loci. Red bar: coloc loci. (B) Number of coloc loci across traits. (C) Coloc proportion across traits." width="40%" type="application/pdf" />
<p class="caption">
Figure: Summary of gwas loci and coloc loci. 36 blood traits. (A)
x-axis: trait. Grey bar: GWAS loci. Red bar: coloc loci. (B) Number of
coloc loci across traits. (C) Coloc proportion across traits.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-10_04_coloc_num">
Past versions of 10_04_coloc_num.pdf
</button>
</p>
<div id="fig-10_04_coloc_num" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/10_04_coloc_num.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/10_04_coloc_compare.pdf" title="Figure: Coloc loci compare with (A) if coloc loci coloc with new cQTLs compared to univariate. (B) if coloc loci overlap with any eQTL." width="40%" type="application/pdf" />
<p class="caption">
Figure: Coloc loci compare with (A) if coloc loci coloc with new cQTLs
compared to univariate. (B) if coloc loci overlap with any eQTL.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-10_04_coloc_compare">
Past versions of 10_04_coloc_compare.pdf
</button>
</p>
<div id="fig-10_04_coloc_compare" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/186602a01af3bcb5f185e0ab72499da3b049a802/docs/asset/2023/10_04_coloc_compare.pdf" target="_blank">186602a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Add loci of H3K27ac?</li>
</ul>
</div>
<div id="multicaq-s-on-mark-h3k27ac" class="section level3">
<h3>MultiCaQ-S on mark H3K27ac</h3>
<ul>
<li>Compare with results of same mark by MultiCaQ-P</li>
</ul>
</div>
</div>
</div>
<div id="sep-06-sep-13" class="section level1">
<h1>Sep 06 &amp; Sep 13</h1>
<p>As shown in last week, to compre to Dutta et. al., I proposed three
ways -</p>
<ol style="list-style-type: decimal">
<li><p>Direct comparison - compare eQTLGen results</p></li>
<li><p>Under our case - reduce archie multiple variants to one single
variant</p>
<p>Use our simulated data.</p></li>
<li><p>Under their case - <del>aggregate trans-pco single variants as a
set</del></p>
<p>Instead, run trans-pco on archie selected gene sets and all eQTLGen
single variants.</p></li>
</ol>
<p>I did way 1. This week I will do 2 and 3.</p>
<div id="trans-pco-on-archie-selected-gene-sets-way-3"
class="section level2">
<h2>1. Trans-PCO on archie selected gene sets (Way 3)</h2>
<div id="how" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Estimate <span
class="math inline">\(\hat{\Sigma}_{nullZ}\)</span> for each gene
set</p>
<p>A SNP has p &gt; 1e-4 for all genes in the set, as a null SNP. Same
threshold as used in paper. Use independent null SNPs to estimate <span
class="math inline">\(\hat{\Sigma}_{nullZ}\)</span>.</p></li>
<li><p>Define trans variants</p>
<p>Run for only trans variants. Same def of trans as used in archie
paper. Either more 5Mb away from all genes or on a different
chromosome.</p></li>
</ul>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<ul>
<li><p>#independent null SNPs for estimating <span
class="math inline">\(\Sigma_{nullZ}\)</span> v.s. gene set
size</p></li>
<li><p>p value distribution</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_06_plt_gene_indepnull_size.pdf" title="Figure: #independent null SNPs for estimating Sigma_{nullZ} v.s. gene set size for archie signal sets. X-axis is archie significant gene sets. Y-axis is either gene set size (light blue) or number of independent null SNPs used for estimating Sigma for that set (dark blue)." width="40%" type="application/pdf" />
<p class="caption">
Figure: #independent null SNPs for estimating Sigma_{nullZ} v.s. gene
set size for archie signal sets. X-axis is archie significant gene sets.
Y-axis is either gene set size (light blue) or number of independent
null SNPs used for estimating Sigma for that set (dark blue).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_06_plt_gene_indepnull_size">
Past versions of 09_06_plt_gene_indepnull_size.pdf
</button>
</p>
<div id="fig-09_06_plt_gene_indepnull_size" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_06_plt_gene_indepnull_size.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_06_plt_archie_pco.pdf" title="Figure: P value of archie gene sets by trans-pco. Each panel is a trait and a significant archie component (selected genes, selected snps). X-axis is snps, y-axis is -log10(P). Each point is a snp. (A) Archie gene set v.s. all eQTLGen SNPs (trans). Red cross are variants selected by archie corresponding to the gene set. (B) Draw archie variants only. Dash line is p cutoff 0.05." width="40%" type="application/pdf" />
<p class="caption">
Figure: P value of archie gene sets by trans-pco. Each panel is a trait
and a significant archie component (selected genes, selected snps).
X-axis is snps, y-axis is -log10(P). Each point is a snp. (A) Archie
gene set v.s. all eQTLGen SNPs (trans). Red cross are variants selected
by archie corresponding to the gene set. (B) Draw archie variants only.
Dash line is p cutoff 0.05.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_06_plt_archie_pco">
Past versions of 09_06_plt_archie_pco.pdf
</button>
</p>
<div id="fig-09_06_plt_archie_pco" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_06_plt_archie_pco.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations" class="section level3">
<h3>Observations</h3>
<ul>
<li><p>Some archie gene sets also have small p values for their
corresponding selected variants by trans-pco (fig. B), e.g. gene sets of
Schizophrenia_C1, Prosc_C1, UC_C1. Mostly first component.</p></li>
<li><p>But some archie gene sets have large p values for their
corresponding selected variants by trans-pco (fig. B), e.g. gene sets of
Pros_C2, UC_C2. Mostly second component.</p></li>
<li><p>Some archie gene sets have small p values for additional variants
by trans-pco (fig. A).</p></li>
</ul>
</div>
<div id="updated-results" class="section level3">
<h3>Updated results</h3>
<p>For each trait, I adjusted pco p using bonferroni correction by
multiplying nominal p by total number of test for each trait (~9918 *
#components). FDR level 0.05. Figure below shows adjusted pco p
values.</p>
<p>We observe that -</p>
<ul>
<li><p>For most archie signal components (except UC_C2), there is at
least one archie-selected snp that are also significant by
trans-pco.</p></li>
<li><p>Similar observation as above, less variants from the second
component (Prosc_C2, UC_C2) are replicated.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_13_plt_archie_pco.pdf" title="Figure: Adjusted P value of archie gene sets by trans-pco. Each panel is a trait and a significant archie component (selected genes, selected snps). X-axis is snps, y-axis is -log10(P). Each point is a snp. (A) Archie gene set v.s. all eQTLGen SNPs (trans). Red cross are variants selected by archie corresponding to the gene set. (B) Draw archie variants only. Dash line is p cutoff 0.05. (C) Number of archie selected variants of each component that are (dark) or not (light) trans-pco signals." width="40%" type="application/pdf" />
<p class="caption">
Figure: Adjusted P value of archie gene sets by trans-pco. Each panel is
a trait and a significant archie component (selected genes, selected
snps). X-axis is snps, y-axis is -log10(P). Each point is a snp. (A)
Archie gene set v.s. all eQTLGen SNPs (trans). Red cross are variants
selected by archie corresponding to the gene set. (B) Draw archie
variants only. Dash line is p cutoff 0.05. (C) Number of archie selected
variants of each component that are (dark) or not (light) trans-pco
signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_13_plt_archie_pco">
Past versions of 09_13_plt_archie_pco.pdf
</button>
</p>
<div id="fig-09_13_plt_archie_pco" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_13_plt_archie_pco.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="archie-on-our-simulated-data-way-2" class="section level2">
<h2>2. Archie on our simulated data (Way 2)</h2>
<div id="how-1" class="section level3">
<h3>How?</h3>
<p>Input of archie -</p>
<ul>
<li><p><span class="math inline">\(\Sigma_{GG}\)</span></p>
<p>Column-correlations of genetic variants.</p></li>
<li><p><span class="math inline">\(\Sigma_{EE}\)</span></p>
<p>Column-correlations of gene expressions.</p></li>
<li><p><span class="math inline">\(\Sigma_{GE}\)</span></p>
<p>Cross-covariance matrix between variants and gene
expressions.</p></li>
</ul>
<p>Our simulated data -</p>
<ul>
<li><p>Gene module of size 101, and one snp.</p></li>
<li><p>In this case, <span class="math inline">\(\Sigma_{GG}=1\)</span>,
<span class="math inline">\(\Sigma_{EE}=\Sigma\)</span></p></li>
<li><p><span class="math inline">\(\Sigma_{GE}\)</span> is tricky
<strong>[Is it correct?]</strong></p>
<p>According to archie paper, <span
class="math inline">\((\Sigma_{GE})_{ij} =
\frac{Z_{ij}}{\sqrt{2Nmaf_i(1-maf_i)}}\)</span>.</p>
<p>We don’t have <span class="math inline">\(maf_i\)</span>. Instead, I
used <span class="math inline">\((\Sigma_{GE})_{ij} =
\frac{\beta}{\sigma_b}\)</span>.</p>
<p>So, <span class="math inline">\((\Sigma_{GE})_{ij}\)</span> only
depends on <span class="math inline">\(prop_caus\)</span>, <span
class="math inline">\(\sigma^2_b\)</span>.</p></li>
<li><p>Simulation params <strong>[Is it correct?]</strong></p>
<p>I change various <span class="math inline">\(\sigma^2_b\)</span>,
generate <span class="math inline">\(\beta \sim N(0,
\sigma^2_b)\)</span> for <span class="math inline">\(\gamma\)</span>
SNPs, and <span class="math inline">\(0\)</span> otherwise, estimate
<span class="math inline">\((\Sigma_{GE})\)</span>, run 1000 simulation,
calculate 1000 cc value by archie.</p>
<p>A bit different than trans-pco simulation, where I generated 10k z
scores for each simulation.</p></li>
</ul>
</div>
<div id="results-1" class="section level3">
<h3>Results</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_06_plt_archie_cc_value.pdf" title="Figure: Distribution of archie cc value of simulated data across various genetic variance. Box plot for 1000 simulations." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of archie cc value of simulated data across various
genetic variance. Box plot for 1000 simulations.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_06_plt_archie_cc_value">
Past versions of 09_06_plt_archie_cc_value.pdf
</button>
</p>
<div id="fig-09_06_plt_archie_cc_value" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_06_plt_archie_cc_value.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations-1" class="section level3">
<h3>Observations</h3>
<ul>
<li><p>All cc values are small. Power is low no matter what cc value
cutoff is used.</p></li>
<li><p>Not sure results are correct, mainly due to confusions on running
archie code -</p>
<ul>
<li><p>Is archie <span class="math inline">\((\Sigma_{GE})\)</span>
correct? Only depend on caus, var? not N?</p></li>
<li><p>Archie code bug in W?</p>
<p>In archie paper, <span class="math inline">\(W = \Sigma_{GG}^{-1/2}
\Sigma_{GE} \Sigma_{EE}^{-1/2}\)</span>.</p>
<p>However, in code
<code>W &lt;- Sigma_GG%*%Sigma_GE%*%Sigma_EE;</code></p></li>
</ul></li>
<li><p>Were my previous simulations of trans-pco correct?</p>
<p>For (gene set, a snp), I generated one <span
class="math inline">\(\beta \sim N(0, \sigma^2_b)\)</span> for 10k snps
in each simulation. In one simulation, 10k SNPs have z scores generated
from same <span class="math inline">\(\beta\)</span> of <span
class="math inline">\(z \sim N_K(\sqrt{n} \beta, \Sigma_{K \times
K})\)</span>. Across simulation, <span
class="math inline">\(\beta\)</span> are different.</p></li>
</ul>
</div>
<div id="new-equation-to-estimate-sigma_ge" class="section level3">
<h3><del>New equation to estimate <span
class="math inline">\(\Sigma_{GE}\)</span></del></h3>
<p><del>It should be <span class="math inline">\((\Sigma_{GE})_{ij} =
\frac{\beta}{\sigma_b} \sqrt{N}\)</span>. So, <span
class="math inline">\((\Sigma_{GE})_{ij}\)</span> depends on <span
class="math inline">\(prop_caus\)</span>, <span
class="math inline">\(\sigma^2_b\)</span>, <span
class="math inline">\(N\)</span>.</del></p>
<p><del>I run 1000 simulation using this new equation. cc values of
simulated data shown in figure below. Observations keep same -</del></p>
<p><del>- All cc values are small. Power is low no matter what cc value
cutoff is used.</del></p>
<p><del>- cc values are even smaller (&lt;1e-4) than above
case.</del></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_13_plt_archie_cc_value_addN.pdf" title="Figure: Distribution of archie cc value of simulated data across various genetic variance. Box plot for 1000 simulations. Use new equation adding sqrt{N}." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of archie cc value of simulated data across various
genetic variance. Box plot for 1000 simulations. Use new equation adding
sqrt{N}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_13_plt_archie_cc_value_addN">
Past versions of 09_13_plt_archie_cc_value_addN.pdf
</button>
</p>
<div id="fig-09_13_plt_archie_cc_value_addN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_13_plt_archie_cc_value_addN.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="new-equation-to-estimate-sigma_ge-correct-version"
class="section level3">
<h3>New equation to estimate <span
class="math inline">\(\Sigma_{GE}\)</span> (Correct version)</h3>
<p><span class="math inline">\((\Sigma_{GE})_{ij} =
\frac{Z_{GE}}{\sqrt{N}}\)</span></p>
<p>I run 1000 simulation using this new equation. cc values of simulated
data shown in figure below. Observations -</p>
<ul>
<li><p>cc values are large.</p></li>
<li><p>Archie selected more genes than real target genes.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_20_plt_archie_cc_value_z.pdf" title="Figure: (LEFT) Distribution of archie cc value of simulated data across various genetic variance. Box plot for 1000 simulations. (RIGHT) Numebr of selected genes by archie on simulated alternative data across various genetic variance. Dashed line is the true number of target genes." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_20_plt_archie_cc_value_z">
Past versions of 09_20_plt_archie_cc_value_z.pdf
</button>
</p>
<div id="fig-09_20_plt_archie_cc_value_z" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/asset/2023/09_20_plt_archie_cc_value_z.pdf" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/09_20_plt_archie_selected_gene_z.pdf" title="Figure: (LEFT) Distribution of archie cc value of simulated data across various genetic variance. Box plot for 1000 simulations. (RIGHT) Numebr of selected genes by archie on simulated alternative data across various genetic variance. Dashed line is the true number of target genes." width="40%" type="application/pdf" />
<p class="caption">
Figure: (LEFT) Distribution of archie cc value of simulated data across
various genetic variance. Box plot for 1000 simulations. (RIGHT) Numebr
of selected genes by archie on simulated alternative data across various
genetic variance. Dashed line is the true number of target genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_20_plt_archie_selected_gene_z">
Past versions of 09_20_plt_archie_selected_gene_z.pdf
</button>
</p>
<div id="fig-09_20_plt_archie_selected_gene_z" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/asset/2023/09_20_plt_archie_selected_gene_z.pdf" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cutoff-of-cc-value" class="section level3">
<h3>Cutoff of cc value</h3>
<p>Previously, I thought ~0.07 cc value is low. But I checked cc-values
of signals in archie paper, turns out the cc values are also ~0.08.</p>
<ul>
<li><p>Cutoff used in archie paper</p>
<p>Their cc value of significant signals are low as ~0.08 <a
href="https://www.nature.com/articles/s41467-022-31845-9/figures/3">figure
in their paper</a>.</p>
<p>So we can’t define significance simply based on a general sense of
correlation.</p></li>
<li><p>Solutions to define cutoff</p>
<p>Use empirical null of cc values of null simulations</p>
<p>I wanted to calculate p value of cc value of null simulations, as the
way they used in the above figure, where they use competitive null
distributions.</p>
<p>Here is how they did it -</p>
<blockquote>
<p>Then we use ARCHIE with the same sparsity levels as the original
analysis, to extract the gene and variant components and calculate
corresponding cc values. We repeat this step multiple (M) times to
generate a competitive null distribution of cc values. We then evaluate
the observed cc-values from the original analysis against the
corresponding competitive null distributions to calculate the
p-value.</p>
</blockquote>
<p>p-value = expected number of null cc-value larger than the observed
cc-value (wrong equation (5) used in the paper). Then we will have P
value distribution and calculate the power under <span
class="math inline">\(\alpha\)</span>.</p>
<p>I ran 1e+6 null simulations, obtained an empirical distribution of
null cc values. Then calculate p values for each observed cc value. Then
do qvalue to adjust for multiple testing. Lastly, calculate power,
i.e. how many adjusted observed p values &lt; 0.05.</p>
<p>Observations -</p>
<ul>
<li><p>P values are deflated.</p></li>
<li><p>None of 1000 simulations are significant under 5% fdr,
i.e. adjusted p &lt; 0.05.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_20_plt_archie_null_cc_value_z.pdf" title="Figure: (LEFT) Distribution of null archie cc value of simulated null data. Box plot for 1e+6 simulations. (RIGHT) Numebr of selected genes by archie on simulated null data. Dashed line is the true number of target genes." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_20_plt_archie_null_cc_value_z">
Past versions of 09_20_plt_archie_null_cc_value_z.pdf
</button>
</p>
<div id="fig-09_20_plt_archie_null_cc_value_z" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/asset/2023/09_20_plt_archie_null_cc_value_z.pdf" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/09_20_plt_archie_null_selected_gene_z.pdf" title="Figure: (LEFT) Distribution of null archie cc value of simulated null data. Box plot for 1e+6 simulations. (RIGHT) Numebr of selected genes by archie on simulated null data. Dashed line is the true number of target genes." width="40%" type="application/pdf" />
<p class="caption">
Figure: (LEFT) Distribution of null archie cc value of simulated null
data. Box plot for 1e+6 simulations. (RIGHT) Numebr of selected genes by
archie on simulated null data. Dashed line is the true number of target
genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_20_plt_archie_null_selected_gene_z">
Past versions of 09_20_plt_archie_null_selected_gene_z.pdf
</button>
</p>
<div id="fig-09_20_plt_archie_null_selected_gene_z" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/asset/2023/09_20_plt_archie_null_selected_gene_z.pdf" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_20_archie_p_sim.pdf" title="Figure: (A) QQ plot of empirical nominal p values. (B) Distribution of empirical nominal p values. (C) Distribution of adjusted p values. Dashed line is 0.05 fdr level." width="50%" type="application/pdf" />
<p class="caption">
Figure: (A) QQ plot of empirical nominal p values. (B) Distribution of
empirical nominal p values. (C) Distribution of adjusted p values.
Dashed line is 0.05 fdr level.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_20_archie_p_sim">
Past versions of 09_20_archie_p_sim.pdf
</button>
</p>
<div id="fig-09_20_archie_p_sim" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a0755ce84dfa6219b49c45710c2464a5cf9fa880/docs/asset/2023/09_20_archie_p_sim.pdf" target="_blank">a0755ce</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Look at selection of SNPs &amp; genes</p>
<p>Another way to look at the performance of archie is to check if the
selected genes by archie are real target genes (we have 30% genes out of
101 have non-zero effect). So I looked at the numebr of selected genes
by archie on simulated alternative data.</p>
<p>Archie selected more genes than real target genes.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/09_13_plt_archie_selected_gene.pdf" title="Figure: Numebr of selected genes by archie on simulated alternative data across various genetic variance. Box plot for 1000 simulations. LEFT: old derivation. RIGHT: new derivation with sqrt{N}." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_13_plt_archie_selected_gene">
Past versions of 09_13_plt_archie_selected_gene.pdf
</button>
</p>
<div id="fig-09_13_plt_archie_selected_gene" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_13_plt_archie_selected_gene.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/09_13_plt_archie_selected_gene_addN.pdf" title="Figure: Numebr of selected genes by archie on simulated alternative data across various genetic variance. Box plot for 1000 simulations. LEFT: old derivation. RIGHT: new derivation with sqrt{N}." width="30%" type="application/pdf" />
<p class="caption">
Figure: Numebr of selected genes by archie on simulated alternative data
across various genetic variance. Box plot for 1000 simulations. LEFT:
old derivation. RIGHT: new derivation with sqrt{N}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_13_plt_archie_selected_gene_addN">
Past versions of 09_13_plt_archie_selected_gene_addN.pdf
</button>
</p>
<div id="fig-09_13_plt_archie_selected_gene_addN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/17f3513a8f932600c6a08a4709de6fd87d490192/docs/asset/2023/09_13_plt_archie_selected_gene_addN.pdf" target="_blank">17f3513</a>
</td>
<td>
liliw-w
</td>
<td>
2023-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="aug-16-aug-23" class="section level1">
<h1>Aug 16 &amp; Aug 23</h1>
<div id="change-the-way-to-define-overlap-of-a-window-and-a-peak"
class="section level2">
<h2>1. Change the way to define overlap of a window and a peak</h2>
<div id="new-way" class="section level3">
<h3>New way</h3>
<center>
<div class="float">
<img src="asset/2023/08_23_def_overlap_window_peak.jpg"
style="width:40.0%" alt="Define overlap of window and peak." />
<div class="figcaption">Define overlap of window and peak.</div>
</div>
</center>
<ul>
<li><p>Original way</p>
<p>In the original def, (1) a window overlap with a peak if this peak’s
center is within the window. (2) a peak overlap with a window if this
peak’s center is within the window.</p></li>
<li><p>Issue</p>
<p>However, there is issue with this def.</p>
<p>When width_window &gt; width_peak (mark H3K27ac), this def is
okay.</p>
<p>But when width_window &lt; width_peak (mark H3K36me3), even when a
window is included within a peak, the peak’s center can still be outside
the window.</p></li>
<li><p>New way</p>
<p>So the new way is, a window overlap with a peak OR a peak overlap
with a window, no matter width_window &gt; width_peak OR width_window
&lt; width_peak, if</p>
<p><strong>Peak center within window OR window center within
peak.</strong></p>
<p>Using center ensures at least half of the shorter one (window or
peak) is included in the other long one.</p></li>
</ul>
</div>
<div id="updated-figures-observations" class="section level3">
<h3>Updated figures &amp; observations</h3>
<ul>
<li><p>New way works</p>
<p>Look at case of overlapseg (Lower panel right purple).</p>
<p>H3K36ME3. Changed. Indicates the new way works.</p>
<p>H3K27ac. Stay same. Indicates both old and new way work, as window
larger than peak size.</p></li>
<li><p>Less new signals but still many</p>
<p>Look at upper panel right purple.</p>
<p>Less purple that are cwindows but not cpeaks.</p></li>
<li><p>Less signals not overlap with any peak, but still many.</p>
<p>Look at lower panel right purple.</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>H3K36ME3_window50kb_cis500kb_seg2kb</li>
</ol>
<ul>
<li><p>peak_overlapseg</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf"
style="width:40.0%" /></p></li>
<li><p>peak_overlap</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf"
style="width:40.0%" /></p></li>
<li><p>prop_40</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40_new.pdf"
width="40%">New</a> <a
href="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf"
width="40%">Old</a></p></li>
<li><p>prop_45</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45_new.pdf"
width="40%">New</a> <a
href="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf"
width="40%">Old</a></p></li>
<li><p>prop_50</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50_new.pdf"
width="40%">New</a> <a
href="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf"
width="40%">Old</a></p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>H3K36ME3_window50kb_cis500kb_seg5kb</li>
</ol>
<ul>
<li><p>peak_overlapseg</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf"
style="width:40.0%" /></p></li>
<li><p>peak_overlap</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf"
style="width:40.0%" /></p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>H3K36ME3_window50kb_cis500kb_seg1kb</li>
</ol>
<ul>
<li><p>peak_overlap</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap_new.pdf"
width="40%">New</a> <a
href="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf"
width="40%">Old</a></p></li>
</ul>
<ol start="4" style="list-style-type: decimal">
<li>Previous figures on H3K27ac</li>
</ol>
<ul>
<li><p>peak_overlapseg</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf"
style="width:40.0%" /></p></li>
<li><p>peak_overlap</p>
<p><embed
src="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr_new.pdf"
style="width:40.0%" /> <embed
src="asset/2023/07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr.pdf"
style="width:40.0%" /></p></li>
<li><p>prop_10</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10_new.pdf"
width="40%">New</a> <a
href="asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10.pdf"
width="40%">Old</a></p></li>
<li><p>prop_15</p>
<p><a
href="asset/2023/08_23_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15_new.pdf"
width="40%">New</a> <a
href="asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15.pdf"
width="40%">Old</a></p></li>
</ul>
</div>
</div>
<div id="revision-on-paper-1---add-simulations" class="section level2">
<h2>2. Revision on paper 1 - Add simulations</h2>
<p><strong>Goal - According to reviewer 3, to show multi is not always
more powerful than uni test, at some cases.</strong> -</p>
</div>
<div id="reviewers-comment" class="section level2">
<h2>Reviewer’s comment</h2>
<ol style="list-style-type: decimal">
<li>at high level of sparsity</li>
</ol>
<p>In current simulations with high sparsity, power of all methods is
too low to be differentiated. So, to increase power at this setting to
differentiate methods, I change to -</p>
<ul>
<li><p>increase effect sizes</p>
<p>e.g. change sparsity level in Fig. S1</p>
<p>[<strong>This is equivalent to say to zoom in Fig. 2A, to make three
tests differiatable from each other - increase power and pull them away
from 0.</strong>]</p></li>
<li><p><del>Or, use a larger sample size</del></p>
<blockquote>
<p>“if you think increasing effect size in high sparsity setting is
artificial”</p>
</blockquote></li>
</ul>
<p><del>2. low level of correlation among variates</del></p>
<p>[<strong>I will do the case of “increase effect sizes”.</strong>]</p>
</div>
<div id="simulation-params" class="section level2">
<h2>Simulation params</h2>
<p><span class="math inline">\(z \sim N_K(\sqrt{n} \beta, \Sigma_{K
\times K})\)</span>, where <span class="math inline">\(\beta \sim N(0,
\sigma^2_b)\)</span> for <span class="math inline">\(\gamma\)</span>
SNPs, and <span class="math inline">\(0\)</span> otherwise.</p>
<p>Params - <span class="math inline">\(\Sigma_{K \times K}\)</span>,
<span class="math inline">\(n\)</span>, <span
class="math inline">\(\gamma\)</span>, <span
class="math inline">\(\sigma^2_b\)</span>.</p>
</div>
<div id="how-high-sparsity-large-effect-size" class="section level2">
<h2>How? (High sparsity, large effect size)</h2>
<p>High sparsity in Fig. 2A (<span class="math inline">\(\gamma = 1\%,
5\%, 10\%\)</span>), with larger genetic variance <span
class="math inline">\(\sigma^2_b = 0.005, 0.01, 0.05, 0.1,
0.2\)</span>.</p>
<p>101 genes, N 500, 10k SNPs, 1k simulations, 5% fdr level.</p>
<ul>
<li><p>Calculate FDR &amp; power</p>
<p>Based on null empirical FDR (<span
class="math inline">\(10^7\)</span> = null SNPs).</p>
<p>#positive SNPs/#SNPs.</p></li>
</ul>
</div>
<div id="figures-observations" class="section level2">
<h2>Figures &amp; Observations</h2>
<ul>
<li><p>With effect sizes being increased, pco and minp have larger power
at high sparsity, thus can be well compared - power away from
0.</p></li>
<li><p>When 1% (1) gene is causal, minp is better.</p></li>
<li><p>When 5% or 10% genes are causal, pco is better.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.005_N500_K101.pdf" title="Figure: Power plot across various genetic variances. x-axis shows causal proportion. y-axis shows power. Color for three methods." width="25%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.005_N500_K101">
Past versions of
08_23_plt_power_lowCaus_highb_changecaus_varb0.005_N500_K101.pdf
</button>
</p>
<div
id="fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.005_N500_K101"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.005_N500_K101.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.01_N500_K101.pdf" title="Figure: Power plot across various genetic variances. x-axis shows causal proportion. y-axis shows power. Color for three methods." width="25%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.01_N500_K101">
Past versions of
08_23_plt_power_lowCaus_highb_changecaus_varb0.01_N500_K101.pdf
</button>
</p>
<div
id="fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.01_N500_K101"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.01_N500_K101.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.05_N500_K101.pdf" title="Figure: Power plot across various genetic variances. x-axis shows causal proportion. y-axis shows power. Color for three methods." width="25%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.05_N500_K101">
Past versions of
08_23_plt_power_lowCaus_highb_changecaus_varb0.05_N500_K101.pdf
</button>
</p>
<div
id="fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.05_N500_K101"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.05_N500_K101.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.1_N500_K101.pdf" title="Figure: Power plot across various genetic variances. x-axis shows causal proportion. y-axis shows power. Color for three methods." width="25%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.1_N500_K101">
Past versions of
08_23_plt_power_lowCaus_highb_changecaus_varb0.1_N500_K101.pdf
</button>
</p>
<div id="fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.1_N500_K101"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.1_N500_K101.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.2_N500_K101.pdf" title="Figure: Power plot across various genetic variances. x-axis shows causal proportion. y-axis shows power. Color for three methods." width="25%" type="application/pdf" />
<p class="caption">
Figure: Power plot across various genetic variances. x-axis shows causal
proportion. y-axis shows power. Color for three methods.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.2_N500_K101">
Past versions of
08_23_plt_power_lowCaus_highb_changecaus_varb0.2_N500_K101.pdf
</button>
</p>
<div id="fig-08_23_plt_power_lowCaus_highb_changecaus_varb0.2_N500_K101"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_23_plt_power_lowCaus_highb_changecaus_varb0.2_N500_K101.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="revision-on-paper-1---compare-to-dutta-et.-al."
class="section level2">
<h2>3. Revision on paper 1 - Compare to Dutta et. al.</h2>
<div id="their-idea" class="section level3">
<h3>Their idea</h3>
<ul>
<li><p>Goal</p>
<p>Estimate correlation between gene sets associated to set of
trait-related SNPs.</p></li>
<li><p>Method</p>
<p>Use sparse CCA.</p></li>
<li><p>Result</p>
<p>Selected genes associated with selected variants from sets.</p></li>
<li><p>Data</p>
<p>eQTLGen sum stats, LD matrix among variants, co-expression matrix
among genes.</p></li>
<li><p>Output</p>
<p>cc value (association between the selected sets of variants and
genes). Not p value. They use empirical p value by re-sampling random
eQTLGen SNPs and genes.</p>
<p>P value is for trait specific trans associations (not general
associations).</p></li>
</ul>
</div>
<div id="how-to-compare-to" class="section level3">
<h3>How to compare to?</h3>
<ul>
<li><p>Why archie and trans-pco are not directly comparable?</p>
<ul>
<li><p>Goals of ARCHIE <del>and the standard analysis</del> and our
method are different and therefore not directly comparable.</p></li>
<li><p>ARCHIE identify gene sets associated to set of variants.</p></li>
<li><p>Output cc value (measure of association between the selected sets
of variants and genes) not p value. empirical p value for trait specific
associations.</p></li>
</ul></li>
</ul>
<p>[<strong>But I come up with several ways for a general comparison
-</strong>]</p>
<ol style="list-style-type: decimal">
<li>Direct comparison - compare eQTLGen results</li>
</ol>
<p>This is to compare <strong>trait specific associations</strong> under
competitive null. To see if archie genes and variants were replicated in
our analysis.</p>
<ul>
<li><p>Compare gene component (selected genes)</p>
<p>See if selected genes are replicated in trans target gene modules.
<del>If same module (number of module distributed). If same
variants.</del></p></li>
</ul>
<blockquote>
<p>The set of selected target genes in the gene component is one of the
key outputs of ARCHIE.</p>
</blockquote>
<ul>
<li><p>Compare variant component (selected variants)</p>
<p>See if selected variants are replicated in trans signals. <del>If
same target module.</del></p></li>
<li><p><del>Association patterns of (module, variants) including these
components.</del></p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Under our case - reduce archie multiple variants to one single
variant</li>
</ol>
<p>This is to compare general associations under general null.</p>
<p>Run archie for each module we analyzed in eQTLGen and a SNP. Only use
top component for significance. Try different cutoff on cc-value to
claim significance and compare signals. Not to use their way of using
empirical null p, computational expensive.</p>
<ol start="3" style="list-style-type: decimal">
<li><del>Under their case - aggregate trans-pco single variants as a
set</del></li>
</ol>
<p>[<strong>I will use the first way -</strong>]</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_23_fig_compare_archie.pdf" title="Figure: Comparison with archie results. x-axis shows significant pairs of selected gene and variant set across three traits. 2/1/2 pairs for prostate cancer/schizophrenia/UC. y-axis shows the number of selected (A) genes and (B) variants. Lightest blue is the total number of selected (A) genes and (B) variants by archie. Middle blue is the number of these (A) genes and (B) variants included in our eQTLGen analysis. Dark blue is (A) genes one of our trans target modules, (B) variants that are also signals in our analysis. Red line shows the number of replicated signals whose target module also include their corresponding selected genes by archie." width="40%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison with archie results. x-axis shows significant pairs
of selected gene and variant set across three traits. 2/1/2 pairs for
prostate cancer/schizophrenia/UC. y-axis shows the number of selected
(A) genes and (B) variants. Lightest blue is the total number of
selected (A) genes and (B) variants by archie. Middle blue is the number
of these (A) genes and (B) variants included in our eQTLGen analysis.
Dark blue is (A) genes one of our trans target modules, (B) variants
that are also signals in our analysis. Red line shows the number of
replicated signals whose target module also include their corresponding
selected genes by archie.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_23_fig_compare_archie">
Past versions of 08_23_fig_compare_archie.pdf
</button>
</p>
<div id="fig-08_23_fig_compare_archie" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/c34957bae42e29e3d3cb607c63032e73c63e55a6/docs/asset/2023/08_23_fig_compare_archie.pdf" target="_blank">c34957b</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Their results</p>
<p>Pairs of (selected genes, selected variants of a trait) with
significant correlations.</p>
<p>Only results of three traits are available. They didn’t update
results of all 29 traits they analyzed.</p>
<p>2/1/2 pairs of (selected genes, selected variants of a trait) for
prostate cancer/schizophrenia/UC.</p></li>
<li><p>Our results</p>
<p>DGN 166 co-expression modules and eQTLGen sum stats.</p>
<blockquote>
<p>Of the 166 co-expression gene modules identified in DGN, we used 129
modules with reliable correlation matrix approximations to ensure the
trans-eQTL signals are well-controlled for inflation.</p>
</blockquote>
<blockquote>
<p>We identified 8116 trans-eQTL SNP-gene coexpression module pairs,
corresponding to 2161 eQTLGen test SNPs and 122 gene modules (Figure 5B,
Table S11);</p>
</blockquote></li>
<li><p>Comparison &amp; observations</p>
<ul>
<li><p>Among selected genes by archie, all of those included in our
eQTLGen analysis were in a trans target module (122). Could this because
122 modules out of 129 modules were all trans target module?</p></li>
<li><p>Among selected variants by archie, most were included in our
analysis. But less than half were also trans signals.</p></li>
<li><p>And most of these trans signals are signals of the target module
that include their corresponding selected genes by archie.</p></li>
</ul></li>
<li><p>Add a paragraph</p>
<blockquote>
<p>Although ARCHIE also … identify gene sets associated to trait-related
SNPs (regulated by sets of variants) by aggregating weaker trans
associations, our method has different goals and are not directly
comparable. Their primary objective is to capture trait specific
trans-associations among gene and variant sets in the background of
broad trans-associations that are expected to be seen in the genome,
whereas our aim is to identify a comprehensive map of association in
single-variant-module pairs irrespective of trait specificity. To
compare generally …, we looked into if the selected genes/variants were
replicated in trans target gene modules/trans signals (eQTLGen, supp
notes, supp fig, supp table).</p>
</blockquote></li>
</ul>
</div>
</div>
<div id="zichengs-experience-on-scataq" class="section level2">
<h2>4. Zicheng’s experience on scataq</h2>
<ul>
<li><p>Observed Inflation</p>
<p>Due to the error of not doing rank normal. Have been
corrected.</p></li>
<li><p>Caution on selecting window size for various cell types</p>
<p>To keep peaks within window fewer than sample size, which can be
small for some minor cell types after samples filtering. For these cell
types, sample size small, window size should be small.</p>
<p>Zicheng did whole blood, large cell type, no problem.</p></li>
<li><p>Caution on normalization by rank normal peaks for minor cell
types</p>
<p>For minor cell types, peak read depth low, many peaks can have few
reads. normalization points clustered at low reads. Should find out a
way to remove or …</p></li>
</ul>
</div>
</div>
<div id="aug-09" class="section level1">
<h1>Aug 09</h1>
<div id="H3K27ac-seg1b-peak-overlapseg" class="section level2">
<h2>0. MultiCaQ-S for H3K27ac using QC peak_overlapseg</h2>
<ul>
<li><p>Goal</p>
<p>To show <strong>peak_overlapseg</strong> for mark H3K27ac. Should be
no much difference from previous <strong>peak_overlap</strong> results,
as H3K27ac peaks are well defined.</p>
<p>[<code>peak_overlapseg</code> means to retain only segs included by a
peak.]</p>
<p>[<code>peak_overlap</code> means to retain top X segs with high cpm.
X is the number of segs included by any peak.]</p></li>
<li><p>Signals</p>
<p>Out of 26462 windows, 2994 are cwindows at FDR &lt; 5%.</p>
<p>[By comparison, peak_overlap: Out of 21290 windows, 2846 are cwindows
at FDR &lt; 5%.]</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf" title="Figure: MultiCaQ-S for mark H3K27ac. QC filter on segments overlap with original peaks. (1) Comparison to univariate cPeaks. (2) Enrichment of eQTL in extra signals." width="25%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf" title="Figure: MultiCaQ-S for mark H3K27ac. QC filter on segments overlap with original peaks. (1) Comparison to univariate cPeaks. (2) Enrichment of eQTL in extra signals." width="25%" type="application/pdf" />
<p class="caption">
Figure: MultiCaQ-S for mark H3K27ac. QC filter on segments overlap with
original peaks. (1) Comparison to univariate cPeaks. (2) Enrichment of
eQTL in extra signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K27AC_window50kb_cis500kb_seg1kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="H3K36me3-seg1b-peak-overlap" class="section level2">
<h2>1. Change segment size of MultiCaQ-S for H3K36me3</h2>
<div id="smaller-seg-size-1kb" class="section level3">
<h3>Smaller seg size (1kb)</h3>
<ul>
<li><p>Goal</p>
<p>To show <strong>seg1kb</strong> results for mark H3K36me3. Compare to
seg2kb.</p></li>
<li><p>Signals</p>
<p>seg1kb: Out of 44171 windows, 1028 are cwindows at FDR &lt; 5%. 790
extra cwindows, 241 overlap with eQTLs.</p>
<p>seg2kb: Out of 36688 windows, 1139 are cwindows at FDR &lt; 5%. 892
extra cwindows, 279 overlap with eQTLs.</p></li>
<li><p>Observations</p>
<ul>
<li><p>Using 1kb seg has fewer cwindows than using 2kb seg.</p></li>
<li><p>Use even larger seg &amp; window size?</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf" title="Figure: MultiCaQ-S for mark H3K36me3. Using 1kb segs. (1) Comparison to univariate cPeaks. (2) Enrichment of eQTL in extra signals." width="25%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf" title="Figure: MultiCaQ-S for mark H3K36me3. Using 1kb segs. (1) Comparison to univariate cPeaks. (2) Enrichment of eQTL in extra signals." width="25%" type="application/pdf" />
<p class="caption">
Figure: MultiCaQ-S for mark H3K36me3. Using 1kb segs. (1) Comparison to
univariate cPeaks. (2) Enrichment of eQTL in extra signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg1kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="larger-seg-size-5kb" class="section level3">
<h3>Larger seg size (5kb)</h3>
<ul>
<li><p>Why?</p>
<p>Due to observations from <a
href="#H3K36me3-seg1b-peak-overlap">section</a> that using smaller seg
size (1kb) has fewer cwindows than using 2kb seg.</p></li>
<li><p>Goal</p>
<p>Increase seg size to 5kb.</p>
<p>Proportion? Group size? Why not window size?</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>Segment group</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlap.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlap">
Past versions of
08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlapseg.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Segment groups of H3K36me3_window50kb_seg5kb, using QC
peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlapseg">
Past versions of
08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg5kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Similar pattern as the case of using 2kb seg.</p>
<p>Many groups include full segments (i.e. 10 = 50kb/5kb) for QC
<code>peak_overlapseg</code>, as segments are adjacent to each other,
while segments spans across groups for QC <code>peak_overlap</code>, as
they were chosen as top segs.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>cWindow</li>
</ol>
<table>
<caption>Out of *** windows/peaks, *** are cwindows/cpeaks at FDR &lt;
5%.</caption>
<colgroup>
<col width="58%" />
<col width="24%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">case</th>
<th align="right">total.window.peak</th>
<th align="right">cwindow.peak</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Univariate</td>
<td align="right">14000</td>
<td align="right">338</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg2kb_peak_overlap</td>
<td align="right">36688</td>
<td align="right">1139</td>
</tr>
<tr class="odd">
<td align="left">H3K36me3_window50kb_seg2kb_peak_overlapseg</td>
<td align="right">24706</td>
<td align="right">636</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg5kb_peak_overlap</td>
<td align="right">28106</td>
<td align="right">1355</td>
</tr>
<tr class="odd">
<td align="left">H3K36me3_window50kb_seg5kb_peak_overlapseg</td>
<td align="right">24165</td>
<td align="right">759</td>
</tr>
</tbody>
</table>
<ol start="3" style="list-style-type: decimal">
<li>Signal comparison v.s. uni</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg5kb,
using QC peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>More new cwindows found by using 5kb seg size than 2kb
seg.</p></li>
<li><p>Similar as case of 2kb seg, we should include more seg than just
segs overlap with any peak.</p></li>
</ul>
<ol start="4" style="list-style-type: decimal">
<li>Enrichment of eQTL in extra signals</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="25%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg5kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="25%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg5kb,
using QC peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="multicaq-s-for-h3k36me3-keeping-larger-proportion-of-segments"
class="section level2">
<h2>2. MultiCaQ-S for H3K36me3 keeping larger proportion of
segments</h2>
<ul>
<li><p>Why?</p>
<p>Due to observations from <a
href="#H3K36me3-seg2kb-peak-overlap">previous section</a> that including
extra segments that do not overlap with peak can generate extra windows.
Therefore, I wanted to include those segs by increasing the proportion
of segs with high cpm.</p></li>
<li><p>Goal</p>
<p>To show results of including more segments that do not overlap with
originally called peaks.</p></li>
<li><p>How?</p>
<ul>
<li><p>Use proportion of segs to keep segs with high cpm.</p></li>
<li><p>~35% (486843/1378402) segments (2kb) overlap with a
peak.</p></li>
<li><p>Larger proportions: 40%, 45%, 50%.</p></li>
</ul></li>
<li><p>Observations</p>
<ul>
<li>In <a href="#H3K27ac-seg1kb-qc">previous section</a> for mark
H3K27ac, we showed including more segments that do not overlap with
originally called peaks did not increase signals.</li>
</ul></li>
</ul>
<ol style="list-style-type: decimal">
<li>Segment group</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_40.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs. Groups including higher proportion of segs have larger group sieze." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_40">
Past versions of
08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_40.pdf
</button>
</p>
<div id="fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_40"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_40.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_45.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs. Groups including higher proportion of segs have larger group sieze." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_45">
Past versions of
08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_45.pdf
</button>
</p>
<div id="fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_45"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_45.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_50.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs. Groups including higher proportion of segs have larger group sieze." width="30%" type="application/pdf" />
<p class="caption">
Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC based on
proportion of included segs. Groups including higher proportion of segs
have larger group sieze.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_50">
Past versions of
08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_50.pdf
</button>
</p>
<div id="fig-08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_50"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_seg_group_size_H3K36ME3_window50kb_seg2kb_prop_50.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>cWindow</li>
</ol>
<table>
<caption>Out of *** windows/peaks, *** are cwindows/cpeaks at FDR &lt;
5%.</caption>
<colgroup>
<col width="58%" />
<col width="24%" />
<col width="17%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">case</th>
<th align="right">total.window.peak</th>
<th align="right">cwindow.peak</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Univariate</td>
<td align="right">14000</td>
<td align="right">338</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg2kb_peak_overlap</td>
<td align="right">36688</td>
<td align="right">1139</td>
</tr>
<tr class="odd">
<td align="left">H3K36me3_window50kb_seg2kb_peak_overlapseg</td>
<td align="right">24706</td>
<td align="right">636</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg5kb_peak_overlap</td>
<td align="right">28106</td>
<td align="right">1355</td>
</tr>
<tr class="odd">
<td align="left">H3K36me3_window50kb_seg5kb_peak_overlapseg</td>
<td align="right">24165</td>
<td align="right">759</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg2kb_prop_40</td>
<td align="right">40490</td>
<td align="right">1143</td>
</tr>
<tr class="odd">
<td align="left">H3K36me3_window50kb_seg2kb_prop_45</td>
<td align="right">43760</td>
<td align="right">1179</td>
</tr>
<tr class="even">
<td align="left">H3K36me3_window50kb_seg2kb_prop_50</td>
<td align="right">46340</td>
<td align="right">1204</td>
</tr>
</tbody>
</table>
<ol start="3" style="list-style-type: decimal">
<li>Signal comparison v.s. uni</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs." width="30%" type="application/pdf" />
<p class="caption">
Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb,
using QC based on proportion of included segs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50">
Past versions of
08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf
</button>
</p>
<div
id="fig-08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="4" style="list-style-type: decimal">
<li>Enrichment of eQTL in extra signals</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs" width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_40.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs" width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_45.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb, using QC based on proportion of included segs" width="30%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb,
using QC based on proportion of included segs
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50">
Past versions of
08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf
</button>
</p>
<div
id="fig-08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_09_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_prop_50.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cwindows-v.s.-cpeaks" class="section level2">
<h2>3. cwindows v.s. cpeaks</h2>
<ul>
<li><p>Issue</p>
<p>There are windows/cwindows not overlap with any peak, while all
windows should consist of segs overlap with a peak.</p>
<p><code>count_based/nonoverlap_window/plot/plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf</code></p>
<p>To see if a window overlap with any I used center of a pea</p>
<p>All peak centers are in a window.</p></li>
<li><p>Why this issue diappears for mark H3K27AC?</p>
<p>See <a href="#H3K27ac-seg1b-peak-overlapseg">section</a>.</p></li>
<li><p>Another way to check overlap</p></li>
</ul>
</div>
</div>
<div id="july-19-july-26" class="section level1">
<h1>July 19 &amp; July 26</h1>
<div id="H3K27ac-seg1kb-qc" class="section level2">
<h2>1. Whole-genome wide QC</h2>
<div id="three-ways" class="section level3">
<h3>Three ways</h3>
<ul>
<li><p>To make #segments ~ #peaks</p></li>
<li><p>To retain segments overlap with any peak</p></li>
<li><p>Use segment cpm proportion - 10%, 15%</p>
<p>To retain seg overlap with any peak, the proportion is ~8%.
Therefore, trying 10% and 15%.</p></li>
</ul>
</div>
<div id="ways-comparison" class="section level3">
<h3>Ways comparison</h3>
<p>Add the old way using chr-wise factor</p>
<ul>
<li><p>#segment</p>
<p>QC_peak_number: 100845</p>
<p>QC_peak_overlap: 210076</p>
<p>QC_prop_10: 274161</p>
<p>QC_prop_15: 411242</p></li>
<li><p>quantile cutoff</p>
<p>0.03678267</p>
<p>0.07662449</p>
<p>0.1</p>
<p>0.15</p></li>
<li><p>cpm cutoff</p>
<p>0.4083591</p>
<p>-0.860688</p>
<p>-1.184399</p>
<p>-1.547873</p></li>
<li><p>Individual total reads</p>
<p>Min. 1st Qu. Median Mean 3rd Qu. Max.</p>
<p>10786452 14324385 15495376 15509431 16699293 19339800</p></li>
</ul>
</div>
<div id="signals" class="section level3">
<h3>Signals</h3>
<ol style="list-style-type: decimal">
<li>Segment group</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_19_plt_seg_group_window50kb_avgcpmfct2_seg1kb_peak_number.pdf" title="Figure: Segment group size of QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_19_plt_seg_group_window50kb_avgcpmfct2_seg1kb_peak_number">
Past versions of
07_19_plt_seg_group_window50kb_avgcpmfct2_seg1kb_peak_number.pdf
</button>
</p>
<div
id="fig-07_19_plt_seg_group_window50kb_avgcpmfct2_seg1kb_peak_number"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_19_plt_seg_group_window50kb_avgcpmfct2_seg1kb_peak_number.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_19_plt_seg_group_window50kb_avgcpmfct2_seg1k_peak_overlap.pdf" title="Figure: Segment group size of QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_19_plt_seg_group_window50kb_avgcpmfct2_seg1k_peak_overlap">
Past versions of
07_19_plt_seg_group_window50kb_avgcpmfct2_seg1k_peak_overlap.pdf
</button>
</p>
<div
id="fig-07_19_plt_seg_group_window50kb_avgcpmfct2_seg1k_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_19_plt_seg_group_window50kb_avgcpmfct2_seg1k_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_10.pdf" title="Figure: Segment group size of QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_10">
Past versions of
07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_10.pdf
</button>
</p>
<div id="fig-07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_10.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_15.pdf" title="Figure: Segment group size of QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p class="caption">
Figure: Segment group size of QC to make segments number be the same as
(1) peak number, (2) segments overlap with any peak, (3) top 10% segs,
and (4) top 15% segs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_15">
Past versions of
07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_15.pdf
</button>
</p>
<div id="fig-07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_15"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_seg_group_H3K27AC_window50kb_seg1k_prop_15.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Largest group has segments. Most groups have &lt;10 segments.</p>
<ol start="2" style="list-style-type: decimal">
<li>cWindow</li>
</ol>
<p>Peak_number: Out of 14523 windows, 2398 are cwindows at FDR &lt;
5%.</p>
<p>Peak_overlap: Out of 21290 windows, 2846 are cwindows at FDR &lt;
5%.</p>
<p>QC_prop_10: Out of 23750, 2806 are cwindows at FDR &lt; 5%.</p>
<p>QC_prop_15: Out of 27567, 2562 are cwindows at FDR &lt; 5%.</p>
<ol start="3" style="list-style-type: decimal">
<li>Signal comparison v.s. uni</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_number_allchr.pdf" title="Figure: Comparison to univariate cPeaks. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_number_allchr">
Past versions of
07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_number_allchr.pdf
</button>
</p>
<div
id="fig-07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_number_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_number_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr.pdf" title="Figure: Comparison to univariate cPeaks. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr">
Past versions of
07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr.pdf
</button>
</p>
<div
id="fig-07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_19_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_peak_overlap_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10.pdf" title="Figure: Comparison to univariate cPeaks. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10">
Past versions of
07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10.pdf
</button>
</p>
<div
id="fig-07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_10.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15.pdf" title="Figure: Comparison to univariate cPeaks. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p class="caption">
Figure: Comparison to univariate cPeaks. Use QC to make segments number
be the same as (1) peak number, (2) segments overlap with any peak, (3)
top 10% segs, and (4) top 15% segs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15">
Past versions of
07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15.pdf
</button>
</p>
<div
id="fig-07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg1kb_prop_15.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="4" style="list-style-type: decimal">
<li>Extra cQTLs enrichment for eQTLs</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_number.pdf" title="Figure: Enrichment of eQTLs in extra cQTLs. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_number">
Past versions of
07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_number.pdf
</button>
</p>
<div
id="fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_number"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_number.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_overlap.pdf" title="Figure: Enrichment of eQTLs in extra cQTLs. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_overlap">
Past versions of
07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_10.pdf" title="Figure: Enrichment of eQTLs in extra cQTLs. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_10">
Past versions of
07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_10.pdf
</button>
</p>
<div id="fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_10.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_15.pdf" title="Figure: Enrichment of eQTLs in extra cQTLs. Use QC to make segments number be the same as (1) peak number, (2) segments overlap with any peak, (3) top 10% segs, and (4) top 15% segs." width="23%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of eQTLs in extra cQTLs. Use QC to make segments
number be the same as (1) peak number, (2) segments overlap with any
peak, (3) top 10% segs, and (4) top 15% segs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_15">
Past versions of
07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_15.pdf
</button>
</p>
<div id="fig-07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_15"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_26_plt_enrich_extra_cqtl_eqtl_window50kb_seg1kb_prop_15.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="H3K36me3-seg2kb-peak-overlap" class="section level2">
<h2>2. MultiCAQ-S on broad mark H3K36me3</h2>
<p>In total, 95 samples.</p>
<div id="way-to-filter-segments" class="section level3">
<h3>Way to filter segments?</h3>
<p>Goal -</p>
<ul>
<li><p>To avoid peak calling (using the above QC dependent on peak
calling is fine)</p></li>
<li><p>Inaccurate peak calling (QC shouldn’t depend on peak
calling)</p></li>
</ul>
<p>There are two ways of seg filtering -</p>
<ul>
<li><p><code>peak_overlap</code></p>
<p>Use the proportion of segments that overlap with any peak as the cpm
cutoff to retain segments.</p>
<p>Clarify - Previous results on mark H3K27ac used this QC.</p></li>
<li><p><code>peak_overlapseg</code></p>
<p>Retain the exact segments that overlap with any peak.</p></li>
</ul>
</div>
<div id="wrangle-bam" class="section level3">
<h3>Wrangle bam</h3>
<ul>
<li><p>Problematic sample reads assignment of mark H3K27ac</p>
<p>There is one sample which had non-zero unassigned reads due to
ambuiguity on chr20, which was supposed to be zero. (All other chr’s and
samples were fine.)</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_02_plt_count_unassigned_ambiguity_H3K27AC_seg1kb.pdf" title="Figure: Problematic sample reads assignment of mark H3K27ac." width="40%" type="application/pdf" />
<p class="caption">
Figure: Problematic sample reads assignment of mark H3K27ac.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_count_unassigned_ambiguity_H3K27AC_seg1kb">
Past versions of 08_02_plt_count_unassigned_ambiguity_H3K27AC_seg1kb.pdf
</button>
</p>
<div id="fig-08_02_plt_count_unassigned_ambiguity_H3K27AC_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_count_unassigned_ambiguity_H3K27AC_seg1kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Speed up featureCounts</p>
<p>Previously feature counting was slow. I did below to speed up.</p>
<ul>
<li><p>Remove <code>-p</code></p></li>
<li><p>Add slurm flag <code>cpus-per-task</code> for
multi-thread</p></li>
</ul></li>
<li><p>For this mark, no samples with Unassigned_Ambiguity reads
(featureCounts summary)</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_02_plt_read_assignment_sum_H3K36ME3_seg2kb.pdf" title="Figure: FeatureCounts summary for mark H3K36me3. No Unassigned_Ambiguity reads. This is seg2kb. Same for seg1kb." width="40%" type="application/pdf" />
<p class="caption">
Figure: FeatureCounts summary for mark H3K36me3. No Unassigned_Ambiguity
reads. This is seg2kb. Same for seg1kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_read_assignment_sum_H3K36ME3_seg2kb">
Past versions of 08_02_plt_read_assignment_sum_H3K36ME3_seg2kb.pdf
</button>
</p>
<div id="fig-08_02_plt_read_assignment_sum_H3K36ME3_seg2kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_read_assignment_sum_H3K36ME3_seg2kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="parameters-and-why" class="section level3">
<h3>Parameters and why?</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_19_plt_peak_width_H3K36ME3.pdf" title="Figure: Peak width of mark H3K36me3." width="40%" type="application/pdf" />
<p class="caption">
Figure: Peak width of mark H3K36me3.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_plt_peak_width_H3K36ME3">
Past versions of 04_19_plt_peak_width_H3K36ME3.pdf
</button>
</p>
<div id="fig-04_19_plt_peak_width_H3K36ME3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_plt_peak_width_H3K36ME3.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Peak width distribution -</p>
<pre><code>Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 

341   12992   32431   70270   77592 2172910</code></pre>
<p>I wanted the window size to be approx median peak width, so that a
window can span a whole peak for at least half of the peaks.</p>
<ul>
<li><p>Segment size</p>
<p>1kb. 2kb.</p></li>
<li><p>Window size</p>
<p>50kb.</p></li>
<li><p>QC seg filter</p>
<p>peak_overlap. peak_overlapseg.</p></li>
</ul>
</div>
<div id="signals-1" class="section level3">
<h3>Signals</h3>
<ol style="list-style-type: decimal">
<li>Segment group</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlap.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlap">
Past versions of
08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlapseg.pdf" title="Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Segment groups of H3K36me3_window50kb_seg2kb, using QC
peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlapseg">
Past versions of
08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_seg_group_size_H3K36ME3_window50kb_seg2kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Window size smaller than peak size? Dynamic? Half windows have
full 2kb-seg (25), i.e. half peaks split into multiple windows.</p></li>
<li><p>We should include more seg than just segs overlap with any
peak.</p>
<p>Why? See results below, using peak_overlap QC generates extra
cwindows that don’t overlap with any peaks.</p>
<p>This makes sense, as those called peaks were not accurate, some
regions with high cpm were not included in the peaks.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>cWindow</li>
</ol>
<ul>
<li><p>Univariate</p>
<p>Out of 14000 called peaks, 338 are cpeaks at FDR &lt; 5%.</p></li>
<li><p>H3K36me3_window50kb_seg2kb_peak_overlap</p>
<p>Out of 36688 windows, 1139 are cwindows at FDR &lt; 5%.</p></li>
<li><p>H3K36me3_window50kb_seg2kb_peak_overlapseg</p>
<p>Out of 24706 windows, 636 are cwindows at FDR &lt; 5%.</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Signal comparison v.s. uni</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap">
Past versions of
08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf" title="Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison to univariate cPeaks. H3K36me3_window50kb_seg2kb,
using QC peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg">
Past versions of
08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Extra cwindows not overlap with any peak, using QC peak_overlap
(left figure, B, upper right)</p>
<p>This indicates peak calling is not good in broad mark. Many segments
with high cpm not called as peaks, but they have cQTLs by
multicaq-S.</p>
<p>Compared to narrow peak H3K27ac. Look at same panel. There is no
purple bar, which means all cwindow overlap with a peak, which means
segments with high cpm overlap with at least one peak, they are included
by peak calling, i.e. peak calling was good.</p></li>
<li><p>Some extra signals overlap with a peak but not a cpeak.</p></li>
<li><p>As mentioned above, we should include more seg than just segs
overlap with any peak.</p>
<p>Why? Using peak_overlap QC generates extra cwindows that don’t
overlap with any peaks.</p>
<p>This makes sense, as those called peaks were not accurate, some
regions with high cpm were not included in the peaks.</p>
<p>Including these extra segments generates extra signals.</p></li>
</ul>
<ol start="4" style="list-style-type: decimal">
<li>Enrichment of eQTL in extra signals</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap">
Past versions of
08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlap.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf" title="Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb, using QC peak_overlap (left) and peak_overlapseg (right)." width="40%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of eQTL in extra signals. H3K36me3_window50kb_seg2kb,
using QC peak_overlap (left) and peak_overlapseg (right).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg">
Past versions of
08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf
</button>
</p>
<div
id="fig-08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/08_02_plt_enrich_extra_cqtl_eqtl_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="july-12" class="section level1">
<h1>July 12</h1>
<div id="multicaq-s-on-mark-h3k27ac-1" class="section level2">
<h2>1. MultiCAQ-S on mark H3K27ac</h2>
<div id="parameters" class="section level3">
<h3>Parameters</h3>
<ul>
<li><p>Mark: H3K27ac</p></li>
<li><p>Segment size: 1kb</p></li>
<li><p>Window size: 50kb</p></li>
<li><p>Nonoverlap window</p></li>
<li><p>QC AVE_CPM_FCT: 2</p>
<p>To filter low abundance segments. Factor multiplying
median(ave_logcpm). CPM is chr-wise, i.e. sum reads of an individual on
a chr. [From the results, should try genome-wise QC.]</p></li>
</ul>
</div>
<div id="signals-itself" class="section level3">
<h3>Signals itself</h3>
<ol style="list-style-type: decimal">
<li>Segments and QC</li>
</ol>
<p>To filter segments, I calculate median(ave_logcpm) for each chr, and
retain segments with logcpm &gt; 2*median(ave_logcpm).</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_12_plt_seg1kb.pdf" title="Figure: QC on segments. (A) median(ave_logcpm) used for filtering segments across chr. (B) Total number of segments across chr. (C) Distribution of ave_logcpm for segments across chr. Leftmost density is for genome-wide segments. (D) Number of retained segments after QC." width="40%" type="application/pdf" />
<p class="caption">
Figure: QC on segments. (A) median(ave_logcpm) used for filtering
segments across chr. (B) Total number of segments across chr. (C)
Distribution of ave_logcpm for segments across chr. Leftmost density is
for genome-wide segments. (D) Number of retained segments after QC.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_12_plt_seg1kb">
Past versions of 07_12_plt_seg1kb.pdf
</button>
</p>
<div id="fig-07_12_plt_seg1kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_12_plt_seg1kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol style="list-style-type: upper-alpha">
<li>Chr-wise cutoff of median(ave_logcpm) is all positive. Positive
median(ave_logcpm) cutoff multiplying a factor elevates cutoff with less
segments left.</li>
</ol>
<p>With (B), shorter chr has less sum reads, thus larger cpm, larger
median(ave_logcpm) cutoff.</p>
<ol start="3" style="list-style-type: upper-alpha">
<li>Genome-wide has negative median(ave_logcpm). Multiplying it by a
factor lowers cutoff with more segments left. It doesn’t work to filter
low abundance segments. Should think about another way to filter
segments by genome-wide QC.</li>
</ol>
<ul>
<li><p>chr-wise QC</p>
<p>cpm &lt; 0: c_i/sum_i(chr) &lt; 1/10^6</p></li>
<li><p>genome-wide QC - median(cpm) &lt; 0</p>
<p>cpm &lt; 0: c_i/sum_i(all_chr) &lt; 1/10^6</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Segment group</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_12_seg_group_size_window50kb_avgcpmfct2_seg1kb.pdf" title="Figure: Segment groups." width="40%" type="application/pdf" />
<p class="caption">
Figure: Segment groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_12_seg_group_size_window50kb_avgcpmfct2_seg1kb">
Past versions of 07_12_seg_group_size_window50kb_avgcpmfct2_seg1kb.pdf
</button>
</p>
<div id="fig-07_12_seg_group_size_window50kb_avgcpmfct2_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_12_seg_group_size_window50kb_avgcpmfct2_seg1kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Largest group has 50 segments. Most groups have &lt;10 segments.</p>
<ol start="3" style="list-style-type: decimal">
<li>cWindow</li>
</ol>
<p>Out of 17921 windows, 2499 are cwindows at FDR &lt; 5%.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_12_plt_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf" title="Figure: cWindows." width="40%" type="application/pdf" />
<p class="caption">
Figure: cWindows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_12_plt_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr">
Past versions of
07_12_plt_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf
</button>
</p>
<div
id="fig-07_12_plt_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_12_plt_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="signal-comparison" class="section level3">
<h3>Signal comparison</h3>
<ol style="list-style-type: decimal">
<li>v.s. uni</li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf" title="Figure: Comparison to univariate cPeaks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison to univariate cPeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr">
Past versions of
07_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf
</button>
</p>
<div
id="fig-07_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_avgcpmfct2_seg1kb_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>MultiCAQ-S found new signals than QTLTools. But less than MultiCAQ-P,
see previous figure.</p>
<p>Try genome-wide QC? Chr-wise QC filtered too many segments on shorter
chr’s, where cutoff is high.</p>
<ul>
<li>with eQTLs</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/07_12_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb.pdf" title="Figure: Enrichment of eQTLs in new cwindows." width="40%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of eQTLs in new cwindows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-07_12_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb">
Past versions of
07_12_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb.pdf
</button>
</p>
<div
id="fig-07_12_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/07_12_plt_enrich_extra_cqtl_eqtl_window50kb_avgcpmfct2_seg1kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Those new cwindows are enriched for eQTLs.</p>
<ul>
<li>with GWAS</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>v.s. MultiCaQ-P</li>
</ol>
<p>At same window size 50kb, MultiCAQ-P found 7259 cwindows at FDR &lt;
5%, out of 20648 windows.</p>
</div>
</div>
</div>
<div id="may-31-jun-07" class="section level1">
<h1>May 31 &amp; Jun 07</h1>
<div id="coloc-cqtls-with-gwas" class="section level2">
<h2>1. coloc cQTLs with GWAS</h2>
<div id="starting-from-cqtl-loci" class="section level3">
<h3>Starting from cQTL loci</h3>
<ul>
<li><p>How?</p>
<ul>
<li><p>With 29 blood trait GWAS from Neale’s lab.</p></li>
<li><p>For each window, define 1Mb (500kb as cis) as coloc region, take
overlapped snps with GWAS.</p></li>
</ul></li>
<li><p>Numbers</p></li>
</ul>
</div>
<div id="starting-from-gwas-loci---neale-lab" class="section level3">
<h3>Starting from GWAS loci - Neale lab</h3>
<ul>
<li><p>How?</p>
<ul>
<li><p>Define GWAS loci - Start with the smallest p, define a 1Mb (500kb
as cis) flanking region as coloc region, search any remained snps with p
&lt; 1e-7.</p></li>
<li><p>For each loci, take overlapped snps with any window.</p></li>
<li><p>PP4 &gt; 0.75</p></li>
</ul></li>
<li><p>Numbers</p>
<ul>
<li>Each trait has ~400 loci, ~20 are colocalized with cQTLs.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/06_07_coloc_gwas_qtl.png" alt="Figure: GWAS loci coloc with cQTLs. 29 blood traits." width="40%" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs. 29 blood traits.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-06_07_coloc_gwas_qtl">
Past versions of 06_07_coloc_gwas_qtl.png
</button>
</p>
<div id="fig-06_07_coloc_gwas_qtl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/06_07_coloc_gwas_qtl.png" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Low coloc number - possible reasons</p>
<ul>
<li><p>Use maf for coloc. I used to use af that can be &gt;
0.5.</p></li>
<li><p>Use 36 blood trait GWAS from Astle et al 2016 Cell that Ben used.
Neale lab GWAS sum stats have some weird snps, e.g. af=1.</p></li>
<li><p>Region too wide? 500kb total.</p></li>
</ul></li>
</ul>
</div>
<div id="multicaqp-H3K27ac-gwas-coloc" class="section level3">
<h3>Use 36 blood traits - Ben used</h3>
<ul>
<li><p>Ben’s coloc results as comparison</p>
<ul>
<li><p>Sample size smaller</p></li>
<li><p>Most of trait have between 100-300 gwas loci for coloc (though a
lot of the loci are overlapping between the different gwas)</p></li>
<li><p>Overall, colocalize 329 gwas loci (not necessarily unique loci)
to 183 distinct H3K27Ac peaks (not necessarily 1 peak per loci, could be
more than one).</p></li>
</ul></li>
<li><p>How?</p>
<ul>
<li><p>Define GWAS loci - Start with the smallest p, define a 1Mb (500kb
as cis) flanking region as coloc region, search any remained snps with p
&lt; 1e-7.</p></li>
<li><p>For each loci, take overlapped snps with any window.</p></li>
<li><p>Remove MHC region (chromosome 6: 25.9–33.4 Mb, genome build
GRCh37).</p></li>
<li><p>PP4 &gt; 0.5</p></li>
</ul></li>
<li><p>Numbers</p>
<ul>
<li><p>Summary of gwas loci and coloc loci.</p>
<p>Each trait has ~200 loci, ~10 (5%) are colocalized with
cQTLs.</p></li>
<li><p>How many coloc gwas loci are extra new cQTLs (than
univariate)?</p>
<p>Most traits have only 1 coloc loci coloc with new cQTLs.</p></li>
<li><p>How many coloc gwas loci are not eQTLs?</p>
<p>Most traits have 5 coloc loci not overlap with any eQTL.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/06_07_coloc_num.pdf" title="Figure: Summary of gwas loci and coloc loci. 36 blood traits. (A) x-axis: trait. Grey bar: GWAS loci. Red bar: coloc loci. (B) Number of coloc loci across traits. (C) Coloc proportion across traits." width="40%" type="application/pdf" />
<p class="caption">
Figure: Summary of gwas loci and coloc loci. 36 blood traits. (A)
x-axis: trait. Grey bar: GWAS loci. Red bar: coloc loci. (B) Number of
coloc loci across traits. (C) Coloc proportion across traits.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-06_07_coloc_num">
Past versions of 06_07_coloc_num.pdf
</button>
</p>
<div id="fig-06_07_coloc_num" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/06_07_coloc_num.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/06_07_coloc_compare.pdf" title="Figure: Coloc loci compare with (A) if coloc loci coloc with new cQTLs compared to univariate. (B) if coloc loci overlap with any eQTL." width="40%" type="application/pdf" />
<p class="caption">
Figure: Coloc loci compare with (A) if coloc loci coloc with new cQTLs
compared to univariate. (B) if coloc loci overlap with any eQTL.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-06_07_coloc_compare">
Past versions of 06_07_coloc_compare.pdf
</button>
</p>
<div id="fig-06_07_coloc_compare" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/06_07_coloc_compare.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="other-two-marks" class="section level2">
<h2>2. Other two marks</h2>
<div id="enrichment" class="section level3">
<h3>Enrichment</h3>
</div>
</div>
</div>
<div id="may-19-may-24" class="section level1">
<h1>May 19 &amp; May 24</h1>
<div id="peak-based-all-chromosome" class="section level2">
<h2>1. Peak-based all chromosome</h2>
<ul>
<li><p>Top p for each window, correction on cis SNPs by ACAT</p></li>
<li><p>Q-value fdr across windows genome-wide</p></li>
<li><p>Univariate - QTLTools permutation p &amp; fdr q-value for each
peak</p></li>
</ul>
<p><strong>Check window &amp; peak overlaps</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_allchr.pdf" title="Figure: Overlaps between cwindow and cpeak. 10kb &amp; 50kb &amp; 100kb window." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_19_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_allchr">
Past versions of
05_19_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_allchr.pdf
</button>
</p>
<div id="fig-05_19_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr.pdf" title="Figure: Overlaps between cwindow and cpeak. 10kb &amp; 50kb &amp; 100kb window." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_19_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr">
Past versions of
05_19_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr.pdf
</button>
</p>
<div id="fig-05_19_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window100kb_cis500kb_allchr.pdf" title="Figure: Overlaps between cwindow and cpeak. 10kb &amp; 50kb &amp; 100kb window." width="30%" type="application/pdf" />
<p class="caption">
Figure: Overlaps between cwindow and cpeak. 10kb &amp; 50kb &amp; 100kb
window.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_19_plt_cpeak_cwindow_H3K27AC_window100kb_cis500kb_allchr">
Past versions of
05_19_plt_cpeak_cwindow_H3K27AC_window100kb_cis500kb_allchr.pdf
</button>
</p>
<div
id="fig-05_19_plt_cpeak_cwindow_H3K27AC_window100kb_cis500kb_allchr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_19_plt_cpeak_cwindow_H3K27AC_window100kb_cis500kb_allchr.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="the-enrichment-of-eqtls-in-extra-identified-cwindowscqtls"
class="section level2">
<h2>2. The enrichment of eQTLs in extra identified cwindows/cQTLs</h2>
<div id="how-2" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Use cwindows from results of 50kb window.</p></li>
<li><p>Find those extra cwindows (top cQTL) that don’t overlap with any
cpeaks by univariate mapping.</p></li>
<li><p>Find those extra top cQTLs that overlap with eQTLGen eQTLs.</p>
<ul>
<li><p>By overlap, I mean (1) simple overlap. (2) in LD
(R2&lt;0.8).</p></li>
<li><p>By eQTLGen eQTLs, I mean (1) all eQTLs. (2) top eQTL for each
gene.</p></li>
</ul></li>
<li><p>Quanfify the enrichment of eQTLs in cQTLs</p>
<ul>
<li><p>I finally used top cQTLs with simple overlap with all eQTLs.</p>
<p>The reason is, if the top cQTL is not the lead eQTL, but secondary
eQTL, we can still capture that kind of overlap. In addition, this way
equals to using LD with top eQTL, because if a top cQTL is in LD with
top eQTL, then this cQTL should also be an eQTL.</p></li>
</ul></li>
<li><p>Other details</p>
<ul>
<li><p>I define matched SNPs between chip-seq and eqtlgen snps by lift
over chip-seq to hg19. All successfully converted snps are considered to
be matched SNPs in eqtlgen.</p></li>
<li><p>To quantify enrichment in those extra cQTLs, I used two sets of
SNPs that belong to (1) matched SNPs &amp; cQTLs &amp; extra cQTLs. (2)
matched SNPs &amp; non-cQTLs (excluding those cQTLs).</p></li>
<li><p>One-side fisher exact test.</p></li>
</ul></li>
</ul>
</div>
<div id="enrichment-figure-with-number-and-pvalue-labelled"
class="section level3">
<h3>Enrichment figure with number and pvalue labelled</h3>
<p>The extra cQTLs are enriched for eQTLs. P-value is 1.8e-12. OR is
1.37.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/05_24_plt_enrich_extra_cqtl_eqtl_window50kb.pdf" title="Figure: The extra cQTLs are enriched for eQTLs." width="40%" type="application/pdf" />
<p class="caption">
Figure: The extra cQTLs are enriched for eQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_24_plt_enrich_extra_cqtl_eqtl_window50kb">
Past versions of 05_24_plt_enrich_extra_cqtl_eqtl_window50kb.pdf
</button>
</p>
<div id="fig-05_24_plt_enrich_extra_cqtl_eqtl_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_24_plt_enrich_extra_cqtl_eqtl_window50kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="results-for-general-cwindows" class="section level3">
<h3>Results for general cwindows</h3>
<p>I was also curious to see the enrichment in all cQTLs. The enrichment
for eQTLs is even more significant. P-value is 1.1e-122. OR is 2.24.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/05_24_plt_enrich_all_cqtl_eqtl_window50kb.pdf" title="Figure: All identified cQTLs are enriched for eQTLs." width="40%" type="application/pdf" />
<p class="caption">
Figure: All identified cQTLs are enriched for eQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_24_plt_enrich_all_cqtl_eqtl_window50kb">
Past versions of 05_24_plt_enrich_all_cqtl_eqtl_window50kb.pdf
</button>
</p>
<div id="fig-05_24_plt_enrich_all_cqtl_eqtl_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_24_plt_enrich_all_cqtl_eqtl_window50kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="may-03" class="section level1">
<h1>May 03</h1>
<div id="compare-cwindow-and-cpeak-overlaps" class="section level2">
<h2>1. Compare cwindow and cpeak overlaps</h2>
<ul>
<li><p>For window-based, get adjusted p of a window by taking the best
hit of the window and multiple by the number of cis SNPs (Bonferroni
correction). Then do q-value to get adjusted p across windows. Compare
with FDR level 5% to get significant cwindow. (Same FDR procedure as
cpeaks of Ben &amp; Carlos results.)</p></li>
<li><p>Plot overlaps between cwindow and cpeak - four types of set
replication</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/05_03_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_chr20_avgcpmfct2_seg1kb.pdf" title="Figure: Overlaps between cwindow and cpeak." width="40%" type="application/pdf" />
<p class="caption">
Figure: Overlaps between cwindow and cpeak.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_03_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_chr20_avgcpmfct2_seg1kb">
Past versions of
05_03_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_chr20_avgcpmfct2_seg1kb.pdf
</button>
</p>
<div
id="fig-05_03_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_chr20_avgcpmfct2_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/493db6ceab3eadf112c4063856ee5494a3dad77f/docs/asset/2023/05_03_plt_cpeak_cwindow_H3K27AC_window10kb_cis500kb_chr20_avgcpmfct2_seg1kb.pdf" target="_blank">493db6c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-08-28
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>genometrack of cwindows do not overlap with cpeaks</p>
<p>Check
<code>/scratch/midway3/liliw1/chromatin/test_results</code></p></li>
<li><p>genometrack of cpeaks do not overlap with cwindows</p>
<p>Check
<code>/scratch/midway3/liliw1/chromatin/test_results</code></p></li>
</ul>
</div>
<div id="add-peak-based" class="section level2">
<h2>2. Add peak-based</h2>
<ul>
<li><p>The figure</p></li>
<li><p>Compare to the seg-based above</p></li>
<li><p>genometrack plot</p></li>
</ul>
</div>
<div id="change-top-p-correction-from-using-bonferroni-to-acat"
class="section level2">
<h2>3. Change top p correction from using Bonferroni to ACAT</h2>
<ul>
<li><p>The figures</p></li>
<li><p>Compare to the seg-based &amp; peak-based above</p></li>
</ul>
</div>
</div>
<div id="apr-26" class="section level1">
<h1>Apr 26</h1>
<div id="show-top-p-of-seg-based-and-peak-based" class="section level2">
<h2>1. Show top p of seg-based and peak-based</h2>
<p>Add this figure to previous section <a
href="#try-other-cpm-cutoff-factors">Try other CPM cutoff
factors</a>.</p>
<ul>
<li><p>Top p’s are bonferroni corrected by multiplying the number of cis
SNPs of a group.</p></li>
<li><p>Only show groups with top p &lt; 1e-3.</p></li>
</ul>
</div>
<div
id="more-figures-of-comparison-between-multi-segments-pco-and-uni-merged-segments"
class="section level2">
<h2>2. More figures of comparison between multi-segments-PCO and
uni-merged-segments</h2>
<div
id="compare-nominal-top-p-values-by-multi-segments-pco-and-uni-merged-segments"
class="section level3">
<h3>Compare nominal top p-values by multi-segments-PCO and
uni-merged-segments</h3>
<p>Add this figure to previous section <a
href="#top-p-comparsion-multi-merged">Compare top p-values by
multi-segments-PCO and uni-merged-segments</a>.</p>
<p>Use <strong>nominal top p</strong>, w.o. bonferroni correction by
multiplifying number of cis SNPs of groups. We wanted to look at this
because we think cis regions should have same #independent SNPs. Not
much difference with top p with bonferroni correction.</p>
</div>
<div id="only-show-groups-with-large-top-p-difference"
class="section level3">
<h3>Only show groups with large top p difference</h3>
<p>i.e. log(topp_diff)&gt;2.</p>
<p>Add this figure to previous section <a
href="#top-p-comparsion-multi-merged">Compare top p-values by
multi-segments-PCO and uni-merged-segments</a>.</p>
</div>
<div id="what-are-these-groups" class="section level3">
<h3>What are these groups?</h3>
<p>Add figures to previous section <a
href="#top-p-comparsion-multi-merged">Compare top p-values by
multi-segments-PCO and uni-merged-segments</a>.</p>
<ul>
<li><p>Group size of seg/merged groups off diagonal line</p></li>
<li><p>#Vars in cis of seg/merged groups off diagonal line</p></li>
</ul>
</div>
</div>
</div>
<div id="apr-19" class="section level1">
<h1>Apr 19</h1>
<div id="quick-summary" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Check peak width for marks</p></li>
<li><p>Compare p-values by multi-segments-PCO and uni-merged-segments in
terms of top p, small p, and all p.</p></li>
<li><p>Not finish comparison in terms of
cmerged_seg/csegment/cpeak-group. With questions.</p></li>
</ul>
</div>
<div id="check-peak-width-for-marks" class="section level2">
<h2>1. Check peak width for marks</h2>
<div id="why" class="section level3">
<h3>Why?</h3>
<p>To see what segment size should be used to cover a whole peak for
different marks.</p>
</div>
<div id="results-2" class="section level3">
<h3>Results</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_19_plt_peak_width_H3K27AC.pdf" title="Figure: Peak width of marks." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_plt_peak_width_H3K27AC">
Past versions of 04_19_plt_peak_width_H3K27AC.pdf
</button>
</p>
<div id="fig-04_19_plt_peak_width_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_plt_peak_width_H3K27AC.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_plt_peak_width_H3K4ME1.pdf" title="Figure: Peak width of marks." width="40%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_plt_peak_width_H3K4ME1">
Past versions of 04_19_plt_peak_width_H3K4ME1.pdf
</button>
</p>
<div id="fig-04_19_plt_peak_width_H3K4ME1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_plt_peak_width_H3K4ME1.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_plt_peak_width_H3K4ME3.pdf" title="Figure: Peak width of marks." width="40%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_plt_peak_width_H3K4ME3">
Past versions of 04_19_plt_peak_width_H3K4ME3.pdf
</button>
</p>
<div id="fig-04_19_plt_peak_width_H3K4ME3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_plt_peak_width_H3K4ME3.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/04_19_plt_peak_width_H3K36ME3.pdf" title="Figure: Peak width of marks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Peak width of marks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_plt_peak_width_H3K36ME3">
Past versions of 04_19_plt_peak_width_H3K36ME3.pdf
</button>
</p>
<div id="fig-04_19_plt_peak_width_H3K36ME3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_plt_peak_width_H3K36ME3.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations-2" class="section level3">
<h3>Observations</h3>
<ul>
<li><p>For mark H3K27AC, the average peak width is ~1kb.</p></li>
<li><p>Mark H3K36ME3 has very wide peaks.</p></li>
</ul>
</div>
</div>
<div id="compare-p-values-by-multi-segments-pco-and-uni-merged-segments"
class="section level2">
<h2>2. Compare p-values by multi-segments-PCO and
uni-merged-segments</h2>
<div id="top-p-comparsion-multi-merged" class="section level3">
<h3>Look top p at all scenarios - seg size and cpm cutoff factor</h3>
<p>Last week, I compared top p for one scenario. See below for all
scenarios across seg sizes and cpm cutoff factors.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_19_1_plt_topp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_1_plt_topp_multi_sum_chr20_avgcpmfct2_seg1.5kb">
Past versions of
04_19_1_plt_topp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf
</button>
</p>
<div id="fig-04_19_1_plt_topp_multi_sum_chr20_avgcpmfct2_seg1.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_1_plt_topp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_2_plt_topp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_2_plt_topp_multi_sum_chr20_avgcpmfct2_seg2kb">
Past versions of 04_19_2_plt_topp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf
</button>
</p>
<div id="fig-04_19_2_plt_topp_multi_sum_chr20_avgcpmfct2_seg2kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_2_plt_topp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_3_plt_topp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_3_plt_topp_multi_sum_chr20_avgcpmfct3_seg0.5kb">
Past versions of
04_19_3_plt_topp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf
</button>
</p>
<div id="fig-04_19_3_plt_topp_multi_sum_chr20_avgcpmfct3_seg0.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_3_plt_topp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_4_plt_topp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_4_plt_topp_multi_sum_chr20_avgcpmfct3_seg1kb">
Past versions of 04_19_4_plt_topp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf
</button>
</p>
<div id="fig-04_19_4_plt_topp_multi_sum_chr20_avgcpmfct3_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_4_plt_topp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_5_plt_topp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_5_plt_topp_multi_sum_chr20_avgcpmfct3_seg1.5kb">
Past versions of
04_19_5_plt_topp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf
</button>
</p>
<div id="fig-04_19_5_plt_topp_multi_sum_chr20_avgcpmfct3_seg1.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_5_plt_topp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/04_19_6_plt_topp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p of groups. Top p's are bonferroni corrected by multiplying the number of cis SNPs of a group." width="30%" type="application/pdf" />
<p class="caption">
Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of top p
of groups. Top p’s are bonferroni corrected by multiplying the number of
cis SNPs of a group.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_6_plt_topp_multi_sum_chr20_avgcpmfct3_seg2kb">
Past versions of 04_19_6_plt_topp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf
</button>
</p>
<div id="fig-04_19_6_plt_topp_multi_sum_chr20_avgcpmfct3_seg2kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_6_plt_topp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We also wanted to look at -</p>
<p><a href="asset/2023/04_26_plt_topp_nominal_multi_sum_chr20.pdf">Use
nominal top p, w.o. bonferroni correction by multiplifying number of cis
SNPs of groups. We wanted to look at this because we think cis regions
should have same #independent SNPs. Not much difference with top p with
bonferroni correction.</a></p>
<p>Look into those seg/merged groups off diagonal line -</p>
<ul>
<li><p><a
href="asset/2023/04_26_plt_topp_bigdiff_multi_sum_chr20.pdf">Only show
groups with large top p difference,
i.e. log(topp_diff)&gt;2.</a></p></li>
<li><p><a
href="asset/2023/04_26_plt_topp_bigdiff_group_size_multi_sum_chr20.pdf">Group
size of seg/merged groups off diagonal line</a></p></li>
<li><p><a
href="asset/2023/04_26_plt_topp_bigdiff_cissnp_multi_sum_chr20.pdf">#Vars
in cis of seg/merged groups off diagonal line</a></p></li>
</ul>
<p><strong>Observations -</strong></p>
<ul>
<li>In terms of top p (with bonferroni correction on cis snps),
seg_multi and seg_merged don’t have much difference.</li>
</ul>
</div>
<div id="look-all-p-from-snp-group-not-only-just-top-p"
class="section level3">
<h3>Look all p from (SNP, group), not only just top p</h3>
<p>No bonferroni correction just nominal p.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_19_1_plt_allp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_1_plt_allp_multi_sum_chr20_avgcpmfct2_seg1.5kb">
Past versions of
04_19_1_plt_allp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf
</button>
</p>
<div id="fig-04_19_1_plt_allp_multi_sum_chr20_avgcpmfct2_seg1.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_1_plt_allp_multi_sum_chr20_avgcpmfct2_seg1.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_2_plt_allp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_2_plt_allp_multi_sum_chr20_avgcpmfct2_seg2kb">
Past versions of 04_19_2_plt_allp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf
</button>
</p>
<div id="fig-04_19_2_plt_allp_multi_sum_chr20_avgcpmfct2_seg2kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_2_plt_allp_multi_sum_chr20_avgcpmfct2_seg2kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_3_plt_allp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_3_plt_allp_multi_sum_chr20_avgcpmfct3_seg0.5kb">
Past versions of
04_19_3_plt_allp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf
</button>
</p>
<div id="fig-04_19_3_plt_allp_multi_sum_chr20_avgcpmfct3_seg0.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_3_plt_allp_multi_sum_chr20_avgcpmfct3_seg0.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_4_plt_allp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_4_plt_allp_multi_sum_chr20_avgcpmfct3_seg1kb">
Past versions of 04_19_4_plt_allp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf
</button>
</p>
<div id="fig-04_19_4_plt_allp_multi_sum_chr20_avgcpmfct3_seg1kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_4_plt_allp_multi_sum_chr20_avgcpmfct3_seg1kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/04_19_5_plt_allp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_5_plt_allp_multi_sum_chr20_avgcpmfct3_seg1.5kb">
Past versions of
04_19_5_plt_allp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf
</button>
</p>
<div id="fig-04_19_5_plt_allp_multi_sum_chr20_avgcpmfct3_seg1.5kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_5_plt_allp_multi_sum_chr20_avgcpmfct3_seg1.5kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/04_19_6_plt_allp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf" title="Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small p and all p of (SNP, seg group/merged group) pairs. Small p's show p &lt; 1e-5. All p's show truncated QQ plot." width="30%" type="application/pdf" />
<p class="caption">
Figure: Compare p of seg_multi_pco and seg_merged_uni in terms of small
p and all p of (SNP, seg group/merged group) pairs. Small p’s show p
&lt; 1e-5. All p’s show truncated QQ plot.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_19_6_plt_allp_multi_sum_chr20_avgcpmfct3_seg2kb">
Past versions of 04_19_6_plt_allp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf
</button>
</p>
<div id="fig-04_19_6_plt_allp_multi_sum_chr20_avgcpmfct3_seg2kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc575927109853d7da4fa40ba70811e7a66404dc/docs/asset/2023/04_19_6_plt_allp_multi_sum_chr20_avgcpmfct3_seg2kb.pdf" target="_blank">fc57592</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li>In terms of all p (without multiple testing correction), seg_multi
is better than seg_merged only in case avgcpmfct3_seg2kb.</li>
</ul>
</div>
<div id="compare-group-in-terms-of-csegment_group"
class="section level3">
<h3>Compare group in terms of csegment_group?</h3>
<blockquote>
<p>Use the way how people find cpeak to find csegment_group. With
multiple testing correction.</p>
</blockquote>
<p><strong>Question -</strong></p>
<ul>
<li><p>Isn’t this the same as <a
href="#top-p-comparsion-multi-merged">what I did</a> where I looked at
top p (with bonferroni correction on cis snps)?</p></li>
<li><p>Since last time we said what people want is csegment/cpeak_group,
and given seg_multi and seg_merged are similar in terms of top p, should
this be concerning?</p></li>
</ul>
</div>
</div>
<div
id="compare-seg-based-and-peak-based-in-terms-of-csegmentcpeak_group"
class="section level2">
<h2>3. Compare seg-based and peak-based in terms of
csegment/cpeak_group</h2>
<p><strong>Question -</strong></p>
<p>Same reason as above.</p>
</div>
</div>
<div id="apr-05-apr-12" class="section level1">
<h1>Apr 05 &amp; Apr 12</h1>
<div id="quick-summary-1" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Updated figures and observations from last time, adding factor 1
and peak-based.</p></li>
<li><p>Re-do segment-based by changing peak filtering from using average
log2 CPM as cutoff to using average CPM.</p></li>
<li><p>Tried using different segment sizes.</p></li>
<li><p>Compared p-values of segment-multi PCO with segment-uni (merged
reads from multiple segments).</p></li>
</ul>
</div>
<div id="re-do-peak-filtering-using-average-cpm-not-log2-based"
class="section level2">
<h2>1. Re-do peak filtering using average CPM, not log2 based</h2>
<p><strong>Two changes on segment filtering based on cpm -</strong></p>
<ol style="list-style-type: decimal">
<li><p>Cutoff to be ave cpm, instead of ave log2 cpm as used in last
time.</p></li>
<li><p>Use median across segments instead of mean, as large variation
expected across segments. [Use mean of one seg across individuals, as
similar count expected for individuals.]</p></li>
</ol>
<ul>
<li><p>Should be adding the factor instead of multiplying. More
stringent cutoff and less segments.</p></li>
<li><p>Under CPM filtering cutoff, check segment numbers</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_plt_seg_avgcpmfct.pdf" title="Figure: How many segments are retained after removing segments based on average log2 CPM?" width="50%" type="application/pdf" />
<p class="caption">
Figure: How many segments are retained after removing segments based on
average log2 CPM?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_plt_seg_avgcpmfct">
Past versions of 04_05_plt_seg_avgcpmfct.pdf
</button>
</p>
<div id="fig-04_05_plt_seg_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_plt_seg_avgcpmfct.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Under CPM filtering cutoff, check group size</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_plt_seg_group_size_avgcpmfct.pdf" title="Figure: Segment group size across various factors of average log2 CPM." width="50%" type="application/pdf" />
<p class="caption">
Figure: Segment group size across various factors of average log2 CPM.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_plt_seg_group_size_avgcpmfct">
Past versions of 04_05_plt_seg_group_size_avgcpmfct.pdf
</button>
</p>
<div id="fig-04_05_plt_seg_group_size_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_plt_seg_group_size_avgcpmfct.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Run all thing - Check p</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_compare_p_avgcpmfct.pdf" title="Figure: P value distribution under various factors of log2 average CPM. Only show small P &lt; 1e-5. Lower panel shows the total number of tests under each scenario." width="50%" type="application/pdf" />
<p class="caption">
Figure: P value distribution under various factors of log2 average CPM.
Only show small P &lt; 1e-5. Lower panel shows the total number of tests
under each scenario.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_compare_p_avgcpmfct">
Past versions of 04_05_compare_p_avgcpmfct.pdf
</button>
</p>
<div id="fig-04_05_compare_p_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_compare_p_avgcpmfct.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_compare_p_avgcpmfct_qqplot.pdf" title="Figure: QQ-plot of P values under various factors of log2 average CPM. I sorted all P values but here I only show P &lt; 1e-7" width="50%" type="application/pdf" />
<p class="caption">
Figure: QQ-plot of P values under various factors of log2 average CPM. I
sorted all P values but here I only show P &lt; 1e-7
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_compare_p_avgcpmfct_qqplot">
Past versions of 04_05_compare_p_avgcpmfct_qqplot.pdf
</button>
</p>
<div id="fig-04_05_compare_p_avgcpmfct_qqplot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_compare_p_avgcpmfct_qqplot.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
<ul>
<li><p>I tried using mean of ave cpm as cutoff, too many segs were
removed.</p></li>
<li><p>More segments left.</p></li>
<li><p>Use 2 or 3 to be comparable or better than peak based.</p></li>
<li><p>Computation time on chr20: using 2 takes ~2h, 3 takes
~1h.</p></li>
<li><p>P values under these factors not differ much, not as results from
last time, because the number of tests do not differ much …</p></li>
<li><p><strong>Again, peak-based is the best in terms of QQ-plot, due to
much less number of tests. Increase segment size?</strong></p></li>
</ul>
<div id="csaw-paper-on-filtering-windows" class="section level3">
<h3>csaw paper on filtering windows</h3>
<ul>
<li><p>Check csaw’s user guide for the histogram of CPM with their
cutoff and background cutoff. They retain very few windows. (Page
25)</p></li>
<li><p>Cutoff depends on mark binding site. Narrow v.s. Broad.</p></li>
</ul>
<blockquote>
<p>Smaller minimum fold changes are recommended for diffuse marks where
the difference from background is less obvious.</p>
</blockquote>
</div>
</div>
<div id="use-different-segment-sizes" class="section level2">
<h2>2. Use different segment sizes</h2>
<div id="how-3" class="section level3">
<h3>How?</h3>
<p>Use 0.5kb, 1kb, 1.5kb, 2kb to define segment size.</p>
<p>Use factor 3 for all segment sizes.</p>
</div>
<div id="results-3" class="section level3">
<h3>Results</h3>
<ul>
<li>Check segment cpm distribution and numbers based on CPM
filtering</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_plt_seg_segsize.pdf" title="Figure: Use 0.5kb, 1kb, 1.5kb, 2kb to define segment size. With that, split chr20 into segments, and then apply cpm filtering using factor 3. (A) Log2(average cpm) distribution across segments under various segment sizes. (B) The number of retained segments after filtering segments based on (median of average cpm) times various factors." width="50%" type="application/pdf" />
<p class="caption">
Figure: Use 0.5kb, 1kb, 1.5kb, 2kb to define segment size. With that,
split chr20 into segments, and then apply cpm filtering using factor 3.
(A) Log2(average cpm) distribution across segments under various segment
sizes. (B) The number of retained segments after filtering segments
based on (median of average cpm) times various factors.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_plt_seg_segsize">
Past versions of 04_05_plt_seg_segsize.pdf
</button>
</p>
<div id="fig-04_05_plt_seg_segsize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_plt_seg_segsize.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Check size of segment groups</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_plt_seg_group_size_segsize.pdf" title="Figure: Segment group size across various segment sizes after filtering segments using 3*median(average CPM)." width="50%" type="application/pdf" />
<p class="caption">
Figure: Segment group size across various segment sizes after filtering
segments using 3*median(average CPM).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_plt_seg_group_size_segsize">
Past versions of 04_05_plt_seg_group_size_segsize.pdf
</button>
</p>
<div id="fig-04_05_plt_seg_group_size_segsize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_plt_seg_group_size_segsize.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Check p</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_compare_p_segsize.pdf" title="Figure: P value distribution under various segment sizes after filtering segments using 3*median(average CPM). Only show small P &lt; 1e-5. Lower panel shows the total number of tests under each scenario." width="50%" type="application/pdf" />
<p class="caption">
Figure: P value distribution under various segment sizes after filtering
segments using 3*median(average CPM). Only show small P &lt; 1e-5. Lower
panel shows the total number of tests under each scenario.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_compare_p_segsize">
Past versions of 04_05_compare_p_segsize.pdf
</button>
</p>
<div id="fig-04_05_compare_p_segsize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_compare_p_segsize.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_05_compare_p_segsize_qqplot.pdf" title="Figure: QQ-plot of P values under various segment sizes after filtering segments using 3*median(average CPM). I sorted all P values but here I only show P &lt; 1e-7" width="50%" type="application/pdf" />
<p class="caption">
Figure: QQ-plot of P values under various segment sizes after filtering
segments using 3*median(average CPM). I sorted all P values but here I
only show P &lt; 1e-7
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_05_compare_p_segsize_qqplot">
Past versions of 04_05_compare_p_segsize_qqplot.pdf
</button>
</p>
<div id="fig-04_05_compare_p_segsize_qqplot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_05_compare_p_segsize_qqplot.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="try-other-cpm-cutoff-factors" class="section level3">
<h3>Try other CPM cutoff factors</h3>
<ul>
<li><p>Why? Use various cpm cutoff for various segment sizes. Larger
segments, more reads, higher CPM, higher average CPM. If using the same
cpm cutoff factor as smaller segments, the cutoff can be much higher and
more segments would be filtered out.</p></li>
<li><p>Check size of segment groups</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_12_plt_seg_group_size_avgcpmfct_segsize.pdf" title="Figure: Segment group size across various segment sizes after filtering segments using various CPM factor cutoffes." width="50%" type="application/pdf" />
<p class="caption">
Figure: Segment group size across various segment sizes after filtering
segments using various CPM factor cutoffes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_12_plt_seg_group_size_avgcpmfct_segsize">
Past versions of 04_12_plt_seg_group_size_avgcpmfct_segsize.pdf
</button>
</p>
<div id="fig-04_12_plt_seg_group_size_avgcpmfct_segsize"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_12_plt_seg_group_size_avgcpmfct_segsize.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Check p</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_12_compare_p_avgcpmfct_segsize.pdf" title="Figure: P value distribution under various segment sizes after filtering segments using 3*median(average CPM). Only show small P &lt; 1e-5. Lower panel shows the total number of tests under each scenario." width="50%" type="application/pdf" />
<p class="caption">
Figure: P value distribution under various segment sizes after filtering
segments using 3*median(average CPM). Only show small P &lt; 1e-5. Lower
panel shows the total number of tests under each scenario.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_12_compare_p_avgcpmfct_segsize">
Past versions of 04_12_compare_p_avgcpmfct_segsize.pdf
</button>
</p>
<div id="fig-04_12_compare_p_avgcpmfct_segsize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_12_compare_p_avgcpmfct_segsize.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_12_compare_p_avgcpmfct_segsize_qqplot.pdf" title="Figure: QQ-plot of P values under various segment sizes after filtering segments using 3*median(average CPM). I sorted all P values but here I only show P &lt; 1e-7" width="50%" type="application/pdf" />
<p class="caption">
Figure: QQ-plot of P values under various segment sizes after filtering
segments using 3*median(average CPM). I sorted all P values but here I
only show P &lt; 1e-7
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_12_compare_p_avgcpmfct_segsize_qqplot">
Past versions of 04_12_compare_p_avgcpmfct_segsize_qqplot.pdf
</button>
</p>
<div id="fig-04_12_compare_p_avgcpmfct_segsize_qqplot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_12_compare_p_avgcpmfct_segsize_qqplot.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_26_compare_topp_avgcpmfct_segsize.pdf" title="Figure: Top P value distribution. Top p's are bonferroni corrected by multiplying the number of cis SNPs of groups. Only show groups with top p &lt; 1e-3." width="50%" type="application/pdf" />
<p class="caption">
Figure: Top P value distribution. Top p’s are bonferroni corrected by
multiplying the number of cis SNPs of groups. Only show groups with top
p &lt; 1e-3.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_26_compare_topp_avgcpmfct_segsize">
Past versions of 04_26_compare_topp_avgcpmfct_segsize.pdf
</button>
</p>
<div id="fig-04_26_compare_topp_avgcpmfct_segsize" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0cd9785583f2a677fd805bb9c6be991dcd63fff4/docs/asset/2023/04_26_compare_topp_avgcpmfct_segsize.pdf" target="_blank">0cd9785</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p><strong>Observations</strong></p>
<ul>
<li><p>Segment size still smaller than peak width.</p></li>
<li><p><strong>Again, peak-based is the best in terms of QQ-plot, due to
much less number of tests. Increase segment size?</strong></p></li>
</ul></li>
<li><p>Solve questions written on the paper sheet - on read counting
&amp; cpm cutoff for various segment size.</p></li>
<li><p>Check if -p -reduce read1.bam<code>has same alignment as</code>
-p -reduce -O`</p></li>
<li><p>Read counting is problematic.</p>
<ul>
<li><p>seg1kb, different assigned reads for two run</p></li>
<li><p><code>count_based/seg_chr/seg1kb_chr20.saf</code></p></li>
<li><p><code>count_based/featurecounts/3_Counts_paired_leftmost_onlyread1.txt</code>
v.s
<code>count_based/featurecounts/Counts_paired_leftmost_onlyread1_seg1kb_chr20.txt</code></p></li>
</ul></li>
<li><p>Try size options, large or small, for current narrow peaks or
future wide peaks. Check segment numbers, group size.</p></li>
</ul>
</div>
</div>
<div id="merge-reads-across-segments" class="section level2">
<h2>3. Merge reads across segments</h2>
<p>See some intuitions from <a href="#sum_reads_across_seg">reading
below</a>.</p>
<div id="how-4" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Filter segments first.</p></li>
<li><p>The total read counts across segs in a window.</p></li>
<li><p>Same QC and normalizations.</p></li>
<li><p>cis - 500kb. Probably more test than PCO.</p></li>
</ul>
<p>2266 groups.</p>
</div>
<div id="comparison" class="section level3">
<h3>Comparison</h3>
<ul>
<li><p>Top p</p></li>
<li><p>#cis SNPs tested for each group</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/04_12_compare_multi_sum.pdf" title="Figure: (A) Comparion of top p of each group for segment_multi and segment_sum. Top p was bonferroni corrected by the number of cis SNPs tested for the group. (B) Distribution of number of cis SNPs tested for each group." width="50%" type="application/pdf" />
<p class="caption">
Figure: (A) Comparion of top p of each group for segment_multi and
segment_sum. Top p was bonferroni corrected by the number of cis SNPs
tested for the group. (B) Distribution of number of cis SNPs tested for
each group.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_12_compare_multi_sum">
Past versions of 04_12_compare_multi_sum.pdf
</button>
</p>
<div id="fig-04_12_compare_multi_sum" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/61c7945a0b9796a3a6ba96911f4c8bb5c4d794ac/docs/asset/2023/04_12_compare_multi_sum.pdf" target="_blank">61c7945</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-12
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="ideas-from-reading" class="section level2">
<h2>4. Ideas from reading</h2>
<div id="sum_reads_across_seg" class="section level3">
<h3>Sum of reads for a peak v.s. slice the peak by windows/segments
<span class="citation">(Lun and Smyth 2016)</span></h3>
<p>If a region/peak is broad/diffuse, QTL effects on all parts of the
peak are weak but consistent. Then summing of reads provides greater
detection power for such regions, as large peaks can collect more read
counts than small windows. And since each subinterval of the region has
the same/positively correlated QTL effects, no benefit is gained from
considering each subinterval separately with small windows.</p>
</div>
<div id="when-is-window-based-strategy-powerful---some-intuitions"
class="section level3">
<h3>When is window-based strategy powerful - some intuitions</h3>
<ul>
<li>When detecting complex DB events, use windowing to increase spatial
resolution missed by peak calling -</li>
</ul>
<blockquote>
<p>csaw was more powerful than other methods at detecting complex DB
events, especially those involving changes in the width of a binding
event.</p>
</blockquote>
<blockquote>
<p>The relatively poor performance of peak-based methods can be
attributed to the fact that each binding site is defined as a single
peak by MACS or HOMER.</p>
</blockquote>
<blockquote>
<p>csaw is able to tile small windows across the binding site. Each
window corresponds a different subinterval of the binding site, such
that the magnitude of any changes in part of the site can be faithfully
captured by a window.</p>
</blockquote>
<p>In my case, also add a case of where intensity difference is low.</p>
<p><strong>This is actually different than our case, where we wanted the
window size to be approximately peak size, so that a window group would
contain multiple peaks to be jointly tested. However, the idea of csaw
is to split a general complex peak to sub-intervals, each of which are
captured by windows for increased resolution.</strong></p>
<ul>
<li>When the peak is narrow and easily identified, window-based strategy
should have comparable power to peak calling.</li>
</ul>
<blockquote>
<p>sharp peaks were easily identified by the peak calling software. In
this case, windowing did not provide any useful gain in spatial
resolution. Nevertheless, csaw still returned the same performance curve
as the best peak-based methods.</p>
</blockquote>
<p><strong>In our case, our idea should be more powerful than peak
calling even when the peaks are well-defined, since we test multiple
peaks jointly.</strong></p>
</div>
</div>
<div id="choose-ways-to-cluster-windowssegments-into-groups"
class="section level2">
<h2>Choose ways to cluster windows/segments into groups</h2>
</div>
</div>
<div id="mar-29" class="section level1">
<h1>Mar 29</h1>
<div id="quick-summary-2" class="section level2">
<h2>Quick summary</h2>
<ul>
<li>Re-do segment-based by adding cpm filtering on segments.</li>
</ul>
</div>
<div id="re-thinking-of-segment-based-idea" class="section level2">
<h2>1. Re-thinking of segment-based idea</h2>
<div id="segment-based-for-narrow-peaks" class="section level3">
<h3>Segment-based for narrow peaks</h3>
<ul>
<li><p>Goal: to skip peak calling</p></li>
<li><p>Illustration figure - ideal case</p>
<p>We want segment size to be peak-width to include multiple peaks
together.</p></li>
</ul>
</div>
<div id="segment-based-for-braod-peaks" class="section level3">
<h3>Segment-based for braod peaks</h3>
<ul>
<li><p>Goal: to compensate peak calling</p></li>
<li><p>Illustration figure - ideal case</p>
<p>We want segment size to be smaller than peak-width to split a broad
peak to multiple segments.</p></li>
</ul>
</div>
</div>
<div id="add-cpm-filtering-on-segments-and-re-do-segment-based"
class="section level2">
<h2>2. Add cpm filtering on segments and re-do segment-based</h2>
<div id="how-5" class="section level3">
<h3>How?</h3>
<p>Average log2 CPM for each segment across samples. Mean of average
log2 CPM. Add a factor.</p>
</div>
<div id="under-cpm-filtering-cutoff-check-segment-numbers"
class="section level3">
<h3>Under CPM filtering cutoff, check segment numbers</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_29_plt_seg_avgcpmfct.pdf" title="Figure: How many segments are retained after removing segments based on average log2 CPM?" width="50%" type="application/pdf" />
<p class="caption">
Figure: How many segments are retained after removing segments based on
average log2 CPM?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_29_plt_seg_avgcpmfct">
Past versions of 03_29_plt_seg_avgcpmfct.pdf
</button>
</p>
<div id="fig-03_29_plt_seg_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/dfa6c34e5fdd493c43adce286558f43ad9da8de2/docs/asset/2023/03_29_plt_seg_avgcpmfct.pdf" target="_blank">dfa6c34</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="under-cpm-filtering-cutoff-check-group-size"
class="section level3">
<h3>Under CPM filtering cutoff, check group size</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_29_plt_seg_group_size_avgcpmfct.pdf" title="Figure: Segment group size across various factors of average log2 CPM." width="50%" type="application/pdf" />
<p class="caption">
Figure: Segment group size across various factors of average log2 CPM.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_29_plt_seg_group_size_avgcpmfct">
Past versions of 03_29_plt_seg_group_size_avgcpmfct.pdf
</button>
</p>
<div id="fig-03_29_plt_seg_group_size_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/dfa6c34e5fdd493c43adce286558f43ad9da8de2/docs/asset/2023/03_29_plt_seg_group_size_avgcpmfct.pdf" target="_blank">dfa6c34</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="run-all-thing" class="section level3">
<h3>Run all thing</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_29_compare_p_avgcpmfct.pdf" title="Figure: P value distribution under various factors of average log2 CPM. Only show small P &lt; 1e-5. Lower panel shows the total number of tests under each scenario." width="50%" type="application/pdf" />
<p class="caption">
Figure: P value distribution under various factors of average log2 CPM.
Only show small P &lt; 1e-5. Lower panel shows the total number of tests
under each scenario.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_29_compare_p_avgcpmfct">
Past versions of 03_29_compare_p_avgcpmfct.pdf
</button>
</p>
<div id="fig-03_29_compare_p_avgcpmfct" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a8ba9a35e0a05265e2ad801f12bada97b91ba752/docs/asset/2023/03_29_compare_p_avgcpmfct.pdf" target="_blank">a8ba9a3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/dfa6c34e5fdd493c43adce286558f43ad9da8de2/docs/asset/2023/03_29_compare_p_avgcpmfct.pdf" target="_blank">dfa6c34</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>See histogram for peak-based and factor 2 and 3 <a
href="asset/2023/03_29_compare_p_avgcpmfct_sub.pdf">here</a>.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_29_compare_p_avgcpmfct_qqplot.pdf" title="Figure: QQ-plot of P values under various factors of average log2 CPM. I sorted all P values but here I only show P &lt; 1e-7" width="50%" type="application/pdf" />
<p class="caption">
Figure: QQ-plot of P values under various factors of average log2 CPM. I
sorted all P values but here I only show P &lt; 1e-7
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_29_compare_p_avgcpmfct_qqplot">
Past versions of 03_29_compare_p_avgcpmfct_qqplot.pdf
</button>
</p>
<div id="fig-03_29_compare_p_avgcpmfct_qqplot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a8ba9a35e0a05265e2ad801f12bada97b91ba752/docs/asset/2023/03_29_compare_p_avgcpmfct_qqplot.pdf" target="_blank">a8ba9a3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-04-04
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/dfa6c34e5fdd493c43adce286558f43ad9da8de2/docs/asset/2023/03_29_compare_p_avgcpmfct_qqplot.pdf" target="_blank">dfa6c34</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-31
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
<ul>
<li><p>Use factor 3, in terms of,</p>
<ul>
<li><p>Absolute number of small p values. Compare to factor 4 and
2.</p></li>
<li><p>Density of small p values. Compare to factor 4 and 2.</p></li>
<li><p>Computation speed.</p></li>
</ul></li>
</ul>
<p><strong>Updated Observation</strong></p>
<ul>
<li><p>Add segment-based using factor 1 and peak-based.</p></li>
<li><p>Peak-based is comparable to segment-based using factor 2 and 3,
in terms of absolute number of small p values.</p></li>
<li><p><strong>But Peak-based is the best in terms of qq plot of all p
values.</strong></p>
<ul>
<li><p>This is due to though peak-based is comparable to segment-based
using factor 2 and 3, it has much fewer tests.</p></li>
<li><p>Increase segment size?</p></li>
</ul></li>
</ul>
</div>
<div id="issues" class="section level3">
<h3>Issues</h3>
<p>I can’t do the following to-do as mentioned last time,</p>
<ul>
<li><p>cdf and histogram of all pvalues</p>
<p>Looks exactly same and highly overlapped under various factors of
average log2 CPM. Because most p values are large, so those small p
values are only a few and their differences don’t matter much when
plotting all p values.</p></li>
<li><p>scatter plot</p>
<p>Impossible to plot all p values on a scatter plot. Instead, I plotted
the QQ plot of all pvalues. Not sure if it is equal.</p></li>
</ul>
</div>
</div>
<div id="ideas-from-reading-1" class="section level2">
<h2>3. Ideas from reading</h2>
<div id="kb-is-not-wide-as-segment-size-for-broad-peaks"
class="section level3">
<h3>1kb is not wide as segment size for broad peaks</h3>
<ul>
<li><p>From literature, they typically use 1kb as bin size for broad
peaks.</p></li>
<li><p>We don’t need higher resolution for wide domain/peak so we
actually want the segment to be wide.</p></li>
<li><p>If we apply this idea to narrow peaks - cut the peak to segments
instead of joint multiple peaks, we need narrower segment.</p></li>
</ul>
</div>
<div id="why-avoid-peak-calling-lun2016" class="section level3">
<h3>Why avoid peak calling? <span class="citation">(Lun and Smyth
2016)</span></h3>
<ul>
<li>It can be problematic even for narrow peaks.</li>
</ul>
<blockquote>
<p>To avoid the biases and loss of resolution associated with peak
calling, the software packages USeq (<a href="javascript:;">13</a>),
diffReps (<a href="javascript:;">14</a>) and PePr (<a
href="javascript:;">15</a>) have implemented windowing strategies.</p>
</blockquote>
<ul>
<li><p>Peak calling followed by other analysis, e.g. DB, can loss error
control. So use a sliding window (de novo detection) instead.</p>
<ul>
<li>Window-based strategy has well controlled error rates - unlike peaks
(peak calling and DB on the same dataset), windows are defined
independently from the data.</li>
</ul></li>
<li><p>In my case, add - peak calling for broad peaks</p></li>
<li><p>In my case, add - no need to learn peak calling</p></li>
</ul>
</div>
</div>
</div>
<div id="mar-08-march-15-march-22" class="section level1">
<h1>Mar 08 &amp; March 15 &amp; March 22</h1>
<div id="quick-summary-3" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Figured out Ben &amp; Carlos’s steps on their histone peak QTLs
we used previously.</p></li>
<li><p>Finish count-based / segment-based version of PCO.</p></li>
</ul>
</div>
<div id="what-they-did-to-process-data-for-peak-qtls"
class="section level2">
<h2>1. What they did to process data for peak QTLs</h2>
<p>To work on count based QTL mapping, I need to first figure out the
exact steps Ben &amp; Carlos’s did to process the histone data and peak
QTL mapping, in order to make consistent window-based and count-based
QTL mapping. Below is a mindmap of steps, QC, normalization etc. of Ben
&amp; Carlos did to generate peak QTLs.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_08_Histone_mark_reads.pdf" title="Figure: What Ben &amp; Carlos did to process data." width="100%" type="application/pdf" />
<p class="caption">
Figure: What Ben &amp; Carlos did to process data.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_Histone_mark_reads">
Past versions of 03_08_Histone_mark_reads.pdf
</button>
</p>
<div id="fig-03_08_Histone_mark_reads" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_Histone_mark_reads.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="count-based---processing-data" class="section level2">
<h2>2. Count based - Processing data</h2>
<ul>
<li><p>Why?</p></li>
<li><p>How? A few things to be changed</p>
<p>Make pre-processing data consistent with window-based, except -</p>
<ul>
<li><p>Redefine segments on genome as .saf rather than peak calling
regions.</p></li>
<li><p>Adaptations to QC</p>
<ul>
<li><p>Remove segments without any reads</p></li>
<li><p>Remove segments without reads in 1-?% individuals (I tried 100%,
50%)</p></li>
</ul></li>
</ul></li>
</ul>
<div id="split-chrs-by-segments" class="section level3">
<h3>Split chr’s by segments</h3>
<ul>
<li><p>How?</p>
<p>Split the chromosome from 0 to chromosome length by a segment size -
1kb.</p></li>
<li><p>Number of segments across chr</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_08_num_features.pdf" title="Figure: Number of split segments compared to the called peaks." width="50%" type="application/pdf" />
<p class="caption">
Figure: Number of split segments compared to the called peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_num_features">
Past versions of 03_08_num_features.pdf
</button>
</p>
<div id="fig-03_08_num_features" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_num_features.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="count-reads-within-each-seg" class="section level3">
<h3>Count reads within each seg</h3>
<ul>
<li><p>4 histone marks (by ChIP-seq or CutAndTag) are all paired-end
reads.</p></li>
<li><p>Count by paired-end</p>
<p><del>No <code>-p</code>. Reads instead of paired-end fragments will
be counted.</del></p>
<p>Use <code>-p</code>. <a
href="https://support.bioconductor.org/p/67534/">Count fragment
instead</a>.</p></li>
<li><p>Ways to count a read overlapped with multiple features</p>
<p>[At first, I didn’t think through this until I observed many
unassigned reads due to Unassigned_Ambiguity.]</p>
<ul>
<li><p><del>Use <code>-p -O</code> to assign the read (1) to all
overlapped features.</del> [FEATURECOUNTS PAPER SUGGESTION ON USING
_O]</p></li>
<li><p><del>Use <code>-p --largestOverlap</code> to assign the read (1)
to the feature that has the largest number of overlapping
bases.</del></p></li>
<li><p><del>Use <code>-p --fraction -O</code> to assign a fractional
count (1/y, where y is the total number of features overlapping with the
read) to all overlapped features.</del></p></li>
<li><p>Use <code>-p --read2pos 5</code>. Think a fragment as a whole,
and take the left-most base.</p></li>
<li><p><del>Use <code>--read2pos 5</code> to reduce the read to an end
base (as said by Yang, left-most-read-base to count in bam file, take
just one base, to avoid that one read in multiple regions) and count the
reduced single base.</del></p></li>
<li><p>Use <code>-p --read2pos 5 single_paired_read1_*.bam</code>. Use
paired-end, but only uses read 1. Reduce the read to
left-most-base.</p></li>
</ul></li>
</ul>
<table>
<colgroup>
<col width="34%" />
<col width="65%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">flag</th>
<th align="left">interpretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">-p</td>
<td align="left">paired-end, ignore overlap count</td>
</tr>
<tr class="even">
<td align="left">-p -O</td>
<td align="left">paired-end, assign overlap count to all regions</td>
</tr>
<tr class="odd">
<td align="left">-p –largestOverlap</td>
<td align="left">paired-end, assign overlap count to region with largest
overlap</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5</td>
<td align="left">paired-end, reduce read to left-most-base</td>
</tr>
<tr class="odd">
<td align="left">–read2pos 5</td>
<td align="left">single-end, reduce read to left-most-base</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5 single_paired_read1_*.bam</td>
<td align="left">paired-end, reduce read to left-most-base, but use only
the first read</td>
</tr>
<tr class="odd">
<td align="left">-p –read2pos 5 -O</td>
<td align="left">paired-end, reduce read to left-most-base, assign
overlap count to all regions</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5 –largestOverlap</td>
<td align="left">paired-end, reduce read to left-most-base, assign to
largest overlap</td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_22_plt_count_mode.pdf" title="Figure: (un)Assigned reads under various counting modes." width="50%" type="application/pdf" />
<p class="caption">
Figure: (un)Assigned reads under various counting modes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_count_mode">
Past versions of 03_22_plt_count_mode.pdf
</button>
</p>
<div id="fig-03_22_plt_count_mode" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_count_mode.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
<ul>
<li><p>Unassigned reads under
<code>-p --read2pos 5 single_paired_read1_*.bam</code>.</p>
<ul>
<li><p>These unassigned reads due to ambiguity are only for one
individual.</p></li>
<li><p>Weirdly, if I do read counting for only this individual, there is
no unassigned reads.</p></li>
<li><p>If I count read all individuals together and with <code>-R</code>
to output the detailed (un)assignment of each read, there is no
unassigned reads again. So I couldn’t check exactly which reads were
unassigned for this individual.</p></li>
<li><p>I used the above read counting results.</p></li>
</ul></li>
<li><p>Why both single-end and paired-end reads occur in one bam
file?</p>
<p>Paired-end sequencing. But those single-end reads are not aligned to
ref genome.</p></li>
</ul>
<p><strong>A few numbers - raw counts</strong></p>
<ul>
<li><p>For current exploratory phase, I only run chromosome 20, which
was split to 64,444 1kb-segments.</p>
<ul>
<li><p>62,910 (~97.62%) segments with at least one read in one
individual.</p></li>
<li><p>%individuals without any reads</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_08_seg_count.pdf" title="Figure: Raw counts across segments." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_seg_count">
Past versions of 03_08_seg_count.pdf
</button>
</p>
<div id="fig-03_08_seg_count" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_seg_count.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/03_08_seg_count_zoom.pdf" title="Figure: Raw counts across segments." width="30%" type="application/pdf" />
<p class="caption">
Figure: Raw counts across segments.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_seg_count_zoom">
Past versions of 03_08_seg_count_zoom.pdf
</button>
</p>
<div id="fig-03_08_seg_count_zoom" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_seg_count_zoom.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_08_perct_ind.pdf" title="Figure: % Individuals with nonzero raw counts." width="30%" type="application/pdf" />
<p class="caption">
Figure: % Individuals with nonzero raw counts.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_perct_ind">
Past versions of 03_08_perct_ind.pdf
</button>
</p>
<div id="fig-03_08_perct_ind" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_perct_ind.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>I feature count 2337 peaks on chr20, and compared the count
summary with segment-version on chr20. A few potential questions needed
further understanding -</p>
<ul>
<li><p>In featureCounts, unassigned reads due to Unassigned_Duplicate,
Unassigned_NoFeatures, many Unassigned_Ambiguity.</p></li>
<li><p>There are many Unassigned_Ambiguity, because reads can fall
between adjacent segments. Solution: use <code>--read2pos</code> to
count only the left-most-base.</p></li>
</ul></li>
</ul>
</div>
<div id="qc-normalization" class="section level3">
<h3>QC &amp; Normalization</h3>
<ul>
<li><p>Remove segments without any reads for any individuals.</p></li>
<li><p>Filter segments by proportion of samples with non-zero reads -
100%, 50%.</p></li>
<li><p>Regular standardization &amp; qqnorm way v.s. leafcutter
way.</p></li>
</ul>
</div>
<div id="covariates-1" class="section level3">
<h3>Covariates</h3>
<p>Temporary workaround - Use PC’s of peaks.</p>
</div>
<div id="qtltools" class="section level3">
<h3>QTLTools</h3>
<p>Chr20 for now. All cis-SNPs around 500kb of each segment.</p>
</div>
</div>
<div id="count-based---perform-pco" class="section level2">
<h2>3. Count based - Perform PCO</h2>
<div id="define-segment-groups" class="section level3">
<h3>Define segment groups</h3>
<ul>
<li><p>Window size - 10kb</p></li>
<li><p>Segment size - 1kb</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_22_plt_seg_group_size.pdf" title="Figure: Segment group size under 10kb window size across prop_nonzero (1, 0.5)." width="50%" type="application/pdf" />
<p class="caption">
Figure: Segment group size under 10kb window size across prop_nonzero
(1, 0.5).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_seg_group_size">
Past versions of 03_22_plt_seg_group_size.pdf
</button>
</p>
<div id="fig-03_22_plt_seg_group_size" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_seg_group_size.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations</strong> -</p>
<ul>
<li><p>#groups used for PCO (remove those groups with 1 seg)</p></li>
<li><p>group size v.s. sample size (72)</p>
<p>Group size controlled under 10, for seg_size = 1kb,
window_size=10kb</p></li>
<li><p>Is 1kb too wide for segment size?</p></li>
</ul>
</div>
<div id="perform-pco" class="section level3">
<h3>Perform PCO</h3>
<ul>
<li><p>P for across prop_nonzero (1, 0.5) &amp; compare to peak
group.</p></li>
<li><p>I expect seg_pco to be similar to peak_pco, as these results are
from mark H3K27ac with narrow peaks.</p>
<ul>
<li>I expect seg_pco to be powerful for marks with broad peaks.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/03_22_plt_smallp.pdf" title="Figure: Comparison of small p of segment group size under 10kb window size across prop_nonzero (1, 0.5)." width="50%" type="application/pdf" />
<p class="caption">
Figure: Comparison of small p of segment group size under 10kb window
size across prop_nonzero (1, 0.5).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_smallp">
Past versions of 03_22_plt_smallp.pdf
</button>
</p>
<div id="fig-03_22_plt_smallp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_smallp.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="feb-24-feb-29" class="section level1">
<h1>Feb 24 &amp; Feb 29</h1>
<div id="quick-summary-4" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Locus zoom of a region to visualize PCO and MinP
associations</p></li>
<li><p>Genome track by genotype</p></li>
</ul>
</div>
<div id="compare-ps-of-specific-association-pairs-by-minp-and-pco"
class="section level2">
<h2>1. Compare p’s of specific association pairs by MinP and PCO</h2>
<p>Change the way of visualization -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare MinP and PCO." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare MinP and PCO.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="locus-zoom-of-a-peak-group-cis-region" class="section level2">
<h2>2. Locus zoom of (a peak group, cis region)</h2>
<div id="why-1" class="section level3">
<h3>Why?</h3>
<p>Significant PCO associations, weak ACAT associations.</p>
<p>For follow up analysis, e.g. coloc with traits.</p>
<p>But since we are not comparing to ACAT associations, is this analysis
still necessary? (Does it really make sense to compare with MinP?)</p>
<p>For coloc purpose, improve detection of mediate effects. How does it
help with coloc power? or anything else?</p>
</div>
<div id="how-6" class="section level3">
<h3>How?</h3>
<p>For a peak group, cis region. Two methods: PCO &amp; MinP.</p>
<ul>
<li><p>Issues</p>
<ul>
<li><p><del><code>locuscomparer</code> not available to install on
midway3.</del></p></li>
<li><p><del>Convert SNP position to rsid.</del></p></li>
<li><p><strong>~20% SNPs no rsid?</strong></p></li>
</ul></li>
</ul>
</div>
<div id="example-regions" class="section level3">
<h3>Example regions</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf" title="Figure: Locus compare MinP and PCO." width="30%" type="application/pdf" />
<p class="caption">
Figure: Locus compare MinP and PCO.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_">
Past versions of 02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf
</button>
</p>
<div id="fig-02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>To select regions we want?</li>
</ul>
</div>
</div>
<div id="genometracks-by-genotype" class="section level2">
<h2>3. Genometracks by genotype</h2>
<div id="issues-i-ran-into" class="section level3">
<h3>Issues I ran into</h3>
<ul>
<li><p>Use the new main function</p></li>
<li><p>Use the wrong flag - unknown snp led to wrong genotype
group</p></li>
<li><p>Use the correct template</p>
<p><code>GeneralPurposeColoredByGenotype.ini</code></p>
<p><code>GeneralPurposeColoredByGenotypeWithPerIndCoverage.ini</code></p></li>
<li><p>Correct template</p>
<p>Change to the correct template.</p>
<p>Minor - change the y max value to
<code>PerGroupMaxPerInd</code></p></li>
</ul>
</div>
<div id="example-regions-1" class="section level3">
<h3>Example regions</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/1.pdf" title="Figure: Genome tracks." width="50%" type="application/pdf" />
<p class="caption">
Figure: Genome tracks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1">
Past versions of 1.pdf
</button>
</p>
<div id="fig-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/1.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2023/1.pdf">Format 1</a></p>
<p><a href="asset/2023/2.pdf">Format 2</a></p>
<ul>
<li>To select regions we want?</li>
</ul>
</div>
</div>
<div id="ideas-from-reading-2" class="section level2">
<h2>4. Ideas from reading</h2>
<ul>
<li><p>Define sliding windows with skip</p>
<p><a href="https://doi.org/10.1038/ng.2671"
class="uri">https://doi.org/10.1038/ng.2671</a> NA Something I learned
that can be used in my cQTL work - Define sliding windows with skip by
sliding the window based on both (1) sliding across at least one peak
(the current way), (2) skip a gap window. Adding a gap sliding distance
can decrease the degree of overlapping between two adjacent peak groups
and reduce the number of tests. Or to be more precisely, use an overlap
fraction.</p></li>
<li><p>The idea of using overlap fraction to merge our significant peak
groups.</p></li>
</ul>
</div>
</div>
<div id="feb-17" class="section level1">
<h1>Feb 17</h1>
<div id="quick-summary-5" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Evaluate PCO_ACAT</p></li>
<li><p>Apply PCO_ACAT to all chr’s and window sizes of sliding windows
with the goal to analyze data</p></li>
<li><p>Thoughts &amp; insights on previous results after reading ACAT
more carefully</p></li>
</ul>
</div>
<div id="evaluate-pco_acat" class="section level2">
<h2>1. Evaluate PCO_ACAT</h2>
<p>On non-overlapping windows as described in <a
href="#non_overlapping_window">previous progress</a>.</p>
<div id="v.s.-pco" class="section level3">
<h3>v.s. PCO</h3>
<p>with the goal to see if PCO_ACAT works.</p>
<ul>
<li>Compare P value distribution</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare PCO and PCO_ACAT. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare PCO and PCO_ACAT. Color: methods. Panel: window size.
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of
(SNP, peak group),(C) Density of (SNP, peak group) with corresponding P.
These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>A more rigorous way to compare distributions?</p></li>
<li><p>Compare small p’s of window sizes</p></li>
</ul>
<p><a
href="asset/2023/02_15_plt_compare_small_p_pco_acat_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare window sizes. Color: window sizes. (A)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP,
peak group) associations with corresponding P. (B) x-axis: -log10(small
P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations
with corresponding P. (C) Numbers of tests across window sizes. These
p’s are from all chr20-22.</a></p>
<ul>
<li>Compare specific association pairs</li>
</ul>
<p><a
href="asset/2023/02_15_plt_compare_small_p_pco_acat_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li>Computation time</li>
</ul>
</div>
<div id="v.s.-acatminp" class="section level3">
<h3>v.s. ACAT/MinP</h3>
<p>with the goal to see if there is any improvement by PC-based
method.</p>
<ul>
<li>Compare P value distribution</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare PCO_ACAT and ACAT?MinP. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare PCO_ACAT and ACAT?MinP. Color: methods. Panel: window
size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number
of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding
P. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare small p’s of window sizes</li>
</ul>
<p><a
href="asset/2023/02_15_plt_small_p_pco_acat_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare window sizes. Color: window sizes. (A)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP,
peak group) associations with corresponding P. (B) x-axis: -log10(small
P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations
with corresponding P. (C) Numbers of tests across window sizes. These
p’s are from all chr20-22.</a></p>
<ul>
<li>Compare specific association pairs</li>
</ul>
<p><a
href="asset/2023/02_15_plt_small_p_pco_acat_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li>Computation time</li>
</ul>
</div>
</div>
<div
id="apply-to-all-chrs-and-window-sizes-of-non-overlapping-sliding-windows-to-analyze-data"
class="section level2">
<h2>2. Apply to all chr’s and window sizes of
(<del>non-overlapping</del> / sliding windows) to analyze data</h2>
<ul>
<li><p>Done.</p></li>
<li><p>Computation time</p>
<p>For <span class="math inline">\(22\times9\times2\)</span> …</p></li>
</ul>
</div>
<div id="thoughts-insgihts-after-read-acat-more-carefully"
class="section level2">
<h2>3. Thoughts &amp; insgihts after read ACAT more carefully</h2>
<p>Thoughts on previous results/observations/questions/concerns -</p>
<ul>
<li><p>Why did we observe similar p value distribution by ACAT and MinP
(corrected by Bonferroni)</p>
<p>As we discussed in ACAT paper JC, ACAT can be dominated by the very
small p values.</p>
<p>MinP corrected by Bonferroni and MinP with p values calculated by
computational heavy permutation are similar.</p></li>
<li><p>Why did we observe less improvements by PCO than ACAT/MinP than
expected?</p>
<ul>
<li><p>ACAT/MinP is not a good/common standard to compare with (as said
by Yang), as (1) it is already powerful, (2) it is not what we wanted to
improve on.</p></li>
<li><p>We should compare with univariate methods, which is commonly used
and less improved and less powerful.</p></li>
<li><p>Not to compare the number of SNPs, as not significant biological
meaning and hard to implement. Instead, we can use other ways, e.g. if
improved coloc signals with traits, etc.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="feb-08" class="section level1">
<h1>Feb 08</h1>
<div id="quick-summary-6" class="section level2">
<h2>Quick summary</h2>
<ul>
<li>Tested (SNP, peak group) of groups defined by non-overlapping
windows.</li>
</ul>
</div>
<div id="non_overlapping_window" class="section level2">
<h2>1. Groups defined by non-overlapping windows</h2>
<div id="why-discussion-on-slack" class="section level3">
<h3>Why? (discussion on slack)</h3>
<p>We observed many improved associations of PCO than ACAT/MinP in <a
href="#p_distribution_across_methods_and_window_sizes">previous
section</a>. One potential issue is these improved associations could
possibly come from a small set of associations, due to many repetitive
peaks &amp; SNPs by sliding window, which appears to be a large set of
associations.</p>
<p>So, we need to look into the case using non-overlapping windows, just
to make sure the PCO improvement over minP is still there after avoiding
repetitive counting of associations.</p>
<ul>
<li><p>Four points we discussed in Tuesday’s meeting</p>
<ul>
<li><p>Why coloc style and MACS2 style not work in this case?</p>
<p>Because these two generate noncontinuous genomic regions, whereas
what we need is to divide the whole genome into non-overlapping
chunks.</p></li>
<li><p>Why would simple useage of non-overlapping windows not make sense
in final version results?</p>
<p>Because choosing the start and end of the dividing window seems
arbitrary.</p></li>
<li><p>It only makes sense if we want to take a rough look at PCO v.s.
ACAT/MinP to avoid repetitive counting of signals in preliminary
results.</p></li>
<li><p>To get final results, we should go back to sliding window + MACS2
merge.</p>
<p>Calculate p-value for groups by sliding window, then merge. Maybe
then test again to obtain updated p for merged group?</p></li>
</ul></li>
</ul>
</div>
<div id="how-7" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Define non-overlapping windows</p>
<p>Dividing the genomic locations starting from 0 into non-overlapping
windows of a given window size.</p></li>
</ul>
</div>
<div id="results-4" class="section level3">
<h3>Results</h3>
<ul>
<li><p>Compare small p’s of three methods - Overall summary figures</p>
<ul>
<li>Panel B shows the advantage of PCO persists under window size 25kb
or smaller.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare three methods under non-overlapping windows. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare three methods under non-overlapping windows. Color:
methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5.
y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak
group) with corresponding P. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/asset/2023/02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Compare small p’s of three methods - Scatter plot</p>
<p><strong>This is an important figure, but it’s too large to show
explicitly.</strong></p>
<ul>
<li><p>For a pair of (SNP, group), PCO improves p-value than ACAT/MinP
under window size 25kb or smaller.</p></li>
<li><p>For pair of (SNP, group) with p around 1e-5 by ACAT/MinP
(i.e. the middle part), some pairs have improved p by PCO. (These pairs
are likely the additionally identified signals by PCO.)</p></li>
<li><p>For pair of (SNP, group) with small p by ACAT/MinP (i.e. the
right part) …</p></li>
<li><p>PCO p in large, middle, and extreme small window sizes …</p></li>
</ul></li>
</ul>
<p><a
href="asset/2023/02_08_nonoverlap_window_plt_small_p_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li><p>Compare small p’s of window sizes</p>
<ul>
<li>PCO better or ACAT/MinP worse?</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare window sizes under non-overlapping windows. Color: window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group) associations with corresponding P. (B) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations with corresponding P. (C) Numbers of tests across window sizes. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare window sizes under non-overlapping windows. Color:
window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis:
density of (SNP, peak group) associations with corresponding P. (B)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak
group) associations with corresponding P. (C) Numbers of tests across
window sizes. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/asset/2023/02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="feb-01" class="section level1">
<h1>Feb 01</h1>
<div id="quick-summary-7" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Compare small p, instead of top p, across three methods &amp;
window sizes</p></li>
<li><p>Try even more extreme window sizes</p></li>
<li><p>Example associations (by R not genome tracks)</p></li>
</ul>
</div>
<div
id="compare-methods-and-window-sizes-using-small-p-rather-than-top-p"
class="section level2">
<h2>1. Compare methods and window sizes using small p rather than top
p</h2>
<div id="why-2" class="section level3">
<h3>Why</h3>
</div>
<div id="how-8" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Available results</p>
<p>Methods - ACAT, MinP, PCO.</p>
<p>Window size - 10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p>
<p>Chr - 20, 21, 22.</p></li>
<li><p>Small p value cut off: 1e-5</p>
<p>For a pair of (SNP, group), keep its association for comparing if p
by at least one method (ACAT, MinP, PCO) &lt; 1e-5.</p></li>
</ul>
</div>
<div id="results-5" class="section level3">
<h3>Results</h3>
<ol style="list-style-type: decimal">
<li>Compare small p’s of three methods</li>
</ol>
<p>[See below section for updated results]</p>
<p><strong>Observation</strong></p>
<ul>
<li><p>Why the density line of PCO and minP under 10kb look so different
from the density lines below?</p>
<p>It’s because the smoothing algorithm to estimate the line for
“density” (Figure C) and “histogram” (Figure A) is different. So the two
lines have subtle difference.</p>
<p>To avoid this, I changed the visualization from using a line of
histogram (Figure A) to using histogram itself (Figure B).</p></li>
<li><p>Figure B shows that PCO has smaller P’s than MinP under 10kb
window.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Compare small p’s of window sizes</li>
</ol>
<p>[See below section for updated results]</p>
</div>
</div>
<div id="try-even-more-extreme-window-sizes" class="section level2">
<h2>2. Try even more extreme window sizes</h2>
<div id="why-3" class="section level3">
<h3>Why</h3>
</div>
<div id="how-9" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Available results</p>
<p>Methods - ACAT, MinP, PCO.</p>
<p>Window size - <strong>1kb, 2.5kb, 5kb</strong>, 10kb, 25kb, 50kb,
100kb, 260kb, 500kb.</p>
<p>Chr - 20, 21, 22.</p></li>
</ul>
</div>
<div id="p_distribution_across_methods_and_window_sizes"
class="section level3">
<h3>Results</h3>
<ul>
<li>Compare small p’s of three methods</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare three methods. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare three methods. Color: methods. Panel: window size.
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of
(SNP, peak group),(C) Density of (SNP, peak group) with corresponding P.
These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare small p’s of window sizes</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" title="Figure: To compare window sizes. Color: window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group) associations with corresponding P. (B) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations with corresponding P. (C) Numbers of tests across window sizes. These p's are from all chr20-22." width="50%" type="application/pdf" />
<p class="caption">
Figure: To compare window sizes. Color: window sizes. (A) x-axis:
-log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group)
associations with corresponding P. (B) x-axis: -log10(small P), small P
&lt; 1e-5. y-axis: number of (SNP, peak group) associations with
corresponding P. (C) Numbers of tests across window sizes. These p’s are
from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/docs/asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Aggregate all p values across chr’s &amp; window sizes and calculate
stats: #groups, #tests</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf" title="Figure: " width="40%" type="application/pdf" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22">
Past versions of
02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf
</button>
</p>
<div id="fig-02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Number of groups decreases as window size decreases</p></li>
<li><p>Number of (SNP, group) tests appears as U-shape</p></li>
<li><p>Use a lower p cutoff</p>
<p>P values lower than 1e-8, 1e-10 will be included.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf" title="Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10." width="50%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf" title="Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10." width="50%" type="application/pdf" />
<p class="caption">
Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
</div>
</div>
<div id="example-associations-to-visualize-p_pco-and-p_acat"
class="section level2">
<h2>3. Example associations to visualize P_pco and P_acat</h2>
<div id="goal" class="section level3">
<h3>Goal</h3>
<p>“For example, pick an example where PCO improved pvalue and pick
another example where PCO doesn’t improve pvalue?”</p>
<ul>
<li><p><del>sashimi plots</del></p></li>
<li><p>GenometracksByGenotype - pyGenomeTracks by genotype</p>
<p>(Not doable yet)</p></li>
</ul>
</div>
<div id="another-way" class="section level3">
<h3>Another way</h3>
<ul>
<li><p>Criteria to choose (SNP, group) to visualize for PCO and ACAT</p>
<ul>
<li><p>Enough large difference: -log(P_pco/P_acat) &gt; 5</p></li>
<li><p>At least one weak association: max(P_pco, P_acat) &gt;
1e-5</p></li>
<li><p>Non-inf values: Non-zero P_pco</p></li>
</ul></li>
<li><p>Window - 100kb; Chr - 20-22</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf" title="Figure: " width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5">
Past versions of
02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf
</button>
</p>
<div
id="fig-02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf" title="Figure: " width="20%" type="application/pdf" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5">
Past versions of
02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf
</button>
</p>
<div
id="fig-02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="jan-25" class="section level1">
<h1>Jan 25</h1>
<div id="quick-summary-8" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Test peak group defined by smaller window sizes using ACAT, minP,
PCO.</li>
</ul>
</div>
<div id="try-smaller-window-sizes" class="section level2">
<h2>1. Try smaller window sizes</h2>
<div id="why-4" class="section level3">
<h3>Why?</h3>
<ul>
<li><p>We observed from last week’s results that, it seems that PCO’s p
move toward ACAT/minP p when window sizes decrease.</p></li>
<li><p>Groups of smaller window sizes that have multiple peaks tend to
have physically closer peaks, thus more shared genetic control.</p></li>
</ul>
<p>Therefore, we wanted to check if that pattern persists under smaller
window sizes. So, I tried extreme window sizes, 10kb, 25kb, 50kb.</p>
</div>
<div id="how-10" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Window sizes</p>
<p>10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p></li>
<li><p>Test groups with size &gt; 1</p>
<p>Smaller window sizes can have less groups tested.</p></li>
<li><p>Test overlapped cis-SNPs within 500kb of peaks</p></li>
<li><p>Available results</p>
<p>window size - 10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p>
<p>chr - 20, 21, 22.</p></li>
</ul>
</div>
<div id="results-6" class="section level3">
<h3>Results</h3>
<ul>
<li>Compare methods under same window size</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf" title="Figure: Scatter plot of top p by three methods under same window size on chr21. x-axis: Each peak group. y-axis: -log10(P). Color show methods. Panel show window sizes." width="50%" type="application/pdf" />
<p class="caption">
Figure: Scatter plot of top p by three methods under same window size on
chr21. x-axis: Each peak group. y-axis: -log10(P). Color show methods.
Panel show window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf" title="Figure: Distribution density of top p by three methods under same window size on chr21. x-axis: -log10(P). y-axis: density of peak group with corresponding P. Color show methods. Panel show window sizes. Same total number of groups within each panel." width="50%" type="application/pdf" />
<p class="caption">
Figure: Distribution density of top p by three methods under same window
size on chr21. x-axis: -log10(P). y-axis: density of peak group with
corresponding P. Color show methods. Panel show window sizes. Same total
number of groups within each panel.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We observe that PCO seems to similar or even more small p’s under
smaller window sizes. The question is - Is it because PCO under small
windows better or ACAT/minP worse?</p>
<ul>
<li>Compare window sizes of same method</li>
</ul>
<p>(more complicated as different numbers of groups tested)</p>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf" title="Figure: Distribution of top p across window sizes by same method on chr21. x-axis: -log10(P). y-axis: (A) density, (B) number of peak group with corresponding P. Color show window sizes. Panel show methods. (C) Different numbers of groups (size &gt; 1) tested for window sizes in each panel." width="50%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p across window sizes by same method on
chr21. x-axis: -log10(P). y-axis: (A) density, (B) number of peak group
with corresponding P. Color show window sizes. Panel show methods. (C)
Different numbers of groups (size &gt; 1) tested for window sizes in
each panel.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Q - Larger window, higher proportion of small p, for ACAT/minP not
pco, why?</p>
<ul>
<li><p>A fair way to compare window sizes?</p></li>
<li><p>Top p of matching groups by three methods across window
sizes</p></li>
<li><p>Others</p>
<ul>
<li><p>Group size tested</p></li>
<li><p>Number of cis-SNPs tested</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="pco-acat" class="section level2">
<h2>2. PCO + ACAT</h2>
</div>
</div>
<div id="jan-18" class="section level1">
<h1>Jan 18</h1>
<div id="quick-summary-9" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Compare across window sizes</p></li>
<li><p>PC-based test on (SNP, peak group)</p></li>
</ul>
</div>
<div id="determine-window-size" class="section level2">
<h2>1. Determine window size</h2>
<div id="use-all-p-not-just-top-p" class="section level3">
<h3>Use all p not just top-p</h3>
<ul>
<li>Compare p using ACAT and minp</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" title="Figure: ." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" title="Figure: ." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" title="Figure: ." width="30%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare p by ACAT across window sizes (different total number of
tests)</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf" title="Figure: ." width="60%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_window_size_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_window_size_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Distribution of small p under same number of peaks in group &amp;
same number of cpeaks in group</p>
<p>small_p_thre &lt;- 1e-8</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf" title="Figure: ." width="40%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_same_peak_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Percentage of cPeaks in group with small p under same number of
peaks in group &amp; same number of cpeaks in group</p>
<p>small_p_thre &lt;- 1e-8</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf" title="Figure: ." width="40%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb">
Past versions of
01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of groups</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf" title="Figure: ." width="40%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of cis-SNPs</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf" title="Figure: ." width="40%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb">
Past versions of
01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="determine-test" class="section level2">
<h2>2. Determine test</h2>
<div id="pc-based" class="section level3">
<h3>PC-based</h3>
<ul>
<li><p>phenotype</p></li>
<li><p>genotype</p></li>
<li><p>covaraites</p>
<p>H3K27AC: 14 PCs</p></li>
<li><p>Sigma</p>
<p>Number of peaks v.s. number of samples (73)</p></li>
<li><p><span class="math inline">\(\lambda&gt;0.1\)</span></p></li>
<li><p>Available results</p>
<p>Window size: 100kb, 250kb, 500kb.</p>
<p>Chr: 21, 22.</p></li>
<li><p>Comparison of top p across three methods</p>
<ul>
<li>Scatter plot</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf" title="Figure: ." width="30%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Distribution of top p</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf" title="Figure: ." width="30%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p v.s. group size</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf" title="Figure: ." width="80%" type="application/pdf" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="compare-multi-cqtls-with-uni-cqtls" class="section level2">
<h2>3. Compare multi-cQTLs with uni-cQTLs</h2>
<p>After discussion on slack, we decided <strong>not to use uni-cQTLs as
a comparison metric for now</strong>. Reasons -</p>
<ul>
<li><p>Comparison with uni-cQTLs not necessary in the long run for
biological interpretations.</p></li>
<li><p>Comparison with minp is sufficient for now</p></li>
<li><p>Other ways for comparison of multi-cQTLs on top of the # of
significant SNPs</p></li>
</ul>
</div>
</div>
<div id="jan-11" class="section level1">
<h1>Jan 11</h1>
<div id="quick-summary-10" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Make the pipeline automatic.</p></li>
<li><p>Tested (SNP, peak group) by ACAT at 100kb, 250kb, 500kb window
size.</p></li>
<li><p>Compare multi-cQTLs across window sizes and <del>with
uni-cQTLs</del>.</p></li>
</ul>
</div>
<div
id="define-peak-groups-under-different-window-sizes-and-test-snp-peak-group"
class="section level2">
<h2>1. Define peak groups under different window sizes and test (SNP,
peak group)</h2>
<p>Last time, I tested (SNP, peak group) of peak groups defined by 500kb
window sizes using SNPs within 500kb of peak group.</p>
<p>This time, I tried various window sizes, including 100kb, 250kb,
500kb.</p>
<div id="how-11" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Peaks are considered as a peak group using a sliding window of
sizes 100kb, 250kb, 500kb.</p></li>
<li><p>Use 500kb as the cis distance to search for the cis-SNPs of each
individual peaks in the group, then tested the overlapped SNPs across
all peaks in the group. [Note: we discussed on slack to use 500kb as cis
distance for all window sizes.]</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_choose_cis_dis.jpeg" alt="Figure: Choose cis distance for different window size." width="40%" />
<p class="caption">
Figure: Choose cis distance for different window size.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_choose_cis_dis">
Past versions of 01_11_choose_cis_dis.jpeg
</button>
</p>
<div id="fig-01_11_choose_cis_dis" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_choose_cis_dis.jpeg" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>ACAT test.</li>
</ul>
</div>
<div id="numbers" class="section level3">
<h3>Numbers</h3>
<p>Use a table or histogram …</p>
</div>
<div id="results-7" class="section level3">
<h3>Results</h3>
<ul>
<li>Number of peaks in groups</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_peaks_in_group_H3K27AC_window100kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window100kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window100kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window100kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window100kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_peaks_in_group_H3K27AC_window250kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window250kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window250kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window250kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window250kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_peaks_in_group_H3K27AC_window500kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" type="application/pdf" />
<p class="caption">
Figure: Number of peaks in peak groups defined by different window
sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window500kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window500kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Number of peaks in groups colored by cPeaks (uni)</p>
<p>[Note: as we discussed on slack, I used the results of cPeaks within
500kb cis distance.]</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf" title="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" type="application/pdf" />
<p class="caption">
Figure: Number of peaks in peak groups defined by different window
sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Peak groups on chr</li>
</ul>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window100kb.pdf">100kb
window</a></p>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window250kb.pdf">250kb
window</a></p>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window250kb.pdf">500kb
window</a></p>
<ul>
<li>Number of cPeaks in peak groups</li>
</ul>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window100kb_cpeak500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window250kb_cpeak500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window500kb_cpeak500kb.pdf">500kb
window</a></p>
<ul>
<li>Distribution of the top p-value of a peak group</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s. number of peaks in the group</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s. number of cPeaks in the group</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group on chromosomal position</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="35%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="35%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="35%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group on chromosomal position v.s number of cPeaks
in the group</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf" title="Figure: Distribution of top p-values across peak groups." width="30%" type="application/pdf" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s chromosomes</li>
</ul>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window100kb_cis500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window250kb_cis500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window500kb_cis500kb.pdf">500kb
window</a></p>
<ul>
<li>Top p of a peak group v.s. the number of peaks in the group</li>
</ul>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window100kb_cis500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window250kb_cis500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb.pdf">500kb
window</a></p>
</div>
</div>
<div id="compare-results-across-window-sizes" class="section level2">
<h2>2. Compare results across window sizes</h2>
<p>I compared the results of peak groups defined by different window
sizes in terms of (1) cQTLs (2) peak groups.</p>
<div id="top-p-value-distribution-across-window-sizes"
class="section level3">
<h3>Top p-value distribution across window sizes</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf" title="Figure: Top p-vaue of a peak group across window sizes." width="50%" type="application/pdf" />
<p class="caption">
Figure: Top p-vaue of a peak group across window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_window_size_H3K27AC_cis500kb">
Past versions of 01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_window_size_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="number-of-significant-cqtls-across-window-sizes"
class="section level3">
<h3>Number of significant cQTLs across window sizes</h3>
<ul>
<li><p>How?</p>
<ul>
<li><p>Use 1e-10 as the uniform p-value cutoff across window
sizes.</p></li>
<li><p>Unique cQTLs for each group</p></li>
</ul></li>
</ul>
<p>Define significant cQTLs under same cutoff, as (1) same number of
peak groups, (2) same cis distance used to search for SNPs to be
tested.</p>
<ul>
<li>But not same number of SNPs tested?</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf" title="Figure: Number of cQTLs across window sizes." width="30%" type="application/pdf" />
<p class="caption">
Figure: Number of cQTLs across window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10">
Past versions of 01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div id="fig-01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cqtls-overlap-across-window-sizes" class="section level3">
<h3>cQTLs overlap across window sizes</h3>
<ul>
<li>Use 1e-10 as the uniform p-value cutoff across window sizes.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf" title="Figure: Overlaps among cQTLs of different window sizes." width="40%" type="application/pdf" />
<p class="caption">
Figure: Overlaps among cQTLs of different window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10">
Past versions of 01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div id="fig-01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Observations</li>
</ul>
<ol style="list-style-type: decimal">
<li><p>Most are shared.</p></li>
<li><p>100kb some specific cQTLs?</p></li>
</ol>
<ul>
<li><p>Missed cQTLs v.s. distance to their target peak groups</p>
<p>Question to ask - Are the missed cQTLs those which are distant to
their target peak groups, therefore, were not considered for association
tests? Or is it because true higher power?</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2023/01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf" title="Figure: Window-size-specific cQTLs v.s. their distances to target peak groups." width="40%" type="application/pdf" />
<p class="caption">
Figure: Window-size-specific cQTLs v.s. their distances to target peak
groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10">
Past versions of
01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div
id="fig-01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="in-terms-of-peak-groups" class="section level3">
<h3>In terms of peak groups</h3>
</div>
</div>
<div id="compare-multi-cqtls-with-uni-cqtls-1" class="section level2">
<h2>3. Compare multi-cQTLs with uni-cQTLs</h2>
<p><strong>Questions?</strong></p>
</div>
</div>
<div id="dec-21" class="section level1">
<h1>Dec 21</h1>
<div id="quick-summary-11" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Ran associations (by ACAT) between SNPs and peak groups using
cis-window 1Mb.</p></li>
<li><p>Looked at the distribution of p-values.</p></li>
</ul>
</div>
<div id="update-test-a-snp-and-a-group-of-adjacent-peaks-by-acat"
class="section level2">
<h2>1. Update: Test a SNP and a group of adjacent peaks by ACAT</h2>
<div id="updates" class="section level3">
<h3>Updates</h3>
<ul>
<li><p>For each peak group, test its association across all overlapped
SNPs (SNPs with cis-cQTL associations with all peaks in the group).
<del><u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del>
<u><strong>[Use 1Mb cis-cQTL]</strong></u></p></li>
<li><p>For now, I only looked at the the peaks groups,</p>
<ul>
<li><p><del>where none of the peaks have significant cis-cQTLs
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del> Run
across all peak groups no matter how many significant cis-cQTLs they
have.</p></li>
<li><p>which have more than 1 peak</p></li>
<li><p>of mark H3K27AC</p></li>
</ul></li>
<li><p>Except peak groups on chr1.</p></li>
</ul>
</div>
<div id="results-8" class="section level3">
<h3>Results</h3>
<ul>
<li>Look at the distribution of top p-values of peak groups.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_density_H3K27AC.pdf" title="LEFT: Distribution of the top p-value (by ACAT and minp_adjusted) across peak groups." width="30%" type="application/pdf" />
<p class="caption">
LEFT: Distribution of the top p-value (by ACAT and minp_adjusted) across
peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_density_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_density_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_density_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_density_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Look at the distribution of top p-values of peak groups on the size
of peak groups and chromosomes.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" title="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf" title="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" type="application/pdf" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_on_chr_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_on_chr_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Distribution of top p-values on chromosomal position.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf" title="Distribution of the top p-value across peak groups on chromosomal position." width="60%" type="application/pdf" />
<p class="caption">
Distribution of the top p-value across peak groups on chromosomal
position.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_manhattan_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_manhattan_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p values of a peak group v.s. the number of peaks?</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_peaks.pdf" title="Similar figure as the above, except dividing the peak groups by the number of peaks in the group. Color: the number of peaks in a peak group." width="50%" type="application/pdf" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group. Color: the number of peaks in a peak
group.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_peaks">
Past versions of 12_21_plt_top_p_peak_group_peaks.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_peaks" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_peaks.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Top p values of a peak group v.s. the number of peaks with
cQTLs?</p>
<ul>
<li>.. and a more detailed zoom-in on chromosomal position given
below</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_w_cqtl.pdf" title="Similar figure as the above, except dividing the peak groups by the number of peaks in the group that have cQTLs. Color: the number of peaks in a peak group that have cQTLs." width="50%" type="application/pdf" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group that have cQTLs. Color: the number of peaks
in a peak group that have cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_w_cqtl">
Past versions of 12_21_plt_top_p_peak_group_w_cqtl.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_w_cqtl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_w_cqtl.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf" title="A more detailed zoom-in. Top p-values across peak groups on chromosomal position, grouped and colored by #peaks with uni-cQTLs." width="70%" type="application/pdf" />
<p class="caption">
A more detailed zoom-in. Top p-values across peak groups on chromosomal
position, grouped and colored by #peaks with uni-cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare the uni-cQTLs and multi-cQTLs of a peak group</li>
</ul>
<p>~~### Numbers</p>
<p>Total peak groups: 100858</p>
<p>Peak groups on chr1: 10038</p>
<p>Peak groups on chr2-chr22: 90150 (90787, 90166)</p>
<p>Peak groups with only 1 peak: 26458</p>
<p>~~</p>
</div>
</div>
</div>
<div id="dec-14" class="section level1">
<h1>Dec 14</h1>
<div id="quick-summary-12" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Run ACAT to test associations between (SNP, adjacent peak group) for
mark H3K27AC.</li>
</ul>
</div>
<div id="cqtl-of-peaks-with-cis-window-of-1mb" class="section level2">
<h2>1. cQTL of peaks with cis-window of 1Mb</h2>
<p>[Waiting for Carlos to run this step for me.]</p>
</div>
<div id="adjacent-peaks-as-peak-groups" class="section level2">
<h2>2. Adjacent peaks as peak groups</h2>
<div id="how-12" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Sliding window of length 500kb.</p></li>
<li><p>Start with a peak, extend from its start position to a 500kb
window, include all peaks in this window as a peak group.</p></li>
<li><p>Slide to next peak.</p></li>
</ul>
</div>
<div id="look-into-peak-groups" class="section level3">
<h3>Look into peak groups</h3>
<ul>
<li><p>Number of peak groups in total</p>
<p>There are 100,858 peak groups in total.</p></li>
<li><p>Number of peaks in each group</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_peaks_in_group_H3K27AC.pdf" title="How many peaks are included in the preak groups? x-axis: the number of peaks. y-axis: the number of peaks groups with the corresponding number of peaks." width="30%" type="application/pdf" />
<p class="caption">
How many peaks are included in the preak groups? x-axis: the number of
peaks. y-axis: the number of peaks groups with the corresponding number
of peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_in_group_H3K27AC">
Past versions of 12_14_peaks_in_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_in_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_in_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>For peak groups with the same number of peaks, how many peaks have
cQTLs? <u><strong>[Here I used the 100kb cis-cQTL]</strong></u></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_peaks_in_group_w_cqtl_H3K27AC.pdf" title="Similar figure as the above, except dividing the peak groups by the number of peaks in the group that have cQTLs. Color: the number of peaks in a peak group that have cQTLs." width="30%" type="application/pdf" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group that have cQTLs. Color: the number of peaks
in a peak group that have cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_in_group_w_cqtl_H3K27AC">
Past versions of 12_14_peaks_in_group_w_cqtl_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_in_group_w_cqtl_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_in_group_w_cqtl_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of peaks with/without cQTLs in each group <u><strong>[Here I
used the 100kb cis-cQTL]</strong></u></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_peaks_w_cqtl_in_group_H3K27AC.pdf" title="How many peaks within a peak group have cQTLs? x-axis: the number of peaks with cQTLs. y-axis: the number of peak groups that have corresponding cPeaks." width="30%" type="application/pdf" />
<p class="caption">
How many peaks within a peak group have cQTLs? x-axis: the number of
peaks with cQTLs. y-axis: the number of peak groups that have
corresponding cPeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_w_cqtl_in_group_H3K27AC">
Past versions of 12_14_peaks_w_cqtl_in_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_w_cqtl_in_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_w_cqtl_in_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="test-a-snp-and-a-group-of-adjacent-peaks-by-acat"
class="section level2">
<h2>3. Test a SNP and a group of adjacent peaks by ACAT</h2>
<div id="how-13" class="section level3">
<h3>How?</h3>
<ul>
<li><p>For each peak group, test its association across all overlapped
SNPs (SNPs with cis-cQTL associations with all peaks in the group).
<del><u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del>
<u><strong>[Use 1Mb cis-cQTL]</strong></u></p></li>
<li><p>For (peak group, SNP), test by ACAT, a method to combine
p-values.</p>
<ul>
<li>As a comparison, I also calculated the adjusted minp, which <span
class="math inline">\(=min\{p_{snp, peak_1}, \dots, p_{snp, peak_k}\}
\times k\)</span>, where <span class="math inline">\(k\)</span> is the
number of peaks in the group.</li>
</ul></li>
<li><p>For now, I only looked at the the peaks groups,</p>
<ul>
<li><p><del>where none of the peaks have significant cis-cQTLs
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del> Run
across all peak groups no matter how many significant cis-cQTLs they
have.</p></li>
<li><p>which have more than 1 peak</p></li>
<li><p>of mark H3K27AC</p></li>
</ul></li>
</ul>
</div>
<div id="results-9" class="section level3">
<h3>Results</h3>
<ul>
<li><p>There are 25,853 peak groups considered for association
test.</p></li>
<li><p>3,450 peak groups have at least one overlapped SNPs.
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></p></li>
</ul>
<p>So, now I have ACAT p-values (and adjusted minp) for peak groups
across their overlapped SNPs. I take the minimum p-value for a peak
group across the overlapped SNPs as the top p-value for the peak
group.</p>
<ul>
<li>Look at the distribution of top p-values of peak groups.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_plt_top_p_peak_group_H3K27AC.pdf" title="LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT: Distribution of the top p-value across peak groups." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/12_14_plt_top_p_peak_group_density_H3K27AC.pdf" title="LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT: Distribution of the top p-value across peak groups." width="30%" type="application/pdf" />
<p class="caption">
LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT:
Distribution of the top p-value across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_density_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_density_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_density_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_density_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Look at the distribution of top p-values of peak groups on the size
of peak groups and chromosomes.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" title="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf" title="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" type="application/pdf" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_on_chr_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_on_chr_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>More detailed p-values for (a SNP, a peak group) for a few peak
groups with significant p_acat. See more <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G56480.pdf">here</a>, <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57648.pdf">here</a>, <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57649.pdf">here</a>.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57644.pdf" title="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="30%" type="application/pdf" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_p_peak_group_H3K27AC_G57644">
Past versions of 12_14_plt_p_peak_group_H3K27AC_G57644.pdf
</button>
</p>
<div id="fig-12_14_plt_p_peak_group_H3K27AC_G57644" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_p_peak_group_H3K27AC_G57644.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations-3" class="section level3">
<h3>Observations</h3>
<ul>
<li><p>Looks like p-values by ACAT are similar to p-values by minp
adjusted by Bonferroni correction.</p></li>
<li><p>The minimum p-values are ~1e-6.</p></li>
<li><p>Why no significant associations?</p>
<ul>
<li><p>I only test ~3k peak groups that have overlapped SNPs within
100kb window. So these peak groups are very close. So, combining these
peaks don’t have much improvement.</p></li>
<li><p>If I include more peak groups (and more SNPs) that are not so
close for association test, by extending 100kb cis window to 1Mb cis
window, maybe there can be more significant p-values.)</p></li>
<li><p>Will update the numbers after Carlos sends me the 1Mb cQTL
associations.</p></li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="dec-09" class="section level1">
<h1>Dec 09</h1>
<div id="quick-summary-13" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Preliminary analysis on Carlos’s hQTL results.</li>
</ul>
</div>
<div id="carloss-analysis-and-results" class="section level2">
<h2>Carlos’s analysis and results</h2>
<ul>
<li><p>A few questions</p>
<ul>
<li><p>Sample size - H3K27AC (78/72), H3K4ME1 (77), H3K4ME3 (78),
H3K36ME3 (95)</p></li>
<li><p>Why use genes as phenotype for H3K36ME3 but not peaks?</p></li>
<li><p>For mark H3K36ME3, peak names used for <code>Counts.txt</code>,
but gene names used for the other files. A conversion file?</p></li>
<li><p>In file <code>NominalPass.txt.gz</code>, what’s the nominal
p-value cutoff?</p></li>
</ul></li>
<li><p>Data of H3K27AC, H3K4ME1, H3K4ME3, and H3K36ME3 chromatin
marks</p>
<ul>
<li><p>ChIP-seq data for H3K27AC, H3K4ME1 and H3K4ME3 chromatin marks in
75 LCLs from Yoruba invididuals was publicly available from a study by
Grubert et al (2015).</p></li>
<li><p>We produced in this study the Cut&amp;Tag data for the H3K36ME3
chromatin mark in 96 LCLs from Yoruba individuals.</p></li>
</ul></li>
<li><p>Pre-processing</p>
<ul>
<li><p>We aligned the reads from all experiments to the GRCh38 human
genome using HISAT.</p></li>
<li><p>Peak calling for the ChIP-seq data was performed with MACS2 for
H3K27AC and H3K4ME1 (narrow peaks), and for H3K4ME3 (broad
peaks).</p></li>
<li><p>For each individual, quantified ChIP-seq coverage for the three
marks across the peaks using featureCounts.</p></li>
<li><p>After filtering for low coverage and low variance, we were left
with 100858 peaks for H3K27AC, 182749 peaks for H3K4ME1, and 55769 peaks
for H3K4ME3.</p></li>
<li><p>H3K36ME3 is a chromatin mark associated with transcription, and
it doesn’t form clearly defined peaks like the other marks. For
H3K36ME3, we quantified coverage across the top expressed 14000 protein
coding genes with featureCounts. The set of top expressed genes was
determined with mRNA expression in LCLs from Yoruba invidivuals in
RNA-seq data from the Geuvadis consortium.</p></li>
</ul></li>
<li><p>Normalization</p>
<p>Each phenotype (peaks or genes) by individuals coverage table was
normalized as follows:</p>
<ul>
<li><p>first, for each individual we normalized the raw peak or gene
counts to counts per million (CPM).</p></li>
<li><p>Second, for each phenotype we applied the standard normalization
across all individuals.</p></li>
<li><p>Finally, for each individual we applied the rank-based inverse
normal transformation across all phenotypes.</p></li>
</ul></li>
<li><p>hQTL calling</p>
<ul>
<li>hQTLs were called using QTLTools with permutation pass and nominal
pass. We used a cis window of 100 kb, and the principal components as
covariates.</li>
</ul></li>
<li><p>Results</p>
<p>In summary, for each of H3K27AC, H3K4ME1 and H3K4ME3 we have a set of
peaks. For each of the four chromatin marks (including H3K36ME3), we
have the following data:</p>
<ul>
<li><p><strong>Raw counts</strong>: Phenotype (peaks or gene) by
individual table of raw counts.</p></li>
<li><p><strong>Normalized counts</strong>: Phenotype by individual table
of qqnorm data (normalization described above)</p></li>
<li><p><strong>Permutation pass (top hQTL-peak/gene
association)</strong>: Table with top hQTLs for each phenotype from
QTLTools in permutation pass.</p></li>
<li><p><strong>Nominal pass (all hQTL-peak/gene associations within
100kb windows of the peak)</strong>: Table with hQTLs results for each
SNP within 100kb of each phenotype from QTLTools in permutation
pass.</p></li>
</ul></li>
</ul>
</div>
<div id="strong-tag-enriched-peaks" class="section level2">
<h2>Strong tag enriched peaks</h2>
<div id="look-at-peaks" class="section level3">
<h3>Look at peaks</h3>
<p>Questions of interest:</p>
<ul>
<li><p>Length of merged peaks</p></li>
<li><p>Number of sub-peaks merged into large peaks</p></li>
<li><p>Distance between peaks</p></li>
</ul>
</div>
<div id="look-at-pqtl-effects" class="section level3">
<h3>Look at pQTL effects</h3>
<p>Questions of interest:</p>
<ul>
<li><p>What is the proportion of peaks with pQTLs?</p>
<ul>
<li><p>For mark H3K27AC , there are 105049 peaks in total. After
preprocessing, 100858 peaks are left for QTL calling. <strong>7380 peaks
( 0.07317218 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
<li><p>For mark H3K4ME1 , there are 190250 peaks in total. After
preprocessing, 182749 peaks are left for QTL calling. <strong>4383 peaks
( 0.02398372 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
<li><p>For mark H3K4ME3 , there are 58171 peaks in total. After
preprocessing, 55769 peaks are left for QTL calling. <strong>3246 peaks
( 0.05820438 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/1209_dis_sig_peak_H3K27AC.pdf" title="Significant peaks of mark H3K27AC." width="30%" type="application/pdf" />
<p class="caption">
Significant peaks of mark H3K27AC.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1209_dis_sig_peak_H3K27AC">
Past versions of 1209_dis_sig_peak_H3K27AC.pdf
</button>
</p>
<div id="fig-1209_dis_sig_peak_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/1209_dis_sig_peak_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Length of peaks with &amp; w.o. pQTLs</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/1209_dis_peak_len_H3K27AC.pdf" title="Peak length of mark H3K27AC." width="50%" type="application/pdf" />
<p class="caption">
Peak length of mark H3K27AC.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1209_dis_peak_len_H3K27AC">
Past versions of 1209_dis_peak_len_H3K27AC.pdf
</button>
</p>
<div id="fig-1209_dis_peak_len_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/1209_dis_peak_len_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>- Reads count of peaks with &amp; w.o. pQTLs</strong>
<strong>- Peak enrichment (weak v.s. strong) for peaks with &amp; w.o.
pQTLs</strong></p>
<ul>
<li><p>What is the distribution of effect sizes for peaks (with pQTL
&amp; without pQTLs)?</p>
<ul>
<li><p>Distribution of all effects</p></li>
<li><p>Distribution of all effects on the dimension of SNPs and peaks
coordinates</p></li>
<li><p>Distribution of the most significant effect for each peaks across
SNPs</p></li>
</ul></li>
</ul>
<p><del>- Distribution of peaks with pQTL &amp; without pQTLs?</del></p>
<ul>
<li>Are peaks without pQTLs more likely to be peaks with merged
sub-peaks? (Can combining multiple sub-peaks in a large peak improve
power than considering the large peak as a whole?)</li>
</ul>
<p><strong>- Are peaks without pQTLs more adjacent than peaks with
pQTLs? (Can combining multiple adjacent peaks improve power than using
individual peaks?)</strong></p>
</div>
</div>
<div id="weak-tag-enriched-peaks" class="section level2">
<h2>Weak tag enriched peaks</h2>
</div>
<div id="what-type-of-peaks-should-have-improved-qtl-calling"
class="section level2">
<h2>What type of peaks should have improved QTL calling?</h2>
</div>
<div id="how-to-improve" class="section level2">
<h2>How to improve?</h2>
<ol style="list-style-type: decimal">
<li><p>Combine two peaks</p></li>
<li><p>Combine sub-peaks for one peak</p></li>
</ol>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-lun2016" class="csl-entry">
Lun, Aaron T. L., and Gordon K. Smyth. 2016. <span>“Csaw: A Bioconductor
Package for Differential Binding Analysis of ChIP-Seq Data Using Sliding
Windows.”</span> <em>Nucleic Acids Research</em> 44 (5): e45. <a
href="https://doi.org/10.1093/nar/gkv1191">https://doi.org/10.1093/nar/gkv1191</a>.
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       highr_0.10        compiler_4.2.3    pillar_1.9.0     
 [5] bslib_0.5.0       later_1.3.1       git2r_0.32.0      jquerylib_0.1.4  
 [9] tools_4.2.3       getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7   
[13] evaluate_0.21     lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3  
[17] rlang_1.1.1       cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7       
[21] xfun_0.39         fastmap_1.1.1     httr_1.4.6        stringr_1.5.0    
[25] knitr_1.43        fs_1.6.2          vctrs_0.6.3       sass_0.4.6       
[29] rprojroot_2.0.3   glue_1.6.2        R6_2.5.1          processx_3.8.2   
[33] fansi_1.0.4       rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3   
[37] whisker_0.4.1     ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5  
[41] httpuv_1.6.11     utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
