<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2021</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Log2021</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-02-06
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree74c629e7a179ab4fa37672fc63d508c3ff2b5b8dtargetblank74c629ea"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liliw-w/GradLog/tree/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d" target="_blank">74c629e</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree74c629e7a179ab4fa37672fc63d508c3ff2b5b8dtargetblank74c629ea" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liliw-w/GradLog/tree/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d" target="_blank">74c629e</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    code/.DS_Store
    Ignored:    data/.DS_Store
    Ignored:    output/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Log2021.Rmd</code>) and HTML (<code>docs/Log2021.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d/analysis/Log2021.Rmd" target="_blank">74c629e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “docs/power.png”, “docs/Muscle_Skeleta.p.funcExplorer.png”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/86b2e826a9533b862b6715d0f7de193374e4a3b3/docs/Log2021.html" target="_blank">86b2e82</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/analysis/Log2021.Rmd" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “docs/modules.png”, “docs/Muscle_Skeleta.p.WGCNA.png”,
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bb3c0167c3a5fe978c67b3ee26743b21e08d7514/docs/Log2021.html" target="_blank">bb3c016</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/83d3f1df74b031551273c819b1badc3cad2ddcce/docs/Log2021.html" target="_blank">83d3f1d</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/301726295134b063cc529b09d126e305e44f4b04/analysis/Log2021.Rmd" target="_blank">3017262</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “data/signals.all.txt”,
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a753e96616e6107b3903d6520d76e907334055b3/docs/Log2021.html" target="_blank">a753e96</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b7111b226ef2465c1082c1bf829c3c72e56de4d3/analysis/Log2021.Rmd" target="_blank">b7111b2</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “docs/signalsv.s.FDR.png”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1b48368459d0a166d3bfc3723f18a344babcb81c/docs/Log2021.html" target="_blank">1b48368</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9526e7a4db2194d4466bfb704904b23f4e671808/analysis/Log2021.Rmd" target="_blank">9526e7a</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “docs/signalsv.s.FDR.png”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/9b6da5ccad8ca7b6d03a1d25d165d6e69310e006/docs/Log2021.html" target="_blank">9b6da5c</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/16f7005f5912525fa5678a416cabad511d9c0e42/analysis/Log2021.Rmd" target="_blank">16f7005</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-26
</td>
<td>
wflow_publish(c(“analysis/Log2021.Rmd”, “data/signals.all.txt”))
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/82152fd9cf25b5181433540129f1caabec8921f3/docs/Log2021.html" target="_blank">82152fd</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a3644196362b1562516fb85a55f6f3f2103f171e/docs/Log2021.html" target="_blank">a364419</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b8815e3a5cad91b286dd209681a60875e4ea6d2c/analysis/Log2021.Rmd" target="_blank">b8815e3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
docs/nCross.png
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0611c355e9eae2bcad4336efdf32a86a777ee6e4/docs/Log2021.html" target="_blank">0611c35</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/11c79eb5f073cdff8f48f909a8c3d1e3974d2b03/analysis/Log2021.Rmd" target="_blank">11c79eb</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
<td>
Add simulation.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/51465aef79db94aa2c17b98069e03c1e23630d1f/docs/Log2021.html" target="_blank">51465ae</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cc5fc2819c50a235676ebf78877d78ee4499b218/analysis/Log2021.Rmd" target="_blank">cc5fc28</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
wflow_publish(“analysis/Log2021.Rmd”)
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1f9d60f1b33018a546171bd15d14f3a20120eb57/docs/Log2021.html" target="_blank">1f9d60f</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b4ce27bdf62d804dca1bd3ff81f689620dddf4dd/analysis/Log2021.Rmd" target="_blank">b4ce27b</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
<td>
Update GTEx.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/51b1a4cc02e4dd63c9a84c02ea8c597e171bc5ca/docs/Log2021.html" target="_blank">51b1a4c</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/23d78aacbfd5ed4ed07e702b4071961aec7928bb/analysis/Log2021.Rmd" target="_blank">23d78aa</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
<td>
wflow_git_commit(c("analysis/_site.yml“,”analysis/index.Rmd",
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="jan-26" class="section level2">
<h2>Jan 26</h2>
<div id="simulation" class="section level3">
<h3>simulation</h3>
<ul>
<li><p>Methods: Oracle, PC1, PCO, MinP, minp.</p></li>
<li><p>Models: <span class="math inline">\(z \sim N(\sqrt{N} \beta, \Sigma)\)</span>. <span class="math inline">\(\beta \sim N(0, \sigma_{b}^2 I_{K \times K})\)</span>.</p></li>
</ul>
<ol style="list-style-type: decimal">
<li><p>various sample size <span class="math inline">\(N\)</span>. 200, 400, 600, 800, 1000.</p></li>
<li><p>various <span class="math inline">\(\sigma_{b}^2\)</span>. 0.005, 0.01, 0.05, 0.1, 0.2.</p></li>
<li><p>various causality percentage. ???</p></li>
</ol>
<ul>
<li><p>simulation.</p>
<p>Generate <span class="math inline">\(10^4\)</span> zscores under each model. <span class="math inline">\(10^4\)</span> simulations.</p></li>
</ul>
<p>The following plot gives the boxplot, mean plot with standard deviation, and mean plot with standard error of the mean. Which one to use?</p>
<div class="figure" style="text-align: center">
<img src="power.png" alt="power comparison"  />
<p class="caption">
power comparison
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-power">
Past versions of power.png
</button>
</p>
<div id="fig-power" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d/docs/power.png" target="_blank">74c629e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/docs/power.png" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="larger-fdr-level-signals-in-large-and-small-module" class="section level3">
<h3>Larger FDR level, #signals in large and small module?</h3>
<p>Here I will look at how the number of signals changes in the original large and current small modules when I increase the FDR level.</p>
<p>The plot below is based on Muscle tissue from GTEx_v8. The x-axis is for various FDR levels, including 5%, 10%, 15%, and 20%. The y-axis is for the number of signals under these FDR levels. I plot three types of signals, i.e. module and SNP pairs (left subplot), unique SNPs (middle subplot), independent SNPs (right subplot). The two colors represent our previous WGCNA modules (red) and current smaller WGCNA modules (blue).</p>
<p>We can observe (1) When increasing FDR levels, number of signals only increase a little bit, say 3-5 more signals. This means that the corrected pvalues are bipolar, i.e. either very small or very large. (2) Signals based on smaller modules are similar to that of previous standard modules.</p>
<div class="figure" style="text-align: center">
<img src="./signalsv.s.FDR.Muscle.png" alt="Muscle: #signals v.s.FDR"  />
<p class="caption">
Muscle: #signals v.s.FDR
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-signalsv.s.FDR.Muscle">
Past versions of signalsv.s.FDR.Muscle.png
</button>
</p>
<div id="fig-signalsv.s.FDR.Muscle" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/docs/signalsv.s.FDR.Muscle.png" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="./signalsv.s.FDR.DGN.png" alt="DGN: #signals v.s.FDR"  />
<p class="caption">
DGN: #signals v.s.FDR
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-signalsv.s.FDR.DGN">
Past versions of signalsv.s.FDR.DGN.png
</button>
</p>
<div id="fig-signalsv.s.FDR.DGN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/docs/signalsv.s.FDR.DGN.png" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="smaller-gene-modules-more-signals" class="section level3">
<h3>Smaller gene modules, more signals?</h3>
<p>I changed the parameter <code>deepSplit</code> in function <code>cutreeDynamic</code> from package WGCNA to get modules with smaller size. I run it on the Muscle tissue from GTEx (see <code>GTEx_v8.Muscle_Skeletal.WGCNA</code> in the GTEx results table). Compared to <code>GTEx_v8.Muscle_Skeletal</code>, though more modules (32 v.s. 18), they have same independent signals (3), located on chr5, chr10, and chr22.</p>
<p>The corresponding modules of these signals from small and large WGCNA share a large proportion of genes.</p>
<table>
<thead>
<tr class="header">
<th align="left">signal.Chr</th>
<th align="left">module.small (#genes)</th>
<th align="left">module.big (#genes)</th>
<th align="right">#overlapped genes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">chr5</td>
<td align="left">module21 (89)</td>
<td align="left">module15 (114)</td>
<td align="right">88</td>
</tr>
<tr class="even">
<td align="left">chr10</td>
<td align="left">module21 (89)</td>
<td align="left">module15 (114)</td>
<td align="right">88</td>
</tr>
<tr class="odd">
<td align="left">chr22</td>
<td align="left">module2 (297)</td>
<td align="left">module4 (350)</td>
<td align="right">257</td>
</tr>
</tbody>
</table>
<p>The above table is for clusters obtained by method ‘WGCNA.min20_deep2’ (big) and ‘WGCNA.min20_deep4’ (small). Next, I give more info about the clusters by more methods, i.e. #genes in modules (A), #unclassfied genes (B), and #modules (C) by the six methods. I also give the pvalue distribution corresponding to these clusters below.</p>
<p>Generally, funcExplorer gives more modules, smaller module, and much more unclassified genes. Though the smaller modules, there aren’t many extreme pvalues. Say, the minimum p of all is about <span class="math inline">\(10^{-9}\)</span>, as compared to <span class="math inline">\(10^{-14}\)</span> by other methods.</p>
<p>From the pvalue plot, it looks like ‘WGCNA.min10_deep4’ gives smaller clusters and some extreme pvalues. So next step I will run the whole pipleline for this setting.</p>
<div class="figure" style="text-align: center">
<img src="./modules.png" alt="Modules v.s. methods/parameters"  />
<p class="caption">
Modules v.s. methods/parameters
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-modules">
Past versions of modules.png
</button>
</p>
<div id="fig-modules" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0b5d8d3467486ab96260d08796bd15f84bd8b368/docs/modules.png" target="_blank">0b5d8d3</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-02
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/af77a67144476c5f5d538dd369be98755ac3a669/docs/modules.png" target="_blank">af77a67</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/301726295134b063cc529b09d126e305e44f4b04/docs/modules.png" target="_blank">3017262</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="./Muscle_Skeleta.p.funcExplorer.png" alt="p distribution of Muscle"  />
<p class="caption">
p distribution of Muscle
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Muscle_Skeleta.p.funcExplorer">
Past versions of Muscle_Skeleta.p.funcExplorer.png
</button>
</p>
<div id="fig-Muscle_Skeleta.p.funcExplorer" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/74c629e7a179ab4fa37672fc63d508c3ff2b5b8d/docs/Muscle_Skeleta.p.funcExplorer.png" target="_blank">74c629e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-02-06
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="jan-05-jan-19" class="section level2">
<h2>Jan 05, Jan 19</h2>
<div id="simulation-1" class="section level3">
<h3>Simulation</h3>
<p>The simulations here aim to compare the power of PC-based tests including PC1 (using only the primary PC) and PCO (using combined PC’s), and non-PC based test MinP. The simulations consist of two parts: (1) verify that type 1 error is well controlled; (2) compare power of tests. At this time, I simply assume there is no LD among SNPs, i.e. tests are independent.</p>
<ol style="list-style-type: decimal">
<li>Verify that type 1 error is well controlled.</li>
</ol>
<p>Since the tests are assumed to be independent, the null pvalues should be uniformly distributed. To verify that, I estimate the null pvalues by empirical p’s obtained from the following steps.</p>
<p>Step1. Generate <span class="math inline">\(z_0 \sim N(0, \Sigma) | H_0\)</span>.</p>
<p>Step2. Run tests including PC1, PCO, MinP for <span class="math inline">\(10^6\)</span> times and obtain <span class="math inline">\(10^6\)</span> p’s for each test.</p>
<p>Step3. Compute the empirical T1E at significance <span class="math inline">\(\alpha\)</span> by <span class="math inline">\(\frac{I\{p&lt;\alpha\}}{10^6}\)</span>. Draw qqplot of <span class="math inline">\(-log_{10}p\)</span>.</p>
<p>The correlation matrix <span class="math inline">\(\Sigma\)</span> is the correlation of genes in module15 of GTEx dataset Muscle_Skeletal (Sigma.Muscle_Skeletal.module15.chr7.rds).</p>
<table>
<thead>
<tr class="header">
<th align="left">alpha</th>
<th align="center">PC1</th>
<th align="right">PCO M</th>
<th align="left">inP</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">5e-02</td>
<td align="center">0.050356</td>
<td align="right">0.053584</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">1e-04</td>
<td align="center">0.000093</td>
<td align="right">0.000114</td>
<td align="left"></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="./qqplot.simulation.null.png" alt="qqplot.simulation.null"  />
<p class="caption">
qqplot.simulation.null
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-qqplot.simulation.null">
Past versions of qqplot.simulation.null.png
</button>
</p>
<div id="fig-qqplot.simulation.null" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/11c79eb5f073cdff8f48f909a8c3d1e3974d2b03/docs/qqplot.simulation.null.png" target="_blank">11c79eb</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>Compare power of tests.</li>
</ol>
<p>To compare the power of non-/PC-based tests, I will run the oracle test in addition to PC1, PCO, and MinP to benchmark the power. It’s mentioned in the PCO paper that given a fixed correlation relationship among the phenotypes, the power of a test depends on the relationship bewteen the true effects (<span class="math inline">\(\beta\)</span>) and the phenotype correlation (<span class="math inline">\(\Sigma\)</span>). Therefore, to see which tests perform well in what cases, I will consider different models, including (1) <span class="math inline">\(\beta=10 u_1\)</span>; (2) <span class="math inline">\(\beta=4 u_k\)</span>; (3) <span class="math inline">\(\beta=1.5 u_K\)</span>; (4) <span class="math inline">\(\beta=rnorm(K)\)</span>; (5) <span class="math inline">\(\beta=rnorm(0.7K)\)</span>; (6) <span class="math inline">\(\beta=rnorm(0.3K)\)</span>.</p>
<p>Step1. Generate <span class="math inline">\(z_0 \sim N(\beta, \Sigma) | H_1\)</span>.</p>
<p>Step2. Run tests including Oracle, PC1, PCO, MinP for <span class="math inline">\(10^6\)</span> times and obtain <span class="math inline">\(10^6\)</span> p’s for each test.</p>
<p>Step3. Compute the power at significance <span class="math inline">\(\alpha\)</span> by <span class="math inline">\(\frac{I\{p&lt;\alpha\}}{10^6}\)</span>.</p>
<table>
<thead>
<tr class="header">
<th align="left">Model</th>
<th align="right">Oracle</th>
<th align="right">PC1</th>
<th align="right">PCO</th>
<th align="left">MinP</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">u1</td>
<td align="right">0.946341</td>
<td align="right">0.902540</td>
<td align="right">0.746301</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">uPCO</td>
<td align="right">0.989835</td>
<td align="right">0.050178</td>
<td align="right">0.713434</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">uK</td>
<td align="right">0.953727</td>
<td align="right">0.049719</td>
<td align="right">0.054140</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">100%</td>
<td align="right">1.000000</td>
<td align="right">0.058867</td>
<td align="right">0.945918</td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">70%</td>
<td align="right">1.000000</td>
<td align="right">0.058910</td>
<td align="right">0.580368</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">30%</td>
<td align="right">0.999995</td>
<td align="right">0.050201</td>
<td align="right">0.149904</td>
<td align="left"></td>
</tr>
</tbody>
</table>
</div>
<div id="gtex" class="section level3">
<h3>GTEx</h3>
<div id="results" class="section level4">
<h4>Results</h4>
<p>The table below summarizes all results I have so far, followed by figures of the distributions of pvalues in various datasets.</p>
<table>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="left">Nsample</th>
<th align="left">All</th>
<th align="left">Annotated</th>
<th align="left">Filtered</th>
<th align="left">Filtered_info</th>
<th align="left">Final</th>
<th align="left">Nmodule</th>
<th align="left">FDR</th>
<th align="left">Nperm</th>
<th align="left">QTL_Module</th>
<th align="left">unique_QTL</th>
<th align="left">independent_QTL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">11979</td>
<td align="left">8120</td>
<td align="left">0;165;1663;8073</td>
<td align="left">3859</td>
<td align="left">21</td>
<td align="left">combined chr+module</td>
<td align="left">20</td>
<td align="left">659</td>
<td align="left">623</td>
<td align="left">40</td>
</tr>
<tr class="even">
<td align="left">DGN_new</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">12585</td>
<td align="left">9939</td>
<td align="left">1;453;1798;8705</td>
<td align="left">3695</td>
<td align="left">19</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">331</td>
<td align="left">275</td>
<td align="left">26</td>
</tr>
<tr class="odd">
<td align="left">TCGA</td>
<td align="left">788</td>
<td align="left">17656</td>
<td align="left">15994</td>
<td align="left">12694</td>
<td align="left">336;3504;10697;598</td>
<td align="left">4962</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Whole_Blood</td>
<td align="left">670</td>
<td align="left">20315</td>
<td align="left">20315</td>
<td align="left">15245</td>
<td align="left">619;4544;3677;13096</td>
<td align="left">5070</td>
<td align="left">22</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">4</td>
<td align="left">4</td>
<td align="left">2</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Muscle_Skeletal</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">15601</td>
<td align="left">675;4447;3631;13490</td>
<td align="left">5430</td>
<td align="left">18</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">38</td>
<td align="left">38</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Skin_Sun_Exposed_Lower_leg</td>
<td align="left">605</td>
<td align="left">25196</td>
<td align="left">25196</td>
<td align="left">18801</td>
<td align="left">807;6458;4557;15650</td>
<td align="left">6395</td>
<td align="left">30</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">1</td>
<td align="left">1</td>
<td align="left">1</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Artery_Tibial</td>
<td align="left">584</td>
<td align="left">23304</td>
<td align="left">23304</td>
<td align="left">17390</td>
<td align="left">795;5588;4202;14730</td>
<td align="left">5914</td>
<td align="left">20</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">0</td>
<td align="left">0</td>
<td align="left">0</td>
</tr>
<tr class="even">
<td align="left">GTEx_v8.Muscle_Skeletal.cross</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">7141</td>
<td align="left">675;4447;3631;0</td>
<td align="left">13890</td>
<td align="left">39</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">43</td>
<td align="left">43</td>
<td align="left">4</td>
</tr>
<tr class="odd">
<td align="left">GTEx_v8.Muscle_Skeletal.WGCNA</td>
<td align="left">706</td>
<td align="left">21031</td>
<td align="left">21031</td>
<td align="left">15601</td>
<td align="left">675;4447;3631;13490</td>
<td align="left">5430</td>
<td align="left">32</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left">44</td>
<td align="left">44</td>
<td align="left">3</td>
</tr>
<tr class="even">
<td align="left">DGN_new.WGCNA</td>
<td align="left">913</td>
<td align="left">13634</td>
<td align="left">12585</td>
<td align="left">9939</td>
<td align="left">1;453;1798;8705</td>
<td align="left">3695</td>
<td align="left">40</td>
<td align="left">combined chr+module</td>
<td align="left">10</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
<tr class="odd">
<td align="left">TCGA_new</td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
<td align="left"></td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="DGN.p.png" alt="DGN" width="70%" height="70%" />
<p class="caption">
DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-DGN.p">
Past versions of DGN.p.png
</button>
</p>
<div id="fig-DGN.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b1109759a6eae54fe2d872646616312b1f8aa73f/docs/DGN.p.png" target="_blank">b110975</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-12-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="DGN_new.p.png" alt="DGN_new" width="70%" height="70%" />
<p class="caption">
DGN_new
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-DGN_new.p">
Past versions of DGN_new.p.png
</button>
</p>
<div id="fig-DGN_new.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/361c43ee4f6361a32103c987e6e5ec3ad30edb08/docs/DGN_new.p.png" target="_blank">361c43e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="Whole_Blood.p.png" alt="Whole_Blood" width="70%" height="70%" />
<p class="caption">
Whole_Blood
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Whole_Blood.p">
Past versions of Whole_Blood.p.png
</button>
</p>
<div id="fig-Whole_Blood.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/361c43ee4f6361a32103c987e6e5ec3ad30edb08/docs/Whole_Blood.p.png" target="_blank">361c43e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="Muscle_Skeletal.p.png" alt="Muscle_Skeletal" width="70%" height="70%" />
<p class="caption">
Muscle_Skeletal
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Muscle_Skeletal.p">
Past versions of Muscle_Skeletal.p.png
</button>
</p>
<div id="fig-Muscle_Skeletal.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/361c43ee4f6361a32103c987e6e5ec3ad30edb08/docs/Muscle_Skeletal.p.png" target="_blank">361c43e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="Skin_Sun_Exposed_Lower_leg.p.png" alt="Skin_Sun_Exposed_Lower_leg" width="70%" height="70%" />
<p class="caption">
Skin_Sun_Exposed_Lower_leg
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Skin_Sun_Exposed_Lower_leg.p">
Past versions of Skin_Sun_Exposed_Lower_leg.p.png
</button>
</p>
<div id="fig-Skin_Sun_Exposed_Lower_leg.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/361c43ee4f6361a32103c987e6e5ec3ad30edb08/docs/Skin_Sun_Exposed_Lower_leg.p.png" target="_blank">361c43e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="Artery_Tibial.p.png" alt="Artery_Tibial" width="70%" height="70%" />
<p class="caption">
Artery_Tibial
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Artery_Tibial.p">
Past versions of Artery_Tibial.p.png
</button>
</p>
<div id="fig-Artery_Tibial.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/361c43ee4f6361a32103c987e6e5ec3ad30edb08/docs/Artery_Tibial.p.png" target="_blank">361c43e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="Muscle_Skeletal.cross.p.png" alt="Muscle_Skeletal.cross" width="70%" height="70%" />
<p class="caption">
Muscle_Skeletal.cross
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-Muscle_Skeletal.cross.p">
Past versions of Muscle_Skeletal.cross.p.png
</button>
</p>
<div id="fig-Muscle_Skeletal.cross.p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/60cf10e13c677eb4ad76bd2be51ad8a4815357b6/docs/Muscle_Skeletal.cross.p.png" target="_blank">60cf10e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="remarks" class="section level4">
<h4>Remarks</h4>
<ol style="list-style-type: decimal">
<li><p>The dataset “DGN_new” represents DGN through the standard filtering (see “Gene filter” on Dec 10).</p></li>
<li><p>GTEx datasets generally have few significant signals and relatively large pvalues (compared to DGN).</p></li>
<li><p>The dataset “GTEx_v8.Muscle_Skeletal.cross” uses Muscle_Skeletal samples from GTEx_v8 (similar as GTEx_v8.Muscle_Skeletal), but without removing cross-mapped genes before constructing gene modules. Therefore, there are 13890 genes (v.s. 5430 in GTEx_v8.Muscle_Skeletal) in total which result in 39 modules (v.s. 18 in GTEx_v8.Muscle_Skeletal). We do this step because we observed that there are relatively few signals using our original pipeline and we wonder if the reason to this observation is us filtering too many genes in the first step and leaving too few signals. To check on this, we put the “filtering” to the last step, i.e. including potentially cross-mapped genes into the analysis and generate significant variant-module pairs. We then exclude those where target eGene in the module is cross-mappable with any gene within 1Mb of the variant. Hopefully we could have more signals.</p>
<p>However, though the increased genes and modules, the number of identified signals (43) is similar as that using the original pipeline (38). Next, I will look into these signals.</p></li>
</ol>
<table>
<caption>GTEx_v8.Muscle_Skeletal</caption>
<thead>
<tr class="header">
<th align="left">variant-module</th>
<th align="left">p</th>
<th align="left">q</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module15:10:48930105</td>
<td align="left">8.63e-10</td>
<td align="left">8.33e-03</td>
</tr>
<tr class="even">
<td align="left">module15:5:132440814</td>
<td align="left">1.56e-09</td>
<td align="left">2.16e-02</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132448891</td>
<td align="left">3.36e-09</td>
<td align="left">4.74e-02</td>
</tr>
<tr class="even">
<td align="left">module15:5:132450078</td>
<td align="left">5.81e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132450726</td>
<td align="left">5.96e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132450916</td>
<td align="left">8.89e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132451361</td>
<td align="left">5.97e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132451586</td>
<td align="left">1.11e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132453865</td>
<td align="left">1.05e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132454053</td>
<td align="left">9.94e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132454171</td>
<td align="left">1.03e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132454631</td>
<td align="left">1.44e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132454724</td>
<td align="left">1.08e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132455672</td>
<td align="left">1.07e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132455979</td>
<td align="left">1.04e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132456154</td>
<td align="left">9.96e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132456710</td>
<td align="left">6.04e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132458606</td>
<td align="left">5.33e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132459905</td>
<td align="left">4.84e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132459971</td>
<td align="left">5.04e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132460190</td>
<td align="left">5.05e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132460375</td>
<td align="left">5.56e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132460917</td>
<td align="left">5.03e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132461111</td>
<td align="left">5.17e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132463834</td>
<td align="left">5.47e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132464413</td>
<td align="left">3.90e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132464907</td>
<td align="left">1.01e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132466034</td>
<td align="left">4.56e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132468333</td>
<td align="left">1.32e-10</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132468353</td>
<td align="left">5.74e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132468564</td>
<td align="left">8.53e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132469724</td>
<td align="left">5.63e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132469899</td>
<td align="left">6.96e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132470043</td>
<td align="left">1.03e-10</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132470796</td>
<td align="left">5.42e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module15:5:132471932</td>
<td align="left">8.01e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module15:5:132474927</td>
<td align="left">2.71e-10</td>
<td align="left">2.94e-03</td>
</tr>
<tr class="even">
<td align="left">module4:22:23508295</td>
<td align="left">3.76e-10</td>
<td align="left">5.71e-03</td>
</tr>
</tbody>
</table>
<table>
<caption>GTEx_v8.Muscle_Skeletal.cross</caption>
<thead>
<tr class="header">
<th align="left">variant-module</th>
<th align="left">p</th>
<th align="left">q</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module8:1:123955560</td>
<td align="left">9.97e-12</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:1:123955561</td>
<td align="left">1.06e-11</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:16:36353041</td>
<td align="left">3.49e-11</td>
<td align="left">6.67e-03</td>
</tr>
<tr class="even">
<td align="left">module8:16:36353056</td>
<td align="left">8.29e-11</td>
<td align="left">1.25e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132453865</td>
<td align="left">1.56e-09</td>
<td align="left">3.12e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132454053</td>
<td align="left">1.59e-09</td>
<td align="left">2.86e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132454171</td>
<td align="left">1.52e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132454631</td>
<td align="left">1.43e-09</td>
<td align="left">3.57e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132454724</td>
<td align="left">1.53e-09</td>
<td align="left">3.23e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132455672</td>
<td align="left">1.50e-09</td>
<td align="left">3.45e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132455979</td>
<td align="left">1.56e-09</td>
<td align="left">3.03e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132456154</td>
<td align="left">1.78e-09</td>
<td align="left">3.42e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132456710</td>
<td align="left">1.36e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132458606</td>
<td align="left">3.37e-10</td>
<td align="left">1.76e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132459905</td>
<td align="left">6.50e-10</td>
<td align="left">1.74e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132459971</td>
<td align="left">3.92e-10</td>
<td align="left">1.43e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132460190</td>
<td align="left">3.50e-10</td>
<td align="left">1.58e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132460375</td>
<td align="left">3.37e-10</td>
<td align="left">1.67e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132460917</td>
<td align="left">3.98e-10</td>
<td align="left">1.36e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132461111</td>
<td align="left">3.57e-10</td>
<td align="left">1.50e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132463834</td>
<td align="left">8.21e-10</td>
<td align="left">2.40e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132464413</td>
<td align="left">9.93e-10</td>
<td align="left">2.69e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132466034</td>
<td align="left">6.75e-10</td>
<td align="left">1.67e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132468333</td>
<td align="left">1.86e-09</td>
<td align="left">3.50e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132468353</td>
<td align="left">1.69e-09</td>
<td align="left">3.06e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132468564</td>
<td align="left">2.43e-09</td>
<td align="left">4.19e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132469724</td>
<td align="left">1.93e-09</td>
<td align="left">3.66e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132469899</td>
<td align="left">1.73e-09</td>
<td align="left">3.24e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132470043</td>
<td align="left">1.56e-09</td>
<td align="left">2.94e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:132470796</td>
<td align="left">1.85e-09</td>
<td align="left">3.33e-02</td>
</tr>
<tr class="odd">
<td align="left">module8:5:132471932</td>
<td align="left">2.37e-09</td>
<td align="left">4.29e-02</td>
</tr>
<tr class="even">
<td align="left">module8:5:46659000</td>
<td align="left">9.86e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:46906585</td>
<td align="left">2.34e-11</td>
<td align="left">7.14e-03</td>
</tr>
<tr class="even">
<td align="left">module8:5:47027928</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47068540</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47166316</td>
<td align="left">3.33e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47210790</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47210990</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:47258679</td>
<td align="left">4.44e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:47258917</td>
<td align="left">3.33e-16</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:49809727</td>
<td align="left">2.70e-13</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="even">
<td align="left">module8:5:49930686</td>
<td align="left">0.00e+00</td>
<td align="left">0.00e+00</td>
</tr>
<tr class="odd">
<td align="left">module8:5:49936394</td>
<td align="left">1.10e-11</td>
<td align="left">0.00e+00</td>
</tr>
</tbody>
</table>
<p>For GTEx_v8.Muscle_Skeletal, there are 38 variant-module pairs, corresponding to 2 module (module 15, module4) and 3 independent loci on (chr5, chr10, chr22). GTEx_v8.Muscle_Skeletal.cross has 43 variant-module pairs, corresponding to 1 module (module 8) and 4 independent loci on (chr1, chr5, chr16).</p>
<p>The signal on chr5 is significant for both module 15 (114 genes) and module 8 (394 genes) in two datasets, which have 82 shared genes. Take SNP <a href="https://www.nature.com/articles/nature24277#Sec9">rs2706381</a> (chr5:132474927) for example. It is <a href="https://www.nature.com/articles/nature24277#Sec9">“in cis with IRF1 (<span class="math inline">\(P \le 2\times10^{-10}\)</span>; Fig. 6c), a transcription factor that facilitates regulation of the interferon-induced immune response”</a>. It is also <a href="https://www.nature.com/articles/nature24277#Sec9">“associated in trans with PSME1 (<span class="math inline">\(P \le 1.1\times10^{-11}\)</span>) and PARP10 (<span class="math inline">\(P \le 7.8\times10^{-10}\)</span>)”</a>. These two genes are included in module 15. The reference also gives additional results to “suggest that cis-regulatory loci affecting IRF1 are regulators of interferon-responsive inflammatory processes involving genes including PSME1 and PARP10, with implications for complex traits specific to muscle tissue”.</p>
<p>I also looked at the enrichment of the genes in module 15. These genes are mainly enriched in immunity-related terms and tuberculosis. To reproduce, use the gene list <a href="./gene.GTEx_v8.Muscle_Skeletal.module15.txt">here</a>.</p>
<p>SNP 10:48930105. module 15.</p>
<p>module4:22:23508295</p>
<ul>
<li><p>Check if the signals’ nearby genes are cross-mapped with genes in their corresponding module. The signal on chr5 corresponding to module8 has 20 genes +/- 500Kb away from its TSS. Many of them are crosspable with genes in the module 8. The number of of genes in the module crossmapped with each of the 20 genes are: <code>1, 144, 0, 62, 54, 0, 63, 4, 0, 0, 0, 59, 145, 0, 0, 128, 0, 23, 53</code>, where the nearest gene is crossmappable with 1 gene in the module.</p></li>
<li><p>The other signals on chr1, chr16, and chr5:47210790 don’t have any nearby genes around its TSS.</p></li>
<li><p>See the following plot for number of cross-mapped genes in each module.</p></li>
</ul>
<p><img src="nCross.png" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-nCross">
Past versions of nCross.png
</button>
</p>
<div id="fig-nCross" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d22021e4b203e7f09b0095a5bed0db071302b976/docs/nCross.png" target="_blank">d22021e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="new-tcga-by-the-standard-filtering" class="section level3">
<h3>New TCGA by the standard filtering</h3>
</div>
<div id="eqtlgen" class="section level3">
<h3>eQTLGen</h3>
<ol style="list-style-type: decimal">
<li><p>eQTLGen description</p>
<p><em>This full dataset includes 19942 genes that showed expression in blood tested and 10317 SNPs that are trait-associated SNPs based on GWAS Catalog.</em></p>
<p>After gene filter steps described in <code>Dec 10</code>, there are 4963 genes left. Applying the same filtering to 13634 DGN genes, there are 3695 genes left, among which 3642 are also included in eQTLGen. So, I will use these genes to do the downstream analysis, e.g. constructing co-expressed gene modules. These 3642 genes result in 19 gene modules.</p></li>
<li><p>Replication of DGN signals in eQTLGen</p>
<p>The table below gives the signals found in eQTLGen and DGN.</p>
<p>The first two rows give results based on eQTLGen zscores, with row 1 using <code>qvalue</code> for FDR correction (threshold <span class="math inline">\(0.05\)</span>) and row 2 using <span class="math inline">\(\frac{0.05}{\#DGN signals}\)</span> as significance threshold. The third row is based on the same gene modules and SNPs but tensorQTL zscores using DGN expression data. The FDR correction uses the empirical distribution of pvalues from the combined chr’s and modules (10 permutations).</p></li>
</ol>
<table>
<thead>
<tr class="header">
<th align="left">Dataset</th>
<th align="left">FDR</th>
<th align="right">minp</th>
<th align="right">(QTL, module)</th>
<th align="right">unique QTL</th>
<th align="right">independent QTL</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">eQTLGen</td>
<td align="left">qvalue</td>
<td align="right">5.82e-04</td>
<td align="right">2195</td>
<td align="right">909</td>
<td align="right">348</td>
</tr>
<tr class="even">
<td align="left">eQTLGen</td>
<td align="left">0.05/#DGN signals</td>
<td align="right">1.18e-04</td>
<td align="right">1707</td>
<td align="right">762</td>
<td align="right">286</td>
</tr>
<tr class="odd">
<td align="left">eQTLGen_DGN</td>
<td align="left">combined chr+module #10-perms</td>
<td align="right">1.00e-07</td>
<td align="right">420</td>
<td align="right">374</td>
<td align="right">28</td>
</tr>
</tbody>
</table>
<p>Among 374 eQTLGen_DGN signals, 16 are replicated in 762 eQTLGen signals. These 16 replicated signals consists of 6 independent SNPs, including (based on GRCh37),</p>
<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs12485738">rs12485738</a> : 3:56865776, intron variant of <em>ARHGEF3</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs643381">rs643381</a> : 6:139839423. (<a href="https://www.ncbi.nlm.nih.gov/snp/rs590856">rs590856</a>, 6:139844429).</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs149007767">rs149007767</a> 7:50370254: , intron variant of <em>IKZF1</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs12718597">rs12718597</a> : 7:50428428, intron variant of <em>IKZF1</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs35979828">rs35979828</a> : 12:54685880, 500B downstream variant of <em>NFE2</em>.</li>
<li><a href="https://www.ncbi.nlm.nih.gov/snp/rs7210990">rs7210990</a> : 17:16170764, intron variant of <em>PIGL</em>.</li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Replication of DGN signals in eQTLGen</li>
</ol>
<div class="figure" style="text-align: center">
<img src="./eQTLGen_DGN_venn.jpeg" alt="DGN" width="70%" height="70%" />
<p class="caption">
DGN
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-eQTLGen_DGN_venn">
Past versions of eQTLGen_DGN_venn.jpeg
</button>
</p>
<div id="fig-eQTLGen_DGN_venn" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/60cf10e13c677eb4ad76bd2be51ad8a4815357b6/docs/eQTLGen_DGN_venn.jpeg" target="_blank">60cf10e</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 3.6.3 (2020-02-29)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.6

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.6.2

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.4      rprojroot_1.3-2 digest_0.6.25   later_1.0.0    
 [5] R6_2.4.1        backports_1.1.5 git2r_0.27.1    magrittr_1.5   
 [9] evaluate_0.14   highr_0.8       stringi_1.4.6   rlang_0.4.7    
[13] fs_1.3.2        promises_1.1.0  whisker_0.4     rmarkdown_2.1  
[17] tools_3.6.3     stringr_1.4.0   glue_1.3.2      httpuv_1.5.2   
[21] xfun_0.12       yaml_2.2.1      compiler_3.6.3  htmltools_0.4.0
[25] knitr_1.28     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
