<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2022</title>

<script src="site_libs/header-attrs-2.14.2/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2022</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-06-11
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree4be96854d4f33e6238c6026de728ffea1469d3fatargetblank4be9685a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/4be96854d4f33e6238c6026de728ffea1469d3fa" target="_blank">4be9685</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree4be96854d4f33e6238c6026de728ffea1469d3fatargetblank4be9685a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/4be96854d4f33e6238c6026de728ffea1469d3fa" target="_blank">4be9685</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

Untracked files:
    Untracked:  Log2020.Rmd
    Untracked:  Trans.Rmd

Unstaged changes:
    Deleted:    analysis/Log2020.Rmd
    Deleted:    analysis/Trans.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2022.Rmd</code>) and HTML
(<code>docs/Log2022.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e6a219967e37e37280572b63403aa4dd684738cd/docs/Log2022.html" target="_blank">e6a2199</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ed7392ac29ac9c3bc61f7ca2e3054b64dbdce56b/analysis/Log2022.Rmd" target="_blank">ed7392a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-10
</td>
<td>
update rmd
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f60b0b9b9826f90567e7e0ea843b7ee2ce93f23f/docs/Log2022.html" target="_blank">f60b0b9</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/06afdff95ba63f4c4921ca43985a159904248918/analysis/Log2022.Rmd" target="_blank">06afdff</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-09
</td>
<td>
update error bar from se to 0.95 CI
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0155a0fe37e0797776139bca94f1e814f140e216/docs/Log2022.html" target="_blank">0155a0f</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/518a1c0b1cc18d00fc95c40d23493d1a1eb4b562/analysis/Log2022.Rmd" target="_blank">518a1c0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add more details
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/607ff2b1555cc61e6395a5ad8c3a765c33313958/docs/Log2022.html" target="_blank">607ff2b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/397fb52e1b7304df8501600cd1fcc959f3426336/analysis/Log2022.Rmd" target="_blank">397fb52</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/21ae027be39b4c1ff7c145334aef15997e0b1dce/docs/Log2022.html" target="_blank">21ae027</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0db9d7426b737c2a9901a671045b1a2e73485d3a/analysis/Log2022.Rmd" target="_blank">0db9d74</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0835a9c2081ac25f70b320c7fe58609511fc41aa/docs/Log2022.html" target="_blank">0835a9c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6d481ca00d02c00197745445efbaafd75bf8232b/analysis/Log2022.Rmd" target="_blank">6d481ca</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
fix a typo
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1b8143daad903e7b6882bf9c8eba4163a742fb0a/docs/Log2022.html" target="_blank">1b8143d</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0626baa5c59dfa308ff90956f7c13b0c65f80d1f/analysis/Log2022.Rmd" target="_blank">0626baa</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/59c3d71239d38efbc42825c93788c13f500bed6a/docs/Log2022.html" target="_blank">59c3d71</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9d92b99b9d5aa7746f02630955a7a8f23ed02853/analysis/Log2022.Rmd" target="_blank">9d92b99</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2e6c03934e4cd52fbb585020fffa30a179c6e572/docs/Log2022.html" target="_blank">2e6c039</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fae1f4ee357d211f63286d228c9bcc6c0b68d4c9/analysis/Log2022.Rmd" target="_blank">fae1f4e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
adjust fig position
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/3395a267589af572688e609b13621db4d8ca104d/docs/Log2022.html" target="_blank">3395a26</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d0a08e63e7fc87b48409fd0dcc879ce6d71ffae6/analysis/Log2022.Rmd" target="_blank">d0a08e6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
adjust fig position
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8b9071e4982b644e5868ebd9e0a417e578ab0257/docs/Log2022.html" target="_blank">8b9071e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/424015b9c6ebab80f3ade02c9c2f48f5c173bba3/analysis/Log2022.Rmd" target="_blank">424015b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/72025bf9b9f91c57745d7910e076d79a5fc2ee18/docs/Log2022.html" target="_blank">72025bf</a>
</td>
<td>
liliw-w
</td>
<td>
2022-02-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/089c60942bfc4e7416bbc5bcf73accd65aeef082/analysis/Log2022.Rmd" target="_blank">089c609</a>
</td>
<td>
liliw-w
</td>
<td>
2022-02-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1274e244c3b7d06966b3351caa0bff74870a3382/docs/Log2022.html" target="_blank">1274e24</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9976af6f087dd6b4546a78fc8936e5dc54c7e01a/analysis/Log2022.Rmd" target="_blank">9976af6</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bd69a21c8b6909c5174347c0c0c724d7169dc47c/docs/Log2022.html" target="_blank">bd69a21</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/28d0a2b13cae4c13c5979ffcf05154bc2aa15603/analysis/Log2022.Rmd" target="_blank">28d0a2b</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ec0a2af10bac6589b98ea6818de13cd3bda98a33/docs/Log2022.html" target="_blank">ec0a2af</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9e5c2e02a0e8f42e2fd9c4f697dd3d8c16fff6e2/analysis/Log2022.Rmd" target="_blank">9e5c2e0</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/97b4ca1336ea92e396991f1569177c14360eb07c/docs/Log2022.html" target="_blank">97b4ca1</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4c90bd8ce853f24d0248eee7fe6a8e038cb3d0a0/docs/Log2022.html" target="_blank">4c90bd8</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6dc3f9c96110c0be94a620950616ee99eaa806f0/analysis/Log2022.Rmd" target="_blank">6dc3f9c</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
add a new year
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="jun-08" class="section level2">
<h2>Jun 08</h2>
<div id="blood-trait-ldsc" class="section level3">
<h3>1. blood trait ldsc</h3>
<p>PENDING.</p>
<p><strong>Update enrichment figure error bar from se to 95%
CI.</strong></p>
</div>
<div id="autoimmune-disease-ldsc" class="section level3">
<h3>2. autoimmune disease ldsc</h3>
<div id="coloc" class="section level4">
<h4>coloc</h4>
<p>Before we look at the partitioned <span
class="math inline">\(h^2\)</span> of colocalized module in autoimmune
diseases, see below for the coloc regions of diseases and modules. Six
diseases have coloc regions, corresponding to 8 modules. Number in each
cell is the number of coloc regions.</p>
<p><img src="asset/2022/0608_immune_coloc_modules.png"
style="width:45.0%" /> <img
src="asset/2022/0608_traits.module.coloc.region.module_QTL_sig.order_by_n_coloc_trait.png"
style="width:45.0%" /></p>
</div>
<div id="the-enrichment" class="section level4">
<h4>The enrichment</h4>
<ul>
<li><p>Since there are only a few modules coloc with a disease, I ran
S-LDSC only on these coloc modules for each disease. Except Crohns’
disease (cd), I ran all 166 modules, bc this gwas has the highest coloc
proportion.</p>
<p>Note: S-LDSC throws an error running on module 156 (haven’t figured
out why). So module 156 were not analyzed.</p></li>
<li><p>Enrichment: (low resolution figures to save storage space. High
resolution figures on RCC)</p></li>
</ul>
<p><img src="asset/2022/0608_T_28067908_cd_all_modules.results.png"
style="width:50.0%" /></p>
<p><img src="asset/2022/0608_T_28067908_ibd_all_modules.results.png"
style="width:45.0%" /> <img
src="asset/2022/0608_T_29083406_Allergy_all_modules.results.png"
style="width:45.0%" /></p>
<ul>
<li>Observation:</li>
</ul>
<ol style="list-style-type: decimal">
<li><p>None of coloc modules have significant h2 enrich.</p></li>
<li><p>But module 100 looks promising for Crohns’ disease
(Enrichment=3.22, sd=1.83, p=0.23) and ibd (Enrichment=4.6, sd=2.45,
p=0.14). So, I looked into it.</p></li>
</ol>
<p>(As for why the other modules don’t have enriched h2 at all, maybe
that’s because these modules are mostly zinc finger genes? as we
discussed in <a href="Log2022.html#gene_SENP7">earlier section</a>.)</p>
</div>
<div id="module-100" class="section level4">
<h4>Module 100</h4>
<p>trans-eQTLs of module 100 include two loci:</p>
<p><img src="asset/2022/0608_M100_signals.png"
style="width:50.0%" /></p>
<ol style="list-style-type: decimal">
<li><code>4:123205143</code></li>
</ol>
<ul>
<li><p>See loci location and nearby genes <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr4%3A122352294%2D124046923&amp;hgsid=1373866789_g0iFr0hp4gIFQHP8VwAKAXnEDSpE">here</a>.</p></li>
<li><p>Near genes: KIAA1109, ADAD1, IL2, IL21.</p></li>
<li><p>This loci’s relation with immune traits?</p>
<p>Looks like this loci has been found to be associated with autoimmune
diseases back in 2007. Relevant stories include <a
href="https://www-sciencedirect-com.proxy.uchicago.edu/science/article/pii/S0002929707637768">ref
1</a>, <a
href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C14&amp;q=KIAA1109&amp;btnG=">more
stories</a>.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><code>5:82996199</code></li>
</ol>
<ul>
<li><p>See loci location and nearby genes <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr5%3A82756735%2D83104320&amp;hgsid=1373866789_g0iFr0hp4gIFQHP8VwAKAXnEDSpE">here</a>.</p></li>
<li><p>Near gene: HAPLN1.</p></li>
<li><p>This loci’s relation with autoimmune diseases?</p>
<p>I didn’t find direct link bewteen HAPLN1 and autoimmune diseases. But
this gene is relevant with NF-KB activity, see <a
href="https://www-jbc-org.proxy.uchicago.edu/article/S0021-9258(20)31707-5/fulltext">ref</a>.</p></li>
</ul>
</div>
</div>
</div>
<div id="may-27" class="section level2">
<h2>May 27</h2>
<div id="a-few-numbers-filled-up-in-manuscript" class="section level3">
<h3>1. A few numbers filled up in manuscript</h3>
<ul>
<li><p>64% of trans-eQTLs colocalize with GWAS signals of blood traits
and autoimmune diseases.</p></li>
<li><p>62% of trans-eQTLs colocalize with GWAS signals of blood
traits.</p></li>
<li><p>11% of trans-eQTLs colocalize with GWAS signals of autoimmune
diseases.</p></li>
</ul>
</div>
<div id="module-v.s.-gwas-heritability" class="section level3">
<h3>2. Module v.s. GWAS heritability</h3>
<p>Yang:</p>
<blockquote>
<p>Test whether gene modules show enrichment in specific GWAS h2g?
Especially GWAS with a coloc trans-eQTL.</p>
</blockquote>
<p>Xuanyao:</p>
<blockquote>
<p>The colocalized trans-eQTL gene modules showed significant
heritability enrichment in the colocalized complex trait.</p>
</blockquote>
<blockquote>
<p>Just to reiterate our analysis: for each co-localized gene module, we
will take all trans genes in the module, get their gene regions+500bp
buffer to each end. We will combine all gene regions into a single
annotation by using LDSC. Then we run LDSC baseline model+the gene
annotation on co-localized traits.</p>
</blockquote>
<div id="understand-ldsc" class="section level4">
<h4>understand LDSC</h4>
<ul>
<li><p>Goal: Partition the heritability of BMI by functional category,
using the baseline model.</p></li>
<li><p>Computation goal: estimate <span
class="math inline">\(\tau_C\)</span> via the multiple regression of
<span class="math inline">\(\chi^2\)</span> statistics against <span
class="math inline">\(l(j, C)\)</span>.</p></li>
<li><p>Define the enrichment of a category to be the proportion of SNP
heritability in the category divided by the proportion of SNPs in that
category.</p></li>
<li><p>Functional categories</p>
<ul>
<li><p>full baseline model’ from 24 publicly available main
annotations.</p></li>
<li><p>For each of these 24 categories, we added a 500-bp window around
the category as an additional category to keep our heritability
estimates from being inflated by heritability in flanking
regions.</p></li>
<li><p>In total, <em>53</em> functional categories, derived from 24 main
annotations.</p></li>
</ul></li>
</ul>
</div>
<div id="download-ldsc" class="section level4">
<h4>download LDSC</h4>
</div>
<div id="prepare-inputs" class="section level4">
<h4>prepare inputs</h4>
<ol start="0" style="list-style-type: decimal">
<li>Define functional categories</li>
</ol>
<ul>
<li><code>baseline.@.annot.gz</code> (updated with new annot) SNP v.s.
category, if in category</li>
</ul>
<ol style="list-style-type: decimal">
<li>LD scores (baseline model LD scores)</li>
</ol>
<ul>
<li><p><code>baseline.@.l2.ldscore.gz</code> (updated with new annot)
SNP v.s. category, ldscore</p></li>
<li><p><code>baseline.@.l2.M_5_50</code> (updated with new annot) for
each category, the number of SNPs with MAF&gt;5%</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><p>regression weights</p></li>
<li><p>allele frequencies</p></li>
<li><p>GWAS summary statistics</p></li>
</ol>
</div>
<div id="run-ldsc" class="section level4">
<h4>run LDSC</h4>
<ol start="0" style="list-style-type: decimal">
<li>run partitioned LD scores for new defined functional category and
update baseline inputs</li>
</ol>
<ul>
<li><p>Get annotation file: <code>{prefix}.${chr}.annot.gz</code> (same
order of SNPs as in <code>.bim</code>)</p>
<p>Use <code>make_annot.py</code> script to compute annot files from a
gene set.</p></li>
<li><p>calculate LD scores: <code>${prefix}.${chr}.l2.ldscore</code>,
<code>{prefix}.${chr}.l2.M_5_50</code></p></li>
<li><p>Update <code>baseline.@.annot.gz</code></p></li>
<li><p>Update <code>baseline.@.l2.ldscore.gz</code>, calculate ldscores
of SNPs for self-defined category</p></li>
<li><p>Update <code>baseline.@.l2.M_5_50</code>, calculate
baseline.@.l2.M_5_50 for self-defined category</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>run <code>munge_sumstats.py</code> to convert GWAS sum stat file to
<code>.sumstats</code> format</li>
</ol>
<blockquote>
<p>We recommend only keeping HapMap3 SNPs; to do this, you can download
a list of HapMap3 SNPs on LDSC page.</p>
</blockquote>
<ol start="2" style="list-style-type: decimal">
<li>run <code>ldsc.py</code></li>
</ol>
</div>
</div>
<div id="hapmap3-snps" class="section level3">
<h3>HapMap3 SNPs</h3>
<ul>
<li><p>For GWAS SNPs, “recommend only keeping HapMap3 SNPs”.</p></li>
<li><p><code>--w-ld-chr</code> flag gives the location of the LD scores
used for regression weights. It should be a non-partitioned
.l2.ldscore.gz file that was computed using the regression SNPs. We
recommend using HapMap3 SNPs, excluding the HLA region, as default
regression SNPs.</p></li>
</ul>
<div id="concerns" class="section level4">
<h4>Concerns</h4>
<ol style="list-style-type: decimal">
<li>Too few genes in a gene set (module)?</li>
</ol>
<p>Can lead to negative correlation between MAF and L2.</p>
<p>e.g. module 66, 42 genes.</p>
<p>In LDSC tutorial of partitioned ldsc, they have &gt;2400 genes in the
example set.</p>
</div>
</div>
</div>
<div id="apr-20-apr-27" class="section level2">
<h2>Apr 20 &amp; Apr 27</h2>
<div id="updated-figures" class="section level3">
<h3>0. Updated figures</h3>
<ul>
<li>The ratio of number of NULL SNPs used to estimate <span
class="math inline">\(\Sigma\)</span> over the module size for each
module</li>
</ul>
<p><img
src="asset/2022/0420_plot_nullSNP_module_size_pthre1e-04_ratio.png" /></p>
<p>As a result, I choose to use ratio 50 as the threshold for selecting
modules to further look at for signals.</p>
<ul>
<li>Updated signal distribution</li>
</ul>
<div class="figure">
<img src="asset/2022/0420_signal_module_chr_rm_infl_ratio_50.png"
alt="" />
<p class="caption">signal distribution on modules and chrs</p>
</div>
<p><img src="asset/2022/0420_signal_module_rm_infl_ratio_50.png"
alt="signal distribution on modules" /> <img
src="asset/2022/0420_signal_chr_rm_infl_ratio_50.png"
style="width:50.0%" alt="signal distribution on chrs" /></p>
</div>
<div id="some-numbers-of-eqtlgen-signals" class="section level3">
<h3>1. Some numbers of eQTLGen signals</h3>
<ul>
<li><p>Out of 166 modules, 129 modules are left after removing modules
with the ratio of number of NULL SNPs used to estimate <span
class="math inline">\(\Sigma\)</span> over the module size is smaller
than 50.</p></li>
<li><p>There are 8116 significant (trans-eQTL, module) pairs under
Bonferroni <span
class="math inline">\(p&lt;0.05/9918/129\)</span>.</p></li>
<li><p>Out of 129 modules, 122 have at least one trans-eQTL from ~10k
eQTLGen SNPs.</p></li>
<li><p>There are 2161 eQTLGen SNPs that are significant trans-eQTLs for
at least one module.</p></li>
</ul>
</div>
<div id="overlaps-between-trans-and-cis-esqtls" class="section level3">
<h3>2. Overlaps between trans and cis-e/sQTLs</h3>
<div id="overlap-of-10k-eqtlgen-snps-and-dgn-snps"
class="section level4">
<h4>Overlap of ~10k eQTLGen SNPs and DGN SNPs?</h4>
</div>
<div id="are-trans-eqtls-cis-esqtls-in-dgn" class="section level4">
<h4>Are trans-eQTLs cis-e/sQTLs in DGN?</h4>
<ol style="list-style-type: decimal">
<li><p>1240 trans-eQTLs (out of 2161 ) are also cis-eQTLs, corresponding
to 1302 cis-eQTL-Genes.</p></li>
<li><p>930 trans-eQTLs (out of 2161 ) are also cis-sQTLs, corresponding
to 2807 cis-sQTL-Genes.</p></li>
</ol>
<p><img src="asset/2022/0420_trans_cis_venn.png" style="width:45.0%"
alt="venn" /> <img src="asset/2022/0420_trans_cis_upset.png"
style="width:45.0%" alt="upset" /></p>
</div>
<div id="are-trans-eqtls-cis-esqtls-in-eqtlgen" class="section level4">
<h4>Are trans-eQTLs cis-e/sQTLs in eQTLGen?</h4>
</div>
</div>
<div id="eqtlgen-replication" class="section level3">
<h3>3. eQTLGen replication</h3>
<div id="how" class="section level4">
<h4>How?</h4>
</div>
<div id="dgn-replication-in-eqtlgen" class="section level4">
<h4>DGN replication in eQTLGen</h4>
<p>Out of 3899 DGN (SNP, module) signal pairs, 38 of them are also
analyzed in eQTLGen.</p>
<p>Based on these overlaps, the updated p-values threshold for defining
replication is: 0.002631579 , under FDR level 0.1.</p>
<p><strong>As a result, out of 38 DGN signals, 38 are replicated in
eQTLGen.</strong></p>
<p>The p-values of these signals in eQTLGen range: ( 0 , 8.163248e-12
).</p>
</div>
<div id="eqtlgen-replciation-in-dgn" class="section level4">
<h4>eQTLGen replciation in DGN</h4>
<ul>
<li><p>Summary</p>
<p>Out of 8116 eQTLGen (SNP, module) signal pairs, 7577 of them are also
analyzed in DGN.</p>
<p>Based on these overlaps, the updated p-values threshold for defining
replication is: 1.319784e-05, under FDR level 0.1.</p>
<p><strong>As a result, out of 7577 eQTLGen signals, 69 are replicated
in DGN.</strong></p>
<p>The p-values of these signals in DGN range: ( 0 , 1.315094e-05
).</p></li>
<li><p>Interesting signals, in terms of those in eQTLGen but not in DGN.
Particularly, those with extremely small eQTLGen p but extremely large
DGN p, <span class="math inline">\(p_{eQTLGen}=0,
p_{DGN}=1\)</span>.</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>p value comparison in eqtlgen and dgn, for replicated and not
replicated signals</li>
</ol>
<p>I looked at the pvalue differences for eqtlgen and dgn. And found
that SNPs that were not replicated mostly have extremely small eQTLGen p
and extremely large DGN p.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_rep_eqtlgen_in_dgn_p.png" alt="Figure: -log10(trans-PCO P) of eQTLGen and DGN for replicated and not replicated signals." width="70%" />
<p class="caption">
Figure: -log10(trans-PCO P) of eQTLGen and DGN for replicated and not
replicated signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_rep_eqtlgen_in_dgn_p">
Past versions of 0427_rep_eqtlgen_in_dgn_p.png
</button>
</p>
<div id="fig-0427_rep_eqtlgen_in_dgn_p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_rep_eqtlgen_in_dgn_p.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>p value differences across modules</li>
</ol>
<p>I further looked at the p value differences between eqtlgen and dgn
across modules, to see if larger modules have larger p difference? in
case eQTLGen large modules have inflated signals.</p>
<p>I didn’t see an association between the p-value difference and module
size.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_rep_eqtlgen_in_dgn_p_diff.png" alt="Figure: log10 of p difference between eQTLGen and DGN across modules for replicated and not replicated signals." width="70%" />
<p class="caption">
Figure: log10 of p difference between eQTLGen and DGN across modules for
replicated and not replicated signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_rep_eqtlgen_in_dgn_p_diff">
Past versions of 0427_rep_eqtlgen_in_dgn_p_diff.png
</button>
</p>
<div id="fig-0427_rep_eqtlgen_in_dgn_p_diff" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_rep_eqtlgen_in_dgn_p_diff.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Next, I will pick some interesting signals in terms of those in
eQTLGen but not in DGN.</p>
</div>
<div id="interesting-signal-example-1---loci-near-gene-ikzf1"
class="section level4">
<h4>Interesting signal Example 1 - loci near gene <em>IKZF1</em></h4>
<p>This loci is on Chr7, pos <code>^50\\d{6}$</code>. The nearby gene
<em>IKZF1</em> is a TF.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_IKZF1.png" alt="Figure: P values for SNPs near loci IKZF1 and their corresponding modules, for eQTLGen and DGN. SNPs ordered by freq." width="100%" />
<p class="caption">
Figure: P values for SNPs near loci IKZF1 and their corresponding
modules, for eQTLGen and DGN. SNPs ordered by freq.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_IKZF1">
Past versions of 0427_eg_snp_near_IKZF1.png
</button>
</p>
<div id="fig-0427_eg_snp_near_IKZF1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_IKZF1.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In DGN, SNPs in this loci are also identified as signals for a few
modules.</p>
</div>
<div id="interesting-signal-example-2---loci-near-gene-irf1"
class="section level4">
<h4>Interesting signal Example 2 - loci near gene <em>IRF1</em></h4>
<p>This loci is on Chr5, pos <code>^131\\d{6}$</code>. The nearby gene
<em>IRF1</em> is a TF.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_IRF1.png" alt="Figure: P values for SNPs near loci IRF1 and their corresponding modules, for eQTLGen and DGN. Modules ordered by freq." width="60%" />
<p class="caption">
Figure: P values for SNPs near loci IRF1 and their corresponding
modules, for eQTLGen and DGN. Modules ordered by freq.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_IRF1">
Past versions of 0427_eg_snp_near_IRF1.png
</button>
</p>
<div id="fig-0427_eg_snp_near_IRF1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_IRF1.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In DGN, this whole loci is missed.</p>
</div>
<div
id="interesting-signal-example-3---two-snps-356849749-and-356865776-near-gene-arhgef3"
class="section level4">
<h4>Interesting signal Example 3 - Two SNPs <code>3:56849749</code> and
<code>3:56865776</code> near gene <em>ARHGEF3</em></h4>
<p>Their p-value is 0 in eQTLGen and 1 in DGN for 53 modules.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_ARHGEF3.png" alt="Figure: Two SNPs that are signals in eQTLGen but not in DGN. And their corresponding modules." width="80%" />
<p class="caption">
Figure: Two SNPs that are signals in eQTLGen but not in DGN. And their
corresponding modules.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_ARHGEF3">
Past versions of 0427_eg_snp_near_ARHGEF3.png
</button>
</p>
<div id="fig-0427_eg_snp_near_ARHGEF3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_ARHGEF3.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>However, in DGN, these two SNPs are identified as signals for module
4. Specifically, for <code>module4:3:56849749</code> and
<code>module4:3:56865776</code>, p is 0. In fact, many other SNPs in the
loci around these two SNPs are also DGN signals and they all correspond
to module 4.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_ARHGEF3_module4_DGN.png" alt="Figure: DGN p values of signals for module 4 in loci ARHGEF3" width="80%" />
<p class="caption">
Figure: DGN p values of signals for module 4 in loci ARHGEF3
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_ARHGEF3_module4_DGN">
Past versions of 0427_eg_snp_near_ARHGEF3_module4_DGN.png
</button>
</p>
<div id="fig-0427_eg_snp_near_ARHGEF3_module4_DGN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_ARHGEF3_module4_DGN.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Module 4 was removed from eQTLGen analysis due to its low ratio
of <span class="math inline">\(\frac{\#NULL\ SNPs}{Module\
size}\)</span>.</p></li>
<li><p>Those additional SNPs in the loci are not included in eQTLGen
~10k SNPs.</p></li>
</ul>
</div>
</div>
<div id="many-signals-on-chr6" class="section level3">
<h3>4. Many signals on Chr6</h3>
<p>Signals near HLA region.</p>
</div>
<div id="cis--genes" class="section level3">
<h3>5. cis- genes</h3>
<div id="how-1" class="section level4">
<h4>How?</h4>
<ol style="list-style-type: decimal">
<li><p>Protein coding &amp; lincRNA genes</p></li>
<li><p>Three types of “near genes”: nearest, genes within 1e+5, genes
within 1e+6.</p></li>
</ol>
</div>
<div id="result-file" class="section level4">
<h4>Result file</h4>
<p>See file
<code>/scratch/midway2/liliw1/eQTGen_est_Sigma/postanalysis/signal_rm_infl_ratio_50.txt_cis_genes.txt</code>.</p>
</div>
<div id="enrichment-of-the-nearest-genes" class="section level4">
<h4>Enrichment of the nearest genes</h4>
<p>See <a href="https://biit.cs.ut.ee/gplink/l/1y5o1qk9Qk">g:Profiler
enrich here</a>.</p>
</div>
</div>
</div>
<div id="apr-13" class="section level2">
<h2>Apr 13</h2>
<div id="run-pco-on-eqtlgen" class="section level3">
<h3>1. Run PCO on eQTLGen</h3>
<div id="why" class="section level4">
<h4>Why?</h4>
<ul>
<li><p>To see if new signals in eQTLGen with much larger sample
size.</p></li>
<li><p>To see the replcation of DGN signals in eQTLGen.</p></li>
</ul>
</div>
<div id="how-to-run-pco-on-eqtlgen-zs" class="section level4">
<h4>How to run PCO on eQTLGen z’s?</h4>
<ul>
<li><p>Determine modules - 166 DGN modules.</p></li>
<li><p>Sum stat of each SNP on genes in the module.</p></li>
</ul>
</div>
<div id="problem" class="section level4">
<h4>Problem</h4>
<ul>
<li><p><span class="math inline">\(\Sigma\)</span>: no raw expression
data for estimating <span
class="math inline">\(\Sigma\)</span>.</p></li>
<li><p>eQTLGen sum stats from meta analysis, didn’t remove mappability
reads.</p></li>
</ul>
</div>
<div id="solution" class="section level4">
<h4>Solution</h4>
<ul>
<li><p>Use eQTLGen NULL z’s to estimate <span
class="math inline">\(\Sigma\)</span>. By definition.</p></li>
<li><p>Remove genes in module that are cross-map with cis-genes of the
SNP of interest (SNP-wise PCO).</p></li>
</ul>
</div>
<div id="observation" class="section level4">
<h4>Observation</h4>
<ul>
<li><p>signals claimed by Bonferroni correction, i.e. <span
class="math inline">\(0.05/166/9918\)</span></p></li>
<li><p>p-values distribution</p></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/0413_p_hist.png" alt="Figure: Distribution of p-values." width="50%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_p_hist">
Past versions of 0413_p_hist.png
</button>
</p>
<div id="fig-0413_p_hist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_p_hist.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_p_hist.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/0413_p_qq.png" alt="Figure: Distribution of p-values." width="50%" />
<p class="caption">
Figure: Distribution of p-values.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_p_qq">
Past versions of 0413_p_qq.png
</button>
</p>
<div id="fig-0413_p_qq" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_p_qq.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_p_qq.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Signals distribution for each (module, chr), module, and chr.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_signal_module_chr.png" alt="Figure: Signal distribution for pair of (module, chr)." width="100%" />
<p class="caption">
Figure: Signal distribution for pair of (module, chr).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_module_chr">
Past versions of 0413_signal_module_chr.png
</button>
</p>
<div id="fig-0413_signal_module_chr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_module_chr.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_module_chr.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/0413_signal_module.png" alt="Figure: Signal distribution on each module and chr." width="50%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_module">
Past versions of 0413_signal_module.png
</button>
</p>
<div id="fig-0413_signal_module" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_module.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_module.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/0413_signal_chr.png" alt="Figure: Signal distribution on each module and chr." width="50%" />
<p class="caption">
Figure: Signal distribution on each module and chr.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_chr">
Past versions of 0413_signal_chr.png
</button>
</p>
<div id="fig-0413_signal_chr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_chr.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_chr.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="simulation---are-eqtlgen-pco-signals-of-large-modules-inflated"
class="section level3">
<h3>2. Simulation - Are eQTLGen PCO signals of large modules
inflated?</h3>
<div id="problem-1" class="section level4">
<h4>Problem</h4>
<p>We see from the above figure the trend, which is larges modules have
more signals. There can be two possible reasons,</p>
<ul>
<li><p>Those are true signals, as larger modules with more genes could
have more trans-eQTLd.</p></li>
<li><p>Those are inflated signals (false positives).</p></li>
</ul>
</div>
<div id="i-looked-into-the-second-reason" class="section level4">
<h4>I looked into the second reason</h4>
<p>Why is it possible? Because at the very beginning, we first used the
correlation matrix of gene samples from DGN to estiamte <span
class="math inline">\(\Sigma\)</span> and use it as <span
class="math inline">\(\hat\Sigma\)</span> for eQTLGen. And we observed
all 9918 SNPs are significant, and SNPs’ p-values for each module are
almost all 0. That seemed unreasonable.</p>
<p>And we found that may be because of the use of inaccurate <span
class="math inline">\(\hat\Sigma\)</span>. So we decided to use <span
class="math inline">\(\hat\Sigma\)</span> from <span
class="math inline">\(cor(Z^{NULL})\)</span> of eQTLGen’s own null sum
stat.</p>
<p>We learned that inaccurate <span
class="math inline">\(\hat\Sigma\)</span> could inflate signals.</p>
<p>And using <span class="math inline">\(cor(Z^{NULL})\)</span> can also
lead to inaccurate <span class="math inline">\(\hat\Sigma\)</span> is
because for a module of <span class="math inline">\(K\)</span> genes, we
need to estimate a <span class="math inline">\(K \times K\)</span>
matrix, using <span class="math inline">\(Z^{NULL}\)</span> of dimension
<span class="math inline">\(m \times K\)</span> with <span
class="math inline">\(m\)</span> NULL SNPs of these genes from
eQTLGen.</p>
<p>This can be problematic for large modules with large <span
class="math inline">\(K\)</span>, because</p>
<ul>
<li><p>There are fewer SNPs with null sum stat for all <span
class="math inline">\(K\)</span> genes in the module, i.e. smaller <span
class="math inline">\(m\)</span>.</p></li>
<li><p>If <span class="math inline">\(m\)</span> is not larger enough
than <span class="math inline">\(K\)</span>, <span
class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span> is inaccurate.</p></li>
<li><p>There can be lots of extremely small p-values. Inflate PCO
signals.</p></li>
</ul>
</div>
<div id="observation-1" class="section level4">
<h4>Observation</h4>
<p>We did observe a decreasing number of null SNPs for larger
modules.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png" alt="Figure: We did observe a decreasing number of null SNPs for larger modules." width="100%" />
<p class="caption">
Figure: We did observe a decreasing number of null SNPs for larger
modules.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_moduleplot_nullSNP_module_size_pthre1e-04">
Past versions of
0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png
</button>
</p>
<div id="fig-0413_signal_moduleplot_nullSNP_module_size_pthre1e-04"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="">m/K ratio</a></p>
</div>
<div id="does-hatsigma-by-corznull-lead-to-inflated-pco-signals"
class="section level4">
<h4>Does <span class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span> lead to inflated PCO
signals?</h4>
<p>We want to check if <span class="math inline">\(\hat\Sigma\)</span>
by <span class="math inline">\(cor(Z^{NULL})\)</span> really lead to
extremely small pvalues and inflated PCO signals.</p>
<ul>
<li><p>How to check?</p>
<p>We use simulations. Specifically,</p>
<ol style="list-style-type: decimal">
<li>Choose a real module from DGN.</li>
<li>Generate sum stat (Z) of NULL SNPs from the real <span
class="math inline">\(\Sigma\)</span>.</li>
<li>Use these NULL SNPs to estimate a <span
class="math inline">\(\hat\Sigma\)</span>.</li>
<li>Run PCO on another set of NULL SNPs using the true <span
class="math inline">\(\Sigma\)</span> and <span
class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span>.</li>
<li>Compare their p-values.</li>
</ol>
<p>The p-values using true <span class="math inline">\(\Sigma\)</span>
should represent the true case. By comparing p-values using <span
class="math inline">\(\hat\Sigma\)</span> to it, we should be able to
see if <span class="math inline">\(P_{\hat\Sigma}\)</span> have extreme
values.</p></li>
<li><p>Simulation setting</p>
<ol style="list-style-type: decimal">
<li><p>I looked at multiple modules with various module size, including
module 1-11, 15, 20, 30, 40, 50, 60, 70, 90, 100, 150, 166.</p></li>
<li><p>I looked at various <span class="math inline">\(m/K\)</span>
ratios for estimating <span class="math inline">\(\hat\Sigma\)</span>,
including 1, 5, 10, 50, 100, 150.</p></li>
</ol></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_plot_hist_Sigma_DGN_module1.png" alt="Figure: Histogram of p-values using various Sigma's." width="100%" />
<p class="caption">
Figure: Histogram of p-values using various Sigma’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_plot_hist_Sigma_DGN_module1">
Past versions of 0413_plot_hist_Sigma_DGN_module1.png
</button>
</p>
<div id="fig-0413_plot_hist_Sigma_DGN_module1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_plot_hist_Sigma_DGN_module1.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_plot_qq_Sigma_DGN_module1.png" alt="Figure: QQ-plot of p-values using various Sigma's." width="70%" />
<p class="caption">
Figure: QQ-plot of p-values using various Sigma’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_plot_qq_Sigma_DGN_module1">
Past versions of 0413_plot_qq_Sigma_DGN_module1.png
</button>
</p>
<div id="fig-0413_plot_qq_Sigma_DGN_module1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_plot_qq_Sigma_DGN_module1.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observation-2" class="section level4">
<h4>Observation</h4>
<ul>
<li><p>P-values looked similar to the true case when ratio <span
class="math inline">\(&gt;50\)</span>.</p></li>
<li><p>The ratio has smaller inflation effects on smaller modules,
e.g. p-values of module 166 using <span
class="math inline">\(\Sigma\)</span> of ratio 20 are closer to that of
module 1 using <span class="math inline">\(\Sigma\)</span> of same
ratio.</p></li>
<li><p>In eQTLGen analysis, ??? modules have <span
class="math inline">\(m/K\)</span> ratio <span
class="math inline">\(&gt;50\)</span>.</p></li>
</ul>
</div>
</div>
<div id="in-dgn-larger-module-more-signals" class="section level3">
<h3>3. In DGN, larger module, more signals?</h3>
<p>We wonder if this trend also exists in DGN data.</p>
<p><span class="math inline">\(\hat\Sigma\)</span>: module size v.s.
sample size</p>
</div>
<div id="remaining-questions" class="section level3">
<h3>4. Remaining questions</h3>
<ul>
<li>Inaccurate <span class="math inline">\(\hat\Sigma\)</span> leads to
false negatives? in DGN?</li>
</ul>
</div>
</div>
<div id="mar-30" class="section level2">
<h2>Mar 30</h2>
<ol style="list-style-type: decimal">
<li>eQTLGen analysis</li>
</ol>
</div>
<div id="mar-25" class="section level2">
<h2>Mar 25</h2>
<ol style="list-style-type: decimal">
<li>Signal SNPs genomic location annotation</li>
</ol>
<ul>
<li><p>torus: GTEx</p></li>
<li><p>snpEff</p></li>
</ul>
</div>
<div id="mar-02" class="section level2">
<h2>Mar 02</h2>
<div id="about-replication-of-signals-in-eqtlgen"
class="section level3">
<h3>1. About “replication of signals in eQTLGen”</h3>
<p>In the abstract of The Biology of Genome conference, you added
something as</p>
<blockquote>
<p>The trans-eQTLs are highly replicable, and all were replicated in the
eQTLGen.</p>
</blockquote>
<p>At first, I thought you were referring to the replication of the
identified trans-eQTLs in eQTLGen. But I didn’t remember we had this
observation. So I wondered where this impression came from.</p>
<p>I think maybe because <a
href="Log2021.html#1_Are_the_dark_green_signals_eQTLGen_signals">previous
section</a>. I looked at what are those lead-SNPs of the colocalized
regions (dark green regions). Specifically, are those dark green signals
eQTLGen signals (cis- or trans- signals)?</p>
<p>I observed that, out of 74 unique lead SNPs of colocalized regions
for all blood-related traits, 65 are included in eQTLGen SNPs. And 63
are eQTLGen cis- or trans- signals.</p>
</div>
<div id="preious-eqtgen-signals-using-trans-pco" class="section level3">
<h3>2. preious eQTGen signals using trans-PCO</h3>
<table>
<thead>
<tr class="header">
<th align="left">Sigma</th>
<th align="left">#modules</th>
<th align="right">PCO_lambda_thre</th>
<th align="left">#signals</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">less</td>
<td align="right">1.0</td>
<td align="left">~367</td>
</tr>
<tr class="even">
<td align="left">DGN</td>
<td align="left">less</td>
<td align="right">0.1</td>
<td align="left">9918</td>
</tr>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">more</td>
<td align="right">1.0</td>
<td align="left">~460</td>
</tr>
<tr class="even">
<td align="left">DGN</td>
<td align="left">more</td>
<td align="right">0.1</td>
<td align="left">9905</td>
</tr>
<tr class="odd">
<td align="left">nullz</td>
<td align="left">more</td>
<td align="right">0.1</td>
<td align="left">NA</td>
</tr>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li><p>At first, we used <span
class="math inline">\(\text{PCO}_{\lambda&lt;1}\)</span> and observed
hundreds of eQTLGen SNPs that are significant trans-eQTL
signals.</p></li>
<li><p>We decided to choose <span
class="math inline">\(\text{PCO}_{\lambda&lt;0.1}\)</span> over <span
class="math inline">\(\text{PCO}_{\lambda&lt;1}\)</span>.</p></li>
<li><p>Then we observed almost all 9918 eQTLGen SNPs are trans-eQTL
signals. We doubted some signals are false positives.</p></li>
</ol>
<p><a
href="Log2021.html#Replication_of_eQTLGen_results_in_trans-PCO_of_eQTLGen_summary_stats">Replication_of_eQTLGen_results_in_trans-PCO_of_eQTLGen_summary_stats</a></p>
<p><a
href="Log2021.html#Why_there_are_so_many_eQTLGen_signals_using_PCO_(%7Blambda01%7D)">Why_there_are_so_many_eQTLGen_signals</a></p>
<ol start="4" style="list-style-type: decimal">
<li>We thought false positives occurred because we used <span
class="math inline">\(\Sigma\)</span> of DGN for estimating <span
class="math inline">\(\hat{\Sigma}\)</span> of eQTLGen. So we <a
href="Log2021.html#Build_Sigma_(hat%7BSigma%7D_%7BDGN%7D)_and_(hat%7BSigma%7D_%7Bnullz%7D)_and_check_PCO_pvalues">tried
using null zscores from eQTLGen to build sigma</a>. We observed a
significant drop of pvalues.</li>
</ol>
</div>
</div>
<div id="feb-16" class="section level2">
<h2>Feb 16</h2>
<div id="simulation" class="section level3">
<h3>1. Simulation</h3>
<ul>
<li><p>Re-run simulation using <span
class="math inline">\(\Sigma\)</span> from the updated 166 modules of
DGN_no_filter_on_mappability.</p></li>
<li><p>QQ plot of null distribution.</p>
<p>Figures are stored on server.</p>
<p><a
href="/scratch/midway2/liliw1/simulation_lambda0.1/new_Sigma/qqplot_null.png">qq
plot</a> <a
href="/scratch/midway2/liliw1/simulation_lambda0.1/new_Sigma/qqplot_null_with_hist.png">qqplot
with histogram</a></p></li>
<li><p>Power plot.</p></li>
</ul>
</div>
</div>
<div id="jan-26" class="section level2">
<h2>Jan 26</h2>
<div id="update-coloc-figures-with-less-stringent-pvalue-threshold"
class="section level3">
<h3>1. Update coloc figures with less stringent pvalue threshold</h3>
<p>Previously, we defined the green regions with module-QTL <span
class="math inline">\(p&lt;1e-8\)</span>. This threshold is not
premutation based. I looked at the permutation-based p threshold, and it
is <span class="math inline">\(\approx 3e-8\)</span>. We’d like to use a
less stringent threshold, so I chose <span
class="math inline">\(p&lt;1e-7\)</span> to define the green
regions.</p>
<p>In total, we considered three (four) types of coloc. Coloc with (1)
29 blood related traits from ukbb, (2) 11 immune traits, (3) 27 more
traits from ukbb. (The 4-th type is coloc with cis genes.)</p>
<p>I updated the figures and tables.</p>
<ul>
<li>figures</li>
</ul>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">29
blood related traits from ukbb</a>; <a
href="../docs/asset/1115_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<p><a
href="../docs/asset/2022/0126_immune_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">11
immune traits</a>; <a
href="../docs/asset/1129_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_more_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">27
more traits from ukbb</a>; <a
href="../docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<ul>
<li>tables with detailed numerical results</li>
</ul>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_coloc_region_prop_pvalThre-1e-07.txt">29
blood related traits from ukbb</a></p>
<p><a
href="../docs/asset/2022/0126_immune_traits_coloc_region_prop_pvalThre-1e-07.txt">11
immune traits</a></p>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_more_traits_coloc_region_prop_pvalThre-1e-07.txt">27
more traits from ukbb</a></p>
</div>
<div id="other-types-of-modules-coloc-with-traits"
class="section level3">
<h3>2. Other types of modules coloc with traits</h3>
</div>
</div>
<div id="jan-19" class="section level2">
<h2>Jan 19</h2>
<div id="comprehensive-analysis-of-coloc-regions-of-trait-ms"
class="section level3">
<h3>1. Comprehensive analysis of coloc regions of trait MS</h3>
<div
id="add-mss-result-to-previous-coloc-analysis-dark-green-coloc-regions"
class="section level4">
<h4>Add MS’s result to previous coloc analysis (dark green coloc
regions)</h4>
<p>See <a
href="Log2021.html#coloc-region-between-module-and-immune-trait">earlier
section</a>.</p>
</div>
<div id="dark-blue-coloc-regions" class="section level4">
<h4>dark blue coloc regions</h4>
<p>There are 4 dark blue coloc regions, i.e. coloc regions with
module-QTL lead pvalue &lt; 1e-5.</p>
<table>
<thead>
<tr class="header">
<th align="left">Region</th>
<th align="right">nSNPs</th>
<th align="right">Pval</th>
<th align="right">PP4</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module121:19:10423815</td>
<td align="right">140</td>
<td align="right">1.5e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="even">
<td align="left">module108:7:50308811</td>
<td align="right">122</td>
<td align="right">1.6e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="odd">
<td align="left">module50:7:149331042</td>
<td align="right">265</td>
<td align="right">2.0e-06</td>
<td align="right">0.94</td>
</tr>
<tr class="even">
<td align="left">module44:7:50308527</td>
<td align="right">69</td>
<td align="right">0.0e+00</td>
<td align="right">0.89</td>
</tr>
</tbody>
</table>
<p>In <a href="#make-a-story-out-of-trans-signals">earlier section</a>,
the region “module108:7:50308811” coloc with MS is included in these
dark blue regions.</p>
</div>
</div>
<div id="more-traits-from-ukbb" class="section level3">
<h3>2. More traits, from UKBB</h3>
<div id="extracted-traits" class="section level4">
<h4>Extracted traits</h4>
<ol style="list-style-type: decimal">
<li>How?</li>
</ol>
<p>I extracted specific traits from ukbb traits manifest file, based
on:</p>
<ul>
<li><p>Traits from eQTLGen traits</p></li>
<li><p>Traits from Mu et al paper</p></li>
<li><p>Powerful (and interesting) ukbb GWASs</p>
<p>Sorted by case number of EUR <code>n_cases_EUR</code>.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>What traits?</li>
</ol>
<ul>
<li><p>26 ukbb GWASs in total, including height, BMI, Cholesterol,
Hypertension, Asthma, Type 2 diabetes etc.</p></li>
<li><p>Exclude height, as not GWASed for EUR.</p></li>
<li><p>Use height sum stat from <a
href="height_BMI_meta_data_source">another meta analysis</a> instead. It
also has another BMI sum stat.</p>
<p>Meta analyze inds from GIANT and ukbb.</p>
<blockquote>
<p>Meta-analysis of genome-wide association studies for height and body
mass index in ~700,000 individuals of European ancestry.</p>
</blockquote></li>
<li><p>Thus, as a result, 27 GWASs. 22 unique traits.</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Visualize the sample sizes of selected traits</li>
</ol>
<p><img src="asset/2022/0119_phenotype_sub_N.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_phenotype_sub_N">
Past versions of 0119_phenotype_sub_N.png
</button>
</p>
<div id="fig-0119_phenotype_sub_N" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_phenotype_sub_N.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="coloc-results-more-traits-from-ukbb" class="section level4">
<h4>coloc results {More-traits-from-UKBB}</h4>
<ol style="list-style-type: decimal">
<li>Visualize</li>
</ol>
<p>See <a
href="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">figure</a>.</p>
<p><img src="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_selected_ukbb_all.traits.coloc.region.summary">
Past versions of 0119_selected_ukbb_all.traits.coloc.region.summary.png
</button>
</p>
<div id="fig-0119_selected_ukbb_all.traits.coloc.region.summary"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For detailed numbers, see <a
href="asset/2022/0119_selected_ukbb_coloc_region_prop.txt">file</a>.</p>
<ol start="2" style="list-style-type: decimal">
<li>In-depth visualization for selected traits (dark blue regions)</li>
</ol>
<p><a href="asset/2022/0119_BMI_coloc.region.summary.png">BMI</a></p>
<p><a href="asset/2022/0119_BMI-2_coloc.region.summary.png">BMI</a></p>
<p><a
href="asset/2022/0119_Cholesterol_coloc.region.summary.png">Cholesterol</a></p>
<p><a
href="asset/2022/0119_HDL-cholesterol_coloc.region.summary.png">HDL-cholesterol</a></p>
<p><a
href="asset/2022/0119_Hayfever,-allergic-rhinitis-or-eczema_coloc.region.summary.png">Hayfever,-allergic-rhinitis-or-eczema</a></p>
<p><a
href="asset/2022/0119_Hypertension_coloc.region.summary.png">Hypertension</a></p>
</div>
</div>
</div>
<div id="jan-12" class="section level2">
<h2>Jan 12</h2>
<div id="make-a-story-out-of-trans-signals" class="section level3">
<h3>1. Make a story out of the trans- signals</h3>
<div id="more-concrete-examples" class="section level4">
<h4>More concrete examples</h4>
<ol style="list-style-type: decimal">
<li>Trait, gene, trans- signal</li>
</ol>
<p>SLE signals in Fig. 5.</p>
<blockquote>
<p>reflecting the involvement of interferon signaling in SLE
pathophysiology</p>
</blockquote>
<p><img src="asset/2022/0112_Systemic lupus erythematosus_tile.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-0112_Systemic-lupus-erythematosus_tile">
Past versions of "0112_Systemic lupus erythematosus_tile.png"
</button>
</p>
<div id="fig-no-spaces-0112_Systemic-lupus-erythematosus_tile"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/664d0d7bdcedd0fb3164db435cb8bffdce9ea41e/docs/asset/2022/0112_Systemic lupus erythematosus_tile.png" target="_blank">664d0d7</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>additional examples of trans-eQTL variants associated with
traits</li>
</ol>
<ul>
<li><p>ZNF131 locus, age of menarche</p>
<p>Supplementary Figure 11A, rs1532331</p></li>
<li><p>FADS1/FADS2 locus, lipid levels</p>
<p>Supplementary Figure 11B, rs174574</p></li>
<li><p>IFIH1 locus, inflammatory bowel disease and SLE</p>
<p>Supplementary Figure 11C, rs1990760</p></li>
<li><p>GSDMB locus, asthma</p>
<p>Supplementary Figure 11D, rs7216389</p></li>
<li><p>CLOCK locus, height</p>
<p>Supplementary Figure 11E, rs1311351834</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li><p>eQTSs genes GO enrichment</p></li>
<li><p>eQTSs genes associated with HDL</p></li>
</ol>
</div>
</div>
</div>
<div id="jan-05" class="section level2">
<h2>Jan 05</h2>
<div id="trans-eqtl-and-complex-disease-in-eqtlgen-võsa-franke-et-al"
class="section level3">
<h3>1. trans-eQTL and complex disease in <a
href="https://www.nature.com/articles/s41588-021-00913-z">eQTLGen, Võsa,
Franke et al</a></h3>
<div id="question-are-there-any-trans-eqtls-that-are-gwas-hits"
class="section level4">
<h4>Question: are there any trans-eQTLs that are GWAS hits?</h4>
<p>Observations from the paper:</p>
<ul>
<li><p>Meta-analysis using up to <strong>31,684 blood samples</strong>
from 37 eQTLGen Consortium cohorts.</p></li>
<li><p>For trans, they focused on <strong>10,317 trait-associated
SNPs</strong>.</p></li>
</ul>
<p>The paper linked trans-eQTLs with traits in two ways to find the
potential driver genes for traits.</p>
<ol style="list-style-type: decimal">
<li>trans-eQTL analysis</li>
</ol>
<ul>
<li><p>One-third of trait-associated variants have distal effects.</p>
<p>Identified 59,786 trans-eQTL, representing 3,853 SNPs (<strong>37% of
tested GWAS SNPs</strong>) and 6,298 genes (32% of tested genes).</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311)
identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote></li>
<li><p>Identify genes that are coordinately affected by <strong>multiple
independent trait-associated SNPs</strong>.</p>
<p>Identified <strong>47 GWAS traits</strong> for which at least four
independent variants affected the same gene in trans (Supplementary
Tables 10). Examples genes affected by at least three SLE-associated
genetic variants.</p></li>
</ul>
<p>But, Individual trans-eQTL effects too weak to detect. Another way to
look for the potential driver genes for traits and the “core” genes:</p>
<ol start="2" style="list-style-type: decimal">
<li>eQTSs (associations between PGSs and gene expression, Fig. 6a,
6b)</li>
</ol>
<ul>
<li><p>Individual trait-associated SNPs are combined into a PGS that is
associated with gene expression.</p></li>
<li><p>when the PGS for a trait correlates with the expression of a
gene, trans-eQTL effects of individual risk variants converge on that
gene, and it can be prioritized as a putative driver of the
disease.</p></li>
<li><p>1,263 traits in total. 18,210 eQTSs representing 689 unique
traits (<strong>55% of tested traits</strong>) and 2,568 genes
(<strong>13% of tested genes</strong>).</p></li>
<li><p>Of these genes, 719 (<strong>28%</strong>) were not identified in
the trans-eQTL analysis.</p></li>
</ul>
<p><strong>Therefore, all 3,853 trans-eQTLs eQTLGen identified are GWAS
hits.</strong></p>
</div>
<div id="question-if-so-can-we-replicate-them" class="section level4">
<h4>Question: if so, can we replicate them?</h4>
<p>Among 10,317 trait-associated SNPs, 9,056 (~89%) are included in DGN
SNPs. Among 3,853 eQTLGen trans- eQTLs, <strong>27 (~0.7%) are
replicated in DGN signals (1,863,</strong> <span
class="math inline">\(p&lt;1e-8\)</span>).</p>
<p>In the paper, another trans- study was mentioned,</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311)
identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote>
</div>
</div>
<div id="gene_SENP7" class="section level3">
<h3>2. gene <em>SENP7</em></h3>
<div
id="the-cross-mappability-of-senp7-and-the-genes-in-the-trans-modules"
class="section level4">
<h4>the cross mappability of SENP7 and the genes in the trans
modules</h4>
<p>As noted in <a href="Log2021.html#what_is_gene_SENP7">earlier
section</a>, we observed,</p>
<ul>
<li><em>SENP7</em> is the neasrest gene of the lead SNPs of coloc
regions.</li>
<li>Gene <em>ZNF90P1</em> is located within <em>SENP7</em>.</li>
<li>Modules 25, 51, 153, 156 have coloc regions with immune traits and
consist of many zinc finger genes.</li>
</ul>
<p>Therefore, we wanted to know,</p>
<ol style="list-style-type: decimal">
<li>Does gene <em>ZNF90P1</em> have expression data?</li>
</ol>
<p>I looked at the expression matrix of 13634 genes. <em>ZNF90P1</em>
doesn’t have expression data in this matrix.</p>
<ol start="2" style="list-style-type: decimal">
<li>What is the cross mappability between <em>SENP7</em> and the zinc
finger genes in the modules?</li>
</ol>
<p>See files for the cross mappability for <a
href="asset/2022/0105_SENP7_module25_size123_crossMap.txt">module
25</a>, <a
href="asset/2022/0105_SENP7_module51_size61_crossMap.txt">module 51</a>,
<a href="asset/2022/0105_SENP7_module153_size15_crossMap.txt">module
153</a>, <a
href="asset/2022/0105_SENP7_module156_size13_crossMap.txt">module
156</a>.</p>
<p>The columns include: “score” for cross mappability score,
“score_map1” for mappability score of gene1, “score_map2” for
mappability score of gene2.</p>
<ol start="3" style="list-style-type: decimal">
<li>For the genes in the trans modules,</li>
</ol>
<p>See a previous <a
href="asset/1213_cd_ibd_Allergy_coloc_module_david_query.xlsx">file
here</a>.</p>
</div>
<div id="highlighting-senp7-as-a-trans-meqtl" class="section level4">
<h4>highlighting SENP7 as a trans-meQTL</h4>
<p><a
href="https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x"
class="uri">https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x</a>
Another paper highlighting SENP7 as a trans-meQTL.</p>
</div>
</div>
<div id="pqtl-resource" class="section level3">
<h3>3. pQTL resource</h3>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8       highr_0.9        bslib_0.3.1.9000 compiler_4.1.2  
 [5] pillar_1.7.0     later_1.3.0      git2r_0.29.0     jquerylib_0.1.4 
 [9] tools_4.1.2      getPass_0.2-2    digest_0.6.29    jsonlite_1.8.0  
[13] evaluate_0.15    tibble_3.1.6     lifecycle_1.0.1  pkgconfig_2.0.3 
[17] rlang_1.0.2      cli_3.1.1        rstudioapi_0.13  yaml_2.3.5      
[21] xfun_0.31        fastmap_1.1.0    httr_1.4.2       stringr_1.4.0   
[25] knitr_1.39       sass_0.4.1       fs_1.5.2         vctrs_0.3.8     
[29] rprojroot_2.0.2  glue_1.6.2       R6_2.5.1         processx_3.5.2  
[33] fansi_1.0.2      rmarkdown_2.14.2 callr_3.7.0      magrittr_2.0.3  
[37] whisker_0.4      ps_1.6.0         promises_1.2.0.1 htmltools_0.5.2 
[41] ellipsis_0.3.2   httpuv_1.6.5     utf8_1.2.2       stringi_1.7.6   
[45] cachem_1.0.6     crayon_1.4.2    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
