<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2023</title>

<script src="site_libs/header-attrs-2.14.2/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2023</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2023-03-24
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree3094ce997a05a84325f063673ce9eea4cdebacf7targetblank3094ce9a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/3094ce997a05a84325f063673ce9eea4cdebacf7" target="_blank">3094ce9</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree3094ce997a05a84325f063673ce9eea4cdebacf7targetblank3094ce9a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/3094ce997a05a84325f063673ce9eea4cdebacf7" target="_blank">3094ce9</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Untracked files:
    Untracked:  analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2023_a_fresh_start.Rmd</code>)
and HTML (<code>docs/Log2023_a_fresh_start.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3094ce997a05a84325f063673ce9eea4cdebacf7/analysis/Log2023_a_fresh_start.Rmd" target="_blank">3094ce9</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-24
</td>
<td>
add more details
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f5a165be0df7f78fcb8cbf9c5eb49cb71bb9797c/docs/Log2023_a_fresh_start.html" target="_blank">f5a165b</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/88593d492a7edee3dcdf85e9550875db6e27cec2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">88593d4</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/9ecc7a8af04512c57de40fb708845460f572f601/docs/Log2023_a_fresh_start.html" target="_blank">9ecc7a8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/7588eb75a18aee67634d632523da454d2f4cdb30/analysis/Log2023_a_fresh_start.Rmd" target="_blank">7588eb7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/analysis/Log2023_a_fresh_start.Rmd" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/Log2023_a_fresh_start.html" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/analysis/Log2023_a_fresh_start.Rmd" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/Log2023_a_fresh_start.html" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e225f5d45714ad8bf33e21a4da04a2c975cbf16e/docs/Log2023_a_fresh_start.html" target="_blank">e225f5d</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/analysis/Log2023_a_fresh_start.Rmd" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/15872511eb2e7eeb833bd01c6cc6b85473cc8c90/docs/Log2023_a_fresh_start.html" target="_blank">1587251</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/281c4602ae2a7ffa01409b9233c4b9fcee9272b2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">281c460</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
add reason why to test smaller window size
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/279fae6f6cb02c60adbf1d62cece753e127f1fc2/docs/Log2023_a_fresh_start.html" target="_blank">279fae6</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/analysis/Log2023_a_fresh_start.Rmd" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4104dc026ba662ac6d89da7b243ac22b89c55cf0/docs/Log2023_a_fresh_start.html" target="_blank">4104dc0</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/245fe228f05daceb01b1293b77c3cb5733f42c52/analysis/Log2023_a_fresh_start.Rmd" target="_blank">245fe22</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e7b284cf13b85c027f1760424017abe9745292c5/docs/Log2023_a_fresh_start.html" target="_blank">e7b284c</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6e5a414fc6364413fa987028f07d382ec32dfd7f/analysis/Log2023_a_fresh_start.Rmd" target="_blank">6e5a414</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
correct a figure path
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a6d19e33b90fd7a2db46c7e4c55f5416b62f2828/docs/Log2023_a_fresh_start.html" target="_blank">a6d19e3</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/bc0a2f7db3f7f82573f5f6297cebac528967c5c2/analysis/Log2023_a_fresh_start.Rmd" target="_blank">bc0a2f7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/92bc59e2447452c7e7f39e4c8ed4eeb3d58a319b/docs/Log2023_a_fresh_start.html" target="_blank">92bc59e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/bf7aad20b6be25071e0a04bb971b7550ed02d8ff/analysis/Log2023_a_fresh_start.Rmd" target="_blank">bf7aad2</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/cdbf8d48acae909e8dae436520fbc510ffa7fd95/docs/Log2023_a_fresh_start.html" target="_blank">cdbf8d4</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f28308c60221d21a3fadd6691b79480fd5729a17/analysis/Log2023_a_fresh_start.Rmd" target="_blank">f28308c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
<td>
Add a new wekk
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="mar-08-march-15-march-22" class="section level1">
<h1>Mar 08 &amp; March 15 &amp; March 22</h1>
<div id="quick-summary" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Figured out Ben &amp; Carlos’s steps on their histone peak QTLs
we used previously.</p></li>
<li><p>Count-based QTL</p></li>
</ul>
</div>
<div id="what-they-did-to-process-data-for-peak-qtls"
class="section level2">
<h2>1. What they did to process data for peak QTLs</h2>
<p>To work on count based QTL mapping, I need to first figure out the
exact steps Ben &amp; Carlos’s did to process the histone data and peak
QTL mapping, in order to make consistent window-based and count-based
QTL mapping. Below is a mindmap of steps, QC, normalization etc. of Ben
&amp; Carlos did to generate peak QTLs.</p>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_08_Histone_mark_reads.pdf" alt="Figure: What Ben &amp; Carlos did to process data." width="100%" />
<p class="caption">
Figure: What Ben &amp; Carlos did to process data.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_Histone_mark_reads">
Past versions of 03_08_Histone_mark_reads.pdf
</button>
</p>
<div id="fig-03_08_Histone_mark_reads" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_Histone_mark_reads.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="count-based---processing-data" class="section level2">
<h2>2. Count based - Processing data</h2>
<ul>
<li><p>Why?</p></li>
<li><p>How? A few things to be changed</p>
<p>Make pre-processing data consistent with window-based, except -</p>
<ul>
<li><p>Redefine segments on genome as .saf rather than peak calling
regions.</p></li>
<li><p>Adaptations to QC</p>
<ul>
<li><p>Remove segments without any reads</p></li>
<li><p>Remove segments without reads in 1-?% individuals (I tried 100%,
50%)</p></li>
</ul></li>
</ul></li>
</ul>
<div id="split-chrs-by-segments" class="section level3">
<h3>Split chr’s by segments</h3>
<ul>
<li><p>How?</p>
<p>Split the chromosome from 0 to chromosome length by a segment size -
1kb.</p></li>
<li><p>Number of segments across chr</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_08_num_features.pdf" alt="Figure: Number of split segments compared to the called peaks." width="50%" />
<p class="caption">
Figure: Number of split segments compared to the called peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_num_features">
Past versions of 03_08_num_features.pdf
</button>
</p>
<div id="fig-03_08_num_features" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_num_features.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="count-reads-within-each-seg" class="section level3">
<h3>Count reads within each seg</h3>
<ul>
<li><p>4 histone marks (by ChIP-seq or CutAndTag) are all paired-end
reads.</p></li>
<li><p>Count by paired-end</p>
<p><del>No <code>-p</code>. Reads instead of paired-end fragments will
be counted.</del></p>
<p>Use <code>-p</code>. <a
href="https://support.bioconductor.org/p/67534/">Count fragment
instead</a>.</p></li>
<li><p>Ways to count a read overlapped with multiple features</p>
<p>[At first, I didn’t think through this until I observed many
unassigned reads due to Unassigned_Ambiguity.]</p>
<ul>
<li><p><del>Use <code>-p -O</code> to assign the read (1) to all
overlapped features.</del> [FEATURECOUNTS PAPER SUGGESTION ON USING
_O]</p></li>
<li><p><del>Use <code>-p --largestOverlap</code> to assign the read (1)
to the feature that has the largest number of overlapping
bases.</del></p></li>
<li><p><del>Use <code>-p --fraction -O</code> to assign a fractional
count (1/y, where y is the total number of features overlapping with the
read) to all overlapped features.</del></p></li>
<li><p>Use <code>-p --read2pos 5</code>. Think a fragment as a whole,
and take the left-most base.</p></li>
<li><p><del>Use <code>--read2pos 5</code> to reduce the read to an end
base (as said by Yang, left-most-read-base to count in bam file, take
just one base, to avoid that one read in multiple regions) and count the
reduced single base.</del></p></li>
<li><p>Use <code>-p --read2pos 5 single_paired_read1_*.bam</code>. Use
paired-end, but only uses read 1. Reduce the read to
left-most-base.</p></li>
</ul></li>
</ul>
<table>
<colgroup>
<col width="34%" />
<col width="65%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">flag</th>
<th align="left">interpretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">-p</td>
<td align="left">paired-end, ignore overlap count</td>
</tr>
<tr class="even">
<td align="left">-p -O</td>
<td align="left">paired-end, assign overlap count to all regions</td>
</tr>
<tr class="odd">
<td align="left">-p –largestOverlap</td>
<td align="left">paired-end, assign overlap count to region with largest
overlap</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5</td>
<td align="left">paired-end, reduce read to left-most-base</td>
</tr>
<tr class="odd">
<td align="left">–read2pos 5</td>
<td align="left">single-end, reduce read to left-most-base</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5 single_paired_read1_*.bam</td>
<td align="left">paired-end, reduce read to left-most-base, but use only
the first read</td>
</tr>
<tr class="odd">
<td align="left">-p –read2pos 5 -O</td>
<td align="left">paired-end, reduce read to left-most-base, assign
overlap count to all regions</td>
</tr>
<tr class="even">
<td align="left">-p –read2pos 5 –largestOverlap</td>
<td align="left">paired-end, reduce read to left-most-base, assign to
largest overlap</td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_22_plt_count_mode.pdf" alt="Figure: (un)Assigned reads under various counting modes." width="50%" />
<p class="caption">
Figure: (un)Assigned reads under various counting modes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_count_mode">
Past versions of 03_22_plt_count_mode.pdf
</button>
</p>
<div id="fig-03_22_plt_count_mode" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_count_mode.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
<ul>
<li><p>Unassigned reads under
<code>-p --read2pos 5 single_paired_read1_*.bam</code>.</p>
<ul>
<li><p>These unassigned reads due to ambiguity are only for one
individual.</p></li>
<li><p>Weirdly, if I do read counting for only this individual, there is
no unassigned reads.</p></li>
<li><p>If I count read all individuals together and with <code>-R</code>
to output the detailed (un)assignment of each read, there is no
unassigned reads again. So I couldn’t check exactly which reads were
unassigned for this individual.</p></li>
<li><p>I used the above read counting results.</p></li>
</ul></li>
<li><p>Why both single-end and paired-end reads occur in one bam
file?</p>
<p>Paired-end sequencing. But those single-end reads are not aligned to
ref genome.</p></li>
</ul>
<p><strong>A few numbers - raw counts</strong></p>
<ul>
<li><p>For current exploratory phase, I only run chromosome 20, which
was split to 64,444 1kb-segments.</p>
<ul>
<li><p>62,910 (~97.62%) segments with at least one read in one
individual.</p></li>
<li><p>%individuals without any reads</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/03_08_seg_count.pdf" alt="Figure: Raw counts across segments." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_seg_count">
Past versions of 03_08_seg_count.pdf
</button>
</p>
<div id="fig-03_08_seg_count" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_seg_count.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/03_08_seg_count_zoom.pdf" alt="Figure: Raw counts across segments." width="30%" />
<p class="caption">
Figure: Raw counts across segments.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_seg_count_zoom">
Past versions of 03_08_seg_count_zoom.pdf
</button>
</p>
<div id="fig-03_08_seg_count_zoom" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_seg_count_zoom.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_08_perct_ind.pdf" alt="Figure: % Individuals with nonzero raw counts." width="30%" />
<p class="caption">
Figure: % Individuals with nonzero raw counts.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_08_perct_ind">
Past versions of 03_08_perct_ind.pdf
</button>
</p>
<div id="fig-03_08_perct_ind" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_08_perct_ind.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>I feature count 2337 peaks on chr20, and compared the count
summary with segment-version on chr20. A few potential questions needed
further understanding -</p>
<ul>
<li><p>In featureCounts, unassigned reads due to Unassigned_Duplicate,
Unassigned_NoFeatures, many Unassigned_Ambiguity.</p></li>
<li><p>There are many Unassigned_Ambiguity, because reads can fall
between adjacent segments. Solution: use <code>--read2pos</code> to
count only the left-most-base.</p></li>
</ul></li>
</ul>
</div>
<div id="qc-normalization" class="section level3">
<h3>QC &amp; Normalization</h3>
<ul>
<li><p>Remove segments without any reads for any individuals.</p></li>
<li><p>Filter segments by proportion of samples with non-zero reads -
100%, 50%.</p></li>
<li><p>Regular standardization &amp; qqnorm way v.s. leafcutter
way.</p></li>
</ul>
</div>
<div id="covariates" class="section level3">
<h3>Covariates</h3>
<p>Temporary workaround - Use PC’s of peaks.</p>
</div>
<div id="qtltools" class="section level3">
<h3>QTLTools</h3>
<p>Chr20 for now. All cis-SNPs around 500kb of each segment.</p>
</div>
</div>
<div id="count-based---perform-pco-pick-one-region"
class="section level2">
<h2>3. Count based - Perform PCO (pick one region)</h2>
<div id="define-segment-groups" class="section level3">
<h3>Define segment groups</h3>
<ul>
<li><p>Window size - 10kb</p></li>
<li><p>Segment size - 1kb</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_22_plt_seg_group_size.pdf" alt="Figure: Segment group size under 10kb window size across prop_nonzero (1, 0.5)." width="50%" />
<p class="caption">
Figure: Segment group size under 10kb window size across prop_nonzero
(1, 0.5).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_seg_group_size">
Past versions of 03_22_plt_seg_group_size.pdf
</button>
</p>
<div id="fig-03_22_plt_seg_group_size" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_seg_group_size.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations</strong> -</p>
<ul>
<li><p>#groups used for PCO (remove those groups with 1 seg)</p></li>
<li><p>group size v.s. sample size (72)</p>
<p>Group size controlled under 10, for seg_size = 1kb,
window_size=10kb</p></li>
<li><p>Is 1kb too wide for segment size?</p></li>
</ul>
</div>
<div id="perform-pco" class="section level3">
<h3>Perform PCO</h3>
<ul>
<li><p>P for across prop_nonzero (1, 0.5) &amp; compare to peak
group.</p></li>
<li><p>I expect seg_pco to be similar to peak_pco, as these results are
from mark H3K27ac with narrow peaks.</p>
<ul>
<li>I expect seg_pco to be powerful for marks with broad peaks.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/03_22_plt_smallp.pdf" alt="Figure: Comparison of small p of segment group size under 10kb window size across prop_nonzero (1, 0.5)." width="50%" />
<p class="caption">
Figure: Comparison of small p of segment group size under 10kb window
size across prop_nonzero (1, 0.5).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_22_plt_smallp">
Past versions of 03_22_plt_smallp.pdf
</button>
</p>
<div id="fig-03_22_plt_smallp" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/03_22_plt_smallp.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="feb-24-feb-29" class="section level1">
<h1>Feb 24 &amp; Feb 29</h1>
<div id="quick-summary-1" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Locus zoom of a region to visualize PCO and MinP
associations</p></li>
<li><p>Genome track by genotype</p></li>
</ul>
</div>
<div id="compare-ps-of-specific-association-pairs-by-minp-and-pco"
class="section level2">
<h2>1. Compare p’s of specific association pairs by MinP and PCO</h2>
<p>Change the way of visualization -</p>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare MinP and PCO." width="50%" />
<p class="caption">
Figure: To compare MinP and PCO.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_24_plt_small_p_pco_acat_methods_diag_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="locus-zoom-of-a-peak-group-cis-region" class="section level2">
<h2>2. Locus zoom of (a peak group, cis region)</h2>
<div id="why" class="section level3">
<h3>Why?</h3>
<p>Significant PCO associations, weak ACAT associations.</p>
<p>For follow up analysis, e.g. coloc with traits.</p>
<p>But since we are not comparing to ACAT associations, is this analysis
still necessary? (Does it really make sense to compare with MinP?)</p>
<p>For coloc purpose, improve detection of mediate effects. How does it
help with coloc power? or anything else?</p>
</div>
<div id="how" class="section level3">
<h3>How?</h3>
<p>For a peak group, cis region. Two methods: PCO &amp; MinP.</p>
<ul>
<li><p>Issues</p>
<ul>
<li><p><del><code>locuscomparer</code> not available to install on
midway3.</del></p></li>
<li><p><del>Convert SNP position to rsid.</del></p></li>
<li><p><strong>~20% SNPs no rsid?</strong></p></li>
</ul></li>
</ul>
</div>
<div id="example-regions" class="section level3">
<h3>Example regions</h3>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf" alt="Figure: Locus compare MinP and PCO." width="30%" />
<p class="caption">
Figure: Locus compare MinP and PCO.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_">
Past versions of 02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf
</button>
</p>
<div id="fig-02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_24_G55137_chr20_2_peaks_0_4588_snps_10kb_.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>To select regions we want?</li>
</ul>
</div>
</div>
<div id="genometracks-by-genotype" class="section level2">
<h2>3. Genometracks by genotype</h2>
<div id="issues-i-ran-into" class="section level3">
<h3>Issues I ran into</h3>
<ul>
<li><p>Use the new main function</p></li>
<li><p>Use the wrong flag - unknown snp led to wrong genotype
group</p></li>
<li><p>Use the correct template</p>
<p><code>GeneralPurposeColoredByGenotype.ini</code></p>
<p><code>GeneralPurposeColoredByGenotypeWithPerIndCoverage.ini</code></p></li>
<li><p>Correct template</p>
<p>Change to the correct template.</p>
<p>Minor - change the y max value to
<code>PerGroupMaxPerInd</code></p></li>
</ul>
</div>
<div id="example-regions-1" class="section level3">
<h3>Example regions</h3>
<div class="figure" style="text-align: center">
<img src="asset/2023/1.pdf" alt="Figure: Genome tracks." width="50%" />
<p class="caption">
Figure: Genome tracks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1">
Past versions of 1.pdf
</button>
</p>
<div id="fig-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/1.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2023/1.pdf">Format 1</a></p>
<p><a href="asset/2023/2.pdf">Format 2</a></p>
<ul>
<li>To select regions we want?</li>
</ul>
</div>
</div>
<div id="ideas-from-reading" class="section level2">
<h2>4. Ideas from reading</h2>
<ul>
<li><p>Define sliding windows with skip</p>
<p><a href="https://doi.org/10.1038/ng.2671"
class="uri">https://doi.org/10.1038/ng.2671</a> NA Something I learned
that can be used in my cQTL work - Define sliding windows with skip by
sliding the window based on both (1) sliding across at least one peak
(the current way), (2) skip a gap window. Adding a gap sliding distance
can decrease the degree of overlapping between two adjacent peak groups
and reduce the number of tests. Or to be more precisely, use an overlap
fraction.</p></li>
<li><p>The idea of using overlap fraction to merge our significant peak
groups.</p></li>
</ul>
</div>
</div>
<div id="feb-17" class="section level1">
<h1>Feb 17</h1>
<div id="quick-summary-2" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Evaluate PCO_ACAT</p></li>
<li><p>Apply PCO_ACAT to all chr’s and window sizes of sliding windows
with the goal to analyze data</p></li>
<li><p>Thoughts &amp; insights on previous results after reading ACAT
more carefully</p></li>
</ul>
</div>
<div id="evaluate-pco_acat" class="section level2">
<h2>1. Evaluate PCO_ACAT</h2>
<p>On non-overlapping windows as described in <a
href="#non_overlapping_window">previous progress</a>.</p>
<div id="v.s.-pco" class="section level3">
<h3>v.s. PCO</h3>
<p>with the goal to see if PCO_ACAT works.</p>
<ul>
<li>Compare P value distribution</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare PCO and PCO_ACAT. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare PCO and PCO_ACAT. Color: methods. Panel: window size.
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of
(SNP, peak group),(C) Density of (SNP, peak group) with corresponding P.
These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_15_plt_compare_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>A more rigorous way to compare distributions?</p></li>
<li><p>Compare small p’s of window sizes</p></li>
</ul>
<p><a
href="asset/2023/02_15_plt_compare_small_p_pco_acat_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare window sizes. Color: window sizes. (A)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP,
peak group) associations with corresponding P. (B) x-axis: -log10(small
P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations
with corresponding P. (C) Numbers of tests across window sizes. These
p’s are from all chr20-22.</a></p>
<ul>
<li>Compare specific association pairs</li>
</ul>
<p><a
href="asset/2023/02_15_plt_compare_small_p_pco_acat_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li>Computation time</li>
</ul>
</div>
<div id="v.s.-acatminp" class="section level3">
<h3>v.s. ACAT/MinP</h3>
<p>with the goal to see if there is any improvement by PC-based
method.</p>
<ul>
<li>Compare P value distribution</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare PCO_ACAT and ACAT?MinP. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare PCO_ACAT and ACAT?MinP. Color: methods. Panel: window
size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number
of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding
P. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1fe0e18bdd092c47b65b91a134b1378bdaa7d9ab/docs/asset/2023/02_15_plt_small_p_pco_acat_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">1fe0e18</a>
</td>
<td>
liliw-w
</td>
<td>
2023-03-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare small p’s of window sizes</li>
</ul>
<p><a
href="asset/2023/02_15_plt_small_p_pco_acat_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare window sizes. Color: window sizes. (A)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP,
peak group) associations with corresponding P. (B) x-axis: -log10(small
P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations
with corresponding P. (C) Numbers of tests across window sizes. These
p’s are from all chr20-22.</a></p>
<ul>
<li>Compare specific association pairs</li>
</ul>
<p><a
href="asset/2023/02_15_plt_small_p_pco_acat_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li>Computation time</li>
</ul>
</div>
</div>
<div
id="apply-to-all-chrs-and-window-sizes-of-non-overlapping-sliding-windows-to-analyze-data"
class="section level2">
<h2>2. Apply to all chr’s and window sizes of
(<del>non-overlapping</del> / sliding windows) to analyze data</h2>
<ul>
<li><p>Done.</p></li>
<li><p>Computation time</p>
<p>For <span class="math inline">\(22\times9\times2\)</span> …</p></li>
</ul>
</div>
<div id="thoughts-insgihts-after-read-acat-more-carefully"
class="section level2">
<h2>3. Thoughts &amp; insgihts after read ACAT more carefully</h2>
<p>Thoughts on previous results/observations/questions/concerns -</p>
<ul>
<li><p>Why did we observe similar p value distribution by ACAT and MinP
(corrected by Bonferroni)</p>
<p>As we discussed in ACAT paper JC, ACAT can be dominated by the very
small p values.</p>
<p>MinP corrected by Bonferroni and MinP with p values calculated by
computational heavy permutation are similar.</p></li>
<li><p>Why did we observe less improvements by PCO than ACAT/MinP than
expected?</p>
<ul>
<li><p>ACAT/MinP is not a good/common standard to compare with (as said
by Yang), as (1) it is already powerful, (2) it is not what we wanted to
improve on.</p></li>
<li><p>We should compare with univariate methods, which is commonly used
and less improved and less powerful.</p></li>
<li><p>Not to compare the number of SNPs, as not significant biological
meaning and hard to implement. Instead, we can use other ways, e.g. if
improved coloc signals with traits, etc.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="feb-08" class="section level1">
<h1>Feb 08</h1>
<div id="quick-summary-3" class="section level2">
<h2>Quick summary</h2>
<ul>
<li>Tested (SNP, peak group) of groups defined by non-overlapping
windows.</li>
</ul>
</div>
<div id="non_overlapping_window" class="section level2">
<h2>1. Groups defined by non-overlapping windows</h2>
<div id="why-discussion-on-slack" class="section level3">
<h3>Why? (discussion on slack)</h3>
<p>We observed many improved associations of PCO than ACAT/MinP in <a
href="#p_distribution_across_methods_and_window_sizes">previous
section</a>. One potential issue is these improved associations could
possibly come from a small set of associations, due to many repetitive
peaks &amp; SNPs by sliding window, which appears to be a large set of
associations.</p>
<p>So, we need to look into the case using non-overlapping windows, just
to make sure the PCO improvement over minP is still there after avoiding
repetitive counting of associations.</p>
<ul>
<li><p>Four points we discussed in Tuesday’s meeting</p>
<ul>
<li><p>Why coloc style and MACS2 style not work in this case?</p>
<p>Because these two generate noncontinuous genomic regions, whereas
what we need is to divide the whole genome into non-overlapping
chunks.</p></li>
<li><p>Why would simple useage of non-overlapping windows not make sense
in final version results?</p>
<p>Because choosing the start and end of the dividing window seems
arbitrary.</p></li>
<li><p>It only makes sense if we want to take a rough look at PCO v.s.
ACAT/MinP to avoid repetitive counting of signals in preliminary
results.</p></li>
<li><p>To get final results, we should go back to sliding window + MACS2
merge.</p>
<p>Calculate p-value for groups by sliding window, then merge. Maybe
then test again to obtain updated p for merged group?</p></li>
</ul></li>
</ul>
</div>
<div id="how-1" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Define non-overlapping windows</p>
<p>Dividing the genomic locations starting from 0 into non-overlapping
windows of a given window size.</p></li>
</ul>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<ul>
<li><p>Compare small p’s of three methods - Overall summary figures</p>
<ul>
<li>Panel B shows the advantage of PCO persists under window size 25kb
or smaller.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare three methods under non-overlapping windows. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare three methods under non-overlapping windows. Color:
methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5.
y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak
group) with corresponding P. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/asset/2023/02_08_nonoverlap_window_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Compare small p’s of three methods - Scatter plot</p>
<p><strong>This is an important figure, but it’s too large to show
explicitly.</strong></p>
<ul>
<li><p>For a pair of (SNP, group), PCO improves p-value than ACAT/MinP
under window size 25kb or smaller.</p></li>
<li><p>For pair of (SNP, group) with p around 1e-5 by ACAT/MinP
(i.e. the middle part), some pairs have improved p by PCO. (These pairs
are likely the additionally identified signals by PCO.)</p></li>
<li><p>For pair of (SNP, group) with small p by ACAT/MinP (i.e. the
right part) …</p></li>
<li><p>PCO p in large, middle, and extreme small window sizes …</p></li>
</ul></li>
</ul>
<p><a
href="asset/2023/02_08_nonoverlap_window_plt_small_p_methods_point_H3K27AC_window_all_cis500kb_chr20-22.pdf"
width="50%">Figure: To compare each (SNP, group) association by three
methods under non-overlapping windows. x-axis: (SNP, group)
associations, with small P &lt; 1e-5. Point is a (SNP, group)
association. y-axis: -log10(small P). Color: methods. Panel: window
size. These p’s are from all chr20-22.</a></p>
<ul>
<li><p>Compare small p’s of window sizes</p>
<ul>
<li>PCO better or ACAT/MinP worse?</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare window sizes under non-overlapping windows. Color: window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group) associations with corresponding P. (B) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations with corresponding P. (C) Numbers of tests across window sizes. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare window sizes under non-overlapping windows. Color:
window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis:
density of (SNP, peak group) associations with corresponding P. (B)
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak
group) associations with corresponding P. (C) Numbers of tests across
window sizes. These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0a7c2b7d4d9ae8dab31b3d22aa23fa75e31717ee/docs/asset/2023/02_08_nonoverlap_window_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0a7c2b7</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-09
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="feb-01" class="section level1">
<h1>Feb 01</h1>
<div id="quick-summary-4" class="section level2">
<h2>Quick summary</h2>
<ul>
<li><p>Compare small p, instead of top p, across three methods &amp;
window sizes</p></li>
<li><p>Try even more extreme window sizes</p></li>
<li><p>Example associations (by R not genome tracks)</p></li>
</ul>
</div>
<div
id="compare-methods-and-window-sizes-using-small-p-rather-than-top-p"
class="section level2">
<h2>1. Compare methods and window sizes using small p rather than top
p</h2>
<div id="why-1" class="section level3">
<h3>Why</h3>
</div>
<div id="how-2" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Available results</p>
<p>Methods - ACAT, MinP, PCO.</p>
<p>Window size - 10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p>
<p>Chr - 20, 21, 22.</p></li>
<li><p>Small p value cut off: 1e-5</p>
<p>For a pair of (SNP, group), keep its association for comparing if p
by at least one method (ACAT, MinP, PCO) &lt; 1e-5.</p></li>
</ul>
</div>
<div id="results-1" class="section level3">
<h3>Results</h3>
<ol style="list-style-type: decimal">
<li>Compare small p’s of three methods</li>
</ol>
<p>[See below section for updated results]</p>
<p><strong>Observation</strong></p>
<ul>
<li><p>Why the density line of PCO and minP under 10kb look so different
from the density lines below?</p>
<p>It’s because the smoothing algorithm to estimate the line for
“density” (Figure C) and “histogram” (Figure A) is different. So the two
lines have subtle difference.</p>
<p>To avoid this, I changed the visualization from using a line of
histogram (Figure A) to using histogram itself (Figure B).</p></li>
<li><p>Figure B shows that PCO has smaller P’s than MinP under 10kb
window.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Compare small p’s of window sizes</li>
</ol>
<p>[See below section for updated results]</p>
</div>
</div>
<div id="try-even-more-extreme-window-sizes" class="section level2">
<h2>2. Try even more extreme window sizes</h2>
<div id="why-2" class="section level3">
<h3>Why</h3>
</div>
<div id="how-3" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Available results</p>
<p>Methods - ACAT, MinP, PCO.</p>
<p>Window size - <strong>1kb, 2.5kb, 5kb</strong>, 10kb, 25kb, 50kb,
100kb, 260kb, 500kb.</p>
<p>Chr - 20, 21, 22.</p></li>
</ul>
</div>
<div id="p_distribution_across_methods_and_window_sizes"
class="section level3">
<h3>Results</h3>
<ul>
<li>Compare small p’s of three methods</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare three methods. Color: methods. Panel: window size. x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of (SNP, peak group),(C) Density of (SNP, peak group) with corresponding P. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare three methods. Color: methods. Panel: window size.
x-axis: -log10(small P), small P &lt; 1e-5. y-axis: (A-B) number of
(SNP, peak group),(C) Density of (SNP, peak group) with corresponding P.
These p’s are from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare small p’s of window sizes</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" alt="Figure: To compare window sizes. Color: window sizes. (A) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group) associations with corresponding P. (B) x-axis: -log10(small P), small P &lt; 1e-5. y-axis: number of (SNP, peak group) associations with corresponding P. (C) Numbers of tests across window sizes. These p's are from all chr20-22." width="50%" />
<p class="caption">
Figure: To compare window sizes. Color: window sizes. (A) x-axis:
-log10(small P), small P &lt; 1e-5. y-axis: density of (SNP, peak group)
associations with corresponding P. (B) x-axis: -log10(small P), small P
&lt; 1e-5. y-axis: number of (SNP, peak group) associations with
corresponding P. (C) Numbers of tests across window sizes. These p’s are
from all chr20-22.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22">
Past versions of
02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b796c451d6d02c04943554d72fb517c0285f89ca/docs/asset/2023/02_01_plt_small_p_windows_H3K27AC_window_all_cis500kb_chr20-22.pdf" target="_blank">b796c45</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Aggregate all p values across chr’s &amp; window sizes and calculate
stats: #groups, #tests</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf" alt="Figure: " width="40%" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22">
Past versions of
02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf
</button>
</p>
<div id="fig-02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_all_p_stats_H3K27AC_windowall_cis500kb_chr20-22.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Number of groups decreases as window size decreases</p></li>
<li><p>Number of (SNP, group) tests appears as U-shape</p></li>
<li><p>Use a lower p cutoff</p>
<p>P values lower than 1e-8, 1e-10 will be included.</p></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf" alt="Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10." width="50%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-08.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf" alt="Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10." width="50%" />
<p class="caption">
Figure: LEFT: P cutoff 1e-8. RIGHT: P cutoff 1e-10.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10">
Past versions of
02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf
</button>
</p>
<div
id="fig-02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_plt_small_p_methods_H3K27AC_window_all_cis500kb_chr20-22_pthre1e-10.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation</strong></p>
</div>
</div>
<div id="example-associations-to-visualize-p_pco-and-p_acat"
class="section level2">
<h2>3. Example associations to visualize P_pco and P_acat</h2>
<div id="goal" class="section level3">
<h3>Goal</h3>
<p>“For example, pick an example where PCO improved pvalue and pick
another example where PCO doesn’t improve pvalue?”</p>
<ul>
<li><p><del>sashimi plots</del></p></li>
<li><p>GenometracksByGenotype - pyGenomeTracks by genotype</p>
<p>(Not doable yet)</p></li>
</ul>
</div>
<div id="another-way" class="section level3">
<h3>Another way</h3>
<ul>
<li><p>Criteria to choose (SNP, group) to visualize for PCO and ACAT</p>
<ul>
<li><p>Enough large difference: -log(P_pco/P_acat) &gt; 5</p></li>
<li><p>At least one weak association: max(P_pco, P_acat) &gt;
1e-5</p></li>
<li><p>Non-inf values: Non-zero P_pco</p></li>
</ul></li>
<li><p>Window - 100kb; Chr - 20-22</p></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf" alt="Figure: " width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5">
Past versions of
02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf
</button>
</p>
<div
id="fig-02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G58821_22-21484558-CT-C_7_0_5.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf" alt="Figure: " width="20%" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5">
Past versions of
02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf
</button>
</p>
<div
id="fig-02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0feec1286c70b68058407e1b9c1249b33d8ed83c/docs/asset/2023/02_01_eg_H3K27AC_window100kb_cis500kb_G57611_21-8836656-C-CAA_32_0_-5.pdf" target="_blank">0feec12</a>
</td>
<td>
liliw-w
</td>
<td>
2023-02-01
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="jan-25" class="section level1">
<h1>Jan 25</h1>
<div id="quick-summary-5" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Test peak group defined by smaller window sizes using ACAT, minP,
PCO.</li>
</ul>
</div>
<div id="try-smaller-window-sizes" class="section level2">
<h2>1. Try smaller window sizes</h2>
<div id="why-3" class="section level3">
<h3>Why?</h3>
<ul>
<li><p>We observed from last week’s results that, it seems that PCO’s p
move toward ACAT/minP p when window sizes decrease.</p></li>
<li><p>Groups of smaller window sizes that have multiple peaks tend to
have physically closer peaks, thus more shared genetic control.</p></li>
</ul>
<p>Therefore, we wanted to check if that pattern persists under smaller
window sizes. So, I tried extreme window sizes, 10kb, 25kb, 50kb.</p>
</div>
<div id="how-4" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Window sizes</p>
<p>10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p></li>
<li><p>Test groups with size &gt; 1</p>
<p>Smaller window sizes can have less groups tested.</p></li>
<li><p>Test overlapped cis-SNPs within 500kb of peaks</p></li>
<li><p>Available results</p>
<p>window size - 10kb, 25kb, 50kb, 100kb, 260kb, 500kb.</p>
<p>chr - 20, 21, 22.</p></li>
</ul>
</div>
<div id="results-2" class="section level3">
<h3>Results</h3>
<ul>
<li>Compare methods under same window size</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf" alt="Figure: Scatter plot of top p by three methods under same window size on chr21. x-axis: Each peak group. y-axis: -log10(P). Color show methods. Panel show window sizes." width="50%" />
<p class="caption">
Figure: Scatter plot of top p by three methods under same window size on
chr21. x-axis: Each peak group. y-axis: -log10(P). Color show methods.
Panel show window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_methods_scatter_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf" alt="Figure: Distribution density of top p by three methods under same window size on chr21. x-axis: -log10(P). y-axis: density of peak group with corresponding P. Color show methods. Panel show window sizes. Same total number of groups within each panel." width="50%" />
<p class="caption">
Figure: Distribution density of top p by three methods under same window
size on chr21. x-axis: -log10(P). y-axis: density of peak group with
corresponding P. Color show methods. Panel show window sizes. Same total
number of groups within each panel.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_methods_density_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>We observe that PCO seems to similar or even more small p’s under
smaller window sizes. The question is - Is it because PCO under small
windows better or ACAT/minP worse?</p>
<ul>
<li>Compare window sizes of same method</li>
</ul>
<p>(more complicated as different numbers of groups tested)</p>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf" alt="Figure: Distribution of top p across window sizes by same method on chr21. x-axis: -log10(P). y-axis: (A) density, (B) number of peak group with corresponding P. Color show window sizes. Panel show methods. (C) Different numbers of groups (size &gt; 1) tested for window sizes in each panel." width="50%" />
<p class="caption">
Figure: Distribution of top p across window sizes by same method on
chr21. x-axis: -log10(P). y-axis: (A) density, (B) number of peak group
with corresponding P. Color show window sizes. Panel show methods. (C)
Different numbers of groups (size &gt; 1) tested for window sizes in
each panel.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21">
Past versions of
01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff0daa8357e5c2aa80f9bd5d2d6f1dc031658832/docs/asset/2023/01_25_plt_top_p_windows_density_H3K27AC_window_all_cis500kb_chr21.pdf" target="_blank">ff0daa8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-25
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Q - Larger window, higher proportion of small p, for ACAT/minP not
pco, why?</p>
<ul>
<li><p>A fair way to compare window sizes?</p></li>
<li><p>Top p of matching groups by three methods across window
sizes</p></li>
<li><p>Others</p>
<ul>
<li><p>Group size tested</p></li>
<li><p>Number of cis-SNPs tested</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="pco-acat" class="section level2">
<h2>2. PCO + ACAT</h2>
</div>
</div>
<div id="jan-18" class="section level1">
<h1>Jan 18</h1>
<div id="quick-summary-6" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Compare across window sizes</p></li>
<li><p>PC-based test on (SNP, peak group)</p></li>
</ul>
</div>
<div id="determine-window-size" class="section level2">
<h2>1. Determine window size</h2>
<div id="use-all-p-not-just-top-p" class="section level3">
<h3>Use all p not just top-p</h3>
<ul>
<li>Compare p using ACAT and minp</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: ." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: ." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: ." width="30%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb">
Past versions of
01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare p by ACAT across window sizes (different total number of
tests)</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf" alt="Figure: ." width="60%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_window_size_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_window_size_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_window_size_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Distribution of small p under same number of peaks in group &amp;
same number of cpeaks in group</p>
<p>small_p_thre &lt;- 1e-8</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf" alt="Figure: ." width="40%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_same_peak_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Percentage of cPeaks in group with small p under same number of
peaks in group &amp; same number of cpeaks in group</p>
<p>small_p_thre &lt;- 1e-8</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf" alt="Figure: ." width="40%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb">
Past versions of
01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_perct_cpeak_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of groups</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf" alt="Figure: ." width="40%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb">
Past versions of 01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_group_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of cis-SNPs</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf" alt="Figure: ." width="40%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb">
Past versions of
01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_small_p_cis_snp_same_peak_H3K27AC_cis500kb.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="determine-test" class="section level2">
<h2>2. Determine test</h2>
<div id="pc-based" class="section level3">
<h3>PC-based</h3>
<ul>
<li><p>phenotype</p></li>
<li><p>genotype</p></li>
<li><p>covaraites</p>
<p>H3K27AC: 14 PCs</p></li>
<li><p>Sigma</p>
<p>Number of peaks v.s. number of samples (73)</p></li>
<li><p><span class="math inline">\(\lambda&gt;0.1\)</span></p></li>
<li><p>Available results</p>
<p>Window size: 100kb, 250kb, 500kb.</p>
<p>Chr: 21, 22.</p></li>
<li><p>Comparison of top p across three methods</p>
<ul>
<li>Scatter plot</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf" alt="Figure: ." width="30%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Distribution of top p</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf" alt="Figure: ." width="30%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_density_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p v.s. group size</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf" alt="Figure: ." width="80%" />
<p class="caption">
Figure: .
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21">
Past versions of
01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf
</button>
</p>
<div
id="fig-01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/775e3b8f7184425cd2932a349e430501943edd92/docs/asset/2023/01_18_plt_top_p_pco_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb_chr21.pdf" target="_blank">775e3b8</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="compare-multi-cqtls-with-uni-cqtls" class="section level2">
<h2>3. Compare multi-cQTLs with uni-cQTLs</h2>
<p>After discussion on slack, we decided <strong>not to use uni-cQTLs as
a comparison metric for now</strong>. Reasons -</p>
<ul>
<li><p>Comparison with uni-cQTLs not necessary in the long run for
biological interpretations.</p></li>
<li><p>Comparison with minp is sufficient for now</p></li>
<li><p>Other ways for comparison of multi-cQTLs on top of the # of
significant SNPs</p></li>
</ul>
</div>
</div>
<div id="jan-11" class="section level1">
<h1>Jan 11</h1>
<div id="quick-summary-7" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Make the pipeline automatic.</p></li>
<li><p>Tested (SNP, peak group) by ACAT at 100kb, 250kb, 500kb window
size.</p></li>
<li><p>Compare multi-cQTLs across window sizes and <del>with
uni-cQTLs</del>.</p></li>
</ul>
</div>
<div
id="define-peak-groups-under-different-window-sizes-and-test-snp-peak-group"
class="section level2">
<h2>1. Define peak groups under different window sizes and test (SNP,
peak group)</h2>
<p>Last time, I tested (SNP, peak group) of peak groups defined by 500kb
window sizes using SNPs within 500kb of peak group.</p>
<p>This time, I tried various window sizes, including 100kb, 250kb,
500kb.</p>
<div id="how-5" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Peaks are considered as a peak group using a sliding window of
sizes 100kb, 250kb, 500kb.</p></li>
<li><p>Use 500kb as the cis distance to search for the cis-SNPs of each
individual peaks in the group, then tested the overlapped SNPs across
all peaks in the group. [Note: we discussed on slack to use 500kb as cis
distance for all window sizes.]</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_choose_cis_dis.jpeg" alt="Figure: Choose cis distance for different window size." width="40%" />
<p class="caption">
Figure: Choose cis distance for different window size.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_choose_cis_dis">
Past versions of 01_11_choose_cis_dis.jpeg
</button>
</p>
<div id="fig-01_11_choose_cis_dis" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_choose_cis_dis.jpeg" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>ACAT test.</li>
</ul>
</div>
<div id="numbers" class="section level3">
<h3>Numbers</h3>
<p>Use a table or histogram …</p>
</div>
<div id="results-3" class="section level3">
<h3>Results</h3>
<ul>
<li>Number of peaks in groups</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_peaks_in_group_H3K27AC_window100kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window100kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window100kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window100kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window100kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_peaks_in_group_H3K27AC_window250kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window250kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window250kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window250kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window250kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_peaks_in_group_H3K27AC_window500kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="20%" />
<p class="caption">
Figure: Number of peaks in peak groups defined by different window
sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_H3K27AC_window500kb">
Past versions of 01_11_peaks_in_group_H3K27AC_window500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_H3K27AC_window500kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_H3K27AC_window500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Number of peaks in groups colored by cPeaks (uni)</p>
<p>[Note: as we discussed on slack, I used the results of cPeaks within
500kb cis distance.]</p></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window100kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window250kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf" alt="Figure: Number of peaks in peak groups defined by different window sizes." width="30%" />
<p class="caption">
Figure: Number of peaks in peak groups defined by different window
sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb">
Past versions of
01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf
</button>
</p>
<div id="fig-01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_peaks_in_group_w_cqtl_H3K27AC_window500kb_cpeak500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Peak groups on chr</li>
</ul>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window100kb.pdf">100kb
window</a></p>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window250kb.pdf">250kb
window</a></p>
<p><a href="asset/2023/01_11_peaks_on_chr_H3K27AC_window250kb.pdf">500kb
window</a></p>
<ul>
<li>Number of cPeaks in peak groups</li>
</ul>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window100kb_cpeak500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window250kb_cpeak500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_peaks_w_cqtl_in_group_H3K27AC_window500kb_cpeak500kb.pdf">500kb
window</a></p>
<ul>
<li>Distribution of the top p-value of a peak group</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_density_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s. number of peaks in the group</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s. number of cPeaks in the group</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_at_peaks_w_cqtl_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group on chromosomal position</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="35%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="35%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="35%" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group on chromosomal position v.s number of cPeaks
in the group</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window100kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window250kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf" alt="Figure: Distribution of top p-values across peak groups." width="30%" />
<p class="caption">
Figure: Distribution of top p-values across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb">
Past versions of
01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_peak_group_manhattan_cqtl_H3K27AC_window500kb_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p of a peak group v.s chromosomes</li>
</ul>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window100kb_cis500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window250kb_cis500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_chr_H3K27AC_window500kb_cis500kb.pdf">500kb
window</a></p>
<ul>
<li>Top p of a peak group v.s. the number of peaks in the group</li>
</ul>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window100kb_cis500kb.pdf">100kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window250kb_cis500kb.pdf">250kb
window</a></p>
<p><a
href="asset/2023/01_11_plt_top_p_peak_group_on_num_peaks_H3K27AC_window500kb_cis500kb.pdf">500kb
window</a></p>
</div>
</div>
<div id="compare-results-across-window-sizes" class="section level2">
<h2>2. Compare results across window sizes</h2>
<p>I compared the results of peak groups defined by different window
sizes in terms of (1) cQTLs (2) peak groups.</p>
<div id="top-p-value-distribution-across-window-sizes"
class="section level3">
<h3>Top p-value distribution across window sizes</h3>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf" alt="Figure: Top p-vaue of a peak group across window sizes." width="50%" />
<p class="caption">
Figure: Top p-vaue of a peak group across window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_top_p_window_size_H3K27AC_cis500kb">
Past versions of 01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf
</button>
</p>
<div id="fig-01_11_plt_top_p_window_size_H3K27AC_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_top_p_window_size_H3K27AC_cis500kb.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="number-of-significant-cqtls-across-window-sizes"
class="section level3">
<h3>Number of significant cQTLs across window sizes</h3>
<ul>
<li><p>How?</p>
<ul>
<li><p>Use 1e-10 as the uniform p-value cutoff across window
sizes.</p></li>
<li><p>Unique cQTLs for each group</p></li>
</ul></li>
</ul>
<p>Define significant cQTLs under same cutoff, as (1) same number of
peak groups, (2) same cis distance used to search for SNPs to be
tested.</p>
<ul>
<li>But not same number of SNPs tested?</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf" alt="Figure: Number of cQTLs across window sizes." width="30%" />
<p class="caption">
Figure: Number of cQTLs across window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10">
Past versions of 01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div id="fig-01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_cqtl_window_size_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cqtls-overlap-across-window-sizes" class="section level3">
<h3>cQTLs overlap across window sizes</h3>
<ul>
<li>Use 1e-10 as the uniform p-value cutoff across window sizes.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf" alt="Figure: Overlaps among cQTLs of different window sizes." width="40%" />
<p class="caption">
Figure: Overlaps among cQTLs of different window sizes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10">
Past versions of 01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div id="fig-01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_cqtl_overlap_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Observations</li>
</ul>
<ol style="list-style-type: decimal">
<li><p>Most are shared.</p></li>
<li><p>100kb some specific cQTLs?</p></li>
</ol>
<ul>
<li><p>Missed cQTLs v.s. distance to their target peak groups</p>
<p>Question to ask - Are the missed cQTLs those which are distant to
their target peak groups, therefore, were not considered for association
tests? Or is it because true higher power?</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2023/01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf" alt="Figure: Window-size-specific cQTLs v.s. their distances to target peak groups." width="40%" />
<p class="caption">
Figure: Window-size-specific cQTLs v.s. their distances to target peak
groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10">
Past versions of
01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf
</button>
</p>
<div
id="fig-01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/3439b271ac6c04695864e0b1a540fff95f940472/docs/asset/2023/01_11_plt_window_spec_cqtl_dist_group_H3K27AC_cis500kb_logp10.pdf" target="_blank">3439b27</a>
</td>
<td>
liliw-w
</td>
<td>
2023-01-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="in-terms-of-peak-groups" class="section level3">
<h3>In terms of peak groups</h3>
</div>
</div>
<div id="compare-multi-cqtls-with-uni-cqtls-1" class="section level2">
<h2>3. Compare multi-cQTLs with uni-cQTLs</h2>
<p><strong>Questions?</strong></p>
</div>
</div>
<div id="dec-21" class="section level1">
<h1>Dec 21</h1>
<div id="quick-summary-8" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li><p>Ran associations (by ACAT) between SNPs and peak groups using
cis-window 1Mb.</p></li>
<li><p>Looked at the distribution of p-values.</p></li>
</ul>
</div>
<div id="update-test-a-snp-and-a-group-of-adjacent-peaks-by-acat"
class="section level2">
<h2>1. Update: Test a SNP and a group of adjacent peaks by ACAT</h2>
<div id="updates" class="section level3">
<h3>Updates</h3>
<ul>
<li><p>For each peak group, test its association across all overlapped
SNPs (SNPs with cis-cQTL associations with all peaks in the group).
<del><u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del>
<u><strong>[Use 1Mb cis-cQTL]</strong></u></p></li>
<li><p>For now, I only looked at the the peaks groups,</p>
<ul>
<li><p><del>where none of the peaks have significant cis-cQTLs
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del> Run
across all peak groups no matter how many significant cis-cQTLs they
have.</p></li>
<li><p>which have more than 1 peak</p></li>
<li><p>of mark H3K27AC</p></li>
</ul></li>
<li><p>Except peak groups on chr1.</p></li>
</ul>
</div>
<div id="results-4" class="section level3">
<h3>Results</h3>
<ul>
<li>Look at the distribution of top p-values of peak groups.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_21_plt_top_p_peak_group_density_H3K27AC.pdf" alt="LEFT: Distribution of the top p-value (by ACAT and minp_adjusted) across peak groups." width="30%" />
<p class="caption">
LEFT: Distribution of the top p-value (by ACAT and minp_adjusted) across
peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_density_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_density_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_density_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_density_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Look at the distribution of top p-values of peak groups on the size
of peak groups and chromosomes.</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" alt="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf" alt="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_on_chr_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_on_chr_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_on_chr_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Distribution of top p-values on chromosomal position.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf" alt="Distribution of the top p-value across peak groups on chromosomal position." width="60%" />
<p class="caption">
Distribution of the top p-value across peak groups on chromosomal
position.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_manhattan_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_manhattan_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_manhattan_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Top p values of a peak group v.s. the number of peaks?</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_21_plt_top_p_peak_group_peaks.pdf" alt="Similar figure as the above, except dividing the peak groups by the number of peaks in the group. Color: the number of peaks in a peak group." width="50%" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group. Color: the number of peaks in a peak
group.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_peaks">
Past versions of 12_21_plt_top_p_peak_group_peaks.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_peaks" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_peaks.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Top p values of a peak group v.s. the number of peaks with
cQTLs?</p>
<ul>
<li>.. and a more detailed zoom-in on chromosomal position given
below</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_21_plt_top_p_peak_group_w_cqtl.pdf" alt="Similar figure as the above, except dividing the peak groups by the number of peaks in the group that have cQTLs. Color: the number of peaks in a peak group that have cQTLs." width="50%" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group that have cQTLs. Color: the number of peaks
in a peak group that have cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_w_cqtl">
Past versions of 12_21_plt_top_p_peak_group_w_cqtl.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_w_cqtl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_w_cqtl.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf" alt="A more detailed zoom-in. Top p-values across peak groups on chromosomal position, grouped and colored by #peaks with uni-cQTLs." width="70%" />
<p class="caption">
A more detailed zoom-in. Top p-values across peak groups on chromosomal
position, grouped and colored by #peaks with uni-cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC">
Past versions of 12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf
</button>
</p>
<div id="fig-12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2baacd09a2d62386870ce4d0260504aa92c470e7/docs/asset/2022/12_21_plt_top_p_peak_group_manhattan_cqtl_H3K27AC.pdf" target="_blank">2baacd0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-22
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Compare the uni-cQTLs and multi-cQTLs of a peak group</li>
</ul>
<p>~~### Numbers</p>
<p>Total peak groups: 100858</p>
<p>Peak groups on chr1: 10038</p>
<p>Peak groups on chr2-chr22: 90150 (90787, 90166)</p>
<p>Peak groups with only 1 peak: 26458</p>
<p>~~</p>
</div>
</div>
</div>
<div id="dec-14" class="section level1">
<h1>Dec 14</h1>
<div id="quick-summary-9" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Run ACAT to test associations between (SNP, adjacent peak group) for
mark H3K27AC.</li>
</ul>
</div>
<div id="cqtl-of-peaks-with-cis-window-of-1mb" class="section level2">
<h2>1. cQTL of peaks with cis-window of 1Mb</h2>
<p>[Waiting for Carlos to run this step for me.]</p>
</div>
<div id="adjacent-peaks-as-peak-groups" class="section level2">
<h2>2. Adjacent peaks as peak groups</h2>
<div id="how-6" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Sliding window of length 500kb.</p></li>
<li><p>Start with a peak, extend from its start position to a 500kb
window, include all peaks in this window as a peak group.</p></li>
<li><p>Slide to next peak.</p></li>
</ul>
</div>
<div id="look-into-peak-groups" class="section level3">
<h3>Look into peak groups</h3>
<ul>
<li><p>Number of peak groups in total</p>
<p>There are 100,858 peak groups in total.</p></li>
<li><p>Number of peaks in each group</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_14_peaks_in_group_H3K27AC.pdf" alt="How many peaks are included in the preak groups? x-axis: the number of peaks. y-axis: the number of peaks groups with the corresponding number of peaks." width="30%" />
<p class="caption">
How many peaks are included in the preak groups? x-axis: the number of
peaks. y-axis: the number of peaks groups with the corresponding number
of peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_in_group_H3K27AC">
Past versions of 12_14_peaks_in_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_in_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_in_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>For peak groups with the same number of peaks, how many peaks have
cQTLs? <u><strong>[Here I used the 100kb cis-cQTL]</strong></u></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_14_peaks_in_group_w_cqtl_H3K27AC.pdf" alt="Similar figure as the above, except dividing the peak groups by the number of peaks in the group that have cQTLs. Color: the number of peaks in a peak group that have cQTLs." width="30%" />
<p class="caption">
Similar figure as the above, except dividing the peak groups by the
number of peaks in the group that have cQTLs. Color: the number of peaks
in a peak group that have cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_in_group_w_cqtl_H3K27AC">
Past versions of 12_14_peaks_in_group_w_cqtl_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_in_group_w_cqtl_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_in_group_w_cqtl_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Number of peaks with/without cQTLs in each group <u><strong>[Here I
used the 100kb cis-cQTL]</strong></u></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_14_peaks_w_cqtl_in_group_H3K27AC.pdf" alt="How many peaks within a peak group have cQTLs? x-axis: the number of peaks with cQTLs. y-axis: the number of peak groups that have corresponding cPeaks." width="30%" />
<p class="caption">
How many peaks within a peak group have cQTLs? x-axis: the number of
peaks with cQTLs. y-axis: the number of peak groups that have
corresponding cPeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_peaks_w_cqtl_in_group_H3K27AC">
Past versions of 12_14_peaks_w_cqtl_in_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_peaks_w_cqtl_in_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_peaks_w_cqtl_in_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="test-a-snp-and-a-group-of-adjacent-peaks-by-acat"
class="section level2">
<h2>3. Test a SNP and a group of adjacent peaks by ACAT</h2>
<div id="how-7" class="section level3">
<h3>How?</h3>
<ul>
<li><p>For each peak group, test its association across all overlapped
SNPs (SNPs with cis-cQTL associations with all peaks in the group).
<del><u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del>
<u><strong>[Use 1Mb cis-cQTL]</strong></u></p></li>
<li><p>For (peak group, SNP), test by ACAT, a method to combine
p-values.</p>
<ul>
<li>As a comparison, I also calculated the adjusted minp, which <span
class="math inline">\(=min\{p_{snp, peak_1}, \dots, p_{snp, peak_k}\}
\times k\)</span>, where <span class="math inline">\(k\)</span> is the
number of peaks in the group.</li>
</ul></li>
<li><p>For now, I only looked at the the peaks groups,</p>
<ul>
<li><p><del>where none of the peaks have significant cis-cQTLs
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></del> Run
across all peak groups no matter how many significant cis-cQTLs they
have.</p></li>
<li><p>which have more than 1 peak</p></li>
<li><p>of mark H3K27AC</p></li>
</ul></li>
</ul>
</div>
<div id="results-5" class="section level3">
<h3>Results</h3>
<ul>
<li><p>There are 25,853 peak groups considered for association
test.</p></li>
<li><p>3,450 peak groups have at least one overlapped SNPs.
<u><strong>[Here I used the 100kb cis-cQTL]</strong></u></p></li>
</ul>
<p>So, now I have ACAT p-values (and adjusted minp) for peak groups
across their overlapped SNPs. I take the minimum p-value for a peak
group across the overlapped SNPs as the top p-value for the peak
group.</p>
<ul>
<li>Look at the distribution of top p-values of peak groups.</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/12_14_plt_top_p_peak_group_H3K27AC.pdf" alt="LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT: Distribution of the top p-value across peak groups." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/12_14_plt_top_p_peak_group_density_H3K27AC.pdf" alt="LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT: Distribution of the top p-value across peak groups." width="30%" />
<p class="caption">
LEFT: Top p-value (by ACAT and minp_adjusted) across peak groups. RIGHT:
Distribution of the top p-value across peak groups.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_density_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_density_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_density_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_density_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Look at the distribution of top p-values of peak groups on the size
of peak groups and chromosomes.</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" alt="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_on_num_peaks_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf" alt="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="40%" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_top_p_peak_group_on_chr_H3K27AC">
Past versions of 12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf
</button>
</p>
<div id="fig-12_14_plt_top_p_peak_group_on_chr_H3K27AC"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_top_p_peak_group_on_chr_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>More detailed p-values for (a SNP, a peak group) for a few peak
groups with significant p_acat. See more <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G56480.pdf">here</a>, <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57648.pdf">here</a>, <a
href="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57649.pdf">here</a>.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/12_14_plt_p_peak_group_H3K27AC_G57644.pdf" alt="Distribution of the top p-value across peak groups on, LEFT: the groups with different number of peaks. RIGHT: chromosomes." width="30%" />
<p class="caption">
Distribution of the top p-value across peak groups on, LEFT: the groups
with different number of peaks. RIGHT: chromosomes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-12_14_plt_p_peak_group_H3K27AC_G57644">
Past versions of 12_14_plt_p_peak_group_H3K27AC_G57644.pdf
</button>
</p>
<div id="fig-12_14_plt_p_peak_group_H3K27AC_G57644" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/12_14_plt_p_peak_group_H3K27AC_G57644.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations" class="section level3">
<h3>Observations</h3>
<ul>
<li><p>Looks like p-values by ACAT are similar to p-values by minp
adjusted by Bonferroni correction.</p></li>
<li><p>The minimum p-values are ~1e-6.</p></li>
<li><p>Why no significant associations?</p>
<ul>
<li><p>I only test ~3k peak groups that have overlapped SNPs within
100kb window. So these peak groups are very close. So, combining these
peaks don’t have much improvement.</p></li>
<li><p>If I include more peak groups (and more SNPs) that are not so
close for association test, by extending 100kb cis window to 1Mb cis
window, maybe there can be more significant p-values.)</p></li>
<li><p>Will update the numbers after Carlos sends me the 1Mb cQTL
associations.</p></li>
</ul></li>
</ul>
</div>
</div>
</div>
<div id="dec-09" class="section level1">
<h1>Dec 09</h1>
<div id="quick-summary-10" class="section level2">
<h2>Quick Summary</h2>
<ul>
<li>Preliminary analysis on Carlos’s hQTL results.</li>
</ul>
</div>
<div id="carloss-analysis-and-results" class="section level2">
<h2>Carlos’s analysis and results</h2>
<ul>
<li><p>A few questions</p>
<ul>
<li><p>Sample size - H3K27AC (78/72), H3K4ME1 (77), H3K4ME3 (78),
H3K36ME3 (95)</p></li>
<li><p>Why use genes as phenotype for H3K36ME3 but not peaks?</p></li>
<li><p>For mark H3K36ME3, peak names used for <code>Counts.txt</code>,
but gene names used for the other files. A conversion file?</p></li>
<li><p>In file <code>NominalPass.txt.gz</code>, what’s the nominal
p-value cutoff?</p></li>
</ul></li>
<li><p>Data of H3K27AC, H3K4ME1, H3K4ME3, and H3K36ME3 chromatin
marks</p>
<ul>
<li><p>ChIP-seq data for H3K27AC, H3K4ME1 and H3K4ME3 chromatin marks in
75 LCLs from Yoruba invididuals was publicly available from a study by
Grubert et al (2015).</p></li>
<li><p>We produced in this study the Cut&amp;Tag data for the H3K36ME3
chromatin mark in 96 LCLs from Yoruba individuals.</p></li>
</ul></li>
<li><p>Pre-processing</p>
<ul>
<li><p>We aligned the reads from all experiments to the GRCh38 human
genome using HISAT.</p></li>
<li><p>Peak calling for the ChIP-seq data was performed with MACS2 for
H3K27AC and H3K4ME1 (narrow peaks), and for H3K4ME3 (broad
peaks).</p></li>
<li><p>For each individual, quantified ChIP-seq coverage for the three
marks across the peaks using featureCounts.</p></li>
<li><p>After filtering for low coverage and low variance, we were left
with 100858 peaks for H3K27AC, 182749 peaks for H3K4ME1, and 55769 peaks
for H3K4ME3.</p></li>
<li><p>H3K36ME3 is a chromatin mark associated with transcription, and
it doesn’t form clearly defined peaks like the other marks. For
H3K36ME3, we quantified coverage across the top expressed 14000 protein
coding genes with featureCounts. The set of top expressed genes was
determined with mRNA expression in LCLs from Yoruba invidivuals in
RNA-seq data from the Geuvadis consortium.</p></li>
</ul></li>
<li><p>Normalization</p>
<p>Each phenotype (peaks or genes) by individuals coverage table was
normalized as follows:</p>
<ul>
<li><p>first, for each individual we normalized the raw peak or gene
counts to counts per million (CPM).</p></li>
<li><p>Second, for each phenotype we applied the standard normalization
across all individuals.</p></li>
<li><p>Finally, for each individual we applied the rank-based inverse
normal transformation across all phenotypes.</p></li>
</ul></li>
<li><p>hQTL calling</p>
<ul>
<li>hQTLs were called using QTLTools with permutation pass and nominal
pass. We used a cis window of 100 kb, and the principal components as
covariates.</li>
</ul></li>
<li><p>Results</p>
<p>In summary, for each of H3K27AC, H3K4ME1 and H3K4ME3 we have a set of
peaks. For each of the four chromatin marks (including H3K36ME3), we
have the following data:</p>
<ul>
<li><p><strong>Raw counts</strong>: Phenotype (peaks or gene) by
individual table of raw counts.</p></li>
<li><p><strong>Normalized counts</strong>: Phenotype by individual table
of qqnorm data (normalization described above)</p></li>
<li><p><strong>Permutation pass (top hQTL-peak/gene
association)</strong>: Table with top hQTLs for each phenotype from
QTLTools in permutation pass.</p></li>
<li><p><strong>Nominal pass (all hQTL-peak/gene associations within
100kb windows of the peak)</strong>: Table with hQTLs results for each
SNP within 100kb of each phenotype from QTLTools in permutation
pass.</p></li>
</ul></li>
</ul>
</div>
<div id="strong-tag-enriched-peaks" class="section level2">
<h2>Strong tag enriched peaks</h2>
<div id="look-at-peaks" class="section level3">
<h3>Look at peaks</h3>
<p>Questions of interest:</p>
<ul>
<li><p>Length of merged peaks</p></li>
<li><p>Number of sub-peaks merged into large peaks</p></li>
<li><p>Distance between peaks</p></li>
</ul>
</div>
<div id="look-at-pqtl-effects" class="section level3">
<h3>Look at pQTL effects</h3>
<p>Questions of interest:</p>
<ul>
<li><p>What is the proportion of peaks with pQTLs?</p>
<ul>
<li><p>For mark H3K27AC , there are 105049 peaks in total. After
preprocessing, 100858 peaks are left for QTL calling. <strong>7380 peaks
( 0.07317218 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
<li><p>For mark H3K4ME1 , there are 190250 peaks in total. After
preprocessing, 182749 peaks are left for QTL calling. <strong>4383 peaks
( 0.02398372 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
<li><p>For mark H3K4ME3 , there are 58171 peaks in total. After
preprocessing, 55769 peaks are left for QTL calling. <strong>3246 peaks
( 0.05820438 ) have significant pQTLs</strong>, under FDR &lt; 0.05
.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/1209_dis_sig_peak_H3K27AC.pdf" alt="Significant peaks of mark H3K27AC." width="30%" />
<p class="caption">
Significant peaks of mark H3K27AC.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1209_dis_sig_peak_H3K27AC">
Past versions of 1209_dis_sig_peak_H3K27AC.pdf
</button>
</p>
<div id="fig-1209_dis_sig_peak_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/1209_dis_sig_peak_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Length of peaks with &amp; w.o. pQTLs</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/1209_dis_peak_len_H3K27AC.pdf" alt="Peak length of mark H3K27AC." width="50%" />
<p class="caption">
Peak length of mark H3K27AC.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1209_dis_peak_len_H3K27AC">
Past versions of 1209_dis_peak_len_H3K27AC.pdf
</button>
</p>
<div id="fig-1209_dis_peak_len_H3K27AC" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/271b94b3689648b94d0fc9f1460d6d0751b5b384/docs/asset/2022/1209_dis_peak_len_H3K27AC.pdf" target="_blank">271b94b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-12-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>- Reads count of peaks with &amp; w.o. pQTLs</strong>
<strong>- Peak enrichment (weak v.s. strong) for peaks with &amp; w.o.
pQTLs</strong></p>
<ul>
<li><p>What is the distribution of effect sizes for peaks (with pQTL
&amp; without pQTLs)?</p>
<ul>
<li><p>Distribution of all effects</p></li>
<li><p>Distribution of all effects on the dimension of SNPs and peaks
coordinates</p></li>
<li><p>Distribution of the most significant effect for each peaks across
SNPs</p></li>
</ul></li>
</ul>
<p><del>- Distribution of peaks with pQTL &amp; without pQTLs?</del></p>
<ul>
<li>Are peaks without pQTLs more likely to be peaks with merged
sub-peaks? (Can combining multiple sub-peaks in a large peak improve
power than considering the large peak as a whole?)</li>
</ul>
<p><strong>- Are peaks without pQTLs more adjacent than peaks with
pQTLs? (Can combining multiple adjacent peaks improve power than using
individual peaks?)</strong></p>
</div>
</div>
<div id="weak-tag-enriched-peaks" class="section level2">
<h2>Weak tag enriched peaks</h2>
</div>
<div id="what-type-of-peaks-should-have-improved-qtl-calling"
class="section level2">
<h2>What type of peaks should have improved QTL calling?</h2>
</div>
<div id="how-to-improve" class="section level2">
<h2>How to improve?</h2>
<ol style="list-style-type: decimal">
<li><p>Combine two peaks</p></li>
<li><p>Combine sub-peaks for one peak</p></li>
</ol>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8       highr_0.9        bslib_0.3.1.9000 compiler_4.1.2  
 [5] pillar_1.7.0     later_1.3.0      git2r_0.29.0     jquerylib_0.1.4 
 [9] tools_4.1.2      getPass_0.2-2    digest_0.6.29    jsonlite_1.8.0  
[13] evaluate_0.15    lifecycle_1.0.3  tibble_3.1.6     pkgconfig_2.0.3 
[17] rlang_1.0.6      cli_3.6.0        rstudioapi_0.13  yaml_2.3.5      
[21] xfun_0.31        fastmap_1.1.0    httr_1.4.2       stringr_1.5.0   
[25] knitr_1.39       sass_0.4.1       fs_1.5.2         vctrs_0.5.1     
[29] rprojroot_2.0.2  glue_1.6.2       R6_2.5.1         processx_3.5.2  
[33] fansi_1.0.2      rmarkdown_2.14.2 callr_3.7.0      magrittr_2.0.3  
[37] whisker_0.4      ps_1.6.0         promises_1.2.0.1 htmltools_0.5.4 
[41] ellipsis_0.3.2   httpuv_1.6.5     utf8_1.2.2       stringi_1.7.12  
[45] cachem_1.0.6     crayon_1.4.2    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
