<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2022</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>




<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2022</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-02-04
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree089c60942bfc4e7416bbc5bcf73accd65aeef082targetblank089c609a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liliw-w/GradLog/tree/089c60942bfc4e7416bbc5bcf73accd65aeef082" target="_blank">089c609</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree089c60942bfc4e7416bbc5bcf73accd65aeef082targetblank089c609a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liliw-w/GradLog/tree/089c60942bfc4e7416bbc5bcf73accd65aeef082" target="_blank">089c609</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/test_log.Rmd
    Ignored:    code/.DS_Store
    Ignored:    data/.DS_Store
    Ignored:    output/.DS_Store

Untracked files:
    Untracked:  analysis/references.bib

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Log2022.Rmd</code>) and HTML (<code>docs/Log2022.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/089c60942bfc4e7416bbc5bcf73accd65aeef082/analysis/Log2022.Rmd" target="_blank">089c609</a>
</td>
<td>
liliw-w
</td>
<td>
2022-02-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1274e244c3b7d06966b3351caa0bff74870a3382/docs/Log2022.html" target="_blank">1274e24</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9976af6f087dd6b4546a78fc8936e5dc54c7e01a/analysis/Log2022.Rmd" target="_blank">9976af6</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bd69a21c8b6909c5174347c0c0c724d7169dc47c/docs/Log2022.html" target="_blank">bd69a21</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/28d0a2b13cae4c13c5979ffcf05154bc2aa15603/analysis/Log2022.Rmd" target="_blank">28d0a2b</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ec0a2af10bac6589b98ea6818de13cd3bda98a33/docs/Log2022.html" target="_blank">ec0a2af</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9e5c2e02a0e8f42e2fd9c4f697dd3d8c16fff6e2/analysis/Log2022.Rmd" target="_blank">9e5c2e0</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/97b4ca1336ea92e396991f1569177c14360eb07c/docs/Log2022.html" target="_blank">97b4ca1</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4c90bd8ce853f24d0248eee7fe6a8e038cb3d0a0/docs/Log2022.html" target="_blank">4c90bd8</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6dc3f9c96110c0be94a620950616ee99eaa806f0/analysis/Log2022.Rmd" target="_blank">6dc3f9c</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
add a new year
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="jan-26" class="section level2">
<h2>Jan 26</h2>
<div id="update-coloc-figures-with-less-stringent-pvalue-threshold" class="section level3">
<h3>1. Update coloc figures with less stringent pvalue threshold</h3>
<p>Previously, we defined the green regions with module-QTL <span class="math inline">\(p&lt;1e-8\)</span>. This threshold is not premutation based. I looked at the permutation-based p threshold, and it is <span class="math inline">\(\approx 3e-8\)</span>. We’d like to use a less stringent threshold, so I chose <span class="math inline">\(p&lt;1e-7\)</span> to define the green regions.</p>
<p>In total, we considered three (four) types of coloc. Coloc with (1) 29 blood related traits from ukbb, (2) 11 immune traits, (3) 27 more traits from ukbb. (The 4-th type is coloc with cis genes.)</p>
<p>I updated the figures and tables.</p>
<ul>
<li>figures</li>
</ul>
<p><a href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">29 blood related traits from ukbb</a>; <a href="../docs/asset/1115_all.traits.coloc.region.summary.png">earlier result with 1e-8 thre</a></p>
<p><a href="../docs/asset/2022/0126_immune_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">11 immune traits</a>; <a href="../docs/asset/1129_all.traits.coloc.region.summary.png">earlier result with 1e-8 thre</a></p>
<p><a href="../docs/asset/2022/0126_ukbb_coloc_more_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">27 more traits from ukbb</a>; <a href="../docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">earlier result with 1e-8 thre</a></p>
<ul>
<li>tables with detailed numerical results</li>
</ul>
<p><a href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_coloc_region_prop_pvalThre-1e-07.txt">29 blood related traits from ukbb</a></p>
<p><a href="../docs/asset/2022/0126_immune_traits_coloc_region_prop_pvalThre-1e-07.txt">11 immune traits</a></p>
<p><a href="../docs/asset/2022/0126_ukbb_coloc_more_traits_coloc_region_prop_pvalThre-1e-07.txt">27 more traits from ukbb</a></p>
</div>
<div id="other-types-of-modules-coloc-with-traits" class="section level3">
<h3>2. Other types of modules coloc with traits</h3>
</div>
</div>
<div id="jan-19" class="section level2">
<h2>Jan 19</h2>
<div id="comprehensive-analysis-of-coloc-regions-of-trait-ms" class="section level3">
<h3>1. Comprehensive analysis of coloc regions of trait MS</h3>
<div id="add-mss-result-to-previous-coloc-analysis-dark-green-coloc-regions" class="section level4">
<h4>Add MS’s result to previous coloc analysis (dark green coloc regions)</h4>
<p>See <a href="Log2021.html#coloc-region-between-module-and-immune-trait">earlier section</a>.</p>
</div>
<div id="dark-blue-coloc-regions" class="section level4">
<h4>dark blue coloc regions</h4>
<p>There are 4 dark blue coloc regions, i.e. coloc regions with module-QTL lead pvalue &lt; 1e-5.</p>
<table>
<thead>
<tr class="header">
<th align="left">Region</th>
<th align="right">nSNPs</th>
<th align="right">Pval</th>
<th align="right">PP4</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module121:19:10423815</td>
<td align="right">140</td>
<td align="right">1.5e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="even">
<td align="left">module108:7:50308811</td>
<td align="right">122</td>
<td align="right">1.6e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="odd">
<td align="left">module50:7:149331042</td>
<td align="right">265</td>
<td align="right">2.0e-06</td>
<td align="right">0.94</td>
</tr>
<tr class="even">
<td align="left">module44:7:50308527</td>
<td align="right">69</td>
<td align="right">0.0e+00</td>
<td align="right">0.89</td>
</tr>
</tbody>
</table>
<p>In <a href="#make-a-story-out-of-trans-signals">earlier section</a>, the region “module108:7:50308811” coloc with MS is included in these dark blue regions.</p>
</div>
</div>
<div id="more-traits-from-ukbb" class="section level3">
<h3>2. More traits, from UKBB</h3>
<div id="extracted-traits" class="section level4">
<h4>Extracted traits</h4>
<ol style="list-style-type: decimal">
<li>How?</li>
</ol>
<p>I extracted specific traits from ukbb traits manifest file, based on:</p>
<ul>
<li><p>Traits from eQTLGen traits</p></li>
<li><p>Traits from Mu et al paper</p></li>
<li><p>Powerful (and interesting) ukbb GWASs</p>
<p>Sorted by case number of EUR <code>n_cases_EUR</code>.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>What traits?</li>
</ol>
<ul>
<li><p>26 ukbb GWASs in total, including height, BMI, Cholesterol, Hypertension, Asthma, Type 2 diabetes etc.</p></li>
<li><p>Exclude height, as not GWASed for EUR.</p></li>
<li><p>Use height sum stat from <a href="height_BMI_meta_data_source">another meta analysis</a> instead. It also has another BMI sum stat.</p>
<p>Meta analyze inds from GIANT and ukbb.</p>
<blockquote>
<p>Meta-analysis of genome-wide association studies for height and body mass index in ~700,000 individuals of European ancestry.</p>
</blockquote></li>
<li><p>Thus, as a result, 27 GWASs. 22 unique traits.</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Visualize the sample sizes of selected traits</li>
</ol>
<p><img src="asset/2022/0119_phenotype_sub_N.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_phenotype_sub_N">
Past versions of 0119_phenotype_sub_N.png
</button>
</p>
<div id="fig-0119_phenotype_sub_N" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_phenotype_sub_N.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="coloc-results-more-traits-from-ukbb" class="section level4">
<h4>coloc results {More-traits-from-UKBB}</h4>
<ol style="list-style-type: decimal">
<li>Visualize</li>
</ol>
<p>See <a href="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">figure</a>.</p>
<p><img src="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_selected_ukbb_all.traits.coloc.region.summary">
Past versions of 0119_selected_ukbb_all.traits.coloc.region.summary.png
</button>
</p>
<div id="fig-0119_selected_ukbb_all.traits.coloc.region.summary" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For detailed numbers, see <a href="asset/2022/0119_selected_ukbb_coloc_region_prop.txt">file</a>.</p>
<ol start="2" style="list-style-type: decimal">
<li>In-depth visualization for selected traits (dark blue regions)</li>
</ol>
<p><a href="asset/2022/0119_BMI_coloc.region.summary.png">BMI</a></p>
<p><a href="asset/2022/0119_BMI-2_coloc.region.summary.png">BMI</a></p>
<p><a href="asset/2022/0119_Cholesterol_coloc.region.summary.png">Cholesterol</a></p>
<p><a href="asset/2022/0119_HDL-cholesterol_coloc.region.summary.png">HDL-cholesterol</a></p>
<p><a href="asset/2022/0119_Hayfever,-allergic-rhinitis-or-eczema_coloc.region.summary.png">Hayfever,-allergic-rhinitis-or-eczema</a></p>
<p><a href="asset/2022/0119_Hypertension_coloc.region.summary.png">Hypertension</a></p>
</div>
</div>
</div>
<div id="jan-12" class="section level2">
<h2>Jan 12</h2>
<div id="make-a-story-out-of-trans-signals" class="section level3">
<h3>1. Make a story out of the trans- signals</h3>
<div id="more-concrete-examples" class="section level4">
<h4>More concrete examples</h4>
<ol style="list-style-type: decimal">
<li>Trait, gene, trans- signal</li>
</ol>
<p>SLE signals in Fig. 5.</p>
<blockquote>
<p>reflecting the involvement of interferon signaling in SLE pathophysiology</p>
</blockquote>
<p><img src="asset/2022/0112_Systemic lupus erythematosus_tile.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-0112_Systemic-lupus-erythematosus_tile">
Past versions of "0112_Systemic lupus erythematosus_tile.png"
</button>
</p>
<div id="fig-no-spaces-0112_Systemic-lupus-erythematosus_tile" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/664d0d7bdcedd0fb3164db435cb8bffdce9ea41e/docs/asset/2022/0112_Systemic lupus erythematosus_tile.png" target="_blank">664d0d7</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>additional examples of trans-eQTL variants associated with traits</li>
</ol>
<ul>
<li><p>ZNF131 locus, age of menarche</p>
<p>Supplementary Figure 11A, rs1532331</p></li>
<li><p>FADS1/FADS2 locus, lipid levels</p>
<p>Supplementary Figure 11B, rs174574</p></li>
<li><p>IFIH1 locus, inflammatory bowel disease and SLE</p>
<p>Supplementary Figure 11C, rs1990760</p></li>
<li><p>GSDMB locus, asthma</p>
<p>Supplementary Figure 11D, rs7216389</p></li>
<li><p>CLOCK locus, height</p>
<p>Supplementary Figure 11E, rs1311351834</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li><p>eQTSs genes GO enrichment</p></li>
<li><p>eQTSs genes associated with HDL</p></li>
</ol>
</div>
</div>
</div>
<div id="jan-05" class="section level2">
<h2>Jan 05</h2>
<div id="trans-eqtl-and-complex-disease-in-eqtlgen-võsa-franke-et-al" class="section level3">
<h3>1. trans-eQTL and complex disease in <a href="https://www.nature.com/articles/s41588-021-00913-z">eQTLGen, Võsa, Franke et al</a></h3>
<div id="question-are-there-any-trans-eqtls-that-are-gwas-hits" class="section level4">
<h4>Question: are there any trans-eQTLs that are GWAS hits?</h4>
<p>Observations from the paper:</p>
<ul>
<li><p>Meta-analysis using up to <strong>31,684 blood samples</strong> from 37 eQTLGen Consortium cohorts.</p></li>
<li><p>For trans, they focused on <strong>10,317 trait-associated SNPs</strong>.</p></li>
</ul>
<p>The paper linked trans-eQTLs with traits in two ways to find the potential driver genes for traits.</p>
<ol style="list-style-type: decimal">
<li>trans-eQTL analysis</li>
</ol>
<ul>
<li><p>One-third of trait-associated variants have distal effects.</p>
<p>Identified 59,786 trans-eQTL, representing 3,853 SNPs (<strong>37% of tested GWAS SNPs</strong>) and 6,298 genes (32% of tested genes).</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311) identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote></li>
<li><p>Identify genes that are coordinately affected by <strong>multiple independent trait-associated SNPs</strong>.</p>
<p>Identified <strong>47 GWAS traits</strong> for which at least four independent variants affected the same gene in trans (Supplementary Tables 10). Examples genes affected by at least three SLE-associated genetic variants.</p></li>
</ul>
<p>But, Individual trans-eQTL effects too weak to detect. Another way to look for the potential driver genes for traits and the “core” genes:</p>
<ol start="2" style="list-style-type: decimal">
<li>eQTSs (associations between PGSs and gene expression, Fig. 6a, 6b)</li>
</ol>
<ul>
<li><p>Individual trait-associated SNPs are combined into a PGS that is associated with gene expression.</p></li>
<li><p>when the PGS for a trait correlates with the expression of a gene, trans-eQTL effects of individual risk variants converge on that gene, and it can be prioritized as a putative driver of the disease.</p></li>
<li><p>1,263 traits in total. 18,210 eQTSs representing 689 unique traits (<strong>55% of tested traits</strong>) and 2,568 genes (<strong>13% of tested genes</strong>).</p></li>
<li><p>Of these genes, 719 (<strong>28%</strong>) were not identified in the trans-eQTL analysis.</p></li>
</ul>
<p><strong>Therefore, all 3,853 trans-eQTLs eQTLGen identified are GWAS hits.</strong></p>
</div>
<div id="question-if-so-can-we-replicate-them" class="section level4">
<h4>Question: if so, can we replicate them?</h4>
<p>Among 10,317 trait-associated SNPs, 9,056 (~89%) are included in DGN SNPs. Among 3,853 eQTLGen trans- eQTLs, <strong>27 (~0.7%) are replicated in DGN signals (1,863,</strong> <span class="math inline">\(p&lt;1e-8\)</span>).</p>
<p>In the paper, another trans- study was mentioned,</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311) identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote>
</div>
</div>
<div id="gene-senp7" class="section level3">
<h3>2. gene <em>SENP7</em></h3>
<div id="the-cross-mappability-of-senp7-and-the-genes-in-the-trans-modules" class="section level4">
<h4>the cross mappability of SENP7 and the genes in the trans modules</h4>
<p>As noted in <a href="Log2021.html#what_is_gene_SENP7">earlier section</a>, we observed,</p>
<ul>
<li><em>SENP7</em> is the neasrest gene of the lead SNPs of coloc regions.</li>
<li>Gene <em>ZNF90P1</em> is located within <em>SENP7</em>.</li>
<li>Modules 25, 51, 153, 156 have coloc regions with immune traits and consist of many zinc finger genes.</li>
</ul>
<p>Therefore, we wanted to know,</p>
<ol style="list-style-type: decimal">
<li>Does gene <em>ZNF90P1</em> have expression data?</li>
</ol>
<p>I looked at the expression matrix of 13634 genes. <em>ZNF90P1</em> doesn’t have expression data in this matrix.</p>
<ol start="2" style="list-style-type: decimal">
<li>What is the cross mappability between <em>SENP7</em> and the zinc finger genes in the modules?</li>
</ol>
<p>See files for the cross mappability for <a href="asset/2022/0105_SENP7_module25_size123_crossMap.txt">module 25</a>, <a href="asset/2022/0105_SENP7_module51_size61_crossMap.txt">module 51</a>, <a href="asset/2022/0105_SENP7_module153_size15_crossMap.txt">module 153</a>, <a href="asset/2022/0105_SENP7_module156_size13_crossMap.txt">module 156</a>.</p>
<p>The columns include: “score” for cross mappability score, “score_map1” for mappability score of gene1, “score_map2” for mappability score of gene2.</p>
<ol start="3" style="list-style-type: decimal">
<li>For the genes in the trans modules,</li>
</ol>
<p>See a previous <a href="asset/1213_cd_ibd_Allergy_coloc_module_david_query.xlsx">file here</a>.</p>
</div>
<div id="highlighting-senp7-as-a-trans-meqtl" class="section level4">
<h4>highlighting SENP7 as a trans-meQTL</h4>
<p><a href="https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x" class="uri">https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x</a> Another paper highlighting SENP7 as a trans-meQTL.</p>
</div>
</div>
<div id="pqtl-resource" class="section level3">
<h3>3. pQTL resource</h3>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.2 (2021-11-01)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.8       rstudioapi_0.13  knitr_1.37       whisker_0.4     
 [5] magrittr_2.0.2   getPass_0.2-2    R6_2.5.1         rlang_1.0.0     
 [9] fastmap_1.1.0    highr_0.9        stringr_1.4.0    httr_1.4.2      
[13] tools_4.1.2      xfun_0.29        cli_3.1.1        jquerylib_0.1.4 
[17] git2r_0.29.0     htmltools_0.5.2  yaml_2.2.2       digest_0.6.29   
[21] rprojroot_2.0.2  processx_3.5.2   callr_3.7.0      later_1.3.0     
[25] promises_1.2.0.1 fs_1.5.2         ps_1.6.0         glue_1.6.1      
[29] evaluate_0.14    rmarkdown_2.11   stringi_1.7.6    compiler_4.1.2  
[33] httpuv_1.6.5    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
