<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2024</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2024</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-02-20
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef6aa2c741cc8496342de6bfac181650222aafd2ftargetblankf6aa2c7a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/f6aa2c741cc8496342de6bfac181650222aafd2f" target="_blank">f6aa2c7</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef6aa2c741cc8496342de6bfac181650222aafd2ftargetblankf6aa2c7a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/f6aa2c741cc8496342de6bfac181650222aafd2f" target="_blank">f6aa2c7</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2024_new_beginning.Rmd</code>)
and HTML (<code>docs/Log2024_new_beginning.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f6aa2c741cc8496342de6bfac181650222aafd2f/analysis/Log2024_new_beginning.Rmd" target="_blank">f6aa2c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
fix toc
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f6c98b089319bf6c885ad76c146733207f0fad75/docs/Log2024_new_beginning.html" target="_blank">f6c98b0</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/24a0be1963d9bb1f4aabf38bd9f386b81c04c1dd/analysis/Log2024_new_beginning.Rmd" target="_blank">24a0be1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2ab6eac96840c3dea03bec5c7a23fab6b9dcb315/docs/Log2024_new_beginning.html" target="_blank">2ab6eac</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/55c9df22c52dc3e6c03eff7173d4da55e9157df1/analysis/Log2024_new_beginning.Rmd" target="_blank">55c9df2</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8b691e2e4bcb05f0031ed5e2d54300d101cb60c8/docs/Log2024_new_beginning.html" target="_blank">8b691e2</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8bfc0386a02aec0687e4c678333f7bf06d7f10c4/analysis/Log2024_new_beginning.Rmd" target="_blank">8bfc038</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
add a new year
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="feb-15" class="section level1">
<h1>Feb 15</h1>
<div id="simulations-to-evaluate-pco-on-non-zscore-tests"
class="section level2">
<h2>1. Simulations to evaluate PCO on non-zscore tests</h2>
<div id="ways-to-estimate-sigma" class="section level3">
<h3>Ways to estimate Sigma</h3>
<ul>
<li><p>P-value of test statistic from a non-zscore test is converted to
a z-score by inverse-normal transformation.</p></li>
<li><p>Sigma of the converted z-scores is the corresponding
cov.</p></li>
<li><p>Run PCO on the converted z-scores.</p></li>
</ul>
</div>
<div id="null-simulations" class="section level3">
<h3>Null simulations</h3>
<ul>
<li><p><span class="math inline">\(Y\)</span> from residualized
expression matrix of <span class="math inline">\(K=101\)</span> gene
module.</p></li>
<li><p><span class="math inline">\(G\)</span> from binomial
distribution, with maf being 0.2, 1k snps.</p></li>
<li><p><span class="math inline">\(G\)</span> is independent with <span
class="math inline">\(Y\)</span>.</p></li>
<li><p>Run a non-zscore test, e.g. LRT, between every pairs of genes and
snps.</p></li>
<li><p>Obtain chi-square test statistics and p-value.</p></li>
<li><p>Convert p to z-score by <span class="math inline">\(z_{convert} =
\Phi^{-1}(p)\)</span>, where <span
class="math inline">\(\Phi^{-1}\)</span> is the inverse cdf of standard
normal distribution.</p></li>
<li><p>Estimate <span class="math inline">\(\Sigma_{convert} =
cov(z_{convert})\)</span></p></li>
<li><p>Run PCO on <span class="math inline">\(z_{convert}\)</span> using
<span class="math inline">\(\Sigma_{convert}\)</span>.</p></li>
</ul>
</div>
<div id="pco-p-value-inflation-under-null-distribution"
class="section level3">
<h3>PCO p-value inflation under null distribution</h3>
<p>To look at if PCO p-values are inflated under the null simulations
using <span class="math inline">\(z_{convert}\)</span> and <span
class="math inline">\(\Sigma_{convert}\)</span> -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_14_pco_nullp_int.pdf" title="Figure: PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}." width="40%" type="application/pdf" />
<p class="caption">
Figure: PCO p-value inflation under null distribution using z_{convert}
and Sigma_{convert}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_14_pco_nullp_int">
Past versions of 02_14_pco_nullp_int.pdf
</button>
</p>
<div id="fig-02_14_pco_nullp_int" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87402c7054fa6d1127a4c4cb79a57153472fdf89/docs/asset/2024/02_14_pco_nullp_int.pdf" target="_blank">87402c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Problem</strong> -</p>
<ul>
<li>The way Zhonghua mentioned should be <span
class="math inline">\(\Phi^{-1}(p)\)</span>, where <span
class="math inline">\(\Phi\)</span> is the cdf of standard normal
distribution. However, it might not work, as while a small p is
converted to a positively large z, a large p can be falsely converted to
a negatively large z. Therefore, some non signals with large p can be a
false signal and lead to inflation. We want both the converted test to
be normally distributed and correspond to magnitude of p-values as
well.</li>
</ul>
</div>
<div id="other-ways-to-convert-p" class="section level3">
<h3>Other ways to convert p</h3>
<p>I tried other ways to convert p-values to “z-scores”.</p>
<ul>
<li>Convert to a chi-square test, then take square root and assign signs
randomly.</li>
<li>Rank-based inverse-normal transformation.</li>
</ul>
<p>They all have inflations as well.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_14_pco_nullp_other.pdf" title="Figure: Other normal transformation. PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}." width="40%" type="application/pdf" />
<p class="caption">
Figure: Other normal transformation. PCO p-value inflation under null
distribution using z_{convert} and Sigma_{convert}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_14_pco_nullp_other">
Past versions of 02_14_pco_nullp_other.pdf
</button>
</p>
<div id="fig-02_14_pco_nullp_other" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87402c7054fa6d1127a4c4cb79a57153472fdf89/docs/asset/2024/02_14_pco_nullp_other.pdf" target="_blank">87402c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="jan-25" class="section level1">
<h1>Jan 25</h1>
<div id="saige-qtl-saige-qtl-pco" class="section level2">
<h2>1. SAIGE-QTL &amp; SAIGE-QTL-PCO</h2>
<div id="how-does-saige-qtl-work" class="section level3">
<h3>How does SAIGE-QTL work?</h3>
<ul>
<li><p>Step 1: fitting the null Poisson mixed model</p></li>
<li><p>Step 2: performing single-variant association tests</p></li>
<li><p>Step 3: obtaining gene-level p-values using the ACAT
test</p></li>
</ul>
</div>
<div id="questions-to-be-confirmed-before-running-saige-qtl"
class="section level3">
<h3>Questions to be confirmed before running SAIGE-QTL</h3>
<ul>
<li>Is association test of SAIGE-QTL (score test statistic) linear to Y?
It has to be so in order to calculate the correlation matrix of the
tests using correlation matrix of Y’s.</li>
</ul>
</div>
<div id="run-saige-qtl" class="section level3">
<h3>Run SAIGE-QTL</h3>
<ul>
<li>How is the number of cpeaks by SAIGE-QTL compared to RASQUAL and
scPME?</li>
</ul>
</div>
</div>
<div id="using-crd-defined-windows" class="section level2">
<h2>2. Using CRD-defined windows</h2>
</div>
<div id="using-two-datasets" class="section level2">
<h2>3. Using two datasets</h2>
</div>
</div>
<div id="jan-18" class="section level1">
<h1>Jan 18</h1>
<div id="distance-from-lead-snp-to-cpeaks-of-histone-marks-in-bulk-data"
class="section level2">
<h2>1. Distance from lead SNP to cpeaks of histone marks in bulk
data</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf" title="Figure: cpeaks by QTLTools across cis regions." width="40%" type="application/pdf" />
<p class="caption">
Figure: cpeaks by QTLTools across cis regions.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_dis_leadsnp_cpeak_histone_marks_cis500kb">
Past versions of 01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_dis_leadsnp_cpeak_histone_marks_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Change of cpeaks by QTLTools across various cis regions for bulk
histone marks</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_cpeak_qtltools_cis_reg_bulk.pdf" title="Figure: Change of cpeaks by QTLTools across various cis regions for bulk histone marks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Change of cpeaks by QTLTools across various cis regions for bulk
histone marks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_cpeak_qtltools_cis_reg_bulk">
Past versions of 01_18_cpeak_qtltools_cis_reg_bulk.pdf
</button>
</p>
<div id="fig-01_18_cpeak_qtltools_cis_reg_bulk" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_cpeak_qtltools_cis_reg_bulk.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="compare-our-cqtls-to-activity-cqtls-in-delaunea-paper"
class="section level2">
<h2>2. Compare our cQTLs to activity cQTLs in Delaunea paper</h2>
<div id="how-acrdqtls-were-defined" class="section level3">
<h3>How aCRDQTLs were defined?</h3>
<ul>
<li><p>LCL.aCRDQTLs - QTLs for the activity of CRDs in LCLs.</p></li>
<li><p>The activity of CRDs are defined as the mean activity of the
peaks it contains.</p></li>
<li><p>6157 significant aCRDs (out of 12583 total CRDs) at 5%
FDR.</p></li>
</ul>
</div>
<div id="signal-comparison-cwindowsccrds" class="section level3">
<h3>Signal comparison cwindows/cCRDs</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Comparison of cwindows and aCRDQTLs." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison of cwindows and aCRDQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<p>CACTI v.s. cCRDs -</p>
<ul>
<li><p>Many signals are missed by CACTI.</p>
<ul>
<li>Those aCRDs overlap multiple windows, while missed cwindows overlap
mostly one aCRD.</li>
</ul></li>
<li><p>Increase window size? 50kb, similar case.</p></li>
<li><p>Does the comparison even make sense given CRD (calculated using
three histone marks by ChIP-seq) v.s. ATAC-seq?</p></li>
</ul>
</div>
<div id="enrichment-of-ccrds-in-cwindows" class="section level3">
<h3>Enrichment of cCRDs in cwindows</h3>
<p><strong>Questions to be asked -</strong></p>
<ol style="list-style-type: decimal">
<li><p>If being a cwindow more likely be a aCRDQTL?</p></li>
<li><p>Restrict to cwindow overlap with CRDs</p></li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: If being a cwindow more likely be a aCRDQTL? If being a cwindow that overlaps with CRDs more likely be a aCRDQTL?" width="40%" type="application/pdf" />
<p class="caption">
Figure: If being a cwindow more likely be a aCRDQTL? If being a cwindow
that overlaps with CRDs more likely be a aCRDQTL?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>cwindows are enriched for aCRDQTLs, no matter for windows overlap
with a CRD or not.</p></li>
<li><p>The enrichment and OR are more significant than the enrichment of
cwindows for CRDs.</p></li>
</ul>
<p>To-do’s -</p>
<ul>
<li><p>Apply to previous other cases to see if there is significant
enrichment.</p></li>
<li><p>Change the name of cQTLs to something like cWindowQTLs?</p></li>
</ul>
</div>
</div>
<div
id="another-angle-to-explain-power-difference-between-cacti-and-rasqual---rasqual-mostly-captures-allelic-effects"
class="section level2">
<h2>3. Another angle to explain power difference between CACTI and
RASQUAL - RASQUAL mostly captures allelic effects?</h2>
<ul>
<li><p>No such information in RASQUAL output.</p></li>
<li><p>Previously, I said RASQUAL uses 10kb as cis window because the
SNP should be within the feature to have aligned reads overlap with the
SNP. This is wrong. RASQUAL can take larger cis window outside the
feature. They combined the pop effect and allelic effect for any SNPs.
See their paper.</p></li>
<li><p>So, to have a fair power comparsion, we still need to think of a
way to harmonize the multiple testing burden, since QTLTools have almost
three times more signals at 10kb cis region than 500kb cis
region.</p></li>
</ul>
</div>
</div>
<div id="jan-11" class="section level1">
<h1>Jan 11</h1>
<div id="some-insights" class="section level2">
<h2>0. some insights</h2>
<ul>
<li><p>Why cis region used by RASQUAL is so small?</p>
<p>Because in order to calculate the allelic effect, snps should be
located inside the sequenced feature within where the feature reads are
counted. Otherwise, RASQUAL effects are just between-individual signals
not allele-specific signals. Therefore, RASQUAL cis regions should be
the feature length on average.</p></li>
</ul>
</div>
<div id="pco-using-rasqua-sum-stats" class="section level2">
<h2>1. PCO using RASQUA sum stats</h2>
<div id="why" class="section level3">
<h3>Why?</h3>
<p>Current CACTI uses sum stats by QTLTools (uni-pop). We wanted to show
the advantage of aggregating muliple peaks, so to fairly compare with
RASQUAL which uses allelic effects, we want to use PCO on RASQUAL sum
stats.</p>
<p><strong>To-do’s -</strong></p>
<ul>
<li><p>Check PCO on RASQUAL p-values (not from z-scores), if it’s
feasible.</p></li>
<li><p>Use PCO to combine RASQUAL p’s.</p></li>
</ul>
</div>
<div id="why-pco-wont-work-to-combine-rasqual-ps"
class="section level3">
<h3>Why PCO won’t work to combine RASQUAL p’s?</h3>
<ul>
<li><p>Six tests of PCO combine p-values, not depending on the test
statistics.</p></li>
<li><p>But the distribution of six tests depends on <span
class="math inline">\(\Sigma\)</span>, which is calculated as the
correlation among residual y’s, is actually the covariance matrix of the
test statistic corresponding to the p-values (z-scores) across y’s. The
equivalence depends on the fact that the test statistic is linear to y
(see equation 2 in <a
href="https://onlinelibrary-wiley-com.proxy.uchicago.edu/doi/full/10.1111/biom.12735">paper</a>.</p></li>
<li><p>The issue is RASQUAL test statistic is not linear to y, which
make the calculation of <span class="math inline">\(\Sigma\)</span>
impossible. RASQUAL p-value for a pair of snp and feature comes from LRT
testing for <span class="math inline">\(\pi=0.5\)</span>, where <span
class="math inline">\(\pi\)</span> is the metric for cis regulatory
effect parameter (see “Hypothesis testing for inference of QTLs” in <a
href="https://www.nature.com/articles/ng.3467#Sec13">paper</a>). We need
to know the correlation among LRT test statistics of a snp across peaks,
but we can’t. It is difference case than where the test statistic is OLS
estimate and we can easily derive the correlations.</p></li>
</ul>
<blockquote>
<p>Under the null hypothesis, all parameters except π are estimated for
each feature independently, whereas π is set to 0.5, and we use a
likelihood-ratio test to compare the null and alternative hypotheses for
each SNP-feature combination using the <span
class="math inline">\(\chi^2\)</span> distribution with 1 degree of
freedom (for <span class="math inline">\(\pi\)</span>).</p>
</blockquote>
</div>
</div>
<div id="effect-of-the-choice-of-cis-region-on-cpeaks-by-qtltools"
class="section level2">
<h2>2. Effect of the choice of cis region on #cpeaks by QTLTools</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_cpeak_qtltools_cis_reg.pdf" title="Figure: cpeaks by QTLTools across cis regions." width="40%" type="application/pdf" />
<p class="caption">
Figure: cpeaks by QTLTools across cis regions.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_cpeak_qtltools_cis_reg">
Past versions of 01_11_cpeak_qtltools_cis_reg.pdf
</button>
</p>
<div id="fig-01_11_cpeak_qtltools_cis_reg" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_cpeak_qtltools_cis_reg.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li><p>Using 500kb as cis region in previous results should have less
signals than using shorter cis regions.</p></li>
<li><p>Lead SNPs are very close to cpeaks across various cis
regions.</p></li>
</ul></li>
</ul>
</div>
<div id="use-qtltools-cpeaks-for-signal-to-noise-ratio-plot"
class="section level2">
<h2>3. Use QTLTools cpeaks for signal-to-noise ratio plot</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Signal to noise ratio of missed &amp; replicated cwindows compared to QTLTools and RASQUAL." width="40%" type="application/pdf" />
<p class="caption">
Figure: Signal to noise ratio of missed &amp; replicated cwindows
compared to QTLTools and RASQUAL.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I added the third panel, where I compared the signal to noise ratio
of missed &amp; replicated cwindows compared to RASQUAL and the number
of signals (cpeaks) in a window was from QTLTools. To compare with the
first panel.</p>
<ul>
<li><p>Observations</p>
<ul>
<li><p>The missed &amp; replicated cwindows compared to RASQUAL have
even fewer cpeaks in the windows by QTLTools. This indicate that RASQUAL
lends power from allelic effects.</p></li>
<li><p>The power difference between RASQUAL and CACTI comes from two
aspects - aggregating multiple peaks and combining allelic
effects.</p></li>
</ul></li>
</ul>
</div>
<div id="characterization-of-cacti-signals-in-bulk-in-terms-of-crd"
class="section level2">
<h2>5. Characterization of CACTI signals in bulk in terms of CRD</h2>
<div id="rational" class="section level3">
<h3>Rational</h3>
<p>To check if CACTI finds cwindows of co-regulated peaks, we
investigate two questions -</p>
<ul>
<li><p>Do cwindows more likely overlap with any CRDs than
non-cwindows?</p></li>
<li><p>Are cwindows more likely with same CRDs than
non-cwindows?</p></li>
</ul>
<p>I checked cwindows by CACTI-P for mark H3K27ac and CACTI-S for mark
H3K36me.</p>
<p>(12583 CRDs on hg38. 12433 converted to hg19. 150 CRDs deleted.)</p>
</div>
<div id="cacti-p-for-mark-h3k27ac" class="section level3">
<h3>CACTI-P for mark H3K27ac</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" title="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb">
Past versions of
01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" title="Figure: Are cwindows more likely with same CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Are cwindows more likely with same CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb">
Past versions of
01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
</div>
<div id="cacti-s-for-mark-h3k36me3" class="section level3">
<h3>CACTI-S for mark H3K36me3</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" title="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new">
Past versions of
01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" title="Figure: Are cwindows more likely with same CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Are cwindows more likely with same CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new">
Past versions of
01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
</div>
</div>
<div id="use-50kb-window-to-be-consistent-with-previous-bulk-results"
class="section level2">
<h2>6. Use 50kb window to be consistent with previous bulk results?</h2>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       highr_0.10        compiler_4.2.3    pillar_1.9.0     
 [5] bslib_0.5.0       later_1.3.1       git2r_0.32.0      jquerylib_0.1.4  
 [9] tools_4.2.3       getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7   
[13] evaluate_0.21     lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3  
[17] rlang_1.1.1       cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7       
[21] xfun_0.39         fastmap_1.1.1     httr_1.4.6        stringr_1.5.0    
[25] knitr_1.43        fs_1.6.2          vctrs_0.6.3       sass_0.4.6       
[29] rprojroot_2.0.3   glue_1.6.2        R6_2.5.1          processx_3.8.2   
[33] fansi_1.0.4       rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3   
[37] whisker_0.4.1     ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5  
[41] httpuv_1.6.11     utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
