<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2022</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2022</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2024-04-25
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef1090b1fecef54522cbb07c77c3b22db0f7dfc3etargetblankf1090b1a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e" target="_blank">f1090b1</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreef1090b1fecef54522cbb07c77c3b22db0f7dfc3etargetblankf1090b1a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e" target="_blank">f1090b1</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Unstaged changes:
    Modified:   analysis/Trans.Rmd
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2022.Rmd</code>) and HTML
(<code>docs/Log2022.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e/docs/Log2022.html" target="_blank">f1090b1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ee589703409e38f18cd760bf00a45681c72df3b6/docs/Log2022.html" target="_blank">ee58970</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0/docs/Log2022.html" target="_blank">c50b98f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2551271290da8aed12b4fadc67e2b68f50eb5c19/docs/Log2022.html" target="_blank">2551271</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d902cc148bcf404da30b62cfc9d3f8cf512523a7/analysis/Log2022.Rmd" target="_blank">d902cc1</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
<td>
typo edits
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a638585ae01fe4d3ff8fc8a9b031d3c7bcda718f/docs/Log2022.html" target="_blank">a638585</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6b293721f225095bef9142e95d56826dd2721abd/analysis/Log2022.Rmd" target="_blank">6b29372</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0a2db0770accbe46e6e026cd7b42a88d93c91734/docs/Log2022.html" target="_blank">0a2db07</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0d49cac50f5facf9177437d8ca9ef31bd9c5e2b1/analysis/Log2022.Rmd" target="_blank">0d49cac</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/27b1c909810d37115979a93caf079e52a99c5cf0/docs/Log2022.html" target="_blank">27b1c90</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cab555069807783a4ff726fcdd0cba2c1154921d/analysis/Log2022.Rmd" target="_blank">cab5550</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
<td>
add ref link of section of ratio of module and chr
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/124a92ba1923b6174f49371f38415c3eed785ce7/docs/Log2022.html" target="_blank">124a92b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/21796d911d17c644197b63975ce56e6b9a2dd20a/analysis/Log2022.Rmd" target="_blank">21796d9</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-16
</td>
<td>
correct the bonferroni cutoff for signals when keep cell
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1744c2122664466804777ee23bcbd8103cab9c06/docs/Log2022.html" target="_blank">1744c21</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8c62d610393a06fcbace69e644139681eaa280ac/analysis/Log2022.Rmd" target="_blank">8c62d61</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
<td>
update msig in DGN signals’ numbers and figures by add
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/cef27a7c11cee2d8704cdb046c14a50c1fde180e/docs/Log2022.html" target="_blank">cef27a7</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8a0e1cd8d96e024e10b2e02ac715d1e67b7c4be2/analysis/Log2022.Rmd" target="_blank">8a0e1cd</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/b602908a091560058bfdace4b09d676c9fd81648/docs/Log2022.html" target="_blank">b602908</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2a43458a23e3e8e477613e85cfbf7be152075a61/analysis/Log2022.Rmd" target="_blank">2a43458</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/afd30244c1d6015b3399a11b39f133d197e71312/docs/Log2022.html" target="_blank">afd3024</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ff45d00e7593c9e9c5608b83cb6d032a7701d46c/analysis/Log2022.Rmd" target="_blank">ff45d00</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-24
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/56c966a125d5e4834bcd06957bc3d068ef148b9b/docs/Log2022.html" target="_blank">56c966a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/31d1e52e30f3155fa87a73144d7d3f33844555e3/analysis/Log2022.Rmd" target="_blank">31d1e52</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-18
</td>
<td>
test side by side figures
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0ca4da8f01681dacc86c49badd5e7d5467a8a137/docs/Log2022.html" target="_blank">0ca4da8</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/985ba117837716cd83aca130ba3316b715148a46/analysis/Log2022.Rmd" target="_blank">985ba11</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-17
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a85e562f2b4a5adda738859e9b6a2894e92967f1/docs/Log2022.html" target="_blank">a85e562</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/4ea65028eb2ba186b24054028db88fff168bf86e/analysis/Log2022.Rmd" target="_blank">4ea6502</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
<td>
adjust figures
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d3ffddd3a91ecb0fe9f511b9f8087b879530bbc5/docs/Log2022.html" target="_blank">d3ffddd</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/06cbd57a0aa8c6fdb3d645f1412f44c1b29746bf/analysis/Log2022.Rmd" target="_blank">06cbd57</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
<td>
New week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4edeb7eb9bdeed02c1c3bd883556c8b595f134f5/docs/Log2022.html" target="_blank">4edeb7e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/4870a2e0a059fec8c8edacf9e7c940a382ef8c40/analysis/Log2022.Rmd" target="_blank">4870a2e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/dbaa2360ad1f43a34cff6436d18dfaf25fb3c912/docs/Log2022.html" target="_blank">dbaa236</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ba5c42bc41e8ce0ba72092d5e5693061d733e69e/analysis/Log2022.Rmd" target="_blank">ba5c42b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
<td>
Add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/07720fcb6be6466440965eb9e93c92dd290b9af8/docs/Log2022.html" target="_blank">07720fc</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/41a531bf06460b1055ba55bdde47675fdbfeba05/analysis/Log2022.Rmd" target="_blank">41a531b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-11
</td>
<td>
update theme
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/dacdae72dc64e4e22ad915006c0cb24a5c938cc1/docs/Log2022.html" target="_blank">dacdae7</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-11
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/e6a219967e37e37280572b63403aa4dd684738cd/docs/Log2022.html" target="_blank">e6a2199</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-10
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ed7392ac29ac9c3bc61f7ca2e3054b64dbdce56b/analysis/Log2022.Rmd" target="_blank">ed7392a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-10
</td>
<td>
update rmd
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f60b0b9b9826f90567e7e0ea843b7ee2ce93f23f/docs/Log2022.html" target="_blank">f60b0b9</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/06afdff95ba63f4c4921ca43985a159904248918/analysis/Log2022.Rmd" target="_blank">06afdff</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-09
</td>
<td>
update error bar from se to 0.95 CI
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0155a0fe37e0797776139bca94f1e814f140e216/docs/Log2022.html" target="_blank">0155a0f</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/518a1c0b1cc18d00fc95c40d23493d1a1eb4b562/analysis/Log2022.Rmd" target="_blank">518a1c0</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add more details
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/607ff2b1555cc61e6395a5ad8c3a765c33313958/docs/Log2022.html" target="_blank">607ff2b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/397fb52e1b7304df8501600cd1fcc959f3426336/analysis/Log2022.Rmd" target="_blank">397fb52</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/21ae027be39b4c1ff7c145334aef15997e0b1dce/docs/Log2022.html" target="_blank">21ae027</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0db9d7426b737c2a9901a671045b1a2e73485d3a/analysis/Log2022.Rmd" target="_blank">0db9d74</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-08
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/0835a9c2081ac25f70b320c7fe58609511fc41aa/docs/Log2022.html" target="_blank">0835a9c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6d481ca00d02c00197745445efbaafd75bf8232b/analysis/Log2022.Rmd" target="_blank">6d481ca</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
fix a typo
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1b8143daad903e7b6882bf9c8eba4163a742fb0a/docs/Log2022.html" target="_blank">1b8143d</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/0626baa5c59dfa308ff90956f7c13b0c65f80d1f/analysis/Log2022.Rmd" target="_blank">0626baa</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
<td>
add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/59c3d71239d38efbc42825c93788c13f500bed6a/docs/Log2022.html" target="_blank">59c3d71</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9d92b99b9d5aa7746f02630955a7a8f23ed02853/analysis/Log2022.Rmd" target="_blank">9d92b99</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
<td>
Add a new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2e6c03934e4cd52fbb585020fffa30a179c6e572/docs/Log2022.html" target="_blank">2e6c039</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fae1f4ee357d211f63286d228c9bcc6c0b68d4c9/analysis/Log2022.Rmd" target="_blank">fae1f4e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
adjust fig position
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/3395a267589af572688e609b13621db4d8ca104d/docs/Log2022.html" target="_blank">3395a26</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d0a08e63e7fc87b48409fd0dcc879ce6d71ffae6/analysis/Log2022.Rmd" target="_blank">d0a08e6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
adjust fig position
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8b9071e4982b644e5868ebd9e0a417e578ab0257/docs/Log2022.html" target="_blank">8b9071e</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/424015b9c6ebab80f3ade02c9c2f48f5c173bba3/analysis/Log2022.Rmd" target="_blank">424015b</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/72025bf9b9f91c57745d7910e076d79a5fc2ee18/docs/Log2022.html" target="_blank">72025bf</a>
</td>
<td>
liliw-w
</td>
<td>
2022-02-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/089c60942bfc4e7416bbc5bcf73accd65aeef082/analysis/Log2022.Rmd" target="_blank">089c609</a>
</td>
<td>
liliw-w
</td>
<td>
2022-02-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1274e244c3b7d06966b3351caa0bff74870a3382/docs/Log2022.html" target="_blank">1274e24</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9976af6f087dd6b4546a78fc8936e5dc54c7e01a/analysis/Log2022.Rmd" target="_blank">9976af6</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/bd69a21c8b6909c5174347c0c0c724d7169dc47c/docs/Log2022.html" target="_blank">bd69a21</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/28d0a2b13cae4c13c5979ffcf05154bc2aa15603/analysis/Log2022.Rmd" target="_blank">28d0a2b</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ec0a2af10bac6589b98ea6818de13cd3bda98a33/docs/Log2022.html" target="_blank">ec0a2af</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9e5c2e02a0e8f42e2fd9c4f697dd3d8c16fff6e2/analysis/Log2022.Rmd" target="_blank">9e5c2e0</a>
</td>
<td>
llw
</td>
<td>
2022-01-04
</td>
<td>
new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/97b4ca1336ea92e396991f1569177c14360eb07c/docs/Log2022.html" target="_blank">97b4ca1</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4c90bd8ce853f24d0248eee7fe6a8e038cb3d0a0/docs/Log2022.html" target="_blank">4c90bd8</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6dc3f9c96110c0be94a620950616ee99eaa806f0/analysis/Log2022.Rmd" target="_blank">6dc3f9c</a>
</td>
<td>
llw
</td>
<td>
2022-01-03
</td>
<td>
add a new year
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="oct-5" class="section level2">
<h2>Oct 5</h2>
<div id="coloc-between-trans-eqlts-of-msigdb-modules-and-blood-traits"
class="section level3">
<h3>1. Coloc between trans-eQLTs of MSigDB modules and blood traits</h3>
<p>We observed something interesting on one of the MSigDB module:
HALLMARK_HEME_METABOLISM (module 9). The trans signals corresponding to
this pathway are near genes with relevant functions with the pathway in
red blood cells.</p>
<p>Therefore, we decided to look at the colocalization of
HALLMARK_HEME_METABOLISM and red blood cell traits.</p>
<p>So, I performed colocalization of trans-eQTLs of 50 MSigDB pathways
and GWAS loci of 29 blood related traits.</p>
<div id="a-few-numbers" class="section level4">
<h4>A few numbers</h4>
<ul>
<li><p>50 MSigDB pathways</p></li>
<li><p>29 blood related traits</p></li>
<li><p>129 trans- regions and 112 merged regions</p>
<p>There are 129 regions (with the region’s lead SNP as a trans-eQTL)
that were used to do coloc. After merging close lead SNPs of the
regions, there are 112 merged regions left.</p></li>
</ul>
</div>
<div id="results" class="section level4">
<h4>Results</h4>
<ul>
<li>Proportion of regions &amp; merged regions that coloc with each of
traits</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/1005_coloc_region_prop_merged.pdf" title="Figure: Coloc proportion of trans loci of MSig pathways and blood traits, based on 129 originally defined regions and 112 merged regions. Blue: original non-merged regions. Green: merged regions. Dark: coloc." width="80%" type="application/pdf" />
<p class="caption">
Figure: Coloc proportion of trans loci of MSig pathways and blood
traits, based on 129 originally defined regions and 112 merged regions.
Blue: original non-merged regions. Green: merged regions. Dark: coloc.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1005_coloc_region_prop_merged">
Past versions of 1005_coloc_region_prop_merged.pdf
</button>
</p>
<div id="fig-1005_coloc_region_prop_merged" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2a8ec6a6e193a65b126aca8b450efa469ba6fa0a/docs/asset/2022/1005_coloc_region_prop_merged.pdf" target="_blank">2a8ec6a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>See numerical result</strong>:</p>
<p><code>/project2/xuanyao/llw/MODULES/MSigDB/coloc_MSigDB/ukbb_all/coloc_region_prop_merged.txt</code>.</p>
<ul>
<li>Number of merged regions that have coloc for each pair of (pathway,
trait)</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/1005_reg_module_trait_merged.pdf" title="Figure: Number of coloc merged regions for all pairs of (pathway, trait). Colors: Traits of various types." width="80%" type="application/pdf" />
<p class="caption">
Figure: Number of coloc merged regions for all pairs of (pathway,
trait). Colors: Traits of various types.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-1005_reg_module_trait_merged">
Past versions of 1005_reg_module_trait_merged.pdf
</button>
</p>
<div id="fig-1005_reg_module_trait_merged" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2a8ec6a6e193a65b126aca8b450efa469ba6fa0a/docs/asset/2022/1005_reg_module_trait_merged.pdf" target="_blank">2a8ec6a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>See numerical result for the specific coloc regions of
(pathway, trait):</strong></p>
<p><code>/project2/xuanyao/llw/MODULES/MSigDB/coloc_MSigDB/ukbb_all/coloc_region_summary_all_merged_annot.txt</code>.</p>
<p><strong>A few important columns</strong>:</p>
<ol style="list-style-type: decimal">
<li><p>Region: the coloc region defined by the lead trans-eQTLs in that
region and the correpsonding module</p></li>
<li><p>module, module_annot: the coloc module and its MSig pathway
annotation</p></li>
<li><p>Phenocode, trait: the coloc trait</p></li>
<li><p>PP.H4.abf: coloc PP4</p></li>
<li><p>merged_region: the merged region that the region belong
to</p></li>
<li><p>merged_region_n: the number of regions in the merged
region</p></li>
</ol>
<p>See file
<code>/project2/xuanyao/llw/MODULES/MSigDB/coloc_MSigDB/ukbb_all/coloc_region_summary_M9_HEME.txt</code>
for the <strong>coloc regions of only
HALLMARK_HEME_METABOLISM</strong>.</p>
<ul>
<li><p>Number of (not merged) regions that have coloc for each pair of
(pathway, trait)</p>
<p>See <a href="asset/2022/1005_reg_module_trait.pdf"
width="45%">non-merged coloc regions</a>.</p></li>
</ul>
</div>
<div id="observations" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>HALLMARK_HEME_METABOLISM (module9) and red blood cells traits
share many coloc loci.</p></li>
<li><p>Particularly more than white blood cell traits.</p></li>
<li><p>HALLMARK_HEME_METABOLISM (module9) and platelet traits also share
a few coloc regions.</p></li>
</ul>
</div>
</div>
</div>
<div id="sep-21" class="section level2">
<h2>Sep 21</h2>
<div id="h2-enrichment-of-module-trait-re-ordered-by-clustering"
class="section level3">
<h3>1. h2 enrichment of (module, trait) re-ordered by clustering</h3>
<div id="how" class="section level4">
<h4>How?</h4>
<ul>
<li><p>Use distance of -log10(p of h2 enrich) to cluster modules and
traits.</p></li>
<li><p>Draw -log10(p of h2 enrich) as shades.</p></li>
<li><p>Annotate with trait types.</p></li>
</ul>
</div>
<div id="figures-observations" class="section level4">
<h4>Figures &amp; observations</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0921_h2_enrich_order_by_dist.pdf" title="Figure: LEFT: only coloc modules. RIGHT: all modules." width="50%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_h2_enrich_order_by_dist">
Past versions of 0921_h2_enrich_order_by_dist.pdf
</button>
</p>
<div id="fig-0921_h2_enrich_order_by_dist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_h2_enrich_order_by_dist.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0921_h2_enrich_order_by_dist_all_modules.pdf" title="Figure: LEFT: only coloc modules. RIGHT: all modules." width="50%" type="application/pdf" />
<p class="caption">
Figure: LEFT: only coloc modules. RIGHT: all modules.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_h2_enrich_order_by_dist_all_modules">
Past versions of 0921_h2_enrich_order_by_dist_all_modules.pdf
</button>
</p>
<div id="fig-0921_h2_enrich_order_by_dist_all_modules" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_h2_enrich_order_by_dist_all_modules.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="eqtlgen-with-msigdb-modules" class="section level3">
<h3>2. eQTLGen with MSigDB modules</h3>
<div id="relative-path" class="section level4">
<h4>Relative path</h4>
<p><strong>All results &amp; figures for this section are in folder
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/</code>.</strong></p>
</div>
<div id="signals-before-removing-any-modules-based-on-ratio"
class="section level4">
<h4><del>Signals before removing any modules based on ratio</del></h4>
<ul>
<li><p><del>Signal file:
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/postanalysis/signal.txt</code></del></p></li>
<li><p><del>Numbers: 72696 significant (snp, module) pairs,
corresponding to 9309 trans-eQTLs and all 50 modules.</del></p></li>
<li><p><del>Signal distribution</del></p>
<ul>
<li><p><del>on modules, <a
href="asset/2022/0921_signal_module.pdf">figure</a></del></p></li>
<li><p><del>on chr’s, <a
href="asset/2022/0921_signal_chr.pdf">figure</a></del></p></li>
<li><p><del>on (module, chr), <a
href="asset/2022/0921_signal_module_chr.pdf">figure</a></del></p></li>
</ul></li>
<li><p><del>Observation</del></p>
<p><del>Inflated signals for the first few large module. Thus, remove
these modules based on their <span class="math inline">\(\frac{null \
snps}{module \ size}\)</span> ratios. See updated signals in next
section.</del></p></li>
</ul>
</div>
<div id="signals-after-removing-module-chr-with-ratio-50"
class="section level4">
<h4>Signals after removing (module, chr) with ratio &lt; 50</h4>
<p>In <a href="#ratio_of_module_and_chr">previous section</a>, I gave
<span class="math inline">\(\frac{null \ snps}{module \ size}\)</span>
ratio for each pair of (module, chr). So here, I remove a (module, chr)
from signals if its ratio &lt; 50.</p>
<ul>
<li><p>How many (module, chr) are saved compared to only module-based
ratio?</p>
<ul>
<li><p>module-based ratio: 11 modules, 242 (module, chr) in
total.</p></li>
<li><p>chr-based ratio: 253 (module, chr) in total.</p></li>
<li><p>11 more were saved.</p></li>
</ul></li>
<li><p>Result files</p>
<ul>
<li><p>Signal file:
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/postanalysis/signal_rm_infl_ratio_50_module_chr.txt</code></p></li>
<li><p>Annotation files with module pathway, gwas catalog trait, cis-
genes:
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/postanalysis/signal_cis_genes_w_annot_rm_infl_ratio_50_module_chr.txt</code></p></li>
</ul></li>
<li><p>Numbers: 2134 significant (snp, module) pairs, corresponding to
988 trans-eQTLs and 15 modules.</p></li>
<li><p>Signal distribution</p>
<ul>
<li><p>on modules, <a
href="asset/2022/0921_signal_module_rm_infl_ratio_50_module_chr.pdf">figure</a></p></li>
<li><p>on chr’s, <a
href="asset/2022/0921_signal_chr_rm_infl_ratio_50_module_chr.pdf">figure</a></p></li>
<li><p>on (module, chr), <a
href="asset/2022/0921_signal_module_chr_rm_infl_ratio_50_module_chr.pdf">figure</a></p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0921_signal_module_rm_infl_ratio_50_module_chr.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_module_rm_infl_ratio_50_module_chr">
Past versions of 0921_signal_module_rm_infl_ratio_50_module_chr.pdf
</button>
</p>
<div id="fig-0921_signal_module_rm_infl_ratio_50_module_chr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_module_rm_infl_ratio_50_module_chr.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2022/0921_signal_chr_rm_infl_ratio_50_module_chr.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_chr_rm_infl_ratio_50_module_chr">
Past versions of 0921_signal_chr_rm_infl_ratio_50_module_chr.pdf
</button>
</p>
<div id="fig-0921_signal_chr_rm_infl_ratio_50_module_chr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_chr_rm_infl_ratio_50_module_chr.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0921_signal_module_chr_rm_infl_ratio_50_module_chr.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" />
<p class="caption">
Figure: LEFT: on modules. MIDDLE: on chr’s. RIGHT: on (module, chr).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_module_chr_rm_infl_ratio_50_module_chr">
Past versions of 0921_signal_module_chr_rm_infl_ratio_50_module_chr.pdf
</button>
</p>
<div id="fig-0921_signal_module_chr_rm_infl_ratio_50_module_chr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_module_chr_rm_infl_ratio_50_module_chr.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observation</p>
<p>Looks like the additionally added module 37 has many signals, I
suspect if these signals are inflated. So I also looked at signals after
removing modules based on module-based ratios, as in <a
href="#msig_eqtlgen_module_ratio">previous section</a> and what we used
in eQTLGen with co-expression modules.</p></li>
</ul>
</div>
<div id="signals-after-removing-modules-with-ratio-50"
class="section level4">
<h4>Signals after removing modules with ratio &lt; 50</h4>
<ul>
<li><p>Result files</p>
<ul>
<li><p>Signal file:
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/postanalysis/signal_rm_infl_ratio_50.txt</code></p></li>
<li><p>Annotation files with module pathway, gwas catalog trait, cis-
genes:
<code>/project2/xuanyao/llw/eQTLGen/MSigDB_est_Sigma/postanalysis/signal_cis_genes_w_annot_rm_infl_ratio_50.txt</code></p></li>
</ul></li>
<li><p>Numbers: 1756 significant (snp, module) pairs, corresponding to
870 trans-eQTLs and 11 modules.</p></li>
<li><p>Signal distribution</p>
<ul>
<li><p>on modules, <a
href="asset/2022/0921_signal_module_rm_infl_ratio_50.pdf">figure</a></p></li>
<li><p>on chr’s, <a
href="asset/2022/0921_signal_chr_rm_infl_ratio_50.pdf">figure</a></p></li>
<li><p>on (module, chr), <a
href="asset/2022/0921_signal_module_chr_rm_infl_ratio_50.pdf">figure</a></p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0921_signal_module_rm_infl_ratio_50.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_module_rm_infl_ratio_50">
Past versions of 0921_signal_module_rm_infl_ratio_50.pdf
</button>
</p>
<div id="fig-0921_signal_module_rm_infl_ratio_50" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_module_rm_infl_ratio_50.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2022/0921_signal_chr_rm_infl_ratio_50.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_chr_rm_infl_ratio_50">
Past versions of 0921_signal_chr_rm_infl_ratio_50.pdf
</button>
</p>
<div id="fig-0921_signal_chr_rm_infl_ratio_50" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_chr_rm_infl_ratio_50.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0921_signal_module_chr_rm_infl_ratio_50.pdf" title="Figure: LEFT: on modules. MIDDLE: on chr's. RIGHT: on (module, chr)." width="30%" type="application/pdf" />
<p class="caption">
Figure: LEFT: on modules. MIDDLE: on chr’s. RIGHT: on (module, chr).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_module_chr_rm_infl_ratio_50">
Past versions of 0921_signal_module_chr_rm_infl_ratio_50.pdf
</button>
</p>
<div id="fig-0921_signal_module_chr_rm_infl_ratio_50" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/be8dd1c20d43d279c051423c02e0f6d67dd01a1c/docs/asset/2022/0921_signal_module_chr_rm_infl_ratio_50.pdf" target="_blank">be8dd1c</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="eqtlgen-with-co-expression-modules" class="section level3">
<h3>3. eQTLGen with co-expression modules</h3>
<div id="relative-path-1" class="section level4">
<h4>Relative path</h4>
<p><strong>All results &amp; figures for this section are in folder
<code>/project2/xuanyao/llw/eQTLGen/DGN_est_Sigma/</code>.</strong></p>
</div>
<div id="annotate-trans-eqtls-with-gwas-catalog-snps-and-traits"
class="section level4">
<h4>Annotate trans-eQTLs with gwas catalog snps and traits</h4>
<ul>
<li><p>Result files</p>
<ul>
<li><p>Signal file:
<code>/project2/xuanyao/llw/eQTLGen/DGN_est_Sigma/postanalysis/signal_rm_infl_ratio_50.txt</code></p></li>
<li><p>Annotation files with module pathway, gwas catalog trait, cis-
genes:
<code>/project2/xuanyao/llw/eQTLGen/DGN_est_Sigma/postanalysis/signal_cis_genes_w_annot_rm_infl_ratio_50.txt</code></p></li>
</ul></li>
<li><p>Numbers: 8116 significant (snp, module) pairs, corresponding to
2161 trans-eQTLs and 122 modules.</p></li>
</ul>
</div>
</div>
<div id="re-draw-signals-for-module-chr" class="section level3">
<h3>4. Re-draw #signals for (module, chr)</h3>
<p>Make chr the chromosome position.</p>
<p>P-value as point size.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0921_signal_module_chr_pos.pdf" title="Figure: Signals for (chr, module). Y-axis is chromosome postiion. X-axis is module. Each point is a signal. Point size means -logP." width="70%" type="application/pdf" />
<p class="caption">
Figure: Signals for (chr, module). Y-axis is chromosome postiion. X-axis
is module. Each point is a signal. Point size means -logP.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0921_signal_module_chr_pos">
Past versions of 0921_signal_module_chr_pos.pdf
</button>
</p>
<div id="fig-0921_signal_module_chr_pos" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2a8ec6a6e193a65b126aca8b450efa469ba6fa0a/docs/asset/2022/0921_signal_module_chr_pos.pdf" target="_blank">2a8ec6a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-10-04
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="figures-for-eqtlgen-results-with-co-expression-modules"
class="section level3">
<h3>5. Figures for eQTLGen results with co-expression modules</h3>
<div id="signal-figure" class="section level4">
<h4>Signal figure</h4>
<ul>
<li><p>#signals for (chr, module)</p>
<p><code>/project2/xuanyao/llw/eQTLGen/DGN_est_Sigma/plot/signal_module_chr_rm_infl_ratio_50.pdf</code></p></li>
<li><p>-logp for (chr postion, module)</p>
<p><code>/project2/xuanyao/llw/eQTLGen/DGN_est_Sigma/plot/signal_module_chr_rm_infl_ratio_50_chr_pos.pdf</code></p></li>
</ul>
</div>
<div id="cis--gene-manhattan-figure" class="section level4">
<h4>Cis- gene manhattan figure</h4>
</div>
</div>
<div id="trans--coloc-with-cis--genes" class="section level3">
<h3>6. trans- coloc with cis- genes</h3>
<p>Updated numbers on merged regions,</p>
<ul>
<li><p>255 trans- loci</p></li>
<li><p>179 merged trans- loci</p></li>
<li><p>51 merged trans- loci coloc with at least one gene’s
eQTLs</p></li>
<li><p>41 merged trans- loci coloc with at least one gene’s
sQTLs</p></li>
<li><p>Overall, 60 merged trans- loci coloc with at least one gene’s
eQTL or sQTLs</p></li>
</ul>
</div>
</div>
<div id="sep-14" class="section level2">
<h2>Sep 14</h2>
<div id="msigdb-modules-in-eqtlgen" class="section level3">
<h3>1. MSigDB modules in eQTLGen</h3>
<div id="fix-error" class="section level4">
<h4>Fix error</h4>
<ul>
<li><p>Weird observation</p>
<p>Why for some modules, so many genes were removed due to being cis
genes in the module (<code>num_gene_snp_used_M*.pdf</code>)? In
contrast, for eQTLGen-DGN modules, not many genes were removed due to
being cis to SNPs.</p></li>
<li><p>The error (code)</p>
<p>Wrong genes in splitted z matrices
<code>z.module*.chr*.txt.gz</code>.</p></li>
</ul>
</div>
<div id="updates" class="section level4">
<h4>Updates</h4>
<ol style="list-style-type: decimal">
<li><p>Re-split z matrices for (module, chr).</p></li>
<li><p>Update sizes of various gene sets for each module
(<code>num_gene_snp_used_M*.pdf</code>).</p></li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0914_num_gene_snp_used_M1.pdf" title="Figure: Updated number of genes for a gene used for trans-PCO." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_num_gene_snp_used_M1">
Past versions of 0914_num_gene_snp_used_M1.pdf
</button>
</p>
<div id="fig-0914_num_gene_snp_used_M1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_num_gene_snp_used_M1.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0914_num_gene_snp_used_M50.pdf" title="Figure: Updated number of genes for a gene used for trans-PCO." width="40%" type="application/pdf" />
<p class="caption">
Figure: Updated number of genes for a gene used for trans-PCO.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_num_gene_snp_used_M50">
Past versions of 0914_num_gene_snp_used_M50.pdf
</button>
</p>
<div id="fig-0914_num_gene_snp_used_M50" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_num_gene_snp_used_M50.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="3" style="list-style-type: decimal">
<li>Re-run trans-PCO on the updated splitted z matrices.</li>
</ol>
</div>
<div id="signals" class="section level4">
<h4>Signals</h4>
</div>
</div>
<div id="inflation-simulation-to-choose-modules" class="section level3">
<h3>2. Inflation simulation to choose modules</h3>
<p>As in <a href="#eqtlgen_inflation_simulation">previous section</a>, I
looked at how <span class="math inline">\(\hat{\Sigma}\)</span> can lead
to inflated signals given different ratios of <span
class="math inline">\(\frac{null \ snps}{module \ size}\)</span>. I
simulated ratios 1, 5, 10, 50, 100, 150. As a result, we decided to use
50 as the ratio cutoff to select modules for final signal
identification.</p>
<p>However, in the case of MSigDB modules in eQTLGen, not many modules
have ratios larger than 50. So, we were interested if any ratio between
10 and 50 can still lead to non-inflated signals, so that we can save
more MSigDB modules for signal identification.</p>
<div id="add-ratios-between-10-and-50" class="section level4">
<h4>Add ratios between 10 and 50</h4>
<p>Therefore, I added a few ratios between 10 and 50, <em>(10, 20, 30,
40, 50, 60, 70)</em>, to simulations to look at if there is any p-value
inflations.</p>
</div>
<div id="results-and-observations" class="section level4">
<h4>Results and observations</h4>
<ul>
<li>Histogram and qq-plots</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0914_plot_qq_Sigma_DGN_module100.rds.pdf" title="Figure: QQ-plots of p-values under estimated sigmas of various ratios between 10 and 50." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_plot_qq_Sigma_DGN_module100.rds">
Past versions of 0914_plot_qq_Sigma_DGN_module100.rds.pdf
</button>
</p>
<div id="fig-0914_plot_qq_Sigma_DGN_module100.rds" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_plot_qq_Sigma_DGN_module100.rds.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2022/0914_plot_qq_Sigma_DGN_module150.rds.pdf" title="Figure: QQ-plots of p-values under estimated sigmas of various ratios between 10 and 50." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_plot_qq_Sigma_DGN_module150.rds">
Past versions of 0914_plot_qq_Sigma_DGN_module150.rds.pdf
</button>
</p>
<div id="fig-0914_plot_qq_Sigma_DGN_module150.rds" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_plot_qq_Sigma_DGN_module150.rds.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0914_plot_qq_Sigma_DGN_module166.rds.pdf" title="Figure: QQ-plots of p-values under estimated sigmas of various ratios between 10 and 50." width="30%" type="application/pdf" />
<p class="caption">
Figure: QQ-plots of p-values under estimated sigmas of various ratios
between 10 and 50.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_plot_qq_Sigma_DGN_module166.rds">
Past versions of 0914_plot_qq_Sigma_DGN_module166.rds.pdf
</button>
</p>
<div id="fig-0914_plot_qq_Sigma_DGN_module166.rds" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_plot_qq_Sigma_DGN_module166.rds.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Ratio 20 is too low. There are obvious inflations.</li>
</ul>
</div>
<div id="ratio_of_module_and_chr" class="section level4">
<h4>Update ratio plot</h4>
<p>In <a href="#msig_eqtlgen_module_ratio">previous section</a>, I
looked at the ratio of each MSigDB module, by the definition of <span
class="math inline">\(\frac{null \ snps}{eqtlgen \ genes \ in \
module}\)</span>.</p>
<p>I looked at a more accurate definition of the actually ratio for each
module, i.e. $. Because for a (SNP, module), the actually used genes are
module size - gene not in eqtlgen - remove cis genes - remove cross map
gene. So I thought using this new definition can increase the ratio.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0914_plot_nullSNP_module_size_pthre1e-04_ratio_chr.pdf" title="Figure: Ratio used for each (module, chr). Each dot is ratio for a chr. Red dot is the ratio for each module by #null SNPs/#genes in module." width="70%" type="application/pdf" />
<p class="caption">
Figure: Ratio used for each (module, chr). Each dot is ratio for a chr.
Red dot is the ratio for each module by #null SNPs/#genes in module.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0914_plot_nullSNP_module_size_pthre1e-04_ratio_chr">
Past versions of 0914_plot_nullSNP_module_size_pthre1e-04_ratio_chr.pdf
</button>
</p>
<div id="fig-0914_plot_nullSNP_module_size_pthre1e-04_ratio_chr"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0914_plot_nullSNP_module_size_pthre1e-04_ratio_chr.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>The ratio increases to some extent.</p>
</div>
</div>
</div>
<div id="aug-30-sep-7" class="section level2">
<h2>Aug 30 &amp; Sep 7</h2>
<div id="eqtlgen-trans-eqtls-signals-under-msigdb-modules"
class="section level3">
<h3>1. eQTLGen trans-eQTLs signals under MSigDB modules</h3>
<div id="msigdb-modules-and-overlap-with-eqtlgen-genes"
class="section level4">
<h4>MSigDB modules and overlap with eQTLGen genes</h4>
<ul>
<li><p>Numbers</p>
<p>There are 19250 genes analyzed in eQTLGen for trans-
associations.</p>
<p>There are 7321 (uniquely 4383 ) genes in MSigDB modules.</p>
<p>And there are 6114 (uniquely 3558) MSigDB genes overlapped with
eQTLGen genes.</p></li>
<li><p>Figures</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0830_nmodule_msigdb_gene.pdf" title="Figure: LEFT: Each module has overlapped genes. How many modules do every gene belongs to? RIGHT: Size of MSigDB modules and how many are eQTLGen genes?" width="50%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0830_nmodule_msigdb_gene">
Past versions of 0830_nmodule_msigdb_gene.pdf
</button>
</p>
<div id="fig-0830_nmodule_msigdb_gene" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0830_nmodule_msigdb_gene.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0830_msigdb_module_eqtlgen.pdf" title="Figure: LEFT: Each module has overlapped genes. How many modules do every gene belongs to? RIGHT: Size of MSigDB modules and how many are eQTLGen genes?" width="50%" type="application/pdf" />
<p class="caption">
Figure: LEFT: Each module has overlapped genes. How many modules do
every gene belongs to? RIGHT: Size of MSigDB modules and how many are
eQTLGen genes?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0830_msigdb_module_eqtlgen">
Past versions of 0830_msigdb_module_eqtlgen.pdf
</button>
</p>
<div id="fig-0830_msigdb_module_eqtlgen" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0830_msigdb_module_eqtlgen.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="choose-z-cutoff-to-define-null-snps-for-estimating-sigma"
class="section level4">
<h4>Choose z cutoff to define null snps for estimating <span
class="math inline">\(\Sigma\)</span></h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0907_msigdb_num_nullSNP.pdf" title="Figure: Which z-score cutoff should be chosen for defining null SNPs to estimate Sigma?" width="70%" type="application/pdf" />
<p class="caption">
Figure: Which z-score cutoff should be chosen for defining null SNPs to
estimate Sigma?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_msigdb_num_nullSNP">
Past versions of 0907_msigdb_num_nullSNP.pdf
</button>
</p>
<div id="fig-0907_msigdb_num_nullSNP" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0907_msigdb_num_nullSNP.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Use <span class="math inline">\(10^{-4}\)</span>.</p>
</div>
<div id="msig_eqtlgen_module_ratio" class="section level4">
<h4>Ratios</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0907_plot_nullSNP_module_size_pthre1e-04.pdf" title="Figure: LEFT: module size v.s. null SNPs that can be used for Sigma estimation. RIGHT: Ratio of these two numbers for each module." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_plot_nullSNP_module_size_pthre1e-04">
Past versions of 0907_plot_nullSNP_module_size_pthre1e-04.pdf
</button>
</p>
<div id="fig-0907_plot_nullSNP_module_size_pthre1e-04" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0907_plot_nullSNP_module_size_pthre1e-04.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0907_plot_nullSNP_module_size_pthre1e-04_ratio.pdf" title="Figure: LEFT: module size v.s. null SNPs that can be used for Sigma estimation. RIGHT: Ratio of these two numbers for each module." width="40%" type="application/pdf" />
<p class="caption">
Figure: LEFT: module size v.s. null SNPs that can be used for Sigma
estimation. RIGHT: Ratio of these two numbers for each module.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_plot_nullSNP_module_size_pthre1e-04_ratio">
Past versions of 0907_plot_nullSNP_module_size_pthre1e-04_ratio.pdf
</button>
</p>
<div id="fig-0907_plot_nullSNP_module_size_pthre1e-04_ratio"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0907_plot_nullSNP_module_size_pthre1e-04_ratio.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="genes-in-a-module-used-by-each-snp" class="section level4">
<h4>Genes in a module used by each SNP</h4>
<p>Why? For a pair of (SNP, module),</p>
<ul>
<li><p>Remove genes in the module on the same chr as the SNP.</p></li>
<li><p>Remove genes in the module cross-mappable with cis genes of the
SNP.</p></li>
</ul>
<p>Therefore, each SNP may correspond to a different gene set for a same
module.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0907_num_gene_snp_used_M1.pdf" title="Figure: How many genes in a module were finally used for each SNP, after removing cis genes on the same chr and removing cross map genes with the SNP's cis genes." width="50%" type="application/pdf" />
<p class="caption">
Figure: How many genes in a module were finally used for each SNP, after
removing cis genes on the same chr and removing cross map genes with the
SNP’s cis genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_num_gene_snp_used_M1">
Past versions of 0907_num_gene_snp_used_M1.pdf
</button>
</p>
<div id="fig-0907_num_gene_snp_used_M1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68005649b9fe0ffe3394a959921a52fe70009bf1/docs/asset/2022/0907_num_gene_snp_used_M1.pdf" target="_blank">6800564</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="run-trans-pco-on-eqtlgen-sum-stats-under-msigdb-modules"
class="section level4">
<h4>Run trans-PCO on eQTLGen sum stats under MSigDB modules</h4>
<p>Running…</p>
</div>
</div>
<div id="annotate-cis-genes-of-signals-in-dgn-under-msigdb-modules"
class="section level3">
<h3>2. Annotate cis genes of signals in DGN under MSigDB modules</h3>
<p>See
<code>/project2/xuanyao/llw/MODULES/MSigDB/postanalysis/signal_cis_genes.txt</code>.</p>
<p>File columns are same as in previous signal file of DGN under
co-expressed modules. The first few columns are signal info: module,
chr, pos, p-value, etc. The last few columns are the SNP’s nearest gene
and distance, cis- genes and distance (&lt;1Mb).</p>
</div>
<div id="add-annotations-to-modules-and-snps-msigdb-signals-in-dgn"
class="section level3">
<h3>3. Add annotations to modules and SNPs (MSigDB signals in DGN)</h3>
<div id="result-file" class="section level4">
<h4>Result file</h4>
<p>See
<code>/project2/xuanyao/llw/MODULES/MSigDB/postanalysis/signal_cis_genes_w_annot.txt</code>.</p>
<p><strong>Columns explanation</strong>:</p>
<ul>
<li><p>signal: module:snp signal</p></li>
<li><p>annot_module: MSigDB pathway annotation of the signal
module</p></li>
<li><p>if_gwas_catalog: if the signal SNP is in gwas catalog
SNP</p></li>
<li><p>gwas_catalog_trait: the signal SNP’s corresponding associated
trait(s) in gwas catalog</p></li>
<li><p>if_cis_gwas_catalog: if the signal SNP is cis (within 100kb) to
any gwas catalog SNP</p></li>
<li><p>cis_gwas_catalog_trait: the cis SNPs’ corresponding associated
trait(s) in gwas catalog</p></li>
<li><p>module</p></li>
<li><p>SNP</p></li>
<li><p>chr</p></li>
<li><p>pos</p></li>
<li><p>p</p></li>
<li><p>q</p></li>
<li><p>nearest_gene: the nearest gene of the signal SNP</p></li>
<li><p>nearest_dis: the distance of the signal SNP to the nearest
gene</p></li>
<li><p>near_genes: genes near (within 1Mb) the signal SNP</p></li>
<li><p>near_dis: the distance of the signal SNP to the near
genes</p></li>
<li><p>rsid</p></li>
<li><p>gwas_catalog_report_gene: report gene of the corresponding gwas
catalog SNP of the signal snp</p></li>
<li><p>gwas_catalog_mapped_gene: experimental mapped genes of the
corresponding gwas catalog SNP of the signal snp</p></li>
<li><p>cis_rsid: rsid of gwas catalog cis snps</p></li>
<li><p>cis_dis_gwas_catalog: distance between the signal SNP and the
gwas catalog cis snps</p></li>
<li><p>cis_gwas_catalog_report_gene: the report genes of the gwas
catalog cis snps</p></li>
<li><p>cis_gwas_catalog_mapped_gene: the experimental mapped genes of
the gwas catalog cis snps</p></li>
</ul>
</div>
<div id="signal-module-annotation" class="section level4">
<h4>Signal module annotation</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0907_signal_module_annot.pdf" title="Figure: How many genes in a module were finally used for each SNP, after removing cis genes on the same chr and removing cross map genes with the SNP's cis genes." width="80%" type="application/pdf" />
<p class="caption">
Figure: How many genes in a module were finally used for each SNP, after
removing cis genes on the same chr and removing cross map genes with the
SNP’s cis genes.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0907_signal_module_annot">
Past versions of 0907_signal_module_annot.pdf
</button>
</p>
<div id="fig-0907_signal_module_annot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cf3ab0ad6b1c315acecd4fd2b1a3dc9377e07c9f/docs/asset/2022/0907_signal_module_annot.pdf" target="_blank">cf3ab0a</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="signal-snp-annotation-by-gwas-catalog-snps-with-disease"
class="section level4">
<h4>Signal SNP annotation by GWAS catalog SNPs (with disease)</h4>
<p>gwas catalog association features:</p>
<ul>
<li><p>SNPs are based on hg38.</p></li>
<li><p>Association tests: SNP, haplotype, interaction.</p></li>
<li><p>merged rsid.</p></li>
<li><p>only show lead SNPs. So I also looked at if a singal SNP is cis
to any gwas catalog SNPs. (Though to be rigorous, I should have checked
if they are in LD. but for simplicity, for now I just checked the
distance)</p></li>
</ul>
<p>965 (SNP, module) pairs. 220 are gwas catalog significant SNPs. All
are within 100kb of gwas catalog significant SNPs.</p>
<p><strong>??? look into the specific pathways v.s. disease/trait
???</strong></p>
</div>
</div>
</div>
<div id="aug-17-aug-24" class="section level2">
<h2>Aug 17 &amp; Aug 24</h2>
<div id="merged-coloc-regions" class="section level3">
<h3>1. Merged coloc regions</h3>
<div id="how-1" class="section level4">
<h4>How?</h4>
<ul>
<li><p>If two trans-eQTL regions under previous definition have close
lead SNPs within 200kb, they are merged to one region. As a result, 255
regions were merged to 179 regions.</p></li>
<li><p>I counted a merged region as a coloc region if any of its
sub-regions has coloc signals.</p></li>
<li><p>I calculated the coloc proportion as #coloc/(255 or 179), instead
of using those “candidate regions” as denominator as calculated
previously.</p></li>
<li><p>I looked at 29 blood related traits, autoimmune diseases, and
some other traits (including height).</p></li>
</ul>
</div>
<div id="figures-observations-1" class="section level4">
<h4>Figures &amp; Observations</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0817_coloc_region_prop_merged_blood_immu_other.pdf" title="Figure: Coloc proportion of blood traits, autoimmune diseases, and other traits, based on 255 originally defined regions and 179 merged regions. Blue: original non-merged regions. Green: merged regions. Dark: coloc." width="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc proportion of blood traits, autoimmune diseases, and other
traits, based on 255 originally defined regions and 179 merged regions.
Blue: original non-merged regions. Green: merged regions. Dark: coloc.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0817_coloc_region_prop_merged_blood_immu_other">
Past versions of 0817_coloc_region_prop_merged_blood_immu_other.pdf
</button>
</p>
<div id="fig-0817_coloc_region_prop_merged_blood_immu_other"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/985ba117837716cd83aca130ba3316b715148a46/docs/asset/2022/0817_coloc_region_prop_merged_blood_immu_other.pdf" target="_blank">985ba11</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="other-files" class="section level4">
<h4>Other files:</h4>
<ul>
<li><p>Numerical results of the figure, see</p>
<ul>
<li><p>For blood,
<code>/project2/xuanyao/llw/coloc/ukbb_coloc_blood_traits/data/coloc_region_prop_merged.txt</code>.</p></li>
<li><p>For autoimmune,
<code>/project2/xuanyao/llw/coloc/immune_traits/pmid_all/coloc_region_prop_merged.txt</code>.</p></li>
<li><p>For other ukbb traits,
<code>/project2/xuanyao/llw/coloc/ukbb_coloc_more_traits/all_trait/data/coloc_region_prop_merged.txt</code>.</p></li>
</ul></li>
<li><p>Detailed coloc merged region for each trait, e.g. the original
coloc region was merged to which region, see</p>
<ul>
<li><p>For blood,
<code>/project2/xuanyao/llw/coloc/ukbb_coloc_blood_traits/data/pheno*.coloc_reg_w_merged.txt</code>.</p></li>
<li><p>For autoimmune,
<code>/project2/xuanyao/llw/coloc/immune_traits/pmid*/data/coloc_reg_w_merged.txt</code>.</p></li>
<li><p>For other ukbb traits,
<code>/project2/xuanyao/llw/coloc/ukbb_coloc_more_traits/ukbb*/data/coloc_reg_w_merged.txt</code>.</p></li>
</ul></li>
</ul>
<p>For height, modules : M153, M156, M25, M51. Loci: 3:101044144.
Similar for autoimmune traits.</p>
</div>
<div id="other-updated-figures" class="section level4">
<h4>Other updated figures</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0817_reg_module_trait.pdf" title="Figure: Number of coloc regions for pairs of (module, trait) for all traits. Left: non-merged coloc regions. Right: merged coloc regions. Colors: Traits of various types." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0817_reg_module_trait">
Past versions of 0817_reg_module_trait.pdf
</button>
</p>
<div id="fig-0817_reg_module_trait" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/985ba117837716cd83aca130ba3316b715148a46/docs/asset/2022/0817_reg_module_trait.pdf" target="_blank">985ba11</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0817_reg_module_trait_merged.pdf" title="Figure: Number of coloc regions for pairs of (module, trait) for all traits. Left: non-merged coloc regions. Right: merged coloc regions. Colors: Traits of various types." width="40%" type="application/pdf" />
<p class="caption">
Figure: Number of coloc regions for pairs of (module, trait) for all
traits. Left: non-merged coloc regions. Right: merged coloc regions.
Colors: Traits of various types.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0817_reg_module_trait_merged">
Past versions of 0817_reg_module_trait_merged.pdf
</button>
</p>
<div id="fig-0817_reg_module_trait_merged" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/985ba117837716cd83aca130ba3316b715148a46/docs/asset/2022/0817_reg_module_trait_merged.pdf" target="_blank">985ba11</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-17
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="s-ldsc-for-other-traits" class="section level2">
<h2>2. S-LDSC for “other traits”</h2>
<ul>
<li><p>All modules for all blood traits</p>
<p><code>/project2/xuanyao/llw/ldsc/h2_enrich_comb/M*_blood_traits.results</code></p></li>
<li><p>All modules for all autoimmune diseases and all other traits</p>
<p><code>/project2/xuanyao/llw/ldsc/h2_enrich_comb/T_*_all_modules.results</code></p></li>
<li><p>Figures for all</p>
<ul>
<li>Module based (blood traits)</li>
</ul>
<p><code>/project2/xuanyao/llw/ldsc/h2_enrich_comb/plots/M*_all_traits.results.pdf</code></p>
<ul>
<li>Trait based (autoimmune diseases &amp; other traits)</li>
</ul>
<p><code>/project2/xuanyao/llw/ldsc/plots/T_*_all_modules.results.pdf</code></p>
<ul>
<li>Enrich p v.s. module v.s. trait</li>
</ul>
<p><code>/project2/xuanyao/llw/ldsc/plots/h2_enrich_all.pdf</code></p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0824_h2_enrich_w_other.pdf" title="Figure: Heatmap of coloc regions and h2 enrichment for pairs of module and trait." width="60%" type="application/pdf" />
<p class="caption">
Figure: Heatmap of coloc regions and h2 enrichment for pairs of module
and trait.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0824_h2_enrich_w_other">
Past versions of 0824_h2_enrich_w_other.pdf
</button>
</p>
<div id="fig-0824_h2_enrich_w_other" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b3ba2972ef0aeca821affc63015d5e15075a9bf2/docs/asset/2022/0824_h2_enrich_w_other.pdf" target="_blank">b3ba297</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="covariates" class="section level2">
<h2>3. Covariates</h2>
<ul>
<li>117 modules</li>
</ul>
<p>Use <span class="math inline">\(0.1/10^6/117=8.5e-10\)</span> as
signal cutoff,</p>
<ul>
<li><p>1135 (trans-eQTL, module) pairs</p></li>
<li><p>21 modules</p></li>
<li><p>1028 unique trans-eQTLs</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0824_sig_m.pdf" title="Figure: Heatmap of coloc regions and h2 enrichment for pairs of module and trait." width="60%" type="application/pdf" />
<p class="caption">
Figure: Heatmap of coloc regions and h2 enrichment for pairs of module
and trait.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0824_sig_m">
Past versions of 0824_sig_m.pdf
</button>
</p>
<div id="fig-0824_sig_m" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8abfa54dc2f3c77a99d18dd31a4cd409b2b7a1bc/docs/asset/2022/0824_sig_m.pdf" target="_blank">8abfa54</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b3ba2972ef0aeca821affc63015d5e15075a9bf2/docs/asset/2022/0824_sig_m.pdf" target="_blank">b3ba297</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0824_sig_chr.pdf" title="Figure: Heatmap of coloc regions and h2 enrichment for pairs of module and trait." width="60%" type="application/pdf" />
<p class="caption">
Figure: Heatmap of coloc regions and h2 enrichment for pairs of module
and trait.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0824_sig_chr">
Past versions of 0824_sig_chr.pdf
</button>
</p>
<div id="fig-0824_sig_chr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8abfa54dc2f3c77a99d18dd31a4cd409b2b7a1bc/docs/asset/2022/0824_sig_chr.pdf" target="_blank">8abfa54</a>
</td>
<td>
liliw-w
</td>
<td>
2022-09-14
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b3ba2972ef0aeca821affc63015d5e15075a9bf2/docs/asset/2022/0824_sig_chr.pdf" target="_blank">b3ba297</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-24
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="aug-10" class="section level2">
<h2>Aug 10</h2>
<div
id="how-many-trans-eqtl-loci-are-cis-to-the-trans-target-gene-module"
class="section level3">
<h3>1. How many trans-eQTL Loci are cis to the trans target gene
module?</h3>
<div id="how-and-result-files" class="section level4">
<h4>How? and result files</h4>
<p>I used two ways:</p>
<ul>
<li><p>If a signal’s nearest gene (cis- genes within 1Mb) is (are) in
its trans- target module.</p></li>
<li><p>What is the distance between a signal and genes in the module? Is
distance to any gene within 1Mb of the signal?</p></li>
</ul>
<p><strong>Result file:</strong></p>
<ul>
<li><p><code>/project2/xuanyao/llw/DGN_no_filter_on_mappability/postanalysis/cis_genes_in_module.txt</code></p></li>
<li><p><code>/project2/xuanyao/llw/DGN_no_filter_on_mappability/postanalysis/dis_to_genes_in_module.txt</code></p></li>
</ul>
</div>
<div id="numerical-results" class="section level4">
<h4>Numerical results</h4>
<p>Among 3899 (SNP, module) signal pairs in DGN,</p>
<ul>
<li><p><strong>The nearest gene</strong> (or at least one gene
<strong>within 1Mb</strong>) of trans-eQTLs in <strong>79 (372)</strong>
(SNP, module) pairs are in the trans- target module.</p></li>
<li><p>trans-eQTLs in <strong>186 (372)</strong> (SNP, module) pairs are
within <strong>100kb (1Mb)</strong> of at least one gene in the trans-
target module.</p></li>
</ul>
</div>
</div>
<div id="simulation-when-beta-has-same-correlation-as-sigma"
class="section level3">
<h3>2. Simulation when beta has same correlation as <span
class="math inline">\(\Sigma\)</span></h3>
<div id="goal" class="section level4">
<h4>Goal</h4>
<p>To see if PC1 has any power in this case.</p>
<ul>
<li><p>Signals of previous simulations have large p-values (max ~0.05),
under null distribution.</p></li>
<li><p>The simulated null distribution has large p-values (0.001% &lt;
1e-5, 0.01% &lt; 1e-4, 0.1% &lt; 1e-3).</p></li>
<li><p>Signals in DGN have largest p-value ~1e-7.</p></li>
<li><p>Why simulated null distribution has large p-values?</p></li>
</ul>
</div>
<div id="procedure" class="section level4">
<h4>Procedure</h4>
<p>To look into how <span class="math inline">\(cor(\beta)\)</span>
affects the power, I considered the following parameter settings,</p>
<ol style="list-style-type: decimal">
<li><p><span class="math inline">\(cor(\beta)=\Sigma\)</span>, <span
class="math inline">\(caus=0.3\)</span>, <span
class="math inline">\(N=800\)</span></p></li>
<li><p><span class="math inline">\(cor(\beta)=\Sigma\)</span>, <span
class="math inline">\(caus=1\)</span>, <span
class="math inline">\(N=800\)</span></p></li>
<li><p><span class="math inline">\(cor(\beta)=I\)</span>, <span
class="math inline">\(caus=1\)</span>, <span
class="math inline">\(N=800\)</span></p></li>
<li><p><span class="math inline">\(cor(\beta)=I\)</span>, <span
class="math inline">\(caus=0.3\)</span>, <span
class="math inline">\(N=800\)</span> (previous setting)</p></li>
</ol>
</div>
<div id="observation" class="section level4">
<h4>Observation</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0810_power_b_corr.pdf" title="Figure: coloc between trans and cis loci." width="60%" type="application/pdf" />
<p class="caption">
Figure: coloc between trans and cis loci.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0810_power_b_corr">
Past versions of 0810_power_b_corr.pdf
</button>
</p>
<div id="fig-0810_power_b_corr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b724aee76fb6ecf965b0e9c5ed5e5b045c60c37/docs/asset/2022/0810_power_b_corr.pdf" target="_blank">1b724ae</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>PC1 still has low power.</p></li>
<li><p>Using <span class="math inline">\(\Sigma\)</span> as <span
class="math inline">\(cor(\beta)\)</span> doesn’t matter much to power
under <span class="math inline">\(caus=0.3\)</span>.</p></li>
<li><p>Using <span class="math inline">\(\Sigma\)</span> as <span
class="math inline">\(cor(\beta)\)</span> increase power under <span
class="math inline">\(caus=1\)</span>.</p></li>
<li><p>Increased <span class="math inline">\(caus\)</span> doesn’t
matter much to power under independent <span
class="math inline">\(cor(\beta)\)</span>.</p></li>
<li><p>Increased <span class="math inline">\(caus\)</span> increase
power under <span
class="math inline">\(cor(\beta)=\Sigma\)</span>.</p></li>
</ul>
</div>
</div>
<div id="cell-type-analysis" class="section level3">
<h3>3. Cell type analysis</h3>
<p>Aggregate p-values, Bonferroni, signals. Except M1 &amp; M2.</p>
<p>Run <code>z.sh</code> for M1 &amp; M2.</p>
</div>
<div id="re-define-coloc-regions" class="section level3">
<h3>4. Re-define coloc regions</h3>
<p>Previously, a region for coloc is defined to be a region of length
100kb centered at a lead SNP. As a result, for two coloc regions, their
lead SNPs can be as closest as 50kb. In most cases (only one causal
variant), it doesn’t make much sense to see these two regions as two
different regions.</p>
<p>Therefore, I need to merge these kind of regions. How?</p>
<p>I focus on the regions with significant coloc. If two regions’ lead
SNPs are within 200kb, then these regions are counted as one.</p>
<p>See result file:
<code>/project2/xuanyao/llw/coloc/ukbb_coloc_blood_traits/res_coloc_reg_merged.txt</code></p>
</div>
</div>
<div id="aug-03" class="section level2">
<h2>Aug 03</h2>
<div
id="check-enriched-h2-of-all-modules-in-all-three-autoimmune-diseases"
class="section level3">
<h3>1. Check enriched <span class="math inline">\(h^2\)</span> of all
modules in all three autoimmune diseases</h3>
<p>To look at if there is h2 enrichment of any module in three
autoimmune diseases (i.e. cd, ibd, allergy), I ran S-LDSC for all
modules.</p>
<p>See <a href="asset/2022/T_28067908_cd_all_modules.results.pdf">h2
enrich for cd</a>, <a
href="asset/2022/T_28067908_ibd_all_modules.results.pdf">h2 enrich for
ibd</a>, <a
href="asset/2022/T_29083406_Allergy_all_modules.results.pdf">h2 enrich
for allergy</a>.</p>
</div>
<div
id="run-trans-pco-in-dgn-with-keeping-cell-type-proportion-and-expression-pcs"
class="section level3">
<h3>2. Run Trans-PCO in DGN with keeping cell type proportion and
expression pc’s</h3>
<p>This is a suggestion from the committee meeting. We observed a high
proportion of trans-eQTLs and blood traits. We interpreted it as the
enrichment of trans-eQTLs in cell type proportion.</p>
<p>However, since we attempted to remove the cell type proportion as
covariates, the above intepretion doesn’t send a good message.</p>
<p>So, the suggestion was to not remove cell type proportion at all, and
see what happens. Therefore, I need to run trans-PCO without regression
out cell type proportions (and expression PC’s, since they are also
relevant to cell type proportions), and see how the gene modules,
signals, coloc with blood traits change.</p>
<p>Progress: running trans-PCO on DGN.</p>
</div>
<div id="update-eqtlgen-using-ratio-100-instead-of-50"
class="section level3">
<h3>3. Update eQTLGen using ratio 100 (instead of 50)</h3>
<div id="numbers" class="section level4">
<h4>Numbers</h4>
<ul>
<li><p>108 (previously, 130) modules are used for signal
detection.</p></li>
<li><p>There are 5610 significant (trans-eQTL, module) pairs.</p></li>
<li><p>There are 101 modules that have at least one trans-eQTL.</p></li>
<li><p>There are 1697 eQTLGen SNPs that are significant trans-eQTL for
at least one module.</p></li>
</ul>
</div>
<div id="signal-figures" class="section level4">
<h4>Signal figures</h4>
<p>Updates see
<code>/project2/xuanyao/llw/eQTLGen_est_Sigma/plot</code>.</p>
</div>
<div id="update-dgn-replication-in-eqtlgen" class="section level4">
<h4>Update: DGN replication in eQTLGen</h4>
<ul>
<li><p>Out of 3899 DGN (SNP, module) signal pairs, 33 of them are also
analyzed in eQTLGen.</p></li>
<li><p>Out of 33 DGN signals, 33 are replicated in eQTLGen.</p></li>
</ul>
</div>
<div id="update-eqtlgen-replication-in-dgn" class="section level4">
<h4>Update: eQTLGen replication in DGN</h4>
<ul>
<li><p>Out of 5610 eQTLGen (SNP, module) signal pairs, 5230 of them are
also analyzed in DGN.</p></li>
<li><p>Out of 5230 eQTLGen signals, 69 are replicated in DGN.</p></li>
</ul>
<p><strong>Add buffer region to DGN signals to see
replication</strong></p>
<ul>
<li><p>eQTLGen specific signal (SNP, module) pairs not in DGN</p>
<p>Out of 5230 eQTLGen (SNP, module) signal pairs that were also
analyzed by DGN,</p>
<p>145 190 212 236 are near at least one DGN signal within same loci of
distance:</p>
<p>1e+05 2e+05 5e+05 1e+06 bp, respectively.</p>
<p>4672 are not close to any DGN signals.</p></li>
<li><p>eQTLGen specific signal SNPs not in DGN</p>
<p>Out of 1558 eQTLGen signal SNPs that were also analyzed by DGN,</p>
<p>153 216 263 378 are near at least one DGN signal within same loci of
distance:</p>
<p>1e+05 2e+05 5e+05 1e+06 respectively.</p>
<p>0 are not close to any DGN signals.</p></li>
<li><p>Interesting examples detected in eQTLGen but missed in
DGN</p></li>
</ul>
<p>In <a href="Log2022.html#eQTLGen_replciation_in_DGN">earlier
section</a> where I used ratio 50 to see the replications, I gave a few
interesting examples of signals detected in eQTLGen but missed in DGN
and have more significant z-scores.</p>
<p>After updating the ratio to 100, those examples still hold.</p>
</div>
</div>
<div id="h2-enrichment-heatmap" class="section level3">
<h3>4. h2 enrichment heatmap</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2022/ukbb_blood_immune_module.pdf" title="Figure: Heatmap of coloc regions and h2 enrichment for pairs of module and trait." width="60%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-ukbb_blood_immune_module">
Past versions of ukbb_blood_immune_module.pdf
</button>
</p>
<div id="fig-ukbb_blood_immune_module" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b724aee76fb6ecf965b0e9c5ed5e5b045c60c37/docs/asset/2022/ukbb_blood_immune_module.pdf" target="_blank">1b724ae</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/h2_enrich_all.pdf" title="Figure: Heatmap of coloc regions and h2 enrichment for pairs of module and trait." width="60%" type="application/pdf" />
<p class="caption">
Figure: Heatmap of coloc regions and h2 enrichment for pairs of module
and trait.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-h2_enrich_all">
Past versions of h2_enrich_all.pdf
</button>
</p>
<div id="fig-h2_enrich_all" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b724aee76fb6ecf965b0e9c5ed5e5b045c60c37/docs/asset/2022/h2_enrich_all.pdf" target="_blank">1b724ae</a>
</td>
<td>
liliw-w
</td>
<td>
2022-08-11
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Add other non-blood traits?</li>
</ul>
</div>
</div>
<div id="june-29-july-06" class="section level2">
<h2>June 29 &amp; July 06</h2>
<div id="coloc-between-trans--and-cis-qtls" class="section level3">
<h3>1. coloc between trans- and cis-QTLs</h3>
<div id="with-cis-sqtls" class="section level4">
<h4>With cis-sQTLs</h4>
<ul>
<li><p>255 trans- regions in total.</p></li>
<li><p>163 candidate coloc regions, i.e. regions which have at least one
intron with min p &lt; 1e-5. (Light green regions).</p></li>
<li><p>66 regions are coloced with at least one intron. (Dark green
regions).</p></li>
</ul>
</div>
<div id="with-cis-eqtls-updates" class="section level4">
<h4>With cis-eQTLs (updates)</h4>
<p>When I was looking into coloc with cis-s, I found some results were
weird about the coloc numbers with cis-e. So I re-analyzed cis-e coloc
with trans. Numbers are updated as below.</p>
<ul>
<li><p>255</p></li>
<li><p>120 –&gt; <strong>222</strong></p></li>
<li><p>54 –&gt; <strong>93</strong></p></li>
</ul>
</div>
<div id="with-either-cis-e-or-cis-s" class="section level4">
<h4>With either cis-e or cis-s</h4>
<ul>
<li><p>255</p></li>
<li><p>222</p></li>
<li><p>104</p></li>
</ul>
</div>
<div id="upset-plot" class="section level4">
<h4>Upset plot</h4>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0629_coloc_trans_cis_upset.pdf" title="Figure: coloc between trans and cis loci." width="40%" type="application/pdf" />
<p class="caption">
Figure: coloc between trans and cis loci.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0629_coloc_trans_cis_upset">
Past versions of 0629_coloc_trans_cis_upset.pdf
</button>
</p>
<div id="fig-0629_coloc_trans_cis_upset" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/521ea43567904a163cd3743c43f5d0693995508a/docs/asset/2022/0629_coloc_trans_cis_upset.pdf" target="_blank">521ea43</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="module-gprofiler-enrichment" class="section level3">
<h3>2. Module g:profiler enrichment</h3>
<p>Files for enrichment of each 166 modules,
<code>/project2/xuanyao/llw/DGN_no_filter_on_mappability/module_enrich</code>.</p>
</div>
<div id="simulations" class="section level3">
<h3>3. Simulations</h3>
<div id="simulation-with-large-n-caus-var" class="section level4">
<h4>Simulation with large N, caus, var</h4>
<ul>
<li><p>Aim</p>
<p>To look at PC1 and PCO power. If PC1 has any power in this
case.</p></li>
<li><p>Parameters</p>
<p>var.b: 0.2 (0.001 by default), caus: 1 (0.3 by default), N: 800 (500
by default). 10^3 simulations, 10^4 samples.</p>
<p>I looked at only one scenario.</p></li>
<li><p>Figures &amp; Observations</p>
<ul>
<li><p>Trans-PCO and MinP both have power of 1 across all
simulations.</p></li>
<li><p>PC1 tends to be either extremely powerful or powerless.</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0706_power.large.paras.rds3.pdf" title="Figure: Simulation power when parameters, N, caus, var, are large. Green point is mean +- 95% CI." width="40%" type="application/pdf" />
<p class="caption">
Figure: Simulation power when parameters, N, caus, var, are large. Green
point is mean +- 95% CI.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0706_power.large.paras.rds3">
Past versions of 0706_power.large.paras.rds3.pdf
</button>
</p>
<div id="fig-0706_power.large.paras.rds3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26e2dad5dbc221c73cadf844831fb159a6f60bd3/docs/asset/2022/0706_power.large.paras.rds3.pdf" target="_blank">26e2dad</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="simulation-where-pc1-has-the-highest-power"
class="section level4">
<h4>Simulation where PC1 has the highest power</h4>
<ul>
<li><p>Aim</p>
<p>When true beta’s align with the first eigenvector, what does power
look like for PC1? and the other two methods?</p></li>
<li><p>Parameters, and how?</p>
<p>To make beta’s have same direction with u1, I first make beta equal
u1 (which has length 1), then scale beta by a constant to set its
genetic variance.</p>
<p>I set beta’s to have the same length as beta generated by default
setting, i.e. var.b=0.001, caus=0.3, N=500. However, in this case,
powers of all three methods are low, even for the oracle test, which
indicates power in this case is intrinsically low.</p>
<p>This may be due to the smaller effects of each individual gene given
a fixed var and more genes with nonzero effects.</p>
<p>Therefore, I increased beta’s length by <span
class="math inline">\(u_1 \times ||beta^{default}||/0.3 \times
1.2\)</span>, i.e. 4 times longer.</p>
<p>I looked at power at various N’s.</p></li>
<li><p>Figures &amp; Observations</p>
<ul>
<li>PC1 has the highest power.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0706_power.pc1.rds3.pdf" title="Figure: Simulation power when PC1 has the highest power. Error bar is mean +- 95% CI." width="40%" type="application/pdf" />
<p class="caption">
Figure: Simulation power when PC1 has the highest power. Error bar is
mean +- 95% CI.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0706_power.pc1.rds3">
Past versions of 0706_power.pc1.rds3.pdf
</button>
</p>
<div id="fig-0706_power.pc1.rds3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26e2dad5dbc221c73cadf844831fb159a6f60bd3/docs/asset/2022/0706_power.pc1.rds3.pdf" target="_blank">26e2dad</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="simulation-3" class="section level4">
<h4>Simulation 3</h4>
</div>
</div>
<div id="trans--target-module-of-gene-klf14" class="section level3">
<h3>4. trans- target module of gene <em>KLF14</em></h3>
<div id="goal-1" class="section level4">
<h4>Goal</h4>
<p><em>KLF14</em> relates to T1D, an autoimmune disease. Check if its
trans- target module is enriched in T1D related pathways.</p>
</div>
<div id="observations-1" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>Signal is (7:130536491, module30).</p></li>
<li><p>Genomic location of the trans-eQTL and <em>KLF14</em>, see <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr7%3A129647434%2D131499539&amp;hgsid=1395121101_3u30PbMsiNLjlbJNTXJUkjY4KLWH">genome
browser</a>. The distance is 119,090 bp.</p></li>
<li><p>GO enrichment of the module 30</p>
<p><a href="https://biit.cs.ut.ee/gplink/l/kLhSRPZuSP">See enrichment
here</a>.</p>
<p>Module 30 is enriched in two GO molecular functions:</p>
<ul>
<li>pantetheine hydrolase activity</li>
</ul>
<p>The Vanin genes are a family that encode pantetheinases (or
pantetheine hydrolase).</p>
<p>VNN1, VNN2, VNN3 are in module 30.</p>
<ul>
<li>CXCR chemokine receptor binding</li>
</ul></li>
</ul>
</div>
</div>
<div id="check-coloc-between-m71-m74-m154-v.s.-plagl1"
class="section level3">
<h3>5. Check coloc between M71 &amp; M74 (M154) v.s. PLAGL1</h3>
<div id="coloc-overview" class="section level4">
<h4>coloc overview</h4>
<ul>
<li><p>M154, coloc</p>
<p><strong>module154:6:144291150</strong> 0 PLAGL1 206
6.81723922870508e-21 6.90867766367242e-12 7.55979453410063e-11
0.0756876149474007 <strong>0.924312384970091</strong> PLAGL1</p></li>
<li><p>M74, no coloc</p>
<p><strong>module74:6:144304358</strong> 1.765254609154e-14 PLAGL1 209
2.57183433027403e-17 9.47954145652225e-13 2.36642981576199e-05
0.872116097251446 <strong>0.127860238449448</strong> PLAGL1</p></li>
<li><p>M71, no coloc</p>
<p><strong>module71:6:144304358</strong> 9.50528573628873e-08 PLAGL1 209
6.79628848096945e-14 2.30495794945591e-13 0.0625348977909173
0.211360698835187 <strong>0.726104403373597</strong> PLAGL1</p></li>
</ul>
</div>
<div id="figures-observations-2" class="section level4">
<h4>Figures &amp; Observations</h4>
<ul>
<li><p>Coloc between PLAGL1 loci and three modules look very similar.
But only M154 has significant coloc.</p></li>
<li><p>no coloc because there are two independent causal variants for
PLAGL1 and M154, M74, and one causal variant for M71?</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0706_module154.6.144291150.PLAGL1.reg.coloc.pdf" title="Figure: Coloc near loci PLAGL1." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0706_module154.6.144291150.PLAGL1.reg.coloc">
Past versions of 0706_module154.6.144291150.PLAGL1.reg.coloc.pdf
</button>
</p>
<div id="fig-0706_module154.6.144291150.PLAGL1.reg.coloc"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26e2dad5dbc221c73cadf844831fb159a6f60bd3/docs/asset/2022/0706_module154.6.144291150.PLAGL1.reg.coloc.pdf" target="_blank">26e2dad</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2022/0706_module74.6.144304358.PLAGL1.reg.coloc.pdf" title="Figure: Coloc near loci PLAGL1." width="30%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0706_module74.6.144304358.PLAGL1.reg.coloc">
Past versions of 0706_module74.6.144304358.PLAGL1.reg.coloc.pdf
</button>
</p>
<div id="fig-0706_module74.6.144304358.PLAGL1.reg.coloc"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26e2dad5dbc221c73cadf844831fb159a6f60bd3/docs/asset/2022/0706_module74.6.144304358.PLAGL1.reg.coloc.pdf" target="_blank">26e2dad</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0706_module71.6.144304358.PLAGL1.reg.coloc.pdf" title="Figure: Coloc near loci PLAGL1." width="30%" type="application/pdf" />
<p class="caption">
Figure: Coloc near loci PLAGL1.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0706_module71.6.144304358.PLAGL1.reg.coloc">
Past versions of 0706_module71.6.144304358.PLAGL1.reg.coloc.pdf
</button>
</p>
<div id="fig-0706_module71.6.144304358.PLAGL1.reg.coloc"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26e2dad5dbc221c73cadf844831fb159a6f60bd3/docs/asset/2022/0706_module71.6.144304358.PLAGL1.reg.coloc.pdf" target="_blank">26e2dad</a>
</td>
<td>
liliw-w
</td>
<td>
2022-07-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="coloc.susie-allow-more-than-one-causal-variants"
class="section level4">
<h4>coloc.susie allow more than one causal variants</h4>
<p><code>coloc.susie</code> &lt;– <code>runsusie</code> &lt;–
<code>z</code> &lt;– sign of effect matters &lt;– no sign info of
module-qtls.</p>
<p>Run <code>coloc.susie</code> forcely by assuming module-qtls have
same signs, I observed significant coloc.</p>
</div>
</div>
</div>
<div id="june-15-june-22" class="section level2">
<h2>June 15 &amp; June 22</h2>
<div id="eqtlgen-specific-signals" class="section level3">
<h3>1. eQTLGen specific signals</h3>
<div id="specific-snp-module-pairs" class="section level4">
<h4>Specific (SNP, module) pairs</h4>
<ul>
<li><p>Observations</p>
<p>Out of 7577 eQTLGen (SNP, module) signal pairs that were also
analyzed by DGN,</p>
<p><strong>184, 240, 262, 286</strong> pairs are near at least one DGN
signal within same loci of distance:</p>
<p><strong>1e+05, 2e+05, 5e+05, 1e+06</strong> bp, respectively.</p>
<p><strong>6876</strong> pairs are not close to any DGN
signals.</p></li>
<li><p>Result file:</p>
<p><code>/project2/xuanyao/llw/eQTLGen_est_Sigma/postanalysis/eqtlgen_spec_pair_to_dgn.txt</code></p>
<ul>
<li><p>Why so many Inf distances?</p>
<p>Inf distance means no DGN signal SNPs on the same chr as eQTLGen
(SNP, module).</p>
<p>In fact, overlapped SNPs, but not right module.</p></li>
</ul></li>
<li><p>Example</p>
<p>I obversed a lot of SNPs (and corresponding modules) that are signals
in eQTLGen, but not in DGN.</p>
<p>Among those, one interesting loci SNP: 3:56849749. This loci is near
gene <em>ARHGEF3</em>, and it has been detected in DGN (for module 4).
However, in eQTLGen, it is a signal for a few more modules missed in
DGN.</p>
<p>Therefore, I looked at z-scores of this SNP and those
eQTLGen-specific modules, in both QTLGen and DGN. The goal is to see if
eQTLGen-z’s are generally more significant than DGN-z’s. If so, then it
indicates larger sample size of eQTLGen improves detection power.</p>
<ul>
<li><p>Figure of z distributions</p>
<p><a href="asset/2022/0615_z_eqtlgen_dgn_snp3_56849749.pdf">See
here</a></p>
<p>Add summary stats of distribution difference?</p></li>
</ul></li>
</ul>
</div>
<div id="specific-snps" class="section level4">
<h4>Specific SNPs</h4>
<ul>
<li><p>Observations</p>
<p>Out of 1975 eQTLGen signal SNPs that were also analyzed by DGN,</p>
<p><strong>166 236 303 430</strong> are near at least one DGN signal
within same loci of distance:</p>
<p><strong>1e+05 2e+05 5e+05 1e+06</strong> bp, respectively.</p></li>
<li><p>Result file:</p>
<p><code>/project2/xuanyao/llw/eQTLGen_est_Sigma/postanalysis/eqtlgen_spec_snp_to_dgn.txt</code></p>
<ul>
<li><p>No Inf distances?</p>
<p>which means all eQTLGen SNPs have at least one DGN signal SNP on the
same chr.</p></li>
</ul></li>
</ul>
</div>
</div>
<div id="compare-z-of-pc1-and-pco-signals" class="section level3">
<h3>2. Compare z of PC1 and PCO signals</h3>
<div id="goal-2" class="section level4">
<h4>Goal</h4>
<p>Given that signals detected by PC1 are more than what we expected
from simulations, we want to know why PC1 can get so many signals than
in simulations and look into what are these signals.</p>
<p>PC1 is powerful when <span class="math inline">\(\beta\)</span>’s
align with the first eigen vector of the expression matrix. <strong>How
did we come to the conclusion that PC1-detected signals should have
large effects?</strong></p>
<p>Therefore, I looked at z-scores of signals detected by PC1 and
trans-PCO.</p>
</div>
<div id="signals-review" class="section level4">
<h4>Signals review</h4>
<ul>
<li><p>PCO signals: 3899</p></li>
<li><p>PC1 signals: 1483, 1464 (98.72% are PCO signals).</p></li>
</ul>
</div>
<div id="comparison-procedure" class="section level4">
<h4>Comparison procedure</h4>
<p>For a pair of (SNP, module) signal, I plotted <span
class="math inline">\(|Z|\)</span>’s of the SNP across genes in this
module. I divided (SNP, module) signals into three categories:</p>
<ul>
<li><ol style="list-style-type: decimal">
<li>pair detected only on trans-PCO</li>
</ol></li>
<li><ol start="2" style="list-style-type: decimal">
<li>pair detected by both trans-PCO and PC1</li>
</ol></li>
<li><ol start="3" style="list-style-type: decimal">
<li>pair detected by only PC1.</li>
</ol></li>
</ul>
<p>To compare z-score distribution of these three types of signals, I
used (1) cdf and (2) pdf of</p>
<ul>
<li><ol style="list-style-type: decimal">
<li>abs z-scores of one SNP across genes in module</li>
</ol></li>
<li><ol start="2" style="list-style-type: decimal">
<li>the maximum abs z-scores for each (SNP, module) pair.</li>
</ol></li>
</ul>
<p><strong>Note: I first plotted cdf of min z as we discussed. But for
signal category (2) and (3), points for estimating cdf are only a few,
making cdf estimation inaccurate. So, I also tried other three ways to
compare z-distribution.</strong></p>
<p>The above comparison is for individual modules. I also compared the
max abs z of each (SNP, module) pair across all modules.</p>
</div>
<div id="comparison-figures" class="section level4">
<h4>Comparison figures</h4>
<ul>
<li><p>See all figures in
<code>/scratch/midway2/liliw1/compare_z/pc1</code>.</p></li>
<li><p>Max abs z comparison of all modules, (<strong>If you can’t see
figures below, try opening this site in safari</strong>)</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0615_M_all_maxz_cdf.pdf" title="Figure: cdf and pdf of max abs for all signal (SNP, module)." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0615_M_all_maxz_cdf">
Past versions of 0615_M_all_maxz_cdf.pdf
</button>
</p>
<div id="fig-0615_M_all_maxz_cdf" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68148ef7d907c8eb91e97f7fea917fcd95e91c83/docs/asset/2022/0615_M_all_maxz_cdf.pdf" target="_blank">68148ef</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0615_M_all_maxz_pdf.pdf" title="Figure: cdf and pdf of max abs for all signal (SNP, module)." width="40%" type="application/pdf" />
<p class="caption">
Figure: cdf and pdf of max abs for all signal (SNP, module).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0615_M_all_maxz_pdf">
Past versions of 0615_M_all_maxz_pdf.pdf
</button>
</p>
<div id="fig-0615_M_all_maxz_pdf" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68148ef7d907c8eb91e97f7fea917fcd95e91c83/docs/asset/2022/0615_M_all_maxz_pdf.pdf" target="_blank">68148ef</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>I also looked at min abs z comparison for all modules.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2022/0615_M_all_minz_cdf.pdf" title="Figure: cdf and pdf of min abs z for all signal (SNP, module)." width="30%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0615_M_all_minz_cdf">
Past versions of 0615_M_all_minz_cdf.pdf
</button>
</p>
<div id="fig-0615_M_all_minz_cdf" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68148ef7d907c8eb91e97f7fea917fcd95e91c83/docs/asset/2022/0615_M_all_minz_cdf.pdf" target="_blank">68148ef</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2022/0615_M_all_minz_pdf.pdf" title="Figure: cdf and pdf of min abs z for all signal (SNP, module)." width="30%" type="application/pdf" />
<p class="caption">
Figure: cdf and pdf of min abs z for all signal (SNP, module).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0615_M_all_minz_pdf">
Past versions of 0615_M_all_minz_pdf.pdf
</button>
</p>
<div id="fig-0615_M_all_minz_pdf" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/68148ef7d907c8eb91e97f7fea917fcd95e91c83/docs/asset/2022/0615_M_all_minz_pdf.pdf" target="_blank">68148ef</a>
</td>
<td>
liliw-w
</td>
<td>
2022-06-29
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observations-2" class="section level4">
<h4>Observations</h4>
<ul>
<li><p>PCO only signals have smaller max abs z than signals detected by
both PCO and PC1.</p></li>
<li><p>PC1 only signals have smaller max abs z and larger min abs z than
PCO only and both signals.</p>
<ul>
<li>19 PC1 only signals have similar PCO p-values. And mostly have pco
corrected p &lt; 15%. See <a
href="asset/2022/0615_pc1_sig_mis_pco.txt">file of PC1 signals missed by
PCO</a>.</li>
</ul></li>
</ul>
<p><strong>Update following numbers!!!</strong></p>
<ul>
<li><p>Z(PCO) &lt; Z(both)</p>
<p>Module 1, 32, 51, 57, 66, 108, 121, 134, 154, 155, 158</p></li>
<li><p>No difference</p>
<p>Module 4, 35, 47, 74, 90, 100, 113, 140, 150</p></li>
<li><p>Z(PCO) &gt; Z(both)?</p>
<p>Module 27, 42, 65, 156</p></li>
<li><p>No accurate cdf estimation due to too few signals.</p>
<p>Module 11, 13, 21, 30, 31, 32, 57, 65, 66, 87, 90, 100, 106, 108,
117, 121, 134, 142, 144, 147, 149, 150, 164, 155, 158</p></li>
</ul>
<p>Note: Module 142: Z(PCO) &lt; Z(PC1)? PC1 only signal with large z
but missed by PCO?</p>
</div>
</div>
<div id="compare-z-of-uni-variate-and-multi-variate-signals"
class="section level3">
<h3>3. Compare z of uni-variate and multi-variate signals</h3>
<div id="goal-3" class="section level4">
<h4>Goal</h4>
</div>
<div id="signal-review" class="section level4">
<h4>Signal review</h4>
</div>
<div id="procedure-1" class="section level4">
<h4>Procedure</h4>
</div>
<div id="figures-and-observations" class="section level4">
<h4>Figures and observations</h4>
</div>
</div>
</div>
<div id="jun-08" class="section level2">
<h2>Jun 08</h2>
<div id="blood-trait-s-ldsc" class="section level3">
<h3>1. blood trait S-LDSC</h3>
<div id="goal-4" class="section level4">
<h4>Goal</h4>
<p>Test whether colocalized trans-eQTL gene modules show significant
heritability enrichment in the colocalized complex trait. By S-LDSC.</p>
<p>In this section, I looked at 29 blood traits.</p>
</div>
<div id="how-2" class="section level4">
<h4>How?</h4>
<p>See <a href="#how_to_run_s_ldsc">previous section</a>.</p>
</div>
<div id="s-ldsc-results-on-partitioned-h2-of-module-4"
class="section level4">
<h4>S-LDSC results on partitioned h2 of module 4</h4>
<p>I show here results of module 4 for 29 traits as an example. Why
module 4 interesting? Because it has interesting trans-eQTLs coloc with
interesting traits. And it’s an example in the manuscript.</p>
<p>The figure is large, so I won’t show it here, but <a
href="asset/2022/0608_M4_all_traits.results.pdf">see file here</a>.</p>
<ul>
<li>x-axis: enrichment</li>
<li>y-axis: 29 blood traits</li>
<li>error bar: enrichment 95% CI</li>
<li>color: trait types</li>
<li>green diamond: there is coloc between module 4 and corresponding
trait</li>
</ul>
<p>Compared to figures I sent on slack, here I <strong>updated
enrichment figure error bar from se to 95% CI.</strong></p>
</div>
<div id="observations-3" class="section level4">
<h4>Observations</h4>
<ul>
<li>Module 4 has enriched h2 in platelet-relevant traits. Consistent
with trans-eQTLs loci, module enrichment, and coloc results.</li>
</ul>
</div>
</div>
<div id="autoimmune-disease-ldsc" class="section level3">
<h3>2. autoimmune disease ldsc</h3>
<div id="coloc" class="section level4">
<h4>coloc</h4>
<p>Before we look at the partitioned <span
class="math inline">\(h^2\)</span> of colocalized module in autoimmune
diseases, see below for the coloc regions of diseases and modules. Six
diseases have coloc regions, corresponding to 8 modules. Number in each
cell is the number of coloc regions.</p>
<p><img src="asset/2022/0608_immune_coloc_modules.png"
style="width:45.0%" /> <img
src="asset/2022/0608_traits.module.coloc.region.module_QTL_sig.order_by_n_coloc_trait.png"
style="width:45.0%" /></p>
</div>
<div id="the-enrichment" class="section level4">
<h4>The enrichment</h4>
<ul>
<li><p>Since there are only a few modules coloc with a disease, I ran
S-LDSC only on these coloc modules for each disease. Except Crohns’
disease (cd), I ran all 166 modules, bc this gwas has the highest coloc
proportion.</p>
<p>Note: S-LDSC throws an error running on module 156 (haven’t figured
out why). So module 156 were not analyzed.</p></li>
<li><p>Enrichment: (low resolution figures to save storage space. High
resolution figures on RCC)</p></li>
</ul>
<p><img src="asset/2022/0608_T_28067908_cd_all_modules.results.png"
style="width:50.0%" /></p>
<p><img src="asset/2022/0608_T_28067908_ibd_all_modules.results.png"
style="width:45.0%" /> <img
src="asset/2022/0608_T_29083406_Allergy_all_modules.results.png"
style="width:45.0%" /></p>
<ul>
<li>Observation:</li>
</ul>
<ol style="list-style-type: decimal">
<li><p>None of coloc modules have significant h2 enrich.</p></li>
<li><p>But module 100 looks promising for Crohns’ disease
(Enrichment=3.22, sd=1.83, p=0.23) and ibd (Enrichment=4.6, sd=2.45,
p=0.14). So, I looked into it.</p></li>
</ol>
<p>(As for why the other modules don’t have enriched h2 at all, maybe
that’s because these modules are mostly zinc finger genes? as we
discussed in <a href="Log2022.html#gene_SENP7">earlier section</a>.)</p>
</div>
<div id="module-100" class="section level4">
<h4>Module 100</h4>
<p>trans-eQTLs of module 100 include two loci:</p>
<p><img src="asset/2022/0608_M100_signals.png"
style="width:50.0%" /></p>
<ol style="list-style-type: decimal">
<li><code>4:123205143</code></li>
</ol>
<ul>
<li><p>See loci location and nearby genes <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr4%3A122352294%2D124046923&amp;hgsid=1373866789_g0iFr0hp4gIFQHP8VwAKAXnEDSpE">here</a>.</p></li>
<li><p>Near genes: KIAA1109, ADAD1, IL2, IL21.</p></li>
<li><p>This loci’s relation with immune traits?</p>
<p>Looks like this loci has been found to be associated with autoimmune
diseases back in 2007. Relevant stories include <a
href="https://www-sciencedirect-com.proxy.uchicago.edu/science/article/pii/S0002929707637768">ref
1</a>, <a
href="https://scholar.google.com/scholar?hl=en&amp;as_sdt=0%2C14&amp;q=KIAA1109&amp;btnG=">more
stories</a>.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><code>5:82996199</code></li>
</ol>
<ul>
<li><p>See loci location and nearby genes <a
href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&amp;lastVirtModeType=default&amp;lastVirtModeExtraState=&amp;virtModeType=default&amp;virtMode=0&amp;nonVirtPosition=&amp;position=chr5%3A82756735%2D83104320&amp;hgsid=1373866789_g0iFr0hp4gIFQHP8VwAKAXnEDSpE">here</a>.</p></li>
<li><p>Near gene: HAPLN1.</p></li>
<li><p>This loci’s relation with autoimmune diseases?</p>
<p>I didn’t find direct link bewteen HAPLN1 and autoimmune diseases. But
this gene is relevant with NF-KB activity, see <a
href="https://www-jbc-org.proxy.uchicago.edu/article/S0021-9258(20)31707-5/fulltext">ref</a>.</p></li>
</ul>
</div>
</div>
</div>
<div id="may-27" class="section level2">
<h2>May 27</h2>
<div id="a-few-numbers-filled-up-in-manuscript" class="section level3">
<h3>1. A few numbers filled up in manuscript</h3>
<ul>
<li><p>64% of trans-eQTLs colocalize with GWAS signals of blood traits
and autoimmune diseases.</p></li>
<li><p>62% of trans-eQTLs colocalize with GWAS signals of blood
traits.</p></li>
<li><p>11% of trans-eQTLs colocalize with GWAS signals of autoimmune
diseases.</p></li>
</ul>
</div>
<div id="module-v.s.-gwas-heritability" class="section level3">
<h3>2. Module v.s. GWAS heritability</h3>
<p>Yang:</p>
<blockquote>
<p>Test whether gene modules show enrichment in specific GWAS h2g?
Especially GWAS with a coloc trans-eQTL.</p>
</blockquote>
<p>Xuanyao:</p>
<blockquote>
<p>The colocalized trans-eQTL gene modules showed significant
heritability enrichment in the colocalized complex trait.</p>
</blockquote>
<blockquote>
<p>Just to reiterate our analysis: for each co-localized gene module, we
will take all trans genes in the module, get their gene regions+500bp
buffer to each end. We will combine all gene regions into a single
annotation by using LDSC. Then we run LDSC baseline model+the gene
annotation on co-localized traits.</p>
</blockquote>
<div id="understand-ldsc" class="section level4">
<h4>understand LDSC</h4>
<ul>
<li><p>Goal: Partition the heritability of BMI by functional category,
using the baseline model.</p></li>
<li><p>Computation goal: estimate <span
class="math inline">\(\tau_C\)</span> via the multiple regression of
<span class="math inline">\(\chi^2\)</span> statistics against <span
class="math inline">\(l(j, C)\)</span>.</p></li>
<li><p>Define the enrichment of a category to be the proportion of SNP
heritability in the category divided by the proportion of SNPs in that
category.</p></li>
<li><p>Functional categories</p>
<ul>
<li><p>full baseline model’ from 24 publicly available main
annotations.</p></li>
<li><p>For each of these 24 categories, we added a 500-bp window around
the category as an additional category to keep our heritability
estimates from being inflated by heritability in flanking
regions.</p></li>
<li><p>In total, <em>53</em> functional categories, derived from 24 main
annotations.</p></li>
</ul></li>
</ul>
</div>
<div id="download-ldsc" class="section level4">
<h4>download LDSC</h4>
</div>
<div id="prepare-inputs" class="section level4">
<h4>prepare inputs</h4>
<ol start="0" style="list-style-type: decimal">
<li>Define functional categories</li>
</ol>
<ul>
<li><code>baseline.@.annot.gz</code> (updated with new annot) SNP v.s.
category, if in category</li>
</ul>
<ol style="list-style-type: decimal">
<li>LD scores (baseline model LD scores)</li>
</ol>
<ul>
<li><p><code>baseline.@.l2.ldscore.gz</code> (updated with new annot)
SNP v.s. category, ldscore</p></li>
<li><p><code>baseline.@.l2.M_5_50</code> (updated with new annot) for
each category, the number of SNPs with MAF&gt;5%</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li><p>regression weights</p></li>
<li><p>allele frequencies</p></li>
<li><p>GWAS summary statistics</p></li>
</ol>
</div>
<div id="how_to_run_s_ldsc" class="section level4">
<h4>run LDSC</h4>
<ol start="0" style="list-style-type: decimal">
<li>run partitioned LD scores for new defined functional category and
update baseline inputs</li>
</ol>
<ul>
<li><p>Get annotation file: <code>{prefix}.${chr}.annot.gz</code> (same
order of SNPs as in <code>.bim</code>)</p>
<p>Use <code>make_annot.py</code> script to compute annot files from a
gene set.</p></li>
<li><p>calculate LD scores: <code>${prefix}.${chr}.l2.ldscore</code>,
<code>{prefix}.${chr}.l2.M_5_50</code></p></li>
<li><p>Update <code>baseline.@.annot.gz</code></p></li>
<li><p>Update <code>baseline.@.l2.ldscore.gz</code>, calculate ldscores
of SNPs for self-defined category</p></li>
<li><p>Update <code>baseline.@.l2.M_5_50</code>, calculate
baseline.@.l2.M_5_50 for self-defined category</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>run <code>munge_sumstats.py</code> to convert GWAS sum stat file to
<code>.sumstats</code> format</li>
</ol>
<blockquote>
<p>We recommend only keeping HapMap3 SNPs; to do this, you can download
a list of HapMap3 SNPs on LDSC page.</p>
</blockquote>
<ol start="2" style="list-style-type: decimal">
<li>run <code>ldsc.py</code></li>
</ol>
</div>
</div>
<div id="hapmap3-snps" class="section level3">
<h3>HapMap3 SNPs</h3>
<ul>
<li><p>For GWAS SNPs, “recommend only keeping HapMap3 SNPs”.</p></li>
<li><p><code>--w-ld-chr</code> flag gives the location of the LD scores
used for regression weights. It should be a non-partitioned
.l2.ldscore.gz file that was computed using the regression SNPs. We
recommend using HapMap3 SNPs, excluding the HLA region, as default
regression SNPs.</p></li>
</ul>
<div id="concerns" class="section level4">
<h4>Concerns</h4>
<ol style="list-style-type: decimal">
<li>Too few genes in a gene set (module)?</li>
</ol>
<p>Can lead to negative correlation between MAF and L2.</p>
<p>e.g. module 66, 42 genes.</p>
<p>In LDSC tutorial of partitioned ldsc, they have &gt;2400 genes in the
example set.</p>
</div>
</div>
</div>
<div id="apr-20-apr-27" class="section level2">
<h2>Apr 20 &amp; Apr 27</h2>
<div id="updated-figures" class="section level3">
<h3>0. Updated figures</h3>
<ul>
<li>The ratio of number of NULL SNPs used to estimate <span
class="math inline">\(\Sigma\)</span> over the module size for each
module</li>
</ul>
<p><img
src="asset/2022/0420_plot_nullSNP_module_size_pthre1e-04_ratio.png" /></p>
<p>As a result, I choose to use ratio 50 as the threshold for selecting
modules to further look at for signals.</p>
<ul>
<li>Updated signal distribution</li>
</ul>
<div class="float">
<img src="asset/2022/0420_signal_module_chr_rm_infl_ratio_50.png"
alt="signal distribution on modules and chrs" />
<div class="figcaption">signal distribution on modules and chrs</div>
</div>
<p><img src="asset/2022/0420_signal_module_rm_infl_ratio_50.png"
alt="signal distribution on modules" /> <img
src="asset/2022/0420_signal_chr_rm_infl_ratio_50.png"
style="width:50.0%" alt="signal distribution on chrs" /></p>
</div>
<div id="some-numbers-of-eqtlgen-signals" class="section level3">
<h3>1. Some numbers of eQTLGen signals</h3>
<ul>
<li><p>Out of 166 modules, 129 modules are left after removing modules
with the ratio of number of NULL SNPs used to estimate <span
class="math inline">\(\Sigma\)</span> over the module size is smaller
than 50.</p></li>
<li><p>There are 8116 significant (trans-eQTL, module) pairs under
Bonferroni <span
class="math inline">\(p&lt;0.05/9918/129\)</span>.</p></li>
<li><p>Out of 129 modules, 122 have at least one trans-eQTL from ~10k
eQTLGen SNPs.</p></li>
<li><p>There are 2161 eQTLGen SNPs that are significant trans-eQTLs for
at least one module.</p></li>
</ul>
</div>
<div id="overlaps-between-trans-and-cis-esqtls" class="section level3">
<h3>2. Overlaps between trans and cis-e/sQTLs</h3>
<div id="overlap-of-10k-eqtlgen-snps-and-dgn-snps"
class="section level4">
<h4>Overlap of ~10k eQTLGen SNPs and DGN SNPs?</h4>
</div>
<div id="are-trans-eqtls-cis-esqtls-in-dgn" class="section level4">
<h4>Are trans-eQTLs cis-e/sQTLs in DGN?</h4>
<ol style="list-style-type: decimal">
<li><p>1240 trans-eQTLs (out of 2161 ) are also cis-eQTLs, corresponding
to 1302 cis-eQTL-Genes.</p></li>
<li><p>930 trans-eQTLs (out of 2161 ) are also cis-sQTLs, corresponding
to 2807 cis-sQTL-Genes.</p></li>
</ol>
<p><img src="asset/2022/0420_trans_cis_venn.png" style="width:45.0%"
alt="venn" /> <img src="asset/2022/0420_trans_cis_upset.png"
style="width:45.0%" alt="upset" /></p>
</div>
<div id="are-trans-eqtls-cis-esqtls-in-eqtlgen" class="section level4">
<h4>Are trans-eQTLs cis-e/sQTLs in eQTLGen?</h4>
</div>
</div>
<div id="eqtlgen-replication" class="section level3">
<h3>3. eQTLGen replication</h3>
<div id="how-3" class="section level4">
<h4>How?</h4>
</div>
<div id="dgn-replication-in-eqtlgen" class="section level4">
<h4>DGN replication in eQTLGen</h4>
<p>Out of 3899 DGN (SNP, module) signal pairs, 38 of them are also
analyzed in eQTLGen.</p>
<p>Based on these overlaps, the updated p-values threshold for defining
replication is: 0.002631579 , under FDR level 0.1.</p>
<p><strong>As a result, out of 38 DGN signals, 38 are replicated in
eQTLGen.</strong></p>
<p>The p-values of these signals in eQTLGen range: ( 0 , 8.163248e-12
).</p>
</div>
<div id="eQTLGen_replciation_in_DGN" class="section level4">
<h4>eQTLGen replciation in DGN</h4>
<ul>
<li><p>Summary</p>
<p>Out of 8116 eQTLGen (SNP, module) signal pairs, 7577 of them are also
analyzed in DGN.</p>
<p>Based on these overlaps, the updated p-values threshold for defining
replication is: 1.319784e-05, under FDR level 0.1.</p>
<p><strong>As a result, out of 7577 eQTLGen signals, 69 are replicated
in DGN.</strong></p>
<p>The p-values of these signals in DGN range: ( 0 , 1.315094e-05
).</p></li>
<li><p>Interesting signals, in terms of those in eQTLGen but not in DGN.
Particularly, those with extremely small eQTLGen p but extremely large
DGN p, <span class="math inline">\(p_{eQTLGen}=0,
p_{DGN}=1\)</span>.</p></li>
</ul>
<ol style="list-style-type: decimal">
<li>p value comparison in eqtlgen and dgn, for replicated and not
replicated signals</li>
</ol>
<p>I looked at the pvalue differences for eqtlgen and dgn. And found
that SNPs that were not replicated mostly have extremely small eQTLGen p
and extremely large DGN p.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_rep_eqtlgen_in_dgn_p.png" alt="Figure: -log10(trans-PCO P) of eQTLGen and DGN for replicated and not replicated signals." width="70%" />
<p class="caption">
Figure: -log10(trans-PCO P) of eQTLGen and DGN for replicated and not
replicated signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_rep_eqtlgen_in_dgn_p">
Past versions of 0427_rep_eqtlgen_in_dgn_p.png
</button>
</p>
<div id="fig-0427_rep_eqtlgen_in_dgn_p" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_rep_eqtlgen_in_dgn_p.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>p value differences across modules</li>
</ol>
<p>I further looked at the p value differences between eqtlgen and dgn
across modules, to see if larger modules have larger p difference? in
case eQTLGen large modules have inflated signals.</p>
<p>I didn’t see an association between the p-value difference and module
size.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_rep_eqtlgen_in_dgn_p_diff.png" alt="Figure: log10 of p difference between eQTLGen and DGN across modules for replicated and not replicated signals." width="70%" />
<p class="caption">
Figure: log10 of p difference between eQTLGen and DGN across modules for
replicated and not replicated signals.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_rep_eqtlgen_in_dgn_p_diff">
Past versions of 0427_rep_eqtlgen_in_dgn_p_diff.png
</button>
</p>
<div id="fig-0427_rep_eqtlgen_in_dgn_p_diff" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_rep_eqtlgen_in_dgn_p_diff.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Next, I will pick some interesting signals in terms of those in
eQTLGen but not in DGN.</p>
</div>
<div id="interesting-signal-example-1---loci-near-gene-ikzf1"
class="section level4">
<h4>Interesting signal Example 1 - loci near gene <em>IKZF1</em></h4>
<p>This loci is on Chr7, pos <code>^50\\d{6}$</code>. The nearby gene
<em>IKZF1</em> is a TF.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_IKZF1.png" alt="Figure: P values for SNPs near loci IKZF1 and their corresponding modules, for eQTLGen and DGN. SNPs ordered by freq." width="100%" />
<p class="caption">
Figure: P values for SNPs near loci IKZF1 and their corresponding
modules, for eQTLGen and DGN. SNPs ordered by freq.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_IKZF1">
Past versions of 0427_eg_snp_near_IKZF1.png
</button>
</p>
<div id="fig-0427_eg_snp_near_IKZF1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_IKZF1.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In DGN, SNPs in this loci are also identified as signals for a few
modules.</p>
</div>
<div id="interesting-signal-example-2---loci-near-gene-irf1"
class="section level4">
<h4>Interesting signal Example 2 - loci near gene <em>IRF1</em></h4>
<p>This loci is on Chr5, pos <code>^131\\d{6}$</code>. The nearby gene
<em>IRF1</em> is a TF.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_IRF1.png" alt="Figure: P values for SNPs near loci IRF1 and their corresponding modules, for eQTLGen and DGN. Modules ordered by freq." width="60%" />
<p class="caption">
Figure: P values for SNPs near loci IRF1 and their corresponding
modules, for eQTLGen and DGN. Modules ordered by freq.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_IRF1">
Past versions of 0427_eg_snp_near_IRF1.png
</button>
</p>
<div id="fig-0427_eg_snp_near_IRF1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_IRF1.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>In DGN, this whole loci is missed.</p>
</div>
<div
id="interesting-signal-example-3---two-snps-356849749-and-356865776-near-gene-arhgef3"
class="section level4">
<h4>Interesting signal Example 3 - Two SNPs <code>3:56849749</code> and
<code>3:56865776</code> near gene <em>ARHGEF3</em></h4>
<p>Their p-value is 0 in eQTLGen and 1 in DGN for 53 modules.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_ARHGEF3.png" alt="Figure: Two SNPs that are signals in eQTLGen but not in DGN. And their corresponding modules." width="80%" />
<p class="caption">
Figure: Two SNPs that are signals in eQTLGen but not in DGN. And their
corresponding modules.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_ARHGEF3">
Past versions of 0427_eg_snp_near_ARHGEF3.png
</button>
</p>
<div id="fig-0427_eg_snp_near_ARHGEF3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_ARHGEF3.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>However, in DGN, these two SNPs are identified as signals for module
4. Specifically, for <code>module4:3:56849749</code> and
<code>module4:3:56865776</code>, p is 0. In fact, many other SNPs in the
loci around these two SNPs are also DGN signals and they all correspond
to module 4.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0427_eg_snp_near_ARHGEF3_module4_DGN.png" alt="Figure: DGN p values of signals for module 4 in loci ARHGEF3" width="80%" />
<p class="caption">
Figure: DGN p values of signals for module 4 in loci ARHGEF3
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0427_eg_snp_near_ARHGEF3_module4_DGN">
Past versions of 0427_eg_snp_near_ARHGEF3_module4_DGN.png
</button>
</p>
<div id="fig-0427_eg_snp_near_ARHGEF3_module4_DGN" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9bd1bebcc0791ae637a290b2fb4e7f2cb26e473f/docs/asset/2022/0427_eg_snp_near_ARHGEF3_module4_DGN.png" target="_blank">9bd1beb</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-27
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Module 4 was removed from eQTLGen analysis due to its low ratio
of <span class="math inline">\(\frac{\#NULL\ SNPs}{Module\
size}\)</span>.</p></li>
<li><p>Those additional SNPs in the loci are not included in eQTLGen
~10k SNPs.</p></li>
</ul>
</div>
</div>
<div id="many-signals-on-chr6" class="section level3">
<h3>4. Many signals on Chr6</h3>
<p>Signals near HLA region.</p>
</div>
<div id="cis--genes" class="section level3">
<h3>5. cis- genes</h3>
<div id="how-4" class="section level4">
<h4>How?</h4>
<ol style="list-style-type: decimal">
<li><p>Protein coding &amp; lincRNA genes</p></li>
<li><p>Three types of “near genes”: nearest, genes within 1e+5, genes
within 1e+6.</p></li>
</ol>
</div>
<div id="result-file-1" class="section level4">
<h4>Result file</h4>
<p>See file
<code>/scratch/midway2/liliw1/eQTGen_est_Sigma/postanalysis/signal_rm_infl_ratio_50.txt_cis_genes.txt</code>.</p>
</div>
<div id="enrichment-of-the-nearest-genes" class="section level4">
<h4>Enrichment of the nearest genes</h4>
<p>See <a href="https://biit.cs.ut.ee/gplink/l/1y5o1qk9Qk">g:Profiler
enrich here</a>.</p>
</div>
</div>
</div>
<div id="apr-13" class="section level2">
<h2>Apr 13</h2>
<div id="run-pco-on-eqtlgen" class="section level3">
<h3>1. Run PCO on eQTLGen</h3>
<div id="why" class="section level4">
<h4>Why?</h4>
<ul>
<li><p>To see if new signals in eQTLGen with much larger sample
size.</p></li>
<li><p>To see the replcation of DGN signals in eQTLGen.</p></li>
</ul>
</div>
<div id="how-to-run-pco-on-eqtlgen-zs" class="section level4">
<h4>How to run PCO on eQTLGen z’s?</h4>
<ul>
<li><p>Determine modules - 166 DGN modules.</p></li>
<li><p>Sum stat of each SNP on genes in the module.</p></li>
</ul>
</div>
<div id="problem" class="section level4">
<h4>Problem</h4>
<ul>
<li><p><span class="math inline">\(\Sigma\)</span>: no raw expression
data for estimating <span
class="math inline">\(\Sigma\)</span>.</p></li>
<li><p>eQTLGen sum stats from meta analysis, didn’t remove mappability
reads.</p></li>
</ul>
</div>
<div id="solution" class="section level4">
<h4>Solution</h4>
<ul>
<li><p>Use eQTLGen NULL z’s to estimate <span
class="math inline">\(\Sigma\)</span>. By definition.</p></li>
<li><p>Remove genes in module that are cross-map with cis-genes of the
SNP of interest (SNP-wise PCO).</p></li>
</ul>
</div>
<div id="observation-1" class="section level4">
<h4>Observation</h4>
<ul>
<li><p>signals claimed by Bonferroni correction, i.e. <span
class="math inline">\(0.05/166/9918\)</span></p></li>
<li><p>p-values distribution</p></li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/0413_p_hist.png" alt="Figure: Distribution of p-values." width="50%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_p_hist">
Past versions of 0413_p_hist.png
</button>
</p>
<div id="fig-0413_p_hist" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_p_hist.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_p_hist.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/0413_p_qq.png" alt="Figure: Distribution of p-values." width="50%" />
<p class="caption">
Figure: Distribution of p-values.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_p_qq">
Past versions of 0413_p_qq.png
</button>
</p>
<div id="fig-0413_p_qq" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_p_qq.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_p_qq.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Signals distribution for each (module, chr), module, and chr.</li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_signal_module_chr.png" alt="Figure: Signal distribution for pair of (module, chr)." width="100%" />
<p class="caption">
Figure: Signal distribution for pair of (module, chr).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_module_chr">
Past versions of 0413_signal_module_chr.png
</button>
</p>
<div id="fig-0413_signal_module_chr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_module_chr.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_module_chr.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<p><img src="asset/2022/0413_signal_module.png" alt="Figure: Signal distribution on each module and chr." width="50%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_module">
Past versions of 0413_signal_module.png
</button>
</p>
<div id="fig-0413_signal_module" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_module.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_module.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2022/0413_signal_chr.png" alt="Figure: Signal distribution on each module and chr." width="50%" />
<p class="caption">
Figure: Signal distribution on each module and chr.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_chr">
Past versions of 0413_signal_chr.png
</button>
</p>
<div id="fig-0413_signal_chr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e3006f60bf8cb2e98161fdc7a4ab0ecc8c3d1843/docs/asset/2022/0413_signal_chr.png" target="_blank">e3006f6</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-25
</td>
</tr>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_chr.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="simulation---are-eqtlgen-pco-signals-of-large-modules-inflated"
class="section level3">
<h3>2. Simulation - Are eQTLGen PCO signals of large modules
inflated?</h3>
<div id="problem-1" class="section level4">
<h4>Problem</h4>
<p>We see from the above figure the trend, which is larges modules have
more signals. There can be two possible reasons,</p>
<ul>
<li><p>Those are true signals, as larger modules with more genes could
have more trans-eQTLd.</p></li>
<li><p>Those are inflated signals (false positives).</p></li>
</ul>
</div>
<div id="i-looked-into-the-second-reason" class="section level4">
<h4>I looked into the second reason</h4>
<p>Why is it possible? Because at the very beginning, we first used the
correlation matrix of gene samples from DGN to estiamte <span
class="math inline">\(\Sigma\)</span> and use it as <span
class="math inline">\(\hat\Sigma\)</span> for eQTLGen. And we observed
all 9918 SNPs are significant, and SNPs’ p-values for each module are
almost all 0. That seemed unreasonable.</p>
<p>And we found that may be because of the use of inaccurate <span
class="math inline">\(\hat\Sigma\)</span>. So we decided to use <span
class="math inline">\(\hat\Sigma\)</span> from <span
class="math inline">\(cor(Z^{NULL})\)</span> of eQTLGen’s own null sum
stat.</p>
<p>We learned that inaccurate <span
class="math inline">\(\hat\Sigma\)</span> could inflate signals.</p>
<p>And using <span class="math inline">\(cor(Z^{NULL})\)</span> can also
lead to inaccurate <span class="math inline">\(\hat\Sigma\)</span> is
because for a module of <span class="math inline">\(K\)</span> genes, we
need to estimate a <span class="math inline">\(K \times K\)</span>
matrix, using <span class="math inline">\(Z^{NULL}\)</span> of dimension
<span class="math inline">\(m \times K\)</span> with <span
class="math inline">\(m\)</span> NULL SNPs of these genes from
eQTLGen.</p>
<p>This can be problematic for large modules with large <span
class="math inline">\(K\)</span>, because</p>
<ul>
<li><p>There are fewer SNPs with null sum stat for all <span
class="math inline">\(K\)</span> genes in the module, i.e. smaller <span
class="math inline">\(m\)</span>.</p></li>
<li><p>If <span class="math inline">\(m\)</span> is not larger enough
than <span class="math inline">\(K\)</span>, <span
class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span> is inaccurate.</p></li>
<li><p>There can be lots of extremely small p-values. Inflate PCO
signals.</p></li>
</ul>
</div>
<div id="observation-2" class="section level4">
<h4>Observation</h4>
<p>We did observe a decreasing number of null SNPs for larger
modules.</p>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png" alt="Figure: We did observe a decreasing number of null SNPs for larger modules." width="100%" />
<p class="caption">
Figure: We did observe a decreasing number of null SNPs for larger
modules.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_signal_moduleplot_nullSNP_module_size_pthre1e-04">
Past versions of
0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png
</button>
</p>
<div id="fig-0413_signal_moduleplot_nullSNP_module_size_pthre1e-04"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_signal_moduleplot_nullSNP_module_size_pthre1e-04.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="">m/K ratio</a></p>
</div>
<div id="eqtlgen_inflation_simulation" class="section level4">
<h4>Does <span class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span> lead to inflated PCO
signals?</h4>
<p>We want to check if <span class="math inline">\(\hat\Sigma\)</span>
by <span class="math inline">\(cor(Z^{NULL})\)</span> really lead to
extremely small pvalues and inflated PCO signals.</p>
<ul>
<li><p>How to check?</p>
<p>We use simulations. Specifically,</p>
<ol style="list-style-type: decimal">
<li>Choose a real module from DGN.</li>
<li>Generate sum stat (Z) of NULL SNPs from the real <span
class="math inline">\(\Sigma\)</span>.</li>
<li>Use these NULL SNPs to estimate a <span
class="math inline">\(\hat\Sigma\)</span>.</li>
<li>Run PCO on another set of NULL SNPs using the true <span
class="math inline">\(\Sigma\)</span> and <span
class="math inline">\(\hat\Sigma\)</span> by <span
class="math inline">\(cor(Z^{NULL})\)</span>.</li>
<li>Compare their p-values.</li>
</ol>
<p>The p-values using true <span class="math inline">\(\Sigma\)</span>
should represent the true case. By comparing p-values using <span
class="math inline">\(\hat\Sigma\)</span> to it, we should be able to
see if <span class="math inline">\(P_{\hat\Sigma}\)</span> have extreme
values.</p></li>
<li><p>Simulation setting</p>
<ol style="list-style-type: decimal">
<li><p>I looked at multiple modules with various module size, including
module 1-11, 15, 20, 30, 40, 50, 60, 70, 90, 100, 150, 166.</p></li>
<li><p>I looked at various <span class="math inline">\(m/K\)</span>
ratios for estimating <span class="math inline">\(\hat\Sigma\)</span>,
including 1, 5, 10, 50, 100, 150.</p></li>
</ol></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_plot_hist_Sigma_DGN_module1.png" alt="Figure: Histogram of p-values using various Sigma's." width="100%" />
<p class="caption">
Figure: Histogram of p-values using various Sigma’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_plot_hist_Sigma_DGN_module1">
Past versions of 0413_plot_hist_Sigma_DGN_module1.png
</button>
</p>
<div id="fig-0413_plot_hist_Sigma_DGN_module1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_plot_hist_Sigma_DGN_module1.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<img src="asset/2022/0413_plot_qq_Sigma_DGN_module1.png" alt="Figure: QQ-plot of p-values using various Sigma's." width="70%" />
<p class="caption">
Figure: QQ-plot of p-values using various Sigma’s.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0413_plot_qq_Sigma_DGN_module1">
Past versions of 0413_plot_qq_Sigma_DGN_module1.png
</button>
</p>
<div id="fig-0413_plot_qq_Sigma_DGN_module1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ec19552d32833de8ff0ec04b2ee88705690f0aa3/docs/asset/2022/0413_plot_qq_Sigma_DGN_module1.png" target="_blank">ec19552</a>
</td>
<td>
liliw-w
</td>
<td>
2022-04-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="observation-3" class="section level4">
<h4>Observation</h4>
<ul>
<li><p>P-values looked similar to the true case when ratio <span
class="math inline">\(&gt;50\)</span>.</p></li>
<li><p>The ratio has smaller inflation effects on smaller modules,
e.g. p-values of module 166 using <span
class="math inline">\(\Sigma\)</span> of ratio 20 are closer to that of
module 1 using <span class="math inline">\(\Sigma\)</span> of same
ratio.</p></li>
<li><p>In eQTLGen analysis, ??? modules have <span
class="math inline">\(m/K\)</span> ratio <span
class="math inline">\(&gt;50\)</span>.</p></li>
</ul>
</div>
</div>
<div id="in-dgn-larger-module-more-signals" class="section level3">
<h3>3. In DGN, larger module, more signals?</h3>
<p>We wonder if this trend also exists in DGN data.</p>
<p><span class="math inline">\(\hat\Sigma\)</span>: module size v.s.
sample size</p>
</div>
<div id="remaining-questions" class="section level3">
<h3>4. Remaining questions</h3>
<ul>
<li>Inaccurate <span class="math inline">\(\hat\Sigma\)</span> leads to
false negatives? in DGN?</li>
</ul>
</div>
</div>
<div id="mar-30" class="section level2">
<h2>Mar 30</h2>
<ol style="list-style-type: decimal">
<li>eQTLGen analysis</li>
</ol>
</div>
<div id="mar-25" class="section level2">
<h2>Mar 25</h2>
<ol style="list-style-type: decimal">
<li>Signal SNPs genomic location annotation</li>
</ol>
<ul>
<li><p>torus: GTEx</p></li>
<li><p>snpEff</p></li>
</ul>
</div>
<div id="mar-02" class="section level2">
<h2>Mar 02</h2>
<div id="about-replication-of-signals-in-eqtlgen"
class="section level3">
<h3>1. About “replication of signals in eQTLGen”</h3>
<p>In the abstract of The Biology of Genome conference, you added
something as</p>
<blockquote>
<p>The trans-eQTLs are highly replicable, and all were replicated in the
eQTLGen.</p>
</blockquote>
<p>At first, I thought you were referring to the replication of the
identified trans-eQTLs in eQTLGen. But I didn’t remember we had this
observation. So I wondered where this impression came from.</p>
<p>I think maybe because <a
href="Log2021.html#1_Are_the_dark_green_signals_eQTLGen_signals">previous
section</a>. I looked at what are those lead-SNPs of the colocalized
regions (dark green regions). Specifically, are those dark green signals
eQTLGen signals (cis- or trans- signals)?</p>
<p>I observed that, out of 74 unique lead SNPs of colocalized regions
for all blood-related traits, 65 are included in eQTLGen SNPs. And 63
are eQTLGen cis- or trans- signals.</p>
</div>
<div id="preious-eqtgen-signals-using-trans-pco" class="section level3">
<h3>2. preious eQTGen signals using trans-PCO</h3>
<table>
<thead>
<tr class="header">
<th align="left">Sigma</th>
<th align="left">#modules</th>
<th align="right">PCO_lambda_thre</th>
<th align="left">#signals</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">less</td>
<td align="right">1.0</td>
<td align="left">~367</td>
</tr>
<tr class="even">
<td align="left">DGN</td>
<td align="left">less</td>
<td align="right">0.1</td>
<td align="left">9918</td>
</tr>
<tr class="odd">
<td align="left">DGN</td>
<td align="left">more</td>
<td align="right">1.0</td>
<td align="left">~460</td>
</tr>
<tr class="even">
<td align="left">DGN</td>
<td align="left">more</td>
<td align="right">0.1</td>
<td align="left">9905</td>
</tr>
<tr class="odd">
<td align="left">nullz</td>
<td align="left">more</td>
<td align="right">0.1</td>
<td align="left">NA</td>
</tr>
</tbody>
</table>
<ol style="list-style-type: decimal">
<li><p>At first, we used <span
class="math inline">\(\text{PCO}_{\lambda&lt;1}\)</span> and observed
hundreds of eQTLGen SNPs that are significant trans-eQTL
signals.</p></li>
<li><p>We decided to choose <span
class="math inline">\(\text{PCO}_{\lambda&lt;0.1}\)</span> over <span
class="math inline">\(\text{PCO}_{\lambda&lt;1}\)</span>.</p></li>
<li><p>Then we observed almost all 9918 eQTLGen SNPs are trans-eQTL
signals. We doubted some signals are false positives.</p></li>
</ol>
<p><a
href="Log2021.html#Replication_of_eQTLGen_results_in_trans-PCO_of_eQTLGen_summary_stats">Replication_of_eQTLGen_results_in_trans-PCO_of_eQTLGen_summary_stats</a></p>
<p><a
href="Log2021.html#Why_there_are_so_many_eQTLGen_signals_using_PCO_(%7Blambda01%7D)">Why_there_are_so_many_eQTLGen_signals</a></p>
<ol start="4" style="list-style-type: decimal">
<li>We thought false positives occurred because we used <span
class="math inline">\(\Sigma\)</span> of DGN for estimating <span
class="math inline">\(\hat{\Sigma}\)</span> of eQTLGen. So we <a
href="Log2021.html#Build_Sigma_(hat%7BSigma%7D_%7BDGN%7D)_and_(hat%7BSigma%7D_%7Bnullz%7D)_and_check_PCO_pvalues">tried
using null zscores from eQTLGen to build sigma</a>. We observed a
significant drop of pvalues.</li>
</ol>
</div>
</div>
<div id="feb-16" class="section level2">
<h2>Feb 16</h2>
<div id="simulation" class="section level3">
<h3>1. Simulation</h3>
<ul>
<li><p>Re-run simulation using <span
class="math inline">\(\Sigma\)</span> from the updated 166 modules of
DGN_no_filter_on_mappability.</p></li>
<li><p>QQ plot of null distribution.</p>
<p>Figures are stored on server.</p>
<p><a
href="/scratch/midway2/liliw1/simulation_lambda0.1/new_Sigma/qqplot_null.png">qq
plot</a> <a
href="/scratch/midway2/liliw1/simulation_lambda0.1/new_Sigma/qqplot_null_with_hist.png">qqplot
with histogram</a></p></li>
<li><p>Power plot.</p></li>
</ul>
</div>
</div>
<div id="jan-26" class="section level2">
<h2>Jan 26</h2>
<div id="update-coloc-figures-with-less-stringent-pvalue-threshold"
class="section level3">
<h3>1. Update coloc figures with less stringent pvalue threshold</h3>
<p>Previously, we defined the green regions with module-QTL <span
class="math inline">\(p&lt;1e-8\)</span>. This threshold is not
premutation based. I looked at the permutation-based p threshold, and it
is <span class="math inline">\(\approx 3e-8\)</span>. We’d like to use a
less stringent threshold, so I chose <span
class="math inline">\(p&lt;1e-7\)</span> to define the green
regions.</p>
<p>In total, we considered three (four) types of coloc. Coloc with (1)
29 blood related traits from ukbb, (2) 11 immune traits, (3) 27 more
traits from ukbb. (The 4-th type is coloc with cis genes.)</p>
<p>I updated the figures and tables.</p>
<ul>
<li>figures</li>
</ul>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">29
blood related traits from ukbb</a>; <a
href="../docs/asset/1115_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<p><a
href="../docs/asset/2022/0126_immune_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">11
immune traits</a>; <a
href="../docs/asset/1129_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_more_traits_all.traits.coloc.region.summary.pvalThre-1e-07.png">27
more traits from ukbb</a>; <a
href="../docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">earlier
result with 1e-8 thre</a></p>
<ul>
<li>tables with detailed numerical results</li>
</ul>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_blood_traits_coloc_region_prop_pvalThre-1e-07.txt">29
blood related traits from ukbb</a></p>
<p><a
href="../docs/asset/2022/0126_immune_traits_coloc_region_prop_pvalThre-1e-07.txt">11
immune traits</a></p>
<p><a
href="../docs/asset/2022/0126_ukbb_coloc_more_traits_coloc_region_prop_pvalThre-1e-07.txt">27
more traits from ukbb</a></p>
</div>
<div id="other-types-of-modules-coloc-with-traits"
class="section level3">
<h3>2. Other types of modules coloc with traits</h3>
</div>
</div>
<div id="jan-19" class="section level2">
<h2>Jan 19</h2>
<div id="comprehensive-analysis-of-coloc-regions-of-trait-ms"
class="section level3">
<h3>1. Comprehensive analysis of coloc regions of trait MS</h3>
<div
id="add-mss-result-to-previous-coloc-analysis-dark-green-coloc-regions"
class="section level4">
<h4>Add MS’s result to previous coloc analysis (dark green coloc
regions)</h4>
<p>See <a
href="Log2021.html#coloc-region-between-module-and-immune-trait">earlier
section</a>.</p>
</div>
<div id="dark-blue-coloc-regions" class="section level4">
<h4>dark blue coloc regions</h4>
<p>There are 4 dark blue coloc regions, i.e. coloc regions with
module-QTL lead pvalue &lt; 1e-5.</p>
<table>
<thead>
<tr class="header">
<th align="left">Region</th>
<th align="right">nSNPs</th>
<th align="right">Pval</th>
<th align="right">PP4</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">module121:19:10423815</td>
<td align="right">140</td>
<td align="right">1.5e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="even">
<td align="left">module108:7:50308811</td>
<td align="right">122</td>
<td align="right">1.6e-06</td>
<td align="right">0.98</td>
</tr>
<tr class="odd">
<td align="left">module50:7:149331042</td>
<td align="right">265</td>
<td align="right">2.0e-06</td>
<td align="right">0.94</td>
</tr>
<tr class="even">
<td align="left">module44:7:50308527</td>
<td align="right">69</td>
<td align="right">0.0e+00</td>
<td align="right">0.89</td>
</tr>
</tbody>
</table>
<p>In <a href="#make-a-story-out-of-trans-signals">earlier section</a>,
the region “module108:7:50308811” coloc with MS is included in these
dark blue regions.</p>
</div>
</div>
<div id="more-traits-from-ukbb" class="section level3">
<h3>2. More traits, from UKBB</h3>
<div id="extracted-traits" class="section level4">
<h4>Extracted traits</h4>
<ol style="list-style-type: decimal">
<li>How?</li>
</ol>
<p>I extracted specific traits from ukbb traits manifest file, based
on:</p>
<ul>
<li><p>Traits from eQTLGen traits</p></li>
<li><p>Traits from Mu et al paper</p></li>
<li><p>Powerful (and interesting) ukbb GWASs</p>
<p>Sorted by case number of EUR <code>n_cases_EUR</code>.</p></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>What traits?</li>
</ol>
<ul>
<li><p>26 ukbb GWASs in total, including height, BMI, Cholesterol,
Hypertension, Asthma, Type 2 diabetes etc.</p></li>
<li><p>Exclude height, as not GWASed for EUR.</p></li>
<li><p>Use height sum stat from <a
href="height_BMI_meta_data_source">another meta analysis</a> instead. It
also has another BMI sum stat.</p>
<p>Meta analyze inds from GIANT and ukbb.</p>
<blockquote>
<p>Meta-analysis of genome-wide association studies for height and body
mass index in ~700,000 individuals of European ancestry.</p>
</blockquote></li>
<li><p>Thus, as a result, 27 GWASs. 22 unique traits.</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li>Visualize the sample sizes of selected traits</li>
</ol>
<p><img src="asset/2022/0119_phenotype_sub_N.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_phenotype_sub_N">
Past versions of 0119_phenotype_sub_N.png
</button>
</p>
<div id="fig-0119_phenotype_sub_N" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_phenotype_sub_N.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="coloc-results-more-traits-from-ukbb" class="section level4">
<h4>coloc results {More-traits-from-UKBB}</h4>
<ol style="list-style-type: decimal">
<li>Visualize</li>
</ol>
<p>See <a
href="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png">figure</a>.</p>
<p><img src="asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-0119_selected_ukbb_all.traits.coloc.region.summary">
Past versions of 0119_selected_ukbb_all.traits.coloc.region.summary.png
</button>
</p>
<div id="fig-0119_selected_ukbb_all.traits.coloc.region.summary"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/1b40fde9ad9a399b0c182ac5be7b16bdd4400b98/docs/asset/2022/0119_selected_ukbb_all.traits.coloc.region.summary.png" target="_blank">1b40fde</a>
</td>
<td>
llw
</td>
<td>
2022-01-19
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For detailed numbers, see <a
href="asset/2022/0119_selected_ukbb_coloc_region_prop.txt">file</a>.</p>
<ol start="2" style="list-style-type: decimal">
<li>In-depth visualization for selected traits (dark blue regions)</li>
</ol>
<p><a href="asset/2022/0119_BMI_coloc.region.summary.png">BMI</a></p>
<p><a href="asset/2022/0119_BMI-2_coloc.region.summary.png">BMI</a></p>
<p><a
href="asset/2022/0119_Cholesterol_coloc.region.summary.png">Cholesterol</a></p>
<p><a
href="asset/2022/0119_HDL-cholesterol_coloc.region.summary.png">HDL-cholesterol</a></p>
<p><a
href="asset/2022/0119_Hayfever,-allergic-rhinitis-or-eczema_coloc.region.summary.png">Hayfever,-allergic-rhinitis-or-eczema</a></p>
<p><a
href="asset/2022/0119_Hypertension_coloc.region.summary.png">Hypertension</a></p>
</div>
</div>
</div>
<div id="jan-12" class="section level2">
<h2>Jan 12</h2>
<div id="make-a-story-out-of-trans-signals" class="section level3">
<h3>1. Make a story out of the trans- signals</h3>
<div id="more-concrete-examples" class="section level4">
<h4>More concrete examples</h4>
<ol style="list-style-type: decimal">
<li>Trait, gene, trans- signal</li>
</ol>
<p>SLE signals in Fig. 5.</p>
<blockquote>
<p>reflecting the involvement of interferon signaling in SLE
pathophysiology</p>
</blockquote>
<p><img src="asset/2022/0112_Systemic lupus erythematosus_tile.png" width="100%" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-no-spaces-0112_Systemic-lupus-erythematosus_tile">
Past versions of "0112_Systemic lupus erythematosus_tile.png"
</button>
</p>
<div id="fig-no-spaces-0112_Systemic-lupus-erythematosus_tile"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/664d0d7bdcedd0fb3164db435cb8bffdce9ea41e/docs/asset/2022/0112_Systemic lupus erythematosus_tile.png" target="_blank">664d0d7</a>
</td>
<td>
llw
</td>
<td>
2022-01-13
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ol start="2" style="list-style-type: decimal">
<li>additional examples of trans-eQTL variants associated with
traits</li>
</ol>
<ul>
<li><p>ZNF131 locus, age of menarche</p>
<p>Supplementary Figure 11A, rs1532331</p></li>
<li><p>FADS1/FADS2 locus, lipid levels</p>
<p>Supplementary Figure 11B, rs174574</p></li>
<li><p>IFIH1 locus, inflammatory bowel disease and SLE</p>
<p>Supplementary Figure 11C, rs1990760</p></li>
<li><p>GSDMB locus, asthma</p>
<p>Supplementary Figure 11D, rs7216389</p></li>
<li><p>CLOCK locus, height</p>
<p>Supplementary Figure 11E, rs1311351834</p></li>
</ul>
<ol start="3" style="list-style-type: decimal">
<li><p>eQTSs genes GO enrichment</p></li>
<li><p>eQTSs genes associated with HDL</p></li>
</ol>
</div>
</div>
</div>
<div id="jan-05" class="section level2">
<h2>Jan 05</h2>
<div id="trans-eqtl-and-complex-disease-in-eqtlgen-võsa-franke-et-al"
class="section level3">
<h3>1. trans-eQTL and complex disease in <a
href="https://www.nature.com/articles/s41588-021-00913-z">eQTLGen, Võsa,
Franke et al</a></h3>
<div id="question-are-there-any-trans-eqtls-that-are-gwas-hits"
class="section level4">
<h4>Question: are there any trans-eQTLs that are GWAS hits?</h4>
<p>Observations from the paper:</p>
<ul>
<li><p>Meta-analysis using up to <strong>31,684 blood samples</strong>
from 37 eQTLGen Consortium cohorts.</p></li>
<li><p>For trans, they focused on <strong>10,317 trait-associated
SNPs</strong>.</p></li>
</ul>
<p>The paper linked trans-eQTLs with traits in two ways to find the
potential driver genes for traits.</p>
<ol style="list-style-type: decimal">
<li>trans-eQTL analysis</li>
</ol>
<ul>
<li><p>One-third of trait-associated variants have distal effects.</p>
<p>Identified 59,786 trans-eQTL, representing 3,853 SNPs (<strong>37% of
tested GWAS SNPs</strong>) and 6,298 genes (32% of tested genes).</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311)
identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote></li>
<li><p>Identify genes that are coordinately affected by <strong>multiple
independent trait-associated SNPs</strong>.</p>
<p>Identified <strong>47 GWAS traits</strong> for which at least four
independent variants affected the same gene in trans (Supplementary
Tables 10). Examples genes affected by at least three SLE-associated
genetic variants.</p></li>
</ul>
<p>But, Individual trans-eQTL effects too weak to detect. Another way to
look for the potential driver genes for traits and the “core” genes:</p>
<ol start="2" style="list-style-type: decimal">
<li>eQTSs (associations between PGSs and gene expression, Fig. 6a,
6b)</li>
</ol>
<ul>
<li><p>Individual trait-associated SNPs are combined into a PGS that is
associated with gene expression.</p></li>
<li><p>when the PGS for a trait correlates with the expression of a
gene, trans-eQTL effects of individual risk variants converge on that
gene, and it can be prioritized as a putative driver of the
disease.</p></li>
<li><p>1,263 traits in total. 18,210 eQTSs representing 689 unique
traits (<strong>55% of tested traits</strong>) and 2,568 genes
(<strong>13% of tested genes</strong>).</p></li>
<li><p>Of these genes, 719 (<strong>28%</strong>) were not identified in
the trans-eQTL analysis.</p></li>
</ul>
<p><strong>Therefore, all 3,853 trans-eQTLs eQTLGen identified are GWAS
hits.</strong></p>
</div>
<div id="question-if-so-can-we-replicate-them" class="section level4">
<h4>Question: if so, can we replicate them?</h4>
<p>Among 10,317 trait-associated SNPs, 9,056 (~89%) are included in DGN
SNPs. Among 3,853 eQTLGen trans- eQTLs, <strong>27 (~0.7%) are
replicated in DGN signals (1,863,</strong> <span
class="math inline">\(p&lt;1e-8\)</span>).</p>
<p>In the paper, another trans- study was mentioned,</p>
<blockquote>
<p>The largest previous trans-eQTL meta-analysis in blood (N = 5,311)
identified trans-eQTL for <strong>8% of tested SNPs</strong>.</p>
</blockquote>
</div>
</div>
<div id="gene_SENP7" class="section level3">
<h3>2. gene <em>SENP7</em></h3>
<div
id="the-cross-mappability-of-senp7-and-the-genes-in-the-trans-modules"
class="section level4">
<h4>the cross mappability of SENP7 and the genes in the trans
modules</h4>
<p>As noted in <a href="Log2021.html#what_is_gene_SENP7">earlier
section</a>, we observed,</p>
<ul>
<li><em>SENP7</em> is the neasrest gene of the lead SNPs of coloc
regions.</li>
<li>Gene <em>ZNF90P1</em> is located within <em>SENP7</em>.</li>
<li>Modules 25, 51, 153, 156 have coloc regions with immune traits and
consist of many zinc finger genes.</li>
</ul>
<p>Therefore, we wanted to know,</p>
<ol style="list-style-type: decimal">
<li>Does gene <em>ZNF90P1</em> have expression data?</li>
</ol>
<p>I looked at the expression matrix of 13634 genes. <em>ZNF90P1</em>
doesn’t have expression data in this matrix.</p>
<ol start="2" style="list-style-type: decimal">
<li>What is the cross mappability between <em>SENP7</em> and the zinc
finger genes in the modules?</li>
</ol>
<p>See files for the cross mappability for <a
href="asset/2022/0105_SENP7_module25_size123_crossMap.txt">module
25</a>, <a
href="asset/2022/0105_SENP7_module51_size61_crossMap.txt">module 51</a>,
<a href="asset/2022/0105_SENP7_module153_size15_crossMap.txt">module
153</a>, <a
href="asset/2022/0105_SENP7_module156_size13_crossMap.txt">module
156</a>.</p>
<p>The columns include: “score” for cross mappability score,
“score_map1” for mappability score of gene1, “score_map2” for
mappability score of gene2.</p>
<ol start="3" style="list-style-type: decimal">
<li>For the genes in the trans modules,</li>
</ol>
<p>See a previous <a
href="asset/1213_cd_ibd_Allergy_coloc_module_david_query.xlsx">file
here</a>.</p>
</div>
<div id="highlighting-senp7-as-a-trans-meqtl" class="section level4">
<h4>highlighting SENP7 as a trans-meQTL</h4>
<p><a
href="https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x"
class="uri">https://www-nature-com.proxy.uchicago.edu/articles/s41588-021-00969-x</a>
Another paper highlighting SENP7 as a trans-meQTL.</p>
</div>
</div>
<div id="pqtl-resource" class="section level3">
<h3>3. pQTL resource</h3>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       highr_0.10        compiler_4.2.3    pillar_1.9.0     
 [5] bslib_0.5.0       later_1.3.1       git2r_0.32.0      jquerylib_0.1.4  
 [9] tools_4.2.3       getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7   
[13] evaluate_0.21     lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3  
[17] rlang_1.1.1       cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7       
[21] xfun_0.39         fastmap_1.1.1     httr_1.4.6        stringr_1.5.0    
[25] knitr_1.43        fs_1.6.2          vctrs_0.6.3       sass_0.4.6       
[29] rprojroot_2.0.3   glue_1.6.2        R6_2.5.1          processx_3.8.2   
[33] fansi_1.0.4       rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3   
[37] whisker_0.4.1     ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5  
[41] httpuv_1.6.11     utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
