<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Log2024</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Log2024</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-28
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree8133a83797a22a126901ca88106dd5f46e60a6e5targetblank8133a83a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/8133a83797a22a126901ca88106dd5f46e60a6e5" target="_blank">8133a83</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree8133a83797a22a126901ca88106dd5f46e60a6e5targetblank8133a83a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/8133a83797a22a126901ca88106dd5f46e60a6e5" target="_blank">8133a83</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Unstaged changes:
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Log2024_new_beginning.Rmd</code>)
and HTML (<code>docs/Log2024_new_beginning.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8133a83797a22a126901ca88106dd5f46e60a6e5/analysis/Log2024_new_beginning.Rmd" target="_blank">8133a83</a>
</td>
<td>
Lili Wang
</td>
<td>
2025-04-28
</td>
<td>
wflow_publish("analysis/Log2024_new_beginning.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d3806017b549188b7f4b1b8ea7ad3460f9da628d/docs/Log2024_new_beginning.html" target="_blank">d380601</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/daeae1d773978aad145988c7106f05066f8898b2/analysis/Log2024_new_beginning.Rmd" target="_blank">daeae1d</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
<td>
update figures after harmonizing ref genomes of two
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1aa462228f56e5d571960d08b4793646a1c38c06/docs/Log2024_new_beginning.html" target="_blank">1aa4622</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/e69c1a0ae421d413fbd10c2320a363ba41ca65f3/analysis/Log2024_new_beginning.Rmd" target="_blank">e69c1a0</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
<td>
wflow_publish("analysis/Log2024_new_beginning.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/7c5ef748d49c731790817e97977c3b6cb2351d70/docs/Log2024_new_beginning.html" target="_blank">7c5ef74</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/13fe371e67517ac3d29a30e782497764c3b5db3d/analysis/Log2024_new_beginning.Rmd" target="_blank">13fe371</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
<td>
update
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/a94a2f1d1c6312f94926f0ce9db56a42f9643ca2/analysis/Log2024_new_beginning.Rmd" target="_blank">a94a2f1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a94a2f1d1c6312f94926f0ce9db56a42f9643ca2/docs/Log2024_new_beginning.html" target="_blank">a94a2f1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
<td>
update
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8b813c24b4e0392a2a982455fd964e15dc108843/docs/Log2024_new_beginning.html" target="_blank">8b813c2</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f1090b1fecef54522cbb07c77c3b22db0f7dfc3e/docs/Log2024_new_beginning.html" target="_blank">f1090b1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ee589703409e38f18cd760bf00a45681c72df3b6/docs/Log2024_new_beginning.html" target="_blank">ee58970</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c50b98f4c75cb14be2dcf4f0cecc8988b4fb55b0/docs/Log2024_new_beginning.html" target="_blank">c50b98f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-25
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d010e244d32bddbd389f6364cd7b8d26b9330b0c/docs/Log2024_new_beginning.html" target="_blank">d010e24</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/713cef5241858d1b7a141810812aee50d563b70e/analysis/Log2024_new_beginning.Rmd" target="_blank">713cef5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8c8f6168e9e9acdfc99046a082f9f5f9ec90aae5/docs/Log2024_new_beginning.html" target="_blank">8c8f616</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f6aa2c741cc8496342de6bfac181650222aafd2f/analysis/Log2024_new_beginning.Rmd" target="_blank">f6aa2c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
fix toc
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f6c98b089319bf6c885ad76c146733207f0fad75/docs/Log2024_new_beginning.html" target="_blank">f6c98b0</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/24a0be1963d9bb1f4aabf38bd9f386b81c04c1dd/analysis/Log2024_new_beginning.Rmd" target="_blank">24a0be1</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
add new week
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2ab6eac96840c3dea03bec5c7a23fab6b9dcb315/docs/Log2024_new_beginning.html" target="_blank">2ab6eac</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/55c9df22c52dc3e6c03eff7173d4da55e9157df1/analysis/Log2024_new_beginning.Rmd" target="_blank">55c9df2</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
<td>
add new weeks
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8b691e2e4bcb05f0031ed5e2d54300d101cb60c8/docs/Log2024_new_beginning.html" target="_blank">8b691e2</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8bfc0386a02aec0687e4c678333f7bf06d7f10c4/analysis/Log2024_new_beginning.Rmd" target="_blank">8bfc038</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-09
</td>
<td>
add a new year
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
</style>
<p><strong>If any figures don’t show, try opening in
Safari.</strong></p>
<div id="manuscript-finalization" class="section level1">
<h1>Manuscript finalization</h1>
<div id="rerun-signals-for-narrow-marks" class="section level2">
<h2>Rerun signals for narrow marks</h2>
<p>Rerun all datasets by combining peak groups containing multiple peaks
and single peaks.</p>
<ul>
<li>Macrophage - three narrow marks, two conditions</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K27ac_Flu_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K27ac_NI_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K4me1_Flu_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K4me1_NI_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K4me3_Flu_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K4me3_NI_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p><strong>Group meta</strong></p>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/H3K27ac_Flu_peak_group_window50kb.rds</code></p>
<ul>
<li>LCL - three narrow marks</li>
</ul>
<p><code>finalized_pip/lcl/results/{h_mark}_p_comb_window50kb_cis500kb_allchr_fdr_added.txt.gz</code></p>
<ul>
<li>GTEx narrow mark in four tissues</li>
</ul>
<p><code>finalized_pip/multitissue/results/{tissue_type}_p_comb_window50kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<div id="compare-to-single-peak" class="section level3">
<h3>Compare to single peak</h3>
<ul>
<li>Macrophage</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/figs/plt_cpeak_ccomb_*_window50kb_cis100kb.pdf</code></p>
<ul>
<li>LCL</li>
</ul>
<p><code>finalized_pip/lcl/figs/plt_cpeak_ccomb_{h_mark}_window50kb_cis500kb.pdf</code></p>
<ul>
<li>GTEx</li>
</ul>
<p><code>finalized_pip/multitissue/figs/plt_cpeak_ccomb_{tissue_type}_window50kb_cis100kb.pdf</code></p>
</div>
<div id="enrichment-of-cqtls-for-eqtls" class="section level3">
<h3>Enrichment of cQTLs for eQTLs</h3>
<ul>
<li>LCL</li>
</ul>
<p><code>finalized_pip/lcl/figs/plt_enrich_extra_cqtl_eqtl_{h_mark}_window50kb_cis500kb.pdf</code></p>
</div>
</div>
<div id="rerun-signals-for-broad-marks" class="section level2">
<h2>Rerun signals for broad marks</h2>
<p>Finalize the pipeline - Determine how to filter segments</p>
<ul>
<li>Macrophage - one broad mark H3K27me3, two conditions</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/p/p_pco_acat_seg_group_H3K27me3_(NI|Flu)_window50kb_seg5kb_cis100kb_allchr_fdr_added.txt.gz</code></p>
<ul>
<li>??? LCL - one broad mark H3K36ME3</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/p/p_pco_acat_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap_allchr_fdr_added.txt.gz</code></p>
<ul>
<li>??? LCL - narrow mark H3K27ac, cacti-p v.s. cacti-s</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/p/p_pco_acat_seg_group_H3K27AC_window50kb_cis500kb_seg5kb_peak_overlap_allchr_fdr_added.txt.gz</code></p>
<div id="compare-to-single-peak-1" class="section level3">
<h3>Compare to single peak</h3>
<ul>
<li>Macrophage - one broad mark H3K27me3, two conditions</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/plot/plt_cpeak_cwindow_seg_group_H3K27me3_(NI|Flu)_window50kb_seg5kb_cis100kb.pdf</code></p>
<ul>
<li>??? LCL - one broad mark H3K36ME3</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/plot/plt_cpeak_cwindow_seg_group_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf</code></p>
<ul>
<li>??? LCL - narrow mark H3K27ac, cacti-p v.s. cacti-s</li>
</ul>
<p><del><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/p/p_pco_acat_seg_group_H3K27AC_window50kb_cis500kb_seg5kb_(overlap_peak)_allchr_fdr_added.txt.gz</code></del></p>
<p>Rerun with <code>peak_overlap</code> QC mode -</p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/plot/plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg5kb_peak_overlap.pdf</code></p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/nonoverlap_window/plot/plt_cwindow_cactip_vs_cactis_H3K27AC_window50kb_cis500kb_seg5kb_peak_overlap.pdf</code></p>
</div>
</div>
<div
id="experimented-with-different-window-sizes-after-finalize-pipeline-for-both-narrow-and-broad-marks"
class="section level2">
<h2>Experimented with different window sizes (after finalize pipeline
for both narrow and broad marks)</h2>
<p><code>/project/xuanyao/llw/chromatin/aracena_et_al/results/(H3K27ac|H3K4me1|H3K4me3)_NI_p_comb_window(10kb|25kb|50kb)_cis100kb_allchr_fdr_added.txt.gz</code></p>
<p>Need to come up with a better way to visualize the results and
demonstrate the point.</p>
</div>
<div id="coloc-with-the-finalized-signals" class="section level2">
<h2>Coloc with the finalized signals</h2>
<ul>
<li><p>Use the finalized signals to do coloc (merged with uni
coloc)</p></li>
<li><p>Blood traits and immune diseases</p></li>
</ul>
<div id="lcl-multi-cqtls" class="section level3">
<h3>LCL multi cQTLs</h3>
<ul>
<li>blood traits</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/coloc_merge_uni_multi/results/coloc_{gwasPhenocode_36bloodtraits}_(H3K27AC|H3K4ME1|H3K4ME3)_window50kb_cis500kb_merge_uni_multi_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/coloc/results/coloc_{gwasPhenocode_36bloodtraits}_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap_allchr.txt.gz</code></p>
<ul>
<li>immune diseases</li>
</ul>
<p><code>coloc/results/coloc_{gwasPhenocode_immu}_(H3K27AC|H3K4ME1|H3K4ME3)_window50kb_cis500kb_merge_uni_multi_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/coloc/results/coloc_{gwasPhenocode_immu}_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap_allchr.txt.gz</code></p>
</div>
<div id="macrophage-multi-cqtls" class="section level3">
<h3>Macrophage multi cQTLs</h3>
<ul>
<li>blood traits</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/coloc_merge_uni_multi/results/coloc_{gwasPhenocode_36bloodtraits}_(H3K27ac|H3K4me1|H3K4me3)_(Flu|NI)_window50kb_cis100kb_merge_uni_multi_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/coloc/results/coloc_{gwasPhenocode_36bloodtraits}_aracena_H3K27me3_(Flu|NI)_window50kb_seg5kb_cis100kb_allchr.txt.gz</code></p>
<ul>
<li>immune diseases</li>
</ul>
<p><code>coloc/results/coloc_{gwasPhenocode_immu}_aracena_(H3K27ac|H3K4me1|H3K4me3)_(Flu|NI)_window50kb_cis100kb_merge_uni_multi_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/count_based4/coloc/results/coloc_{gwasPhenocode_immu}_aracena_H3K27me3_(Flu|NI)_window50kb_seg5kb_cis100kb_allchr.txt.gz</code></p>
</div>
<div id="uni-cqtls" class="section level3">
<h3>uni cQTLs</h3>
<ul>
<li>LCL</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/coloc_uni_cqtl/results/coloc_{gwasPhenocode_36bloodtraits}_{h_mark}_uni_cqtl_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/coloc_uni_cqtl/results/coloc_{gwasPhenocode_immu}_{h_mark}_uni_cqtl_allchr.txt.gz</code></p>
<ul>
<li>Macrophage</li>
</ul>
<p><code>/project/xuanyao/llw/chromatin/coloc_uni_cqtl/results/coloc_{gwasPhenocode_36bloodtraits}_aracena_{mark_aracena}_{condition_aracena}_uni_cqtl_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/coloc_uni_cqtl/results/coloc_{gwasPhenocode_immu}_aracena_{mark_aracena}_{condition_aracena}_uni_cqtl_allchr.txt.gz</code></p>
</div>
<div id="eqtlgen-genes" class="section level3">
<h3>eQTLGen genes</h3>
<p><code>/project/xuanyao/llw/chromatin/coloc_eqtlgen_gene/results/coloc_{gwasPhenocode_36bloodtraits}_eqtlgen_gene_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/coloc_eqtlgen_gene/results/coloc_{gwasPhenocode_immu}_eqtlgen_gene_allchr.txt.gz</code></p>
</div>
<div id="gtex-genes" class="section level3">
<h3>GTEx genes</h3>
<p><code>/project/xuanyao/llw/chromatin/coloc_gtex_gene/results/coloc_{gwasPhenocode_36bloodtraits}_gtex_gene_Cells_EBV-transformed_lymphocytes_allchr.txt.gz</code></p>
<p><code>/project/xuanyao/llw/chromatin/coloc_gtex_gene/results/coloc_{gwasPhenocode_immu}_gtex_gene_Cells_EBV-transformed_lymphocytes_allchr.txt.gz</code></p>
</div>
</div>
</div>
<div id="jan-16" class="section level1">
<h1>Jan 16</h1>
<div id="downsample-size" class="section level2">
<h2>Downsample size</h2>
<div id="downsample-to-35" class="section level3">
<h3>Downsample to 35</h3>
<p>For one mark H3K27ac in LCL, downsized to 35 -</p>
<ul>
<li><p>QTLTools under downsample size</p></li>
<li><p>Multivaraite using sum stats under downsample</p></li>
<li><p>Coloc with uni and multi under downsample</p></li>
<li><p>Visualize the comparison</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_16_coloc_num_uni_multi_single_dataset.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs of H3K27ac in LCL under downsample size, 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with uni and multi cQTLs of H3K27ac in LCL under
downsample size, 35.
</p>
</div>
<p><strong>Observations</strong></p>
<ul>
<li><p>Most traits has only 1 loci coloc with uni cQTLs under downsample
size.</p></li>
<li><p>All traits has no loci coloc with multi cQTLs under downsample
size.</p></li>
<li><p>For loci coloc with uni cQTLs, there are 17 (uni peak, trait)
pairs, corresponding to 8 unique uni peaks, which are included in peak
groups of size 3 to 10, mostly 5.</p></li>
</ul>
</div>
<div id="downsample-to-60" class="section level3">
<h3>Downsample to 60</h3>
<p>I then increase the sample size from 35 to 60 -</p>
<p>Results in folder <code>downsample_lcl_60</code></p>
</div>
</div>
</div>
<div id="compare-signal-number-of-multi_n35-uni_n35-and-uni_n72"
class="section level1">
<h1>Compare signal number of multi_N35, uni_N35, and uni_N72</h1>
<ul>
<li><p>multi_N35 vs uni_N35</p></li>
<li><p>multi_N35 vs uni_N72</p></li>
<li><p>multi_N60 vs uni_N60</p></li>
<li><p>multi_N60 vs uni_N72</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_plt_cpeak_cwindow_H3K27AC_window50kb_multi_N35.pdf" title="Figure: Comparison between uni and multi cQTLs under downsample size 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs under downsample size 35.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_plt_cpeak_cwindow_H3K27AC_window50kb_multi_N35_uni_N72.pdf" title="Figure: Comparison between uni and multi cQTLs. Uni under sample size 72, multi under downsample 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs. Uni under sample size
72, multi under downsample 35.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_plt_cpeak_cwindow_H3K27AC_window50kb_multi_N60.pdf" title="Figure: Comparison between uni and multi cQTLs under downsample size 60." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs under downsample size 60.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_plt_cpeak_cwindow_H3K27AC_window50kb_multi_N60_uni_N72.pdf" title="Figure: Comparison between uni and multi cQTLs. Uni under sample size 72, multi under downsample 60." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs. Uni under sample size
72, multi under downsample 60.
</p>
</div>
<div id="downsample-to-40" class="section level3">
<h3>Downsample to 40</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_plt_cpeak_cwindow_H3K27AC_window50kb_downsample_40.pdf" title="Figure: Comparison between uni and multi cQTLs under downsample size 40." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs under downsample size 40.
</p>
</div>
</div>
<div id="downsample-to-50" class="section level3">
<h3>Downsample to 50</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_plt_cpeak_cwindow_H3K27AC_window50kb_downsample_50.pdf" title="Figure: Comparison between uni and multi cQTLs under downsample size 50." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs under downsample size 50.
</p>
</div>
</div>
<div id="use-original-sample-size-72-as-sanity-check"
class="section level3">
<h3>Use original sample size 72 as sanity check</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_plt_cpeak_cwindow_H3K27AC_window50kb_downsample_72.pdf" title="Figure: Comparison between uni and multi cQTLs under downsample size 72." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs under downsample size 72.
</p>
</div>
<p>Compare to the original signal numbers -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_plt_cpeak_cwindow_H3K27AC_window50kb_cis500kb_allchr.pdf" title="Figure: Comparison between uni and multi cQTLs in orignal results." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Comparison between uni and multi cQTLs in orignal results.
</p>
</div>
<p>Something wrong with window cQTLs …</p>
</div>
</div>
<div id="add-pvalue-cutoff-on-qtl-coloc-regions" class="section level1">
<h1>Add pvalue cutoff on QTL coloc regions</h1>
<div id="how-does-it-change-the-coloc-results" class="section level2">
<h2>How does it change the coloc results?</h2>
<p>I added significance threshold to cQTLs, which means only cQTLs of
cpeaks and cwindows are considered for coloc.</p>
<p>In coloc regions, are most uni or multi QTLs signals or not?</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_coloc_num_multi_new_any_dataset.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_any_dataset.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_single_dataset.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_single_dataset_distribution.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with uni and multi cQTLs when adding
significance threshold to cQTLs.
</p>
</div>
<p><a href="asset/2024/09_12_coloc_num_uni_multi_single_dataset.pdf">Old
version before applying threshold on cQTLs</a>.</p>
<p><a
href="asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution_new.pdf">Old
version before applying threshold on cQTLs</a></p>
<p><a
href="asset/2024/09_12_coloc_num_uni_multi_any_dataset_new.pdf">Old
version before applying threshold on cQTLs</a></p>
</div>
<div id="coloc-results-with-not-eqtls" class="section level2">
<h2>Coloc results with not eQTLs</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_coloc_num_cqtl_not_eqtl_any_dataset.pdf" title="Figure: GWAS loci coloc with eQTLs and cQTLs when adding significance threshold to cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with eQTLs and cQTLs when adding significance
threshold to cQTLs.
</p>
</div>
<p><a
href="asset/2024/09_12_coloc_num_eqtl_cqtl_any_dataset_eqtlgen.pdf">Old
version before applying threshold on cQTLs</a></p>
</div>
<div id="further-change-pp4-cutoff-from-0.5-to-0.75"
class="section level2">
<h2>Further change PP4 cutoff from 0.5 to 0.75</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_30_coloc_num_multi_new_any_dataset_pp4_75.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs and increase PP4 cutoff to 0.75." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_any_dataset_pp4_75.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs and increase PP4 cutoff to 0.75." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_single_dataset_pp4_75.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs and increase PP4 cutoff to 0.75." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/01_30_coloc_num_uni_multi_single_dataset_distribution_pp4_75.pdf" title="Figure: GWAS loci coloc with uni and multi cQTLs when adding significance threshold to cQTLs and increase PP4 cutoff to 0.75." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with uni and multi cQTLs when adding
significance threshold to cQTLs and increase PP4 cutoff to 0.75.
</p>
</div>
</div>
</div>
<div id="data-quality-of-lcl-and-macrophage" class="section level1">
<h1>Data quality of LCL and Macrophage</h1>
<p>LCL - 105043 peaks</p>
<p>Macrophage - 45233 peaks</p>
<ul>
<li><p>reads</p></li>
<li><p>peak length</p></li>
<li><p>Average reads on peak</p></li>
<li><p>38 cpeaks under downsample size 35</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_peak_reads.pdf" title="Figure: Number of reads aligned to peaks of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_07_peak_reads_cpeaks_35.pdf" title="Figure: Number of reads aligned to peaks of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of reads aligned to peaks of H3K27ac in dataset LCL and
Macrophage. All peaks and cpeaks under downsampled size 35.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_peak_length.pdf" title="Figure: Peak length of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_07_peak_length_cpeaks_35.pdf" title="Figure: Peak length of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Peak length of H3K27ac in dataset LCL and Macrophage. All peaks
and cpeaks under downsampled size 35.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_07_reads_ave.pdf" title="Figure: Average number of reads aligned to single base of peaks of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_07_reads_ave_cpeaks_35.pdf" title="Figure: Average number of reads aligned to single base of peaks of H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under downsampled size 35." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Average number of reads aligned to single base of peaks of
H3K27ac in dataset LCL and Macrophage. All peaks and cpeaks under
downsampled size 35.
</p>
</div>
<div id="why" class="section level2">
<h2>Why?</h2>
<p>hope to better understand what factors determine the power of cQTL
mapping. Sample size is one, but for data of the sample size, the
performance also differs a lot.</p>
</div>
<div id="library-size-sequencing-depth-read-depth"
class="section level2">
<h2>Library size / sequencing depth / read depth</h2>
<p>Sequencing depth, also known as read depth or depth of coverage,
refers to the number of times a specific base (nucleotide) in the DNA is
read during the sequencing process. In other words, it’s the average
number of times a given position in the genome is sequenced. A higher
sequencing depth provides more confidence in the accuracy of the base
calls at that position and helps to reduce sequencing errors and
noise.</p>
<p>For example, if a specific nucleotide is sequenced 30 times, the
sequencing depth at that position is 30x.</p>
<ul>
<li><p>Definition: Sequencing depth refers to the number of times a
particular nucleotide is read during the sequencing process.</p></li>
<li><p>Purpose: Higher sequencing depth increases the confidence in
calling a variant at a specific location. It is particularly important
when looking for rare variants or when sequencing heterogeneous samples
(like tumor tissues).</p></li>
<li><p>Measurement: Expressed as an average, e.g., “100x depth.” This
means that, on average, each nucleotide in the sequenced region was read
100 times.</p></li>
<li><p>library size</p></li>
<li><p>The total number of reads mapped to all peaks for each data</p>
<ul>
<li><p>Total number of reads</p></li>
<li><p>Total number of peaks</p></li>
<li><p>Average depth</p></li>
</ul></li>
</ul>
</div>
<div id="read-depth-of-cpeaks-total-reads-and-peak-length"
class="section level2">
<h2>Read depth of cpeaks (total reads and peak length)</h2>
<p>Peak properties of the few hundred cpeaks at sample size 35 in LCL
-</p>
<p>Why they are still cpeaks under downsampled size? Their pvalues?
Their peak length? Total mapped reads? Read depth? Are they on the
higher end?</p>
<p>Does it matter given normalization? If any?</p>
</div>
<div id="something-wrong-with-normalization-under-downsampled-size"
class="section level2">
<h2>Something wrong with normalization under downsampled size?</h2>
</div>
</div>
<div id="coloc" class="section level1">
<h1>Coloc</h1>
<div id="add-other-traits-for-coloc" class="section level2">
<h2>Add other traits for coloc</h2>
<p>Coloc cQTLs with 11 immune diseases -</p>
<ul>
<li>cQTLs of three marks in macrophage</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_coloc_num_immu_aracena_H3K27ac_NI_window50kb_cis100kb.pdf" title="Figure: Coloc between cQTLs of three marks in macrophage and 11 immune diseases." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_13_coloc_num_immu_aracena_H3K4me1_NI_window50kb_cis100kb.pdf" title="Figure: Coloc between cQTLs of three marks in macrophage and 11 immune diseases." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_13_coloc_num_immu_aracena_H3K4me3_NI_window50kb_cis100kb.pdf" title="Figure: Coloc between cQTLs of three marks in macrophage and 11 immune diseases." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc between cQTLs of three marks in macrophage and 11 immune
diseases.
</p>
</div>
</div>
<div id="new-coloc-loci" class="section level2">
<h2>New coloc loci</h2>
<ul>
<li>Coloc immune gwas loci with uni cQTLs of three marks</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_coloc_num_immu_uni_multi_aracena_H3K27ac_NI_window50kb_cis100kb.pdf" title="Figure: Loci coloc with uni cQTLs, multi cQTLs, or both." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_13_coloc_num_immu_uni_multi_aracena_H3K4me1_NI_window50kb_cis100kb.pdf" title="Figure: Loci coloc with uni cQTLs, multi cQTLs, or both." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/02_13_coloc_num_immu_uni_multi_aracena_H3K4me3_NI_window50kb_cis100kb.pdf" title="Figure: Loci coloc with uni cQTLs, multi cQTLs, or both." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with uni cQTLs, multi cQTLs, or both.
</p>
</div>
<div id="example" class="section level3">
<h3>Example</h3>
</div>
</div>
</div>
<div id="change-examples-to-macrophage-signals" class="section level1">
<h1>Change examples to Macrophage signals</h1>
<div id="blood-traits" class="section level2">
<h2>Blood traits</h2>
<div id="new-cqtl-by-cacti-p" class="section level3">
<h3>New cQTL by cacti-p</h3>
</div>
<div id="new-cqtl-by-cacti-s" class="section level3">
<h3>New cQTL by cacti-s</h3>
<ul>
<li><p>with no pre-defined peaks</p></li>
<li><p>with pre-defined peaks</p></li>
</ul>
</div>
<div id="coloc-example-with-new-cqtls" class="section level3">
<h3>Coloc example with new cQTLs</h3>
<p><strong>Example 1</strong></p>
<p>Loci coloc with new cQTL and eQTL of relevant gene.
<code>c('hgb_N172925', '3:58456811', 'G20859', 'PDHB')</code></p>
<ul>
<li><p>Trait: HGB, hgb, Hemoglobin concentration</p></li>
<li><p>Gene: PDHB – Affects cellular metabolism and has implications for
red blood cell function.</p></li>
<li><p>See full figure of coloc between all nearby genes <a
href="asset/2024/02_13_coloc_new_cqtl_hgb_N172925_reg_3_58456811_gene_PDHB.pdf">here</a>.</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2024/02_13_trunc_coloc_new_cqtl_hgb_N172925_reg_3_58456811_gene_PDHB.png" alt="Figure: Loci coloc with new cQTL. The trait is HGB, hgb, Hemoglobin concentration. The relevant nearby gene of cQTLs is PDHB. But there is no coloc in eqTLGen or GTEx." width="50%" height="30%" />
<p class="caption">
Figure: Loci coloc with new cQTL. The trait is HGB, hgb, Hemoglobin
concentration. The relevant nearby gene of cQTLs is PDHB. But there is
no coloc in eqTLGen or GTEx.
</p>
</div>
<p><strong>Example 2, related to monocyte traits</strong></p>
<p>Loci coloc with new cQTL and eQTL of relevant gene.
<code>c(Trait: MONO_P, Region: 17:16176883, Macrophage, mark: H3K27ac_NI, peak: G7756, near gene: NCOR1)</code></p>
<ul>
<li><p>Trait: MONO_P, mono_p, Monocyte percentage of white
cells</p></li>
<li><p>Gene: NCOR1 - NCOR1 is a critical regulator of macrophage
inflammation and proliferation and that deficiency of NCOR1 in
macrophages attenuates myocardial infarction and neointimal
hyperplasia.</p>
<p>Not nearest gene, but close gene, distance 238,224bp.</p></li>
<li><p>Coloc figure</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_trunc_coloc_new_cqtl_mono_p_N170494_reg_17_16176883_gene_NCOR1.pdf" title="Figure: Loci coloc with new cQTL. The trait is MONO_P, mono_p, Monocyte percentage of white cells. The relevant nearby gene of cQTLs is NCOR1. There is also coloc with eQTLs in eQTLGen but not in GTEx." width="50%" height="30%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with new cQTL. The trait is MONO_P, mono_p, Monocyte
percentage of white cells. The relevant nearby gene of cQTLs is NCOR1.
There is also coloc with eQTLs in eQTLGen but not in GTEx.
</p>
</div>
</div>
<div id="coloc-example-with-cqtls-andor-eqtls" class="section level3">
<h3>Coloc example with cQTLs and/or eQTLs</h3>
<p><strong>Example 1</strong></p>
<p>Loci coloc with cQTL not eQTL of relevant gene.
<code>c('eo_p_gran_N170536', '11:88090210', 'RAB38')</code></p>
<ul>
<li><p>Trait: EO%GRAN, eo_p_gran, Eosinophil percentage of
granulocytes</p></li>
<li><p>Gene: RAB38 – Implicated in platelet function and granule
secretion.</p></li>
<li><p>See full figure of coloc between all nearby genes <a
href="asset/2024/02_13_coloc_cqtl_not_eqtl_eo_p_gran_N170536_reg_11_88090210_gene_RAB38.pdf">here</a>.</p></li>
</ul>
<div class="figure" style="text-align: center">
<img src="asset/2024/02_13_trunc_coloc_cqtl_not_eqtl_eo_p_gran_N170536_reg_11_88090210_gene_RAB38.png" alt="Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is eo_p_gran, Eosinophil percentage of granulocytes. The relevant nearby gene of cQTLs is RAB38. There is no coloc between this gene and the locus either in eQTLGen or GTEx." width="50%" height="30%" />
<p class="caption">
Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is
eo_p_gran, Eosinophil percentage of granulocytes. The relevant nearby
gene of cQTLs is RAB38. There is no coloc between this gene and the
locus either in eQTLGen or GTEx.
</p>
</div>
<p><strong>Example 2, related to monocyte traits</strong></p>
<p>Loci coloc with cQTL not eQTL of relevant gene.
<code>c('mono_N170721', '4:185251371', 'IRF2')</code></p>
<ul>
<li><p>Trait: mono, Monocyte count</p></li>
<li><p>Gene: IRF2 – IRF2 is an important regulator of the
proinflammatory response in macrophages …</p></li>
<li><p>Coloc figure</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_trunc_coloc_cqtl_not_eqtl_mono_N170721_reg_4_185251371_gene_IRF2.pdf" title="Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is mono, Monocyte count. The relevant nearby gene of cQTLs is IRF2. There is no coloc between this gene and the locus either in eQTLGen or GTEx." width="50%" height="30%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is
mono, Monocyte count. The relevant nearby gene of cQTLs is IRF2. There
is no coloc between this gene and the locus either in eQTLGen or GTEx.
</p>
</div>
</div>
</div>
<div id="immune-diseases" class="section level2">
<h2>Immune diseases</h2>
<div id="coloc-example-with-new-cqtls-1" class="section level3">
<h3>Coloc example with new cQTLs</h3>
<p><strong>Example 1</strong></p>
<p>Loci coloc with new cQTL and eQTL of relevant gene.
<code>c(Trait: MS Multiple sclerosis, Region: 17:45702280, Macrophage, mark: H3K4me3_NI, peak: G6479, near gene: TBKBP1)</code></p>
<ul>
<li><p>Trait: pmid31604244_MS, Multiple sclerosis</p></li>
<li><p>Gene: TBKBP1 - Adapter protein which constitutively binds TBK1
and IKBKE playing a role in antiviral innate immunity.</p></li>
<li><p>Coloc figure</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_trunc_coloc_cqtl_not_eqtl_pmid31604244_MS_reg_17_45702280_gene_TBKBP1.pdf" title="Figure: Loci coloc with new cQTL. The trait is pmid31604244_MS. The relevant nearby gene of cQTLs is TBKBP1. There is also coloc with eQTLs in eQTLGen." width="50%" height="30%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with new cQTL. The trait is pmid31604244_MS. The
relevant nearby gene of cQTLs is TBKBP1. There is also coloc with eQTLs
in eQTLGen.
</p>
</div>
<p><strong>Example 2 - New coloc cQTL near a relevant gene with no
coloc</strong></p>
<p>Loci coloc with new cQTL and eQTL of relevant gene.
<code>c(Trait: pmid29892013_ALLERGY_ECZEMA, Region: 2:204585956, Macrophage, mark: H3K27ac_NI, peak: G11035, near gene: CD28)</code></p>
<ul>
<li><p>Trait: pmid29892013_ALLERGY_ECZEMA</p></li>
<li><p>Gene: CD28 - T-cell activation is a critical driver of immune
responses. The CD28 costimulation is an essential regulator of CD4
T-cell responses …</p></li>
<li><p>Coloc figure</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_trunc_coloc_cqtl_not_eqtl_pmid29892013_ALLERGY_ECZEMA_reg_2_204585956_gene_CD28.pdf" title="Figure: Loci coloc with new cQTL. The trait is pmid29892013_ALLERGY_ECZEMA. The relevant nearby gene of cQTLs is CD28. There is no coloc with eQTLs in eQTLGen." width="50%" height="30%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with new cQTL. The trait is
pmid29892013_ALLERGY_ECZEMA. The relevant nearby gene of cQTLs is CD28.
There is no coloc with eQTLs in eQTLGen.
</p>
</div>
</div>
<div id="coloc-example-with-cqtls-andor-eqtls-1" class="section level3">
<h3>Coloc example with cQTLs and/or eQTLs</h3>
<p>Loci coloc with cQTL not eQTL of relevant gene.
<code>c(Trait: MS Multiple sclerosis, Region: 6:137959455, Macrophage, mark: H3K4me1_NI, peak: G27458, near gene: TNFAIP3)</code></p>
<ul>
<li><p>Trait: pmid31604244_MS, Multiple sclerosis</p></li>
<li><p>Gene: TNFAIP3 – The protein encoded by this gene is a zinc finger
protein and ubiqitin-editing enzyme, and has been shown to inhibit
NF-kappa B activation as well as TNF-mediated apoptosis.</p></li>
<li><p>Coloc figure</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_trunc_coloc_cqtl_not_eqtl_pmid31604244_MS_reg_6_137959455_gene_TNFAIP3.pdf" title="Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is pmid31604244_MS, Multiple sclerosis. The relevant nearby gene of cQTLs is TNFAIP3. There is no coloc between this gene and the locus either in eQTLGen or GTEx." width="50%" height="30%" type="application/pdf" />
<p class="caption">
Figure: Loci coloc with cQTL not eQTL of relevant gene. The trait is
pmid31604244_MS, Multiple sclerosis. The relevant nearby gene of cQTLs
is TNFAIP3. There is no coloc between this gene and the locus either in
eQTLGen or GTEx.
</p>
</div>
</div>
</div>
</div>
<div id="check-the-importance-of-genes-near-windows-coloc-with-traits"
class="section level1">
<h1>Check the importance of genes near windows coloc with traits</h1>
<p>Goal - we wanted to look into those loci only coloc with multi QTLs
but not uni QTLs, i.e. new coloc loci. We think the cwindows
corresponding to those new coloc loci are weak signals, and wanted to
see if they have important function by checking if their nearest genes
are important in terms of pLI score.</p>
<p>Some numbers -</p>
<ul>
<li><p>Overall coloc pairs between loci and multi cQTLs: 6153</p></li>
<li><p>Strong coloc signals, coloc with multi and uni: 5132</p></li>
<li><p>Weak coloc signals, coloc with multi but not uni: 1021</p></li>
</ul>
<p>I define the cwindows to be a new coloc signal if all the loci they
coloc with do not coloc with any uni cQTLs. I then checked their nearest
genes are different from those of the other cwindows. I also categorized
the cwindows into multiple categories based on the number loci across
traits that they coloc with but not coloc with any uni cQTLs.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_pli_new_coloc_window.pdf" title="Figure: Blue box shows all cwindows coloc with loci that do not coloc with any cQTLs, i.e. cwindows of new coloc signals. Yellow box shows all the other cwindows coloc with loci that also coloc with at least one uni cQLTs. Numbers give the number of the cwindows in each category. Y-axis shows pLI score of nearest genes of cwindows in two categories. Comparision pvalue is on the top. The genes near new cwindows are more tolerant to loss-of-function variants, although the difference is not statistically different. Most of the gene score are not higher than 0.9." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Blue box shows all cwindows coloc with loci that do not coloc
with any cQTLs, i.e. cwindows of new coloc signals. Yellow box shows all
the other cwindows coloc with loci that also coloc with at least one uni
cQLTs. Numbers give the number of the cwindows in each category. Y-axis
shows pLI score of nearest genes of cwindows in two categories.
Comparision pvalue is on the top. The genes near new cwindows are more
tolerant to loss-of-function variants, although the difference is not
statistically different. Most of the gene score are not higher than 0.9.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_pli_new_coloc_window_sorted_by_coloc_num.pdf" title="Figure: All cwindows are categoried into groups based on how many new loci they coloc with but are not coloc with uni cQTLs. This metric is associated with the level of the windows being new coloc signal. Y-axis shows pLI score of nearest genes of cwindows in each category." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: All cwindows are categoried into groups based on how many new
loci they coloc with but are not coloc with uni cQTLs. This metric is
associated with the level of the windows being new coloc signal. Y-axis
shows pLI score of nearest genes of cwindows in each category.
</p>
</div>
<div id="updates" class="section level2">
<h2>Updates</h2>
<ul>
<li>The numbers are windows, not nearest genes. The number can contain
duplicated genes.</li>
</ul>
<p><strong>Number of genes with pLI larger than 0.9 for different gene
groups</strong></p>
<ul>
<li>Enrichment of genes with large pli in nearest genes</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_27_pli_new_coloc_window_large_pli_09_enrich.pdf" title="Figure: Enrichment of genes with large pli (&gt;0.9) in nearest genes of new coloc cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Enrichment of genes with large pli (&gt;0.9) in nearest genes of
new coloc cQTLs.
</p>
</div>
<ul>
<li>Proportion of genes with large pli in nearest genes of coloc
cQTLs</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_27_pli_new_coloc_window_large_pli_09_sorted_by_coloc_num.pdf" title="Figure: Proportion of genes with large pli in nearest genes of coloc cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Proportion of genes with large pli in nearest genes of coloc
cQTLs.
</p>
</div>
</div>
<div id="check-eds-score" class="section level2">
<h2>Check EDS score</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_EDS_new_coloc_window.pdf" title="Figure: EDS score of nearest genes of new coloc cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: EDS score of nearest genes of new coloc cQTLs.
</p>
</div>
<p>EDS score of genes near windows with new coloc signals have higher
score, which means they are more resilient to perturbations, indicating
their importance.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_13_EDS_new_coloc_window_sorted_by_coloc_num.pdf" title="Figure: EDS score of nearest genes of window coloc with different number of new coloc cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: EDS score of nearest genes of window coloc with different number
of new coloc cQTLs.
</p>
</div>
</div>
</div>
<div id="sep-12" class="section level1">
<h1>Sep 12</h1>
<div id="insert-numbers-to-current-draft" class="section level2">
<h2>0. Insert numbers to current draft</h2>
</div>
<div id="cacti-s-on-mark-h3k27ac-lcl-and-macrophage"
class="section level2">
<h2>1. CACTI-s on mark H3K27ac (LCL and macrophage)</h2>
<p>Goal -</p>
<p>To show cacti-s is slightly worse than cacti-p but still better than
univariates, which indicates a balance of skipping peak calling and
reduced power.</p>
<p>[Not include this part in the paper. Put it in package README. Add
observations to README. Add this part in thesis.]</p>
<div id="how" class="section level3">
<h3>How?</h3>
<ul>
<li><p>segment size option</p>
<p>‘1kb’, ‘2kb’, ‘5kb’, ‘10kb’</p></li>
<li><p>segment filter - Exclude segments not overlap with peaks</p>
<p>Reason - to keep comparison fair</p></li>
</ul>
</div>
<div id="compare-h3k27ac-cacti-s-and-h3k27ac-cacti-p"
class="section level3">
<h3>Compare H3K27ac-cacti-s and H3K27ac-cacti-p</h3>
<ul>
<li><p>Signal number</p></li>
<li><p>Signal overlap</p></li>
<li><p>Compare H3K27ac-cacti-s to H3K27ac-fastQTL</p></li>
</ul>
<p><strong>Observations &amp; Conclusions -</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb.pdf" title="Figure: CACTI-P on H3K27AC in LCL v.s. uni. window50kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-P on H3K27AC in LCL v.s. uni. window50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb">
Past versions of
09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg5kb_overlap_peak.pdf" title="Figure:  CACTI-S on H3K27AC in LCL v.s. uni. window50kb. seg10kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-S on H3K27AC in LCL v.s. uni. window50kb. seg10kb.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cwindow_cactip_vs_cactis_H3K27AC_window50kb_cis500kb_seg5kb_overlap_peak.pdf" title="Figure:  CACTI-P on H3K27AC in LCL v.s. CACTI-S. window50kb. seg10kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-P on H3K27AC in LCL v.s. CACTI-S. window50kb. seg10kb.
</p>
</div>
</div>
<div id="results-using-other-segment-options" class="section level3">
<h3>Results using other segment options</h3>
<p>Done.</p>
</div>
<div id="results-using-the-same-segment-filtering-as-in-broad-mark"
class="section level3">
<h3>Results using the same segment filtering as in broad mark</h3>
<p>Done.</p>
</div>
<div id="try-another-filtering-based-on-cpm-check-bens-code"
class="section level3">
<h3>try another filtering based on cpm (check Ben’s code)</h3>
</div>
</div>
<div id="coloc-analysis-on-cqtls-of-mark-h3k27ac-by-cacti-s"
class="section level2">
<h2>2. coloc analysis on cQTLs of mark H3K27ac by CACTI-s</h2>
<p>Done.</p>
</div>
<div id="proportion-of-cwindow-sharing-among-marks-details-see-on-slack"
class="section level2">
<h2>Proportion of cwindow sharing among marks (details see on
slack)</h2>
<blockquote>
<p>Figure 4G of your manuscript, I think enrichment values are less
intuitive. Do you have proportion of sharing instead?</p>
</blockquote>
<p>Done.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_enrich_cwindow_across_mark_lcl.pdf" title="Figure:" width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_enrich_cwindow_across_mark_lcl">
Past versions of 09_12_enrich_cwindow_across_mark_lcl.pdf
</button>
</p>
<div id="fig-09_12_enrich_cwindow_across_mark_lcl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_enrich_cwindow_across_mark_lcl.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="number-of-cwindows-and-overlap-with-cpeaks-figure"
class="section level2">
<h2>Number of cwindows (and overlap with cpeaks) figure</h2>
<p>Done.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_cwindow_across_datasets.pdf" title="Figure:" width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure:
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_cwindow_across_datasets">
Past versions of 09_12_cwindow_across_datasets.pdf
</button>
</p>
<div id="fig-09_12_cwindow_across_datasets" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_cwindow_across_datasets.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cacti-s-on-broad-mark-h3k27me3-macrophage"
class="section level2">
<h2>3. CACTI-s on broad mark H3K27me3 (macrophage)</h2>
<p>Done.</p>
<p>To make cacti-s ’s data processing consistent with Aracena et al.,
see their analysis path <a
href="asset/2024/09_12_Aracena_analysis_path.png">here</a>.</p>
<ul>
<li><del>Get chrome size for hg19 for H3K27me3 in macrophage</del></li>
</ul>
<div id="sample-filtering" class="section level3">
<h3>sample filtering</h3>
<blockquote>
<p>3 samples labels will differ between the bams and sample labels in
the paper and all paper inputs - AF22, AF36 and AF38. Despite having the
AF prefix these samples are nearly 100% European ancestry. We changed
these sample labels to EU prefixes (EU22, EU36 and EU38) in the
manuscript but the bams have these “original” names. I would recommend
changing any output files you create to EU22, EU36 and EU38 if you’re
going to publish this work eventually just to keep things concordant</p>
</blockquote>
<p><del>change sample labels in snmk pipeline, as the updated sample
labels will be used in meta and regress covariates</del></p>
</div>
<div id="peakseg-filtering" class="section level3">
<h3>peak/seg filtering</h3>
<p>To be consistent with the peak filtering in Aracena et al., I keep
only the segments that are present in at least 50% of Flu or NI samples
(&gt;=14).</p>
</div>
<div id="geno-filtering" class="section level3">
<h3>geno filtering</h3>
<blockquote>
<p>WGS fastq files were processed using GenPipes’ (v3.1.4) DNA
sequencing pipeline62 Briefly, raw reads were trimmed using Skewer
(v0.2.2)63 and the resulting reads were aligned to the hg19 human
reference genome using BWA-MEM (v0.7.17)64. Insertion/deletion
realignment and base quality score recalibration were performed using
GATK (v3.8)65, and duplicates were marked using Picard (v2.23.3)66. We
used GATK’s HaplotypeCaller in gvcf mode to perform SNV and insertion
and deletions calling. We filtered the joint genotyped file to exclude
nonautosomal and nonbiallelic variants. Additionally, we used PLINK
(v1.90b6.9)67,68 to remove SNPs that had a call rate of &lt;90% across
all samples, that deviated from Hardy–Weinberg equilibrium at
P&lt;10e−5, and with minor allele frequency less than 5%. We used the
resulting 7,383,243 SNPs in QTL mapping and other downstream analyses.
We annotated the SNPs using dbSNP (human_9606_b151_GRCH37p13)69.</p>
</blockquote>
</div>
<div id="covariates" class="section level3">
<h3>covariates</h3>
<ul>
<li><p>Non genetic factors, e.g. batch, age, et al</p></li>
<li><p>Expression PCs</p>
<p>I used same number PCs used for peaks in Aracena et al, where they
tried different number of PCs.</p>
<p>For H3K27me3, 1 and 2 PCs were used for condition NI and
Flu.</p></li>
<li><p>Genotype first PC</p>
<p>PC1 of genotype data is the only covariate included (to account for
population structure)</p></li>
</ul>
</div>
<div id="compare-h3k27me3-cacti-s-h3k27me3-fastqtl-and-h3k27me3-cacti-p"
class="section level3">
<h3>Compare H3K27me3-cacti-s, H3K27me3-fastQTL, and
H3K27me3-cacti-p</h3>
<ul>
<li><p>Signal number</p></li>
<li><p>Signal overlap</p></li>
</ul>
<p><strong>Observations &amp; Conclusions -</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb.pdf" title="Figure: CACTI-P on H3K27me3 v.s. uni. window50kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-P on H3K27me3 v.s. uni. window50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb">
Past versions of
09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" title="Figure:  CACTI-S on H3K27me3 v.s. uni. window50kb. seg5kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-S on H3K27me3 v.s. uni. window50kb. seg5kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb">
Past versions of
09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf
</button>
</p>
<div
id="fig-09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" title="Figure:  CACTI-P on H3K27me3 v.s. CACTI-S. window50kb. seg5kb." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: CACTI-P on H3K27me3 v.s. CACTI-S. window50kb. seg5kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb">
Past versions of
09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf
</button>
</p>
<div
id="fig-09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="other-options-of-cacti-s-on-broad-mark-h3k27me3-macrophage"
class="section level2">
<h2>other options of CACTI-s on broad mark H3K27me3 (macrophage)</h2>
</div>
<div
id="coloc-analysis-on-cqtls-of-broad-marks-h3k36me3-lcl-and-h3k27me3-macrophage-by-cacti-s"
class="section level2">
<h2>4. coloc analysis on cQTLs of broad marks H3K36me3 (LCL) and
H3K27me3 (macrophage) by CACTI-s</h2>
<p>Done.</p>
<ul>
<li>Figures H3K36me3 (LCL)</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" title="Figure: Coloc with H3K36me3 in LCL by cacti-p." width="50%" height="100%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap">
Past versions of
09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" title="Figure: Coloc with H3K36me3 in LCL by cacti-p." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc with H3K36me3 in LCL by cacti-p.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap">
Past versions of
09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf
</button>
</p>
<div
id="fig-09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Figures H3K27me3 (macrophage)</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" title="Figure: Coloc with H3K27me3 in Macrophage by cacti-p." width="50%" height="100%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb">
Past versions of
09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf
</button>
</p>
<div
id="fig-09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" title="Figure: Coloc with H3K27me3 in Macrophage by cacti-p." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc with H3K27me3 in Macrophage by cacti-p.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb">
Past versions of
09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf
</button>
</p>
<div
id="fig-09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5bd4413b03f7f7ff36be5e3b9fc758e3a865127c/docs/asset/2024/09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf" target="_blank">5bd4413</a>
</td>
<td>
liliw-w
</td>
<td>
2024-10-30
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="update-coloc-any" class="section level2">
<h2>Update coloc any</h2>
<p>Done.</p>
<ul>
<li><p>Any</p></li>
<li><p>Any of narrow</p></li>
<li><p>Any of broad</p></li>
<li><p>Any of cell type</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_any_dataset_celltype_cactips.pdf" title="Figure: Coloc with any cQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc with any cQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_any_dataset_celltype_cactips">
Past versions of 09_12_coloc_any_dataset_celltype_cactips.pdf
</button>
</p>
<div id="fig-09_12_coloc_any_dataset_celltype_cactips" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_any_dataset_celltype_cactips.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2024/09_12_coloc_any_dataset_cacti_ps.pdf">Old version
before harmonizing ref genome</a>.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_any_dataset_mark.pdf" title="Figure: Coloc with any cQTLs of each mark." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc with any cQTLs of each mark.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_any_dataset_mark">
Past versions of 09_12_coloc_any_dataset_mark.pdf
</button>
</p>
<div id="fig-09_12_coloc_any_dataset_mark" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_any_dataset_mark.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="add-other-traits-to-coloc-analysis" class="section level2">
<h2>5. Add other traits to coloc analysis</h2>
<ul>
<li><p>coloc brain cQTLs of mark H3K27ac in GTEx with
schizophrenia</p></li>
<li><p>coloc with immune related diseases</p></li>
</ul>
</div>
<div id="coloc-with-new-cqtls-in-def-of-cqtls-of-single-peaks-not-coloc"
class="section level2">
<h2>7. Coloc with new cQTLs in def of cQTLs of single peaks not
coloc</h2>
<ul>
<li><p>LCL</p>
<ul>
<li><p>mark set 1</p></li>
<li><p>mark set 2: H3K36ME</p></li>
</ul></li>
<li><p>Macrophage</p>
<ul>
<li><p>mark set 1</p></li>
<li><p>mark set 2: H3K27me3</p></li>
</ul></li>
<li><p>coloc number with uni cqtls</p>
<ul>
<li><p>Single dataset</p></li>
<li><p>All datasets</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_uni_cqtl_new.pdf" title="Figure: Coloc loci with uni cQTLs. Upper panel - coloc with single dataset. Lower panel - coloc with cQTLs of any dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc loci with uni cQTLs. Upper panel - coloc with single
dataset. Lower panel - coloc with cQTLs of any dataset.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_uni_cqtl_new">
Past versions of 09_12_coloc_num_uni_cqtl_new.pdf
</button>
</p>
<div id="fig-09_12_coloc_num_uni_cqtl_new" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_num_uni_cqtl_new.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2024/09_12_coloc_num_uni_cqtl.pdf">Old version before
harmonizing ref genome</a>.</p>
<ul>
<li><p>coloc number with multi cqtls</p>
<ul>
<li><p>Single dataset</p></li>
<li><p>All dataset</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_multi_cqtl_new.pdf" title="Figure: Coloc loci with multi cQTLs. Upper panel - coloc with single dataset. Lower panel - coloc with cQTLs of any dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Coloc loci with multi cQTLs. Upper panel - coloc with single
dataset. Lower panel - coloc with cQTLs of any dataset.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_multi_cqtl_new">
Past versions of 09_12_coloc_num_multi_cqtl_new.pdf
</button>
</p>
<div id="fig-09_12_coloc_num_multi_cqtl_new" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_num_multi_cqtl_new.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2024/09_12_coloc_num_multi_cqtl.pdf">Old version
before harmonizing ref genome</a>.</p>
<ul>
<li><p>Compare to cacti p &amp; cacti s (single dataset)</p>
<ul>
<li><p>LCL</p></li>
<li><p>Macrophage</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_uni_multi_single_dataset_new.pdf" title="Figure: Compare loci coloc with multi or uni cQTLs across single dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Compare loci coloc with multi or uni cQTLs across single
dataset.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_uni_multi_single_dataset_new">
Past versions of 09_12_coloc_num_uni_multi_single_dataset_new.pdf
</button>
</p>
<div id="fig-09_12_coloc_num_uni_multi_single_dataset_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_num_uni_multi_single_dataset_new.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2024/09_12_coloc_num_uni_multi_single_dataset.pdf">Old
version before harmonizing ref genome</a>.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution_new.pdf" title="Figure: Distribution of coloc loci with multi or uni cQTLs across single dataset and traits." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Distribution of coloc loci with multi or uni cQTLs across single
dataset and traits.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_uni_multi_single_dataset_distribution_new">
Past versions of
09_12_coloc_num_uni_multi_single_dataset_distribution_new.pdf
</button>
</p>
<div id="fig-09_12_coloc_num_uni_multi_single_dataset_distribution_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution_new.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a
href="asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution.pdf">Old
version before harmonizing ref genome</a>.</p>
<ul>
<li><p>Compare to cacti p &amp; cacti s (all datasets)</p>
<p>Make a bar plot of three categories of coloc loci - uni cQTL only,
multi cQTL only, shared.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_uni_multi_any_dataset_new.pdf" title="Figure: Compare loci coloc with multi or uni cQTLs of any dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Compare loci coloc with multi or uni cQTLs of any dataset.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-09_12_coloc_num_uni_multi_any_dataset_new">
Past versions of 09_12_coloc_num_uni_multi_any_dataset_new.pdf
</button>
</p>
<div id="fig-09_12_coloc_num_uni_multi_any_dataset_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9087667d12c15c2aa6b3c88b31c283f99535ba46/docs/asset/2024/09_12_coloc_num_uni_multi_any_dataset_new.pdf" target="_blank">9087667</a>
</td>
<td>
liliw-w
</td>
<td>
2024-11-03
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><a href="asset/2024/09_12_coloc_num_uni_multi_any_dataset.pdf">Old
version before harmonizing ref genome</a>.</p>
<div id="why-uni-cqtls-coloc-with-many-loci" class="section level3">
<h3>Why uni cQTLs coloc with many loci?</h3>
<p>The essential question becomes whether those uni peaks that
specifically coloc with loci (grey bars) are included in any windows. If
they are not, it means they are missed by our multi signals simply bc
not being included into the test. If they are inclued in a window, and
that window not coloc with loci, could mean out multi test masked that
uni signal.</p>
<p>Therefore, as a first-step simple check, I only need to check if the
uni peaks corresponding to grey bars are included in any window of any
dataset.</p>
<p><strong>Among those uni peaks which coloc with gwas loci that noc
loci with any windows, 398 are isolated peaks not included in any
window, while 6809 are included in windows that not coloc with the
loci.</strong></p>
<p>As a second step, I add uni peaks and update figure. What I did was
to include all isolated single peaks that were not included in our multi
test into coloc analysis. I run coloc with both windows and those
isolated uni peaks. If a loci coloc with a uni peak, it’s accounted as
being explained by multi.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_merge_uni_multi_single_dataset.pdf" title="Figure: Compare loci coloc with multi or uni cQTLs across single dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Compare loci coloc with multi or uni cQTLs across single
dataset.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_merge_uni_multi_single_dataset_distribution.pdf" title="Figure: Distribution of coloc loci with multi or uni cQTLs across single dataset and traits." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Distribution of coloc loci with multi or uni cQTLs across single
dataset and traits.
</p>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_merge_uni_multi_any_dataset.pdf" title="Figure: Compare loci coloc with multi or uni cQTLs of any dataset." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Compare loci coloc with multi or uni cQTLs of any dataset.
</p>
</div>
</div>
</div>
<div
id="gwas-coloc-with-cqtl-and-eqtls-in-gtex-lcl-and-eqtlgen-whole-blood"
class="section level2">
<h2>6. GWAS coloc with cQTL and eQTLs in GTEx LCL and eQTLGen whole
blood</h2>
<div id="gwas-loci-coloc-with-cqtls-in-lcl-but-not-eqtls-from-gtex-lcl"
class="section level3">
<h3>GWAS loci coloc with cQTLs in LCL but not eQTLs from GTEx LCL</h3>
<ul>
<li>Bars divided into three categories - coloc specific to cQTLs,
specific to eQTLs, shared. (as in the case compare to uni cQTLs) (GWAS
loci coloc with eQTLs from GTEx LCL)</li>
</ul>
<p>Restricted to LCL cQTLs -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_any_dataset_LCL_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_LCL_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_distribution_LCL_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL.
</p>
</div>
<p>Use cQTLs of both LCL and Macrophage -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_any_dataset_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_distribution_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in GTEx LCL." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in
GTEx LCL.
</p>
</div>
<ul>
<li>Compare to loci not coloc with eQTLs from whole blood in
eQTLGen</li>
</ul>
<p>Restricted to LCL cQTLs -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_any_dataset_LCL_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_LCL_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_distribution_LCL_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in eQTLGen whole
blood.
</p>
</div>
<p>Use cQTLs of both LCL and Macrophage -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_any_dataset_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<embed src="asset/2024/09_12_coloc_num_eqtl_cqtl_single_dataset_distribution_eqtlgen.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs in LCL or Macrophage and eQTLs in
eQTLGen whole blood.
</p>
</div>
</div>
<div id="gwas-loci-coloc-with-cqtls-from-lcl-or-whole-blood"
class="section level3">
<h3>GWAS loci coloc with cQTLs from LCL or whole blood</h3>
<p>Based the relationship of gwas loci that are coloc with cQTLs with
eQTLs from LCL or whole blood, the coloc loci can be divided into three
categories and intepreted in different ways.</p>
<table>
<thead>
<tr class="header">
<th align="left">LCL</th>
<th align="left">Whole.blood</th>
<th align="left">Intepretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Yes</td>
<td align="left">Yes</td>
<td align="left"></td>
</tr>
<tr class="even">
<td align="left">Yes</td>
<td align="left">No</td>
<td align="left">Using the right cell type helps</td>
</tr>
<tr class="odd">
<td align="left">No</td>
<td align="left">Yes</td>
<td align="left">Large sample size helps</td>
</tr>
<tr class="even">
<td align="left">No</td>
<td align="left">No</td>
<td align="left">cQTL capture the right cell type</td>
</tr>
</tbody>
</table>
<p>Restricted to LCL cQTLs -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_loci_with_eqtl_cell_type_gtex.pdf" title="Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL or eQTLGen whole blood." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: GWAS loci coloc with cQTLs in LCL and eQTLs in GTEx LCL or
eQTLGen whole blood.
</p>
</div>
<ul>
<li><p>Visualize different categories</p>
<p>Manhattan plot of examples of different categories.</p></li>
</ul>
</div>
</div>
<div id="gwas-coloc-with-cqtls-of-multiple-marks"
class="section level2">
<h2>6. GWAS coloc with cQTLs of multiple marks</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_num_coloc_mark.pdf" title="Figure: Number of marks a region coloc with." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of marks a region coloc with.
</p>
</div>
<p>I picked one region coloc with 5 marks - trait is
<code>baso_neut_sum_N170143</code>, region is
<code>8:8094870</code>.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_manhattan_coloc_multi_mark_nmark5_baso_neut_sum_N170143_8/8094870.pdf.pdf" title="Figure: Manhattan plot of a gwas loci coloc with five marks." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Manhattan plot of a gwas loci coloc with five marks.
</p>
</div>
</div>
<div
id="manhattan-plot-of-gwas-loci-cqtls-and-eqtls-in-multiple-tissues"
class="section level2">
<h2>Manhattan plot of GWAS loci, cQTLs and eQTLs in multiple
tissues</h2>
<p>Manhattan plot of examples of different categories.</p>
<ul>
<li>HFE example used in the manuscript</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_cqtl_eqtl_HFE.pdf" title="Figure: Manhattan plot of GWAS loci, cQTLs and eQTLs in multiple tissues. HFE example used in the manuscript." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Manhattan plot of GWAS loci, cQTLs and eQTLs in multiple
tissues. HFE example used in the manuscript.
</p>
</div>
<ul>
<li>An example of coloc with eQTLs in both tissues</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/09_12_coloc_cqtl_eqtl_both.pdf" title="Figure: Manhattan plot of GWAS loci, cQTLs and eQTLs in multiple tissues. An example of coloc with both cQTLs and eQTLs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Manhattan plot of GWAS loci, cQTLs and eQTLs in multiple
tissues. An example of coloc with both cQTLs and eQTLs.
</p>
</div>
</div>
<div id="update-figures" class="section level2">
<h2>11. Update figures</h2>
<div id="why-uni-cqtls-coloc-with-many-loci-1" class="section level3">
<h3>Why uni cQTLs coloc with many loci?</h3>
<p>First add uni peaks and update figure.</p>
</div>
</div>
<div id="sarahs-edits" class="section level2">
<h2>10. Sarah’s edits</h2>
<p>Done.</p>
</div>
<div id="check-mark-h3k27me3-peak-length" class="section level2">
<h2>9. Check mark H3K27me3 peak length</h2>
<p>Mixed? Broad? Narrow?</p>
</div>
<div id="link-cwindows-to-genes" class="section level2">
<h2><del>8. Link cwindows to genes</del></h2>
<p>Not necessary.</p>
</div>
<div id="performance-with-varying-window-sizes" class="section level2">
<h2>7. Performance with varying window sizes</h2>
</div>
<div id="add-analysis---pathway-or-gene-set-enrichment-analysis"
class="section level2">
<h2>Add analysis - pathway or gene set enrichment analysis</h2>
<p>Add analysis of pathway or gene set enrichment analysis on the genes
associated with cQTLs identified by CACTI but missed by single-peak
mapping -</p>
<ul>
<li><p>Assess whether these genes or regions are enriched for specific
biological functions, pathways, or diseases.</p></li>
<li><p>This adds a layer of biological interpretation to the findings,
showing that the additional cQTLs identified by CACTI are not just
statistically significant but also biologically relevant, thereby
reinforcing the practical importance of CACTI’s improved detection
power.</p></li>
</ul>
</div>
<div id="cqtl-enrichment-in-cell-type-specific-variants"
class="section level2">
<h2>cQTL enrichment in cell type specific variants</h2>
<p>Explain why cQTLs not eQTLs …</p>
</div>
</div>
<div id="july-25" class="section level1">
<h1>July 25</h1>
<div id="sharing" class="section level2">
<h2>Sharing</h2>
<p>Ways to define sharing -</p>
<ul>
<li><p>mashr</p></li>
<li><p>cQTL sharing as in Luis paper</p>
<ul>
<li>get (SNP, window) level significance</li>
</ul></li>
</ul>
</div>
</div>
<div id="may-27" class="section level1">
<h1>May 27</h1>
<div id="cqtl-sharing-across-cell-types" class="section level2">
<h2>1. cQTL sharing across cell types</h2>
<div id="how-1" class="section level3">
<h3>How?</h3>
<p>I compared for one mark, what is the sharing of cQTLs across cell
types?</p>
<ul>
<li><p>I looked at three marks H3K27ac, H3K4me1, and H3K4me3.</p></li>
<li><p>Three cell types LCL, Microphage-NI, and Microphage-Flu.</p></li>
<li><p>I define the sharing of cQTLs by -</p>
<p>For a focal cell type, I took the lead cQTL associations of all
cwindows, I extracted all associations of the lead cQTLs from a compared
cell type, took the minumum pvalue of the lead cQTLs from a compared
cell type, and finally calculate the correlation (spearman) among the
lead pvalue in the focal cell type and the corresponding minimum pvalue
in the compared cell type.</p>
<p>Is it better to use lead SNPs of only significant cwindows?</p></li>
<li><p>Why not use the number of replicated cQTLs?</p>
<p>Another intuitive way is to look at if a significant cQTL in a focal
cell type is also a cQTL in the compared cell type. This won’t work in
our case as there is no easy for use to obtain all cQTLs rather than
only lead cQTLs. QTLTools makes it easy to identify all cQTLs by
defining a feature-level significance cutoff based on the feature-level
empiircal pvalue obtained from beta approximation. However, we can’t do
this for pco pvalues.</p></li>
</ul>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_27_sharing_cell_type.pdf" title="Figure: The correlation (spearman) among the lead pvalue in the focal cell type and the corresponding minimum pvalue in the compared cell type. Across three marks. (A) Use all windows. (B) Use only the significant windows" width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: The correlation (spearman) among the lead pvalue in the focal
cell type and the corresponding minimum pvalue in the compared cell
type. Across three marks. (A) Use all windows. (B) Use only the
significant windows
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_sharing_cell_type">
Past versions of 05_27_sharing_cell_type.pdf
</button>
</p>
<div id="fig-05_27_sharing_cell_type" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_sharing_cell_type.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Note the cis region of LCL (500kb) and Microphage (100kb) are
different.</li>
</ul>
</div>
<div id="compare-to-uni-results" class="section level3">
<h3>Compare to uni results</h3>
<p>Run R^2 in their datasets which uses uni for fair comparison.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_27_sharing_cell_type_uni.pdf" title="Figure: The correlation (spearman) among the lead pvalue by uni test." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: The correlation (spearman) among the lead pvalue by uni test.
</p>
</div>
</div>
</div>
<div id="cqtl-sharing-across-marks" class="section level2">
<h2>2. cQTL sharing across marks</h2>
<div id="how-2" class="section level3">
<h3>How?</h3>
<ul>
<li><p>Three cell types LCL, Microphage-NI, and Microphage-Flu.</p></li>
<li><p>Marks H3K27ac, H3K4me1, and H3K4me3.</p></li>
</ul>
<p>I compared in cell type, what is the sharing of cQTLs across
marks?</p>
<ul>
<li><p>I define the sharing of cQTLs by -</p>
<p>The enrichment of cwindows of one mark in another mark - if a peak of
a mark being significant also means it’s significant for another mark
(the peak should be overlapped between two marks).</p></li>
</ul>
</div>
<div id="results-1" class="section level3">
<h3>Results</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf" title="Figure: cQTL sharing across marks in LCL. (A) The enrichment of cwindows of one mark in another mark. (B) Enrich pvalue. (C) Proportion of cwindows of one mark are also significant for another mark." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: cQTL sharing across marks in LCL. (A) The enrichment of cwindows
of one mark in another mark. (B) Enrich pvalue. (C) Proportion of
cwindows of one mark are also significant for another mark.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_enrich_cwindow_across_mark_lcl">
Past versions of 05_27_enrich_cwindow_across_mark_lcl.pdf
</button>
</p>
<div id="fig-05_27_enrich_cwindow_across_mark_lcl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Similar results of cell type <a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni.pdf">microphage
NI</a> and <a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu.pdf">microphage
flu</a>.</p>
</div>
<div id="compare-to-uni-results-1" class="section level3">
<h3>Compare to uni results</h3>
<p>Run same enrichment analysis in their datasets which uses uni for
fair comparison.</p>
<p>Three cell types.</p>
<ul>
<li><p>Overlap peak number</p>
<ul>
<li>Compare to previous number</li>
</ul></li>
<li><p>Enrich</p>
<ul>
<li>Compare to previous enrichment</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_27_num_overlap_window_lcl.pdf" title="Figure: (LEFT) Windows (RIGHT) Peaks." width="50%" height="100%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_num_overlap_window_lcl">
Past versions of 05_27_num_overlap_window_lcl.pdf
</button>
</p>
<div id="fig-05_27_num_overlap_window_lcl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_num_overlap_window_lcl.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/05_27_num_overlap_window_lcl_uni.pdf" title="Figure: (LEFT) Windows (RIGHT) Peaks." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: (LEFT) Windows (RIGHT) Peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_num_overlap_window_lcl_uni">
Past versions of 05_27_num_overlap_window_lcl_uni.pdf
</button>
</p>
<div id="fig-05_27_num_overlap_window_lcl_uni" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_num_overlap_window_lcl_uni.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf" title="Figure: (LEFT) Windows (RIGHT) Peaks." width="50%" height="100%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_enrich_cwindow_across_mark_lcl">
Past versions of 05_27_enrich_cwindow_across_mark_lcl.pdf
</button>
</p>
<div id="fig-05_27_enrich_cwindow_across_mark_lcl" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/05_27_enrich_cwindow_across_mark_lcl_uni.pdf" title="Figure: (LEFT) Windows (RIGHT) Peaks." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: (LEFT) Windows (RIGHT) Peaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_27_enrich_cwindow_across_mark_lcl_uni">
Past versions of 05_27_enrich_cwindow_across_mark_lcl_uni.pdf
</button>
</p>
<div id="fig-05_27_enrich_cwindow_across_mark_lcl_uni" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_27_enrich_cwindow_across_mark_lcl_uni.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Similar results of other cell type -</p>
<ul>
<li><p><a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni.pdf">window
- microphage NI</a> v.s. <a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni_uni.pdf">peak
- microphage NI</a></p></li>
<li><p><a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu.pdf">window
- microphage flu</a> v.s. <a
href="asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu_uni.pdf">peak
- microphage flu</a></p></li>
</ul>
<p>Overlap peak number of other cell type -</p>
<ul>
<li><p><a
href="asset/2024/05_27_num_overlap_window_microphage_ni.pdf">window -
microphage NI</a> v.s. <a
href="asset/2024/05_27_num_overlap_window_microphage_ni_uni.pdf">peak -
microphage NI</a></p></li>
<li><p><a
href="asset/2024/05_27_num_overlap_window_microphage_flu.pdf">window -
microphage flu</a> v.s. <a
href="asset/2024/05_27_num_overlap_window_microphage_flu_uni.pdf">peak -
microphage flu</a></p></li>
</ul>
<p><strong>Observations</strong></p>
<ul>
<li><p>More overlaps of single peaks than windows across marks</p></li>
<li><p>Less sharing using cwindows</p></li>
</ul>
</div>
</div>
</div>
<div id="may-16" class="section level1">
<h1>May 16</h1>
<div id="gwas-loci-coloc-with-cqtls" class="section level2">
<h2>1. GWAS loci coloc with cQTLs</h2>
<div id="traits-v.s.-cqtls-of-three-marks-in-lcl"
class="section level3">
<h3>36 traits v.s. cQTLs of three marks in LCL</h3>
<ul>
<li>Window size 50kb.</li>
</ul>
<p>I looked at a set of numbers, including …</p>
<p><strong>Overall number -</strong></p>
<ul>
<li><p>Number of GWAS loci</p></li>
<li><p>Number of GWAS loci coloc with any cqtls</p></li>
</ul>
<p><strong>Single mark &amp; mark specificity &amp; mark sharing
-</strong></p>
<ul>
<li><p>Number of GWAS loci coloc with each mark</p></li>
<li><p>Number of GWAS loci coloc with only one mark</p></li>
<li><p>Number of GWAS loci coloc with more than two marks</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_16_coloc_num_alltrait_allmark.pdf" title="Figure: Number of coloc loci across 36 traits and three histone marks in LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of coloc loci across 36 traits and three histone marks in
LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc
with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci
coloc with any mark in red, gwas loci coloc with only one mark in cream.
(D-F) gwas loci coloc with any mark in red, gwas loci coloc with the
mark in violet, gwas loci coloc with only the mark in cream.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_16_coloc_num_alltrait_allmark">
Past versions of 05_16_coloc_num_alltrait_allmark.pdf
</button>
</p>
<div id="fig-05_16_coloc_num_alltrait_allmark" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_16_coloc_num_alltrait_allmark.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_16_coloc_prop_alltrait_allmark.pdf" title="Figure: Proportion of coloc loci across 36 traits and three histone marks in LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Proportion of coloc loci across 36 traits and three histone
marks in LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas
loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C)
gwas loci coloc with any mark in red, gwas loci coloc with only one mark
in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc
with the mark in violet, gwas loci coloc with only the mark in cream.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_16_coloc_prop_alltrait_allmark">
Past versions of 05_16_coloc_prop_alltrait_allmark.pdf
</button>
</p>
<div id="fig-05_16_coloc_prop_alltrait_allmark" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_16_coloc_prop_alltrait_allmark.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div
id="traits-v.s.-cqtls-of-four-marks-in-microphage-of-aracena-et-al."
class="section level3">
<h3>36 traits v.s. cQTLs of four marks in Microphage of Aracena et
al.</h3>
<ul>
<li><p>Window size 50kb.</p></li>
<li><p>Flu only.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_16_coloc_aracena_num_alltrait_allmark.pdf" title="Figure: Number of coloc loci across 36 traits and four histone marks in Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of coloc loci across 36 traits and four histone marks in
Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci
coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas
loci coloc with any mark in red, gwas loci coloc with only one mark in
cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with
the mark in violet, gwas loci coloc with only the mark in cream.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_16_coloc_aracena_num_alltrait_allmark">
Past versions of 05_16_coloc_aracena_num_alltrait_allmark.pdf
</button>
</p>
<div id="fig-05_16_coloc_aracena_num_alltrait_allmark" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_16_coloc_aracena_num_alltrait_allmark.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_16_coloc_aracena_prop_alltrait_allmark.pdf" title="Figure: Proportion of coloc loci across 36 traits and four histone marks in Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Proportion of coloc loci across 36 traits and four histone marks
in Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas
loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C)
gwas loci coloc with any mark in red, gwas loci coloc with only one mark
in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc
with the mark in violet, gwas loci coloc with only the mark in cream.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_16_coloc_aracena_prop_alltrait_allmark">
Past versions of 05_16_coloc_aracena_prop_alltrait_allmark.pdf
</button>
</p>
<div id="fig-05_16_coloc_aracena_prop_alltrait_allmark"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_16_coloc_aracena_prop_alltrait_allmark.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cell-types" class="section level3">
<h3>Cell types</h3>
<ul>
<li>LCL v.s. Microphage</li>
</ul>
</div>
<div id="new-coloc-loci-1" class="section level3">
<h3>New coloc loci</h3>
<ul>
<li><p>Number of GWAS loci coloc with only new cqtls</p></li>
<li><p>Number of GWAS loci new coloc proportion by each mark</p></li>
<li><p>Number of GWAS loci new coloc proportion by more than two
marks</p></li>
<li><p>Number of GWAS loci new coloc proportion by only one
mark</p></li>
</ul>
</div>
<div id="coloc-loci-not-with-eqtls" class="section level3">
<h3>Coloc loci not with eQTLs</h3>
</div>
</div>
</div>
<div id="apr-25-may-2-may-9" class="section level1">
<h1>Apr 25 &amp; May 2 &amp; May 9</h1>
<div id="peak-merging-results-of-lcl-dataset" class="section level2">
<h2>1. Peak merging results of LCL dataset</h2>
<div id="groups-changes-after-merging" class="section level3">
<h3>Groups changes after merging</h3>
<ul>
<li><p>Number of groups</p></li>
<li><p>Size of groups</p></li>
<li><p>Single left out peaks not merged thus not tested</p></li>
</ul>
<p>Below are results of tissue Brain.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/H3K27AC_peaks_in_group_combined.pdf" title="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for tissue Brain." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Distribution of peak group size across window sizes and maximum
merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-H3K27AC_peaks_in_group_combined">
Past versions of H3K27AC_peaks_in_group_combined.pdf
</button>
</p>
<div id="fig-H3K27AC_peaks_in_group_combined" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/H3K27AC_peaks_in_group_combined.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined">
Past versions of H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined.pdf
</button>
</p>
<div id="fig-H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined">
Past versions of H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined.pdf
</button>
</p>
<div id="fig-H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="signal-change-after-merging" class="section level3">
<h3>Signal change after merging</h3>
<ul>
<li><p>Compare to the original results with results of merging single
peaks</p></li>
<li><p>(Minor) Compare window sizes under the same maximum merging
window size</p></li>
</ul>
<p>Below I show results for tissue Brian.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_combined.pdf" title="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for tissue Brian." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals across window sizes and
maximum merging window size. This is for tissue Brian.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-plt_cpeak_cwindow_H3K27AC_cis500kb_combined">
Past versions of plt_cpeak_cwindow_H3K27AC_cis500kb_combined.pdf
</button>
</p>
<div id="fig-plt_cpeak_cwindow_H3K27AC_cis500kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_combined.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined.pdf" title="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for tissue Brian." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals proportions across window
sizes and maximum merging window size. This is for tissue Brian.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined">
Past versions of plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined.pdf
</button>
</p>
<div id="fig-plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="combine-p-values-from-two-tests" class="section level2">
<h2>2. Combine p-values from two tests</h2>
<div id="zhonghuas-idea-not-work" class="section level3">
<h3>Zhonghua’s idea not work</h3>
<p>FDR on window and single peak pvalues separately not work, as the
total number of tests in the single-peak-test pool decreased.</p>
<p>FDR on the window and single peak pvalues together makes more
sense.</p>
<ul>
<li><p>Why XY’s idea of a/b=c/d=a+b/c+d=not work?</p>
<p>The discoveries (c and d) and true positives in these discoveries
would change, as the tests included change, i.e. changing from all
windows (all peaks) to all windows plus isolated peaks.</p></li>
</ul>
</div>
<div id="how-simulation" class="section level3">
<h3>How simulation?</h3>
<p>100k overall tests, including -</p>
<ul>
<li><p>uni test (z-score), to simulate the single left out peaks.</p>
<p>NULL - <span class="math inline">\(z \sim N(0, 1)\)</span></p></li>
<li><p>multi test (PCO), to simulate a group of peaks.</p>
<p>NULL - $ z_multi N_k(0, <em>{101</em>{DGN}})$</p></li>
</ul>
<p>To look at if T1E in simulation are well calibreated, I evaluated
different scenarios -</p>
<ul>
<li><p>across different proportion of uni test (z-score), the rest are
pco test.</p>
<p><code>c(0.01, 0.05, 0.1, 0.3, 0.5, 0.7, 0.9, 0.95, 0.99)</code></p></li>
<li><p>across different sigma dimension K</p>
<p>Randomly take a subset of <span class="math inline">\(\Sigma\)</span>
with dimension <code>c(2, 10, 40, 50, 80)</code>. I chose proportion of
uni tests to be 50% and 20%.</p></li>
</ul>
<p>Caveats -</p>
<p>Not gonna be useful for follow up analysis that require test
statistics …</p>
</div>
<div id="t1e-in-simulation---well-calibrated" class="section level3">
<h3>T1E in simulation - well calibrated</h3>
<p>Calculate T1E by {#nominal P &lt; alpha}/{#tests}.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_new_fig_num_null_prop_test.pdf" title="Figure: T1E and pvalue range across scenarios of different proportion of uni tests." width="100%" type="application/pdf" />
<p class="caption">
Figure: T1E and pvalue range across scenarios of different proportion of
uni tests.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_new_fig_num_null_prop_test">
Past versions of 05_02_new_fig_num_null_prop_test.pdf
</button>
</p>
<div id="fig-05_02_new_fig_num_null_prop_test" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_new_fig_num_null_prop_test.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>See <a href="asset/2024/05_02_fig_qq_null_prop_test.pdf">here for qq
plot</a>.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_new_fig_num_null_K.pdf" title="Figure: T1E and pvalue range across scenarios of different sigma dimension using 50% uni tests (Upper) and 20% uni tests (Lower)." width="100%" type="application/pdf" />
<p class="caption">
Figure: T1E and pvalue range across scenarios of different sigma
dimension using 50% uni tests (Upper) and 20% uni tests (Lower).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_new_fig_num_null_K">
Past versions of 05_02_new_fig_num_null_K.pdf
</button>
</p>
<div id="fig-05_02_new_fig_num_null_K" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_new_fig_num_null_K.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="simulation-on-fdr-control" class="section level3">
<h3>Simulation on FDR control</h3>
<p>I first simulated z-scores of (SNP, peak) and (SNP, peak group) with
a mixture of null and alt tests, then I perform z test and pco test, get
the pvalues, perform fdr correction under a given fdr cutoff, identify
the signals, calculated the false discovery rate by <span
class="math inline">\(\frac{\# signals not true}{\# signals}\)</span>,
iterated simulation multiple times, in order to check if the false
positive discovery rate is well controlled.</p>
<p><strong>Parameter settings</strong></p>
<ul>
<li><p>overall tests - 10^5</p></li>
<li><p>A proportion of tests are uni tests
<code>prop_uni_seq &lt;- c(0.01, 0.2, 0.5, 0.8, 0.99)</code></p></li>
<li><p>Among uni tests and multi tests, a proportion are casual</p>
<p><code>caus_prop_test_seq &lt;- c(0, 0.1, 0.3, 0.5, 0.8, 1)</code></p>
<p>[NOTE - There is no point to look at 0 and 1. Explanation see
below.]</p></li>
<li><p>Uni test - (SNP, peak)</p>
<p>NULL: <span class="math inline">\(z \sim N(0, 1)\)</span></p>
<p>ALT: <span class="math inline">\(z \sim N(\beta \sqrt{N}, 1), \beta
\sim N(0, \sigma_b)\)</span>. <span
class="math inline">\(\sigma_b\)</span> is the genetic variance
explained by each SNP.</p></li>
<li><p>Multi test - (SNP, peak group)</p>
<p>NULL: <span class="math inline">\(z \sim N(0_{K},
\Sigma)\)</span></p>
<p>ALT: <span class="math inline">\(z \sim N(B_{K} \sqrt{N},
\Sigma)\)</span>. <span class="math inline">\(B_{K}\)</span> is the
effect of a SNP on K peaks in the group. I set a proportion of these K
peaks (<code>caus_prop_K</code>) are causal peaks that have non zero
effects. So, for zero-effect peaks <span class="math inline">\(\beta_k =
0\)</span>. For non-zero-effect peaks, <span
class="math inline">\(\beta_k \sim N(0, \sigma_b)\)</span> <span
class="math inline">\(\sigma_b\)</span> is the genetic variance
explained by each SNP.</p></li>
<li><p>single snp variance</p>
<p><code>var.b_seq &lt;- c(0.001, 0.005, 0.01, 0.05)</code></p></li>
<li><p>fdr cutoff</p>
<p><code>fdr_seq &lt;- c(1e-4, 1e-3, 1e-2, 5e-2, 1e-1)</code></p></li>
<li><p>sample size <code>N &lt;- 100</code></p></li>
<li><p>causal proportion among multi K
<code>caus_prop_K &lt;- 0.3</code></p></li>
<li><p>number of simulations <code>ITER_sim &lt;- 10</code></p></li>
</ul>
<p><strong>Results</strong></p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_fig_sim_fdr.pdf" title="Figure: To check if FDR is well controlled across settings. x-axis is single snp variance, y-axis is average false discovery rate across 10 simualtions, colors are causal test proportion. Column panels are uni test proportion. Rows are fdr cutoff." width="100%" type="application/pdf" />
<p class="caption">
Figure: To check if FDR is well controlled across settings. x-axis is
single snp variance, y-axis is average false discovery rate across 10
simualtions, colors are causal test proportion. Column panels are uni
test proportion. Rows are fdr cutoff.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_fig_sim_fdr">
Past versions of 05_02_fig_sim_fdr.pdf
</button>
</p>
<div id="fig-05_02_fig_sim_fdr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_fig_sim_fdr.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Under 0.01 FDR cutoff, fdr are well controlled under most
settings.</p></li>
<li><p>Under 0.05 FDR cutoff, fdr are well controlled under most
settings, except when single snp variance is very low. This is because
for most of 10 simulations, there was no signals. For 1 or 2
simulations, there was 1 signal that are not true positive. FDR will be
well controlled with more simulations.</p></li>
<li><p>Some fdr are 1 under the case where no tests are causal. This is
because for most of 10 simulations, there was no signals. For 1
simulation, there was 1 signal that are not true positive. FDR will be
well controlled with more simulations.</p></li>
<li><p>There is no point to show FDR for cases where causal proportion
tests are 0 (all NULL) or 1 (all ALT).</p>
<ul>
<li><p>When all tests are null, then there is no true signals, any
signals will be false, fdr will always be 1. Should check T1E in this
case.</p></li>
<li><p>When all tests are alt, then any signals will be true signals,
fdr will always be 0. Should check power in this case.</p></li>
</ul></li>
</ul>
</div>
<div id="power-in-real-data" class="section level3">
<h3>Power in real data</h3>
<ul>
<li><p>How?</p>
<p>I included those single left out single peaks in included in any
windows into FDR procedure with multi tests. Specifically, I take all
pairs of cis SNPs and those single peaks, take the best lead association
(ACAT on all cis SNP pvalues), then use the adjusted lead pvalue of each
single peaks, finally perform FDR on these lead pvalues with those of
windows.</p></li>
<li><p>From ChatGPT -</p>
<p>Consideration of power in interpretation -</p>
<p>When interpreting the results of multiple testing correction, it’s
important to consider the power of the individual tests. Significant
p-values surviving correction may indicate not only statistical
significance but also the relative strength of evidence provided by each
test.</p>
<p>Confidence in non-significant results: Non-significant results from
tests with higher power may be more credible than those from tests with
lower power. If a test has sufficient power to detect meaningful
effects, the absence of a significant result may suggest weaker evidence
against the null hypothesis. Conversely, non-significant results from
tests with lower power may be less informative and may not necessarily
provide strong evidence in favor of the null hypothesis.</p>
<p>Interpretation of effect sizes: Considering the power of each test
can also inform the interpretation of effect sizes. Tests with higher
power are more likely to detect smaller effect sizes as statistically
significant. Therefore, significant results from tests with higher power
may indicate not only the presence of an effect but also the magnitude
of that effect, whereas significant results from tests with lower power
may be more indicative of larger effect sizes.</p></li>
<li><p>Results. Take one dataset from Aracena et al. as
example.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb.pdf" title="Figure: Signal comparison with uni test and multi test by including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb." width="100%" type="application/pdf" />
<p class="caption">
Figure: Signal comparison with uni test and multi test by including
single left out peaks in to FDR procedure. This is an example result for
mark H3K27ac_Flu data in Aracena et al. Window size is 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb">
Past versions of
05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Compared to previous one with no single peak.</p>
<p>The four panel plot compared to previous one with no single peak
(combined plot)</p>
<ul>
<li><p>Original peak groups</p></li>
<li><p>Merged peak groups (2 maximum merging window)</p></li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb.pdf" title="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb." width="100%" type="application/pdf" />
<p class="caption">
Figure: Signal comparison before and after including single left out
peaks in to FDR procedure. This is an example result for mark
H3K27ac_Flu data in Aracena et al. Window size is 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb.pdf
</button>
</p>
<div
id="fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb.pdf" title="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum window size is 50kb." width="100%" type="application/pdf" />
<p class="caption">
Figure: Signal comparison before and after including single left out
peaks in to FDR procedure. This is an example result for mark
H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum
window size is 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb.pdf
</button>
</p>
<div
id="fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb.pdf" title="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum window size is 100kb." width="100%" type="application/pdf" />
<p class="caption">
Figure: Signal comparison before and after including single left out
peaks in to FDR procedure. This is an example result for mark
H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum
window size is 100kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb.pdf
</button>
</p>
<div
id="fig-05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Changes in numbers. Across all datasets and parameters.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb.pdf" title="Figure: Total number of cwindows." width="100%" type="application/pdf" />
<p class="caption">
Figure: Total number of cwindows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb.pdf
</button>
</p>
<div id="fig-05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb.pdf" title="Figure: Number of new cwindows." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of new cwindows.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb.pdf
</button>
</p>
<div id="fig-05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb.pdf" title="Figure: Number of missed cpeaks." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of missed cpeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb.pdf
</button>
</p>
<div id="fig-05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb.pdf" title="Figure: Number of non signal windows containing cpeaks." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of non signal windows containing cpeaks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb">
Past versions of
05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb.pdf
</button>
</p>
<div
id="fig-05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/5ddc7b79dd1bbf902d2485ae17108e3bb17a1722/docs/asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb.pdf" target="_blank">5ddc7b7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-09-07
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>What are those lost cwindows after fdr on comb? Pvalue, corrected
pvalue changes …</p></li>
<li><p>What are those still significant cwindows? Pvalue, corrected
pvalue changes …</p></li>
<li><p>Are those new cwindows after fdr on comb?</p></li>
<li><p>What are those repeated cpeaks? pvalue and corrected pvalue
changes …</p></li>
<li><p>What are those still missed cpeaks? pvalue and corrected pvalue
changes …</p></li>
<li><p>Are those new cpeaks after fdr on comb?</p></li>
</ul>
<p><strong>Observations and speculated reasons</strong></p>
<ul>
<li><p>Observation –&gt; indication</p></li>
<li><p>The cost of including those single peaks is to lose some weak
cwindows.</p></li>
</ul>
<p><strong>Prove reasons and indications</strong></p>
<ul>
<li><p>Pattern of changes</p></li>
<li><p>Example of changes</p></li>
</ul>
<p><strong>Conclusions</strong></p>
<ul>
<li>We should use 50kb window size, 100kb maximum merging window (or
not?), perform FDR on both windows and left out peaks.</li>
</ul>
</div>
</div>
</div>
<div id="apr-18" class="section level1">
<h1>Apr 18</h1>
<div id="merge-left-out-single-peaks-to-closest-peak-groups"
class="section level2">
<h2>1. Merge left out single peaks to closest peak groups</h2>
<div id="how-3" class="section level3">
<h3>How?</h3>
<p>I merge a left out single peak to its closest peak group, which is
within a maximum window size. This is basically to expand some peak
groups to include nearby single peaks. If a peak is far from any peak
group, then it is not merged and tested.</p>
<p>I used two maximum merging window sizes -</p>
<ul>
<li><p>50kb. The maximum window size can be 100kb + orignal window
size.</p></li>
<li><p>100kb. The maximum window size can be 200kb + orignal window
size.</p></li>
</ul>
</div>
<div id="analysis" class="section level3">
<h3>Analysis</h3>
<p>I redefined peak groups with merged single peaks, of datasets in
Aracena et al and GTEx. I performed CACTI on the new peak groups across
-</p>
<ul>
<li><p>window sizes</p>
<p>‘50kb’, ‘25kb’, ‘20kb’, ‘10kb’</p></li>
<li><p>maximum merging window sizes</p>
<p>50kb, 100kb</p></li>
<li><p>marks and conditions in Aracena et al</p>
<p>‘H3K27ac’, ‘H3K27me3’, ‘H3K4me1’, ‘H3K4me3’</p>
<p>‘Flu’, ‘NI’</p></li>
<li><p>tissues of mark H3K27ac in GTEx</p>
<p>‘Brain’, ‘Heart’, ‘Lung’, ‘Muscle’</p></li>
</ul>
</div>
<div id="comparison" class="section level3">
<h3>Comparison</h3>
<p>I performed a series of comparisons to investigate different
questions -</p>
<ul>
<li><p>Originals window v.s. extended merged window under same window
size</p>
<p>Check if merging single peaks improves power and reduces missing
signals, or worse, or no change at all.</p></li>
<li><p>(Minor) Extended merged window across window sizes</p>
<p>To check if using smaller window size is better in the case of
extended merging windows, given last time we saw it is not better in
data of Aracena et al. </p></li>
</ul>
</div>
</div>
<div id="aracena-et-al-by-cacti-with-merged-single-peaks"
class="section level2">
<h2>2. Aracena et al by CACTI with merged single peaks</h2>
<div id="datasets-analyzed" class="section level3">
<h3>Datasets analyzed</h3>
<ul>
<li><p>MARK_ARACENA = [‘H3K27ac’, ‘H3K27me3’, ‘H3K4me1’,
‘H3K4me3’]</p></li>
<li><p>CONDITION_ARACENA = [‘Flu’, ‘NI’]</p></li>
<li><p>L_WINDOW = [‘50kb’, ‘25kb’, ‘20kb’, ‘10kb’]</p></li>
<li><p>MAX_L_WINDOW = [‘100kb’, ‘50kb’]</p></li>
</ul>
</div>
<div
id="distance-distribution-from-single-left-out-peaks-to-peak-groups"
class="section level3">
<h3>Distance distribution from single left out peaks to peak groups</h3>
<p>To have a general sense of how far single peaks are from already
defined peak groups and what maximum merging window size we should use,
I looked at -</p>
<ul>
<li><p>Distance from single left-out peak to its closest peak
group</p></li>
<li><p>Distance from single peak to its closest peak group</p></li>
</ul>
<p>Below I only show results for Mark H3K27ac, condition Flu.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb.pdf" title="Figure: Distribution of (A) distance of a peak to its left closest peak, and (B) distance from a left-out single peak to its closest peak group, across window sizes 10kb, 25kb, 50kb. Mark H3K27ac, condition Flu." width="60%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb">
Past versions of
04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb.pdf" title="Figure: Distribution of (A) distance of a peak to its left closest peak, and (B) distance from a left-out single peak to its closest peak group, across window sizes 10kb, 25kb, 50kb. Mark H3K27ac, condition Flu." width="60%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb">
Past versions of
04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb.pdf" title="Figure: Distribution of (A) distance of a peak to its left closest peak, and (B) distance from a left-out single peak to its closest peak group, across window sizes 10kb, 25kb, 50kb. Mark H3K27ac, condition Flu." width="60%" type="application/pdf" />
<p class="caption">
Figure: Distribution of (A) distance of a peak to its left closest peak,
and (B) distance from a left-out single peak to its closest peak group,
across window sizes 10kb, 25kb, 50kb. Mark H3K27ac, condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb">
Past versions of
04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation -</strong></p>
<ul>
<li><p>50kb and 100kb include ???% single peaks.</p></li>
<li><p>???% are still left out and not tested.</p></li>
<li><p>If to include ~50% single peaks, use at least 50kb -
100kb.</p></li>
<li><p>Larger window size needs larger maximum window size to include
more singe peaks.</p></li>
<li><p>Using the same maximum window size, smaller windows can include
more singe peaks. But at the same time, more single peaks were left out
already.</p></li>
<li><p>Increase maximum merging group size to include more signle peaks?
To include all single peaks, the maximum window size is ???.</p></li>
</ul>
<p>As for other datasets -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_peak_dis_to_left_peak.pdf" title="Figure: Distance distribution of a peak to its left closest peak defined for marks. Two condition NI and Flu have same peak sets." width="100%" type="application/pdf" />
<p class="caption">
Figure: Distance distribution of a peak to its left closest peak defined
for marks. Two condition NI and Flu have same peak sets.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_to_left_peak">
Past versions of 04_18_peak_dis_to_left_peak.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_to_left_peak" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_to_left_peak.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation -</strong></p>
<ul>
<li><p>Other marks has similar results, except mark H3K4me3 has larger
peak distance.</p></li>
<li><p>Do we use larger maximum merging window size?</p></li>
<li><p>Do we use larger window size for this mark?</p></li>
</ul>
</div>
<div id="groups-changes-after-merging-1" class="section level3">
<h3>Groups changes after merging</h3>
<ul>
<li><p>Number of groups</p></li>
<li><p>Size of groups</p></li>
<li><p>Single left out peaks not merged thus not tested</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_H3K27ac_Flu_peaks_in_group_combined.pdf" title="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu." width="100%" type="application/pdf" />
<p class="caption">
Figure: Distribution of peak group size across window sizes and maximum
merging window size. This is for Mark H3K27ac, condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_H3K27ac_Flu_peaks_in_group_combined">
Past versions of 04_18_H3K27ac_Flu_peaks_in_group_combined.pdf
</button>
</p>
<div id="fig-04_18_H3K27ac_Flu_peaks_in_group_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_H3K27ac_Flu_peaks_in_group_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for Mark H3K27ac, condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined">
Past versions of
04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf
</button>
</p>
<div
id="fig-04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for Mark H3K27ac, condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined">
Past versions of
04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf
</button>
</p>
<div
id="fig-04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Merging single peaks into peak group increase group
size.</p></li>
<li><p>Merging single peaks into peak group increase number of uni
cpeaks in peak groups, which means we are able to include those left-out
single cpeaks to our windows and include them into test.</p></li>
</ul>
</div>
<div id="signal-change-after-merging-1" class="section level3">
<h3>Signal change after merging</h3>
<ul>
<li><p>Compare to the original results with results of merging single
peaks</p></li>
<li><p>(Minor) Compare window sizes under the same maximum merging
window size</p></li>
</ul>
<p>Below I show results for Mark H3K27ac, condition Flu.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined.pdf" title="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals across window sizes and
maximum merging window size. This is for Mark H3K27ac, condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined">
Past versions of
04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined.pdf
</button>
</p>
<div id="fig-04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined.pdf" title="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals proportions across window
sizes and maximum merging window size. This is for Mark H3K27ac,
condition Flu.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined">
Past versions of
04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined.pdf
</button>
</p>
<div id="fig-04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Merging single peaks can decrease the missed uni cpeaks by CACTI,
by including them into our test.</p></li>
<li><p>Merging single peaks can increase cwindows and new
cwindows.</p></li>
<li><p>Merging helps less under larger window size, in terms of
including less uni cpeaks to windows and less improvement in finding
more cwindows.</p></li>
<li><p>Other marks has similar results, except mark H3K4me3, which has
less improvement.</p></li>
</ul>
</div>
</div>
<div id="gtex-by-cacti-with-merged-single-peaks" class="section level2">
<h2>3. GTEx by CACTI with merged single peaks</h2>
<div id="datasets-analyzed-1" class="section level3">
<h3>Datasets analyzed</h3>
<ul>
<li><p>TISSUE_TYPE = [‘Brain’, ‘Heart’, ‘Lung’, ‘Muscle’]</p></li>
<li><p>L_WINDOW = [‘50kb’, ‘25kb’, ‘20kb’, ‘10kb’]</p></li>
<li><p>MAX_L_WINDOW = [‘100kb’, ‘50kb’]</p></li>
</ul>
</div>
<div
id="distance-distribution-from-single-left-out-peaks-to-peak-groups-1"
class="section level3">
<h3>Distance distribution from single left out peaks to peak groups</h3>
<p>To have a general sense of how far single peaks are from already
defined peak groups and what maximum merging window size we should use,
I looked at -</p>
<ul>
<li>Distance from single left-out peak to its closest peak group</li>
</ul>
<p>Below I show results across tissues (rows) and window sizes
(columns).</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_peak_dis_to_group_gtex.pdf" title="Figure: Distribution of distance from a left-out single peak to its closest peak group, across tissues (rows) and window sizes (columns)." width="60%" type="application/pdf" />
<p class="caption">
Figure: Distribution of distance from a left-out single peak to its
closest peak group, across tissues (rows) and window sizes (columns).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_to_group_gtex">
Past versions of 04_18_peak_dis_to_group_gtex.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_to_group_gtex" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_to_group_gtex.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation -</strong></p>
<ul>
<li><p>50kb and 100kb include over 20% and 50% single peaks using 50kb
window.</p></li>
<li><p>If to include ~50% single peaks, use at least 100kb.</p></li>
<li><p>Using the same maximum window size, smaller windows can include
more singe peaks. But at the same time, more single peaks were left out
already.</p></li>
<li><p>Increase maximum merging group size to include more signle peaks?
To include all single peaks, the maximum window size is ???.</p></li>
<li><p>No distinct difference across tissues. (though distinct
difference observed across marks in Aracena et al.)</p></li>
</ul>
<p>I then looked at the overall distance distribution among peaks -</p>
<ul>
<li>Distance of a peak to its left peak to show distance among
peaks</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_peak_dis_to_left_peak_gtex.pdf" title="Figure: Distance distribution of a peak to its left closest peak defined for marks." width="100%" type="application/pdf" />
<p class="caption">
Figure: Distance distribution of a peak to its left closest peak defined
for marks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_peak_dis_to_left_peak_gtex">
Past versions of 04_18_peak_dis_to_left_peak_gtex.pdf
</button>
</p>
<div id="fig-04_18_peak_dis_to_left_peak_gtex" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_peak_dis_to_left_peak_gtex.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observation -</strong></p>
<ul>
<li>GTEx has more H3K27ac peaks. They are closer (distance ~1kb - 10kb)
than H3K27ac peaks in Aracena et al (distance ~10kb).</li>
</ul>
</div>
<div id="groups-changes-after-merging-2" class="section level3">
<h3>Groups changes after merging</h3>
<ul>
<li><p>Number of groups</p></li>
<li><p>Size of groups</p></li>
<li><p>Single left out peaks not merged thus not tested</p></li>
</ul>
<p>Below are results of tissue Brain.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_Brain_peaks_in_group_combined.pdf" title="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for tissue Brain." width="100%" type="application/pdf" />
<p class="caption">
Figure: Distribution of peak group size across window sizes and maximum
merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_Brain_peaks_in_group_combined">
Past versions of 04_18_Brain_peaks_in_group_combined.pdf
</button>
</p>
<div id="fig-04_18_Brain_peaks_in_group_combined" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_Brain_peaks_in_group_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined">
Past versions of
04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf
</button>
</p>
<div id="fig-04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain." width="100%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes and
maximum merging window size. This is for tissue Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined">
Past versions of
04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf
</button>
</p>
<div id="fig-04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For other tissues, check <a
href="asset/2024/04_18_gourp_results_heart.pdf">here for Heart</a>, <a
href="asset/2024/04_18_gourp_results_lung.pdf">here for Lung</a>, <a
href="asset/2024/04_18_gourp_results_muscle.pdf">here for
Muscle</a>.</p>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Merging single peaks into peak group increase group
size.</p></li>
<li><p>Merging single peaks into peak group increase number of uni
cpeaks in peak groups, which means we are able to include those left-out
single cpeaks to our windows and include them into test.</p></li>
<li><p>The increase of uni cpeaks in merged windows is not as much as
observed in Aracena et al, where more uni cpeaks were
identified.</p></li>
</ul>
</div>
<div id="signal-change-after-merging-2" class="section level3">
<h3>Signal change after merging</h3>
<ul>
<li><p>Compare to the original results with results of merging single
peaks</p></li>
<li><p>(Minor) Compare window sizes under the same maximum merging
window size</p></li>
</ul>
<p>Below I show results for tissue Brian.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_combined.pdf" title="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for tissue Brian." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals across window sizes and
maximum merging window size. This is for tissue Brian.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_plt_cpeak_cwindow_Brain_cis100kb_combined">
Past versions of 04_18_plt_cpeak_cwindow_Brain_cis100kb_combined.pdf
</button>
</p>
<div id="fig-04_18_plt_cpeak_cwindow_Brain_cis100kb_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined.pdf" title="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for tissue Brian." width="100%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals proportions across window
sizes and maximum merging window size. This is for tissue Brian.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined">
Past versions of
04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined.pdf
</button>
</p>
<div id="fig-04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>For other tissues, check <a
href="asset/2024/04_18_plt_cpeak_cwindow_Heart_cis100kb_combined.pdf">here
for Heart</a>, <a
href="asset/2024/04_18_plt_cpeak_cwindow_Lung_cis100kb_combined.pdf">here
for Lung</a>, <a
href="asset/2024/04_18_plt_cpeak_cwindow_Muscle_cis100kb_combined.pdf">here
for Muscle</a>.</p>
<p>Proportions - check <a
href="asset/2024/04_18_plt_cpeak_cwindow_Heart_cis100kb_prop_combined.pdf">here
for Heart</a>, <a
href="asset/2024/04_18_plt_cpeak_cwindow_Lung_cis100kb_prop_combined.pdf">here
for Lung</a>, <a
href="asset/2024/04_18_plt_cpeak_cwindow_Muscle_cis100kb_prop_combined.pdf">here
for Muscle</a>.</p>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Merging single peaks can decrease the missed uni cpeaks by CACTI,
by including them into our test. Although less cpeaks were included by
merging than Aracena et al, as not many uni cpeaks were found in
GTEx.</p></li>
<li><p>Although Merging single peaks can increase cwindows and new
cwindows, it’s much less than improvement seen in Aracena et al
dataset.</p></li>
<li><p>Merging helps less under larger window size, in terms of
including less uni cpeaks to windows and less improvement in finding
more cwindows. Same as in Aracena et al dataset.</p></li>
</ul>
</div>
</div>
</div>
<div id="apr-04-apr-11" class="section level1">
<h1>Apr 04 &amp; Apr 11</h1>
<div id="perform-test-on-window-of-single-peak-by-mixing-two-tests"
class="section level2">
<h2>1. Perform test on window of single peak by mixing two tests</h2>
<p>To reduce CACTI missing signals, by reducing the purple bar of uni
that do not overlap with any window, and therefore reduce the purple bar
of uni that do not overlap with any CACTI cwindows.</p>
<p>So we have two tests -</p>
<ul>
<li><p>Test on window of single peak</p></li>
<li><p>Test on window of multiple peaks</p></li>
</ul>
<p>We wanted to include the test on window of single peak in our test. A
few candidate options are -</p>
<div id="acatpco-minp" class="section level3">
<h3>ACAT(pco, minp)</h3>
</div>
<div id="mixture-test" class="section level3">
<h3>mixture test</h3>
</div>
</div>
<div
id="look-into-the-43-non-cwindows-missed-by-cacti-move-this-section-to-below"
class="section level2">
<h2>2. Look into the 43 non-cwindows missed by CACTI (move this section
to <a href="#what-are-those-cacti-missing-signals">below</a>)</h2>
<ul>
<li><p>a genometrack by genotype plots etc</p></li>
<li><p>a plot shows all the univariate pvalues</p></li>
</ul>
</div>
<div id="dig-into-gtex-peaks-and-windows" class="section level2">
<h2>3. Dig into GTEx peaks and windows</h2>
<p>Is there something weird going on in GTEx results?</p>
<div id="check-gtex-univariate-peak-pvalues-distribution"
class="section level3">
<h3>Check GTEx univariate peak pvalues distribution</h3>
<p><strong>Goal -</strong></p>
<p>To make sure the pvalues are well calibrated before combining
them.</p>
<p><strong>How?</strong></p>
<p>As XY suggested - for a peak, look at the pvalue distribution,
i.e. histogram and qqplot.</p>
<p>Instead, inspired by the way MatrixeQTL used to visualize the
associations, I looked at the distribution of all peak and SNP pairs, in
terms of the histogram and qqplot.</p>
<ul>
<li>Histogram</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_04_p_hist_allchr_gtex_Brain.pdf" title="Figure: Histogram of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_hist_allchr_gtex_Brain">
Past versions of 04_04_p_hist_allchr_gtex_Brain.pdf
</button>
</p>
<div id="fig-04_04_p_hist_allchr_gtex_Brain" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_hist_allchr_gtex_Brain.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_04_p_hist_allchr_gtex_Heart.pdf" title="Figure: Histogram of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_hist_allchr_gtex_Heart">
Past versions of 04_04_p_hist_allchr_gtex_Heart.pdf
</button>
</p>
<div id="fig-04_04_p_hist_allchr_gtex_Heart" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_hist_allchr_gtex_Heart.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_04_p_hist_allchr_gtex_Muscle.pdf" title="Figure: Histogram of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_hist_allchr_gtex_Muscle">
Past versions of 04_04_p_hist_allchr_gtex_Muscle.pdf
</button>
</p>
<div id="fig-04_04_p_hist_allchr_gtex_Muscle" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_hist_allchr_gtex_Muscle.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_04_p_hist_allchr_gtex_Lung.pdf" title="Figure: Histogram of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" type="application/pdf" />
<p class="caption">
Figure: Histogram of GTEx p-values across all pairs of peaks and snps,
for four tissues, Brain, Heart, Muscle, and Lung.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_hist_allchr_gtex_Lung">
Past versions of 04_04_p_hist_allchr_gtex_Lung.pdf
</button>
</p>
<div id="fig-04_04_p_hist_allchr_gtex_Lung" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_hist_allchr_gtex_Lung.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>qqplot</li>
</ul>
<div class="figure" style="text-align: center">
<p><img src="asset/2024/04_04_p_qqplot_allchr_gtex_Brain.png" alt="Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_qqplot_allchr_gtex_Brain">
Past versions of 04_04_p_qqplot_allchr_gtex_Brain.png
</button>
</p>
<div id="fig-04_04_p_qqplot_allchr_gtex_Brain" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_qqplot_allchr_gtex_Brain.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2024/04_04_p_qqplot_allchr_gtex_Heart.png" alt="Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_qqplot_allchr_gtex_Heart">
Past versions of 04_04_p_qqplot_allchr_gtex_Heart.png
</button>
</p>
<div id="fig-04_04_p_qqplot_allchr_gtex_Heart" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_qqplot_allchr_gtex_Heart.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="asset/2024/04_04_p_qqplot_allchr_gtex_Muscle.png" alt="Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_qqplot_allchr_gtex_Muscle">
Past versions of 04_04_p_qqplot_allchr_gtex_Muscle.png
</button>
</p>
<div id="fig-04_04_p_qqplot_allchr_gtex_Muscle" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_qqplot_allchr_gtex_Muscle.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2024/04_04_p_qqplot_allchr_gtex_Lung.png" alt="Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung." width="20%" />
<p class="caption">
Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for
four tissues, Brain, Heart, Muscle, and Lung.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_p_qqplot_allchr_gtex_Lung">
Past versions of 04_04_p_qqplot_allchr_gtex_Lung.png
</button>
</p>
<div id="fig-04_04_p_qqplot_allchr_gtex_Lung" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_p_qqplot_allchr_gtex_Lung.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li>The power is pretty low compared to data in Aracena et al and
Grubert et al (LCL).</li>
</ul>
<p><strong>Aracena et al</strong></p>
<p>Similar p-value distribution for four marks (‘H3K27ac’, ‘H3K27me3’,
‘H3K4me1’, ‘H3K4me3’) and two conditions (‘Flu’, “NI”). Below is an
example of Mark H3K27ac, Condition Flu, Tissue primary macrophages.</p>
<div class="figure" style="text-align: center">
<p><img src="asset/2024/04_04_aracena_H3K27ac_pval_histogram.png" alt="Figure: Histogram and QQplot of p-values across all pairs of peaks and snps used in Aracena et al. Mark H3K27ac, Condition Flu, Tissue primary macrophages." width="20%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_aracena_H3K27ac_pval_histogram">
Past versions of 04_04_aracena_H3K27ac_pval_histogram.png
</button>
</p>
<div id="fig-04_04_aracena_H3K27ac_pval_histogram" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_aracena_H3K27ac_pval_histogram.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2024/04_04_aracena_H3K27ac_pval_qqplot.png" alt="Figure: Histogram and QQplot of p-values across all pairs of peaks and snps used in Aracena et al. Mark H3K27ac, Condition Flu, Tissue primary macrophages." width="20%" />
<p class="caption">
Figure: Histogram and QQplot of p-values across all pairs of peaks and
snps used in Aracena et al. Mark H3K27ac, Condition Flu, Tissue primary
macrophages.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_04_aracena_H3K27ac_pval_qqplot">
Past versions of 04_04_aracena_H3K27ac_pval_qqplot.png
</button>
</p>
<div id="fig-04_04_aracena_H3K27ac_pval_qqplot" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_04_aracena_H3K27ac_pval_qqplot.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Grubert et al</strong></p>
<div class="figure" style="text-align: center">
<img src="asset/2024/04_04_" alt="Figure: Histogram of p-values across all pairs of peaks and snps used in Grubert et al. Mark H3K27ac, Tissue LCL." width="20%" />
<p class="caption">
Figure: Histogram of p-values across all pairs of peaks and snps used in
Grubert et al. Mark H3K27ac, Tissue LCL.
</p>
</div>
</div>
<div id="compare-to-lcl-signals" class="section level3">
<h3>Compare to LCL signals</h3>
<ul>
<li><p>Number of cpeaks in GTEx and LCL cwindows</p></li>
<li><p>Number of weak cpeaks in LCL cwindows</p>
<p>Use a loose p value cutoff like 0.0005, to see if on average LCL
datasets have more weak signals in each window.</p></li>
<li><p>Check the correlation between peaks in both GTEx and LCL
datasets</p></li>
</ul>
</div>
</div>
<div id="analyze-data-in-aracena-et-al---part-1" class="section level2">
<h2>4. Analyze data in Aracena et al - Part 1</h2>
<p>Their paper is <a
href="https://www.nature.com/articles/s41588-024-01668-z#data-availability">online</a>.</p>
<div id="number-of-peaks-cpeaks-across-marks-conditions"
class="section level3">
<h3>Number of peaks (&amp;cpeaks) across marks &amp; conditions</h3>
<ul>
<li><p>Number of total peaks. Number of cpeaks under 5% FDR, and 10% FDR
(see <a
href="https://www.nature.com/articles/s41588-024-01668-z/figures/11">Extended
Data Fig. 4a</a>).</p></li>
<li><p>Compare to GTEx and LCL, more cpeaks and higher cpeak
proportion?</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_11_signal_aracena_et_al.pdf" title="Figure: Uni cpeak signal results in Aracena et al (LEFT) and GTEx (RIGHT). (A) Number of cpeaks, (B) Proportion of cpeaks among total peaks. Across datasets, 5% FDR, and 10% FDR. X-axis is datasets in Aracena et al and GTEx. Aracena et al used 10% fdr in the paper, while GTEx used empp&lt;0.005 in the paper. I added results using 5% fdr to both datasets. Note, GTEx results are of mark H3K27ac across four tissues, while Aracena et al analyzed several marks in primary macrophages under two conditions." width="45%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_signal_aracena_et_al">
Past versions of 04_11_signal_aracena_et_al.pdf
</button>
</p>
<div id="fig-04_11_signal_aracena_et_al" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_signal_aracena_et_al.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_11_signal_gtex_v2.pdf" title="Figure: Uni cpeak signal results in Aracena et al (LEFT) and GTEx (RIGHT). (A) Number of cpeaks, (B) Proportion of cpeaks among total peaks. Across datasets, 5% FDR, and 10% FDR. X-axis is datasets in Aracena et al and GTEx. Aracena et al used 10% fdr in the paper, while GTEx used empp&lt;0.005 in the paper. I added results using 5% fdr to both datasets. Note, GTEx results are of mark H3K27ac across four tissues, while Aracena et al analyzed several marks in primary macrophages under two conditions." width="45%" type="application/pdf" />
<p class="caption">
Figure: Uni cpeak signal results in Aracena et al (LEFT) and GTEx
(RIGHT). (A) Number of cpeaks, (B) Proportion of cpeaks among total
peaks. Across datasets, 5% FDR, and 10% FDR. X-axis is datasets in
Aracena et al and GTEx. Aracena et al used 10% fdr in the paper, while
GTEx used empp&lt;0.005 in the paper. I added results using 5% fdr to
both datasets. Note, GTEx results are of mark H3K27ac across four
tissues, while Aracena et al analyzed several marks in primary
macrophages under two conditions.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_signal_gtex_v2">
Past versions of 04_11_signal_gtex_v2.pdf
</button>
</p>
<div id="fig-04_11_signal_gtex_v2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_signal_gtex_v2.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations</strong></p>
<ul>
<li><p>GTEx has much more total called peaks, ~3x more.</p></li>
<li><p>GTEx has less significant cpeaks under the same 5% FDR.</p></li>
<li><p>GTEx has lower signal cpeak proportion.</p></li>
</ul>
</div>
<div id="sample-size-across-marks-conditions" class="section level3">
<h3>Sample size across marks &amp; conditions</h3>
<p>35 individuals.</p>
</div>
<div id="covaraites-they-used" class="section level3">
<h3>Covaraites they used</h3>
<p>batch and age are already accounted for in input reads.</p>
<p>PC1 of genotype data is the only covariate included (to account for
population structure).</p>
</div>
<div id="figure-how-their-matrixeqtl-works" class="section level3">
<h3>Figure how their matrixeqtl works</h3>
<ul>
<li><p>what is the statistics? From t-test?</p></li>
<li><p>Do the statistics correspond to pvalues?</p></li>
<li><p>What is the beta column? How is it generated?</p></li>
<li><p>Does the statistics equal z-scores? Can their correlation be
estimated from cor(residual pheno)?</p></li>
<li><p>Run fastqtl and see if the number of cpeaks changes?</p></li>
</ul>
</div>
</div>
<div id="analyze-data-in-aracena-et-al---part-2" class="section level2">
<h2>5. Analyze data in Aracena et al - Part 2</h2>
<div id="datasets-analyzed-2" class="section level3">
<h3>Datasets analyzed</h3>
<ul>
<li><p>MARK_ARACENA = [‘H3K27ac’, ‘H3K27me3’, ‘H3K4me1’,
‘H3K4me3’]</p></li>
<li><p>CONDITION_ARACENA = [‘Flu’, ‘NI’]</p></li>
<li><p>L_WINDOW = [‘50kb’, ‘25kb’, ‘20kb’, ‘10kb’]</p></li>
</ul>
</div>
<div id="number-of-peak-groups-group-size" class="section level3">
<h3>Number of peak groups &amp; group size</h3>
<ul>
<li>Compare to LCL and GTEx, fewer groups? more single-peak-group?</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window10kb.pdf" title="Figure: Distribution of peak group size across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_window10kb">
Past versions of 04_11_H3K27ac_Flu_peaks_in_group_window10kb.pdf
</button>
</p>
<div id="fig-04_11_H3K27ac_Flu_peaks_in_group_window10kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window10kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window20kb.pdf" title="Figure: Distribution of peak group size across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_window20kb">
Past versions of 04_11_H3K27ac_Flu_peaks_in_group_window20kb.pdf
</button>
</p>
<div id="fig-04_11_H3K27ac_Flu_peaks_in_group_window20kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window20kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window25kb.pdf" title="Figure: Distribution of peak group size across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_window25kb">
Past versions of 04_11_H3K27ac_Flu_peaks_in_group_window25kb.pdf
</button>
</p>
<div id="fig-04_11_H3K27ac_Flu_peaks_in_group_window25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window25kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window50kb.pdf" title="Figure: Distribution of peak group size across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p class="caption">
Figure: Distribution of peak group size across window sizes 10kb, 20kb,
25kb, 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_window50kb">
Past versions of 04_11_H3K27ac_Flu_peaks_in_group_window50kb.pdf
</button>
</p>
<div id="fig-04_11_H3K27ac_Flu_peaks_in_group_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes 10kb,
20kb, 25kb, 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb.pdf" title="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p class="caption">
Figure: Number of uni cpeaks in peak groups across window sizes 10kb,
20kb, 25kb, 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb">
Past versions of
04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb.pdf
</button>
</p>
<div
id="fig-04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cacti-signal-comparison-with-univairate-signals"
class="section level3">
<h3>CACTI signal comparison with univairate signals</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb.pdf" title="Figure: Compare CACTI signals to uni signals using FDR 0.05 across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb">
Past versions of
04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb.pdf
</button>
</p>
<div id="fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb.pdf" title="Figure: Compare CACTI signals to uni signals using FDR 0.05 across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb">
Past versions of
04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb.pdf
</button>
</p>
<div id="fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><embed src="asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb.pdf" title="Figure: Compare CACTI signals to uni signals using FDR 0.05 across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb">
Past versions of
04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb.pdf
</button>
</p>
<div id="fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb.pdf" title="Figure: Compare CACTI signals to uni signals using FDR 0.05 across window sizes 10kb, 20kb, 25kb, 50kb." width="20%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to uni signals using FDR 0.05 across
window sizes 10kb, 20kb, 25kb, 50kb.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb">
Past versions of
04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>CACTI found more signals than univariate signals.</p></li>
<li><p>Univariate signals missed by CACTI are due to not being included
in any peak groups, therefore, not being tested.</p></li>
<li><p>CACTI found more signals in this dataset than in the GTEx
dataset.</p></li>
<li><p>Using 50kb window seems to have better power, in terms of finding
more new signals, in this dataset, whereas in GTEx 10kb window seems
better.</p></li>
</ul>
</div>
</div>
</div>
<div id="mar-28" class="section level1">
<h1>Mar 28</h1>
<div id="to-reduce-missing-signals-and-increase-new-signals"
class="section level2">
<h2>1. To reduce missing signals and increase new signals</h2>
<div id="lift-fdr-threshold-narrower-windows" class="section level3">
<h3>Lift fdr threshold &amp; Narrower windows</h3>
<p>No big difference.</p>
<div class="figure" style="text-align: center">
<p><img src="asset/2024/03_28_signal_across_fdr.png" alt="Figure: Compare CACTI signals to their signals in Brain across FDR thresholds (LEFT) and across window sizes (RIGHT)." width="30%" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_28_signal_across_fdr">
Past versions of 03_28_signal_across_fdr.png
</button>
</p>
<div id="fig-03_28_signal_across_fdr" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_28_signal_across_fdr.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<img src="asset/2024/03_28_signal_across_window_size.png" alt="Figure: Compare CACTI signals to their signals in Brain across FDR thresholds (LEFT) and across window sizes (RIGHT)." width="30%" />
<p class="caption">
Figure: Compare CACTI signals to their signals in Brain across FDR
thresholds (LEFT) and across window sizes (RIGHT).
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_28_signal_across_window_size">
Past versions of 03_28_signal_across_window_size.png
</button>
</p>
<div id="fig-03_28_signal_across_window_size" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_28_signal_across_window_size.png" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>To reduce the non-cwindows, decrease window size.</p></li>
<li><p>Compare to results of the mark in LCL using 10kb and 50kb window,
is there a similar trend?</p></li>
<li><p>To use 10kb in GTEx to get fewer missing cwindows.</p></li>
<li><p>To use 10kb in LCL to keep consistent to GTEx window size
choice.</p></li>
</ul>
</div>
</div>
<div id="what-are-those-cacti-missing-signals" class="section level2">
<h2>2. What are those CACTI missing signals?</h2>
<div id="general-aspects" class="section level3">
<h3>General aspects</h3>
<ul>
<li><p>Signal to noise ratio</p>
<p>Replicated signals by multi have higher signal to noise ratio,
i.e. smaller window size and more cpeaks by uni.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_28_fig_sig_2_noise_Brain_window50kb_cis100kb.pdf" title="Figure: Signal to noise ratio of uni signals missed &amp; replicated by multi in Brain." width="40%" type="application/pdf" />
<p class="caption">
Figure: Signal to noise ratio of uni signals missed &amp; replicated by
multi in Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_28_fig_sig_2_noise_Brain_window50kb_cis100kb">
Past versions of 03_28_fig_sig_2_noise_Brain_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-03_28_fig_sig_2_noise_Brain_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_28_fig_sig_2_noise_Brain_window50kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="mar-21" class="section level1">
<h1>Mar 21</h1>
<div id="cacti-on-gtex-multitissue-chip-datasets"
class="section level2">
<h2>1. CACTI on GTEx multitissue ChIP datasets</h2>
<div id="background-on-datasets" class="section level3">
<h3>Background on datasets</h3>
<ul>
<li><p>Sample sizes across tissues.</p>
<ul>
<li>sample size used to estimate </li>
</ul></li>
<li><p>Number of peaks across tissues.</p></li>
<li><p>Covaraites across tissues.</p>
<blockquote>
<p>We inferred the latent factors for our samples in each of the four
tissues by Peer45 (R package v1.0) and identified the optimum number of
Peer factors (5, 5, 10 and 2 for brain, heart, muscle and lung) to
correct for during haQTL calling. Additionally, we took into account
age, batches for H3K27ac ChIP–seq experiment, top five genotype PCs, and
those technical effects related to platform and PCR.</p>
</blockquote></li>
<li><p>Number of cpeaks across tissues (empirical p &lt; 0.005 or
FDR).</p>
<p>Their way to call cpeaks -</p>
<blockquote>
<p>An empirical P value for each ARE was estimated based on the lead
nominal P value and permutation results (–permute 1000 10000) by fitting
a β distribution to account for multiple variants tested. We used an
empirical P value threshold of 0.005 to identify gAREs as in a previous
study, and then for each gARE, we applied the nominal P value threshold
corresponding to the empirical P value of 0.005 for each locus to
identify haQTLs.</p>
</blockquote>
<p>From <a href="https://doi.org/10.1038/s41588-021-00890-3">another
paper of theirs</a> that uses similar method -</p>
<blockquote>
<p>To identify g-m6A sites, the nominal P value of the lead SNP of each
m6A peak was corrected to get an empirical P value that accounts for the
multiple variants tested. The empirical P value was extrapolated based
on beta distribution fitting to the permutation results by FastQTL
(parameter –permute 1,000). We applied a threshold of empirical P
value&lt;0.005 on the lead variant to identify g-m6A sites, and then for
each g-m6A we applied the nominal P value threshold that corresponds to
the empirical P value of 0.005 for each locus to identify m6A QTLs
(Supplementary Table 2). Given the larger sample size in brain, we
provided higher-confidence sets of g-m6A sites and m6A QTLs by further
correcting the empirical P value with the Benjamini–Hochberg procedure
and applying a threshold of FDR&lt;0.2 (Supplementary Table 3).</p>
</blockquote>
<ul>
<li>There are over 500 cpeaks in lung, but the cpeaks are problematic as
they correspond to fdr &lt; 0.966. Lung has only one cpeak under FDR
&lt; 0.05.</li>
</ul></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_21_signal_gtex.pdf" title="Figure: The equivalent FDR level to using empirical p &lt; 0.005." width="40%" type="application/pdf" />
<p class="caption">
Figure: The equivalent FDR level to using empirical p &lt; 0.005.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_signal_gtex">
Past versions of 03_21_signal_gtex.pdf
</button>
</p>
<div id="fig-03_21_signal_gtex" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_signal_gtex.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="distribution-of-peak-windows" class="section level3">
<h3>Distribution of peak windows</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_21_Brain_peaks_in_group_window50kb.pdf" title="Figure: Distribution of peak group size in Brain and Lung." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_Brain_peaks_in_group_window50kb">
Past versions of 03_21_Brain_peaks_in_group_window50kb.pdf
</button>
</p>
<div id="fig-03_21_Brain_peaks_in_group_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_Brain_peaks_in_group_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/03_21_Lung_peaks_in_group_window50kb.pdf" title="Figure: Distribution of peak group size in Brain and Lung." width="40%" type="application/pdf" />
<p class="caption">
Figure: Distribution of peak group size in Brain and Lung.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_Lung_peaks_in_group_window50kb">
Past versions of 03_21_Lung_peaks_in_group_window50kb.pdf
</button>
</p>
<div id="fig-03_21_Lung_peaks_in_group_window50kb" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_Lung_peaks_in_group_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_21_Brain_peaks_in_group_w_cqtl_window50kb.pdf" title="Figure: cPeaks defined by (1) empirical p &lt; 0.005, (2) FDR &lt; 0.05 in peak groups in Brain and Lung." width="40%" type="application/pdf" />
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_Brain_peaks_in_group_w_cqtl_window50kb">
Past versions of 03_21_Brain_peaks_in_group_w_cqtl_window50kb.pdf
</button>
</p>
<div id="fig-03_21_Brain_peaks_in_group_w_cqtl_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_Brain_peaks_in_group_w_cqtl_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<embed src="asset/2024/03_21_Lung_peaks_in_group_w_cqtl_window50kb.pdf" title="Figure: cPeaks defined by (1) empirical p &lt; 0.005, (2) FDR &lt; 0.05 in peak groups in Brain and Lung." width="40%" type="application/pdf" />
<p class="caption">
Figure: cPeaks defined by (1) empirical p &lt; 0.005, (2) FDR &lt; 0.05
in peak groups in Brain and Lung.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_Lung_peaks_in_group_w_cqtl_window50kb">
Past versions of 03_21_Lung_peaks_in_group_w_cqtl_window50kb.pdf
</button>
</p>
<div id="fig-03_21_Lung_peaks_in_group_w_cqtl_window50kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_Lung_peaks_in_group_w_cqtl_window50kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li>The difference in number of cpeaks in each peak groups is huge
between using different ways to define cpeaks.</li>
</ul>
</div>
</div>
<div id="cacti-signal-comparison-with-univairate-signals-1"
class="section level2">
<h2>2. CACTI signal comparison with univairate signals</h2>
<p>To make a fair comparison between CACTI signals and their cpeaks, and
avoid false positives at the same time, I performed multiple testing
correction on the level of peaks and used fdr &lt; 0.05 to define
cwindows or cpeaks. Then I compar</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb.pdf" title="Figure: Compare CACTI signals to their signals using FDR threshold of 0.05 in Brain." width="40%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to their signals using FDR threshold of
0.05 in Brain.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb">
Past versions of 03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>CACTI cwindows overlap with most of the cpeaks.</p></li>
<li><p>Some CACTI cwindows are new signals containing no
cpeaks.</p></li>
<li><p>Some cpeaks are missed by CACTI windows, all containing only one
cpeak.</p></li>
<li><p>Similar story in Heart and Muscle.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb.pdf" title="Figure: Compare CACTI signals to their signals using FDR threshold of 0.05 in Lung." width="40%" type="application/pdf" />
<p class="caption">
Figure: Compare CACTI signals to their signals using FDR threshold of
0.05 in Lung.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb">
Past versions of 03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb.pdf
</button>
</p>
<div id="fig-03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/6c2f683f9e32bdc3aa9f01c78f2e5c03777d8767/docs/asset/2024/03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb.pdf" target="_blank">6c2f683</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>Only one significant cpeak under FDR &lt; 0.05. It is replicated
in a CACTI cwindow.</p></li>
<li><p>CACTI detects two other cwindows, containing no cpeaks.</p></li>
</ul>
</div>
<div id="cacti-signal-comparison-across-different-aspects"
class="section level2">
<h2>3. CACTI signal comparison across different aspects</h2>
<div id="across-tissues" class="section level3">
<h3>Across tissues</h3>
</div>
</div>
</div>
<div id="feb-22" class="section level1">
<h1>Feb 22</h1>
<div id="single-cell-analysis-terminated" class="section level2">
<h2>1. Single cell analysis (Terminated)</h2>
<div id="pco-on-rasqual-in-single-cells" class="section level3">
<h3>PCO on rasqual in single cells</h3>
</div>
<div id="acatskat-combined-rasqual-in-single-cells"
class="section level3">
<h3>ACAT/SKAT combined rasqual in single cells</h3>
<div id="lm-v.s.-acat_lm-v.s.-pco-previous-results-in-bulk"
class="section level4">
<h4>1. lm v.s. acat_lm v.s. pco (previous results in bulk)</h4>
<ul>
<li><p>lm v.s. acat_lm (to show the power of combination)</p></li>
<li><p>acat_lm v.s. pco (to show overall power)</p></li>
</ul>
</div>
<div id="lm-v.s.-acat_lm-v.s.-rasqual-v.s.-acat_rasqual-in-single-cell"
class="section level4">
<h4>2. lm v.s. acat_lm v.s. rasqual v.s. acat_rasqual (in single
cell)</h4>
<ul>
<li><p>lm v.s. acat_lm (to show the power of combination)</p></li>
<li><p>rasqual v.s. acat_rasqual (to show the power of
combination)</p></li>
</ul>
</div>
<div
id="try-other-tests-in-the-simulation-to-understand-the-source-of-inflation-of-estimated-sigma-in-pco"
class="section level4">
<h4>3. try other tests in the simulation to understand the source of
inflation of estimated sigma in PCO</h4>
<ul>
<li>z test</li>
</ul>
</div>
</div>
</div>
<div id="across-tissues-1" class="section level2">
<h2>2. Across tissues</h2>
</div>
<div id="across-marks" class="section level2">
<h2>3. Across marks</h2>
</div>
</div>
<div id="feb-15" class="section level1">
<h1>Feb 15</h1>
<div id="simulations-to-evaluate-pco-on-non-zscore-tests"
class="section level2">
<h2>1. Simulations to evaluate PCO on non-zscore tests</h2>
<div id="ways-to-estimate-sigma" class="section level3">
<h3>Ways to estimate Sigma</h3>
<ul>
<li><p>P-value of test statistic from a non-zscore test is converted to
a z-score by inverse-normal transformation.</p></li>
<li><p>Sigma of the converted z-scores is the corresponding
cov.</p></li>
<li><p>Run PCO on the converted z-scores.</p></li>
</ul>
</div>
<div id="null-simulations" class="section level3">
<h3>Null simulations</h3>
<ul>
<li><p><span class="math inline">\(Y\)</span> from residualized
expression matrix of <span class="math inline">\(K=101\)</span> gene
module.</p></li>
<li><p><span class="math inline">\(G\)</span> from binomial
distribution, with maf being 0.2, 1k snps.</p></li>
<li><p><span class="math inline">\(G\)</span> is independent with <span
class="math inline">\(Y\)</span>.</p></li>
<li><p>Run a non-zscore test, e.g. LRT, between every pairs of genes and
snps.</p></li>
<li><p>Obtain chi-square test statistics and p-value.</p></li>
<li><p>Convert p to z-score by <span class="math inline">\(z_{convert} =
\Phi^{-1}(p)\)</span>, where <span
class="math inline">\(\Phi^{-1}\)</span> is the inverse cdf of standard
normal distribution.</p></li>
<li><p>Estimate <span class="math inline">\(\Sigma_{convert} =
cov(z_{convert})\)</span></p></li>
<li><p>Run PCO on <span class="math inline">\(z_{convert}\)</span> using
<span class="math inline">\(\Sigma_{convert}\)</span>.</p></li>
</ul>
</div>
<div id="pco-p-value-inflation-under-null-distribution"
class="section level3">
<h3>PCO p-value inflation under null distribution</h3>
<p>To look at if PCO p-values are inflated under the null simulations
using <span class="math inline">\(z_{convert}\)</span> and <span
class="math inline">\(\Sigma_{convert}\)</span> -</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_14_pco_nullp_int.pdf" title="Figure: PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}." width="40%" type="application/pdf" />
<p class="caption">
Figure: PCO p-value inflation under null distribution using z_{convert}
and Sigma_{convert}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_14_pco_nullp_int">
Past versions of 02_14_pco_nullp_int.pdf
</button>
</p>
<div id="fig-02_14_pco_nullp_int" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87402c7054fa6d1127a4c4cb79a57153472fdf89/docs/asset/2024/02_14_pco_nullp_int.pdf" target="_blank">87402c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Problem</strong> -</p>
<ul>
<li>The way Zhonghua mentioned should be <span
class="math inline">\(\Phi^{-1}(p)\)</span>, where <span
class="math inline">\(\Phi\)</span> is the cdf of standard normal
distribution. However, it might not work, as while a small p is
converted to a positively large z, a large p can be falsely converted to
a negatively large z. Therefore, some non signals with large p can be a
false signal and lead to inflation. We want both the converted test to
be normally distributed and correspond to magnitude of p-values as
well.</li>
</ul>
</div>
<div id="other-ways-to-convert-p" class="section level3">
<h3>Other ways to convert p</h3>
<p>I tried other ways to convert p-values to “z-scores”.</p>
<ul>
<li>Convert to a chi-square test, then take square root and assign signs
randomly.</li>
<li>Rank-based inverse-normal transformation.</li>
</ul>
<p>They all have inflations as well.</p>
<div class="figure" style="text-align: center">
<embed src="asset/2024/02_14_pco_nullp_other.pdf" title="Figure: Other normal transformation. PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}." width="40%" type="application/pdf" />
<p class="caption">
Figure: Other normal transformation. PCO p-value inflation under null
distribution using z_{convert} and Sigma_{convert}.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-02_14_pco_nullp_other">
Past versions of 02_14_pco_nullp_other.pdf
</button>
</p>
<div id="fig-02_14_pco_nullp_other" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/87402c7054fa6d1127a4c4cb79a57153472fdf89/docs/asset/2024/02_14_pco_nullp_other.pdf" target="_blank">87402c7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
</div>
<div id="jan-25" class="section level1">
<h1>Jan 25</h1>
<div id="saige-qtl-saige-qtl-pco" class="section level2">
<h2>1. SAIGE-QTL &amp; SAIGE-QTL-PCO</h2>
<div id="how-does-saige-qtl-work" class="section level3">
<h3>How does SAIGE-QTL work?</h3>
<ul>
<li><p>Step 1: fitting the null Poisson mixed model</p></li>
<li><p>Step 2: performing single-variant association tests</p></li>
<li><p>Step 3: obtaining gene-level p-values using the ACAT
test</p></li>
</ul>
</div>
<div id="questions-to-be-confirmed-before-running-saige-qtl"
class="section level3">
<h3>Questions to be confirmed before running SAIGE-QTL</h3>
<ul>
<li>Is association test of SAIGE-QTL (score test statistic) linear to Y?
It has to be so in order to calculate the correlation matrix of the
tests using correlation matrix of Y’s.</li>
</ul>
</div>
<div id="run-saige-qtl" class="section level3">
<h3>Run SAIGE-QTL</h3>
<ul>
<li>How is the number of cpeaks by SAIGE-QTL compared to RASQUAL and
scPME?</li>
</ul>
</div>
</div>
<div id="using-crd-defined-windows" class="section level2">
<h2>2. Using CRD-defined windows</h2>
</div>
<div id="using-two-datasets" class="section level2">
<h2>3. Using two datasets</h2>
</div>
</div>
<div id="jan-18" class="section level1">
<h1>Jan 18</h1>
<div id="distance-from-lead-snp-to-cpeaks-of-histone-marks-in-bulk-data"
class="section level2">
<h2>1. Distance from lead SNP to cpeaks of histone marks in bulk
data</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf" title="Figure: cpeaks by QTLTools across cis regions." width="40%" type="application/pdf" />
<p class="caption">
Figure: cpeaks by QTLTools across cis regions.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_dis_leadsnp_cpeak_histone_marks_cis500kb">
Past versions of 01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf
</button>
</p>
<div id="fig-01_18_dis_leadsnp_cpeak_histone_marks_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Change of cpeaks by QTLTools across various cis regions for bulk
histone marks</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_cpeak_qtltools_cis_reg_bulk.pdf" title="Figure: Change of cpeaks by QTLTools across various cis regions for bulk histone marks." width="40%" type="application/pdf" />
<p class="caption">
Figure: Change of cpeaks by QTLTools across various cis regions for bulk
histone marks.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_cpeak_qtltools_cis_reg_bulk">
Past versions of 01_18_cpeak_qtltools_cis_reg_bulk.pdf
</button>
</p>
<div id="fig-01_18_cpeak_qtltools_cis_reg_bulk" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_cpeak_qtltools_cis_reg_bulk.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="compare-our-cqtls-to-activity-cqtls-in-delaunea-paper"
class="section level2">
<h2>2. Compare our cQTLs to activity cQTLs in Delaunea paper</h2>
<div id="how-acrdqtls-were-defined" class="section level3">
<h3>How aCRDQTLs were defined?</h3>
<ul>
<li><p>LCL.aCRDQTLs - QTLs for the activity of CRDs in LCLs.</p></li>
<li><p>The activity of CRDs are defined as the mean activity of the
peaks it contains.</p></li>
<li><p>6157 significant aCRDs (out of 12583 total CRDs) at 5%
FDR.</p></li>
</ul>
</div>
<div id="signal-comparison-cwindowsccrds" class="section level3">
<h3>Signal comparison cwindows/cCRDs</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Comparison of cwindows and aCRDQTLs." width="40%" type="application/pdf" />
<p class="caption">
Figure: Comparison of cwindows and aCRDQTLs.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<p>CACTI v.s. cCRDs -</p>
<ul>
<li><p>Many signals are missed by CACTI.</p>
<ul>
<li>Those aCRDs overlap multiple windows, while missed cwindows overlap
mostly one aCRD.</li>
</ul></li>
<li><p>Increase window size? 50kb, similar case.</p></li>
<li><p>Does the comparison even make sense given CRD (calculated using
three histone marks by ChIP-seq) v.s. ATAC-seq?</p></li>
</ul>
</div>
<div id="enrichment-of-ccrds-in-cwindows" class="section level3">
<h3>Enrichment of cCRDs in cwindows</h3>
<p><strong>Questions to be asked -</strong></p>
<ol style="list-style-type: decimal">
<li><p>If being a cwindow more likely be a aCRDQTL?</p></li>
<li><p>Restrict to cwindow overlap with CRDs</p></li>
</ol>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: If being a cwindow more likely be a aCRDQTL? If being a cwindow that overlaps with CRDs more likely be a aCRDQTL?" width="40%" type="application/pdf" />
<p class="caption">
Figure: If being a cwindow more likely be a aCRDQTL? If being a cwindow
that overlaps with CRDs more likely be a aCRDQTL?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><strong>Observations -</strong></p>
<ul>
<li><p>cwindows are enriched for aCRDQTLs, no matter for windows overlap
with a CRD or not.</p></li>
<li><p>The enrichment and OR are more significant than the enrichment of
cwindows for CRDs.</p></li>
</ul>
<p>To-do’s -</p>
<ul>
<li><p>Apply to previous other cases to see if there is significant
enrichment.</p></li>
<li><p>Change the name of cQTLs to something like cWindowQTLs?</p></li>
</ul>
</div>
</div>
<div
id="another-angle-to-explain-power-difference-between-cacti-and-rasqual---rasqual-mostly-captures-allelic-effects"
class="section level2">
<h2>3. Another angle to explain power difference between CACTI and
RASQUAL - RASQUAL mostly captures allelic effects?</h2>
<ul>
<li><p>No such information in RASQUAL output.</p></li>
<li><p>Previously, I said RASQUAL uses 10kb as cis window because the
SNP should be within the feature to have aligned reads overlap with the
SNP. This is wrong. RASQUAL can take larger cis window outside the
feature. They combined the pop effect and allelic effect for any SNPs.
See their paper.</p></li>
<li><p>So, to have a fair power comparsion, we still need to think of a
way to harmonize the multiple testing burden, since QTLTools have almost
three times more signals at 10kb cis region than 500kb cis
region.</p></li>
</ul>
</div>
</div>
<div id="jan-11" class="section level1">
<h1>Jan 11</h1>
<div id="some-insights" class="section level2">
<h2>0. some insights</h2>
<ul>
<li><p>Why cis region used by RASQUAL is so small?</p>
<p>Because in order to calculate the allelic effect, snps should be
located inside the sequenced feature within where the feature reads are
counted. Otherwise, RASQUAL effects are just between-individual signals
not allele-specific signals. Therefore, RASQUAL cis regions should be
the feature length on average.</p></li>
</ul>
</div>
<div id="pco-using-rasqua-sum-stats" class="section level2">
<h2>1. PCO using RASQUA sum stats</h2>
<div id="why-1" class="section level3">
<h3>Why?</h3>
<p>Current CACTI uses sum stats by QTLTools (uni-pop). We wanted to show
the advantage of aggregating muliple peaks, so to fairly compare with
RASQUAL which uses allelic effects, we want to use PCO on RASQUAL sum
stats.</p>
<p><strong>To-do’s -</strong></p>
<ul>
<li><p>Check PCO on RASQUAL p-values (not from z-scores), if it’s
feasible.</p></li>
<li><p>Use PCO to combine RASQUAL p’s.</p></li>
</ul>
</div>
<div id="why-pco-wont-work-to-combine-rasqual-ps"
class="section level3">
<h3>Why PCO won’t work to combine RASQUAL p’s?</h3>
<ul>
<li><p>Six tests of PCO combine p-values, not depending on the test
statistics.</p></li>
<li><p>But the distribution of six tests depends on <span
class="math inline">\(\Sigma\)</span>, which is calculated as the
correlation among residual y’s, is actually the covariance matrix of the
test statistic corresponding to the p-values (z-scores) across y’s. The
equivalence depends on the fact that the test statistic is linear to y
(see equation 2 in <a
href="https://onlinelibrary-wiley-com.proxy.uchicago.edu/doi/full/10.1111/biom.12735">paper</a>.</p></li>
<li><p>The issue is RASQUAL test statistic is not linear to y, which
make the calculation of <span class="math inline">\(\Sigma\)</span>
impossible. RASQUAL p-value for a pair of snp and feature comes from LRT
testing for <span class="math inline">\(\pi=0.5\)</span>, where <span
class="math inline">\(\pi\)</span> is the metric for cis regulatory
effect parameter (see “Hypothesis testing for inference of QTLs” in <a
href="https://www.nature.com/articles/ng.3467#Sec13">paper</a>). We need
to know the correlation among LRT test statistics of a snp across peaks,
but we can’t. It is difference case than where the test statistic is OLS
estimate and we can easily derive the correlations.</p></li>
</ul>
<blockquote>
<p>Under the null hypothesis, all parameters except π are estimated for
each feature independently, whereas π is set to 0.5, and we use a
likelihood-ratio test to compare the null and alternative hypotheses for
each SNP-feature combination using the <span
class="math inline">\(\chi^2\)</span> distribution with 1 degree of
freedom (for <span class="math inline">\(\pi\)</span>).</p>
</blockquote>
</div>
</div>
<div id="effect-of-the-choice-of-cis-region-on-cpeaks-by-qtltools"
class="section level2">
<h2>2. Effect of the choice of cis region on #cpeaks by QTLTools</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_cpeak_qtltools_cis_reg.pdf" title="Figure: cpeaks by QTLTools across cis regions." width="40%" type="application/pdf" />
<p class="caption">
Figure: cpeaks by QTLTools across cis regions.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_cpeak_qtltools_cis_reg">
Past versions of 01_11_cpeak_qtltools_cis_reg.pdf
</button>
</p>
<div id="fig-01_11_cpeak_qtltools_cis_reg" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_cpeak_qtltools_cis_reg.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li><p>Using 500kb as cis region in previous results should have less
signals than using shorter cis regions.</p></li>
<li><p>Lead SNPs are very close to cpeaks across various cis
regions.</p></li>
</ul></li>
</ul>
</div>
<div id="use-qtltools-cpeaks-for-signal-to-noise-ratio-plot"
class="section level2">
<h2>3. Use QTLTools cpeaks for signal-to-noise ratio plot</h2>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" title="Figure: Signal to noise ratio of missed &amp; replicated cwindows compared to QTLTools and RASQUAL." width="40%" type="application/pdf" />
<p class="caption">
Figure: Signal to noise ratio of missed &amp; replicated cwindows
compared to QTLTools and RASQUAL.
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb">
Past versions of
01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>I added the third panel, where I compared the signal to noise ratio
of missed &amp; replicated cwindows compared to RASQUAL and the number
of signals (cpeaks) in a window was from QTLTools. To compare with the
first panel.</p>
<ul>
<li><p>Observations</p>
<ul>
<li><p>The missed &amp; replicated cwindows compared to RASQUAL have
even fewer cpeaks in the windows by QTLTools. This indicate that RASQUAL
lends power from allelic effects.</p></li>
<li><p>The power difference between RASQUAL and CACTI comes from two
aspects - aggregating multiple peaks and combining allelic
effects.</p></li>
</ul></li>
</ul>
</div>
<div id="characterization-of-cacti-signals-in-bulk-in-terms-of-crd"
class="section level2">
<h2>5. Characterization of CACTI signals in bulk in terms of CRD</h2>
<div id="rational" class="section level3">
<h3>Rational</h3>
<p>To check if CACTI finds cwindows of co-regulated peaks, we
investigate two questions -</p>
<ul>
<li><p>Do cwindows more likely overlap with any CRDs than
non-cwindows?</p></li>
<li><p>Are cwindows more likely with same CRDs than
non-cwindows?</p></li>
</ul>
<p>I checked cwindows by CACTI-P for mark H3K27ac and CACTI-S for mark
H3K36me.</p>
<p>(12583 CRDs on hg38. 12433 converted to hg19. 150 CRDs deleted.)</p>
</div>
<div id="cacti-p-for-mark-h3k27ac" class="section level3">
<h3>CACTI-P for mark H3K27ac</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" title="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb">
Past versions of
01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" title="Figure: Are cwindows more likely with same CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Are cwindows more likely with same CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb">
Past versions of
01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
</div>
<div id="cacti-s-for-mark-h3k36me3" class="section level3">
<h3>CACTI-S for mark H3K36me3</h3>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" title="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new">
Past versions of
01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div class="figure" style="text-align: center">
<embed src="asset/2024/01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" title="Figure: Are cwindows more likely with same CRDs than non-cwindows?" width="40%" type="application/pdf" />
<p class="caption">
Figure: Are cwindows more likely with same CRDs than non-cwindows?
</p>
</div>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new">
Past versions of
01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf
</button>
</p>
<div
id="fig-01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new"
class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/fc2b1c5594509819e9f0a262fca9b3e3e5ec7cae/docs/asset/2024/01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf" target="_blank">fc2b1c5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-01-18
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li><p>Observations</p>
<ul>
<li>No significant enrichment.</li>
</ul></li>
</ul>
</div>
</div>
<div id="use-50kb-window-to-be-consistent-with-previous-bulk-results"
class="section level2">
<h2>6. Use 50kb window to be consistent with previous bulk results?</h2>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.7.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.11       highr_0.10        compiler_4.2.3    pillar_1.9.0     
 [5] bslib_0.5.0       later_1.3.1       git2r_0.32.0      jquerylib_0.1.4  
 [9] tools_4.2.3       getPass_0.2-2     digest_0.6.33     jsonlite_1.8.7   
[13] evaluate_0.21     lifecycle_1.0.3   tibble_3.2.1      pkgconfig_2.0.3  
[17] rlang_1.1.1       cli_3.6.1         rstudioapi_0.15.0 yaml_2.3.7       
[21] xfun_0.39         fastmap_1.1.1     httr_1.4.6        stringr_1.5.0    
[25] knitr_1.43        fs_1.6.2          vctrs_0.6.3       sass_0.4.6       
[29] rprojroot_2.0.3   glue_1.6.2        R6_2.5.1          processx_3.8.2   
[33] fansi_1.0.4       rmarkdown_2.23    callr_3.7.3       magrittr_2.0.3   
[37] whisker_0.4.1     ps_1.7.5          promises_1.2.0.1  htmltools_0.5.5  
[41] httpuv_1.6.11     utf8_1.2.3        stringi_1.7.12    cachem_1.0.8     </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
