<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Trans</title>

<script src="site_libs/header-attrs-2.10/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>








<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Trans</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-01-03
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree68ded6fafda2c750d6daf577fa3019160d657b0dtargetblank68ded6fa"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/liliw-w/GradLog/tree/68ded6fafda2c750d6daf577fa3019160d657b0d" target="_blank">68ded6f</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtree68ded6fafda2c750d6daf577fa3019160d657b0dtargetblank68ded6fa" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version <a href="https://github.com/liliw-w/GradLog/tree/68ded6fafda2c750d6daf577fa3019160d657b0d" target="_blank">68ded6f</a>. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/test_log.Rmd
    Ignored:    code/.DS_Store
    Ignored:    data/.DS_Store
    Ignored:    output/.DS_Store

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/Trans.Rmd</code>) and HTML (<code>docs/Trans.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/51b1a4cc02e4dd63c9a84c02ea8c597e171bc5ca/docs/Trans.html" target="_blank">51b1a4c</a>
</td>
<td>
Lili Wang
</td>
<td>
2021-01-04
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c4d150c228a9ca1724803fc27b2f2bf3c5761d9e/docs/Trans.html" target="_blank">c4d150c</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d13875e9b2538a0a2d5d73b62b6c5d49684cea95/docs/Trans.html" target="_blank">d13875e</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5bc34ec29c863fe52b0cbea6219fd45fae83a381/docs/Trans.html" target="_blank">5bc34ec</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/8fe41846bceb86adb07e5260904835194eba55ea/analysis/Trans.Rmd" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8fe41846bceb86adb07e5260904835194eba55ea/docs/Trans.html" target="_blank">8fe4184</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-11-03
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2a0480828f09a7687b7457d1ad1e9b70d42a1f56/docs/Trans.html" target="_blank">2a04808</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-19
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/c83022290cba1ef90ba05472cf91682f13abbfba/analysis/Trans.Rmd" target="_blank">c830222</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-19
</td>
<td>
Update all code used for breast cancer TCGA data.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c83022290cba1ef90ba05472cf91682f13abbfba/docs/Trans.html" target="_blank">c830222</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-19
</td>
<td>
Update all code used for breast cancer TCGA data.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5ccca4c0b00867eaa521792cd3d47b20667ced29/docs/Trans.html" target="_blank">5ccca4c</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2c4d88192654ecab6d259e023781d5667bc5b32b/analysis/Trans.Rmd" target="_blank">2c4d881</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
rewrite the code chunk using the ‘code’ parameter
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/a62f2e47917cf936538734c85622907f123c1dad/docs/Trans.html" target="_blank">a62f2e4</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/eb3f712312c98162d5503c90ca5b1ed873c6aa0b/analysis/Trans.Rmd" target="_blank">eb3f712</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
test using cat to display code
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f05b42c12f9068d719e5a8bc64cd555a1c2c2e3a/docs/Trans.html" target="_blank">f05b42c</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f9b0a151e7e462d7a379d1e88c8d656b2fd1043b/analysis/Trans.Rmd" target="_blank">f9b0a15</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-17
</td>
<td>
Rewrite make.geno to convert the genotype binary files to txt
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/5a86aaf54e086a519c08f801248b539a1e825c29/docs/Trans.html" target="_blank">5a86aaf</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/4382474f6df22a49cc2915c2de49d8a62ce35fa1/docs/Trans.html" target="_blank">4382474</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/f79b80d03d24c2874a7935558182a8c35ebb4ea5/docs/Trans.html" target="_blank">f79b80d</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d7821adbca951cc2a8ff7ea1a3ddc75b150486be/analysis/Trans.Rmd" target="_blank">d7821ad</a>
</td>
<td>
Lili Wang
</td>
<td>
2020-10-14
</td>
<td>
wflow_publish("./analysis/*.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="outine" class="section level2">
<h2>Outine</h2>
<p><embed src="outline.pdf" style="width:100.0%;height:5in" /></p>
</div>
<div id="preprocess-data" class="section level2">
<h2>Preprocess data</h2>
<div id="convert-plink-files-of-genotype-to-.txt" class="section level3">
<h3>Convert plink files of genotype to .txt</h3>
<pre class="r"><code>module load plink
module load R/3.6.1

cd /scratch/midway2/liliw1/

# dataset DGN: dir_plink=/project2/xuanyao/data/DGN
# dataset DGN: dir_txt=/project2/xuanyao/data/DGN/txt_file_of_DGN
# dataset DGN: dir_script=/project2/xuanyao/data/DGN
# dataset DGN: awk &#39;BEGIN{print &quot;id\tchr\tpos&quot;}{printf(&quot;%s\tchr%s\t%s\n&quot;,$2,$1,$4)}&#39;

dir_plink=/project2/xuanyao/xuanyao/cancer_gbat/breast_cancer
dir_txt=/project2/xuanyao/llw/breastcancerTCGA/txt_file
dir_script=/scratch/midway2/liliw1/TCGA


for CHR in `seq 22`
do
# convert bfiles to .raw files
plink \
--bfile $dir_plink/genotype/chr${CHR}_QCed \
--recodeA \
--out $dir_txt/chr${CHR}

# extract genotype meta
awk \
&#39;BEGIN{print &quot;id\tchr\tpos&quot;}{key=sprintf(&quot;%s:%s&quot;, $1, $4);printf(&quot;%s\tchr%s\t%s\n&quot;,key,$1,$4)}&#39; \
$dir_plink/genotype/chr${CHR}_QCed.bim &gt; \
$dir_txt/chr${CHR}_genotype_meta.txt

# convert genotype to txt file
Rscript --no-restore --no-save $dir_script/make.geno.R $CHR

done</code></pre>
<p><code>make.geno.R</code></p>
<pre class="r"><code>rm(list = ls())

args = commandArgs(trailing = T)
chr = args[1]

# dataset DGN: dir_txt = &quot;/project2/xuanyao/data/DGN/txt_file_of_DGN/&quot;
# dataset DGN: colnames(geno) = unlist(lapply(strsplit(colnames(genotype)[7:ncol(genotype)],&quot;_&quot;),&quot;[&quot;,1))

dir_txt = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file&quot;
genotype = read.table(paste0(dir_txt, &quot;chr&quot;,chr,&quot;.raw&quot;),
                 header = T, check.names = F)
snp.meta = read.table(paste0(dir_txt, &quot;chr&quot;,chr,&quot;_genotype_meta.txt&quot;),
                      header = TRUE, stringsAsFactors = FALSE)

geno = as.matrix(genotype[,7:ncol(genotype)])
rownames(geno) = as.character(genotype$IID)
colnames(geno) = snp.meta$id


write.table(t(geno),
            paste0(dir_txt, &quot;chr&quot;, chr,&quot;.genotype.matrix.eqtl.txt&quot;),
            sep = &quot;\t&quot;, quote = FALSE)

cat(paste0(&quot;chr&quot;, chr, &quot; is done!&quot;))</code></pre>
</div>
<div id="regress-out-covariates-from-expression-data" class="section level3">
<h3>Regress out covariates from expression data</h3>
<pre class="r"><code>rm(list = ls())

dir_data = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file&quot;
setwd(dir_data)

extract_residual &lt;- function(y, x){
  return(lm(y ~ x)$residuals)
}


# import expression and covariates data
load(&quot;./tumor.rdata&quot;)
cov_exp = read.table(&quot;./tumor.exp_pc.txt&quot;,
                     header = TRUE, row.names = NULL, stringsAsFactors = FALSE)
cov_snp = read.table(&quot;./tumor.snp_pc.txt&quot;,
                     header = FALSE, row.names = NULL, stringsAsFactors = FALSE)

colnames(ex) = gnames


# regress out covariates
ex_cov_regressed = apply(ex, 2, function(y) extract_residual(y, as.matrix(cbind(cov_exp, cov_snp))))
saveRDS(ex_cov_regressed, &quot;./ex_var_regressed.rds&quot;)

print(&quot;Done!&quot;)</code></pre>
</div>
<div id="remove-low-quality-genes" class="section level3">
<h3>Remove low-quality genes</h3>
<p>Remove seudogenes, genes with mappability<span class="math inline">\(&lt;0.9\)</span>, and gene pairs with cross-mappability higher than 1.</p>
<p>The code here is for the breast cancer TCGA data. So the genes on chromosomes X and Y are also removed.</p>
<pre class="r"><code>rm(list = ls())

options(stringsAsFactors = FALSE)

# dataDGN: dir_data =
# dataDGN: gene.name = colnames(datExpr)

dir_data = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file&quot;
setwd(dir_data)

### Data preparation ###
datExpr = readRDS(&quot;./ex_var_regressed.rds&quot;)
gene.name = sapply(colnames(datExpr), function(x) strsplit(x, &quot;\\|&quot;)[[1]][1])

# remove genes on chromosome X &amp; Y
gene.meta = read.table(&quot;./tumor.gene_pos.txt&quot;,
                       header = TRUE,
                       stringsAsFactors = F)
rownames(gene.meta) = gene.meta$gene
ind_chrX = gene.meta[colnames(datExpr), &quot;chr&quot;] == &quot;chrX&quot; | gene.meta[colnames(datExpr), &quot;chr&quot;] == &quot;chrY&quot;

# remove pseudogenes
pseudogenes = read.table(&quot;./pseudogenes.txt&quot;, sep =&quot;\t&quot;, stringsAsFactors = F)
ind_pseudo = gene.name %in% unique(pseudogenes$V1)

# remove low mappability genes
mappability = read.table(&quot;./mappability.txt&quot;, sep =&quot;\t&quot;, stringsAsFactors = F, header = T)
ind_map = gene.name %in% unique(mappability$gene_name)

# remove cross-mappable genes
cross_mappability = readRDS(&quot;./cross_mappable_genes.rds&quot;)
cross_map_genes = unique(names(cross_mappability[!is.na(cross_mappability)]))
ind_cross_map = gene.name %in% cross_map_genes

ind_remove = ind_chrX | ind_pseudo | ind_map | ind_cross_map

sum(ind_chrX)
sum(ind_pseudo)
sum(ind_map)
sum(ind_cross_map)
sum(ind_remove)


write.table(as.matrix(data.frame(&quot;gene&quot; = colnames(datExpr),
                                 &quot;gene.name&quot; = gene.name,
                                 &quot;ind_remove&quot; = ind_remove,
                                 &quot;ind_chrX&quot; = ind_chrX,
                                 &quot;pseudo&quot; = ind_pseudo,
                                 &quot;lowmap&quot; = ind_map,
                                 &quot;crossmap&quot; = ind_cross_map)),
            &quot;genes_rm_info.txt&quot;,
            row.names = FALSE, col.names = TRUE, quote = FALSE)</code></pre>
</div>
</div>
<div id="coexpressed-gene-modules" class="section level2">
<h2>Coexpressed gene modules</h2>
<p>Cluster the co-expressed genes by WGCNA.</p>
<pre class="r"><code>rm(list = ls())

library(WGCNA)

dir_data = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file&quot;
setwd(dir_data)

# Data preparation
datExpr = readRDS(&quot;./ex_var_regressed.rds&quot;)
rm_info = read.table(&quot;./genes_rm_info.txt&quot;,
                     header = TRUE, row.names = NULL,
                     stringsAsFactors = FALSE)
datExpr = datExpr[, !rm_info$ind_remove]


# Run WGCNA
### Parameter specification ###
minModuleSize = 30
MEDissThres = 0.15
if_plot_adjacency_mat_parameter_selection = F
if_plot_only_tree = F
if_plot_color_and_tree = F
if_plot_eigengene_heatmap_tree = F
if_heatmap_of_network = T


### Step1: network construction ###
# determine the paramter in adjacency function: pickSoftThreshold() OR pickHardThreshold()
powers = c(c(1:10), seq(from = 12, to=20, by=2))
sft = pickSoftThreshold(datExpr, powerVector = powers, verbose = 5)
softPower = sft$powerEstimate

# network construction
adjacency = adjacency(datExpr, power = softPower)
TOM = TOMsimilarity(adjacency)
dissTOM = 1-TOM


### Step2: module detection ###
# tree construction using hierarchical clustering based on TOM
geneTree = hclust(as.dist(dissTOM), method = &quot;average&quot;)

# branch cutting using dynamic tree cut
dynamicMods = cutreeDynamic(dendro = geneTree, distM = dissTOM,
                            deepSplit = 2, pamRespectsDendro = FALSE,
                            minClusterSize = minModuleSize)
dynamicColors = labels2colors(dynamicMods)

# eigene genes
MEList = moduleEigengenes(datExpr, colors = dynamicColors)
MEs = MEList$eigengenes

# Call an automatic merging function
merge = mergeCloseModules(datExpr, dynamicColors, cutHeight = MEDissThres, verbose = 3)
mergedColors = merge$colors
mergedMEs = merge$newMEs

moduleLabels = match(mergedColors, c(&quot;grey&quot;, standardColors(100)))-1
names(moduleLabels) = colnames(datExpr)

# Save results
result = list(moduleColors = mergedColors,
              moduleLabels = moduleLabels,
              MEs = mergedMEs,
              old_moduleColors = dynamicColors,
              old_moduleLabels = dynamicMods,
              old_MEs = MEs,
              geneTree = geneTree,
              ind_remove = rm_info$ind_remove)
saveRDS(result, file = &quot;./coexp.module.rds&quot;)</code></pre>
</div>
<div id="calculate-zscores" class="section level2">
<h2>Calculate zscores</h2>
<p><del>Here is the code of how I compute null zscores for module <span class="math inline">\(i\)</span> and chromosome <span class="math inline">\(j\)</span>, using randomly permuted samples. Calculating observed zscores (zscores based on the original real data) is similar, only need to remove line <code>genotype = genotype[ind.perm, ]</code>.</del></p>
<p>Update the code to calculate the <strong>observed zscores</strong> from <strong>TCGA data</strong>.</p>
<pre class="r"><code>rm(list = ls())

parameter = commandArgs(trailingOnly = T)
module_id = parameter[1]
chr = parameter[2]

setwd(&quot;/scratch/midway2/liliw1/TCGA/&quot;)

# dataset DGN: dir_data = &quot;/project2/xuanyao/data/DGN/&quot;
# dataset DGN: gene_meta = read.table(paste0(dir_data, &quot;expression_meta_hg19.txt&quot;), ...)
# dataset DGN: gene_module = readRDS(file = paste0(dir_data, &quot;DGN_coexpressed_gene_network.rds&quot;))
# dataset DGN: header = ifelse(chr==1, TRUE, FALSE) when reading genotype


library(foreach)
library(doParallel)

# data input
dir_data = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file/&quot;
datExpr = readRDS(paste0(dir_data, &quot;ex_var_regressed.rds&quot;))
gene_meta = read.table(paste0(dir_data, &quot;tumor.gene_pos.txt&quot;),
                      sep = &#39;\t&#39;,
                      header = T,
                      stringsAsFactors = F)
gene_module = readRDS(file = paste0(dir_data, &quot;coexp.module.rds&quot;))
genotype = read.table(paste0(dir_data, &quot;chr&quot;, chr, &quot;.genotype.matrix.eqtl.txt&quot;),
                      header = TRUE, row.names=NULL,
                      stringsAsFactors = F)
genotype = genotype[!duplicated(genotype[, 1]), ]
rownames(genotype) = genotype[, 1]
genotype = t(genotype[, -1])


# Use only the genes in the module and have postion info in the meta file
gene_in_cluster = data.frame(&quot;gene_name&quot;=names(gene_module$moduleLabels)[gene_module$moduleLabels == module_id], stringsAsFactors = F)
gene_w_pos = merge(gene_in_cluster, gene_meta, by.x = 1, by.y = 1)
exp.data = datExpr[, gene_w_pos$gene_name]
print(&quot;Input part is done!&quot;)

# run parallel on snps
cores=(Sys.getenv(&quot;SLURM_NTASKS_PER_NODE&quot;))
cl = makeCluster(as.numeric(cores), outfile=&quot;&quot;)
registerDoParallel(cl)

M = ncol(genotype); K = ncol(exp.data); N = nrow(exp.data)
#ind.perm = sample(1:N); genotype = genotype[ind.perm, ]
chunk.size = M %/% as.numeric(cores)
system.time(
  res.all &lt;- foreach(i=1:cores, .combine=&#39;rbind&#39;) %dopar%
    {
      res = matrix(nrow = chunk.size, ncol = K,
                   dimnames = list(colnames(genotype)[((i-1)*chunk.size+1):(i*chunk.size)], colnames(exp.data)))

      for(j in 1:chunk.size){
        snp = genotype[, j + (i-1)*chunk.size]
        res[j, ] = apply(exp.data, 2, function(x) summary(lm(x~snp))$coefficients[2, 3] )

        if(j %% 100 == 0){print(j+(i-1)*chunk.size)}
      }
      res
    }
)
stopCluster(cl)
print(&quot;Parallel part is done!&quot;)

# Run PCO on snps left in the parallel part
Nsnps_left = M %% as.numeric(cores)
Nsnps_done = (chunk.size)*as.numeric(cores)
if(Nsnps_left != 0){
  res = matrix(nrow = Nsnps_left, ncol = K,
               dimnames = list(colnames(genotype)[Nsnps_done+1:(Nsnps_left)], colnames(exp.data)) )

  for(j in 1:Nsnps_left){
    snp = genotype[, j + Nsnps_done]
    res[j, ] = apply(exp.data, 2, function(x) summary(lm(x~snp))$coefficients[2, 3] )

    print(Nsnps_done+j)
  }

  res.all = rbind(res.all, res)
}
print(&quot;All part is done!&quot;)


saveRDS(res.all, file = paste0(&quot;./z/z.module&quot;, module_id, &quot;.chr&quot;, chr, &quot;.rds&quot;))</code></pre>
</div>
<div id="calculate-pvalues" class="section level2">
<h2>Calculate pvalues</h2>
<p>Here is the code using PCO to calculate pvalues. This uses only PC’s with eigenvalue <span class="math inline">\(\lambda &gt; 1\)</span>. Modified PCO code in <code>./script/*</code>.</p>
<pre class="r"><code>rm(list = ls())

parameter = commandArgs(trailingOnly = T)
module_id = parameter[1]
chr = parameter[2]
p.method = parameter[3] #&quot;truncPCO&quot; #&quot;PCO&quot;


setwd(&quot;/scratch/midway2/liliw1/TCGA/&quot;)

# dataset DGN: dir_data = &quot;/project2/xuanyao/data/DGN/txt_file_of_DGN/&quot;
# dataset DGN: gene_module = readRDS(file = paste0(dir_data, &quot;DGN_coexpressed_gene_network.rds&quot;))
# dataset DGN: gene_meta = read.table(paste0(dir_data, &quot;expression_meta_hg19.txt&quot;), ...)


source(paste0(&quot;./script/ModifiedPCOMerged.R&quot;))
source(paste0(&quot;./script/liu.r&quot;))
source(paste0(&quot;./script/liumod.R&quot;))
source(paste0(&quot;./script/davies.R&quot;))
source(paste0(&quot;./script/SigmaMetaEstimate.R&quot;))
library(foreach)
library(doParallel)
library(mvtnorm)
library(MPAT)


# zscore input
z_mat = readRDS(file = paste0(&quot;./z/z.module&quot;, module_id, &quot;.chr&quot;, chr, &quot;.rds&quot;))


# data input
dir_data = &quot;/project2/xuanyao/llw/breastcancerTCGA/txt_file/&quot;
datExpr = readRDS(paste0(dir_data, &quot;ex_var_regressed.rds&quot;))
gene_meta = read.table(paste0(dir_data, &quot;tumor.gene_pos.txt&quot;),
                      sep = &#39;\t&#39;,
                      header = T,
                      stringsAsFactors = F)
gene_module = readRDS(file = paste0(dir_data, &quot;coexp.module.rds&quot;))


# Use only the genes in the module &amp; have postion info in the meta file &amp; not on chromosome chr
gene_in_cluster = data.frame(&quot;gene_name&quot; = names(gene_module$moduleLabels[gene_module$moduleLabels == module_id]), stringsAsFactors = F)
gene_w_pos = merge(gene_in_cluster, gene_meta, by.x = 1, by.y = 1)
gene_trans = gene_w_pos[gene_w_pos[,2] != paste0(&quot;chr&quot;, chr), &quot;gene_name&quot;]


Sigma = cor(datExpr[, gene_trans])
SigmaO = ModifiedSigmaOEstimate(Sigma,simNum=1000)
z_mat_trans = z_mat[, gene_trans]; rm(z_mat)


# Do parallel
cores=(Sys.getenv(&quot;SLURM_NTASKS_PER_NODE&quot;))
cl = makeCluster(as.numeric(cores), outfile=&quot;&quot;)
registerDoParallel(cl)

Nsnps = nrow(z_mat_trans); snps = rownames(z_mat_trans)
chunk.size = Nsnps %/% as.numeric(cores)
Nsnps_left = Nsnps %% as.numeric(cores)
Nsnps_done = (chunk.size)*as.numeric(cores)

system.time(
  res.all &lt;- foreach(i=1:cores, .combine=&#39;c&#39;) %dopar%
    {
      library(MPAT)
      library(mvtnorm)

      res &lt;- rep(NA, chunk.size); names(res) &lt;- snps[((i-1)*chunk.size+1):(i*chunk.size)]
      for(x in 1:chunk.size) {
        res[x] &lt;- ModifiedPCOMerged(Z.vec=z_mat_trans[x + (i-1)*chunk.size, ],
                                                       Sigma=Sigma, SigmaO=SigmaO,method = &quot;davies&quot;)

        if(x %% 100 == 0){print(x + (i-1)*chunk.size)}
      }

      res
    }
)
stopCluster(cl)

print(&quot;Parallel part is done!&quot;)


if(Nsnps_left != 0){
  res_left = rep(NA, Nsnps_left); names(res_left) = snps[1:(Nsnps_left) + Nsnps_done]
  for(k in 1:(Nsnps_left)){
    res_left[k] = ModifiedPCOMerged(Z.vec=z_mat_trans[k + Nsnps_done, ],
                               Sigma=Sigma, SigmaO=SigmaO,method = &quot;davies&quot;)

    print(k + Nsnps_done)
  }

  res.all = c(res.all, res_left)
}

print(&quot;All part is done!&quot;)


saveRDS(res.all, file = paste0(&quot;./p/p.module&quot;, module_id, &quot;.chr&quot;, chr, &quot;.&quot;, p.method, &quot;.rds&quot;))</code></pre>
</div>
<div id="multi-testing-correction" class="section level2">
<h2>Multi-testing correction</h2>
<div id="pvalue-adjustment-based-on-uniform-null-pvalue" class="section level3">
<h3>Pvalue adjustment based on uniform null pvalue</h3>
<pre class="r"><code>rm(list = ls())

library(qvalue)

# estimate the p-value cut-off to control the false discovery rate at a certain level

parameter = commandArgs(trailingOnly = T)
fdr.level = parameter[1]
adjust.method = parameter[2]

# input pvalues for all 18 gene modules and 22 chromosomes
pvalue_pc = NULL
for(gene_cluster_id in 1:18){
  for(chr in 1:22){
    tmp_pc = readRDS(file = paste0(&quot;./TruncPCO/GeneCluster&quot;, gene_cluster_id, &quot;/pvalue_pco_&quot;, chr, &quot;.rds&quot;))
    names(tmp_pc) = paste0(&quot;C&quot;, gene_cluster_id, &quot;:&quot;, names(tmp_pc))
    pvalue_pc = c(pvalue_pc, tmp_pc)

    cat(&quot;Gene cluster:&quot;, gene_cluster_id, &quot;. Chr:&quot;, chr, &quot;\n&quot;)
    print(object.size(pvalue_pc))
  }
}

# histogram of all pvalues
png(&quot;./FDR/Hist of all pvalues&quot;)
hist(pvalue_pc)
dev.off()
print(&quot;Histogram of pvalues saved!&quot;)


# adjust pvalues
if(adjust.method == &quot;BH&quot;){
  p.adjusted = p.adjust(pvalue_pc, method = &quot;BH&quot;)
  print(&quot;BH correction of pvalue done!&quot;)


  signal_q = p.adjusted[p.adjusted&lt;fdr.level]
  signal_name = names(signal_q)
  signal_summary = cbind(pvalue_pc[signal_name], signal_q)
  colnames(signal_summary) = c(&quot;pvalue.PCO&quot;, &quot;qvalue.PCO&quot;)
  write.table(signal_summary, file = &quot;./FDR/transeQTL.Summary.BH.txt&quot;,
              quote = FALSE)
  print(&quot;Signals written in the file!&quot;)


  saveRDS(p.adjusted, file = &quot;./FDR/qvalueofPC.BH.rds&quot;)
  print(&quot;qvalue results saved!&quot;)

}else if(adjust.method == &quot;qvalue&quot;){
  qobj_pc = qvalue(p = pvalue_pc, fdr.level=fdr.level)
  print(&quot;FDR analysis of pc has been done!&quot;)


  signal_q = qobj_pc$qvalues[qobj_pc$significant]
  signal_name = names(signal_q)
  signal_summary = cbind(qobj_pc$pvalues[signal_name], signal_q)
  colnames(signal_summary) = c(&quot;pvalue.PCO&quot;, &quot;qvalue.PCO&quot;)
  write.table(signal_summary, file = &quot;./FDR/transeQTL.Summary.q.txt&quot;,
              quote = FALSE)
  print(&quot;Signals written in the file!&quot;)


  saveRDS(qobj_pc, file = &quot;./FDR/qvalueofPC.q.rds&quot;)
  print(&quot;qvalue results saved!&quot;)

}</code></pre>
</div>
<div id="pvalue-adjustment-using-empirical-null-pvalue" class="section level3">
<h3>Pvalue adjustment using empirical null pvalue</h3>
</div>
</div>
<div id="post-analysis-of-signals" class="section level2">
<h2>Post analysis of signals</h2>
<ul>
<li><p>Use all PC’s. Adjust pvalues using <code>qvalue</code>.</p></li>
<li><p>Use PC’s with eigenvalue <span class="math inline">\(\lambda&gt;1\)</span>. Adjust pvalues using empirical null.</p></li>
<li><p>Use the first PC. (elife method)</p></li>
<li><p>Single-variate method. (MatrixeQTL)</p></li>
</ul>
</div>
<div id="others" class="section level2">
<h2>Others</h2>
<div id="use-only-the-first-pc.-use-all-pcs." class="section level3">
<h3>Use only the first PC. Use all PC’s.</h3>
</div>
<div id="find-phseudogenes-poor-mapped-genes-and-cross-mappable-genes" class="section level3">
<h3>Find phseudogenes, poor-mapped genes, and cross-mappable genes</h3>
</div>
<div id="run-matrixeqtl" class="section level3">
<h3>Run MatrixeQTL</h3>
<ul>
<li><p>Prepare files</p></li>
<li><p>Run MatrixeQTL</p></li>
</ul>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
