<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Weekly Log</title>

<script src="site_libs/header-attrs-2.23/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">GradLog</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Projects
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Trans.html">Multi PC asso test for trans</a>
    </li>
    <li>
      <a href="Log2020.html">Log2020</a>
    </li>
    <li>
      <a href="Log2021.html">Log2021</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/liliw-w/GradLog">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Weekly Log</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-04-09
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>GradLog/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown file has unstaged changes. To know which version of
the R Markdown file created these results, you’ll want to first commit
it to the Git repo. If you’re still working on the analysis, you can
ignore this warning. When you’re finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20201014code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20201014)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20201014code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20201014)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreed3806017b549188b7f4b1b8ea7ad3460f9da628dtargetblankd380601a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/liliw-w/GradLog/tree/d3806017b549188b7f4b1b8ea7ad3460f9da628d" target="_blank">d380601</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomliliwwGradLogtreed3806017b549188b7f4b1b8ea7ad3460f9da628dtargetblankd380601a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/liliw-w/GradLog/tree/d3806017b549188b7f4b1b8ea7ad3460f9da628d" target="_blank">d380601</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory

Unstaged changes:
    Modified:   analysis/Log2024_new_beginning.Rmd
    Modified:   analysis/week_log.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/week_log.Rmd</code>) and HTML
(<code>docs/week_log.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ff0bca7b335cd7a4331f7211666bfea05bbe7f53/docs/week_log.html" target="_blank">ff0bca7</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/9fd6e5fde794caf294634ac5f2acd97c05b60c0e/docs/week_log.html" target="_blank">9fd6e5f</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-24
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/81739628828a7a12b81e2126ab84ebb07a6714ab/analysis/week_log.Rmd" target="_blank">8173962</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-24
</td>
<td>
wflow_publish("analysis/week_log.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/151bee49ed9b1c575c2cae28be51893cb98972ed/docs/week_log.html" target="_blank">151bee4</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b7a0bd952f4535cdd59b1d12ba7cfa6a6ec467d8/analysis/week_log.Rmd" target="_blank">b7a0bd9</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
test toc render
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/423f69ecc48dab9a0b59a4670f609d8c887872a7/docs/week_log.html" target="_blank">423f69e</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/1cf9a35d674d71d14776678bd402a5a6c0d196a8/docs/week_log.html" target="_blank">1cf9a35</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/2448521ef95621fce2dfa6ec2c83aeca5f3323a6/analysis/week_log.Rmd" target="_blank">2448521</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
rename
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/ae55f7a39e0c3415a41f6223f940530abb2168a6/docs/week_log.html" target="_blank">ae55f7a</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
test toc render
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f595c33706116e2ab87118cfb168af53e7a330db/analysis/week_log.Rmd" target="_blank">f595c33</a>
</td>
<td>
liliw-w
</td>
<td>
2024-04-22
</td>
<td>
rename
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/76794f9b78aac10f7a732dfa23aa11b3470f39f1/docs/week_log.html" target="_blank">76794f9</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/cd276715dd17defbcb4678497f6e0c83a430504a/analysis/week_log.Rmd" target="_blank">cd27671</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/2b44f3b93dcb612c6ee7afd19de68291293ed465/docs/week_log.html" target="_blank">2b44f3b</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/ab953a441bab56b2416fb3df6edabcbfb0a2f886/analysis/week_log.Rmd" target="_blank">ab953a4</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/79511afb75f80a6125cf148810e7f374667b46ea/docs/week_log.html" target="_blank">79511af</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/b4b7969ea5a16f6340afb83bdd44bea07871b6c0/analysis/week_log.Rmd" target="_blank">b4b7969</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/355d984af4276924aebee518677732347b438635/docs/week_log.html" target="_blank">355d984</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/d469ed58415eeaccd59f097ead4cc4c53f7153e6/analysis/week_log.Rmd" target="_blank">d469ed5</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/387ac23a9997080f7f9886531d827e4b0ab32dbc/docs/week_log.html" target="_blank">387ac23</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/85299aece1b9be553f6594acb56aab4b084e2a51/analysis/week_log.Rmd" target="_blank">85299ae</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/d8c26065320f9c21d500a8e0aabfc91a23942dbb/docs/week_log.html" target="_blank">d8c2606</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/26506ca9655968c5a3343a926fe59f9bbd981478/analysis/week_log.Rmd" target="_blank">26506ca</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/c775adf8c78f2121e7ed49b52fb522ed3db39287/docs/week_log.html" target="_blank">c775adf</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/f3d506db11164142fd3c0dbe806aa8e4c11511f8/analysis/week_log.Rmd" target="_blank">f3d506d</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/8d7f0d82658e84b8758e32ff37b6f75d7503e6c4/docs/week_log.html" target="_blank">8d7f0d8</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/9b5c76ee89b54ca52a4698c40402265414112d55/analysis/week_log.Rmd" target="_blank">9b5c76e</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/liliw-w/GradLog/46b8cbadf76f631c5e35f5ddae7703bb86dd0ac8/docs/week_log.html" target="_blank">46b8cba</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/liliw-w/GradLog/blob/04c567128865761dcd06c3c08fa23d3e0c5175a6/analysis/week_log.Rmd" target="_blank">04c5671</a>
</td>
<td>
liliw-w
</td>
<td>
2024-02-20
</td>
<td>
test
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<style type="text/css">
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 14px;
    border-left: 5px solid #eee;
}
</style>
<div id="april-09" class="section level1">
<h1>April 09</h1>
<div id="progress-summary" class="section level2">
<h2>Progress summary</h2>
<ul>
<li><p>Get familiar with and set up cluster.</p></li>
<li><p>Looked at the serial biopsies data.</p></li>
</ul>
</div>
<div id="genomic_specimen.csv" class="section level2">
<h2><code>GENOMIC_SPECIMEN.csv</code></h2>
<div id="dataset-info" class="section level3">
<h3>Dataset Info</h3>
<ul>
<li><p>Each row is a sample. Multiple samples can come from one patient.
These samples are profiled in different time points.</p></li>
<li><p>One patient’s samples can be tested by multiple TEST_TYPE
(3%).</p></li>
</ul>
</div>
<div id="filter-samples-and-patients" class="section level3">
<h3>Filter samples and patients</h3>
<ul>
<li><p>Only consider patient specimens tested with OncoPanel from
PROFILECOHORT, by filtering
<code>TEST_TYPE == 'ONCOPANEL_PROFILECOHORT'</code>.</p></li>
<li><p>There are 27,148 specimen, 26,307 unique patients, 773 patients
have their multiple samples tested.</p></li>
<li><p>719 patients have been tested twice. The other patients were
tested 3 to 6 times.</p></li>
<li><p>I only keep specimen samples from patients that were sampled
twice, corresponding to 1438 sample left. [This corresponds to what we
mentioned about the data - “~1000 patients”, “sequenced twice”]</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/patient_multi_sampled.pdf" title="Figure: Number of samples each patient has." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of samples each patient has.
</p>
</div>
</div>
<div id="patient-classification" class="section level3">
<h3>Patient classification</h3>
<ul>
<li>To classify a patient of two samples, I use flag
<code>IS_MATCHED_FLG</code> (normal or tumor sample) and
<code>REPORT_DT</code> (Date this report was signed out by the
pathologist), which results in four categories.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">patient_category</th>
<th align="right">n_patients</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">normal−normal</td>
<td align="right">13</td>
</tr>
<tr class="even">
<td align="left">normal−tumor</td>
<td align="right">1</td>
</tr>
<tr class="odd">
<td align="left">tumor−normal</td>
<td align="right">14</td>
</tr>
<tr class="even">
<td align="left">tumor−tumor</td>
<td align="right">691</td>
</tr>
</tbody>
</table>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/patient_categoties_across_biopsy.pdf" title="Figure: Number of patients belong to each catogory." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of patients belong to each catogory.
</p>
</div>
<p><strong>Questions -</strong></p>
<ul>
<li><p>No flag SAMPLE_COLLECTION_DT to record accurate date when sample
was collected first.</p></li>
<li><p>Does “tumor-tumor” mean two samples were collected before and
after treatment?</p></li>
<li><p>Why normal samples have non zero tumor purity? Some samples also
have negative tumor purity.</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/normal_sample_non_zero_tumor_purity.pdf" title="Figure: Normal samples have non zero tumor purity." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Normal samples have non zero tumor purity.
</p>
</div>
</div>
<div id="tumor_purity-change-across-patients-two-biopsies"
class="section level3">
<h3>TUMOR_PURITY change across patients’ two biopsies</h3>
<ul>
<li>TUMOR_PURITY - Estimated percentage of neoplastic cells in the
sample.</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/TUMOR_PURITY_change_across_biopsies.pdf" title="Figure: TUMOR_PURITY changes across two biopsies of each patient belonging to four catogories." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: TUMOR_PURITY changes across two biopsies of each patient
belonging to four catogories.
</p>
</div>
</div>
</div>
<div id="genomic_mutation_results.csv" class="section level2">
<h2><code>GENOMIC_MUTATION_RESULTS.csv</code></h2>
<div id="filter-samples-and-variants" class="section level3">
<h3>Filter samples and variants</h3>
<ul>
<li><p>Use samples from above</p></li>
<li><p>Only SNP variants</p></li>
<li><p>Keep only patients that have two samples across mutation changes
(some patients have only one sample with mutation info)</p></li>
<li><p>Resulting in 650 patients</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/patient_categoties_across_biopsy_with_mutation_change.pdf" title="Figure: Number of patients belong to each catogory." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of patients belong to each catogory.
</p>
</div>
</div>
<div id="allele_fraction-change-across-patients-two-biopsies-for-snps"
class="section level3">
<h3>ALLELE_FRACTION change across patients’ two biopsies for SNPs</h3>
<ul>
<li><p>ALLELE_FRACTION - Fraction of reads for the observed
allele.</p></li>
<li><p>How many SNPs does each patient have?</p></li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/num_snp_for_patient.pdf" title="Figure: Number of SNP varaints each patient has." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: Number of SNP varaints each patient has.
</p>
</div>
<ul>
<li>ALLELE_FRACTION change across patients’ two biopsies for SNPs</li>
</ul>
<div class="figure" style="text-align: center">
<embed src="../../serial_biopsies/figs/af_change_across_biopsies.pdf" title="Figure: ALLELE_FRACTION change across patients' two biopsies for SNPs." width="50%" height="100%" type="application/pdf" />
<p class="caption">
Figure: ALLELE_FRACTION change across patients’ two biopsies for SNPs.
</p>
</div>
<p><strong>Questions -</strong></p>
<ul>
<li>Use ALLELE_FRACTION directly or adjusted for TUMOR_PURITY by
ALLELE_FRACTION/TUMOR_PURITY?</li>
</ul>
</div>
</div>
<div id="misc" class="section level2">
<h2>Misc</h2>
<ul>
<li><p>Set up Eris cluster</p>
<ul>
<li><p>Can’t login after creating account? Wait 24 hours for the account
to be activated.</p></li>
<li><p>Can’t submit job to slurm? Permission issue, contact help
team.</p></li>
</ul></li>
<li><p>Access to rstudio server and jupyter notebook</p>
<ul>
<li><p>Username is lower case</p></li>
<li><p>Incorrect username or password? Contact help team to restart the
server.</p></li>
</ul></li>
</ul>
<p>Things to double check -</p>
<ul>
<li><p>Results put in <code>/data/gusev/USER/llw</code>?</p></li>
<li><p>Dropbox lab access?</p></li>
<li><p>MPG weekly meeting, location?</p></li>
<li><p>Download Doug’s data &amp; upload to cluster</p></li>
</ul>
<div id="refs">

</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.2.3 (2023-03-15)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur ... 10.16

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.2/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] lubridate_1.9.3 forcats_1.0.0   stringr_1.5.0   dplyr_1.1.3    
 [5] purrr_1.0.2     readr_2.1.4     tidyr_1.3.0     tibble_3.2.1   
 [9] ggplot2_3.4.3   tidyverse_2.0.0

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.0  xfun_0.39         bslib_0.5.0       colorspace_2.1-0 
 [5] vctrs_0.6.3       generics_0.1.3    htmltools_0.5.5   yaml_2.3.7       
 [9] utf8_1.2.3        rlang_1.1.1       jquerylib_0.1.4   later_1.3.1      
[13] pillar_1.9.0      glue_1.6.2        withr_2.5.1       lifecycle_1.0.3  
[17] munsell_0.5.0     gtable_0.3.4      workflowr_1.7.0   evaluate_0.21    
[21] knitr_1.43        tzdb_0.4.0        fastmap_1.1.1     httpuv_1.6.11    
[25] fansi_1.0.4       highr_0.10        Rcpp_1.0.11       promises_1.2.0.1 
[29] scales_1.2.1      cachem_1.0.8      jsonlite_1.8.7    fs_1.6.2         
[33] hms_1.1.3         digest_0.6.33     stringi_1.7.12    rprojroot_2.0.3  
[37] grid_4.2.3        cli_3.6.1         tools_4.2.3       magrittr_2.0.3   
[41] sass_0.4.6        whisker_0.4.1     pkgconfig_2.0.3   timechange_0.2.0 
[45] rmarkdown_2.23    rstudioapi_0.15.0 R6_2.5.1          git2r_0.32.0     
[49] compiler_4.2.3   </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
