---
title: "Overview of Chromatin Accessibility"
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
bibliography: references.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
```

**If any figures don't show, try opening in Safari.**

# Basics

## [Chromatin structure](https://en.wikipedia.org/wiki/Chromatin)

- **Chromatin** is a complex of DNA and protein found in eukaryotic cells.

  - The primary function is to package long DNA molecules into more compact, denser structures.
  
  - Histones prevent DNA from becoming tangled and protect it from DNA damage. In addition, histones play important roles in gene regulation and DNA replication.


- Three levels of chromatin organization & The overall structure of the chromatin network further depends on the stage of the cell cycle.
  
  <center>
  ![Chromatin Structures](asset/2023/Chromatin_Structures.png){width="120%"}
  </center>


- The basic unit is a **nucleosome**: histone wound by DNA.

  <center>
  ![Nucleosome Structures](asset/2023/Nucleosome_structure.png){width="80%"}
  </center>
  
  
  - The primary protein components of chromatin (DNA packaging protein) are **[histones](https://en.wikipedia.org/wiki/Histone#Histone_modification)**.

    - There are five major families of histones which are designated H1/H5 (linker histones), H2A, H2B, H3, and H4 (core histones).
    
  - An octamer of two sets of four histone cores (Histone H2A, Histone H2B, Histone H3, and Histone H4) bind to DNA and function as "anchors" around which the strands are wound.
  
  - The tight wrapping of DNA around histones is to a large degree a result of electrostatic attraction between the positively charged histones and negatively charged phosphate backbone of DNA.




## Chromatin accessibility/compaction - Alter the local chromatin structure


### How is it related with gene regulation?

- By altering the level of chromatin compaction, e.g. structurally loose to allow access to RNA and DNA polymerases that transcribe and replicate the DNA.

- Open chromatin regions (OCRs) are nucleosome-depleted regions that can be bound by protein factors.


### Change chromatin accessibility by histone modification

- The chromatin state, whether it's open or closed, is controlled by epigenetic modifications (to the histone, or to the DNA itself).


- Histone proteins are the basic packers and arrangers of chromatin and can be modified by various post-translational modifications to **alter chromatin packing** and their interaction with DNA and nuclear proteins (histone modification).


- The H3 and H4 histones have long tails protruding from the nucleosome, which can be covalently modified at several places. **Modifications of the tail include**, 

  - methylation
  
  - acetylation
  
  - phosphorylation
  
  - ubiquitination
  
  - SUMOylation
  
  - citrullination
  
  - ADP-ribosylation
  
  (The core of the histones H2A and H2B can also be modified.)


- Combinations of modifications, known as **histone marks**. The common nomenclature of histone modifications is,

  - The name of the histone (e.g., H3)
  
  - The single-letter amino acid abbreviation (e.g., K for Lysine)
  
  - the amino acid position in the protein
  
  - The type of modification (Me: methyl, P: phosphate, Ac: acetyl, Ub: ubiquitin)
  
  - The number of modifications (only Me is known to occur in more than one copy per residue. 1, 2 or 3 is mono-, di- or tri-methylation)
  
  So H3K4me1 denotes the monomethylation of the 4th residue (a lysine) from the start (i.e., the N-terminal) of the H3 protein.


- Example of histonen modifications

  - Methylation of arginine or lysine residues
    
    Methylation can affect how other protein such as transcription factors interact with the nucleosomes.
    
  - Acetylation of lysine
    
    Lysine acetylation eliminates a positive charge on lysine thereby weakening the electrostatic attraction between histone and DNA resulting in partial unwinding of the DNA making it more accessible for gene expression.


## Methods to profile chromatin accessibility in open chromatin regions

- chromatin accessibility assays that directly (DNase-seq, FAIRE-seq and ATAC-seq) isolate accessible locations of a genome

- in contrast to histone mark ChIP-seq, where such effects must be inferred by presence or absence of overlapping histone tail modifications.


### ATAC-seq

(Assay for Transposable Accessible Chromatin sequencing) uses the Tn5 transposase to integrate (synthetic) transposons into accessible regions of the genome consequentially highlighting the localisation of nucleosomes and transcription factors across the genome


### DNase-seq

(DNase I hypersensitive sites Sequencing) uses the sensitivity of accessible regions in the genome to the DNase I enzyme to map open or accessible regions in the genome.


### [ChIP-seq](https://www.youtube.com/watch?v=nkWGmaYRues&t=144s) to assay histone modifications

- Identify the binding sites for any DNA-associated proteins of interest.
  
  - Applied to the set of ChIP-able proteins and modifications, such as transcription factors, polymerases and transcriptional machinery, structural proteins, protein modifications, and DNA modifications.

- It is primarily used to determine how transcription factors and other chromatin-associated proteins influence phenotype-affecting mechanisms.


**Types**

- TF ChIP-seq: sonication to fragment nucleosome, antibody against TF

- Histone mark ChIP-seq: sonication or MNase to fragment nucleosome, antibody against histone modification


<center>
![Chromatin Structures](asset/2023/chip_seq_histone_mark.png){width="50%"}
</center>


# Chromatin accesibility analysis

## Bulk chromatin accessibility data

<center>
![Overview of common tasks in the analysis of bulk chromatin accessibility data](asset/2023/common_task_bulk.png){width="80%"}
</center>



## scATAC-seq data

<center>
![Overview of common tasks in the analysis of scATAC-seq data](asset/2023/common_task_sc.png){width="80%"}
</center>


# Peak calling: detect regions of enrichment

<center>
![Chromatin Structures](asset/2023/peak_calling.png){width="80%"}
</center>


- The most widely used tool for peak calling is MACS2, which is also the default in the ENCODE ATAC-seq pipeline.

- Other general and method-specific peak callers exist, for example, ZINBA (general), HMMRATAC and Genrich (ATAC-seq), and F-seq and Hotspot (DNase-seq and ATAC-seq).



## Basics

- 3 prime end & 5 prime end

  A nucleic acid strand is inherently directional, and the "5 prime end" has a free hydroxyl (or phosphate) on a 5' carbon and the "3 prime end" has a free hydroxyl (or phosphate) on a 3' carbon.

- tag
  
  a tag sequence read

- tag seq v.s. standard RNA-seq
  
  - Standard RNA-seq: generate sequencing libraries for full-length mRNAs
  
  - TagSeq: only generate a single fragment from the 3'/5'/paired end of each transcript (a tag read) and quantifies gene expression by tag abundance


- paired-end tag (ditags) & single-end tag
  
  - Paired-end tags (PET) are the short sequences at the 5’ and 3' ends of a DNA fragment, which are unique enough that they (theoretically) exist together only once in a genome, therefore making the sequence of the DNA in between them available upon search (if full-genome sequence data is available) or upon further sequencing (since tag sites are unique enough to serve as primer annealing sites).
  
  - a PET "represents" a larger fragment of genomic
  
  - Since the tag sequences are short, individual PETs are well suited for next-generation sequencing that has short read lengths and higher throughput.
  
  - increased specificity when aligning back to the genome compared to single tags, which involves only one end of the DNA fragment.



## How MACS2 call peaks specifically?

[Ref: Shirley Liu, et al. Genome Biology, 2008.]


### [Steps](https://hbctraining.github.io/Intro-to-ChIPseq/lessons/05_peak_calling_macs.html)

- Determine the site length d
  
  > The distance between the modes of the two peaks in the alignment is defined as ‘d’ and represents the estimated fragment length.

- Define sliding windows
  
  > After MACS shifts every tag by d/2, it then slides across the genome using a window size of 2d to find candidate peaks.

- Identify windows with significant tag enrichment
  
  > A region is considered to have a significant tag enrichment if the p-value < 10e-5 (this can be changed from the default). This is a Poisson distribution p-value based on $\lambda$.

- Merge overlapping peaks with enriched tags
  
  > Overlapping enriched peaks are merged, and each tag position is extended ‘d’ bases from its center.

- Correct for multiple testing
  
  > in MACS2, p-values are now corrected for multiple comparison using the Benjamini-Hochberg correction.




- MACS2 was originally designed for ChIP–seq data, specific parameters (for example, --nomodel) need to be used for peak calling in ATAC-seq or DNase-seq data.

- MACS2 is a model-based algorithm originally designed for ChIP–seq data analysis. It implements a dynamic Poisson distribution to capture local background biases in the genome and to effectively detect peaks


~~~
significantly enriched tags in the ChIP sample relative to the control, 

although a small ChIP peak region often contains too few control tags to robustly estimate the background biases.
~~~


~~~
ChIP-seq analysis algorithms have specialized in identifying one of two types of enrichment: broad domains (i.e. histone modifications that cover entire gene bodies) or narrow/sharp peaks (i.e. a transcription factor bound to an enhancer).
~~~


- control samples: to estimate the background biases



- paired-end

- shift-extend (for single-end tag)







### Construction of a common set of features across different samples

To be able to compare samples with each other in downstream steps.

- Construct a consensus peak file
  
  - comprises the set of peaks that are shared between samples, and in which the start and end location of overlapping peaks are adjusted (through the so-called merging of peaks) to thus yield one consensus peak

- A predefined set of regions

- A binned genome


- Peak calling is a computational method used to identify areas in the genome that have been enriched with aligned reads.




# Thoughts

## Peak calling using MACS2

1. How often that a called peak has multiple sub-peaks?

2. Even when sub-peaks exist, how to determine the "sub-regions" to apply multivariate association?

3. What's the power improvement?



