---
title: "Log2024"
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 3
editor_options:
  chunk_output_type: console
bibliography: references.bib
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, out.width = "50%")
```

```{css style settings, echo = FALSE}
blockquote {
    padding: 10px 20px;
    margin: 0 0 20px;
    font-size: 16px;
    border-left: 5px solid #eee;
}
```

**If any figures don't show, try opening in Safari.**


# Sep 12


## 0. Insert numbers to current draft



## 1. CACTI-s on mark H3K27ac (LCL and macrophage)

Goal -
  
  To show cacti-s is slightly worse than cacti-p but still better than univariates, which indicates a balance of skipping peak calling and reduced power.



[Not include this part in the paper. Put it in package README. Add observations to README. Add this part in thesis.]


### How?

- segment size option
  
  '1kb', '2kb', '5kb', '10kb'

- segment filter - Exclude segments not overlap with peaks
  
  Reason - to keep comparison fair


### Compare H3K27ac-cacti-s and H3K27ac-cacti-p

- Signal number

- Signal overlap

- Compare H3K27ac-cacti-s to H3K27ac-fastQTL


**Observations & Conclusions -**


```{r out.height = "100%", fig.cap="Figure: CACTI-P on H3K27AC in LCL v.s. uni. window50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cpeak_cwindow_H3K27AC_window50kb_maxwindow50kb_cis500kb.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure:  CACTI-S on H3K27AC in LCL v.s. uni. window50kb. seg10kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cpeak_cwindow_seg_group_H3K27AC_window50kb_cis500kb_seg5kb_overlap_peak.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure:  CACTI-P on H3K27AC in LCL v.s. CACTI-S. window50kb. seg10kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cwindow_cactip_vs_cactis_H3K27AC_window50kb_cis500kb_seg5kb_overlap_peak.pdf"), error = FALSE)
```


### Results using other segment options

Done. 


### Results using the same segment filtering as in broad mark

Done. 


### try another filtering based on cpm (check Ben's code)





## 2. coloc analysis on cQTLs of mark H3K27ac by CACTI-s

Done. 



## Proportion of cwindow sharing among marks (details see on slack)

> Figure 4G of your manuscript, I think enrichment values are less intuitive. Do you have proportion of sharing instead?

Done. 

```{r out.height = "100%", fig.cap="Figure:", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_enrich_cwindow_across_mark_lcl.pdf"), error = FALSE)
```



## Number of cwindows (and overlap with cpeaks) figure

Done. 

```{r out.height = "100%", fig.cap="Figure:", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/cwindow_across_datasets.pdf"), error = FALSE)
```



## 3. CACTI-s on broad mark H3K27me3 (macrophage)

Done. 

To make cacti-s 's data processing consistent with Aracena et al., see their analysis path [here](asset/2024/09_12_Aracena_analysis_path.png). 


- ~~Get chrome size for hg19 for H3K27me3 in macrophage~~


### sample filtering

> 3 samples labels will differ between the bams and sample labels in the paper and all paper inputs - AF22, AF36 and AF38. Despite having the AF prefix these samples are nearly 100% European ancestry. We changed these sample labels to EU prefixes (EU22, EU36 and EU38) in the manuscript but the bams have these "original" names. I would recommend changing any output files you create to EU22, EU36 and EU38 if you're going to publish this work eventually just to keep things concordant

~~change sample labels in snmk pipeline, as the updated sample labels will be used in meta and regress covariates~~



### peak/seg filtering

To be consistent with the peak filtering in Aracena et al., I keep only the segments that are present in at least 50% of Flu or NI samples (>=14). 



### geno filtering

> WGS fastq files were processed using GenPipes’ (v3.1.4) DNA sequencing pipeline62 Briefly, raw reads were trimmed using Skewer (v0.2.2)63 and the resulting reads were aligned to the hg19 human reference genome using BWA-MEM (v0.7.17)64. Insertion/deletion realignment and base quality score recalibration were performed using GATK (v3.8)65, and duplicates were marked using Picard (v2.23.3)66. We used GATK’s HaplotypeCaller in gvcf mode to perform SNV and insertion and deletions calling. We filtered the joint genotyped file to exclude nonautosomal and nonbiallelic variants. Additionally, we used PLINK (v1.90b6.9)67,68 to remove SNPs that had a call rate of <90% across all samples, that deviated from Hardy–Weinberg equilibrium at P<10e−5, and with minor allele frequency less than 5%. We used the resulting 7,383,243 SNPs in QTL mapping and other downstream analyses. We annotated the SNPs using dbSNP (human_9606_b151_GRCH37p13)69.


### covariates

- Non genetic factors, e.g. batch, age, et al

- Expression PCs
  
  I used same number PCs used for peaks in Aracena et al, where they tried different number of PCs. 
  
  For H3K27me3, 1 and 2 PCs were used for condition NI and Flu. 

- Genotype first PC
  
  PC1 of genotype data is the only covariate included (to account for population structure)



### Compare H3K27me3-cacti-s, H3K27me3-fastQTL, and H3K27me3-cacti-p

- Signal number

- Signal overlap


**Observations & Conclusions -**


```{r out.height = "100%", fig.cap="Figure: CACTI-P on H3K27me3 v.s. uni. window50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cpeak_cwindow_H3K27me3_NI_window50kb_cis100kb.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure:  CACTI-S on H3K27me3 v.s. uni. window50kb. seg5kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cpeak_cwindow_seg_group_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure:  CACTI-P on H3K27me3 v.s. CACTI-S. window50kb. seg5kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_plt_cwindow_cactip_vs_cactis_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf"), error = FALSE)
```


## other options of CACTI-s on broad mark H3K27me3 (macrophage)





## 4. coloc analysis on cQTLs of broad marks H3K36me3 (LCL) and H3K27me3 (macrophage) by CACTI-s

Done. 

- Figures H3K36me3 (LCL)

```{r out.height = "100%", fig.cap="Figure: Coloc with H3K36me3 in LCL by cacti-p.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf", "asset/2024/09_12_coloc_compare_H3K36ME3_window50kb_cis500kb_seg5kb_peak_overlap.pdf"), error = FALSE)
```


- Figures H3K27me3 (macrophage)

```{r out.height = "100%", fig.cap="Figure: Coloc with H3K27me3 in Macrophage by cacti-p.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf", "asset/2024/09_12_coloc_compare_aracena_H3K27me3_NI_window50kb_seg5kb_cis100kb.pdf"), error = FALSE)
```



## Update coloc any

Done.

- Any

- Any of narrow

- Any of broad

- Any of cell type

```{r out.height = "100%", fig.cap="Figure: Coloc with any cQTLs.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_any_dataset_celltype_cactips.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_any_dataset_cacti_ps.pdf). 


```{r out.height = "100%", fig.cap="Figure: Coloc with any cQTLs of each mark.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_any_dataset_mark.pdf"), error = FALSE)
```




## 5. Add other traits to coloc analysis

- coloc brain cQTLs of mark H3K27ac in GTEx with schizophrenia

- coloc with immune related diseases




## 7. Coloc with new cQTLs in def of cQTLs of single peaks not coloc

- LCL
  
  - mark set 1
  
  - mark set 2: H3K36ME


- Macrophage
  
  - mark set 1
  
  - mark set 2: H3K27me3


- coloc number with uni cqtls
  
  - Single dataset
  
  - All datasets


```{r out.height = "100%", fig.cap="Figure: Coloc loci with uni cQTLs. Upper panel - coloc with single dataset. Lower panel - coloc with cQTLs of any dataset.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_uni_cqtl_new.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_num_uni_cqtl.pdf). 



- coloc number with multi cqtls
  
  - Single dataset
  
  - All dataset

```{r out.height = "100%", fig.cap="Figure: Coloc loci with multi cQTLs. Upper panel - coloc with single dataset. Lower panel - coloc with cQTLs of any dataset.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_multi_cqtl_new.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_num_multi_cqtl.pdf). 



- Compare to cacti p & cacti s (single dataset)
  
  - LCL
  
  - Macrophage


```{r out.height = "100%", fig.cap="Figure: Compare loci coloc with multi or uni cQTLs across single dataset.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_uni_multi_single_dataset_new.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_num_uni_multi_single_dataset.pdf). 



```{r out.height = "100%", fig.cap="Figure: Distribution of coloc loci with multi or uni cQTLs across single dataset and traits.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution_new.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_num_uni_multi_single_dataset_distribution.pdf). 


- Compare to cacti p & cacti s (all datasets)
  
  Make a bar plot of three categories of coloc loci - uni cQTL only, multi cQTL only, shared. 


```{r out.height = "100%", fig.cap="Figure: Compare loci coloc with multi or uni cQTLs of any dataset.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/09_12_coloc_num_uni_multi_any_dataset_new.pdf"), error = FALSE)
```

[Old version before harmonizing ref genome](asset/2024/09_12_coloc_num_uni_multi_any_dataset.pdf). 



## 6. GWAS coloc with cQTL not eQTL in GTEx tissues





## 9. Check mark H3K27me3 peak length

Mixed? Broad? Narrow?





## ~~8. Link cwindows to genes~~

Not necessary. 


## 7. Performance with varying window sizes




## Add analysis - pathway or gene set enrichment analysis

Add analysis of pathway or gene set enrichment analysis on the genes associated with cQTLs identified by CACTI but missed by single-peak mapping -

- Assess whether these genes or regions are enriched for specific biological functions, pathways, or diseases. 

- This adds a layer of biological interpretation to the findings, showing that the additional cQTLs identified by CACTI are not just statistically significant but also biologically relevant, thereby reinforcing the practical importance of CACTI’s improved detection power. 


## cQTL enrichment in cell type specific variants

Explain why cQTLs not eQTLs ...




# July 25

## Sharing

Ways to define sharing -

- mashr

- cQTL sharing as in Luis paper
  
  - get (SNP, window) level significance




# May 27

## 1. cQTL sharing across cell types

### How?

I compared for one mark, what is the sharing of cQTLs across cell types?

- I looked at three marks H3K27ac, H3K4me1, and H3K4me3. 

- Three cell types LCL, Microphage-NI, and Microphage-Flu. 

- I define the sharing of cQTLs by -
  
  For a focal cell type, I took the lead cQTL associations of all cwindows, I extracted all associations of the lead cQTLs from a compared cell type, took the minumum pvalue of the lead cQTLs from a compared cell type, and finally calculate the correlation (spearman) among the lead pvalue in the focal cell type and the corresponding minimum pvalue in the compared cell type. 
  
  Is it better to use lead SNPs of only significant cwindows?


- Why not use the number of replicated cQTLs?
  
  Another intuitive way is to look at if a significant cQTL in a focal cell type is also a cQTL in the compared cell type. This won't work in our case as there is no easy for use to obtain all cQTLs rather than only lead cQTLs. QTLTools makes it easy to identify all cQTLs by defining a feature-level significance cutoff based on the feature-level empiircal pvalue obtained from beta approximation. However, we can't do this for pco pvalues. 


### Results

```{r out.height = "100%", fig.cap="Figure: The correlation (spearman) among the lead pvalue in the focal cell type and the corresponding minimum pvalue in the compared cell type. Across three marks. (A) Use all windows. (B) Use only the significant windows", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_27_sharing_cell_type.pdf"), error = FALSE)
```

- Note the cis region of LCL (500kb) and Microphage (100kb) are different. 


### Compare to uni results

Run R^2 in their datasets which uses uni for fair comparison. 


```{r out.height = "100%", fig.cap="Figure: The correlation (spearman) among the lead pvalue by uni test.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_27_sharing_cell_type_uni.pdf"), error = FALSE)
```




## 2. cQTL sharing across marks

### How?

- Three cell types LCL, Microphage-NI, and Microphage-Flu. 

- Marks H3K27ac, H3K4me1, and H3K4me3. 


I compared in cell type, what is the sharing of cQTLs across marks?

- I define the sharing of cQTLs by -
  
  The enrichment of cwindows of one mark in another mark - if a peak of a mark being significant also means it's significant for another mark (the peak should be overlapped between two marks). 


### Results

```{r out.height = "100%", fig.cap="Figure: cQTL sharing across marks in LCL. (A) The enrichment of cwindows of one mark in another mark. (B) Enrich pvalue. (C) Proportion of cwindows of one mark are also significant for another mark.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf"), error = FALSE)
```

Similar results of cell type [microphage NI](asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni.pdf) and [microphage flu](asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu.pdf). 


### Compare to uni results

Run same enrichment analysis in their datasets which uses uni for fair comparison. 

Three cell types. 


- Overlap peak number
  
  - Compare to previous number

- Enrich
  
  - Compare to previous enrichment


```{r out.height = "100%", fig.cap="Figure: (LEFT) Windows (RIGHT) Peaks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_27_num_overlap_window_lcl.pdf", "asset/2024/05_27_num_overlap_window_lcl_uni.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure: (LEFT) Windows (RIGHT) Peaks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_27_enrich_cwindow_across_mark_lcl.pdf", "asset/2024/05_27_enrich_cwindow_across_mark_lcl_uni.pdf"), error = FALSE)
```


Similar results of other cell type -

- [window - microphage NI](asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni.pdf) v.s. [peak - microphage NI](asset/2024/05_27_enrich_cwindow_across_mark_microphage_ni_uni.pdf)

- [window - microphage flu](asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu.pdf) v.s. [peak - microphage flu](asset/2024/05_27_enrich_cwindow_across_mark_microphage_flu_uni.pdf)


Overlap peak number of other cell type -

- [window - microphage NI](asset/2024/05_27_num_overlap_window_microphage_ni.pdf) v.s. [peak - microphage NI](asset/2024/05_27_num_overlap_window_microphage_ni_uni.pdf)

- [window - microphage flu](asset/2024/05_27_num_overlap_window_microphage_flu.pdf) v.s. [peak - microphage flu](asset/2024/05_27_num_overlap_window_microphage_flu_uni.pdf)



**Observations**

- More overlaps of single peaks than windows across marks

- Less sharing using cwindows







# May 16

## 1. GWAS loci coloc with cQTLs

### 36 traits v.s. cQTLs of three marks in LCL

- Window size 50kb. 


I looked at a set of numbers, including ...


**Overall number -**

- Number of GWAS loci

- Number of GWAS loci coloc with any cqtls


**Single mark & mark specificity & mark sharing -**

- Number of GWAS loci coloc with each mark

- Number of GWAS loci coloc with only one mark

- Number of GWAS loci coloc with more than two marks


```{r out.height = "100%", fig.cap="Figure: Number of coloc loci across 36 traits and three histone marks in LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_16_coloc_num_alltrait_allmark.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure: Proportion of coloc loci across 36 traits and three histone marks in LCL. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_16_coloc_prop_alltrait_allmark.pdf"), error = FALSE)
```


### 36 traits v.s. cQTLs of four marks in Microphage of Aracena et al.

- Window size 50kb. 

- Flu only.


```{r out.height = "100%", fig.cap="Figure: Number of coloc loci across 36 traits and four histone marks in Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_16_coloc_aracena_num_alltrait_allmark.pdf"), error = FALSE)
```


```{r out.height = "100%", fig.cap="Figure: Proportion of coloc loci across 36 traits and four histone marks in Microphage. 36 traits on y-axis. (A) total gwas loci in grey, gwas loci coloc with any mark in red. (B) gwas loci coloc with any mark. (C) gwas loci coloc with any mark in red, gwas loci coloc with only one mark in cream. (D-F) gwas loci coloc with any mark in red, gwas loci coloc with the mark in violet, gwas loci coloc with only the mark in cream.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_16_coloc_aracena_prop_alltrait_allmark.pdf"), error = FALSE)
```





### Cell types

- LCL v.s. Microphage


### New coloc loci

- Number of GWAS loci coloc with only new cqtls

- Number of GWAS loci new coloc proportion by each mark

- Number of GWAS loci new coloc proportion by more than two marks

- Number of GWAS loci new coloc proportion by only one mark


### Coloc loci not with eQTLs





# Apr 25 & May 2 & May 9

## 1. Peak merging results of LCL dataset

### Groups changes after merging

- Number of groups

- Size of groups

- Single left out peaks not merged thus not tested


Below are results of tissue Brain. 


```{r out.height = "100%", fig.cap="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/H3K27AC_peaks_in_group_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/H3K27AC_peaks_in_group_w_cqtl_cpeak500kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/H3K27AC_peaks_w_cqtl_in_group_cpeak500kb_combined.pdf"), error = FALSE)
```


### Signal change after merging

- Compare to the original results with results of merging single peaks

- (Minor) Compare window sizes under the same maximum merging window size


Below I show results for tissue Brian. 

```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for tissue Brian.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for tissue Brian.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/plt_cpeak_cwindow_H3K27AC_cis500kb_prop_combined.pdf"), error = FALSE)
```


## 2. Combine p-values from two tests

### Zhonghua's idea not work

FDR on window and single peak pvalues separately not work, as the total number of tests in the single-peak-test pool decreased. 

FDR on the window and single peak pvalues together makes more sense. 

- Why XY's idea of a/b=c/d=a+b/c+d=\alpha not work?
  
  The discoveries (c and d) and true positives in these discoveries would change, as the tests included change, i.e. changing from all windows (all peaks) to all windows plus isolated peaks. 


### How simulation?

100k overall tests, including -

- uni test (z-score), to simulate the single left out peaks. 
  
  NULL - $z \sim N(0, 1)$

- multi test (PCO), to simulate a group of peaks. 
  
  NULL - $ z_multi \sim N_k(0, \Sigma_{101_{DGN}})$


To look at if T1E in simulation are well calibreated, I evaluated different scenarios -

- across different proportion of uni test (z-score), the rest are pco test. 
  
  `c(0.01, 0.05, 0.1, 0.3, 0.5, 0.7, 0.9, 0.95, 0.99)`
  
- across different sigma dimension K
  
  Randomly take a subset of $\Sigma$ with dimension `c(2, 10, 40, 50, 80)`. I chose proportion of uni tests to be 50% and 20%. 



Caveats - 

Not gonna be useful for follow up analysis that require test statistics ...



### T1E in simulation - well calibrated

Calculate T1E by {#nominal P < alpha}/{#tests}. 

```{r out.width="100%", fig.cap="Figure: T1E and pvalue range across scenarios of different proportion of uni tests.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_new_fig_num_null_prop_test.pdf"), error = FALSE)
```

See [here for qq plot](asset/2024/05_02_fig_qq_null_prop_test.pdf). 

```{r out.width="100%", fig.cap="Figure: T1E and pvalue range across scenarios of different sigma dimension using 50% uni tests (Upper) and 20% uni tests (Lower).", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_new_fig_num_null_K.pdf"), error = FALSE)
```


### Simulation on FDR control

I first simulated z-scores of (SNP, peak) and (SNP, peak group) with a mixture of null and alt tests, then I perform z test and pco test, get the pvalues, perform fdr correction under a given fdr cutoff, identify the signals, calculated the false discovery rate by $\frac{\# signals not true}{\# signals}$, iterated simulation multiple times, in order to check if the false positive discovery rate is well controlled. 

**Parameter settings**

- overall tests - 10^5

- A proportion of tests are uni tests
  `prop_uni_seq <- c(0.01, 0.2, 0.5, 0.8, 0.99)`

- Among uni tests and multi tests, a proportion are casual
  
  `caus_prop_test_seq <- c(0, 0.1, 0.3, 0.5, 0.8, 1)`
  
  [NOTE - There is no point to look at 0 and 1. Explanation see below.]

- Uni test - (SNP, peak)
  
  NULL: $z \sim N(0, 1)$
  
  ALT: $z \sim N(\beta \sqrt{N}, 1), \beta \sim N(0, \sigma_b)$. $\sigma_b$ is the genetic variance explained by each SNP. 

- Multi test - (SNP, peak group)
  
  NULL: $z \sim N(0_{K}, \Sigma)$
  
  ALT: $z \sim N(B_{K} \sqrt{N}, \Sigma)$. $B_{K}$ is the effect of a SNP on K peaks in the group. I set a proportion of these K peaks (`caus_prop_K`) are causal peaks that have non zero effects. So, for zero-effect peaks $\beta_k = 0$. For non-zero-effect peaks, $\beta_k \sim N(0, \sigma_b)$ $\sigma_b$ is the genetic variance explained by each SNP. 

- single snp variance
  
  `var.b_seq <- c(0.001, 0.005, 0.01, 0.05)`

- fdr cutoff
  
  `fdr_seq <- c(1e-4, 1e-3, 1e-2, 5e-2, 1e-1)`

- sample size `N <- 100`

- causal proportion among multi K `caus_prop_K <- 0.3`

- number of simulations `ITER_sim <- 10`



**Results**

```{r out.width="100%", fig.cap="Figure: To check if FDR is well controlled across settings. x-axis is single snp variance, y-axis is average false discovery rate across 10 simualtions, colors are causal test proportion. Column panels are uni test proportion. Rows are fdr cutoff.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_fig_sim_fdr.pdf"), error = FALSE)
```


- Under 0.01 FDR cutoff, fdr are well controlled under most settings. 

- Under 0.05 FDR cutoff, fdr are well controlled under most settings, except when single snp variance is very low. This is because for most of 10 simulations, there was no signals. For 1 or 2 simulations, there was 1 signal that are not true positive. FDR will be well controlled with more simulations. 

- Some fdr are 1 under the case where no tests are causal. This is because for most of 10 simulations, there was no signals. For 1 simulation, there was 1 signal that are not true positive. FDR will be well controlled with more simulations. 

- There is no point to show FDR for cases where causal proportion tests are 0 (all NULL) or 1 (all ALT). 
  
  - When all tests are null, then there is no true signals, any signals will be false, fdr will always be 1. Should check T1E in this case. 
  
  - When all tests are alt, then any signals will be true signals, fdr will always be 0. Should check power in this case. 




### Power in real data

- How?
  
  I included those single left out single peaks in included in any windows into FDR procedure with multi tests. Specifically, I take all pairs of cis SNPs and those single peaks, take the best lead association (ACAT on all cis SNP pvalues), then use the adjusted lead pvalue of each single peaks, finally perform FDR on these lead pvalues with those of windows. 


- From ChatGPT -
  
  Consideration of power in interpretation -
  
  When interpreting the results of multiple testing correction, it's important to consider the power of the individual tests. Significant p-values surviving correction may indicate not only statistical significance but also the relative strength of evidence provided by each test. 
  
  Confidence in non-significant results: Non-significant results from tests with higher power may be more credible than those from tests with lower power. If a test has sufficient power to detect meaningful effects, the absence of a significant result may suggest weaker evidence against the null hypothesis. Conversely, non-significant results from tests with lower power may be less informative and may not necessarily provide strong evidence in favor of the null hypothesis. 
  
  Interpretation of effect sizes: Considering the power of each test can also inform the interpretation of effect sizes. Tests with higher power are more likely to detect smaller effect sizes as statistically significant. Therefore, significant results from tests with higher power may indicate not only the presence of an effect but also the magnitude of that effect, whereas significant results from tests with lower power may be more indicative of larger effect sizes. 


- Results. Take one dataset from Aracena et al. as example. 

```{r out.width="100%", fig.cap="Figure: Signal comparison with uni test and multi test by including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_ccomb_H3K27ac_Flu_window50kb_cis100kb.pdf"), error = FALSE)
```



- Compared to previous one with no single peak. 
  
  The four panel plot compared to previous one with no single peak (combined plot)
  
  - Original peak groups
  
  - Merged peak groups (2 maximum merging window)

```{r out.width="100%", fig.cap="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow0kb_cis100kb.pdf"), error = FALSE)
```

```{r out.width="100%", fig.cap="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum window size is 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow50kb_cis100kb.pdf"), error = FALSE)
```

```{r out.width="100%", fig.cap="Figure: Signal comparison before and after including single left out peaks in to FDR procedure. This is an example result for mark H3K27ac_Flu data in Aracena et al. Window size is 50kb. Merging maximum window size is 100kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_H3K27ac_Flu_maxwindow100kb_cis100kb.pdf"), error = FALSE)
```



- Changes in numbers. Across all datasets and parameters. 


```{r out.width="100%", fig.cap="Figure: Total number of cwindows.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_total_cwindows_cis100kb.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of new cwindows.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_new_cwindows_cis100kb.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of missed cpeaks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeaks_cis100kb.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of non signal windows containing cpeaks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/05_02_plt_cpeak_cwindow_ccomb_num_missed_cpeak_windows_cis100kb.pdf"), error = FALSE)
```


  - What are those lost cwindows after fdr on comb? Pvalue, corrected pvalue changes ...
  
  - What are those still significant cwindows? Pvalue, corrected pvalue changes ...
  
  - Are those new cwindows after fdr on comb?
  
  - What are those repeated cpeaks? pvalue and corrected pvalue changes ...
  
  - What are those still missed cpeaks? pvalue and corrected pvalue changes ...
  
  - Are those new cpeaks after fdr on comb?


**Observations and speculated reasons**

- Observation --> indication

- The cost of including those single peaks is to lose some weak cwindows. 


**Prove reasons and indications**

- Pattern of changes

- Example of changes


**Conclusions**

- We should use 50kb window size, 100kb maximum merging window (or not?), perform FDR on both windows and left out peaks. 







# Apr 18

## 1. Merge left out single peaks to closest peak groups

### How?

I merge a left out single peak to its closest peak group, which is within a maximum window size. This is basically to expand some peak groups to include nearby single peaks. If a peak is far from any peak group, then it is not merged and tested. 

I used two maximum merging window sizes -

- 50kb. The maximum window size can be 100kb + orignal window size. 

- 100kb. The maximum window size can be 200kb + orignal window size. 


### Analysis

I redefined peak groups with merged single peaks, of datasets in Aracena et al and GTEx. I performed CACTI on the new peak groups across -

- window sizes
  
  '50kb', '25kb', '20kb', '10kb'

- maximum merging window sizes
  
  50kb, 100kb

- marks and conditions in Aracena et al
  
  'H3K27ac', 'H3K27me3', 'H3K4me1', 'H3K4me3'
  
  'Flu', 'NI'

- tissues of mark H3K27ac in GTEx
  
  'Brain', 'Heart', 'Lung', 'Muscle'


### Comparison

I performed a series of comparisons to investigate different questions -

- Originals window v.s. extended merged window under same window size
  
  Check if merging single peaks improves power and reduces missing signals, or worse, or no change at all. 

- (Minor) Extended merged window across window sizes
  
  To check if using smaller window size is better in the case of extended merging windows, given last time we saw it is not better in data of Aracena et al. 



## 2. Aracena et al by CACTI with merged single peaks

### Datasets analyzed

- MARK_ARACENA = ['H3K27ac', 'H3K27me3', 'H3K4me1', 'H3K4me3']

- CONDITION_ARACENA = ['Flu', 'NI']

- L_WINDOW = ['50kb', '25kb', '20kb', '10kb']

- MAX_L_WINDOW = ['100kb', '50kb']


### Distance distribution from single left out peaks to peak groups

To have a general sense of how far single peaks are from already defined peak groups and what maximum merging window size we should use, I looked at -

- Distance from single left-out peak to its closest peak group

- Distance from single peak to its closest peak group


Below I only show results for Mark H3K27ac, condition Flu. 

```{r out.width="60%", fig.cap="Figure: Distribution of (A) distance of a peak to its left closest peak, and (B) distance from a left-out single peak to its closest peak group, across window sizes 10kb, 25kb, 50kb. Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window10kb.pdf", "asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window25kb.pdf", "asset/2024/04_18_peak_dis_group_H3K27ac_Flu_peak_group_window50kb.pdf"), error = FALSE)
```


**Observation -**

- 50kb and 100kb include ???% single peaks. 

- ???% are still left out and not tested. 

- If to include ~50% single peaks, use at least 50kb - 100kb. 

- Larger window size needs larger maximum window size to include more singe peaks. 

- Using the same maximum window size, smaller windows can include more singe peaks. But at the same time, more single peaks were left out already. 

- Increase maximum merging group size to include more signle peaks? To include all single peaks, the maximum window size is ???. 


As for other datasets -

```{r out.width="100%", fig.cap="Figure: Distance distribution of a peak to its left closest peak defined for marks. Two condition NI and Flu have same peak sets.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_peak_dis_to_left_peak.pdf"), error = FALSE)
```


**Observation -**

- Other marks has similar results, except mark H3K4me3 has larger peak distance. 

- Do we use larger maximum merging window size? 

- Do we use larger window size for this mark?




### Groups changes after merging

- Number of groups

- Size of groups

- Single left out peaks not merged thus not tested

```{r out.width="100%", fig.cap="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_H3K27ac_Flu_peaks_in_group_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_H3K27ac_Flu_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_H3K27ac_Flu_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf"), error = FALSE)
```

**Observations -**

- Merging single peaks into peak group increase group size. 

- Merging single peaks into peak group increase number of uni cpeaks in peak groups, which means we are able to include those left-out single cpeaks to our windows and include them into test. 



### Signal change after merging

- Compare to the original results with results of merging single peaks

- (Minor) Compare window sizes under the same maximum merging window size


Below I show results for Mark H3K27ac, condition Flu. 

```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for Mark H3K27ac, condition Flu.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_plt_cpeak_cwindow_H3K27ac_Flu_cis100kb_prop_combined.pdf"), error = FALSE)
```


**Observations -**

- Merging single peaks can decrease the missed uni cpeaks by CACTI, by including them into our test. 

- Merging single peaks can increase cwindows and new cwindows. 

- Merging helps less under larger window size, in terms of including less uni cpeaks to windows and less improvement in finding more cwindows. 

- Other marks has similar results, except mark H3K4me3, which has less improvement. 



## 3. GTEx by CACTI with merged single peaks

### Datasets analyzed

- TISSUE_TYPE = ['Brain', 'Heart', 'Lung', 'Muscle']

- L_WINDOW = ['50kb', '25kb', '20kb', '10kb']

- MAX_L_WINDOW = ['100kb', '50kb']


### Distance distribution from single left out peaks to peak groups

To have a general sense of how far single peaks are from already defined peak groups and what maximum merging window size we should use, I looked at -

- Distance from single left-out peak to its closest peak group


Below I show results across tissues (rows) and window sizes (columns). 

```{r out.width="60%", fig.cap="Figure: Distribution of distance from a left-out single peak to its closest peak group, across tissues (rows) and window sizes (columns).", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_peak_dis_to_group_gtex.pdf"), error = FALSE)
```


**Observation -**

- 50kb and 100kb include over 20% and 50% single peaks using 50kb window.

- If to include ~50% single peaks, use at least 100kb. 

- Using the same maximum window size, smaller windows can include more singe peaks. But at the same time, more single peaks were left out already. 

- Increase maximum merging group size to include more signle peaks? To include all single peaks, the maximum window size is ???. 

- No distinct difference across tissues. (though distinct difference observed across marks in Aracena et al.)



I then looked at the overall distance distribution among peaks -

- Distance of a peak to its left peak to show distance among peaks

```{r out.width="100%", fig.cap="Figure: Distance distribution of a peak to its left closest peak defined for marks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_peak_dis_to_left_peak_gtex.pdf"), error = FALSE)
```


**Observation -**

- GTEx has more H3K27ac peaks. They are closer (distance ~1kb - 10kb) than H3K27ac peaks in Aracena et al (distance ~10kb). 



### Groups changes after merging

- Number of groups

- Size of groups

- Single left out peaks not merged thus not tested


Below are results of tissue Brain. 

```{r out.width="100%", fig.cap="Figure: Distribution of peak group size across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_Brain_peaks_in_group_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_Brain_peaks_in_group_w_cqtl_cpeak100kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes and maximum merging window size. This is for tissue Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_Brain_peaks_w_cqtl_in_group_cpeak100kb_combined.pdf"), error = FALSE)
```


For other tissues, check [here for Heart](asset/2024/04_18_gourp_results_heart.pdf), [here for Lung](asset/2024/04_18_gourp_results_lung.pdf), [here for Muscle](asset/2024/04_18_gourp_results_muscle.pdf). 


**Observations -**

- Merging single peaks into peak group increase group size. 

- Merging single peaks into peak group increase number of uni cpeaks in peak groups, which means we are able to include those left-out single cpeaks to our windows and include them into test. 

- The increase of uni cpeaks in merged windows is not as much as observed in Aracena et al, where more uni cpeaks were identified. 



### Signal change after merging

- Compare to the original results with results of merging single peaks

- (Minor) Compare window sizes under the same maximum merging window size


Below I show results for tissue Brian. 

```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals across window sizes and maximum merging window size. This is for tissue Brian.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_combined.pdf"), error = FALSE)
```


```{r out.width="100%", fig.cap="Figure: Compare CACTI signals to uni signals proportions across window sizes and maximum merging window size. This is for tissue Brian.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_18_plt_cpeak_cwindow_Brain_cis100kb_prop_combined.pdf"), error = FALSE)
```


For other tissues, check [here for Heart](asset/2024/04_18_plt_cpeak_cwindow_Heart_cis100kb_combined.pdf), [here for Lung](asset/2024/04_18_plt_cpeak_cwindow_Lung_cis100kb_combined.pdf), [here for Muscle](asset/2024/04_18_plt_cpeak_cwindow_Muscle_cis100kb_combined.pdf). 

Proportions - check [here for Heart](asset/2024/04_18_plt_cpeak_cwindow_Heart_cis100kb_prop_combined.pdf), [here for Lung](asset/2024/04_18_plt_cpeak_cwindow_Lung_cis100kb_prop_combined.pdf), [here for Muscle](asset/2024/04_18_plt_cpeak_cwindow_Muscle_cis100kb_prop_combined.pdf). 


**Observations -**

- Merging single peaks can decrease the missed uni cpeaks by CACTI, by including them into our test. Although less cpeaks were included by merging than Aracena et al, as not many uni cpeaks were found in GTEx. 

- Although Merging single peaks can increase cwindows and new cwindows, it's much less than improvement seen in Aracena et al dataset. 

- Merging helps less under larger window size, in terms of including less uni cpeaks to windows and less improvement in finding more cwindows. Same as in Aracena et al dataset. 





# Apr 04 & Apr 11

## 1. Perform test on window of single peak by mixing two tests

To reduce CACTI missing signals, by reducing the purple bar of uni that do not overlap with any window, and therefore reduce the purple bar of uni that do not overlap with any CACTI cwindows. 

So we have two tests -

- Test on window of single peak

- Test on window of multiple peaks

We wanted to include the test on window of single peak in our test. A few candidate options are -


### ACAT(pco, minp)



### mixture test





## 2. Look into the 43 non-cwindows missed by CACTI (move this section to [below](#what-are-those-cacti-missing-signals))

- a genometrack by genotype plots etc

- a plot shows all the univariate pvalues





## 3. Dig into GTEx peaks and windows

Is there something weird going on in GTEx results?


### Check GTEx univariate peak pvalues distribution

**Goal -**

To make sure the pvalues are well calibrated before combining them. 


**How?**

As XY suggested - for a peak, look at the pvalue distribution, i.e. histogram and qqplot. 

Instead, inspired by the way MatrixeQTL used to visualize the associations, I looked at the distribution of all peak and SNP pairs, in terms of the histogram and qqplot. 

- Histogram

```{r out.width="20%", fig.cap="Figure: Histogram of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_04_p_hist_allchr_gtex_Brain.pdf", "asset/2024/04_04_p_hist_allchr_gtex_Heart.pdf", "asset/2024/04_04_p_hist_allchr_gtex_Muscle.pdf", "asset/2024/04_04_p_hist_allchr_gtex_Lung.pdf"), error = FALSE)
```


- qqplot

```{r out.width="20%", fig.cap="Figure: QQplot of GTEx p-values across all pairs of peaks and snps, for four tissues, Brain, Heart, Muscle, and Lung.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_04_p_qqplot_allchr_gtex_Brain.png", "asset/2024/04_04_p_qqplot_allchr_gtex_Heart.png", "asset/2024/04_04_p_qqplot_allchr_gtex_Muscle.png", "asset/2024/04_04_p_qqplot_allchr_gtex_Lung.png"), error = FALSE)
```

**Observations -**

- The power is pretty low compared to data in Aracena et al and Grubert et al (LCL). 


**Aracena et al**

Similar p-value distribution for four marks ('H3K27ac', 'H3K27me3', 'H3K4me1', 'H3K4me3') and two conditions ('Flu', "NI"). Below is an example of Mark H3K27ac, Condition Flu, Tissue primary macrophages. 

```{r out.width="20%", fig.cap="Figure: Histogram and QQplot of p-values across all pairs of peaks and snps used in Aracena et al. Mark H3K27ac, Condition Flu, Tissue primary macrophages.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_04_aracena_H3K27ac_pval_histogram.png", "asset/2024/04_04_aracena_H3K27ac_pval_qqplot.png"), error = FALSE)
```


**Grubert et al**

```{r out.width="20%", fig.cap="Figure: Histogram of p-values across all pairs of peaks and snps used in Grubert et al. Mark H3K27ac, Tissue LCL.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_04_"), error = FALSE)
```



### Compare to LCL signals

- Number of cpeaks in GTEx and LCL cwindows

- Number of weak cpeaks in LCL cwindows
  
  Use a loose p value cutoff like 0.0005, to see if on average LCL datasets have more weak signals in each window. 

- Check the correlation between peaks in both GTEx and LCL datasets




## 4. Analyze data in Aracena et al - Part 1

Their paper is [online](https://www.nature.com/articles/s41588-024-01668-z#data-availability). 


### Number of peaks (&cpeaks) across marks & conditions
  
- Number of total peaks. Number of cpeaks under 5% FDR, and 10% FDR (see [Extended Data Fig. 4a](https://www.nature.com/articles/s41588-024-01668-z/figures/11)). 

- Compare to GTEx and LCL, more cpeaks and higher cpeak proportion?

```{r out.width="45%", fig.cap="Figure: Uni cpeak signal results in Aracena et al (LEFT) and GTEx (RIGHT). (A) Number of cpeaks, (B) Proportion of cpeaks among total peaks. Across datasets, 5% FDR, and 10% FDR. X-axis is datasets in Aracena et al and GTEx. Aracena et al used 10% fdr in the paper, while GTEx used empp<0.005 in the paper. I added results using 5% fdr to both datasets. Note, GTEx results are of mark H3K27ac across four tissues, while Aracena et al analyzed several marks in primary macrophages under two conditions.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_11_signal_aracena_et_al.pdf", "asset/2024/04_11_signal_gtex_v2.pdf"), error = FALSE)
```

**Observations**

- GTEx has much more total called peaks, ~3x more. 

- GTEx has less significant cpeaks under the same 5% FDR. 

- GTEx has lower signal cpeak proportion. 



### Sample size across marks & conditions

35 individuals. 


### Covaraites they used
  
  batch and age are already accounted for in input reads. 
  
  PC1 of genotype data is the only covariate included (to account for population structure).


### Figure how their matrixeqtl works

- what is the statistics? From t-test?

- Do the statistics correspond to pvalues?

- What is the beta column? How is it generated?

- Does the statistics equal z-scores? Can their correlation be estimated from cor(residual pheno)?

- Run fastqtl and see if the number of cpeaks changes?



## 5. Analyze data in Aracena et al - Part 2

### Datasets analyzed

- MARK_ARACENA = ['H3K27ac', 'H3K27me3', 'H3K4me1', 'H3K4me3']

- CONDITION_ARACENA = ['Flu', 'NI']

- L_WINDOW = ['50kb', '25kb', '20kb', '10kb']



### Number of peak groups & group size
  
  - Compare to LCL and GTEx, fewer groups? more single-peak-group?

```{r out.width="20%", fig.cap="Figure: Distribution of peak group size across window sizes 10kb, 20kb, 25kb, 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window10kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window20kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window25kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_window50kb.pdf"), error = FALSE)
```


```{r out.width="20%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window10kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window20kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window25kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_in_group_w_cqtl_window50kb_cpeak100kb.pdf"), error = FALSE)
```



```{r out.width="20%", fig.cap="Figure: Number of uni cpeaks in peak groups across window sizes 10kb, 20kb, 25kb, 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window10kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window20kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window25kb_cpeak100kb.pdf", "asset/2024/04_11_H3K27ac_Flu_peaks_w_cqtl_in_group_window50kb_cpeak100kb.pdf"), error = FALSE)
```


### CACTI signal comparison with univairate signals

```{r out.width="20%", fig.cap="Figure: Compare CACTI signals to uni signals using FDR 0.05 across window sizes 10kb, 20kb, 25kb, 50kb.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window10kb_cis100kb.pdf", "asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window20kb_cis100kb.pdf", "asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window25kb_cis100kb.pdf", "asset/2024/04_11_plt_cpeak_cwindow_H3K27ac_Flu_window50kb_cis100kb.pdf"), error = FALSE)
```


**Observations -**

- CACTI found more signals than univariate signals. 

- Univariate signals missed by CACTI are due to not being included in any peak groups, therefore, not being tested. 

- CACTI found more signals in this dataset than in the GTEx dataset. 

- Using 50kb window seems to have better power, in terms of finding more new signals, in this dataset, whereas in GTEx 10kb window seems better. 




# Mar 28

## 1. To reduce missing signals and increase new signals

### Lift fdr threshold & Narrower windows

No big difference. 

```{r out.width="30%", fig.cap="Figure: Compare CACTI signals to their signals in Brain across FDR thresholds (LEFT) and across window sizes (RIGHT).", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_28_signal_across_fdr.png", "asset/2024/03_28_signal_across_window_size.png"), error = FALSE)
```

**Observations -**

- To reduce the non-cwindows, decrease window size. 

- Compare to results of the mark in LCL using 10kb and 50kb window, is there a similar trend?

- To use 10kb in GTEx to get fewer missing cwindows. 

- To use 10kb in LCL to keep consistent to GTEx window size choice. 



## 2. What are those CACTI missing signals? {#what-are-those-cacti-missing-signals}

### General aspects

- Signal to noise ratio
  
  Replicated signals by multi have higher signal to noise ratio, i.e. smaller window size and more cpeaks by uni. 

```{r out.width="40%", fig.cap="Figure: Signal to noise ratio of uni signals missed & replicated by multi in Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_28_fig_sig_2_noise_Brain_window50kb_cis100kb.pdf"), error = FALSE)
```





# Mar 21

## 1. CACTI on GTEx multitissue ChIP datasets

### Background on datasets

- Sample sizes across tissues. 
  
  - sample size used to estimate \Sigma

- Number of peaks across tissues. 

- Covaraites across tissues. 
  
  > We inferred the latent factors for our samples in each of the four tissues by Peer45 (R package v1.0) and identified the optimum number of Peer factors (5, 5, 10 and 2 for brain, heart, muscle and lung) to correct for during haQTL calling. Additionally, we took into account age, batches for H3K27ac ChIP–seq experiment, top five genotype PCs, and those technical effects related to platform and PCR.


- Number of cpeaks across tissues (empirical p < 0.005 or FDR). 
  
  Their way to call cpeaks -
  
  > An empirical P value for each ARE was estimated based on the lead nominal P value and permutation results (--permute 1000 10000) by fitting a β distribution to account for multiple variants tested. We used an empirical P value threshold of 0.005 to identify gAREs as in a previous study, and then for each gARE, we applied the nominal P value threshold corresponding to the empirical P value of 0.005 for each locus to identify haQTLs.
  
  From [another paper of theirs](https://doi.org/10.1038/s41588-021-00890-3) that uses similar method -
  
  > To identify g-m6A sites, the nominal P value of the lead SNP of each m6A peak was corrected to get an empirical P value that accounts for the multiple variants tested. The empirical P value was extrapolated based on beta distribution fitting to the permutation results by FastQTL (parameter --permute 1,000). We applied a threshold of empirical P value<0.005 on the lead variant to identify g-m6A sites, and then for each g-m6A we applied the nominal P value threshold that corresponds to the empirical P value of 0.005 for each locus to identify m6A QTLs (Supplementary Table 2). Given the larger sample size in brain, we provided higher-confidence sets of g-m6A sites and m6A QTLs by further correcting the empirical P value with the Benjamini–Hochberg procedure and applying a threshold of FDR<0.2 (Supplementary Table 3).
  
  - There are over 500 cpeaks in lung, but the cpeaks are problematic as they correspond to fdr < 0.966. Lung has only one cpeak under FDR < 0.05. 

```{r out.width="40%", fig.cap="Figure: The equivalent FDR level to using empirical p < 0.005.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_21_signal_gtex.pdf"), error = FALSE)
```



### Distribution of peak windows

```{r out.width="40%", fig.cap="Figure: Distribution of peak group size in Brain and Lung.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_21_Brain_peaks_in_group_window50kb.pdf", "asset/2024/03_21_Lung_peaks_in_group_window50kb.pdf"), error = FALSE)
```

```{r out.width="40%", fig.cap="Figure: cPeaks defined by (1) empirical p < 0.005, (2) FDR < 0.05 in peak groups in Brain and Lung.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_21_Brain_peaks_in_group_w_cqtl_window50kb.pdf", "asset/2024/03_21_Lung_peaks_in_group_w_cqtl_window50kb.pdf"), error = FALSE)
```


**Observations -**

- The difference in number of cpeaks in each peak groups is huge between using different ways to define cpeaks. 



## 2. CACTI signal comparison with univairate signals

To make a fair comparison between CACTI signals and their cpeaks, and avoid false positives at the same time, I performed multiple testing correction on the level of peaks and used fdr < 0.05 to define cwindows or cpeaks. Then I compar

```{r out.width="40%", fig.cap="Figure: Compare CACTI signals to their signals using FDR threshold of 0.05 in Brain.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_21_plt_cpeak_cwindow_Brain_window50kb_cis100kb.pdf"), error = FALSE)
```

**Observations -**

- CACTI cwindows overlap with most of the cpeaks. 

- Some CACTI cwindows are new signals containing no cpeaks. 

- Some cpeaks are missed by CACTI windows, all containing only one cpeak. 

- Similar story in Heart and Muscle. 


```{r out.width="40%", fig.cap="Figure: Compare CACTI signals to their signals using FDR threshold of 0.05 in Lung.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/03_21_plt_cpeak_cwindow_Lung_window50kb_cis100kb.pdf"), error = FALSE)
```

**Observations -**

- Only one significant cpeak under FDR < 0.05. It is replicated in a CACTI cwindow. 

- CACTI detects two other cwindows, containing no cpeaks. 



## 3. CACTI signal comparison across different aspects

### Across tissues







# Feb 22

## 1. Single cell analysis (Terminated)

### PCO on rasqual in single cells



### ACAT/SKAT combined rasqual in single cells

#### 1. lm v.s. acat_lm v.s. pco (previous results in bulk)

- lm v.s. acat_lm (to show the power of combination)

- acat_lm v.s. pco (to show overall power)


#### 2. lm v.s. acat_lm v.s. rasqual v.s. acat_rasqual (in single cell)

- lm v.s. acat_lm (to show the power of combination)

- rasqual v.s. acat_rasqual (to show the power of combination)


#### 3. try other tests in the simulation to understand the source of inflation of estimated sigma in PCO

- z test



## 2. Across tissues



## 3. Across marks




# Feb 15

## 1. Simulations to evaluate PCO on non-zscore tests

### Ways to estimate Sigma

- P-value of test statistic from a non-zscore test is converted to a z-score by inverse-normal transformation. 

- Sigma of the converted z-scores is the corresponding cov. 

- Run PCO on the converted z-scores. 


### Null simulations

- $Y$ from residualized expression matrix of $K=101$ gene module. 

- $G$ from binomial distribution, with maf being 0.2, 1k snps. 

- $G$ is independent with $Y$. 

- Run a non-zscore test, e.g. LRT, between every pairs of genes and snps. 

- Obtain chi-square test statistics and p-value. 

- Convert p to z-score by $z_{convert} = \Phi^{-1}(p)$, where $\Phi^{-1}$ is the inverse cdf of standard normal distribution. 

- Estimate $\Sigma_{convert} = cov(z_{convert})$

- Run PCO on $z_{convert}$ using $\Sigma_{convert}$. 



### PCO p-value inflation under null distribution

To look at if PCO p-values are inflated under the null simulations using $z_{convert}$ and $\Sigma_{convert}$ - 

```{r out.width="40%", fig.cap="Figure: PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/02_14_pco_nullp_int.pdf"), error = FALSE)
```


**Problem** -

- The way Zhonghua mentioned should be $\Phi^{-1}(p)$, where $\Phi$ is the cdf of standard normal distribution. However, it might not work, as while a small p is converted to a positively large z, a large p can be falsely converted to a negatively large z. Therefore, some non signals with large p can be a false signal and lead to inflation. We want both the converted test to be normally distributed and correspond to magnitude of p-values as well. 


### Other ways to convert p

I tried other ways to convert p-values to "z-scores". 

- Convert to a chi-square test, then take square root and assign signs randomly. 
- Rank-based inverse-normal transformation. 

They all have inflations as well. 


```{r out.width="40%", fig.cap="Figure: Other normal transformation. PCO p-value inflation under null distribution using z_{convert} and Sigma_{convert}.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/02_14_pco_nullp_other.pdf"), error = FALSE)
```



# Jan 25

## 1. SAIGE-QTL & SAIGE-QTL-PCO

### How does SAIGE-QTL work?

- Step 1: fitting the null Poisson mixed model

- Step 2: performing single-variant association tests

- Step 3: obtaining gene-level p-values using the ACAT test



### Questions to be confirmed before running SAIGE-QTL

- Is association test of SAIGE-QTL (score test statistic) linear to Y? It has to be so in order to calculate the correlation matrix of the tests using correlation matrix of Y's. 




### Run SAIGE-QTL

- How is the number of cpeaks by SAIGE-QTL compared to RASQUAL and scPME?


## 2. Using CRD-defined windows


## 3. Using two datasets







# Jan 18

## 1. Distance from lead SNP to cpeaks of histone marks in bulk data

```{r out.width="40%", fig.cap="Figure: cpeaks by QTLTools across cis regions.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_18_dis_leadsnp_cpeak_histone_marks_cis500kb.pdf"), error = FALSE)
```

- Change of cpeaks by QTLTools across various cis regions for bulk histone marks

```{r out.width="40%", fig.cap="Figure: Change of cpeaks by QTLTools across various cis regions for bulk histone marks.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_18_cpeak_qtltools_cis_reg_bulk.pdf"), error = FALSE)
```



## 2. Compare our cQTLs to activity cQTLs in Delaunea paper


### How aCRDQTLs were defined?

- LCL.aCRDQTLs - QTLs for the activity of CRDs in LCLs. 

- The activity of CRDs are defined as the mean activity of the peaks it contains. 

- 6157 significant aCRDs (out of 12583 total CRDs) at 5% FDR. 



### Signal comparison cwindows/cCRDs


```{r out.width="40%", fig.cap="Figure: Comparison of cwindows and aCRDQTLs.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_18_fig_cwindow_acrd_peak_group_whole_blood_window25kb_cis25kb.pdf"), error = FALSE)
```


**Observations -**

CACTI v.s. cCRDs -

- Many signals are missed by CACTI. 
  
  - Those aCRDs overlap multiple windows, while missed cwindows overlap mostly one aCRD. 

- Increase window size? 50kb, similar case. 

- Does the comparison even make sense given CRD (calculated using three histone marks by ChIP-seq) v.s. ATAC-seq?




### Enrichment of cCRDs in cwindows

**Questions to be asked -**

1. If being a cwindow more likely be a aCRDQTL?

2. Restrict to cwindow overlap with CRDs

```{r out.width="40%", fig.cap="Figure: If being a cwindow more likely be a aCRDQTL? If being a cwindow that overlaps with CRDs more likely be a aCRDQTL?", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_18_fig_cwindow_sig_crd_enrich_peak_group_whole_blood_window25kb_cis25kb.pdf"), error = FALSE)
```


**Observations -**

- cwindows are enriched for aCRDQTLs, no matter for windows overlap with a CRD or not. 

- The enrichment and OR are more significant than the enrichment of cwindows for CRDs. 


To-do's -

- Apply to previous other cases to see if there is significant enrichment. 

- Change the name of cQTLs to something like cWindowQTLs?




## 3. Another angle to explain power difference between CACTI and RASQUAL - RASQUAL mostly captures allelic effects?

- No such information in RASQUAL output. 

- Previously, I said RASQUAL uses 10kb as cis window because the SNP should be within the feature to have aligned reads overlap with the SNP. This is wrong. RASQUAL can take larger cis window outside the feature. They combined the pop effect and allelic effect for any SNPs. See their paper. 

- So, to have a fair power comparsion, we still need to think of a way to harmonize the multiple testing burden, since QTLTools have almost three times more signals at 10kb cis region than 500kb cis region. 






# Jan 11

## 0. some insights

- Why cis region used by RASQUAL is so small?
  
  Because in order to calculate the allelic effect, snps should be located inside the sequenced feature within where the feature reads are counted. Otherwise, RASQUAL effects are just between-individual signals not allele-specific signals. Therefore, RASQUAL cis regions should be the feature length on average. 


## 1. PCO using RASQUA sum stats

### Why?
  
  Current CACTI uses sum stats by QTLTools (uni-pop). We wanted to show the advantage of aggregating muliple peaks, so to fairly compare with RASQUAL which uses allelic effects, we want to use PCO on RASQUAL sum stats. 


**To-do's -**

- Check PCO on RASQUAL p-values (not from z-scores), if it's feasible. 

- Use PCO to combine RASQUAL p's. 


### Why PCO won't work to combine RASQUAL p's?

- Six tests of PCO combine p-values, not depending on the test statistics. 

- But the distribution of six tests depends on $\Sigma$, which is calculated as the correlation among residual y's, is actually the covariance matrix of the test statistic corresponding to the p-values (z-scores) across y's. The equivalence depends on the fact that the test statistic is linear to y (see equation 2 in [paper](https://onlinelibrary-wiley-com.proxy.uchicago.edu/doi/full/10.1111/biom.12735). 

- The issue is RASQUAL test statistic is not linear to y, which make the calculation of $\Sigma$ impossible. RASQUAL p-value for a pair of snp and feature comes from LRT testing for $\pi=0.5$, where $\pi$ is the metric for cis regulatory effect parameter (see "Hypothesis testing for inference of QTLs" in [paper](https://www.nature.com/articles/ng.3467#Sec13)). We need to know the correlation among LRT test statistics of a snp across peaks, but we can't. It is difference case than where the test statistic is OLS estimate and we can easily derive the correlations. 

> Under the null hypothesis, all parameters except π are estimated for each feature independently, whereas π is set to 0.5, and we use a likelihood-ratio test to compare the null and alternative hypotheses for each SNP-feature combination using the $\chi^2$ distribution with 1 degree of freedom (for $\pi$). 




## 2. Effect of the choice of cis region on #cpeaks by QTLTools

```{r out.width="40%", fig.cap="Figure: cpeaks by QTLTools across cis regions.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_cpeak_qtltools_cis_reg.pdf"), error = FALSE)
```

- Observations
  
  - Using 500kb as cis region in previous results should have less signals than using shorter cis regions. 
  
  - Lead SNPs are very close to cpeaks across various cis regions. 



## 3. Use QTLTools cpeaks for signal-to-noise ratio plot

```{r out.width="40%", fig.cap="Figure: Signal to noise ratio of missed & replicated cwindows compared to QTLTools and RASQUAL.", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_fig_sig_2_noise_peak_group_whole_blood_window25kb_cis25kb.pdf"), error = FALSE)
```

I added the third panel, where I compared the signal to noise ratio of missed & replicated cwindows compared to RASQUAL and the number of signals (cpeaks) in a window was from QTLTools. To compare with the first panel. 


- Observations
  
  - The missed & replicated cwindows compared to RASQUAL have even fewer cpeaks in the windows by QTLTools. This indicate that RASQUAL lends power from allelic effects. 
  
  - The power difference between RASQUAL and CACTI comes from two aspects - aggregating multiple peaks and combining allelic effects. 




## 5. Characterization of CACTI signals in bulk in terms of CRD

### Rational

To check if CACTI finds cwindows of co-regulated peaks, we investigate two questions -

- Do cwindows more likely overlap with any CRDs than non-cwindows?

- Are cwindows more likely with same CRDs than non-cwindows?


I checked cwindows by CACTI-P for mark H3K27ac and CACTI-S for mark H3K36me. 

(12583 CRDs on hg38. 12433 converted to hg19. 150 CRDs deleted.)


### CACTI-P for mark H3K27ac

```{r out.width="40%", fig.cap="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_fig_cwindow_crd_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf"), error = FALSE)
```

```{r out.width="40%", fig.cap="Figure: Are cwindows more likely with same CRDs than non-cwindows?", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_fig_cwindow_crd_same_enrich_peak_group_H3K27AC_window50kb_cis500kb.pdf"), error = FALSE)
```

- Observations

  - No significant enrichment. 


### CACTI-S for mark H3K36me3

```{r out.width="40%", fig.cap="Figure: Do cwindows more likely overlap with any CRDs than non-cwindows?", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_fig_cwindow_crd_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf"), error = FALSE)
```

```{r out.width="40%", fig.cap="Figure: Are cwindows more likely with same CRDs than non-cwindows?", fig.show='hold',fig.align='center'}
knitr::include_graphics(c("asset/2024/01_11_fig_cwindow_crd_same_enrich_seg_group_H3K36ME3_window50kb_cis500kb_seg2kb_peak_overlapseg_new.pdf"), error = FALSE)
```

- Observations

  - No significant enrichment. 



## 6. Use 50kb window to be consistent with previous bulk results?

