---
title: "Log2021"
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```

## May 26

### Are trans-eQTLs cis-eQTLs?

Updated the way to define cis-eQTLs. Previously, cis-eQTL signals are gene-based (from Pheonix using fastQTL). In this section, I compare trans signals with SNP-based cis-eQTLs from two datasets, i.e. GTEx_v7 (whole blood) and DGN.

- GTEx_v7. [Here](https://www.gtexportal.org/home/datasets) provides all significant variant-gene associations.

- DGN. Two steps to obtain SNP-based significant associations.
  1. Gene-based associations. (eGenes using fastQTL, result from Pheonix)
  
  2. Find all cis-eQTLs for each gene.
  
      See "4.2 cis-eQTL mapping" described [here](https://science.sciencemag.org/content/sci/suppl/2020/09/09/369.6509.1318.DC1/aaz1776_Aguet_SM.pdf). (p.s. I checked GTEx source code and turns out that [the actual code GTEx used](https://github.com/francois-a/fastqtl/blob/master/R/calculateSignificanceFastQTL.R) is a bit different. See below for key lines.)

```{r eval=FALSE, echo=TRUE}
pt = mean(c(max(eGene$bpval), min(eGene$bpval)))
eGene$pval_nominal_threshold = signif(qbeta(pt,
                                            eGene$shape1, eGene$shape2, ncp=0, lower.tail=TRUE, log.p=FALSE), 6)
```



```{r}
res = data.frame("Dataset" = c("DGN", "eQTLGen_DGN"),
                 "#signal" = c(313, 784),
                 "GTEx_v7_is_ciseQTL(SNP-based)" = c(51, 221),
                 "GTEx_v7_cis-eGene" = c(16, 32),
                 "DGN_is_ciseQTL(SNP-based)" = c(156, 502),
                 "DGN_cis-eGene" = c(39, 83),
                 "DGN_is_cissQTL(SNP-based)" = c(101, 381),
                 "DGN_cis-sIntro" = c(71, 182),
                 "DGN_cis-e/sQTL" = c(182, 551),
                 check.names = FALSE, stringsAsFactors = FALSE)
knitr::kable(res)
```


**A diagram version:**

```{r fig.cap="Are trans-eQTLs cis-e/sQTLs?"}
knitr::include_graphics("0526-1.png", error = FALSE)
```

**Observations:**

- For DGN, among 313 trans signals, 156 (~49.8%) are also cis-eQTLs, corresponding to 39 eGenes, including:

*AC016753.7, BECN1, C17orf50, C2orf68, CFL2, CTD-2562J15.6, ERN1, FAM177A1, FIGNL1, FKRP, GGCX, HABP4, HSD17B1P1, ICAM2, KIAA0391, MAT2A, MLX, NEK6, NFE2, NFKBIA, OR2W3, PARP2, PEX12, PPP2R3C, PRKD2, PSMC3IP, RETSAT, RNF181, RP11-1094M14.8, RP11-69M1.4, RP11-85K15.2, SLC1A5, SLC2A3, SLFN12L, SYMPK, TMEM150A, TTC5, VAMP8, ZNF782*

- For DGN, 101 (~32.3%) out of 313 trans signals are cis-sQTLs, corresponding to 71 introns. 26 trans-eQTLs are only sQTLs but not eQTLs. These signals are located on chromosome 15 and 19.

- Among 313 trans signals, 182 (~58.1%) are either cis-eQTLs or cis-sQTLs.


## May 19

### How many trans-eQTLs detected in DGN/GTEx whole blood are replicated in eQTLGen?

Replication of DGN signals in eQTLGen. Based on updated results, i.e. smaller and more gene modules & PCO$_{\lambda>0.1}$.


```{r eval=FALSE}
res = data.frame("Dataset" = c("eQTLGen", "eQTLGen_DGN", "DGN"),
                 "FDR" = c("0.05/#DGN signals", "combined chr+module #10-perms", "combined chr+module #10-perms"),
                 "minp" = c(NA, 4.64e-08, 1.91e-08),
                 "(QTL, module)" = c(NA, 882, 334),
                 "unique QTL" = c(NA, 784, 313),
                 "independent QTL" = c(NA, 50, 28),
                 check.names = FALSE, stringsAsFactors = FALSE)
knitr::kable(res)
```



### Are trans-eQTLs cis-eQTLs?

1. Dataset: DGN

```{r}
res = data.frame("Dataset" = c("DGN", "eQTLGen_DGN"),
                 "#signal" = c(313, 784),
                 "is_ciseQTL(gene-based)" = c(7, 17),
                 #"cis-eGene" = c(, ),
                 "is_cissQTL(SNP-based)" = c(9, 30),
                 #"cis-sGene" = c(, ),
                 "is_cise(s)QTL(gene-based)" = c(15, 45),
                 check.names = FALSE, stringsAsFactors = FALSE)
knitr::kable(res)
```

**Observations:**

1.1 DGN

- Among 313 trans signals, 7 are also (gene-based) cis-eQTLs, corresponding to 5~7 independent SNPs. The cis-eGenes include **SLC2A3, NFE2, KIAA0391, NFKBIA, ERN1, AC016753.7, MAT2A**.

- Among 313 unique trans signals, **229 (~73%)** have a neareset gene that are eGenes (with cis-eQTLs). Among 28 independent trans signals, **19 (~68%)** have a neareset gene that are eGenes (with cis-eQTLs).

- Among 313 trans signals, 9 are also cis-sQTLs, corresponding to 6 independent SNPs.

1.2 eQTLGen_DGN

- Among 784 trans signals, 17 are also (gene-based) cis-eQTLs, corresponding to 9 independent SNPs. The cis-eGenes include **AC016753.7, CPEB4, KIAA0391, MAT2A, MRFAP1, NFKBIA, PAQR6, PWP1, RNF181, RP11-539L10.2, RP11-69M1.4, S100P, SENP7, SLC25A44, SLC2A3, SMG5, TMEM79, ZNF782**.

- Among 784 unique trans signals, **590 (~75%)** have a neareset gene that are eGenes (with cis-eQTLs). Among 50 independent trans signals, **29 (~58%)** have a neareset gene that are eGenes (with cis-eQTLs).

- Among 784 trans signals, 30 are also cis-sQTLs, corresponding to 10 independent SNPs.


2. Dataset: TCGA


3. eQTLGen?

## March 8

### Muscle

+ Less modules, PCO$_{\{\lambda: \lambda>0.1\}}$. 1 independent signal, large p.

+ More modules, PCO$_{\{\lambda: \lambda>0.1\}}$. 5 independent signal, small p.

+ More modules, PCO$_{\{\lambda: \lambda>1\}}$. 5 independent signal, relatively large p.


## March 1

### Real data analysis by PCO using more $\lambda$'s

Dataset: Muscle_Skeletal. 

Fewer (independent) signals. $PCO_{\{\lambda: \lambda>0.1\}}$ (1 signal) v.s. $PCO_{\{\lambda: \lambda>1\}}$ (3 signals).

### proteomics data

See the google doc [here][proteomics data].

[proteomics data]: https://docs.google.com/spreadsheets/d/1PizRyY3VnvTHxu3PIM2_2sLbrj0Q_mxzRC9TrIm7vi4/edit?usp=sharing


### Other stuff

+ RCC account storage


## Feb 23

## Updated simulation

1. Adjusted variance of effects

  + tests
  
    + Oracle: $\beta^T \Sigma^{-1}Z \approx \beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k: \lambda_k>0.1\}}\lambda_k u_k u_k^T) Z$
  
    + PCO: Use $\{\lambda_k: \lambda_k>0.1\}$.

  + scenerios ($K=100,K=105$)

    + N: var.b = 0.003; caus = 30%; N.seq = c(200, 400, 600, 800, 1000)
  
    + caus: var.b = 0.003; N = 500; caus.seq = c(1, 5, 10, 30, 50, 70, 100)
  
    + caus_fix: var.b.fix = 0.1; N = 500; caus.seq = c(1, 5, 10, 30, 50, 70, 100)


2. Run for other $\Sigma$'s. $K=100,K=105$

3. FDR level

The minimum null pvalue of PCO is $\ge 10^{-9}$, similar as the significance threshold we use here $10^{-9}$.

4. Results

```{r fig.cap="power under Bonferroni corretion cutoff"}
knitr::include_graphics(c("power.lambda0.1.K100.png"), error = FALSE)
```


```{r fig.cap="Another Sigma (K=105)"}
knitr::include_graphics(c("power.lambda0.1.K105.png"), error = FALSE)
```


## Feb 16

### simulation

+ tests
    
  + Oracle: $\beta^T \Sigma^{-1}Z \approx \beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k: \lambda_k>1\}}\lambda_k u_k u_k^T) Z$
  
  + PC1;
  
  + minp = $min\{p_1, \dots, p_K \} \times K$
  
  + PCO: Use $\{\lambda_k: \lambda_k>1\}$.
  
+ Multiple testing

  The significance threshold is set based on the Bofferroni correction as the $\frac{0.05}{50\times10^6} = 10^{-9}$ for univariate test and PC-based tests.
  
+ scenerios ($K=100$)

  + N: var.b = 0.01; caus = 100 (100%); N.seq = c(200, 400, 600, 800, 1000)
  
  + caus: var.b = 0.02; N = 500; caus.seq = c(1, 10, 30, 50, 70, 100)
  
  + caus_fix: var.b.fix = 0.2; N = 500; caus.seq = c(1, 10, 30, 50, 70, 100)

+ results

```{r fig.cap="power under Bonferroni corretion cutoff"}
knitr::include_graphics(c("power.lambda1.png"), error = FALSE)
```

+ Try other tests
    
  + Oracle: $\beta^T \Sigma^{-1}Z \approx \beta^T \hat\Sigma^{-1}Z = \beta^T (\sum_{\{\lambda_k: \lambda_k>0.1\}}\lambda_k u_k u_k^T) Z$
  
  + PCO: Use $\{\lambda_k: \lambda_k>0.1\}$.

```{r fig.cap="power under Bonferroni corretion cutoff"}
knitr::include_graphics(c("power.lambda0.1.png"), error = FALSE)
```

+ For talk

(This figure should be updated as for the case of 'u1', the threshold is falsely set as 0.05, which should be $10^{-9}$.)

```{r fig.cap="PC1 v.s. PCO"}
knitr::include_graphics(c("talk.png"), error = FALSE)
```


## Jan 26

### simulation

+ Methods: Oracle, PC1, PCO, MinP, minp.

+ Models: $z \sim N(\sqrt{N} \beta, \Sigma)$. $\beta \sim N(0, \sigma_{b}^2 I_{K \times K})$.

1. various sample size $N$. 200, 400, 600, 800, 1000.

2. various $\sigma_{b}^2$. 0.005, 0.01, 0.05, 0.1, 0.2.

3. various causality percentage. ???

+ simulation.
  
  Generate $10^4$ zscores under each model. $10^4$ simulations.
  
The following plot gives the boxplot, mean plot with standard deviation, and mean plot with standard error of the mean. Which one to use?

```{r fig.cap="power under cutoff 0.05"}
knitr::include_graphics(c("power_05.png"), error = FALSE)
```

```{r fig.cap=c("power comparison")}
knitr::include_graphics(c("plot.error_bar.png"), error = FALSE)
```

```{r fig.cap=c("bar plot")}
knitr::include_graphics(c("test.png"), error = FALSE)
```

### Larger FDR level, #signals in large and small module?

Here I will look at how the number of signals changes in the original large and current small modules when I increase the FDR level.

The plot below is based on Muscle tissue from GTEx_v8. The x-axis is for various FDR levels, including 5%, 10%, 15%, and 20%. The y-axis is for the number of signals under these FDR levels. I plot three types of signals, i.e. module and SNP pairs (left subplot), unique SNPs (middle subplot), independent SNPs (right subplot). The two colors represent our previous WGCNA modules (red) and current smaller WGCNA modules (blue).

We can observe (1) When increasing FDR levels, number of signals only increase a little bit, say 3-5 more signals. This means that the corrected pvalues are bipolar, i.e. either very small or very large. (2) Signals based on smaller modules are similar to that of previous standard modules.

```{r fig.cap=c("Muscle: #signals v.s.FDR", "DGN: #signals v.s.FDR")}
knitr::include_graphics(c("./signalsv.s.FDR.Muscle.png", "./signalsv.s.FDR.DGN.png"), error = FALSE)
```


### Smaller gene modules, more signals?

I changed the parameter `deepSplit`  in function `cutreeDynamic` from package WGCNA to get modules with smaller size. I run it on the Muscle tissue from GTEx (see `GTEx_v8.Muscle_Skeletal.WGCNA` in the GTEx results table). Compared to `GTEx_v8.Muscle_Skeletal`, though more modules (32 v.s. 18), they have same independent signals (3), located on chr5, chr10, and chr22.

The corresponding modules of these signals from small and large WGCNA share a large proportion of genes.

```{r}
res = data.frame("signal.Chr" = c("chr5", "chr10", "chr22"),
                 "module.small (#genes)" = paste0("module", c(21, 21, 2), c(' (89)', ' (89)', ' (297)')),
                 "module.big (#genes)" = paste0("module", c(15, 15, 4), c(' (114)', ' (114)', ' (350)')),
                 "#overlapped genes" = c(88, 88, 257),
                 check.names = FALSE)
knitr::kable(res)
```

The above table is for clusters obtained by method 'WGCNA.min20_deep2' (big) and 'WGCNA.min20_deep4' (small). Next, I give more info about the clusters by more methods, i.e. #genes in modules (A), #unclassfied genes (B), and #modules (C) by the six methods. I also give the pvalue distribution corresponding to these clusters below.

Generally, funcExplorer gives more modules, smaller module, and much more unclassified genes. Though the smaller modules, there aren't many extreme pvalues. Say, the minimum p of all is about $10^{-9}$, as compared to $10^{-14}$ by other methods.

From the pvalue plot, it looks like 'WGCNA.min10_deep4' gives smaller clusters and some extreme pvalues. So next step I will run the whole pipleline for this setting.

```{r fig.cap=c("Modules v.s. methods/parameters", "p distribution of Muscle")}
knitr::include_graphics(c("./modules.png", "./Muscle_Skeleta.p.funcExplorer.png"), error = FALSE)
```


## Jan 05, Jan 19

### Simulation

The simulations here aim to compare the power of PC-based tests including PC1 (using only the primary PC) and PCO (using combined PC's), and non-PC based test MinP. The simulations consist of two parts: (1) verify that type 1 error is well controlled; (2) compare power of tests. At this time, I simply assume there is no LD among SNPs, i.e. tests are independent.

1. Verify that type 1 error is well controlled.

  Since the tests are assumed to be independent, the null pvalues should be uniformly distributed. To verify that, I estimate the null pvalues by empirical p's obtained from the following steps.
  
  Step1. Generate $z_0 \sim N(0, \Sigma) | H_0$.
  
  Step2. Run tests including PC1, PCO, MinP for $10^6$ times and obtain $10^6$ p's for each test.
  
  Step3. Compute the empirical T1E at significance $\alpha$ by $\frac{I\{p<\alpha\}}{10^6}$. Draw qqplot of $-log_{10}p$.
  
  The correlation matrix $\Sigma$ is the correlation of genes in module15 of GTEx dataset Muscle_Skeletal (Sigma.Muscle_Skeletal.module15.chr7.rds).
  
```{r warning=FALSE}
res = read.table("data/simulation.null.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = TRUE)
res[is.na(res)] = " "
knitr::kable(res)

```

```{r fig.cap="qqplot.simulation.null"}
knitr::include_graphics("./qqplot.simulation.null.png", error = FALSE)
```


2. Compare power of tests.

  To compare the power of non-/PC-based tests, I will run the oracle test in addition to PC1, PCO, and MinP to benchmark the power. It's mentioned in the PCO paper that given a fixed correlation relationship among the phenotypes, the power of a test depends on the relationship bewteen the true effects ($\beta$) and the phenotype correlation ($\Sigma$). Therefore, to see which tests perform well in what cases, I will consider different models, including (1) $\beta=10 u_1$; (2) $\beta=4 u_k$; (3) $\beta=1.5 u_K$; (4) $\beta=rnorm(K)$; (5) $\beta=rnorm(0.7K)$; (6) $\beta=rnorm(0.3K)$.
  
  Step1. Generate $z_0 \sim N(\beta, \Sigma) | H_1$.
  
  Step2. Run tests including Oracle, PC1, PCO, MinP for $10^6$ times and obtain $10^6$ p's for each test.
  
  Step3. Compute the power at significance $\alpha$ by $\frac{I\{p<\alpha\}}{10^6}$.

```{r warning=FALSE}
res = read.table("data/simulation.alt.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = TRUE)
res[is.na(res)] = " "
knitr::kable(res)

```


### GTEx

#### Results

The table below summarizes all results I have so far, followed by figures of the distributions of pvalues in various datasets.

```{r warning=FALSE}
res = read.table("data/signals.all.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = TRUE)
res[is.na(res)] = " "
knitr::kable(res)
```


```{r fig.cap=c("DGN", "DGN_new", "Whole_Blood", "Muscle_Skeletal", "Skin_Sun_Exposed_Lower_leg", "Artery_Tibial", "Muscle_Skeletal.cross"), out.width="70%", out.height="70%"}
knitr::include_graphics(c("DGN.p.png", "DGN_new.p.png", "Whole_Blood.p.png", "Muscle_Skeletal.p.png", "Skin_Sun_Exposed_Lower_leg.p.png", "Artery_Tibial.p.png", "Muscle_Skeletal.cross.p.png"), error = FALSE)
```

#### Remarks

1. The dataset "DGN_new" represents DGN through the standard filtering (see "Gene filter" on Dec 10).

2. GTEx datasets generally have few significant signals and relatively large pvalues (compared to DGN).

3. The dataset "GTEx_v8.Muscle_Skeletal.cross" uses Muscle_Skeletal samples from GTEx_v8 (similar as GTEx_v8.Muscle_Skeletal), but without removing cross-mapped genes before constructing gene modules. Therefore, there are 13890 genes (v.s. 5430 in GTEx_v8.Muscle_Skeletal) in total which result in 39 modules (v.s. 18 in GTEx_v8.Muscle_Skeletal). We do this step because we observed that there are relatively few signals using our original pipeline and we wonder if the reason to this observation is us filtering too many genes in the first step and leaving too few signals. To check on this, we put the "filtering" to the last step, i.e. including potentially cross-mapped genes into the analysis and generate significant variant-module pairs. We then exclude those where target eGene in the module is cross-mappable with any gene within 1Mb of the variant. Hopefully we could have more signals.

    However, though the increased genes and modules, the number of identified signals (43) is similar as that using the original pipeline (38). Next, I will look into these signals.
    
```{r warning=FALSE}
res = read.table("data/signals.GTEx_v8.Muscle_Skeletal.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = FALSE, col.names = c('variant-module', "p", "q"))
res$p = as.character(format(res$p, scientific = TRUE, digits = 3)); res$q = as.character(format(res$q, scientific = TRUE, digits = 3))

knitr::kable(res, caption = 'GTEx_v8.Muscle_Skeletal')


res = read.table("data/signals.GTEx_v8.Muscle_Skeletal.cross.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = FALSE, col.names = c('variant-module', "p", "q"))
res$p = as.character(format(res$p, scientific = TRUE, digits = 3)); res$q = as.character(format(res$q, scientific = TRUE, digits = 3))

knitr::kable(res, caption = 'GTEx_v8.Muscle_Skeletal.cross')
```

    For GTEx_v8.Muscle_Skeletal, there are 38 variant-module pairs, corresponding to 2 module (module 15, module4) and 3 independent loci on (chr5, chr10, chr22). GTEx_v8.Muscle_Skeletal.cross has 43 variant-module pairs, corresponding to 1 module (module 8) and 4 independent loci on (chr1, chr5, chr16).
    
    The signal on chr5 is significant for both module 15 (114 genes) and module 8 (394 genes) in two datasets, which have 82 shared genes. Take SNP [rs2706381][rs2706381 GTEx ref] (chr5:132474927) for example. It is ["in cis with IRF1 ($P \le 2\times10^{-10}$; Fig. 6c), a transcription factor that facilitates regulation of the interferon-induced immune response"][rs2706381 GTEx ref]. It is also ["associated in trans with PSME1 ($P \le 1.1\times10^{-11}$) and PARP10 ($P \le 7.8\times10^{-10}$)"][rs2706381 GTEx ref]. These two genes are included in module 15. The reference also gives additional results to "suggest that cis-regulatory loci affecting IRF1 are regulators of interferon-responsive inflammatory processes involving genes including PSME1 and PARP10, with implications for complex traits specific to muscle tissue".
    
    I also looked at the enrichment of the genes in module 15. These genes are mainly enriched in immunity-related terms and tuberculosis. To reproduce, use the gene list [here](./gene.GTEx_v8.Muscle_Skeletal.module15.txt).
    
    SNP 10:48930105. module 15.
    
    module4:22:23508295
    
    + Check if the signals' nearby genes are cross-mapped with genes in their corresponding module. The signal on chr5 corresponding to module8 has 20 genes +/- 500Kb away from its TSS. Many of them are crosspable with genes in the module 8. The number of of genes in the module crossmapped with each of the 20 genes are: `1, 144, 0, 62, 54, 0, 63, 4, 0, 0, 0, 59, 145, 0, 0, 128, 0, 23, 53`, where the nearest gene is crossmappable with 1 gene in the module.
    
  + The other signals on chr1, chr16, and chr5:47210790 don't have any nearby genes around its TSS.

  + See the following plot for number of cross-mapped genes in each module.
    
```{r}
knitr::include_graphics('nCross.png', error = FALSE)
```


[rs2706381 GTEx ref]: https://www.nature.com/articles/nature24277#Sec9


### New TCGA by the standard filtering


### eQTLGen


1. eQTLGen description

    *This full dataset includes 19942 genes that showed expression in blood tested and 10317 SNPs that are trait-associated SNPs based on GWAS Catalog.*

    After gene filter steps described in `Dec 10`, there are 4963 genes left. Applying the same filtering to 13634 DGN genes, there are 3695 genes left, among which 3642 are also included in eQTLGen. So, I will use these genes to do the downstream analysis, e.g. constructing co-expressed gene modules. These 3642 genes result in 19 gene modules.

2. Replication of DGN signals in eQTLGen

    The table below gives the signals found in eQTLGen and DGN.

    The first two rows give results based on eQTLGen zscores, with row 1 using `qvalue` for FDR correction (threshold $0.05$) and row 2 using $\frac{0.05}{\#DGN signals}$ as significance threshold. The third row is based on the same gene modules and SNPs but tensorQTL zscores using DGN expression data. The FDR correction uses the empirical distribution of pvalues from the combined chr's and modules (10 permutations).


```{r}
res = data.frame("Dataset" = c("eQTLGen", "eQTLGen", "eQTLGen_DGN"),
                 "FDR" = c("qvalue", "0.05/#DGN signals", "combined chr+module #10-perms"),
                 "minp" = c(5.82e-04, 1.18e-04, 1.11e-07),
                 "(QTL, module)" = c(2195, 1707, 420),
                 "unique QTL" = c(909, 762, 374),
                 "independent QTL" = c(348, 286, 28),
                 check.names = FALSE, stringsAsFactors = FALSE)
knitr::kable(res)
```


Among 374 eQTLGen_DGN signals, 16 are replicated in 762 eQTLGen signals. These 16 replicated signals consists of 6 independent SNPs, including (based on GRCh37),

+ [rs12485738] : 3:56865776, intron variant of *ARHGEF3*.
+ [rs643381] :  6:139839423. ([rs590856], 6:139844429).
+ [rs149007767] 7:50370254: , intron variant of *IKZF1*.
+ [rs12718597] : 7:50428428, intron variant of *IKZF1*.
+ [rs35979828] : 12:54685880, 500B downstream variant of *NFE2*.
+ [rs7210990] : 17:16170764, intron variant of *PIGL*.

[rs12485738]: https://www.ncbi.nlm.nih.gov/snp/rs12485738
[rs643381]: https://www.ncbi.nlm.nih.gov/snp/rs643381
[rs590856]: https://www.ncbi.nlm.nih.gov/snp/rs590856
[rs149007767]: https://www.ncbi.nlm.nih.gov/snp/rs149007767
[rs12718597]: https://www.ncbi.nlm.nih.gov/snp/rs12718597
[rs35979828]: https://www.ncbi.nlm.nih.gov/snp/rs35979828
[rs7210990]: https://www.ncbi.nlm.nih.gov/snp/rs7210990


3. Replication of DGN signals in eQTLGen

```{r fig.cap=c("DGN", "DGN_new", "Whole_Blood", "Muscle_Skeletal", "Skin_Sun_Exposed_Lower_leg", "Artery_Tibial", "Muscle_Skeletal.cross"), out.width="70%", out.height="70%"}
knitr::include_graphics("./eQTLGen_DGN_venn.jpeg", error = FALSE)
```


