---
title: "Log2021"
output:
  workflowr::wflow_html:
              toc: true
              toc_depth: 4
editor_options:
  chunk_output_type: console
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo=FALSE)
```


## Jan 05, Jan 19

### GTEx

#### Results

The table below summarizes all results I have so far, followed by figures of the distributions of pvalues in various datasets.

```{r warning=FALSE}
res = read.table("data/signals.all.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = TRUE)
res[is.na(res)] = " "
knitr::kable(res)
```


```{r fig.cap=c("DGN", "DGN_new", "Whole_Blood", "Muscle_Skeletal", "Skin_Sun_Exposed_Lower_leg", "Artery_Tibial", "Muscle_Skeletal.cross"), out.width="70%", out.height="70%"}
knitr::include_graphics(c("DGN.p.png", "DGN_new.p.png", "Whole_Blood.p.png", "Muscle_Skeletal.p.png", "Skin_Sun_Exposed_Lower_leg.p.png", "Artery_Tibial.p.png", "Muscle_Skeletal.cross.p.png"), error = FALSE)
```

#### Remarks

1. The dataset "DGN_new" represents DGN through the standard filtering (see "Gene filter" on Dec 10).

2. GTEx datasets generally have few significant signals and relatively large pvalues (compared to DGN).

3. The dataset "GTEx_v8.Muscle_Skeletal.cross" uses Muscle_Skeletal samples from GTEx_v8 (similar as GTEx_v8.Muscle_Skeletal), but without removing cross-mapped genes before constructing gene modules. Therefore, there are 13890 genes (v.s. 5430 in GTEx_v8.Muscle_Skeletal) in total which result in 39 modules (v.s. 18 in GTEx_v8.Muscle_Skeletal). We do this step because we observed that there are relatively few signals using our original pipeline and we wonder if the reason to this observation is us filtering too many genes in the first step and leaving too few signals. To check on this, we put the "filtering" to the last step, i.e. including potentially cross-mapped genes into the analysis and generate significant variant-module pairs. We then exclude those where target eGene in the module is cross-mappable with any gene within 1Mb of the variant. Hopefully we could have more signals.

    However, though the increased genes and modules, the number of identified signals (43) is similar as that using the original pipeline (38). Next, I will look into these signals.
    
```{r warning=FALSE}
res = read.table("data/signals.GTEx_v8.Muscle_Skeletal.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = FALSE, col.names = c('variant-module', "p", "q"))
res$p = as.character(format(res$p, scientific = TRUE, digits = 3)); res$q = as.character(format(res$q, scientific = TRUE, digits = 3))

knitr::kable(res, caption = 'GTEx_v8.Muscle_Skeletal')


res = read.table("data/signals.GTEx_v8.Muscle_Skeletal.cross.txt", check.names = F, stringsAsFactors = F, quote = NULL, sep = "\t", header = FALSE, col.names = c('variant-module', "p", "q"))
res$p = as.character(format(res$p, scientific = TRUE, digits = 3)); res$q = as.character(format(res$q, scientific = TRUE, digits = 3))

knitr::kable(res, caption = 'GTEx_v8.Muscle_Skeletal.cross')
```

    For GTEx_v8.Muscle_Skeletal, there are 38 variant-module pairs, corresponding to 2 module (module 15, module4) and 3 independent loci on (chr5, chr10, chr22). GTEx_v8.Muscle_Skeletal.cross has 43 variant-module pairs, corresponding to 1 module (module 8) and 4 independent loci on (chr1, chr5, chr16).
    
    The signal on chr5 is significant for both module 15 (114 genes) and module 8 (394 genes) in two datasets, which have 82 shared genes. Take SNP [rs2706381][rs2706381 GTEx ref] (chr5:132474927) for example. It is ["in cis with IRF1 ($P \le 2\times10^{-10}$; Fig. 6c), a transcription factor that facilitates regulation of the interferon-induced immune response"][rs2706381 GTEx ref]. It is also ["associated in trans with PSME1 ($P \le 1.1\times10^{-11}$) and PARP10 ($P \le 7.8\times10^{-10}$)"][rs2706381 GTEx ref]. These two genes are included in module 15. The reference also gives additional results to "suggest that cis-regulatory loci affecting IRF1 are regulators of interferon-responsive inflammatory processes involving genes including PSME1 and PARP10, with implications for complex traits specific to muscle tissue".
    
    I also looked at the enrichment of the genes in module 15. These genes are mainly enriched in immunity-related terms and tuberculosis. To reproduce, use the gene list [here](./gene.GTEx_v8.Muscle_Skeletal.module15.txt).
    
    SNP 10:48930105. module 15.
    
    module4:22:23508295


[rs2706381 GTEx ref]: https://www.nature.com/articles/nature24277#Sec9


### New TCGA by the standard filtering


### eQTLGen


1. eQTLGen description

    *This full dataset includes 19942 genes that showed expression in blood tested and 10317 SNPs that are trait-associated SNPs based on GWAS Catalog.*

    After gene filter steps described in `Dec 10`, there are 4963 genes left. Applying the same filtering to 13634 DGN genes, there are 3695 genes left, among which 3642 are also included in eQTLGen. So, I will use these genes to do the downstream analysis, e.g. constructing co-expressed gene modules. These 3642 genes result in 19 gene modules.

2. Replication of DGN signals in eQTLGen

    The table below gives the signals found in eQTLGen and DGN.

    The first two rows give results based on eQTLGen zscores, with row 1 using `qvalue` for FDR correction (threshold $0.05$) and row 2 using $\frac{0.05}{\#DGN signals}$ as significance threshold. The third row is based on the same gene modules and SNPs but tensorQTL zscores using DGN expression data. The FDR correction uses the empirical distribution of pvalues from the combined chr's and modules (10 permutations).


```{r}
res = data.frame("Dataset" = c("eQTLGen", "eQTLGen", "eQTLGen_DGN"),
                 "FDR" = c("qvalue", "0.05/#DGN signals", "combined chr+module #10-perms"),
                 "minp" = c(5.82e-04, 1.18e-04, 1.11e-07),
                 "(QTL, module)" = c(2195, 1707, 420),
                 "unique QTL" = c(909, 762, 374),
                 "independent QTL" = c(348, 286, 28),
                 check.names = FALSE, stringsAsFactors = FALSE)
knitr::kable(res)
```


Among 374 eQTLGen_DGN signals, 16 are replicated in 762 eQTLGen signals. These 16 replicated signals consists of 6 independent SNPs, including (based on GRCh37),

+ [rs12485738] : 3:56865776, intron variant of *ARHGEF3*.
+ [rs643381] :  6:139839423. ([rs590856], 6:139844429).
+ [rs149007767] 7:50370254: , intron variant of *IKZF1*.
+ [rs12718597] : 7:50428428, intron variant of *IKZF1*.
+ [rs35979828] : 12:54685880, 500B downstream variant of *NFE2*.
+ [rs7210990] : 17:16170764, intron variant of *PIGL*.

[rs12485738]: https://www.ncbi.nlm.nih.gov/snp/rs12485738
[rs643381]: https://www.ncbi.nlm.nih.gov/snp/rs643381
[rs590856]: https://www.ncbi.nlm.nih.gov/snp/rs590856
[rs149007767]: https://www.ncbi.nlm.nih.gov/snp/rs149007767
[rs12718597]: https://www.ncbi.nlm.nih.gov/snp/rs12718597
[rs35979828]: https://www.ncbi.nlm.nih.gov/snp/rs35979828
[rs7210990]: https://www.ncbi.nlm.nih.gov/snp/rs7210990



