---
title: "Trans"
output:
  workflowr::wflow_html:
              toc: true
editor_options:
  chunk_output_type: console
---


```{r setup, include=FALSE}
knitr::opts_chunk$set(eval = FALSE)
```


## Outine

![](test.pdf){width=100% height=5in}

## Preprocess data

### Convert plink files of genotype to .txt

```{r code=readLines('../code/make.geno.sh')}
```


`make.geno.R`

```{r code=readLines('../code/make.geno.R')}
```

### Regress out covariates from expression data

```{r code=readLines('../code/regress_out_covariates.R')}
```


### Remove low-quality genes

Remove seudogenes, genes with mappability$<0.9$, and gene pairs with cross-mappability higher than 1.

```{r code=readLines('../code/remove.genes.R')}
```


## Coexpressed gene modules

Cluster the co-expressed genes by WGCNA.

```{r code=readLines('../code/coexpressed.gene.network.R')}
```


## Calculate null zscores

Here is the code of how I compute zscores for module $i$ and chromosome $j$, using randomly permuted samples. Calculating observed zscores (zscores based on the original real data) is similar, only need to remove line `genotype = genotype[ind.perm, ]`.

```{r code=readLines('../code/z.null.R')}
```


## Calculate pvalues

Here is the code using PCO to calculate pvalues. This uses only PC's with eigenvalue $\lambda > 1$.

```{r code=readLines('../code/p.null.R')}
```


## Multi-testing correction
### Pvalue adjustment based on uniform null pvalue
```{r code=readLines('../code/fdr.R')}
```

### Pvalue adjustment using empirical null pvalue

## Post analysis of signals

- Use all PC's. Adjust pvalues using `qvalue`.

- Use PC's with eigenvalue $\lambda>1$. Adjust pvalues using empirical null.

- Use the first PC. (elife method)

- Single-variate method. (MatrixeQTL)

## Others
### Use only the first PC. Use all PC's.

### Find phseudogenes, poor-mapped genes, and cross-mappable genes

### Run MatrixeQTL
- Prepare files

- Run MatrixeQTL
